<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Briana Mittleman" />

<meta name="date" content="2019-11-11" />

<title>Differential Splicing</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/journal.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Comparative_APA</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/brimittleman/Comparative_APA">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Differential Splicing</h1>
<h4 class="author"><em>Briana Mittleman</em></h4>
<h4 class="date"><em>11/11/2019</em></h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-12-04
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Comparative_APA/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.5.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20190902code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20190902)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20190902code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20190902)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtree558b39f968809f66fe79c9aba058a81102f0924ftargetblank558b39fa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/brimittleman/Comparative_APA/tree/558b39f968809f66fe79c9aba058a81102f0924f" target="_blank">558b39f</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtree558b39f968809f66fe79c9aba058a81102f0924ftargetblank558b39fa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    code/chimp_log/
    Ignored:    code/human_log/
    Ignored:    data/.DS_Store
    Ignored:    data/HC_filenames.txt.sb-4426323c-IKIs0S/
    Ignored:    data/RNASEQ_metadata.txt.sb-4426323c-TE4ns3/
    Ignored:    data/RNASEQ_metadata.txt.sb-51f67ae1-HXp7Gq/
    Ignored:    data/RNASEQ_metadata_2Removed.txt.sb-4426323c-a4lBwx/
    Ignored:    data/metadata_HCpanel.txt.sb-a3d92a2d-b9cYoF/
    Ignored:    data/metadata_HCpanel.txt.sb-f4823d1e-qihGek/

Untracked files:
    Untracked:  ._.DS_Store
    Untracked:  Chimp/
    Untracked:  Human/
    Untracked:  analysis/assessReadQual.Rmd
    Untracked:  code/._Config_chimp.yaml
    Untracked:  code/._Config_chimp_full.yaml
    Untracked:  code/._Config_human.yaml
    Untracked:  code/._DiffSplice.sh
    Untracked:  code/._DiffSplicePlots.sh
    Untracked:  code/._DiffSplicePlots_gencode.sh
    Untracked:  code/._DiffSplice_gencode.sh
    Untracked:  code/._DiffSplice_removebad.sh
    Untracked:  code/._GetMAPQscore.py
    Untracked:  code/._GetSecondaryMap.py
    Untracked:  code/._LiftOrthoPAS2chimp.sh
    Untracked:  code/._MapBadSamples.sh
    Untracked:  code/._MergeClusters.sh
    Untracked:  code/._QuantMergedClusters.sh
    Untracked:  code/._RunFixLeafCluster.sh
    Untracked:  code/._Snakefile
    Untracked:  code/._SnakefilePAS
    Untracked:  code/._SnakefilePASfilt
    Untracked:  code/._SortIndexBadSamples.sh
    Untracked:  code/._bed215upbed.py
    Untracked:  code/._bed2SAF_gen.py
    Untracked:  code/._buildIndecpantro5
    Untracked:  code/._buildIndecpantro5.sh
    Untracked:  code/._buildStarIndex.sh
    Untracked:  code/._chimpChromprder.sh
    Untracked:  code/._cleanbed2saf.py
    Untracked:  code/._cluster.json
    Untracked:  code/._cluster2bed.py
    Untracked:  code/._clusterLiftReverse.sh
    Untracked:  code/._clusterLiftReverse_removebad.sh
    Untracked:  code/._clusterLiftprimary.sh
    Untracked:  code/._clusterLiftprimary_removebad.sh
    Untracked:  code/._converBam2Junc.sh
    Untracked:  code/._converBam2Junc_removeBad.sh
    Untracked:  code/._extraSnakefiltpas
    Untracked:  code/._filter5percPAS.py
    Untracked:  code/._filterNumChroms.py
    Untracked:  code/._filterPASforMP.py
    Untracked:  code/._filterPostLift.py
    Untracked:  code/._fixExonFC.py
    Untracked:  code/._fixLeafCluster.py
    Untracked:  code/._fixLiftedJunc.py
    Untracked:  code/._fixUTRexonanno.py
    Untracked:  code/._formathg38Anno.py
    Untracked:  code/._formatpantro6Anno.py
    Untracked:  code/._getRNAseqMapStats.sh
    Untracked:  code/._hg19MapStats.sh
    Untracked:  code/._humanChromorder.sh
    Untracked:  code/._intersectLiftedPAS.sh
    Untracked:  code/._liftJunctionFiles.sh
    Untracked:  code/._liftPAS19to38.sh
    Untracked:  code/._liftedchimpJunc2human.sh
    Untracked:  code/._makeSamplyGroupsHuman_TvN.py
    Untracked:  code/._mapRNAseqhg19.sh
    Untracked:  code/._mapRNAseqhg19_newPipeline.sh
    Untracked:  code/._maphg19.sh
    Untracked:  code/._maphg19_subjunc.sh
    Untracked:  code/._mergedBam2BW.sh
    Untracked:  code/._nameClusters.py
    Untracked:  code/._numMultimap.py
    Untracked:  code/._overlapapaQTLPAS.sh
    Untracked:  code/._prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/._preparePAS4lift.py
    Untracked:  code/._primaryLift.sh
    Untracked:  code/._processhg38exons.py
    Untracked:  code/._quantJunc.sh
    Untracked:  code/._quantJunc_TEST.sh
    Untracked:  code/._quantJunc_removeBad.sh
    Untracked:  code/._quantMerged_seperatly.sh
    Untracked:  code/._recLiftchim2human.sh
    Untracked:  code/._revLiftPAShg38to19.sh
    Untracked:  code/._reverseLift.sh
    Untracked:  code/._runChimpDiffIso.sh
    Untracked:  code/._runFilterNumChroms.sh
    Untracked:  code/._runHumanDiffIso.sh
    Untracked:  code/._runNuclearDifffIso.sh
    Untracked:  code/._run_chimpverifybam.sh
    Untracked:  code/._run_verifyBam.sh
    Untracked:  code/._snakemake.batch
    Untracked:  code/._snakemakePAS.batch
    Untracked:  code/._snakemakePASchimp.batch
    Untracked:  code/._snakemakePAShuman.batch
    Untracked:  code/._snakemake_chimp.batch
    Untracked:  code/._snakemake_human.batch
    Untracked:  code/._snakemakefiltPAS.batch
    Untracked:  code/._snakemakefiltPAS_chimp
    Untracked:  code/._snakemakefiltPAS_chimp.sh
    Untracked:  code/._snakemakefiltPAS_human.sh
    Untracked:  code/._submit-snakemake-chimp.sh
    Untracked:  code/._submit-snakemake-human.sh
    Untracked:  code/._submit-snakemakePAS-chimp.sh
    Untracked:  code/._submit-snakemakePAS-human.sh
    Untracked:  code/._submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/._submit-snakemakefiltPAS-human.sh
    Untracked:  code/._subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/._transcriptDTplotsNuclear.sh
    Untracked:  code/._verifyBam4973.sh
    Untracked:  code/._verifyBam4973inHuman.sh
    Untracked:  code/._wrap_chimpverifybam.sh
    Untracked:  code/._wrap_verifyBam.sh
    Untracked:  code/._writeMergecode.py
    Untracked:  code/.snakemake/
    Untracked:  code/Config_chimp.yaml
    Untracked:  code/Config_chimp_full.yaml
    Untracked:  code/Config_human.yaml
    Untracked:  code/DiffSplice.err
    Untracked:  code/DiffSplice.out
    Untracked:  code/DiffSplice.sh
    Untracked:  code/DiffSplicePlots.err
    Untracked:  code/DiffSplicePlots.out
    Untracked:  code/DiffSplicePlots.sh
    Untracked:  code/DiffSplicePlots_gencode.sh
    Untracked:  code/DiffSplice_gencode.sh
    Untracked:  code/DiffSplice_removebad.err
    Untracked:  code/DiffSplice_removebad.out
    Untracked:  code/DiffSplice_removebad.sh
    Untracked:  code/GencodeDiffSplice.err
    Untracked:  code/GencodeDiffSplice.out
    Untracked:  code/GetMAPQscore.py
    Untracked:  code/GetSecondaryMap.py
    Untracked:  code/HchromOrder.err
    Untracked:  code/HchromOrder.out
    Untracked:  code/JunctionLift.err
    Untracked:  code/JunctionLift.out
    Untracked:  code/LiftClustersFirst.err
    Untracked:  code/LiftClustersFirst.out
    Untracked:  code/LiftClustersFirst_remove.err
    Untracked:  code/LiftClustersFirst_remove.out
    Untracked:  code/LiftClustersSecond.err
    Untracked:  code/LiftClustersSecond.out
    Untracked:  code/LiftClustersSecond_remove.err
    Untracked:  code/LiftClustersSecond_remove.out
    Untracked:  code/LiftOrthoPAS2chimp.sh
    Untracked:  code/LiftorthoPAS.err
    Untracked:  code/LiftorthoPASt.out
    Untracked:  code/Log.out
    Untracked:  code/MapBadSamples.err
    Untracked:  code/MapBadSamples.out
    Untracked:  code/MapBadSamples.sh
    Untracked:  code/MapStats.err
    Untracked:  code/MapStats.out
    Untracked:  code/MergeClusters.err
    Untracked:  code/MergeClusters.out
    Untracked:  code/MergeClusters.sh
    Untracked:  code/QuantMergeClusters
    Untracked:  code/QuantMergeClusters.err
    Untracked:  code/QuantMergeClusters.out
    Untracked:  code/QuantMergedClusters.sh
    Untracked:  code/Rev_liftoverPAShg19to38.err
    Untracked:  code/Rev_liftoverPAShg19to38.out
    Untracked:  code/RunFixCluster.err
    Untracked:  code/RunFixCluster.out
    Untracked:  code/RunFixLeafCluster.sh
    Untracked:  code/SAF215upbed_gen.py
    Untracked:  code/Snakefile
    Untracked:  code/SnakefilePAS
    Untracked:  code/SnakefilePASfilt
    Untracked:  code/SortIndexBadSamples.err
    Untracked:  code/SortIndexBadSamples.out
    Untracked:  code/SortIndexBadSamples.sh
    Untracked:  code/TotalTranscriptDTplot.err
    Untracked:  code/TotalTranscriptDTplot.out
    Untracked:  code/Upstream10Bases_general.py
    Untracked:  code/apaQTLsnake.err
    Untracked:  code/apaQTLsnake.out
    Untracked:  code/apaQTLsnakePAS.err
    Untracked:  code/apaQTLsnakePAS.out
    Untracked:  code/apaQTLsnakePAShuman.err
    Untracked:  code/bam2junc.err
    Untracked:  code/bam2junc.out
    Untracked:  code/bam2junc_remove.err
    Untracked:  code/bam2junc_remove.out
    Untracked:  code/bed215upbed.py
    Untracked:  code/bed2SAF_gen.py
    Untracked:  code/bed2saf.py
    Untracked:  code/bg_to_cov.py
    Untracked:  code/buildIndecpantro5
    Untracked:  code/buildIndecpantro5.sh
    Untracked:  code/buildStarIndex.sh
    Untracked:  code/callPeaksYL.py
    Untracked:  code/chimpChromprder.sh
    Untracked:  code/chooseAnno2Bed.py
    Untracked:  code/chooseAnno2SAF.py
    Untracked:  code/chromOrder.err
    Untracked:  code/chromOrder.out
    Untracked:  code/cleanbed2saf.py
    Untracked:  code/cluster.json
    Untracked:  code/cluster2bed.py
    Untracked:  code/clusterLiftReverse.sh
    Untracked:  code/clusterLiftReverse_removebad.sh
    Untracked:  code/clusterLiftprimary.sh
    Untracked:  code/clusterLiftprimary_removebad.sh
    Untracked:  code/clusterPAS.json
    Untracked:  code/clusterfiltPAS.json
    Untracked:  code/comands2Mege.sh
    Untracked:  code/converBam2Junc.sh
    Untracked:  code/converBam2Junc_removeBad.sh
    Untracked:  code/convertNumeric.py
    Untracked:  code/environment.yaml
    Untracked:  code/extraSnakefiltpas
    Untracked:  code/filter5perc.R
    Untracked:  code/filter5percPAS.py
    Untracked:  code/filter5percPheno.py
    Untracked:  code/filterBamforMP.pysam2_gen.py
    Untracked:  code/filterJuncChroms.err
    Untracked:  code/filterJuncChroms.out
    Untracked:  code/filterMissprimingInNuc10_gen.py
    Untracked:  code/filterNumChroms.py
    Untracked:  code/filterPASforMP.py
    Untracked:  code/filterPostLift.py
    Untracked:  code/filterSAFforMP_gen.py
    Untracked:  code/filterSortBedbyCleanedBed_gen.R
    Untracked:  code/filterpeaks.py
    Untracked:  code/fixExonFC.py
    Untracked:  code/fixFChead.py
    Untracked:  code/fixFChead_bothfrac.py
    Untracked:  code/fixLeafCluster.py
    Untracked:  code/fixLiftedJunc.py
    Untracked:  code/fixUTRexonanno.py
    Untracked:  code/formathg38Anno.py
    Untracked:  code/generateStarIndex.err
    Untracked:  code/generateStarIndex.out
    Untracked:  code/generateStarIndexHuman.err
    Untracked:  code/generateStarIndexHuman.out
    Untracked:  code/getRNAseqMapStats.sh
    Untracked:  code/hg19MapStats.err
    Untracked:  code/hg19MapStats.out
    Untracked:  code/hg19MapStats.sh
    Untracked:  code/humanChromorder.sh
    Untracked:  code/humanFiles
    Untracked:  code/intersectAnno.err
    Untracked:  code/intersectAnno.out
    Untracked:  code/intersectLiftedPAS.sh
    Untracked:  code/leafcutter_merge_regtools_redo.py
    Untracked:  code/liftJunctionFiles.sh
    Untracked:  code/liftPAS19to38.sh
    Untracked:  code/liftoverPAShg19to38.err
    Untracked:  code/liftoverPAShg19to38.out
    Untracked:  code/log/
    Untracked:  code/make5percPeakbed.py
    Untracked:  code/makeFileID.py
    Untracked:  code/makePheno.py
    Untracked:  code/makeSamplyGroupsChimp_TvN.py
    Untracked:  code/makeSamplyGroupsHuman_TvN.py
    Untracked:  code/mapRNAseqhg19.sh
    Untracked:  code/mapRNAseqhg19_newPipeline.sh
    Untracked:  code/maphg19.err
    Untracked:  code/maphg19.out
    Untracked:  code/maphg19.sh
    Untracked:  code/maphg19_new.err
    Untracked:  code/maphg19_new.out
    Untracked:  code/maphg19_sub.err
    Untracked:  code/maphg19_sub.out
    Untracked:  code/maphg19_subjunc.sh
    Untracked:  code/merge.err
    Untracked:  code/merge_leafcutter_clusters_redo.py
    Untracked:  code/mergedBam2BW.sh
    Untracked:  code/mergedbam2bw.err
    Untracked:  code/mergedbam2bw.out
    Untracked:  code/nameClusters.py
    Untracked:  code/namePeaks.py
    Untracked:  code/nuclearTranscriptDTplot.err
    Untracked:  code/nuclearTranscriptDTplot.out
    Untracked:  code/numMultimap.py
    Untracked:  code/overlapPAS.err
    Untracked:  code/overlapPAS.out
    Untracked:  code/overlapapaQTLPAS.sh
    Untracked:  code/peak2PAS.py
    Untracked:  code/pheno2countonly.R
    Untracked:  code/prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/preparePAS4lift.py
    Untracked:  code/prepare_phenotype_table.py
    Untracked:  code/primaryLift.err
    Untracked:  code/primaryLift.out
    Untracked:  code/primaryLift.sh
    Untracked:  code/processhg38exons.py
    Untracked:  code/quantJunc.sh
    Untracked:  code/quantJunc_TEST.sh
    Untracked:  code/quantJunc_removeBad.sh
    Untracked:  code/quantLiftedPAS.err
    Untracked:  code/quantLiftedPAS.out
    Untracked:  code/quantLiftedPAS.sh
    Untracked:  code/quatJunc.err
    Untracked:  code/quatJunc.out
    Untracked:  code/recChimpback2Human.err
    Untracked:  code/recChimpback2Human.out
    Untracked:  code/recLiftchim2human.sh
    Untracked:  code/revLift.err
    Untracked:  code/revLift.out
    Untracked:  code/revLiftPAShg38to19.sh
    Untracked:  code/reverseLift.sh
    Untracked:  code/runChimpDiffIso.sh
    Untracked:  code/runFilterNumChroms.sh
    Untracked:  code/runHumanDiffIso.sh
    Untracked:  code/runNuclearDifffIso.sh
    Untracked:  code/run_Chimpleafcutter_ds.err
    Untracked:  code/run_Chimpleafcutter_ds.out
    Untracked:  code/run_Chimpverifybam.err
    Untracked:  code/run_Chimpverifybam.out
    Untracked:  code/run_Humanleafcutter_ds.err
    Untracked:  code/run_Humanleafcutter_ds.out
    Untracked:  code/run_Nuclearleafcutter_ds.err
    Untracked:  code/run_Nuclearleafcutter_ds.out
    Untracked:  code/run_chimpverifybam.sh
    Untracked:  code/run_verifyBam.sh
    Untracked:  code/run_verifybam.err
    Untracked:  code/run_verifybam.out
    Untracked:  code/slurm-62824013.out
    Untracked:  code/slurm-62825841.out
    Untracked:  code/slurm-62826116.out
    Untracked:  code/slurm-64108209.out
    Untracked:  code/slurm-64108521.out
    Untracked:  code/slurm-64108557.out
    Untracked:  code/snakePASChimp.err
    Untracked:  code/snakePASChimp.out
    Untracked:  code/snakePAShuman.out
    Untracked:  code/snakemake.batch
    Untracked:  code/snakemakeChimp.err
    Untracked:  code/snakemakeChimp.out
    Untracked:  code/snakemakeHuman.err
    Untracked:  code/snakemakeHuman.out
    Untracked:  code/snakemakePAS.batch
    Untracked:  code/snakemakePASFiltChimp.err
    Untracked:  code/snakemakePASFiltChimp.out
    Untracked:  code/snakemakePASFiltHuman.err
    Untracked:  code/snakemakePASFiltHuman.out
    Untracked:  code/snakemakePASchimp.batch
    Untracked:  code/snakemakePAShuman.batch
    Untracked:  code/snakemake_chimp.batch
    Untracked:  code/snakemake_human.batch
    Untracked:  code/snakemakefiltPAS.batch
    Untracked:  code/snakemakefiltPAS_chimp.sh
    Untracked:  code/snakemakefiltPAS_human.sh
    Untracked:  code/submit-snakemake-chimp.sh
    Untracked:  code/submit-snakemake-human.sh
    Untracked:  code/submit-snakemakePAS-chimp.sh
    Untracked:  code/submit-snakemakePAS-human.sh
    Untracked:  code/submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/submit-snakemakefiltPAS-human.sh
    Untracked:  code/subset_diffisopheno.py
    Untracked:  code/subset_diffisopheno_Chimp_tvN.py
    Untracked:  code/subset_diffisopheno_Huma_tvN.py
    Untracked:  code/subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/test
    Untracked:  code/transcriptDTplotsNuclear.sh
    Untracked:  code/transcriptDTplotsTotal.sh
    Untracked:  code/verifyBam4973.sh
    Untracked:  code/verifyBam4973inHuman.sh
    Untracked:  code/verifybam4973.err
    Untracked:  code/verifybam4973.out
    Untracked:  code/verifybam4973HumanMap.err
    Untracked:  code/verifybam4973HumanMap.out
    Untracked:  code/wrap_Chimpverifybam.err
    Untracked:  code/wrap_Chimpverifybam.out
    Untracked:  code/wrap_chimpverifybam.sh
    Untracked:  code/wrap_verifyBam.sh
    Untracked:  code/wrap_verifybam.err
    Untracked:  code/wrap_verifybam.out
    Untracked:  code/writeMergecode.py
    Untracked:  data/._.DS_Store
    Untracked:  data/._HC_filenames.txt
    Untracked:  data/._HC_filenames.txt.sb-4426323c-IKIs0S
    Untracked:  data/._HC_filenames.xlsx
    Untracked:  data/._RNASEQ_metadata.txt
    Untracked:  data/._RNASEQ_metadata.txt.sb-4426323c-TE4ns3
    Untracked:  data/._RNASEQ_metadata.txt.sb-51f67ae1-HXp7Gq
    Untracked:  data/._RNASEQ_metadata.xlsx
    Untracked:  data/._RNASEQ_metadata_2Removed.txt
    Untracked:  data/._RNASEQ_metadata_2Removed.txt.sb-4426323c-a4lBwx
    Untracked:  data/._RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/._metadata_HCpanel.txt
    Untracked:  data/._metadata_HCpanel.txt.sb-a3d92a2d-b9cYoF
    Untracked:  data/._metadata_HCpanel.txt.sb-f4823d1e-qihGek
    Untracked:  data/._metadata_HCpanel.xlsx
    Untracked:  data/._~$RNASEQ_metadata.xlsx
    Untracked:  data/._~$metadata_HCpanel.xlsx
    Untracked:  data/CompapaQTLpas/
    Untracked:  data/DTmatrix/
    Untracked:  data/DiffExpression/
    Untracked:  data/DiffIso_Nuclear/
    Untracked:  data/DiffSplice/
    Untracked:  data/DiffSplice_liftedJunc/
    Untracked:  data/DiffSplice_removeBad/
    Untracked:  data/HC_filenames.txt
    Untracked:  data/HC_filenames.xlsx
    Untracked:  data/MapStats/
    Untracked:  data/NuclearHvC/
    Untracked:  data/Peaks_5perc/
    Untracked:  data/Pheno_5perc/
    Untracked:  data/Pheno_5perc_nuclear/
    Untracked:  data/Pheno_5perc_total/
    Untracked:  data/RNASEQ_metadata.txt
    Untracked:  data/RNASEQ_metadata.xlsx
    Untracked:  data/RNASEQ_metadata_2Removed.txt
    Untracked:  data/RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/TwoBadSampleAnalysis/
    Untracked:  data/chainFiles/
    Untracked:  data/cleanPeaks_anno/
    Untracked:  data/cleanPeaks_byspecies/
    Untracked:  data/cleanPeaks_lifted/
    Untracked:  data/liftover_files/
    Untracked:  data/metadata_HCpanel.txt
    Untracked:  data/metadata_HCpanel.xlsx
    Untracked:  data/primaryLift/
    Untracked:  data/reverseLift/
    Untracked:  data/~$RNASEQ_metadata.xlsx
    Untracked:  data/~$metadata_HCpanel.xlsx
    Untracked:  output/dtPlots/
    Untracked:  projectNotes.Rmd

Unstaged changes:
    Modified:   analysis/CorrbetweenInd.Rmd
    Modified:   analysis/PASnumperSpecies.Rmd
    Modified:   analysis/annotationInfo.Rmd
    Modified:   analysis/mapStats.Rmd
    Modified:   analysis/multiMap.Rmd
    Modified:   analysis/verifyBAM.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/558b39f968809f66fe79c9aba058a81102f0924f/analysis/diffSplicing.Rmd" target="_blank">558b39f</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-04
</td>
<td>
add current error for splice and write out DE genes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/2fbf2d4e555c2aba2ffd4e7fd2e6a12aebce07e3/docs/diffSplicing.html" target="_blank">2fbf2d4</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b5df813a453dec3873b6f1d3b733f37a11f9bf33/analysis/diffSplicing.Rmd" target="_blank">b5df813</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-02
</td>
<td>
initial results no overlap
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/12a7fcb76908724d03437cc9f6d610841717b769/docs/diffSplicing.html" target="_blank">12a7fcb</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/6d6e7d59520fc06aa62bc4b88a0bd5ace38432ba/analysis/diffSplicing.Rmd" target="_blank">6d6e7d5</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-20
</td>
<td>
fix label
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/9b73cfffda5d3f6f6ca6d601049568822b016daa/docs/diffSplicing.html" target="_blank">9b73cff</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/3c2512b5d4004c0afbfd8199fcc96d47e1c54116/analysis/diffSplicing.Rmd" target="_blank">3c2512b</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-18
</td>
<td>
wflow_publish(“analysis/diffSplicing.Rmd”)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/7770d55807a69b64dd7384177560c057c816778e/docs/diffSplicing.html" target="_blank">7770d55</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/fa2c075629c56e60fcb93613c4cad55e34399bd2/analysis/diffSplicing.Rmd" target="_blank">fa2c075</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-18
</td>
<td>
code for diff splice
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/106f3c17b5d185e35bc8f6e64bea5e57547006e4/docs/diffSplicing.html" target="_blank">106f3c1</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/4ad7bcefca386f28e3414559dd08b3021b3c3e2d/analysis/diffSplicing.Rmd" target="_blank">4ad7bce</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-15
</td>
<td>
look at results of cluster lift
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/dc91b0af79367ef559ad277a01090d4c950dcc88/docs/diffSplicing.html" target="_blank">dc91b0a</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b5ba82eee24903f1da5a92da2559275ffc71912d/analysis/diffSplicing.Rmd" target="_blank">b5ba82e</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-11
</td>
<td>
add diff expression and diff splicing
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>library(tidyverse)</code></pre>
<pre><code>── Attaching packages ────────────────────────────────── tidyverse 1.2.1 ──</code></pre>
<pre><code>✔ ggplot2 3.1.1       ✔ purrr   0.3.2  
✔ tibble  2.1.1       ✔ dplyr   0.8.0.1
✔ tidyr   0.8.3       ✔ stringr 1.3.1  
✔ readr   1.3.1       ✔ forcats 0.3.0  </code></pre>
<pre><code>── Conflicts ───────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()</code></pre>
<pre class="r"><code>library(reshape2)</code></pre>
<pre><code>
Attaching package: &#39;reshape2&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:tidyr&#39;:

    smiths</code></pre>
<p>I want to use the RNA seq I collected to also perform a differential splicing analysis with leafcutter. I will follow the pipeline found at <a href="http://davidaknowles.github.io/leafcutter/articles/Usage.html" class="uri">http://davidaknowles.github.io/leafcutter/articles/Usage.html</a>. For a first pass I will use the bam files from the snakemake and differential expression analysis pipeline.</p>
<p>I will get clusters in both species then perform reciprocal liftover. I can use a liftover pipeline similar to the one I used for the differnetial PAS analysis.</p>
<p>Pipeline from example on leafcutter github.</p>
<pre class="bash"><code>

for bamfile in `ls run/geuvadis/*chr1.bam`; do
    echo Converting $bamfile to $bamfile.junc
    samtools index $bamfile
    regtools junctions extract -a 8 -m 50 -M 500000 $bamfile -o $bamfile.junc
    echo $bamfile.junc &gt;&gt; test_juncfiles.txt
done

python ../clustering/leafcutter_cluster_regtools.py -j test_juncfiles.txt -m 50 -o testYRIvsEU -l 500000</code></pre>
<p>At this point I will be able to liftover the junctions. I can use the human corrdinates for the differential splicing step.</p>
<pre class="bash"><code>../scripts/leafcutter_ds.R --num_threads 4 ../example_data/testYRIvsEU_perind_numers.counts.gz example_geuvadis/groups_file.txt
</code></pre>
<p>I now have my RNA seq for each species. I can write a script that runs the junctions for each species.</p>
<pre class="bash"><code>sbatch converBam2Junc.sh</code></pre>
<p>Create a script that only keeps the number chromosomes (2A and 2B for chimp). This means I will not have any of the chimp contigs.</p>
<p>I should lift first then filter</p>
<pre class="bash"><code>
mkdir ../data/DiffSplice_liftedJunc
sbatch liftJunctionFiles.sh
</code></pre>
<p><strong>liftover changes the junction files..</strong></p>
<p>For the humans I can simply filter the original junctions that pass the reciprocal liftover. For the chimps I need to figure out why the junctions change with liftover.</p>
<p>Junction file format from regtools: bed12 format: - chrom - start - end - junction name - score- number of reads supporing junction - strand - thick start (same as chrom start) - thick end (same as chrom end) - itemRgb - default 255,0,0 - block cound- number blocks, defauld 2 - block size - comma- separated list of block sizes, number of items in list corresponds to blockCount - block start- comma separated list of block starts, all block star positions should be calculated relative to chromStrat and the number of items in the list shoudl correspond to blockCount</p>
<p>I need to write a script that fixes the lifted files. I need to make column 9 255,0,0 and remove the commas from the last 2 columns.</p>
<p>I can impliment the fix in the filter file.</p>
<pre class="bash"><code>sbatch runFilterNumChroms.sh</code></pre>
<p>Make clusters:</p>
<pre class="r"><code>juncfiles=read.table(&quot;../data/DiffSplice_liftedJunc/BothSpec_juncfiles.txt&quot;, header = F)
humanFiles=juncfiles %&gt;% slice(1:6)
write.table(humanFiles, &quot;../data/DiffSplice_liftedJunc/Human_juncfiles.txt&quot;, quote = F, col.names = F, row.names = F)
chimpFiles=juncfiles %&gt;% slice(7:12)
write.table(chimpFiles, &quot;../data/DiffSplice_liftedJunc/Chimp_juncfiles.txt&quot;, quote = F, col.names = F, row.names = F)</code></pre>
<pre class="bash"><code>sbatch quantJunc.sh
</code></pre>
<p>Now I can merge all of the culsters with: /project2/yangili1/yangili/leafcutter_scripts/merge_leafcutter_clusters.py</p>
<pre class="bash"><code>sbatch MergeClusters.sh
sbatch QuantMergedClusters.sh</code></pre>
<p>Make the sample list:</p>
<pre class="r"><code>combinedCounts=read.table(&quot;../data/DiffSplice_liftedJunc/MergeCombined_perind_numers.counts&quot;, header=T)
x=colnames(combinedCounts)
#YG-BM-S8-18499H-Total_S8_R1_001-sort.bam
indiv=as.data.frame(x)  %&gt;%  separate(x, into=c(&quot;yg&quot;, &quot;bm&quot;,&quot;lane&quot;, &quot;sample&quot;, &quot;total&quot;,  &quot;sort&quot;, &quot;bam&quot;), sep=&quot;[.]&quot;) %&gt;% mutate(sample=paste(yg, &quot;-&quot;, bm, &quot;-&quot;, lane, &quot;-&quot;, sample, &quot;-&quot;, total, &quot;-&quot;, sort, &quot;.&quot;, bam, sep=&quot;&quot;)) %&gt;% select(sample) %&gt;%  mutate(Species=ifelse(grepl(&quot;H&quot;,sample), &quot;Human&quot;, &quot;Chimp&quot;))


write.table(indiv, &quot;../data/DiffSplice_liftedJunc/groups_file.txt&quot;, quote = F, col.names = F, row.names = F, sep = &quot;\t&quot;)</code></pre>
<p>fix to -Total instead of .Total</p>
<pre class="bash"><code>sbatch DiffSplice.sh</code></pre>
<pre class="r"><code>counts=read.table(&#39;../data/DiffSplice_liftedJunc/MergeCombined_perind_numers.counts&#39;, header=T, check.names = F)
meta=read.table(&quot;../data/DiffSplice_liftedJunc/groups_file.txt&quot;, header=F, stringsAsFactors = F)
colnames(meta)[1:2]=c(&quot;sample&quot;,&quot;group&quot;)
counts=counts[,meta$sample]
#rownames(counts)</code></pre>
<p>Error: cluster needs to be clu_###_sign ‘’’chr1:46368431:46369161:clu_386:+’’’</p>
<p>I am going to write a python work around to change the cluster format. This will take the unziped version of the counts file. When I run it, I can unzip and zip the results in the bash script.</p>
<pre class="bash"><code>
sbatch runFixLeafCluster.sh</code></pre>
<p>There are either not enough samples or min coverage problems. Let me compare these to the numbers.</p>
<pre class="r"><code>results=read.table(&quot;../data/DiffSplice_liftedJunc/MergedRes_cluster_significance.txt&quot;,stringsAsFactors = F, header = T, sep=&quot;\t&quot;) %&gt;% separate(cluster, into=c(&quot;chrom&quot;, &quot;clus&quot;),sep=&quot;:&quot;)

counts=read.table(&quot;../data/DiffSplice_liftedJunc/MergeCombined_perind.counts.fixed.gz&quot;, header = T)</code></pre>
<p>Problem is the recluster and quantify has all human and all chimp. I dont have any clusters with counts for both species…</p>
<p>##Old pipeline</p>
<p>Now I need to do reciprocal liftover with the clusters.</p>
<ul>
<li>/project2/gilad/briana/Comparative_APA/Human/data/RNAseq/DiffSplice/</li>
<li>/project2/gilad/briana/Comparative_APA/Chimp/data/RNAseq/DiffSplice/</li>
</ul>
<p>Chain files are in /data/chainFiles/ * panTro5ToHg38.over.chain<br />
* hg38ToPanTro5.over.chain</p>
<p>I first need to make bedfiles with these clusters.</p>
<p>The clusters all have _NA I dont this this is correct.</p>
<pre class="bash"><code>
&lt;!-- gunzip ../Human/data/RNAseq/DiffSplice/humanJunc_perind.counts.gz --&gt;

&lt;!-- gunzip ../Chimp/data/RNAseq/DiffSplice/chimpJunc_perind.counts.gz --&gt;


&lt;!-- python cluster2bed.py ../Human/data/RNAseq/DiffSplice/humanJunc_perind.counts ../Human/data/RNAseq/DiffSplice/humanJunc.bed --&gt;

&lt;!-- python cluster2bed.py ../Chimp/data/RNAseq/DiffSplice/chimpJunc_perind.counts ../Chimp/data/RNAseq/DiffSplice/chimpJunc.bed --&gt;


&lt;!-- #I need to name the clusters before I can do the lift. (this is like the naming in apa 1-n(clusters)) --&gt;
&lt;!-- python nameClusters.py ../Human/data/RNAseq/DiffSplice/humanJunc.bed ../Human/data/RNAseq/DiffSplice/humanJuncNamed.bed --&gt;

&lt;!-- python nameClusters.py ../Chimp/data/RNAseq/DiffSplice/chimpJunc.bed ../Chimp/data/RNAseq/DiffSplice/chimpJuncNamed.bed --&gt;

&lt;!-- sbatch clusterLiftprimary.sh --&gt;

&lt;!-- sbatch clusterLiftReverse.sh --&gt;</code></pre>
<p>Evaluate results:</p>
<p>(this code is from the lift for the PAS)</p>
<pre class="r"><code># unliftedH=read.table(&quot;../Human/data/RNAseq/DiffSplice/humanJunc_unlifted.bed&quot;,stringsAsFactors = F) %&gt;% nrow()
# unliftedC=read.table(&quot;../Chimp/data/RNAseq/DiffSplice/chimpJunc_unlifted.bed&quot;,stringsAsFactors = F) %&gt;% nrow()
# 
# liftedH=read.table(&quot;../Human/data/RNAseq/DiffSplice/humanJunc_inChimp.bed&quot;,stringsAsFactors = F) %&gt;% nrow()
# liftedC=read.table(&quot;../Chimp/data/RNAseq/DiffSplice/chimpJunc_inHuman.bed&quot;,stringsAsFactors = F) %&gt;% nrow()
# 
# primaryUnC=c(&quot;Chimp&quot;,&quot;Unlifted&quot;, unliftedC)
# primaryUnH=c(&quot;Human&quot;,&quot;Unlifted&quot;, unliftedH)
# 
# primaryLH=c(&quot;Human&quot;,&quot;Lifted&quot;, liftedH)
# primaryLC=c(&quot;Chimp&quot;,&quot;Lifted&quot;, liftedC)
# 
# header=c(&quot;species&quot;, &quot;liftStat&quot;, &quot;PAS&quot;)
# primaryDF= as.data.frame(rbind(primaryLH,primaryLC, primaryUnH,primaryUnC)) 
# colnames(primaryDF)=header
# 
# 
# primaryDF$PAS=as.numeric(as.character(primaryDF$PAS)) 
# 
# 
# 
# primaryDF= primaryDF %&gt;% group_by(species) %&gt;% mutate(nPAS=sum(PAS)) %&gt;% ungroup() %&gt;% mutate(proportion=PAS/nPAS)</code></pre>
<pre class="r"><code># ggplot(primaryDF,aes(x=species, y=PAS, fill=liftStat)) + geom_bar(stat=&quot;identity&quot;,position = &quot;dodge&quot;) + scale_fill_brewer(palette = &quot;Dark2&quot;) + labs(title=&quot;Primary Liftover Results&quot;, y=&quot;Isoforms&quot;)
# 
# ggplot(primaryDF,aes(x=species, y=proportion, fill=liftStat)) + geom_bar(stat=&quot;identity&quot;,position = &quot;dodge&quot;) + scale_fill_brewer(palette = &quot;Dark2&quot;) + labs(title=&quot;Primary Liftover Results&quot;)</code></pre>
<p>Look at the lifted:</p>
<pre class="r"><code># OriginalHuman=read.table(&quot;../Human/data/RNAseq/DiffSplice/humanJunc.bed&quot;,stringsAsFactors = F) 
# liftedHuman=read.table(&quot;../Human/data/RNAseq/DiffSplice/humanJunc_inChimp.bed&quot;,stringsAsFactors = F) 
# 
# OriginalChimp=read.table(&quot;../Chimp/data/RNAseq/DiffSplice/chimpJunc.bed&quot;,stringsAsFactors = F)
# liftedChimp=read.table(&quot;../Chimp/data/RNAseq/DiffSplice/chimpJunc_inHuman.bed&quot;,stringsAsFactors = F)</code></pre>
<p>Reverse lift:</p>
<pre class="r"><code># re_unliftedH=read.table(&quot;../Human/data/RNAseq/DiffSplice/humanJunc_inChimp_B2Human_unlifted.bed&quot;,stringsAsFactors = F) %&gt;% nrow()
# re_unliftedC=read.table(&quot;../Chimp/data/RNAseq/DiffSplice/chimpJunc_inHuman_B2Chimp_unlifted.bed&quot;,stringsAsFactors = F) %&gt;% nrow()
# 
# re_liftedH=read.table(&quot;../Human/data/RNAseq/DiffSplice/humanJunc_inChimp_B2Human.bed&quot;,stringsAsFactors = F) %&gt;% nrow()
# re_liftedC=read.table(&quot;../Chimp/data/RNAseq/DiffSplice/chimpJunc_inHuman_B2Chimp.bed&quot;,stringsAsFactors = F) %&gt;% nrow()
# 
# re_UnC=c(&quot;Chimp&quot;,&quot;Unlifted&quot;, re_unliftedC)
# re_UnH=c(&quot;Human&quot;,&quot;Unlifted&quot;, re_unliftedH)
# 
# re_LH=c(&quot;Human&quot;,&quot;Lifted&quot;, re_liftedH)
# re_LC=c(&quot;Chimp&quot;,&quot;Lifted&quot;, re_liftedC)
# 
# header=c(&quot;species&quot;, &quot;liftStat&quot;, &quot;PAS&quot;)
# re_DF= as.data.frame(rbind(re_LH,re_LC, re_UnH,re_UnC)) 
# colnames(re_DF)=header
# 
# 
# re_DF$PAS=as.numeric(as.character(re_DF$PAS)) 
# 
# 
# 
# re_DF= re_DF %&gt;% group_by(species) %&gt;% mutate(nPAS=sum(PAS)) %&gt;% ungroup() %&gt;% mutate(proportion=PAS/nPAS)</code></pre>
<pre class="r"><code># ggplot(re_DF,aes(x=species, y=PAS, fill=liftStat)) + geom_bar(stat=&quot;identity&quot;,position = &quot;dodge&quot;) + scale_fill_brewer(palette = &quot;Dark2&quot;) + labs(title=&quot;Reverse Liftover Results&quot;, y=&quot;Isoforms&quot;)
# ggplot(re_DF,aes(x=species, y=proportion, fill=liftStat)) + geom_bar(stat=&quot;identity&quot;,position = &quot;dodge&quot;) + scale_fill_brewer(palette = &quot;Dark2&quot;)+ labs(title=&quot;Reverse Liftover Results&quot;)</code></pre>
<p>How many lifted both ways?</p>
<pre class="r"><code># #human
# re_liftedH/nrow(OriginalHuman)
# #chimp
# re_liftedC/nrow(OriginalChimp)</code></pre>
<p>The next step will be to find the corresponding clusters. This is important because I will need to get the quantifications for the same introns and clusters. To do this I will need to write code that looks for the intron location from the primary lift in the reverse lift.</p>
<p>For now I will only look at those introns identified in both species. I need to do this because I need junctions we have quantifications for in both species.</p>
<p>I can make files with the human and chimp coordintats for the clusters that lift both ways. I will have to number each cluster</p>
<p>Human cluser:</p>
<pre class="r"><code># 
# humanRevlift=read.table(&quot;../Human/data/RNAseq/DiffSplice/humanJunc_inChimp_B2Human.bed&quot;,stringsAsFactors = F,col.names = c(&quot;Hchr&quot;,&quot;Hstart&quot;, &quot;Hend&quot;, &quot;cluster&quot;, &quot;score&quot;, &quot;strand&quot;)) %&gt;% select(-strand)
# 
# #number clusters
# humanRevlift %&gt;% select(cluster) %&gt;% unique() %&gt;% nrow()
# 
# humanRevlift$score=as.character(humanRevlift$score)
# humanRevlift= humanRevlift %&gt;%  mutate(Name=paste(&quot;Human&quot;, score, sep=&quot;_&quot;)) %&gt;% select(-score)
# 
# 
# humanInChimp=read.table(&quot;../Human/data/RNAseq/DiffSplice/humanJunc_inChimp.bed&quot;,stringsAsFactors = F,col.names = c(&quot;Cchr&quot;,&quot;Cstart&quot;, &quot;Cend&quot;, &quot;cluster&quot;, &quot;score&quot;, &quot;strand&quot;))%&gt;% select(-strand)
# humanInChimp$score=as.character(humanInChimp$score)
# humanInChimp= humanInChimp %&gt;%  mutate(Name=paste(&quot;Human&quot;, score, sep=&quot;_&quot;)) %&gt;% select(-score)
# 
# 
# humanliftedBoth=humanRevlift %&gt;% inner_join(humanInChimp, by=c(&quot;cluster&quot;, &quot;Name&quot;))</code></pre>
<p>Chimp clusters:</p>
<pre class="r"><code># chimpRevLift=read.table(&quot;../Chimp/data/RNAseq/DiffSplice/chimpJunc_inHuman_B2Chimp.bed&quot;,stringsAsFactors = F,col.names = c(&quot;Cchr&quot;,&quot;Cstart&quot;, &quot;Cend&quot;, &quot;cluster&quot;, &quot;score&quot;, &quot;strand&quot;)) %&gt;% select(-strand)
# chimpRevLift$score=as.character(chimpRevLift$score)
# chimpRevLift= chimpRevLift %&gt;%  mutate(Name=paste(&quot;Chimp&quot;, score, sep=&quot;_&quot;)) %&gt;% select(-score)
# chimpRevLift %&gt;% select(cluster) %&gt;% unique() %&gt;% nrow()
# 
# 
# chimpInHuman=read.table(&quot;../Chimp/data/RNAseq/DiffSplice/chimpJunc_inHuman.bed&quot;,stringsAsFactors = F,col.names = c(&quot;Hchr&quot;,&quot;Hstart&quot;, &quot;Hend&quot;, &quot;cluster&quot;, &quot;score&quot;, &quot;strand&quot;))%&gt;% select(-strand) 
# chimpInHuman$score=as.character(chimpInHuman$score)
# chimpInHuman= chimpInHuman %&gt;%  mutate(Name=paste(&quot;Chimp&quot;, score, sep=&quot;_&quot;)) %&gt;% select(-score)
# 
# chimpliftedBoth=chimpRevLift %&gt;% inner_join(chimpInHuman, by=c(&quot;cluster&quot;, &quot;Name&quot;))</code></pre>
<p>Try to join these by the human and chimp coordinates</p>
<pre class="r"><code># AllClusters=chimpliftedBoth %&gt;% inner_join(humanliftedBoth, by=c(&quot;Cchr&quot;, &quot;Cstart&quot;,&quot;Cend&quot;, &quot;Hchr&quot;, &quot;Hstart&quot;, &quot;Hend&quot;)) %&gt;% mutate(ChimpName=paste(Cchr,Cstart,Cend, cluster.x, sep=&quot;:&quot; ),HumanName=paste(Hchr,Hstart,Hend, cluster.y, sep=&quot;:&quot; ) )
# 
# nrow(AllClusters)
# 
# AllClusters %&gt;% select(cluster.x) %&gt;% unique() %&gt;% nrow()
# AllClusters %&gt;% select(cluster.y) %&gt;% unique() %&gt;% nrow()</code></pre>
<p>This means there are ~7k isoforms from about 3k genes. This is from the ~5k clusters I had before. I can move on with these using the human names.</p>
<p>I will have to go back and figure out how to call clusters for more genes.</p>
<p>I need to reformat these back into the counts format.</p>
<pre class="r"><code># #chr1:17055:17233:clu_1
# 
# AllClustersNames=AllClusters %&gt;% select(HumanName, ChimpName)
# 
# ChimpCluster=read.table(&quot;../Chimp/data/RNAseq/DiffSplice/chimpJunc_perind_numers.counts.gz&quot;) %&gt;% rownames_to_column(var=&quot;ChimpName&quot;)
# FilteredChimpCluster= ChimpCluster %&gt;% inner_join(AllClustersNames, by=&quot;ChimpName&quot;)
# 
# #map human onto these
# 
# HumanCluster=read.table(&quot;../Human/data/RNAseq/DiffSplice/humanJunc_perind_numers.counts.gz&quot;) %&gt;% rownames_to_column(var=&quot;HumanName&quot;)
# FilteredClusterBoth=HumanCluster %&gt;% inner_join(FilteredChimpCluster, by=&quot;HumanName&quot;) %&gt;% select(-ChimpName) 
# 
# FilteredClusterBothfixed=FilteredClusterBoth[!duplicated(FilteredClusterBoth$HumanName),]
# 
# 
# 
# 
# 
# 
# #create group file- this should have the name of the bams and the group
# Bams=as.data.frame(colnames(FilteredClusterBothfixed)) %&gt;% mutate(Species=ifelse(grepl(&quot;H&quot;,colnames(FilteredClusterBothfixed)), &quot;Human&quot;, &quot;Chimp&quot;)) %&gt;% slice(2:n())
# 
# #mkdir ../data/DiffSplice 
# write.table(Bams, &quot;../data/DiffSplice/groups_file.txt&quot;, col.names = F, row.names = F, quote = F, sep=&quot;\t&quot; )
# 
# 
# write.table(FilteredClusterBothfixed, &quot;../data/DiffSplice/BothSpec_perind.counts&quot;, col.names = T, row.names = F, quote = F, sep=&quot;\t&quot; )
# </code></pre>
<p>Remove the first name in header and zip the file:</p>
<p><strong>(manually)</strong></p>
<pre class="bash"><code>&lt;!-- vi ../data/DiffSplice/BothSpec_perind.counts --&gt;

&lt;!-- gzip ../data/DiffSplice/BothSpec_perind.counts --&gt;</code></pre>
<p>I will run the differential splicing analysis with the human exon file for now. Download the NCBI refseq exons from the table browser. I also downloaded the names with the common names so I can fix the exon file.</p>
<pre class="bash"><code>&lt;!-- python processhg38exons.py --&gt;
&lt;!-- gzip ../data/DiffSplice/hg38_ncbiRefseq_exonsfixed --&gt;</code></pre>
<p>Run leafcutter with python 2</p>
<pre class="bash"><code>&lt;!-- sbatch DiffSplice.sh --&gt;
</code></pre>
<p>Look at results:</p>
<pre class="r"><code># sig=read.table(&quot;../data/DiffSplice/leafcutter_ds_cluster_significance.txt&quot;,sep=&quot;\t&quot; ,header =T,stringsAsFactors = F) %&gt;% filter(status==&quot;Success&quot;) 
# 
# sig$p.adjust=as.numeric(as.character(sig$p.adjust))
# 
# 
# qqplot(-log10(runif(nrow(sig))), -log10(sig$p.adjust),ylab=&quot;-log10 Total Adjusted Leafcutter pvalue&quot;, xlab=&quot;-log 10 Uniform expectation&quot;, main=&quot;Leafcutter Differential Splicing&quot;)
# abline(0,1)
# 
# sig %&gt;% filter(p.adjust &lt; .05 ) %&gt;% nrow()</code></pre>
<p>Use the leafcutter tool to visualize</p>
<pre class="bash"><code>&lt;!-- sbatch DiffSplicePlots.sh --&gt;</code></pre>
<p>try with gencode exons:</p>
<pre class="bash"><code>&lt;!-- sbatch DiffSplice_gencode.sh --&gt;
&lt;!-- sbatch DiffSplicePlots_gencode.sh --&gt;</code></pre>
<pre class="r"><code># sig_gencode=read.table(&quot;../data/DiffSplice/Gencode__cluster_significance.txt&quot;,sep=&quot;\t&quot; ,header =T,stringsAsFactors = F) %&gt;% filter(status==&quot;Success&quot;) 
# 
# sig_gencode$p.adjust=as.numeric(as.character(sig_gencode$p.adjust))
# 
# 
# qqplot(-log10(runif(nrow(sig_gencode))), -log10(sig_gencode$p.adjust),ylab=&quot;-log10 Total Adjusted Leafcutter pvalue&quot;, xlab=&quot;-log 10 Uniform expectation&quot;, main=&quot;Leafcutter Differential Splicing Gencode exons&quot;)
# abline(0,1)
# 
# sig_gencode %&gt;% filter(p.adjust &lt; .05 ) %&gt;% nrow()</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.5.1 (2018-07-02)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] reshape2_1.4.3  forcats_0.3.0   stringr_1.3.1   dplyr_0.8.0.1  
 [5] purrr_0.3.2     readr_1.3.1     tidyr_0.8.3     tibble_2.1.1   
 [9] ggplot2_3.1.1   tidyverse_1.2.1

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.2       cellranger_1.1.0 plyr_1.8.4       compiler_3.5.1  
 [5] pillar_1.3.1     later_0.7.5      git2r_0.26.1     workflowr_1.5.0 
 [9] tools_3.5.1      digest_0.6.18    lubridate_1.7.4  jsonlite_1.6    
[13] evaluate_0.12    nlme_3.1-137     gtable_0.2.0     lattice_0.20-38 
[17] pkgconfig_2.0.2  rlang_0.4.0      cli_1.1.0        rstudioapi_0.10 
[21] yaml_2.2.0       haven_1.1.2      withr_2.1.2      xml2_1.2.0      
[25] httr_1.3.1       knitr_1.20       hms_0.4.2        generics_0.0.2  
[29] fs_1.3.1         rprojroot_1.3-2  grid_3.5.1       tidyselect_0.2.5
[33] glue_1.3.0       R6_2.3.0         readxl_1.1.0     rmarkdown_1.10  
[37] modelr_0.1.2     magrittr_1.5     whisker_0.3-2    backports_1.1.2 
[41] scales_1.0.0     promises_1.0.1   htmltools_0.3.6  rvest_0.3.2     
[45] assertthat_0.2.0 colorspace_1.3-2 httpuv_1.4.5     stringi_1.2.4   
[49] lazyeval_0.2.1   munsell_0.5.0    broom_0.5.1      crayon_1.3.4    </code></pre>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
