<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Briana Mittleman" />


<title>Use apaQTL Individuals to establish cutoffs</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/journal.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Comparative_APA</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/brimittleman/Comparative_APA">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Use apaQTL Individuals to establish cutoffs</h1>
<h4 class="author"><em>Briana Mittleman</em></h4>
<h4 class="date"><em>1/16/2020</em></h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-01-21
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Comparative_APA/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.5.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20190902code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20190902)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20190902code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20190902)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtreec27674c6d1dee0a9a99514500ecff762564723cdtargetblankc27674ca"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/brimittleman/Comparative_APA/tree/c27674c6d1dee0a9a99514500ecff762564723cd" target="_blank">c27674c</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtreec27674c6d1dee0a9a99514500ecff762564723cdtargetblankc27674ca" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    code/chimp_log/
    Ignored:    code/human_log/
    Ignored:    data/.DS_Store
    Ignored:    data/metadata_HCpanel.txt.sb-a5794dd2-i594qs/

Untracked files:
    Untracked:  ._.DS_Store
    Untracked:  Chimp/
    Untracked:  Human/
    Untracked:  analysis/CrossChimpThreePrime.Rmd
    Untracked:  analysis/DiffTransProtvsExpression.Rmd
    Untracked:  analysis/assessReadQual.Rmd
    Untracked:  analysis/diffExpressionPantro6.Rmd
    Untracked:  code/._ClassifyLeafviz.sh
    Untracked:  code/._Config_chimp.yaml
    Untracked:  code/._Config_chimp_full.yaml
    Untracked:  code/._Config_human.yaml
    Untracked:  code/._ConvertJunc2Bed.sh
    Untracked:  code/._CountNucleotides.py
    Untracked:  code/._CrossMapChimpRNA.sh
    Untracked:  code/._CrossMapThreeprime.sh
    Untracked:  code/._DiffSplice.sh
    Untracked:  code/._DiffSplicePlots.sh
    Untracked:  code/._DiffSplicePlots_gencode.sh
    Untracked:  code/._DiffSplice_gencode.sh
    Untracked:  code/._DiffSplice_removebad.sh
    Untracked:  code/._FindIntronForDomPAS.sh
    Untracked:  code/._GetMAPQscore.py
    Untracked:  code/._GetSecondaryMap.py
    Untracked:  code/._Lift5perPAS.sh
    Untracked:  code/._LiftFinalChimpJunc2Human.sh
    Untracked:  code/._LiftOrthoPAS2chimp.sh
    Untracked:  code/._MapBadSamples.sh
    Untracked:  code/._PAS_ATTAAA.sh
    Untracked:  code/._PASsequences.sh
    Untracked:  code/._PlotNuclearUsagebySpecies.R
    Untracked:  code/._QuantMergedClusters.sh
    Untracked:  code/._ReverseLiftFilter.R
    Untracked:  code/._RunFixLeafCluster.sh
    Untracked:  code/._RunNegMCMediation.sh
    Untracked:  code/._Snakefile
    Untracked:  code/._SnakefilePAS
    Untracked:  code/._SnakefilePASfilt
    Untracked:  code/._SortIndexBadSamples.sh
    Untracked:  code/._bed215upbed.py
    Untracked:  code/._bed2SAF_gen.py
    Untracked:  code/._buildIndecpantro5
    Untracked:  code/._buildIndecpantro5.sh
    Untracked:  code/._buildLeafviz.sh
    Untracked:  code/._buildLeafviz_leadAnno.sh
    Untracked:  code/._buildStarIndex.sh
    Untracked:  code/._chimpChromprder.sh
    Untracked:  code/._cleanbed2saf.py
    Untracked:  code/._cluster.json
    Untracked:  code/._cluster2bed.py
    Untracked:  code/._clusterLiftReverse.sh
    Untracked:  code/._clusterLiftReverse_removebad.sh
    Untracked:  code/._clusterLiftprimary.sh
    Untracked:  code/._clusterLiftprimary_removebad.sh
    Untracked:  code/._converBam2Junc.sh
    Untracked:  code/._converBam2Junc_removeBad.sh
    Untracked:  code/._extraSnakefiltpas
    Untracked:  code/._filter5percPAS.py
    Untracked:  code/._filterNumChroms.py
    Untracked:  code/._filterPASforMP.py
    Untracked:  code/._filterPostLift.py
    Untracked:  code/._fixExonFC.py
    Untracked:  code/._fixLeafCluster.py
    Untracked:  code/._fixLiftedJunc.py
    Untracked:  code/._fixUTRexonanno.py
    Untracked:  code/._formathg38Anno.py
    Untracked:  code/._formatpantro6Anno.py
    Untracked:  code/._getRNAseqMapStats.sh
    Untracked:  code/._hg19MapStats.sh
    Untracked:  code/._humanChromorder.sh
    Untracked:  code/._intersectLiftedPAS.sh
    Untracked:  code/._liftJunctionFiles.sh
    Untracked:  code/._liftPAS19to38.sh
    Untracked:  code/._liftedchimpJunc2human.sh
    Untracked:  code/._makeNuclearDapaplots.sh
    Untracked:  code/._makeSamplyGroupsHuman_TvN.py
    Untracked:  code/._mapRNAseqhg19.sh
    Untracked:  code/._mapRNAseqhg19_newPipeline.sh
    Untracked:  code/._maphg19.sh
    Untracked:  code/._maphg19_subjunc.sh
    Untracked:  code/._mediation_test.R
    Untracked:  code/._mergeChimp3prime_inhg38.sh
    Untracked:  code/._mergedBam2BW.sh
    Untracked:  code/._nameClusters.py
    Untracked:  code/._negativeMediation_montecarlo.R
    Untracked:  code/._numMultimap.py
    Untracked:  code/._overlapapaQTLPAS.sh
    Untracked:  code/._prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/._prepareLeafvizAnno.sh
    Untracked:  code/._preparePAS4lift.py
    Untracked:  code/._primaryLift.sh
    Untracked:  code/._processhg38exons.py
    Untracked:  code/._quantJunc.sh
    Untracked:  code/._quantJunc_TEST.sh
    Untracked:  code/._quantJunc_removeBad.sh
    Untracked:  code/._quantMerged_seperatly.sh
    Untracked:  code/._recLiftchim2human.sh
    Untracked:  code/._revLiftPAShg38to19.sh
    Untracked:  code/._reverseLift.sh
    Untracked:  code/._runCheckReverseLift.sh
    Untracked:  code/._runChimpDiffIso.sh
    Untracked:  code/._runCountNucleotides.sh
    Untracked:  code/._runFilterNumChroms.sh
    Untracked:  code/._runHumanDiffIso.sh
    Untracked:  code/._runNuclearDifffIso.sh
    Untracked:  code/._runTotalDiffIso.sh
    Untracked:  code/._run_chimpverifybam.sh
    Untracked:  code/._run_verifyBam.sh
    Untracked:  code/._snakemake.batch
    Untracked:  code/._snakemakePAS.batch
    Untracked:  code/._snakemakePASchimp.batch
    Untracked:  code/._snakemakePAShuman.batch
    Untracked:  code/._snakemake_chimp.batch
    Untracked:  code/._snakemake_human.batch
    Untracked:  code/._snakemakefiltPAS.batch
    Untracked:  code/._snakemakefiltPAS_chimp
    Untracked:  code/._snakemakefiltPAS_chimp.sh
    Untracked:  code/._snakemakefiltPAS_human.sh
    Untracked:  code/._submit-snakemake-chimp.sh
    Untracked:  code/._submit-snakemake-human.sh
    Untracked:  code/._submit-snakemakePAS-chimp.sh
    Untracked:  code/._submit-snakemakePAS-human.sh
    Untracked:  code/._submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/._submit-snakemakefiltPAS-human.sh
    Untracked:  code/._subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/._subset_diffisopheno_Total_HvC.py
    Untracked:  code/._threeprimeOrthoFC.sh
    Untracked:  code/._transcriptDTplotsNuclear.sh
    Untracked:  code/._verifyBam4973.sh
    Untracked:  code/._verifyBam4973inHuman.sh
    Untracked:  code/._wrap_chimpverifybam.sh
    Untracked:  code/._wrap_verifyBam.sh
    Untracked:  code/._writeMergecode.py
    Untracked:  code/.snakemake/
    Untracked:  code/ClassifyLeafviz.sh
    Untracked:  code/Config_chimp.yaml
    Untracked:  code/Config_chimp_full.yaml
    Untracked:  code/Config_human.yaml
    Untracked:  code/ConvertJunc2Bed.err
    Untracked:  code/ConvertJunc2Bed.out
    Untracked:  code/ConvertJunc2Bed.sh
    Untracked:  code/CountNucleotides.py
    Untracked:  code/CrossMapChimpRNA.sh
    Untracked:  code/CrossMapThreeprime.sh
    Untracked:  code/CrossmapChimp3prime.err
    Untracked:  code/CrossmapChimp3prime.out
    Untracked:  code/CrossmapChimpRNA.err
    Untracked:  code/CrossmapChimpRNA.out
    Untracked:  code/DiffSplice.err
    Untracked:  code/DiffSplice.out
    Untracked:  code/DiffSplice.sh
    Untracked:  code/DiffSplicePlots.err
    Untracked:  code/DiffSplicePlots.out
    Untracked:  code/DiffSplicePlots.sh
    Untracked:  code/DiffSplicePlots_gencode.sh
    Untracked:  code/DiffSplice_gencode.sh
    Untracked:  code/DiffSplice_removebad.err
    Untracked:  code/DiffSplice_removebad.out
    Untracked:  code/DiffSplice_removebad.sh
    Untracked:  code/FilterReverseLift.err
    Untracked:  code/FilterReverseLift.out
    Untracked:  code/FindIntronForDomPAS.err
    Untracked:  code/FindIntronForDomPAS.out
    Untracked:  code/FindIntronForDomPAS.sh
    Untracked:  code/GencodeDiffSplice.err
    Untracked:  code/GencodeDiffSplice.out
    Untracked:  code/GetMAPQscore.py
    Untracked:  code/GetSecondaryMap.py
    Untracked:  code/HchromOrder.err
    Untracked:  code/HchromOrder.out
    Untracked:  code/JunctionLift.err
    Untracked:  code/JunctionLift.out
    Untracked:  code/JunctionLiftFinalChimp.err
    Untracked:  code/JunctionLiftFinalChimp.out
    Untracked:  code/Lift5perPAS.sh
    Untracked:  code/Lift5perPASbed.err
    Untracked:  code/Lift5perPASbed.out
    Untracked:  code/LiftClustersFirst.err
    Untracked:  code/LiftClustersFirst.out
    Untracked:  code/LiftClustersFirst_remove.err
    Untracked:  code/LiftClustersFirst_remove.out
    Untracked:  code/LiftClustersSecond.err
    Untracked:  code/LiftClustersSecond.out
    Untracked:  code/LiftClustersSecond_remove.err
    Untracked:  code/LiftClustersSecond_remove.out
    Untracked:  code/LiftFinalChimpJunc2Human.sh
    Untracked:  code/LiftOrthoPAS2chimp.sh
    Untracked:  code/LiftorthoPAS.err
    Untracked:  code/LiftorthoPASt.out
    Untracked:  code/Log.out
    Untracked:  code/MapBadSamples.err
    Untracked:  code/MapBadSamples.out
    Untracked:  code/MapBadSamples.sh
    Untracked:  code/MapStats.err
    Untracked:  code/MapStats.out
    Untracked:  code/MergeClusters.err
    Untracked:  code/MergeClusters.out
    Untracked:  code/MergeClusters.sh
    Untracked:  code/PAS_ATTAAA.err
    Untracked:  code/PAS_ATTAAA.out
    Untracked:  code/PAS_ATTAAA.sh
    Untracked:  code/PAS_sequence.err
    Untracked:  code/PAS_sequence.out
    Untracked:  code/PASsequences.sh
    Untracked:  code/PlotNuclearUsagebySpecies.R
    Untracked:  code/QuantMergeClusters
    Untracked:  code/QuantMergeClusters.err
    Untracked:  code/QuantMergeClusters.out
    Untracked:  code/QuantMergedClusters.sh
    Untracked:  code/Rev_liftoverPAShg19to38.err
    Untracked:  code/Rev_liftoverPAShg19to38.out
    Untracked:  code/ReverseLiftFilter.R
    Untracked:  code/RunFixCluster.err
    Untracked:  code/RunFixCluster.out
    Untracked:  code/RunFixLeafCluster.sh
    Untracked:  code/RunNegMCMediation.err
    Untracked:  code/RunNegMCMediation.sh
    Untracked:  code/RunNegMCMediationr.out
    Untracked:  code/RunPosMCMediation.err
    Untracked:  code/RunPosMCMediation.sh
    Untracked:  code/RunPosMCMediationr.out
    Untracked:  code/SAF215upbed_gen.py
    Untracked:  code/Snakefile
    Untracked:  code/SnakefilePAS
    Untracked:  code/SnakefilePASfilt
    Untracked:  code/SortIndexBadSamples.err
    Untracked:  code/SortIndexBadSamples.out
    Untracked:  code/SortIndexBadSamples.sh
    Untracked:  code/TotalTranscriptDTplot.err
    Untracked:  code/TotalTranscriptDTplot.out
    Untracked:  code/Upstream10Bases_general.py
    Untracked:  code/apaQTLsnake.err
    Untracked:  code/apaQTLsnake.out
    Untracked:  code/apaQTLsnakePAS.err
    Untracked:  code/apaQTLsnakePAS.out
    Untracked:  code/apaQTLsnakePAShuman.err
    Untracked:  code/bam2junc.err
    Untracked:  code/bam2junc.out
    Untracked:  code/bam2junc_remove.err
    Untracked:  code/bam2junc_remove.out
    Untracked:  code/bed215upbed.py
    Untracked:  code/bed2SAF_gen.py
    Untracked:  code/bed2saf.py
    Untracked:  code/bg_to_cov.py
    Untracked:  code/buildIndecpantro5
    Untracked:  code/buildIndecpantro5.sh
    Untracked:  code/buildLeafviz.err
    Untracked:  code/buildLeafviz.out
    Untracked:  code/buildLeafviz.sh
    Untracked:  code/buildLeafviz_leadAnno.sh
    Untracked:  code/buildLeafviz_leafanno.err
    Untracked:  code/buildLeafviz_leafanno.out
    Untracked:  code/buildStarIndex.sh
    Untracked:  code/callPeaksYL.py
    Untracked:  code/chimpChromprder.sh
    Untracked:  code/chooseAnno2Bed.py
    Untracked:  code/chooseAnno2SAF.py
    Untracked:  code/chromOrder.err
    Untracked:  code/chromOrder.out
    Untracked:  code/classifyLeafviz.err
    Untracked:  code/classifyLeafviz.out
    Untracked:  code/cleanbed2saf.py
    Untracked:  code/cluster.json
    Untracked:  code/cluster2bed.py
    Untracked:  code/clusterLiftReverse.sh
    Untracked:  code/clusterLiftReverse_removebad.sh
    Untracked:  code/clusterLiftprimary.sh
    Untracked:  code/clusterLiftprimary_removebad.sh
    Untracked:  code/clusterPAS.json
    Untracked:  code/clusterfiltPAS.json
    Untracked:  code/comands2Mege.sh
    Untracked:  code/converBam2Junc.sh
    Untracked:  code/converBam2Junc_removeBad.sh
    Untracked:  code/convertNumeric.py
    Untracked:  code/environment.yaml
    Untracked:  code/extraSnakefiltpas
    Untracked:  code/filter5perc.R
    Untracked:  code/filter5percPAS.py
    Untracked:  code/filter5percPheno.py
    Untracked:  code/filterBamforMP.pysam2_gen.py
    Untracked:  code/filterJuncChroms.err
    Untracked:  code/filterJuncChroms.out
    Untracked:  code/filterMissprimingInNuc10_gen.py
    Untracked:  code/filterNumChroms.py
    Untracked:  code/filterPASforMP.py
    Untracked:  code/filterPostLift.py
    Untracked:  code/filterSAFforMP_gen.py
    Untracked:  code/filterSortBedbyCleanedBed_gen.R
    Untracked:  code/filterpeaks.py
    Untracked:  code/fixExonFC.py
    Untracked:  code/fixFChead.py
    Untracked:  code/fixFChead_bothfrac.py
    Untracked:  code/fixLeafCluster.py
    Untracked:  code/fixLiftedJunc.py
    Untracked:  code/fixUTRexonanno.py
    Untracked:  code/formathg38Anno.py
    Untracked:  code/generateStarIndex.err
    Untracked:  code/generateStarIndex.out
    Untracked:  code/generateStarIndexHuman.err
    Untracked:  code/generateStarIndexHuman.out
    Untracked:  code/getRNAseqMapStats.sh
    Untracked:  code/hg19MapStats.err
    Untracked:  code/hg19MapStats.out
    Untracked:  code/hg19MapStats.sh
    Untracked:  code/humanChromorder.sh
    Untracked:  code/humanFiles
    Untracked:  code/intersectAnno.err
    Untracked:  code/intersectAnno.out
    Untracked:  code/intersectAnnoExt.err
    Untracked:  code/intersectAnnoExt.out
    Untracked:  code/intersectLiftedPAS.sh
    Untracked:  code/leafcutter_merge_regtools_redo.py
    Untracked:  code/liftJunctionFiles.sh
    Untracked:  code/liftPAS19to38.sh
    Untracked:  code/liftoverPAShg19to38.err
    Untracked:  code/liftoverPAShg19to38.out
    Untracked:  code/log/
    Untracked:  code/make5percPeakbed.py
    Untracked:  code/makeFileID.py
    Untracked:  code/makeNuclearDapaplots.sh
    Untracked:  code/makeNuclearPlots.err
    Untracked:  code/makeNuclearPlots.out
    Untracked:  code/makePheno.py
    Untracked:  code/makeSamplyGroupsChimp_TvN.py
    Untracked:  code/makeSamplyGroupsHuman_TvN.py
    Untracked:  code/mapRNAseqhg19.sh
    Untracked:  code/mapRNAseqhg19_newPipeline.sh
    Untracked:  code/maphg19.err
    Untracked:  code/maphg19.out
    Untracked:  code/maphg19.sh
    Untracked:  code/maphg19_new.err
    Untracked:  code/maphg19_new.out
    Untracked:  code/maphg19_sub.err
    Untracked:  code/maphg19_sub.out
    Untracked:  code/maphg19_subjunc.sh
    Untracked:  code/mediation_test.R
    Untracked:  code/merge.err
    Untracked:  code/mergeChimp3prime_inhg38.sh
    Untracked:  code/merge_leafcutter_clusters_redo.py
    Untracked:  code/mergeandsort_ChimpinHuman.err
    Untracked:  code/mergeandsort_ChimpinHuman.out
    Untracked:  code/mergedBam2BW.sh
    Untracked:  code/mergedbam2bw.err
    Untracked:  code/mergedbam2bw.out
    Untracked:  code/nameClusters.py
    Untracked:  code/namePeaks.py
    Untracked:  code/negativeMediation_montecarlo.R
    Untracked:  code/nuclearTranscriptDTplot.err
    Untracked:  code/nuclearTranscriptDTplot.out
    Untracked:  code/numMultimap.py
    Untracked:  code/overlapPAS.err
    Untracked:  code/overlapPAS.out
    Untracked:  code/overlapapaQTLPAS.sh
    Untracked:  code/overlapapaQTLPAS_extended.sh
    Untracked:  code/overlapapaQTLPAS_samples.sh
    Untracked:  code/peak2PAS.py
    Untracked:  code/pheno2countonly.R
    Untracked:  code/postiveMediation_montecarlo.R
    Untracked:  code/prepareAnnoLeafviz.err
    Untracked:  code/prepareAnnoLeafviz.out
    Untracked:  code/prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/prepareLeafvizAnno.sh
    Untracked:  code/preparePAS4lift.py
    Untracked:  code/prepare_phenotype_table.py
    Untracked:  code/primaryLift.err
    Untracked:  code/primaryLift.out
    Untracked:  code/primaryLift.sh
    Untracked:  code/processhg38exons.py
    Untracked:  code/quantJunc.sh
    Untracked:  code/quantJunc_TEST.sh
    Untracked:  code/quantJunc_removeBad.sh
    Untracked:  code/quantLiftedPAS.err
    Untracked:  code/quantLiftedPAS.out
    Untracked:  code/quantLiftedPAS.sh
    Untracked:  code/quatJunc.err
    Untracked:  code/quatJunc.out
    Untracked:  code/recChimpback2Human.err
    Untracked:  code/recChimpback2Human.out
    Untracked:  code/recLiftchim2human.sh
    Untracked:  code/revLift.err
    Untracked:  code/revLift.out
    Untracked:  code/revLiftPAShg38to19.sh
    Untracked:  code/reverseLift.sh
    Untracked:  code/runCheckReverseLift.sh
    Untracked:  code/runChimpDiffIso.sh
    Untracked:  code/runCountNucleotides.err
    Untracked:  code/runCountNucleotides.out
    Untracked:  code/runCountNucleotides.sh
    Untracked:  code/runCountNucleotidesPantro6.err
    Untracked:  code/runCountNucleotidesPantro6.out
    Untracked:  code/runCountNucleotides_pantro6.sh
    Untracked:  code/runFilterNumChroms.sh
    Untracked:  code/runHumanDiffIso.sh
    Untracked:  code/runNuclearDifffIso.sh
    Untracked:  code/runTotalDiffIso.sh
    Untracked:  code/run_Chimpleafcutter_ds.err
    Untracked:  code/run_Chimpleafcutter_ds.out
    Untracked:  code/run_Chimpverifybam.err
    Untracked:  code/run_Chimpverifybam.out
    Untracked:  code/run_Humanleafcutter_ds.err
    Untracked:  code/run_Humanleafcutter_ds.out
    Untracked:  code/run_Nuclearleafcutter_ds.err
    Untracked:  code/run_Nuclearleafcutter_ds.out
    Untracked:  code/run_Totalleafcutter_ds.err
    Untracked:  code/run_Totalleafcutter_ds.out
    Untracked:  code/run_chimpverifybam.sh
    Untracked:  code/run_verifyBam.sh
    Untracked:  code/run_verifybam.err
    Untracked:  code/run_verifybam.out
    Untracked:  code/slurm-62824013.out
    Untracked:  code/slurm-62825841.out
    Untracked:  code/slurm-62826116.out
    Untracked:  code/slurm-64108209.out
    Untracked:  code/slurm-64108521.out
    Untracked:  code/slurm-64108557.out
    Untracked:  code/snakePASChimp.err
    Untracked:  code/snakePASChimp.out
    Untracked:  code/snakePAShuman.out
    Untracked:  code/snakemake.batch
    Untracked:  code/snakemakeChimp.err
    Untracked:  code/snakemakeChimp.out
    Untracked:  code/snakemakeHuman.err
    Untracked:  code/snakemakeHuman.out
    Untracked:  code/snakemakePAS.batch
    Untracked:  code/snakemakePASFiltChimp.err
    Untracked:  code/snakemakePASFiltChimp.out
    Untracked:  code/snakemakePASFiltHuman.err
    Untracked:  code/snakemakePASFiltHuman.out
    Untracked:  code/snakemakePASchimp.batch
    Untracked:  code/snakemakePAShuman.batch
    Untracked:  code/snakemake_chimp.batch
    Untracked:  code/snakemake_human.batch
    Untracked:  code/snakemakefiltPAS.batch
    Untracked:  code/snakemakefiltPAS_chimp.sh
    Untracked:  code/snakemakefiltPAS_human.sh
    Untracked:  code/submit-snakemake-chimp.sh
    Untracked:  code/submit-snakemake-human.sh
    Untracked:  code/submit-snakemakePAS-chimp.sh
    Untracked:  code/submit-snakemakePAS-human.sh
    Untracked:  code/submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/submit-snakemakefiltPAS-human.sh
    Untracked:  code/subset_diffisopheno.py
    Untracked:  code/subset_diffisopheno_Chimp_tvN.py
    Untracked:  code/subset_diffisopheno_Huma_tvN.py
    Untracked:  code/subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/subset_diffisopheno_Total_HvC.py
    Untracked:  code/test
    Untracked:  code/threeprimeOrthoFC.out
    Untracked:  code/threeprimeOrthoFC.sh
    Untracked:  code/threeprimeOrthoFCcd.err
    Untracked:  code/transcriptDTplotsNuclear.sh
    Untracked:  code/transcriptDTplotsTotal.sh
    Untracked:  code/verifyBam4973.sh
    Untracked:  code/verifyBam4973inHuman.sh
    Untracked:  code/verifybam4973.err
    Untracked:  code/verifybam4973.out
    Untracked:  code/verifybam4973HumanMap.err
    Untracked:  code/verifybam4973HumanMap.out
    Untracked:  code/wrap_Chimpverifybam.err
    Untracked:  code/wrap_Chimpverifybam.out
    Untracked:  code/wrap_chimpverifybam.sh
    Untracked:  code/wrap_verifyBam.sh
    Untracked:  code/wrap_verifybam.err
    Untracked:  code/wrap_verifybam.out
    Untracked:  code/writeMergecode.py
    Untracked:  data/._.DS_Store
    Untracked:  data/._HC_filenames.txt
    Untracked:  data/._HC_filenames.txt.sb-4426323c-IKIs0S
    Untracked:  data/._HC_filenames.xlsx
    Untracked:  data/._MapPantro6_meta.txt
    Untracked:  data/._MapPantro6_meta.txt.sb-a5794dd2-Cskmlm
    Untracked:  data/._MapPantro6_meta.xlsx
    Untracked:  data/._OppositeSpeciesMap.txt
    Untracked:  data/._OppositeSpeciesMap.txt.sb-a5794dd2-mayWJf
    Untracked:  data/._OppositeSpeciesMap.xlsx
    Untracked:  data/._RNASEQ_metadata.txt
    Untracked:  data/._RNASEQ_metadata.txt.sb-4426323c-TE4ns3
    Untracked:  data/._RNASEQ_metadata.txt.sb-51f67ae1-HXp7Gq
    Untracked:  data/._RNASEQ_metadata_2Removed.txt
    Untracked:  data/._RNASEQ_metadata_2Removed.txt.sb-4426323c-a4lBwx
    Untracked:  data/._RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/._RNASEQ_metadata_stranded.txt
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-a5794dd2-D659m2
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-a5794dd2-ImNMoY
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-e4bf31f0-ZGnGgl
    Untracked:  data/._RNASEQ_metadata_stranded.xlsx
    Untracked:  data/._metadata_HCpanel.txt
    Untracked:  data/._metadata_HCpanel.txt.sb-a3d92a2d-b9cYoF
    Untracked:  data/._metadata_HCpanel.txt.sb-a5794dd2-i594qs
    Untracked:  data/._metadata_HCpanel.txt.sb-f4823d1e-qihGek
    Untracked:  data/._metadata_HCpanel.xlsx
    Untracked:  data/._metadata_HCpanel_frompantro5.xlsx
    Untracked:  data/._~$RNASEQ_metadata.xlsx
    Untracked:  data/._~$metadata_HCpanel.xlsx
    Untracked:  data/._.xlsx
    Untracked:  data/CompapaQTLpas/
    Untracked:  data/DTmatrix/
    Untracked:  data/DiffExpression/
    Untracked:  data/DiffIso_Nuclear/
    Untracked:  data/DiffIso_Total/
    Untracked:  data/DiffSplice/
    Untracked:  data/DiffSplice_liftedJunc/
    Untracked:  data/DiffSplice_removeBad/
    Untracked:  data/DominantPAS/
    Untracked:  data/EvalPantro5/
    Untracked:  data/HC_filenames.txt
    Untracked:  data/HC_filenames.xlsx
    Untracked:  data/Khan_prot/
    Untracked:  data/Li_eqtls/
    Untracked:  data/MapPantro6_meta.txt
    Untracked:  data/MapPantro6_meta.xlsx
    Untracked:  data/MapStats/
    Untracked:  data/NormalizedClusters/
    Untracked:  data/NuclearHvC/
    Untracked:  data/OppositeSpeciesMap.txt
    Untracked:  data/OppositeSpeciesMap.xlsx
    Untracked:  data/OverlapBenchmark/
    Untracked:  data/PAS/
    Untracked:  data/Peaks_5perc/
    Untracked:  data/Pheno_5perc/
    Untracked:  data/Pheno_5perc_nuclear/
    Untracked:  data/Pheno_5perc_nuclear_old/
    Untracked:  data/Pheno_5perc_total/
    Untracked:  data/RNASEQ_metadata.txt
    Untracked:  data/RNASEQ_metadata_2Removed.txt
    Untracked:  data/RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/RNASEQ_metadata_stranded.txt
    Untracked:  data/RNASEQ_metadata_stranded.txt.sb-e4bf31f0-ZGnGgl/
    Untracked:  data/RNASEQ_metadata_stranded.xlsx
    Untracked:  data/SignalSites/
    Untracked:  data/Threeprime2Ortho/
    Untracked:  data/TotalHvC/
    Untracked:  data/TwoBadSampleAnalysis/
    Untracked:  data/Wang_ribo/
    Untracked:  data/apaQTLGenes/
    Untracked:  data/chainFiles/
    Untracked:  data/cleanPeaks_anno/
    Untracked:  data/cleanPeaks_byspecies/
    Untracked:  data/cleanPeaks_lifted/
    Untracked:  data/files4viz_nuclear/
    Untracked:  data/leafviz/
    Untracked:  data/liftover_files/
    Untracked:  data/mediation/
    Untracked:  data/metadata_HCpanel.txt
    Untracked:  data/metadata_HCpanel.xlsx
    Untracked:  data/metadata_HCpanel_frompantro5.txt
    Untracked:  data/metadata_HCpanel_frompantro5.xlsx
    Untracked:  data/primaryLift/
    Untracked:  data/reverseLift/
    Untracked:  data/~$RNASEQ_metadata.xlsx
    Untracked:  data/~$metadata_HCpanel.xlsx
    Untracked:  data/.xlsx
    Untracked:  output/dtPlots/
    Untracked:  projectNotes.Rmd

Unstaged changes:
    Modified:   analysis/OppositeMap.Rmd
    Modified:   analysis/annotationInfo.Rmd
    Modified:   analysis/comp2apaQTLPAS.Rmd
    Modified:   analysis/correlationPhenos.Rmd
    Modified:   analysis/investigatePantro5.Rmd
    Modified:   analysis/multiMap.Rmd
    Modified:   analysis/speciesSpecific.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c27674c6d1dee0a9a99514500ecff762564723cd/analysis/establishCutoffs.Rmd" target="_blank">c27674c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
remove disc in chimp
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/c211e215cc82898901dd43dc9cfac50fbf10154d/docs/establishCutoffs.html" target="_blank">c211e21</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/7a9f60879915ace82b872f58d7209efd957b93a5/analysis/establishCutoffs.Rmd" target="_blank">7a9f608</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-18
</td>
<td>
finished benchmark
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/0c14f5143e61d931815912c7b07b18505db939e5/docs/establishCutoffs.html" target="_blank">0c14f51</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/11b26b4a39756e8570f8a56338c295731df5a4f8/analysis/establishCutoffs.Rmd" target="_blank">11b26b4</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-17
</td>
<td>
add benchmark res
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/9024e8613939acade1890b29abfacb1a95100e43/docs/establishCutoffs.html" target="_blank">9024e86</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/6c72627367fd37058f496305693b1d0e990f5ced/analysis/establishCutoffs.Rmd" target="_blank">6c72627</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-17
</td>
<td>
add code to make sample APS
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/5eef3eb638ca2bf1784cce27cdf859279202a165/docs/establishCutoffs.html" target="_blank">5eef3eb</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/5c24c0cad611959f9d9c0266b270a52242768642/analysis/establishCutoffs.Rmd" target="_blank">5c24c0c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-16
</td>
<td>
add cutoff code files
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>library(tidyverse)</code></pre>
<pre><code>── Attaching packages ─────────────────────────────────────────────────────────────────────────── tidyverse 1.2.1 ──</code></pre>
<pre><code>✔ ggplot2 3.1.1       ✔ purrr   0.3.2  
✔ tibble  2.1.1       ✔ dplyr   0.8.0.1
✔ tidyr   0.8.3       ✔ stringr 1.3.1  
✔ readr   1.3.1       ✔ forcats 0.3.0  </code></pre>
<pre><code>── Conflicts ────────────────────────────────────────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()</code></pre>
<pre class="r"><code>library(cowplot)</code></pre>
<pre><code>
Attaching package: &#39;cowplot&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:ggplot2&#39;:

    ggsave</code></pre>
<p>In this analyisis I will chose the 3 sets of 6 individuals randomly from my previous study to use for a benchmark analysis.</p>
<p>I will need to rerun the snakefile each time.</p>
<pre class="r"><code>indiv=read.table(&quot;../../apaQTL/data/MetaDataSequencing.txt&quot;, header= T, stringsAsFactors = F) %&gt;% dplyr::select(line) %&gt;% unique()</code></pre>
<pre class="bash"><code>mkdir ../data/OverlapBenchmark</code></pre>
<p>Randomly choose 3 sets:</p>
<pre class="r"><code>#sample1= sample(indiv$line, 6)
#sample2= sample(indiv$line, 6)
#sample3= sample(indiv$line, 6)

#save(sample1, sample2,sample3, file = &quot;../data/OverlapBenchmark/samples.RData&quot;)


load(&quot;../data/OverlapBenchmark/samples.RData&quot;)
sample1</code></pre>
<pre><code>[1] &quot;NA19144&quot; &quot;NA18508&quot; &quot;NA18858&quot; &quot;NA19239&quot; &quot;NA18504&quot; &quot;NA19119&quot;</code></pre>
<pre class="r"><code>sample2</code></pre>
<pre><code>[1] &quot;NA18522&quot; &quot;NA18516&quot; &quot;NA18504&quot; &quot;NA19119&quot; &quot;NA19131&quot; &quot;NA19137&quot;</code></pre>
<pre class="r"><code>sample3</code></pre>
<pre><code>[1] &quot;NA19131&quot; &quot;NA18486&quot; &quot;NA19210&quot; &quot;NA19223&quot; &quot;NA19092&quot; &quot;NA19200&quot;</code></pre>
<pre class="bash"><code>mkdir ../../PAS_Sample1
mkdir ../../PAS_Sample1/code
mkdir ../../PAS_Sample1/data
mkdir ../../PAS_Sample1/data/fastq

mkdir ../../PAS_Sample2
mkdir ../../PAS_Sample2/code
mkdir ../../PAS_Sample2/data
mkdir ../../PAS_Sample2/data/fastq


mkdir ../../PAS_Sample3/
mkdir ../../PAS_Sample3/code/
mkdir ../../PAS_Sample3/data/
mkdir ../../PAS_Sample3/data/fastq
</code></pre>
<p>Move over these fastq files.</p>
<p>I will also move the necessary snakefile and run files.</p>
<p>Next steps:</p>
<ul>
<li>assign to genes and get pheno ratio (<a href="https://brimittleman.github.io/Comparative_APA/annotatePAS.html" class="uri">https://brimittleman.github.io/Comparative_APA/annotatePAS.html</a>) - run code in all 3 locations</li>
</ul>
<p>’</p>
<pre class="bash"><code>mkdir ../data/cleanPeaks_anno
bedtools map -a ../data/cleanPeaks/human_APApeaks.ALLChrom.Filtered.Named.Cleaned.bed -b  /project2/gilad/briana/genome_anotation_data/hg38_refseq_anno/hg38_ncbiRefseq_Formatted_Allannotation.sort.bed -c 4 -S -o distinct &gt; ../data/cleanPeaks_anno/AllPAS_postLift.sort_LocAnno.bed 

python chooseAnno2Bed.py ../data/cleanPeaks_anno/AllPAS_postLift.sort_LocAnno.bed  ../data/cleanPeaks_anno/AllPAS_postLift.sort_LocAnnoPARSED.bed

python bed2SAF_gen.py ../data/cleanPeaks_anno/AllPAS_postLift.sort_LocAnnoPARSED.bed  ../data/cleanPeaks_anno/AllPAS_postLift.sort_LocAnnoPARSED.SAF

mkdir  ../data/CleanLiftedPeaks_FC/

sbatch quantLiftedPAS.sh 


###
python fixFChead_bothfrac.py ../data/CleanLiftedPeaks_FC/ALLPAS_postLift_LocParsed_Human ../data/CleanLiftedPeaks_FC/ALLPAS_postLift_LocParsed_Human_fixed.fc


python makeFileID.py ../data/CleanLiftedPeaks_FC/ALLPAS_postLift_LocParsed_Human ../data/CleanLiftedPeaks_FC/HumanFileID.txt


mkdir ../data/phenotype/

python makePheno.py  ../data/CleanLiftedPeaks_FC/ALLPAS_postLift_LocParsed_Human_fixed.fc ../data/CleanLiftedPeaks_FC/HumanFileID.txt ../data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno.txt

Rscript pheno2countonly.R -I ../data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno.txt -O ../data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno_countOnly.txt

python convertNumeric.py ../data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno_countOnly.txt ../data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno_countOnlyNumeric.txt
</code></pre>
<p>Pull these in, pull just the nuclear and get the mean:</p>
<p>Sample 1</p>
<pre class="r"><code>Sample1Anno=read.table(&quot;../../PAS_Sample1/data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno.txt&quot;, header = T, stringsAsFactors = F) %&gt;% tidyr::separate(chrom, sep = &quot;:&quot;, into = c(&quot;start1&quot;, &quot;start&quot;, &quot;end&quot;, &quot;id&quot;)) %&gt;% tidyr::separate(id, sep=&quot;_&quot;, into=c(&quot;gene&quot;, &quot;strand&quot;, &quot;peak&quot;))  %&gt;% separate(peak,into=c(&quot;loc&quot;, &quot;PAS&quot;,&quot;chr&quot;), sep=&quot;-&quot;)

Sample1Ind=colnames(Sample1Anno)[9:ncol(Sample1Anno)]

Sample1Usage=read.table(&quot;../../PAS_Sample1/data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno_countOnlyNumeric.txt&quot;, col.names = Sample1Ind) %&gt;% dplyr::select(contains(&quot;_N&quot;)) 

Sample1All=as.data.frame(cbind(cbind(Sample1Anno[,1:8], Sample1=rowMeans(Sample1Usage))))</code></pre>
<p>Sample 2:</p>
<pre class="r"><code>Sample2Anno=read.table(&quot;../../PAS_Sample2/data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno.txt&quot;, header = T, stringsAsFactors = F) %&gt;% tidyr::separate(chrom, sep = &quot;:&quot;, into = c(&quot;start1&quot;, &quot;start&quot;, &quot;end&quot;, &quot;id&quot;)) %&gt;% tidyr::separate(id, sep=&quot;_&quot;, into=c(&quot;gene&quot;, &quot;strand&quot;, &quot;peak&quot;))  %&gt;% separate(peak,into=c(&quot;loc&quot;, &quot;PAS&quot;,&quot;chr&quot;), sep=&quot;-&quot;)

Sample2Ind=colnames(Sample2Anno)[9:ncol(Sample2Anno)]

Sample2Usage=read.table(&quot;../../PAS_Sample2/data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno_countOnlyNumeric.txt&quot;, col.names = Sample2Ind) %&gt;% dplyr::select(contains(&quot;_N&quot;)) 

Sample2All=as.data.frame(cbind(cbind(Sample2Anno[,1:8], Sample2=rowMeans(Sample2Usage))))</code></pre>
<p>Sample 3</p>
<pre class="r"><code>Sample3Anno=read.table(&quot;../../PAS_Sample3/data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno.txt&quot;, header = T, stringsAsFactors = F) %&gt;% tidyr::separate(chrom, sep = &quot;:&quot;, into = c(&quot;start1&quot;, &quot;start&quot;, &quot;end&quot;, &quot;id&quot;)) %&gt;% tidyr::separate(id, sep=&quot;_&quot;, into=c(&quot;gene&quot;, &quot;strand&quot;, &quot;peak&quot;))  %&gt;% separate(peak,into=c(&quot;loc&quot;, &quot;PAS&quot;,&quot;chr&quot;), sep=&quot;-&quot;)

Sample3Ind=colnames(Sample3Anno)[9:ncol(Sample3Anno)]

Sample3Usage=read.table(&quot;../../PAS_Sample3/data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno_countOnlyNumeric.txt&quot;, col.names = Sample3Ind) %&gt;% dplyr::select(contains(&quot;_N&quot;)) 

Sample3All=as.data.frame(cbind(cbind(Sample3Anno[,1:8], Sample3=rowMeans(Sample3Usage))))</code></pre>
<p>I need to make a bed file with these to overlap with the original PAS. I want 100 bp on each side of the end. Positive strand (actual negative strand) take</p>
<pre class="r"><code>Sample1bed=Sample1All %&gt;% mutate(PASName=paste(gene,PAS, sep=&quot;_&quot;),newStart=ifelse(strand==&quot;+&quot;, as.integer(start)-100, as.integer(end)-100), newEnd=ifelse(strand==&quot;+&quot;,as.integer(start)+100, as.integer(end)+100)) %&gt;%  dplyr::select(chr, newStart, newEnd, PASName, Sample1, strand)

#write.table(Sample1bed,&quot;../data/OverlapBenchmark/sample1PAS.bed&quot;, col.names = F, row.names = F, sep=&quot;\t&quot;, quote = F )

Sample2bed=Sample2All %&gt;% mutate(PASName=paste(gene,PAS, sep=&quot;_&quot;),newStart=ifelse(strand==&quot;+&quot;, as.integer(start)-100, as.integer(end)-100), newEnd=ifelse(strand==&quot;+&quot;,as.integer(start)+100, as.integer(end)+100)) %&gt;%  dplyr::select(chr, newStart, newEnd, PASName, Sample2, strand)

#write.table(Sample2bed,&quot;../data/OverlapBenchmark/sample2PAS.bed&quot;, col.names = F, row.names = F, sep=&quot;\t&quot;, quote = F )

Sample3bed=Sample3All %&gt;% mutate(PASName=paste(gene,PAS, sep=&quot;_&quot;),newStart=ifelse(strand==&quot;+&quot;, as.integer(start)-100, as.integer(end)-100), newEnd=ifelse(strand==&quot;+&quot;,as.integer(start)+100, as.integer(end)+100)) %&gt;%  dplyr::select(chr, newStart, newEnd, PASName, Sample3, strand)

#write.table(Sample3bed,&quot;../data/OverlapBenchmark/sample3PAS.bed&quot;, col.names = F, row.names = F, sep=&quot;\t&quot;, quote = F )</code></pre>
<pre class="bash"><code>sort -k1,1 -k2,2n ../data/OverlapBenchmark/sample1PAS.bed &gt; ../data/OverlapBenchmark/sample1PAS_sort.bed

sort -k1,1 -k2,2n ../data/OverlapBenchmark/sample2PAS.bed &gt; ../data/OverlapBenchmark/sample2PAS_sort.bed

sort -k1,1 -k2,2n ../data/OverlapBenchmark/sample3PAS.bed &gt; ../data/OverlapBenchmark/sample3PAS_sort.bed</code></pre>
<p>Next step is to assess the overlap.</p>
<pre class="bash"><code>sbatch overlapapaQTLPAS_samples.sh</code></pre>
<p>Sample 1 res:</p>
<pre class="r"><code>wOverlap1=read.table(&quot;../data/OverlapBenchmark/sample1PAS_sort.Intersect.bed&quot;, col.names = colnames(Sample1bed)) %&gt;% mutate(overlap=&quot;yes&quot;)
noOverlap1=read.table(&quot;../data/OverlapBenchmark/sample1PAS_sort.Intersect.NoOverlap.bed&quot;, col.names = colnames(Sample1bed)) %&gt;% mutate(overlap=&quot;no&quot;)

AllwOinfo1=as.data.frame(rbind(wOverlap1, noOverlap1))
nrow(AllwOinfo1)</code></pre>
<pre><code>[1] 77850</code></pre>
<pre class="r"><code>overlap1=c()
totalvec1=c()
seq_usage=seq(0, .95, .01)
for (i in seq_usage){
  x=AllwOinfo1 %&gt;% filter(Sample1&gt;i) %&gt;% group_by(overlap) %&gt;% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlap1=c(overlap1,prop)
  totalvec1=c(totalvec1,total)
}

plot(seq_usage,overlap1,main=&quot;Sample 1&quot;, ylab=&quot;Percent Overlap&quot;, xlab=&quot;Usage Cutoff&quot;)
abline(v=.05,col=&quot;red&quot;)
abline(v=.1,col=&quot;blue&quot;)</code></pre>
<p><img src="figure/establishCutoffs.Rmd/unnamed-chunk-14-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-14-1">
Past versions of unnamed-chunk-14-1.png
</button>
</p>
<div id="fig-unnamed-chunk-14-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/0c14f5143e61d931815912c7b07b18505db939e5/docs/figure/establishCutoffs.Rmd/unnamed-chunk-14-1.png" target="_blank">0c14f51</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Sample 2</p>
<pre class="r"><code>wOverlap2=read.table(&quot;../data/OverlapBenchmark/sample2PAS_sort.Intersect.bed&quot;, col.names = colnames(Sample2bed)) %&gt;% mutate(overlap=&quot;yes&quot;)
noOverlap2=read.table(&quot;../data/OverlapBenchmark/sample2PAS_sort.Intersect.NoOverlap.bed&quot;, col.names = colnames(Sample2bed)) %&gt;% mutate(overlap=&quot;no&quot;)

AllwOinfo2=as.data.frame(rbind(wOverlap2, noOverlap2))
nrow(AllwOinfo2)</code></pre>
<pre><code>[1] 106751</code></pre>
<pre class="r"><code>overlap2=c()
totalvec2=c()
seq_usage=seq(0, .95, .01)
for (i in seq_usage){
  x=AllwOinfo2 %&gt;% filter(Sample2&gt;i) %&gt;% group_by(overlap) %&gt;% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlap2=c(overlap2,prop)
  totalve2c=c(totalvec2,total)
}

plot(seq_usage,overlap2,main=&quot;Sample 2&quot;, ylab=&quot;Percent Overlap&quot;, xlab=&quot;Usage Cutoff&quot;)
abline(v=.05,col=&quot;red&quot;)
abline(v=.1,col=&quot;blue&quot;)</code></pre>
<p><img src="figure/establishCutoffs.Rmd/unnamed-chunk-16-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-16-1">
Past versions of unnamed-chunk-16-1.png
</button>
</p>
<div id="fig-unnamed-chunk-16-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/0c14f5143e61d931815912c7b07b18505db939e5/docs/figure/establishCutoffs.Rmd/unnamed-chunk-16-1.png" target="_blank">0c14f51</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Sample 3</p>
<pre class="r"><code>wOverlap3=read.table(&quot;../data/OverlapBenchmark/sample3PAS_sort.Intersect.bed&quot;, col.names = colnames(Sample3bed)) %&gt;% mutate(overlap=&quot;yes&quot;)
noOverlap3=read.table(&quot;../data/OverlapBenchmark/sample3PAS_sort.Intersect.NoOverlap.bed&quot;, col.names = colnames(Sample3bed)) %&gt;% mutate(overlap=&quot;no&quot;)

AllwOinfo3=as.data.frame(rbind(wOverlap3, noOverlap3))
nrow(AllwOinfo3)</code></pre>
<pre><code>[1] 96769</code></pre>
<pre class="r"><code>overlap3=c()
totalvec3=c()
seq_usage=seq(0, .95, .01)
for (i in seq_usage){
  x=AllwOinfo3 %&gt;% filter(Sample3&gt;i) %&gt;% group_by(overlap) %&gt;% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlap3=c(overlap3,prop)
  totalvec3=c(totalvec3,total)
}

plot(seq_usage,overlap3,main=&quot;Sample 2&quot;, ylab=&quot;Percent Overlap&quot;, xlab=&quot;Usage Cutoff&quot;)
abline(v=.05,col=&quot;red&quot;)
abline(v=.1,col=&quot;blue&quot;)</code></pre>
<p><img src="figure/establishCutoffs.Rmd/unnamed-chunk-18-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-18-1">
Past versions of unnamed-chunk-18-1.png
</button>
</p>
<div id="fig-unnamed-chunk-18-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/0c14f5143e61d931815912c7b07b18505db939e5/docs/figure/establishCutoffs.Rmd/unnamed-chunk-18-1.png" target="_blank">0c14f51</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Plot with all:</p>
<pre class="r"><code>allSamp=as.data.frame(cbind(seq_usage,overlap1,overlap2,overlap3))
actualres= read.table(&quot;../data/CompapaQTLpas/ExtendedResoverlap.txt&quot;, header = T)

SampandRes=allSamp %&gt;% inner_join(actualres,by=&quot;seq_usage&quot;) 

colnames(SampandRes)= c(&quot;Cutoff&quot;,&quot;Sample1&quot;, &quot;Sample2&quot;, &quot;Sample3&quot;, &quot;CompPAS&quot;)

SampandRes_gather=SampandRes %&gt;% gather(key=&quot;set&quot;, value=&quot;overlap&quot;,-Cutoff)</code></pre>
<p>Plot:</p>
<pre class="r"><code>bench=ggplot(SampandRes_gather,aes(x=Cutoff,y=overlap, by=set, col=set))+ geom_line() + labs(title=&quot;Benchmark results with 3 sets of 6 samples&quot;) + geom_vline(xintercept=.05,col=&quot;red&quot;)+ geom_vline(xintercept=.1,col=&quot;blue&quot;)

bench</code></pre>
<p><img src="figure/establishCutoffs.Rmd/unnamed-chunk-20-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-20-1">
Past versions of unnamed-chunk-20-1.png
</button>
</p>
<div id="fig-unnamed-chunk-20-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c211e215cc82898901dd43dc9cfac50fbf10154d/docs/figure/establishCutoffs.Rmd/unnamed-chunk-20-1.png" target="_blank">c211e21</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-18
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/0c14f5143e61d931815912c7b07b18505db939e5/docs/figure/establishCutoffs.Rmd/unnamed-chunk-20-1.png" target="_blank">0c14f51</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>I will filter this to only genes passing the filter I established in the previous:</p>
<pre class="r"><code>PassingGenes=read.table(&quot;../data/OverlapBenchmark/genesPassingCuttoff.txt&quot;, header = T, stringsAsFactors = F)</code></pre>
<pre class="r"><code>AllwOinfo1_filt=AllwOinfo1 %&gt;% separate(PASName, into=c(&quot;gene&quot;, &quot;PAS&quot;), sep=&quot;_&quot;)%&gt;% filter(gene %in% PassingGenes$genes)


AllwOinfo2_filt=AllwOinfo2 %&gt;% separate(PASName, into=c(&quot;gene&quot;, &quot;PAS&quot;), sep=&quot;_&quot;) %&gt;% filter(gene %in% PassingGenes$genes)
AllwOinfo3_filt=AllwOinfo3 %&gt;% separate(PASName, into=c(&quot;gene&quot;, &quot;PAS&quot;), sep=&quot;_&quot;) %&gt;% filter(gene %in% PassingGenes$genes)</code></pre>
<p>Actual results:</p>
<pre class="r"><code>chroms=c(&#39;chr10&#39;, &#39;chr11&#39;, &#39;chr12&#39;, &#39;chr13&#39;, &#39;chr14&#39;, &#39;chr15&#39;, &#39;chr16&#39;, &#39;chr17&#39;, &#39;chr18&#39;, &#39;chr19&#39;, &#39;chr1&#39;, &#39;chr2&#39;, &#39;chr20&#39;, &#39;chr21&#39;, &#39;chr22&#39;, &#39;chr3&#39;, &#39;chr4&#39;, &#39;chr5&#39;, &#39;chr6&#39;, &#39;chr7&#39;,&#39;chr8&#39;, &#39;chr9&#39;)

compAPAPAS=read.table(&quot;../data/Peaks_5perc/Peaks_5perc_either_HumanCoordHummanUsage.bed&quot;, header =   T, stringsAsFactors = F)  %&gt;%  filter(Human&gt;=0.05, chr %in% chroms) 
metaDataPAS=read.table(&quot;../data/PAS/PAS_5perc_either_HumanCoord_BothUsage_meta.txt&quot;, header = T, stringsAsFactors = F) %&gt;% dplyr::select(PAS, gene)


wOverlapExt=read.table(&quot;../data/CompapaQTLpas/PAS_5percHuman.sort.Intersect_ext.bed&quot;, col.names = colnames(compAPAPAS)) %&gt;% mutate(overlap=&quot;yes&quot;)
noOverlapExt=read.table(&quot;../data/CompapaQTLpas/PAS_5percHuman.sort.Intersect.NoOverlap_ext.bed&quot;, col.names = colnames(compAPAPAS)) %&gt;% mutate(overlap=&quot;no&quot;)

AllwOinfoExt=as.data.frame(rbind(wOverlapExt, noOverlapExt)) %&gt;% inner_join(metaDataPAS,by=&quot;PAS&quot;) %&gt;% filter(gene %in% PassingGenes$genes)</code></pre>
<pre><code>Warning: Column `PAS` joining factor and character vector, coercing into
character vector</code></pre>
<pre class="r"><code>nrow(AllwOinfoExt)</code></pre>
<pre><code>[1] 36802</code></pre>
<pre class="r"><code>overlapFilt1=c()
seq_usage=seq(0, .95, .01)
for (i in seq_usage){
  x=AllwOinfo1_filt %&gt;% filter(Sample1&gt;i) %&gt;% group_by(overlap) %&gt;% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlapFilt1=c(overlapFilt1,prop)
}

overlapFilt2=c()
for (i in seq_usage){
  x=AllwOinfo2_filt %&gt;% filter(Sample2&gt;i) %&gt;% group_by(overlap) %&gt;% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlapFilt2=c(overlapFilt2,prop)
}

overlapFilt3=c()
for (i in seq_usage){
  x=AllwOinfo3_filt %&gt;% filter(Sample3&gt;i) %&gt;% group_by(overlap) %&gt;% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlapFilt3=c(overlapFilt3,prop)
}

overlapFiltActual=c()
for (i in seq_usage){
  x=AllwOinfoExt %&gt;% filter(Human&gt;i) %&gt;% group_by(overlap) %&gt;% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlapFiltActual=c(overlapFiltActual,prop)
}

FilteredRes=as.data.frame(cbind(seq_usage,overlapFilt1,overlapFilt2,overlapFilt3,overlapFiltActual))
colnames(FilteredRes)= c(&quot;Cutoff&quot;,&quot;Sample1&quot;, &quot;Sample2&quot;, &quot;Sample3&quot;, &quot;CompPAS&quot;)

FilteredRes_gather=FilteredRes %&gt;% gather(key=&quot;set&quot;, value=&quot;overlap&quot;,-Cutoff)</code></pre>
<pre class="r"><code>filteredbench=ggplot(FilteredRes_gather,aes(x=Cutoff,y=overlap, by=set, col=set))+ geom_line() + labs(title=&quot;Benchmark results with 3 sets of 6 samples\n filtered gene expresssion&quot;) + geom_vline(xintercept=.05,col=&quot;red&quot;)+ geom_vline(xintercept=.1,col=&quot;blue&quot;)

filteredbench</code></pre>
<p><img src="figure/establishCutoffs.Rmd/unnamed-chunk-25-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-25-1">
Past versions of unnamed-chunk-25-1.png
</button>
</p>
<div id="fig-unnamed-chunk-25-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c211e215cc82898901dd43dc9cfac50fbf10154d/docs/figure/establishCutoffs.Rmd/unnamed-chunk-25-1.png" target="_blank">c211e21</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Last step is to remove the PAS that were discovered originally in chimp.</p>
<pre class="r"><code>PASdisc=read.table(&quot;../data/PAS/PAS_5perc_either_HumanCoord_BothUsage_meta.txt&quot;, header = T, stringsAsFactors = F) %&gt;% filter(disc!=&quot;Chimp&quot;)</code></pre>
<p>Filter and rerun loop for percent overlap. Only the actual need to be updated for this.</p>
<pre class="r"><code>AllwOinfoExt_noChimp =AllwOinfoExt %&gt;% filter(PAS %in% PASdisc$PAS)</code></pre>
<p>Rerun:</p>
<pre class="r"><code>overlapFiltActualnoChimp=c()
for (i in seq_usage){
  x=AllwOinfoExt_noChimp %&gt;% filter(Human&gt;i) %&gt;% group_by(overlap) %&gt;% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlapFiltActualnoChimp=c(overlapFiltActualnoChimp,prop)
}

FilteredResNochimp=as.data.frame(cbind(seq_usage,overlapFilt1,overlapFilt2,overlapFilt3,overlapFiltActualnoChimp))
colnames(FilteredResNochimp)= c(&quot;Cutoff&quot;,&quot;Sample1&quot;, &quot;Sample2&quot;, &quot;Sample3&quot;, &quot;CompPAS&quot;)  

FilteredResNochimp_gather=FilteredResNochimp %&gt;% gather(key=&quot;set&quot;, value=&quot;overlap&quot;,-Cutoff)</code></pre>
<pre class="r"><code>filteredbencnoChimop=ggplot(FilteredResNochimp_gather,aes(x=Cutoff,y=overlap, by=set, col=set))+ geom_line() + labs(title=&quot;Benchmark results with 3 sets of 6 samples\n filtered gene expresssion, remove discovered in Chimp&quot;) + geom_vline(xintercept=.05,col=&quot;red&quot;)+ geom_vline(xintercept=.1,col=&quot;blue&quot;)

filteredbencnoChimop</code></pre>
<p><img src="figure/establishCutoffs.Rmd/unnamed-chunk-29-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>Plot together:</p>
<pre class="r"><code>plot_grid(bench,filteredbench,filteredbencnoChimop, nrow=1)</code></pre>
<p><img src="figure/establishCutoffs.Rmd/unnamed-chunk-30-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>Looks like using a 10% usage and this expression cuttoff is pretty good.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.5.1 (2018-07-02)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] cowplot_0.9.4   forcats_0.3.0   stringr_1.3.1   dplyr_0.8.0.1  
 [5] purrr_0.3.2     readr_1.3.1     tidyr_0.8.3     tibble_2.1.1   
 [9] ggplot2_3.1.1   tidyverse_1.2.1

loaded via a namespace (and not attached):
 [1] tidyselect_0.2.5 haven_1.1.2      lattice_0.20-38  colorspace_1.3-2
 [5] generics_0.0.2   htmltools_0.3.6  yaml_2.2.0       rlang_0.4.0     
 [9] later_0.7.5      pillar_1.3.1     glue_1.3.0       withr_2.1.2     
[13] modelr_0.1.2     readxl_1.1.0     plyr_1.8.4       munsell_0.5.0   
[17] gtable_0.2.0     workflowr_1.5.0  cellranger_1.1.0 rvest_0.3.2     
[21] evaluate_0.12    labeling_0.3     knitr_1.20       httpuv_1.4.5    
[25] broom_0.5.1      Rcpp_1.0.2       promises_1.0.1   scales_1.0.0    
[29] backports_1.1.2  jsonlite_1.6     fs_1.3.1         hms_0.4.2       
[33] digest_0.6.18    stringi_1.2.4    grid_3.5.1       rprojroot_1.3-2 
[37] cli_1.1.0        tools_3.5.1      magrittr_1.5     lazyeval_0.2.1  
[41] crayon_1.3.4     whisker_0.3-2    pkgconfig_2.0.2  xml2_1.2.0      
[45] lubridate_1.7.4  assertthat_0.2.0 rmarkdown_1.10   httr_1.3.1      
[49] rstudioapi_0.10  R6_2.3.0         nlme_3.1-137     git2r_0.26.1    
[53] compiler_3.5.1  </code></pre>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
