<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Home</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/journal.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Comparative_APA</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/brimittleman/Comparative_APA">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Home</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-04-28
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Comparative_APA/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtree38b161983ba8d19911dc9fc43e8376ffa8376322targetblank38b1619a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/brimittleman/Comparative_APA/tree/38b161983ba8d19911dc9fc43e8376ffa8376322" target="_blank">38b1619</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtree38b161983ba8d19911dc9fc43e8376ffa8376322targetblank38b1619a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    code/chimp_log/
    Ignored:    code/human_log/
    Ignored:    data/.DS_Store
    Ignored:    data/TrialFiltersMeta.txt.sb-9845453e-R58Y0Q/
    Ignored:    data/mediation_prot/
    Ignored:    data/metadata_HCpanel.txt.sb-284518db-RGf0kd/
    Ignored:    data/metadata_HCpanel.txt.sb-a5794dd2-i594qs/
    Ignored:    output/.DS_Store

Untracked files:
    Untracked:  ._.DS_Store
    Untracked:  Chimp/
    Untracked:  Human/
    Untracked:  analysis/AREstabilityScores.Rmd
    Untracked:  analysis/CrossChimpThreePrime.Rmd
    Untracked:  analysis/DiffTransProtvsExpression.Rmd
    Untracked:  analysis/DiffUsedUTR.Rmd
    Untracked:  analysis/GvizPlots.Rmd
    Untracked:  analysis/HandC.TvN
    Untracked:  analysis/PhenotypeOverlap10.Rmd
    Untracked:  analysis/annotationBias.Rmd
    Untracked:  analysis/assessReadQual.Rmd
    Untracked:  analysis/diffExpressionPantro6.Rmd
    Untracked:  code/._AlignmentScores.sh
    Untracked:  code/._BothFCMM.sh
    Untracked:  code/._BothFCMMPrim.sh
    Untracked:  code/._BothFCnewOInclusive.sh
    Untracked:  code/._ChimpStarMM2.sh
    Untracked:  code/._ClassifyLeafviz.sh
    Untracked:  code/._ClosestorthoEx.sh
    Untracked:  code/._Config_chimp.yaml
    Untracked:  code/._Config_chimp_full.yaml
    Untracked:  code/._Config_human.yaml
    Untracked:  code/._ConvertJunc2Bed.sh
    Untracked:  code/._CountNucleotides.py
    Untracked:  code/._CrossMapChimpRNA.sh
    Untracked:  code/._CrossMapThreeprime.sh
    Untracked:  code/._DiffSplice.sh
    Untracked:  code/._DiffSplicePlots.sh
    Untracked:  code/._DiffSplicePlots_gencode.sh
    Untracked:  code/._DiffSplice_gencode.sh
    Untracked:  code/._DiffSplice_removebad.sh
    Untracked:  code/._Filter255MM.sh
    Untracked:  code/._FilterPrimSec.sh
    Untracked:  code/._FindIntronForDomPAS.sh
    Untracked:  code/._FindIntronForDomPAS_DF.sh
    Untracked:  code/._GetMAPQscore.py
    Untracked:  code/._GetSecondaryMap.py
    Untracked:  code/._Lift5perPAS.sh
    Untracked:  code/._LiftFinalChimpJunc2Human.sh
    Untracked:  code/._LiftOrthoPAS2chimp.sh
    Untracked:  code/._MapBadSamples.sh
    Untracked:  code/._MismatchNumbers.sh
    Untracked:  code/._PAS_ATTAAA.sh
    Untracked:  code/._PAS_ATTAAA_df.sh
    Untracked:  code/._PAS_seqExpanded.sh
    Untracked:  code/._PASsequences.sh
    Untracked:  code/._PASsequences_DF.sh
    Untracked:  code/._PlotNuclearUsagebySpecies.R
    Untracked:  code/._PlotNuclearUsagebySpecies_DF.R
    Untracked:  code/._QuantMergedClusters.sh
    Untracked:  code/._RNATranscriptDTplot.sh
    Untracked:  code/._ReverseLiftFilter.R
    Untracked:  code/._RunFixLeafCluster.sh
    Untracked:  code/._RunNegMCMediation.sh
    Untracked:  code/._RunNegMCMediationDF.sh
    Untracked:  code/._RunPosMCMediationDF.err
    Untracked:  code/._RunPosMCMediationDF.sh
    Untracked:  code/._SAF2Bed.py
    Untracked:  code/._Snakefile
    Untracked:  code/._SnakefilePAS
    Untracked:  code/._SnakefilePASfilt
    Untracked:  code/._SortIndexBadSamples.sh
    Untracked:  code/._StarMM2.sh
    Untracked:  code/._TestFC.sh
    Untracked:  code/._assignPeak2Intronicregion
    Untracked:  code/._assignPeak2Intronicregion.sh
    Untracked:  code/._bed215upbed.py
    Untracked:  code/._bed2Bedbothstrand.py
    Untracked:  code/._bed2SAF_gen.py
    Untracked:  code/._buildIndecpantro5
    Untracked:  code/._buildIndecpantro5.sh
    Untracked:  code/._buildLeafviz.sh
    Untracked:  code/._buildLeafviz_leadAnno.sh
    Untracked:  code/._buildStarIndex.sh
    Untracked:  code/._chimpChromprder.sh
    Untracked:  code/._chimpMultiCov.sh
    Untracked:  code/._chimpMultiCov255.sh
    Untracked:  code/._chimpMultiCovInclusive.sh
    Untracked:  code/._chooseSignalSite.py
    Untracked:  code/._cleanbed2saf.py
    Untracked:  code/._cluster.json
    Untracked:  code/._cluster2bed.py
    Untracked:  code/._clusterLiftReverse.sh
    Untracked:  code/._clusterLiftReverse_removebad.sh
    Untracked:  code/._clusterLiftprimary.sh
    Untracked:  code/._clusterLiftprimary_removebad.sh
    Untracked:  code/._converBam2Junc.sh
    Untracked:  code/._converBam2Junc_removeBad.sh
    Untracked:  code/._extraSnakefiltpas
    Untracked:  code/._extractPhyloReg.py
    Untracked:  code/._extractPhyloRegGene.py
    Untracked:  code/._extractPhylopGeneral.ph
    Untracked:  code/._extractPhylopGeneral.py
    Untracked:  code/._extractPhylopReg200down.py
    Untracked:  code/._extractPhylopReg200up.py
    Untracked:  code/._filter5percPAS.py
    Untracked:  code/._filterNumChroms.py
    Untracked:  code/._filterPASforMP.py
    Untracked:  code/._filterPostLift.py
    Untracked:  code/._filterPrimaryread.py
    Untracked:  code/._filterSecondaryread.py
    Untracked:  code/._fixExonFC.py
    Untracked:  code/._fixFCheadforExp.py
    Untracked:  code/._fixLeafCluster.py
    Untracked:  code/._fixLiftedJunc.py
    Untracked:  code/._fixUTRexonanno.py
    Untracked:  code/._formathg38Anno.py
    Untracked:  code/._formatpantro6Anno.py
    Untracked:  code/._getRNAseqMapStats.sh
    Untracked:  code/._hg19MapStats.sh
    Untracked:  code/._humanChromorder.sh
    Untracked:  code/._humanMultiCov.sh
    Untracked:  code/._humanMultiCov255.sh
    Untracked:  code/._humanMultiCov_inclusive.sh
    Untracked:  code/._intersectLiftedPAS.sh
    Untracked:  code/._liftJunctionFiles.sh
    Untracked:  code/._liftPAS19to38.sh
    Untracked:  code/._liftedchimpJunc2human.sh
    Untracked:  code/._makeNuclearDapaplots.sh
    Untracked:  code/._makeNuclearDapaplots_DF.sh
    Untracked:  code/._makeSamplyGroupsHuman_TvN.py
    Untracked:  code/._mapRNAseqhg19.sh
    Untracked:  code/._mapRNAseqhg19_newPipeline.sh
    Untracked:  code/._maphg19.sh
    Untracked:  code/._maphg19_subjunc.sh
    Untracked:  code/._mediation_test.R
    Untracked:  code/._mergeChimp3prime_inhg38.sh
    Untracked:  code/._mergeandBWRNAseq.sh
    Untracked:  code/._mergedBam2BW.sh
    Untracked:  code/._nameClusters.py
    Untracked:  code/._negativeMediation_montecarlo.R
    Untracked:  code/._negativeMediation_montecarloDF.R
    Untracked:  code/._numMultimap.py
    Untracked:  code/._overlapMMandOrthoexon.sh
    Untracked:  code/._overlapPASandOrthoexon.sh
    Untracked:  code/._overlapapaQTLPAS.sh
    Untracked:  code/._parseHg38.py
    Untracked:  code/._postiveMediation_montecarlo_DF.R
    Untracked:  code/._prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/._prepareLeafvizAnno.sh
    Untracked:  code/._preparePAS4lift.py
    Untracked:  code/._primaryLift.sh
    Untracked:  code/._processhg38exons.py
    Untracked:  code/._quantJunc.sh
    Untracked:  code/._quantJunc_TEST.sh
    Untracked:  code/._quantJunc_removeBad.sh
    Untracked:  code/._quantLiftedPASPrimary.sh
    Untracked:  code/._quantMerged_seperatly.sh
    Untracked:  code/._recLiftchim2human.sh
    Untracked:  code/._revLiftPAShg38to19.sh
    Untracked:  code/._reverseLift.sh
    Untracked:  code/._runCheckReverseLift.sh
    Untracked:  code/._runChimpDiffIso.sh
    Untracked:  code/._runCountNucleotides.sh
    Untracked:  code/._runFilterNumChroms.sh
    Untracked:  code/._runHumanDiffIso.sh
    Untracked:  code/._runNuclearDiffIso_DF.sh
    Untracked:  code/._runNuclearDifffIso.sh
    Untracked:  code/._runTotalDiffIso.sh
    Untracked:  code/._run_chimpverifybam.sh
    Untracked:  code/._run_verifyBam.sh
    Untracked:  code/._snakemake.batch
    Untracked:  code/._snakemakePAS.batch
    Untracked:  code/._snakemakePASchimp.batch
    Untracked:  code/._snakemakePAShuman.batch
    Untracked:  code/._snakemake_chimp.batch
    Untracked:  code/._snakemake_human.batch
    Untracked:  code/._snakemakefiltPAS.batch
    Untracked:  code/._snakemakefiltPAS_chimp
    Untracked:  code/._snakemakefiltPAS_chimp.sh
    Untracked:  code/._snakemakefiltPAS_human.sh
    Untracked:  code/._spliceSite2Fasta.py
    Untracked:  code/._submit-snakemake-chimp.sh
    Untracked:  code/._submit-snakemake-human.sh
    Untracked:  code/._submit-snakemakePAS-chimp.sh
    Untracked:  code/._submit-snakemakePAS-human.sh
    Untracked:  code/._submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/._submit-snakemakefiltPAS-human.sh
    Untracked:  code/._subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/._subset_diffisopheno_Nuclear_HvC_DF.py
    Untracked:  code/._subset_diffisopheno_Total_HvC.py
    Untracked:  code/._threeprimeOrthoFC.sh
    Untracked:  code/._transcriptDTplotsNuclear.sh
    Untracked:  code/._verifyBam4973.sh
    Untracked:  code/._verifyBam4973inHuman.sh
    Untracked:  code/._wrap_chimpverifybam.sh
    Untracked:  code/._wrap_verifyBam.sh
    Untracked:  code/._writeMergecode.py
    Untracked:  code/.snakemake/
    Untracked:  code/ALLPAS_sequenceDF.err
    Untracked:  code/ALLPAS_sequenceDF.out
    Untracked:  code/AlignmentScores.err
    Untracked:  code/AlignmentScores.out
    Untracked:  code/AlignmentScores.sh
    Untracked:  code/BothFCMM.err
    Untracked:  code/BothFCMM.out
    Untracked:  code/BothFCMM.sh
    Untracked:  code/BothFCMMPrim.err
    Untracked:  code/BothFCMMPrim.out
    Untracked:  code/BothFCMMPrim.sh
    Untracked:  code/BothFCnewOInclusive.sh
    Untracked:  code/BothFCnewOInclusive.sh.err
    Untracked:  code/BothFCnewOInclusive.sh.out
    Untracked:  code/ChimpStarMM2.err
    Untracked:  code/ChimpStarMM2.out
    Untracked:  code/ChimpStarMM2.sh
    Untracked:  code/ClassifyLeafviz.sh
    Untracked:  code/ClosestorthoEx.err
    Untracked:  code/ClosestorthoEx.out
    Untracked:  code/ClosestorthoEx.sh
    Untracked:  code/Config_chimp.yaml
    Untracked:  code/Config_chimp_full.yaml
    Untracked:  code/Config_human.yaml
    Untracked:  code/ConvertJunc2Bed.err
    Untracked:  code/ConvertJunc2Bed.out
    Untracked:  code/ConvertJunc2Bed.sh
    Untracked:  code/CountNucleotides.py
    Untracked:  code/CrossMapChimpRNA.sh
    Untracked:  code/CrossMapThreeprime.sh
    Untracked:  code/CrossmapChimp3prime.err
    Untracked:  code/CrossmapChimp3prime.out
    Untracked:  code/CrossmapChimpRNA.err
    Untracked:  code/CrossmapChimpRNA.out
    Untracked:  code/DTUTR.sh
    Untracked:  code/DiffSplice.err
    Untracked:  code/DiffSplice.out
    Untracked:  code/DiffSplice.sh
    Untracked:  code/DiffSplicePlots.err
    Untracked:  code/DiffSplicePlots.out
    Untracked:  code/DiffSplicePlots.sh
    Untracked:  code/DiffSplicePlots_gencode.sh
    Untracked:  code/DiffSplice_gencode.sh
    Untracked:  code/DiffSplice_removebad.err
    Untracked:  code/DiffSplice_removebad.out
    Untracked:  code/DiffSplice_removebad.sh
    Untracked:  code/Filter255.err
    Untracked:  code/Filter255.out
    Untracked:  code/Filter255MM.sh
    Untracked:  code/FilterPrimSec.err
    Untracked:  code/FilterPrimSec.out
    Untracked:  code/FilterPrimSec.sh
    Untracked:  code/FilterReverseLift.err
    Untracked:  code/FilterReverseLift.out
    Untracked:  code/FindDomXCutoff.py
    Untracked:  code/FindIntronForDomPAS.err
    Untracked:  code/FindIntronForDomPAS.out
    Untracked:  code/FindIntronForDomPAS.sh
    Untracked:  code/FindIntronForDomPAS_DF.sh
    Untracked:  code/GencodeDiffSplice.err
    Untracked:  code/GencodeDiffSplice.out
    Untracked:  code/GetMAPQscore.py
    Untracked:  code/GetSecondaryMap.py
    Untracked:  code/GetTopminus2Usage.py
    Untracked:  code/HchromOrder.err
    Untracked:  code/HchromOrder.out
    Untracked:  code/InfoContentShannon.py
    Untracked:  code/IntersectMMandOrtho.err
    Untracked:  code/IntersectMMandOrtho.out
    Untracked:  code/IntersectPASandOrtho.err
    Untracked:  code/IntersectPASandOrtho.out
    Untracked:  code/JunctionLift.err
    Untracked:  code/JunctionLift.out
    Untracked:  code/JunctionLiftFinalChimp.err
    Untracked:  code/JunctionLiftFinalChimp.out
    Untracked:  code/Lift5perPAS.sh
    Untracked:  code/Lift5perPASbed.err
    Untracked:  code/Lift5perPASbed.out
    Untracked:  code/LiftClustersFirst.err
    Untracked:  code/LiftClustersFirst.out
    Untracked:  code/LiftClustersFirst_remove.err
    Untracked:  code/LiftClustersFirst_remove.out
    Untracked:  code/LiftClustersSecond.err
    Untracked:  code/LiftClustersSecond.out
    Untracked:  code/LiftClustersSecond_remove.err
    Untracked:  code/LiftClustersSecond_remove.out
    Untracked:  code/LiftFinalChimpJunc2Human.sh
    Untracked:  code/LiftOrthoPAS2chimp.sh
    Untracked:  code/LiftorthoPAS.err
    Untracked:  code/LiftorthoPASt.out
    Untracked:  code/Log.out
    Untracked:  code/MapBadSamples.err
    Untracked:  code/MapBadSamples.out
    Untracked:  code/MapBadSamples.sh
    Untracked:  code/MapStats.err
    Untracked:  code/MapStats.out
    Untracked:  code/MaxEntCode/
    Untracked:  code/MergeClusters.err
    Untracked:  code/MergeClusters.out
    Untracked:  code/MergeClusters.sh
    Untracked:  code/MismatchNumbers.err
    Untracked:  code/MismatchNumbers.out
    Untracked:  code/MismatchNumbers.sh
    Untracked:  code/NuclearDTUTR.err
    Untracked:  code/NuclearDTUTRt.out
    Untracked:  code/NuclearPlotsDEandDiffDom_4.err
    Untracked:  code/NuclearPlotsDEandDiffDom_4.out
    Untracked:  code/NuclearPlotsDEandDiffDom_4.sh
    Untracked:  code/PAS_ATTAAA.err
    Untracked:  code/PAS_ATTAAA.out
    Untracked:  code/PAS_ATTAAA.sh
    Untracked:  code/PAS_ATTAAADF.err
    Untracked:  code/PAS_ATTAAADF.out
    Untracked:  code/PAS_ATTAAA_df.sh
    Untracked:  code/PAS_seqExpanded.sh
    Untracked:  code/PAS_sequence.err
    Untracked:  code/PAS_sequence.out
    Untracked:  code/PAS_sequenceDF.err
    Untracked:  code/PAS_sequenceDF.out
    Untracked:  code/PASexpanded_sequenceDF.err
    Untracked:  code/PASexpanded_sequenceDF.out
    Untracked:  code/PASsequences.sh
    Untracked:  code/PASsequences_DF.sh
    Untracked:  code/PlotNuclearUsagebySpecies.R
    Untracked:  code/PlotNuclearUsagebySpecies_DF.R
    Untracked:  code/PlotNuclearUsagebySpecies_DF_DEout.R
    Untracked:  code/QuantMergeClusters
    Untracked:  code/QuantMergeClusters.err
    Untracked:  code/QuantMergeClusters.out
    Untracked:  code/QuantMergedClusters.sh
    Untracked:  code/RNATranscriptDTplot.err
    Untracked:  code/RNATranscriptDTplot.out
    Untracked:  code/RNATranscriptDTplot.sh
    Untracked:  code/Rev_liftoverPAShg19to38.err
    Untracked:  code/Rev_liftoverPAShg19to38.out
    Untracked:  code/ReverseLiftFilter.R
    Untracked:  code/RunFixCluster.err
    Untracked:  code/RunFixCluster.out
    Untracked:  code/RunFixLeafCluster.sh
    Untracked:  code/RunNegMCMediation.err
    Untracked:  code/RunNegMCMediation.sh
    Untracked:  code/RunNegMCMediationDF.err
    Untracked:  code/RunNegMCMediationDF.out
    Untracked:  code/RunNegMCMediationDF.sh
    Untracked:  code/RunNegMCMediationr.out
    Untracked:  code/RunNewDom.err
    Untracked:  code/RunNewDom.out
    Untracked:  code/RunPosMCMediation.err
    Untracked:  code/RunPosMCMediation.sh
    Untracked:  code/RunPosMCMediationDF.err
    Untracked:  code/RunPosMCMediationDF.out
    Untracked:  code/RunPosMCMediationDF.sh
    Untracked:  code/RunPosMCMediationr.out
    Untracked:  code/SAF215upbed_gen.py
    Untracked:  code/SAF2Bed.py
    Untracked:  code/Snakefile
    Untracked:  code/SnakefilePAS
    Untracked:  code/SnakefilePASfilt
    Untracked:  code/SortIndexBadSamples.err
    Untracked:  code/SortIndexBadSamples.out
    Untracked:  code/SortIndexBadSamples.sh
    Untracked:  code/StarMM2.err
    Untracked:  code/StarMM2.out
    Untracked:  code/StarMM2.sh
    Untracked:  code/TestFC.err
    Untracked:  code/TestFC.out
    Untracked:  code/TestFC.sh
    Untracked:  code/TotalTranscriptDTplot.err
    Untracked:  code/TotalTranscriptDTplot.out
    Untracked:  code/UTR2FASTA.py
    Untracked:  code/Upstream10Bases_general.py
    Untracked:  code/allPASSeq_df.sh
    Untracked:  code/apaQTLsnake.err
    Untracked:  code/apaQTLsnake.out
    Untracked:  code/apaQTLsnakePAS.err
    Untracked:  code/apaQTLsnakePAS.out
    Untracked:  code/apaQTLsnakePAShuman.err
    Untracked:  code/apaQTLsnakefiltPAS.err
    Untracked:  code/apaQTLsnakefiltPAS.out
    Untracked:  code/assignPeak2Intronicregion.err
    Untracked:  code/assignPeak2Intronicregion.out
    Untracked:  code/assignPeak2Intronicregion.sh
    Untracked:  code/bam2junc.err
    Untracked:  code/bam2junc.out
    Untracked:  code/bam2junc_remove.err
    Untracked:  code/bam2junc_remove.out
    Untracked:  code/bed215upbed.py
    Untracked:  code/bed2Bedbothstrand.py
    Untracked:  code/bed2SAF_gen.py
    Untracked:  code/bed2saf.py
    Untracked:  code/bg_to_cov.py
    Untracked:  code/buildIndecpantro5
    Untracked:  code/buildIndecpantro5.sh
    Untracked:  code/buildLeafviz.err
    Untracked:  code/buildLeafviz.out
    Untracked:  code/buildLeafviz.sh
    Untracked:  code/buildLeafviz_leadAnno.sh
    Untracked:  code/buildLeafviz_leafanno.err
    Untracked:  code/buildLeafviz_leafanno.out
    Untracked:  code/buildStarIndex.sh
    Untracked:  code/callPeaksYL.py
    Untracked:  code/chimpChromprder.sh
    Untracked:  code/chimpMultiCov.err
    Untracked:  code/chimpMultiCov.out
    Untracked:  code/chimpMultiCov.sh
    Untracked:  code/chimpMultiCov255.sh
    Untracked:  code/chimpMultiCovInclusive.err
    Untracked:  code/chimpMultiCovInclusive.out
    Untracked:  code/chimpMultiCovInclusive.sh
    Untracked:  code/chooseAnno2Bed.py
    Untracked:  code/chooseAnno2SAF.py
    Untracked:  code/chooseSignalSite.py
    Untracked:  code/chromOrder.err
    Untracked:  code/chromOrder.out
    Untracked:  code/classifyLeafviz.err
    Untracked:  code/classifyLeafviz.out
    Untracked:  code/cleanbed2saf.py
    Untracked:  code/cluster.json
    Untracked:  code/cluster2bed.py
    Untracked:  code/clusterLiftReverse.sh
    Untracked:  code/clusterLiftReverse_removebad.sh
    Untracked:  code/clusterLiftprimary.sh
    Untracked:  code/clusterLiftprimary_removebad.sh
    Untracked:  code/clusterPAS.json
    Untracked:  code/clusterfiltPAS.json
    Untracked:  code/comands2Mege.sh
    Untracked:  code/converBam2Junc.sh
    Untracked:  code/converBam2Junc_removeBad.sh
    Untracked:  code/convertNumeric.py
    Untracked:  code/environment.yaml
    Untracked:  code/extraSnakefiltpas
    Untracked:  code/extractPhyloReg.py
    Untracked:  code/extractPhyloRegGene.py
    Untracked:  code/extractPhylopGeneral.py
    Untracked:  code/extractPhylopReg200down.py
    Untracked:  code/extractPhylopReg200up.py
    Untracked:  code/filter5perc.R
    Untracked:  code/filter5percPAS.py
    Untracked:  code/filter5percPheno.py
    Untracked:  code/filterBamforMP.pysam2_gen.py
    Untracked:  code/filterJuncChroms.err
    Untracked:  code/filterJuncChroms.out
    Untracked:  code/filterMissprimingInNuc10_gen.py
    Untracked:  code/filterNumChroms.py
    Untracked:  code/filterPASforMP.py
    Untracked:  code/filterPostLift.py
    Untracked:  code/filterPrimaryread.py
    Untracked:  code/filterSAFforMP_gen.py
    Untracked:  code/filterSecondaryread.py
    Untracked:  code/filterSortBedbyCleanedBed_gen.R
    Untracked:  code/filterpeaks.py
    Untracked:  code/fixExonFC.py
    Untracked:  code/fixFChead.py
    Untracked:  code/fixFChead_bothfrac.py
    Untracked:  code/fixFCheadforExp.py
    Untracked:  code/fixLeafCluster.py
    Untracked:  code/fixLiftedJunc.py
    Untracked:  code/fixUTRexonanno.py
    Untracked:  code/formathg38Anno.py
    Untracked:  code/generateStarIndex.err
    Untracked:  code/generateStarIndex.out
    Untracked:  code/generateStarIndexHuman.err
    Untracked:  code/generateStarIndexHuman.out
    Untracked:  code/getAlloverlap.py
    Untracked:  code/getRNAseqMapStats.sh
    Untracked:  code/hg19MapStats.err
    Untracked:  code/hg19MapStats.out
    Untracked:  code/hg19MapStats.sh
    Untracked:  code/humanChromorder.sh
    Untracked:  code/humanFiles
    Untracked:  code/humanMultiCov.err
    Untracked:  code/humanMultiCov.out
    Untracked:  code/humanMultiCov.sh
    Untracked:  code/humanMultiCov255.err
    Untracked:  code/humanMultiCov255.out
    Untracked:  code/humanMultiCov255.sh
    Untracked:  code/humanMultiCovInclusive.err
    Untracked:  code/humanMultiCovInclusive.out
    Untracked:  code/humanMultiCov_inclusive.sh
    Untracked:  code/infoContentSimpson.py
    Untracked:  code/intersectAnno.err
    Untracked:  code/intersectAnno.out
    Untracked:  code/intersectAnnoExt.err
    Untracked:  code/intersectAnnoExt.out
    Untracked:  code/intersectLiftedPAS.sh
    Untracked:  code/leafcutter_merge_regtools_redo.py
    Untracked:  code/liftJunctionFiles.sh
    Untracked:  code/liftPAS19to38.sh
    Untracked:  code/liftoverPAShg19to38.err
    Untracked:  code/liftoverPAShg19to38.out
    Untracked:  code/log/
    Untracked:  code/make5percPeakbed.py
    Untracked:  code/makeFileID.py
    Untracked:  code/makeNuclearDapaplots.sh
    Untracked:  code/makeNuclearDapaplots_DF.sh
    Untracked:  code/makeNuclearPlots.err
    Untracked:  code/makeNuclearPlots.out
    Untracked:  code/makeNuclearPlotsDF.err
    Untracked:  code/makeNuclearPlotsDF.out
    Untracked:  code/makePheno.py
    Untracked:  code/makeSamplyGroupsChimp_TvN.py
    Untracked:  code/makeSamplyGroupsHuman_TvN.py
    Untracked:  code/mapRNAseqhg19.sh
    Untracked:  code/mapRNAseqhg19_newPipeline.sh
    Untracked:  code/maphg19.err
    Untracked:  code/maphg19.out
    Untracked:  code/maphg19.sh
    Untracked:  code/maphg19_new.err
    Untracked:  code/maphg19_new.out
    Untracked:  code/maphg19_sub.err
    Untracked:  code/maphg19_sub.out
    Untracked:  code/maphg19_subjunc.sh
    Untracked:  code/mediation_test.R
    Untracked:  code/merge.err
    Untracked:  code/mergeChimp3prime_inhg38.sh
    Untracked:  code/mergeChimpRNA.sh
    Untracked:  code/merge_leafcutter_clusters_redo.py
    Untracked:  code/mergeandBWRNAseq.sh
    Untracked:  code/mergeandsort_ChimpinHuman.err
    Untracked:  code/mergeandsort_ChimpinHuman.out
    Untracked:  code/mergedBam2BW.sh
    Untracked:  code/mergedbam2bw.err
    Untracked:  code/mergedbam2bw.out
    Untracked:  code/mergedbamRNAand2bw.err
    Untracked:  code/mergedbamRNAand2bw.out
    Untracked:  code/nameClusters.py
    Untracked:  code/namePeaks.py
    Untracked:  code/negativeMediation_montecarlo.R
    Untracked:  code/negativeMediation_montecarloDF.R
    Untracked:  code/nuclearTranscriptDTplot.err
    Untracked:  code/nuclearTranscriptDTplot.out
    Untracked:  code/numMultimap.py
    Untracked:  code/overlapMMandOrthoexon.sh
    Untracked:  code/overlapPAS.err
    Untracked:  code/overlapPAS.out
    Untracked:  code/overlapPASandOrthoexon.sh
    Untracked:  code/overlapapaQTLPAS.sh
    Untracked:  code/overlapapaQTLPAS_extended.sh
    Untracked:  code/overlapapaQTLPAS_samples.sh
    Untracked:  code/parseHg38.py
    Untracked:  code/peak2PAS.py
    Untracked:  code/pheno2countonly.R
    Untracked:  code/postiveMediation_montecarlo.R
    Untracked:  code/postiveMediation_montecarlo_DF.R
    Untracked:  code/prepareAnnoLeafviz.err
    Untracked:  code/prepareAnnoLeafviz.out
    Untracked:  code/prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/prepareLeafvizAnno.sh
    Untracked:  code/preparePAS4lift.py
    Untracked:  code/prepare_phenotype_table.py
    Untracked:  code/primaryLift.err
    Untracked:  code/primaryLift.out
    Untracked:  code/primaryLift.sh
    Untracked:  code/processhg38exons.py
    Untracked:  code/quantJunc.sh
    Untracked:  code/quantJunc_TEST.sh
    Untracked:  code/quantJunc_removeBad.sh
    Untracked:  code/quantLiftedPAS.err
    Untracked:  code/quantLiftedPAS.out
    Untracked:  code/quantLiftedPAS.sh
    Untracked:  code/quantLiftedPASPrimary.err
    Untracked:  code/quantLiftedPASPrimary.out
    Untracked:  code/quantLiftedPASPrimary.sh
    Untracked:  code/quatJunc.err
    Untracked:  code/quatJunc.out
    Untracked:  code/recChimpback2Human.err
    Untracked:  code/recChimpback2Human.out
    Untracked:  code/recLiftchim2human.sh
    Untracked:  code/revLift.err
    Untracked:  code/revLift.out
    Untracked:  code/revLiftPAShg38to19.sh
    Untracked:  code/reverseLift.sh
    Untracked:  code/runCheckReverseLift.sh
    Untracked:  code/runChimpDiffIso.sh
    Untracked:  code/runChimpDiffIsoDF.sh
    Untracked:  code/runCountNucleotides.err
    Untracked:  code/runCountNucleotides.out
    Untracked:  code/runCountNucleotides.sh
    Untracked:  code/runCountNucleotidesPantro6.err
    Untracked:  code/runCountNucleotidesPantro6.out
    Untracked:  code/runCountNucleotides_pantro6.sh
    Untracked:  code/runFilterNumChroms.sh
    Untracked:  code/runHumanDiffIso.sh
    Untracked:  code/runHumanDiffIsoDF.sh
    Untracked:  code/runNewDom.sh
    Untracked:  code/runNuclearDiffIso_DF.sh
    Untracked:  code/runNuclearDifffIso.sh
    Untracked:  code/runTotalDiffIso.sh
    Untracked:  code/run_Chimpleafcutter_ds.err
    Untracked:  code/run_Chimpleafcutter_ds.out
    Untracked:  code/run_Chimpverifybam.err
    Untracked:  code/run_Chimpverifybam.out
    Untracked:  code/run_Humanleafcutter_dF.err
    Untracked:  code/run_Humanleafcutter_dF.out
    Untracked:  code/run_Humanleafcutter_ds.err
    Untracked:  code/run_Humanleafcutter_ds.out
    Untracked:  code/run_Nuclearleafcutter_ds.err
    Untracked:  code/run_Nuclearleafcutter_ds.out
    Untracked:  code/run_Nuclearleafcutter_dsDF.err
    Untracked:  code/run_Nuclearleafcutter_dsDF.out
    Untracked:  code/run_Totalleafcutter_ds.err
    Untracked:  code/run_Totalleafcutter_ds.out
    Untracked:  code/run_chimpverifybam.sh
    Untracked:  code/run_verifyBam.sh
    Untracked:  code/run_verifybam.err
    Untracked:  code/run_verifybam.out
    Untracked:  code/slurm-62824013.out
    Untracked:  code/slurm-62825841.out
    Untracked:  code/slurm-62826116.out
    Untracked:  code/slurm-64108209.out
    Untracked:  code/slurm-64108521.out
    Untracked:  code/slurm-64108557.out
    Untracked:  code/snakePASChimp.err
    Untracked:  code/snakePASChimp.out
    Untracked:  code/snakePAShuman.out
    Untracked:  code/snakemake.batch
    Untracked:  code/snakemakeChimp.err
    Untracked:  code/snakemakeChimp.out
    Untracked:  code/snakemakeHuman.err
    Untracked:  code/snakemakeHuman.out
    Untracked:  code/snakemakePAS.batch
    Untracked:  code/snakemakePASFiltChimp.err
    Untracked:  code/snakemakePASFiltChimp.out
    Untracked:  code/snakemakePASFiltHuman.err
    Untracked:  code/snakemakePASFiltHuman.out
    Untracked:  code/snakemakePAS_Human.batch
    Untracked:  code/snakemakePASchimp.batch
    Untracked:  code/snakemakePAShuman.batch
    Untracked:  code/snakemake_chimp.batch
    Untracked:  code/snakemake_human.batch
    Untracked:  code/snakemakefiltPAS.batch
    Untracked:  code/snakemakefiltPAS_chimp.sh
    Untracked:  code/snakemakefiltPAS_human.batch
    Untracked:  code/snakemakefiltPAS_human.sh
    Untracked:  code/spliceSite2Fasta.py
    Untracked:  code/submit-snakemake-chimp.sh
    Untracked:  code/submit-snakemake-human.sh
    Untracked:  code/submit-snakemakePAS-chimp.sh
    Untracked:  code/submit-snakemakePAS-human.sh
    Untracked:  code/submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/submit-snakemakefiltPAS-human.sh
    Untracked:  code/subset_diffisopheno.py
    Untracked:  code/subset_diffisopheno_Chimp_tvN.py
    Untracked:  code/subset_diffisopheno_Chimp_tvN_DF.py
    Untracked:  code/subset_diffisopheno_Huma_tvN.py
    Untracked:  code/subset_diffisopheno_Huma_tvN_DF.py
    Untracked:  code/subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/subset_diffisopheno_Nuclear_HvC_DF.py
    Untracked:  code/subset_diffisopheno_Total_HvC.py
    Untracked:  code/test
    Untracked:  code/test.txt
    Untracked:  code/threeprimeOrthoFC.out
    Untracked:  code/threeprimeOrthoFC.sh
    Untracked:  code/threeprimeOrthoFCcd.err
    Untracked:  code/transcriptDTplotsNuclear.sh
    Untracked:  code/transcriptDTplotsTotal.sh
    Untracked:  code/verifyBam4973.sh
    Untracked:  code/verifyBam4973inHuman.sh
    Untracked:  code/verifybam4973.err
    Untracked:  code/verifybam4973.out
    Untracked:  code/verifybam4973HumanMap.err
    Untracked:  code/verifybam4973HumanMap.out
    Untracked:  code/wrap_Chimpverifybam.err
    Untracked:  code/wrap_Chimpverifybam.out
    Untracked:  code/wrap_chimpverifybam.sh
    Untracked:  code/wrap_verifyBam.sh
    Untracked:  code/wrap_verifybam.err
    Untracked:  code/wrap_verifybam.out
    Untracked:  code/writeMergecode.py
    Untracked:  data/._.DS_Store
    Untracked:  data/._HC_filenames.txt
    Untracked:  data/._HC_filenames.txt.sb-4426323c-IKIs0S
    Untracked:  data/._HC_filenames.xlsx
    Untracked:  data/._MapPantro6_meta.txt
    Untracked:  data/._MapPantro6_meta.txt.sb-a5794dd2-Cskmlm
    Untracked:  data/._MapPantro6_meta.xlsx
    Untracked:  data/._OppositeSpeciesMap.txt
    Untracked:  data/._OppositeSpeciesMap.txt.sb-a5794dd2-mayWJf
    Untracked:  data/._OppositeSpeciesMap.xlsx
    Untracked:  data/._RNASEQ_metadata.txt
    Untracked:  data/._RNASEQ_metadata.txt.sb-4426323c-TE4ns3
    Untracked:  data/._RNASEQ_metadata.txt.sb-51f67ae1-HXp7Gq
    Untracked:  data/._RNASEQ_metadata_2Removed.txt
    Untracked:  data/._RNASEQ_metadata_2Removed.txt.sb-4426323c-a4lBwx
    Untracked:  data/._RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/._RNASEQ_metadata_stranded.txt
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-a5794dd2-D659m2
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-a5794dd2-ImNMoY
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-e4bf31f0-ZGnGgl
    Untracked:  data/._RNASEQ_metadata_stranded.xlsx
    Untracked:  data/._TrialFiltersMeta.txt
    Untracked:  data/._TrialFiltersMeta.txt.sb-9845453e-R58Y0Q
    Untracked:  data/._metadata_HCpanel.txt
    Untracked:  data/._metadata_HCpanel.txt.sb-a3d92a2d-b9cYoF
    Untracked:  data/._metadata_HCpanel.txt.sb-a5794dd2-i594qs
    Untracked:  data/._metadata_HCpanel.txt.sb-f4823d1e-qihGek
    Untracked:  data/._metadata_HCpanel_frompantro5.xlsx
    Untracked:  data/._~$RNASEQ_metadata.xlsx
    Untracked:  data/._~$metadata_HCpanel.xlsx
    Untracked:  data/._.xlsx
    Untracked:  data/AREelements/
    Untracked:  data/BaseComp/
    Untracked:  data/CleanLiftedPeaks_FC_primary/
    Untracked:  data/CompapaQTLpas/
    Untracked:  data/DNDS/
    Untracked:  data/DTmatrix/
    Untracked:  data/DiffDomandDE_example/
    Untracked:  data/DiffExpression/
    Untracked:  data/DiffIso_Nuclear/
    Untracked:  data/DiffIso_Nuclear_DF/
    Untracked:  data/DiffIso_Total/
    Untracked:  data/DiffSplice/
    Untracked:  data/DiffSplice_liftedJunc/
    Untracked:  data/DiffSplice_removeBad/
    Untracked:  data/DomDefGreaterX/
    Untracked:  data/DomStructure_4/
    Untracked:  data/DominantPAS/
    Untracked:  data/DominantPAS_DF/
    Untracked:  data/EvalPantro5/
    Untracked:  data/HC_filenames.txt
    Untracked:  data/HC_filenames.xlsx
    Untracked:  data/InfoContent/
    Untracked:  data/Khan_prot/
    Untracked:  data/Li_eqtls/
    Untracked:  data/MapPantro6_meta.txt
    Untracked:  data/MapPantro6_meta.xlsx
    Untracked:  data/MapStats/
    Untracked:  data/NormalizedClusters/
    Untracked:  data/NuclearHvC/
    Untracked:  data/NuclearHvC_DF/
    Untracked:  data/OppositeSpeciesMap.txt
    Untracked:  data/OppositeSpeciesMap.xlsx
    Untracked:  data/OrthoExonBed/
    Untracked:  data/OverlapBenchmark/
    Untracked:  data/OverlappingPAS/
    Untracked:  data/PAS/
    Untracked:  data/PAS_SAF/
    Untracked:  data/PAS_doubleFilter/
    Untracked:  data/Peaks_5perc/
    Untracked:  data/Pheno_5perc/
    Untracked:  data/Pheno_5perc_DF_nuclear/
    Untracked:  data/Pheno_5perc_nuclear/
    Untracked:  data/Pheno_5perc_nuclear_old/
    Untracked:  data/Pheno_5perc_total/
    Untracked:  data/PhyloP/
    Untracked:  data/Pol2Chip/
    Untracked:  data/RNASEQ_metadata.txt
    Untracked:  data/RNASEQ_metadata_2Removed.txt
    Untracked:  data/RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/RNASEQ_metadata_stranded.txt
    Untracked:  data/RNASEQ_metadata_stranded.txt.sb-e4bf31f0-ZGnGgl/
    Untracked:  data/RNASEQ_metadata_stranded.xlsx
    Untracked:  data/SignalSites/
    Untracked:  data/SignalSites_doublefilter/
    Untracked:  data/SpliceSite/
    Untracked:  data/TestAnnoBiasOE/
    Untracked:  data/TestMM2/
    Untracked:  data/TestMM2_AS/
    Untracked:  data/TestMM2_PrimaryRead/
    Untracked:  data/TestMM2_SeondaryRead/
    Untracked:  data/TestMM2_mismatch/
    Untracked:  data/TestMM2_quality/
    Untracked:  data/TestWithinMergePAS/
    Untracked:  data/Test_FC_methods/
    Untracked:  data/Threeprime2Ortho/
    Untracked:  data/TotalFractionPAS/
    Untracked:  data/TotalHvC/
    Untracked:  data/TrialFiltersMeta.txt
    Untracked:  data/TwoBadSampleAnalysis/
    Untracked:  data/Wang_ribo/
    Untracked:  data/apaQTLGenes/
    Untracked:  data/bioGRID/
    Untracked:  data/chainFiles/
    Untracked:  data/cleanPeaks_anno/
    Untracked:  data/cleanPeaks_byspecies/
    Untracked:  data/cleanPeaks_lifted/
    Untracked:  data/files4viz_nuclear/
    Untracked:  data/files4viz_nuclear_DF/
    Untracked:  data/gviz/
    Untracked:  data/leafviz/
    Untracked:  data/liftover_files/
    Untracked:  data/mediation/
    Untracked:  data/mediation_DF/
    Untracked:  data/metadata_HCpanel.txt
    Untracked:  data/metadata_HCpanel.xlsx
    Untracked:  data/metadata_HCpanel_extra.txt
    Untracked:  data/metadata_HCpanel_frompantro5.txt
    Untracked:  data/metadata_HCpanel_frompantro5.xlsx
    Untracked:  data/miRNA/
    Untracked:  data/multimap/
    Untracked:  data/orthoUTR/
    Untracked:  data/paiDecay/
    Untracked:  data/primaryLift/
    Untracked:  data/reverseLift/
    Untracked:  data/testQuant/
    Untracked:  data/~$RNASEQ_metadata.xlsx
    Untracked:  data/~$metadata_HCpanel.xlsx
    Untracked:  data/.xlsx
    Untracked:  output/._.DS_Store
    Untracked:  output/dAPAandDomEnrich.png
    Untracked:  output/dEandDomEnrich.png
    Untracked:  output/dtPlots/
    Untracked:  projectNotes.Rmd
    Untracked:  proteinModelSet.Rmd

Unstaged changes:
    Modified:   analysis/DiffTop2SecondDom.Rmd
    Modified:   analysis/ExploredAPA.Rmd
    Modified:   analysis/ExploredAPA_DF.Rmd
    Modified:   analysis/MMExpreiment.Rmd
    Modified:   analysis/OppositeMap.Rmd
    Modified:   analysis/PTM_analysis.Rmd
    Modified:   analysis/TotalDomStructure.Rmd
    Modified:   analysis/TotalVNuclearBothSpecies.Rmd
    Modified:   analysis/annotationInfo.Rmd
    Modified:   analysis/changeMisprimcut.Rmd
    Modified:   analysis/comp2apaQTLPAS.Rmd
    Modified:   analysis/correlationPhenos.Rmd
    Modified:   analysis/establishCutoffs.Rmd
    Modified:   analysis/investigatePantro5.Rmd
    Modified:   analysis/mRNADecay.Rmd
    Modified:   analysis/multiMap.Rmd
    Modified:   analysis/pol2.Rmd
    Modified:   analysis/signalsites_doublefilter.Rmd
    Modified:   analysis/speciesSpecific.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/38b161983ba8d19911dc9fc43e8376ffa8376322/analysis/index.Rmd" target="_blank">38b1619</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-28
</td>
<td>
add dapa and info
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/448aa08e5082af854cce17a9c996bc6bc2f58580/docs/index.html" target="_blank">448aa08</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/93d00f37b4fce84278e01296321b01b441ae5638/analysis/index.Rmd" target="_blank">93d00f3</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-24
</td>
<td>
add info content
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/53a75707083d909ad1c9cf54e2a9c8f94ce45bc3/docs/index.html" target="_blank">53a7570</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b4e617ede962869e7ef3784c37a96f64a025a527/analysis/index.Rmd" target="_blank">b4e617e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-22
</td>
<td>
add color and prop dom, add decay
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/9d14c53f8a8dddef8982cb7e8d4f879ab0005e1b/docs/index.html" target="_blank">9d14c53</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/22071284efc0f4f3dff04a2e9e844928fc9f89fd/analysis/index.Rmd" target="_blank">2207128</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-22
</td>
<td>
add prot to dom and add effect size plots
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/d01cd841bfe66abba07ec897219a369335430ab3/docs/index.html" target="_blank">d01cd84</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/0f53338a6d3957471d0d86ff5625708b2d192f1f/analysis/index.Rmd" target="_blank">0f53338</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-21
</td>
<td>
add de and diff dom mech
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/fbc65999dd3844de9ac467ff3fbf6e03c9f2a3a1/docs/index.html" target="_blank">fbc6599</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/95685efb3a0c1ea3b1cbb510c7ed5c6f71862162/analysis/index.Rmd" target="_blank">95685ef</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-21
</td>
<td>
add length diff analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/a60094e6c07cea1e1dab986fd0bd36b27da8e301/docs/index.html" target="_blank">a60094e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/2b63d021b10c674a9631737203bfb525ff86a161/analysis/index.Rmd" target="_blank">2b63d02</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-21
</td>
<td>
add new dom and de/dapa
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/3e42f78a90ab3cd1ed1920d511b8f0ac94e0b9b0/docs/index.html" target="_blank">3e42f78</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b00be28e9ab1b2d97da47bb5b2b4baedf1e99f5b/analysis/index.Rmd" target="_blank">b00be28</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-20
</td>
<td>
add new dom to anaylses
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/25d194e5612a370eeaf8dea9f0f68aa17664f246/docs/index.html" target="_blank">25d194e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/40ba921ed1ecfa5cc10bf71715d1bdb53d28916f/analysis/index.Rmd" target="_blank">40ba921</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-20
</td>
<td>
add effect size with dom
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/1dc519ab5e05ba948757bcda028136895a2271ae/docs/index.html" target="_blank">1dc519a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/9f8e75fc4bd144eb4cb016469f6a58b675fb4e26/analysis/index.Rmd" target="_blank">9f8e75f</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-17
</td>
<td>
add enrich
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/8f48b3a9b0d58d26cb5af9baf053d5c74358bdaa/docs/index.html" target="_blank">8f48b3a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1d7b9d22d58b2a494ef3bd65f04dddd2b5804702/analysis/index.Rmd" target="_blank">1d7b9d2</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-17
</td>
<td>
add diff top 2 sec
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/218a3d40d59418b055c05563e54f9ed0ccf5057b/docs/index.html" target="_blank">218a3d4</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/2461fb7bf628afc5e809ef902569507efeb154c2/analysis/index.Rmd" target="_blank">2461fb7</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-16
</td>
<td>
new dom integration
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/38c6dc941bb83e3a5bca26dd0cb022935d642f96/docs/index.html" target="_blank">38c6dc9</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/2d0668ed07656cd2a04fdcf4fc731946173802c2/analysis/index.Rmd" target="_blank">2d0668e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-14
</td>
<td>
add new way to call dom
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/edd894076e37f6683d3c43c9355e92787f1de2ff/docs/index.html" target="_blank">edd8940</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/6997920be291f00cace4847fbda6a4a6cf5fab51/analysis/index.Rmd" target="_blank">6997920</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-13
</td>
<td>
initial miRNA
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/88b3edd9c21f8a347bb6e7ac5f52df2cbb67f073/docs/index.html" target="_blank">88b3edd</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c5fc23ced81250f2148f0c8458c5aa01ddbe0aeb/analysis/index.Rmd" target="_blank">c5fc23c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-13
</td>
<td>
add long short info
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/a7e29b9be53d534be7cb0fd16adecb1a3c4d0335/docs/index.html" target="_blank">a7e29b9</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/014951a93df8e8a1c6477562f9706689556b7466/analysis/index.Rmd" target="_blank">014951a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-10
</td>
<td>
remove 18499
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/83ab280e1444789b5405826102c2694246aea6d4/docs/index.html" target="_blank">83ab280</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/21e8a4f1f237b21c7cbec06a0cf139524512c08f/analysis/index.Rmd" target="_blank">21e8a4f</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-09
</td>
<td>
remove ind map
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/7968e3cc48ff78cbb3c36900be4870b775fb4f77/docs/index.html" target="_blank">7968e3c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/fc93eb98ab4c8209440226924fd0d4d2358b3fef/analysis/index.Rmd" target="_blank">fc93eb9</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-09
</td>
<td>
red for no 18499
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/60a370d126196d603b90bceff513416c6b162b3f/docs/index.html" target="_blank">60a370d</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/4fd390333e1cecf510e2e107f3e164ff2f9ebbb6/analysis/index.Rmd" target="_blank">4fd3903</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-08
</td>
<td>
add corr betweenfrac
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/4c131c9daf9f07bac3f38d08eb3ffb5b0d8cc2ce/docs/index.html" target="_blank">4c131c9</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/2f5d4d5ce06bcc8615008906ad97f1e0d1376607/analysis/index.Rmd" target="_blank">2f5d4d5</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-08
</td>
<td>
change number and add 3; utr
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/e45850ef849995805158ec801c9e05c97bfa4942/docs/index.html" target="_blank">e45850e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/856c506b99e9f2d20389f0254dc02a5409dd8a08/analysis/index.Rmd" target="_blank">856c506</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-08
</td>
<td>
fix labels
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/5e58f0335ba75db658a38da1a8e3c4a3b8a98385/docs/index.html" target="_blank">5e58f03</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/d81b1488b71ae6f4d9d7e1685c1e769112e5ff2c/docs/index.html" target="_blank">d81b148</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a672b029e3419ec44cf5b6c7e4c9cdd08f917fe7/analysis/index.Rmd" target="_blank">a672b02</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-07
</td>
<td>
numb dist for dom
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/7877726ba67911a574dd2203a82cd172ee1ae182/docs/index.html" target="_blank">7877726</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/81e9838116418443ebbbedf4ef26e2d4a1b1bf6c/analysis/index.Rmd" target="_blank">81e9838</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-07
</td>
<td>
add pol2 initial data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/b3cce4d894defd81a4e11baf723acf6cd1ce7ece/docs/index.html" target="_blank">b3cce4d</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/71101c1a6c37f5a2b91c240ebfeb429399cc59fc/analysis/index.Rmd" target="_blank">71101c1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-06
</td>
<td>
add dom pattern and de
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/a2818491b8228361248ac5ce624fbfbce3d22d0f/docs/index.html" target="_blank">a281849</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/637094ae060a6fd09f126b8f516b20197fbbbeff/analysis/index.Rmd" target="_blank">637094a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-06
</td>
<td>
look at total dominance
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/c2a07789889eddb977de30d8a838db0ef57758fb/docs/index.html" target="_blank">c2a0778</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/9676d72d60e2f9c82d2bfed1d7634d306b9ef956/analysis/index.Rmd" target="_blank">9676d72</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-06
</td>
<td>
updated anno
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/2242e4af582d177c2cef86a1e242760c9246c205/docs/index.html" target="_blank">2242e4a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/92f06e4499cd41e5172d560e481c51065573db7c/analysis/index.Rmd" target="_blank">92f06e4</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-06
</td>
<td>
update anno
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/c59a7de004937a8b96169f972b7108618e8010c1/docs/index.html" target="_blank">c59a7de</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/68c949f0f5b48425e78c9e5ab2f26088bc55152e/analysis/index.Rmd" target="_blank">68c949f</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-05
</td>
<td>
update post sno
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/065ffdc236115d0db323bc96d42c0bdfc950b5fa/docs/index.html" target="_blank">065ffdc</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/8269a9ae272c6d13ffce9d80b2da7781cac29526/analysis/index.Rmd" target="_blank">8269a9a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-03
</td>
<td>
update post sno
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/72ab9ea72e610546e5c540bb76b1be1bb5311b52/docs/index.html" target="_blank">72ab9ea</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/767bb782152fa84112ee7560fdc8a3df2bbbff93/analysis/index.Rmd" target="_blank">767bb78</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-03
</td>
<td>
update anno
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/4722051162d1da99151a9e122e69c5a63f3ea488/docs/index.html" target="_blank">4722051</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/9ba1428b897204366c9b0ffada85650e8110abd1/analysis/index.Rmd" target="_blank">9ba1428</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-02
</td>
<td>
add start of merge analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/3b7192e14c1cdf6cc3fb3868ac323d9ffa40bf77/docs/index.html" target="_blank">3b7192e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/788b72cc6d8a7851557d2ead790d897e2ab4ba18/analysis/index.Rmd" target="_blank">788b72c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-02
</td>
<td>
add list of problems
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/04171bc4faaf9f994201944cb5c68ecfd36b92b9/docs/index.html" target="_blank">04171bc</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c76175755b1215934cf2f9a983035f4538d4c4dc/analysis/index.Rmd" target="_blank">c761757</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-02
</td>
<td>
add normalized
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/348d9fe80e275cee7c295f5a165fa4e6b07b85cc/docs/index.html" target="_blank">348d9fe</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-01
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/06793995403ddc8eae4956194436db6cb1695481/analysis/index.Rmd" target="_blank">0679399</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-01
</td>
<td>
add no tie analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/8cb8b0a9d222c13d8de89c686209c47931ce085b/docs/index.html" target="_blank">8cb8b0a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-01
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/7447f2ddc3110b73a8cc2409a58eb38d7869754f/analysis/index.Rmd" target="_blank">7447f2d</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-01
</td>
<td>
add norm dom and add filter comp plots
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/1274a02b6e4dc91f6b42eb08ea4ffcee9fccc857/docs/index.html" target="_blank">1274a02</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/205249491f9d1af116829c18a7bb728c810402af/analysis/index.Rmd" target="_blank">2052494</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-30
</td>
<td>
add dom with primary
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/a70f2fe686bf3bdfef8a251dbef22938f9d6d33d/docs/index.html" target="_blank">a70f2fe</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/706fa0d50038656a0ed1504362f62640173ba444/analysis/index.Rmd" target="_blank">706fa0d</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-28
</td>
<td>
add mm exp code
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/3d4dd4722d951a6205c8db1fdf2d6a513b4b402e/docs/index.html" target="_blank">3d4dd47</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f92b89c48e3cdd34486ac237f56bef2d9e8acd5f/analysis/index.Rmd" target="_blank">f92b89c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-26
</td>
<td>
add ortho exon and new mm
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/10fb0fb29d160aee177fb2717e726475fb82b8cf/docs/index.html" target="_blank">10fb0fb</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/6adb4d85f679095d48e320e099e651624b0aeb88/analysis/index.Rmd" target="_blank">6adb4d8</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
<td>
add check res no MM
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/f1800640456b1924a6f3f54553a6d0f90ae9361f/docs/index.html" target="_blank">f180064</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/73127b5d2d6bc20d96fd837161ced3a95dc9aac3/analysis/index.Rmd" target="_blank">73127b5</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-24
</td>
<td>
add con and dom
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/eebb3195e2bdff8d2afd9fe72e59cfb2f5f110d7/docs/index.html" target="_blank">eebb319</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/81d06e8f872581c03c2c4d78e0798cdf02b09c0b/analysis/index.Rmd" target="_blank">81d06e8</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
<td>
add remove sno analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/246d60c22c9418dce848d46bf9ce2808ad3386e4/docs/index.html" target="_blank">246d60c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b6ccdbcbc61bfb23d707cbf1be10f9a6b23e0b1c/analysis/index.Rmd" target="_blank">b6ccdbc</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-19
</td>
<td>
add new quat
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/020ff7bc87a4b35ddc3b49eb0a27482b2359fa9d/docs/index.html" target="_blank">020ff7b</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/01f2aba96b3741a6aa8bf6c46ac75a0d89ebcca7/analysis/index.Rmd" target="_blank">01f2aba</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-19
</td>
<td>
add start to coding pas explore
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/050da80aac2ecbabd17af86f27b46355390c6291/docs/index.html" target="_blank">050da80</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/536958cb0ce8b0d05d99eee60c601330c578092b/analysis/index.Rmd" target="_blank">536958c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-18
</td>
<td>
add res for overlapping PAS
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/74e186a5f5ff98cfbb5cd5c5300cc5ccc5fe0a91/docs/index.html" target="_blank">74e186a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/e85defb5bb6deb2b7516cb7f0746ca3d30a83636/analysis/index.Rmd" target="_blank">e85defb</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-17
</td>
<td>
add res from diff mis prime filters
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/63212bb9e42efe9dfd1130e0f1c992c260061b08/docs/index.html" target="_blank">63212bb</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/bec8ae9758b7e17349f2f939a0432eebfd9e9a6f/analysis/index.Rmd" target="_blank">bec8ae9</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-17
</td>
<td>
add mis filter annotation and pheno
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/45bfe96f8465b20080af82873ce3deb8a3fb2373/docs/index.html" target="_blank">45bfe96</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c4282a6c121764fc4ac160c23ddb91ecd703007b/analysis/index.Rmd" target="_blank">c4282a6</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-17
</td>
<td>
add change filter lifts
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/86238eb4617ff71668af2d726c1adbb078fbde3e/docs/index.html" target="_blank">86238eb</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/164c2371d5d45057c0027bdd37a66b083bc7cf74/analysis/index.Rmd" target="_blank">164c237</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-16
</td>
<td>
ask about anno bias
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/30bfaaa1adfa227ee39e77c5fcf917edd55f8d35/docs/index.html" target="_blank">30bfaaa</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/94211e8035a377ea874ffe699eeac000dc528227/analysis/index.Rmd" target="_blank">94211e8</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-16
</td>
<td>
add change misprime analysis and dapa qtls
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/3bcfc08ab51beeb5f9590ab7ce6b46057cedd7ce/docs/index.html" target="_blank">3bcfc08</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/6f7dfd7f4ef4e6d58abd387a60a9998b39358d21/analysis/index.Rmd" target="_blank">6f7dfd7</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-14
</td>
<td>
add basecomp analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/d1e93513b0e0e6fea308ede3b1dff88f7d275d98/docs/index.html" target="_blank">d1e9351</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/044572086797f9e3dd039853277deb0ed08fb7c3/analysis/index.Rmd" target="_blank">0445720</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-13
</td>
<td>
add map stats when you remove Ns
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/51190b4b3c89f607785c41b79b5c3763af9bf690/docs/index.html" target="_blank">51190b4</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/f1ec3e0777a070010b8894f265fdc3936c2bc95e/docs/index.html" target="_blank">f1ec3e0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/552842c2e00d7ca66f9b4426a834cb0c8e7c7a3a/analysis/index.Rmd" target="_blank">552842c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-09
</td>
<td>
add TvN
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/25a479024019e85622f53af5b0feb82c735785b3/docs/index.html" target="_blank">25a4790</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c6a11f00dbcd38e20964f4a8d82d690d2004ae12/analysis/index.Rmd" target="_blank">c6a11f0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-07
</td>
<td>
add expression indep
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/22926ec6140cb1e4a6866acbb4e56a09983013a7/docs/index.html" target="_blank">22926ec</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/680d435cd9ebc983d9448ddf0ee8e6a9c18dbe11/analysis/index.Rmd" target="_blank">680d435</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-06
</td>
<td>
add ptm analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/a1fd883fda88dea7e55f5d53ee8824e5ecfbe9e4/docs/index.html" target="_blank">a1fd883</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a71e15c015c2de7e0d4822100d35ae2f388bc7ee/analysis/index.Rmd" target="_blank">a71e15c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-05
</td>
<td>
wflow_publish(c(“analysis/index.Rmd”, “analysis/DirSelectionKhan.Rmd”))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/7b73ce76c443223d7c879bf066fa2b6c667419f2/docs/index.html" target="_blank">7b73ce7</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/ad952719ac1bed60805fbcff35b349ed1ba5831a/analysis/index.Rmd" target="_blank">ad95271</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-04
</td>
<td>
add upstream control
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/09ad48202c9b6cca3c08aa15f8d8af777526c6f9/docs/index.html" target="_blank">09ad482</a>
</td>
<td>
brimittleman
</td>
<td>
2020-02-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/7385496764bfb27c830cf11223f138997a0f20ad/analysis/index.Rmd" target="_blank">7385496</a>
</td>
<td>
brimittleman
</td>
<td>
2020-02-24
</td>
<td>
add correlations plotted by location
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/5f821eeff2d8a089e1561a65b9c77ac3c0aff7f1/docs/index.html" target="_blank">5f821ee</a>
</td>
<td>
brimittleman
</td>
<td>
2020-02-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f4ae857f8fbacb81df9398e779042bda1c749282/analysis/index.Rmd" target="_blank">f4ae857</a>
</td>
<td>
brimittleman
</td>
<td>
2020-02-23
</td>
<td>
wflow_publish(c(“analysis/index.Rmd”, “analysis/DiffUsedIntronic.Rmd”))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/5bcde2f165fa7f4dc6b260aa01644ed70b1b1759/docs/index.html" target="_blank">5bcde2f</a>
</td>
<td>
brimittleman
</td>
<td>
2020-02-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/2e8ed44a32216fa67ac83880a5e30b7bf6d5c267/analysis/index.Rmd" target="_blank">2e8ed44</a>
</td>
<td>
brimittleman
</td>
<td>
2020-02-21
</td>
<td>
add Splice site strength
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/f02217e9833c9e0f9b4041d3ea870e387f8c9b17/docs/index.html" target="_blank">f02217e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/86aabf27b3f3489bf4342ea9585753cedf3b9e31/analysis/index.Rmd" target="_blank">86aabf2</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-28
</td>
<td>
add expression overlap with APA exploration
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/b1b07d51e1e9512fbd668da46dc9dacd51c9f3ae/docs/index.html" target="_blank">b1b07d5</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/9642296adbf8ed4ab2422b035be8cb5ee06377a2/docs/index.html" target="_blank">9642296</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1673f189ffa704d689413d623e9ae7821e6c706c/analysis/index.Rmd" target="_blank">1673f18</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-23
</td>
<td>
start the prot mediation cant do diff indiv
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/838da9ba6d3ca6ea1f2a42821c15e65f5abb795a/docs/index.html" target="_blank">838da9b</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a018c33ff0177b4b40492a343f1d63f658932766/analysis/index.Rmd" target="_blank">a018c33</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-23
</td>
<td>
add diff usgage and ss
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/2d84fb1cc3b71c3f710c07e4b29bcf9dfb3c59b8/docs/index.html" target="_blank">2d84fb1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/908f02d60a4a84b5ace4dd57246881825135d217/analysis/index.Rmd" target="_blank">908f02d</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-23
</td>
<td>
add compare filter and write out for top SS
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/8a24b5986baf0058f225536dd327294fe71b54c0/docs/index.html" target="_blank">8a24b59</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/97ef45e819253bd06708ea181f98f23121f9dbec/analysis/index.Rmd" target="_blank">97ef45e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-22
</td>
<td>
add mediation DF
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/f2421a45df5fc112608285c9b459a1d9efda5fb2/docs/index.html" target="_blank">f2421a4</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/d8868667b91be9b09bcca0ee8c6adba4d69b6f96/analysis/index.Rmd" target="_blank">d886866</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-22
</td>
<td>
add correlation between pheno DF
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/8536528851f483bf8a720d0eba211af89085aa55/docs/index.html" target="_blank">8536528</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/07fd45952afd6e6586320a0f6ae9230c1e3f7ff0/analysis/index.Rmd" target="_blank">07fd459</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-22
</td>
<td>
DF for upset and translation protein ven
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/58002317b3430a3dfa938cc9e111cce4f41f77a4/docs/index.html" target="_blank">5800231</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/117fd632969209613fed6fed461293da502a2986/analysis/index.Rmd" target="_blank">117fd63</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
redo differential analysis with double filt
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/f00297802381bb687f60fb10c2ccea7e9cae8280/docs/index.html" target="_blank">f002978</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/46c489bf544f38874f9e7e15db4dbbd640136cb2/analysis/index.Rmd" target="_blank">46c489b</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
nuclear diff filtered
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/6c39a2a7656f78c3cef5b7cda1a5163d19f548f2/docs/index.html" target="_blank">6c39a2a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/010c249a1fac4b9e302603953455894bfd148d9e/analysis/index.Rmd" target="_blank">010c249</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
DF species specific
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/48db342468338de2ec45be8692fa7ff7aee80118/docs/index.html" target="_blank">48db342</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/414b47c494215de92521c0276f3847bd092ae798/analysis/index.Rmd" target="_blank">414b47c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
update dominant PAS
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/5525b39f555800466e8d97a5ca8e849940f68ac7/docs/index.html" target="_blank">5525b39</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/2e66af99c18acdebb901d0e367eeec84e940fbde/analysis/index.Rmd" target="_blank">2e66af9</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
add ss and PAS num DF
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/80f647a5782e2b58660ae49c0bb5d4a8cee5efbf/docs/index.html" target="_blank">80f647a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/96b46ba75ca9a73b88e4c38e83dc4b949baf42d6/analysis/index.Rmd" target="_blank">96b46ba</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-19
</td>
<td>
add enrichment in apapQTLs
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/c211e215cc82898901dd43dc9cfac50fbf10154d/docs/index.html" target="_blank">c211e21</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/7a9f60879915ace82b872f58d7209efd957b93a5/analysis/index.Rmd" target="_blank">7a9f608</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-18
</td>
<td>
finished benchmark
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/5eef3eb638ca2bf1784cce27cdf859279202a165/docs/index.html" target="_blank">5eef3eb</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/5c24c0cad611959f9d9c0266b270a52242768642/analysis/index.Rmd" target="_blank">5c24c0c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-16
</td>
<td>
add cutoff code files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/7e3ab25222fa694c8b875389b0ff50beea884583/docs/index.html" target="_blank">7e3ab25</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c0b3ef6823219aae50e3b6e8c85f9b1ba7ac9a79/analysis/index.Rmd" target="_blank">c0b3ef6</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-16
</td>
<td>
add correlation between pheons
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/0a3545a5989e5163fbd6abaf57a9715cdd295541/docs/index.html" target="_blank">0a3545a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/5dd5e3520a1db76aac47d0cd2ff68516635a8be6/analysis/index.Rmd" target="_blank">5dd5e35</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-15
</td>
<td>
add code to viz nuclear dapa
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/4f5d4929bd9915b744db58348b0e6abd8dbf17e9/docs/index.html" target="_blank">4f5d492</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/3cce3da7f1e29f6354c8ba13cfbf5670d31b266f/analysis/index.Rmd" target="_blank">3cce3da</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-14
</td>
<td>
start mediation analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/c74c3dff8fe4e0f556c85f6e87a52b98af045052/docs/index.html" target="_blank">c74c3df</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/e1c6d25cab3b2417ff0cb817baa5b2ce20c2bc18/analysis/index.Rmd" target="_blank">e1c6d25</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-14
</td>
<td>
add analysis on species spec PAS
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/1b9ae04751591afa424b45746658ede0a2fe8265/docs/index.html" target="_blank">1b9ae04</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/5ea47104fe4b593dc225c6982b9287a75d033568/analysis/index.Rmd" target="_blank">5ea4710</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-13
</td>
<td>
add questions about dAPA pas and genes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/b963392cd54baea2025e643b0f5c7fa5b8a22ce2/docs/index.html" target="_blank">b963392</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1121102d5c13611abe70c2f43c03cea96e44fbc0/analysis/index.Rmd" target="_blank">1121102</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-13
</td>
<td>
add dpau and de overlap with dom intronic
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/df07f1ae847e014b069b024db3499632d98b7199/docs/index.html" target="_blank">df07f1a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/928a30dd19be22274470defdc5d88b3e87a05048/analysis/index.Rmd" target="_blank">928a30d</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
add same and diff dom ss analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/ced196676b623df77391971e34c2804b6ec4d087/docs/index.html" target="_blank">ced1966</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1cdc21d6416fd4732da758fe59f13769a5a06c29/analysis/index.Rmd" target="_blank">1cdc21d</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
add initial signal site analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/dd02dabec8be8444f027abc10b667f1c6e8c5f3a/docs/index.html" target="_blank">dd02dab</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/a41141b42e3d041fb7643c6b4fe21e9144dddc58/docs/index.html" target="_blank">a41141b</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/93ced1e42aac3e088f3a548d93f2ca501a5b0736/analysis/index.Rmd" target="_blank">93ced1e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
peak 2 pas
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/6d9e3694a2b7c72dc2ed9d8647c74328504d1602/docs/index.html" target="_blank">6d9e369</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/203eae574eb54842cf37786d93397c01153ea3b3/analysis/index.Rmd" target="_blank">203eae5</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-08
</td>
<td>
first steps for intron loc analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/4c87e962435f44f7059b09a4b9647ac5aa9ab9b2/docs/index.html" target="_blank">4c87e96</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/6e3027ad7348b369c091f8fb903edfee7cd13b3d/analysis/index.Rmd" target="_blank">6e3027a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-07
</td>
<td>
upset plots for comparisons
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/776853582da60e8c2f6bb43a1a0b14e339c47463/docs/index.html" target="_blank">7768535</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/670020f30d49aae921fa8368b8e7db48a43d4ce5/analysis/index.Rmd" target="_blank">670020f</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-07
</td>
<td>
add translation protien overlap
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/dec23c17b58af879e35616b4e87db89ea5b422fe/docs/index.html" target="_blank">dec23c1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/54015c83b7d5a5583ee4c3cba8e9f12734ca0000/analysis/index.Rmd" target="_blank">54015c8</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-06
</td>
<td>
add overlap dom with dAPA
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/c633e63b7908ec3c5e4a1dd03454495fe8d80f38/docs/index.html" target="_blank">c633e63</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/5e857040d3c1c7aa994183714a5c0a7c84632596/analysis/index.Rmd" target="_blank">5e85704</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-06
</td>
<td>
add overlap with eQTLs
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/dc41b3bed4b09ecfbadc5582fe9344c478a36f26/docs/index.html" target="_blank">dc41b3b</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/6214ec5ac07800db9fcab129908c163965822c19/analysis/index.Rmd" target="_blank">6214ec5</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-30
</td>
<td>
add venn diagram for diff apa
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/42a8b12141c2eaa3041cbbb71c6923e210c46dc6/docs/index.html" target="_blank">42a8b12</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/385057f01b6b0ca687e6e47d3a4bc8203d56768b/analysis/index.Rmd" target="_blank">385057f</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-30
</td>
<td>
fix dom PAS on index
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/0176b490c9d47ce9a1a57debcb33dd33a8678879/docs/index.html" target="_blank">0176b49</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/5b25363cd6afdf6d3159d57a65fb785318f7fab7/analysis/index.Rmd" target="_blank">5b25363</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-30
</td>
<td>
add total and nuclear dominant PAS analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/26447479c3188b6643bf89ec83ac1c4042b4369b/docs/index.html" target="_blank">2644747</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/4c8973d5a093a6450fa78c96b3fc524bb3ca971a/analysis/index.Rmd" target="_blank">4c8973d</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-27
</td>
<td>
add total human vs chimp
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/ffecde802b87c4d1b1578945db9807fab043f0b4/docs/index.html" target="_blank">ffecde8</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/d541bb5b0b501844781d1db3cb73f9182dc492d6/analysis/index.Rmd" target="_blank">d541bb5</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-17
</td>
<td>
move extra pcas
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/8b074b88d8eddfe32f775bf287b0ee2308d0a16e/docs/index.html" target="_blank">8b074b8</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/de41e07c5a865e0fe881b5301917b46a3213b9ef/analysis/index.Rmd" target="_blank">de41e07</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-16
</td>
<td>
add gene comp 5v6
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/1b69793cd30f749ee4b0ba21d45783b7752cc282/docs/index.html" target="_blank">1b69793</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/3b64d8c2dfa1c6a7c2c5e3687a09e2963987fed6/analysis/index.Rmd" target="_blank">3b64d8c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-16
</td>
<td>
add compare ortho exon
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/cbadb32d4f2c53f97ffab3f48cd89c838f4ed583/docs/index.html" target="_blank">cbadb32</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/8068598fcdb9515d4c3f7b6b429971c0d2c7eec7/analysis/index.Rmd" target="_blank">8068598</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-12
</td>
<td>
meta data by species
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/3ee838285a224d7b74f683ccc93ff29a21a4333e/docs/index.html" target="_blank">3ee8382</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f04a8042f4b698ab19e130b931b5aa8075a125a4/analysis/index.Rmd" target="_blank">f04a804</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-08
</td>
<td>
add DE vs DS analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/b32d961465fc2811af401d0ac4ff09dee895107c/docs/index.html" target="_blank">b32d961</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1452408d077ed41af636026bbe45b5aaf6066893/analysis/index.Rmd" target="_blank">1452408</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-05
</td>
<td>
add investigation pantro5
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/144bc43946f728c856c1039f0d8b7d0f6eb6e936/docs/index.html" target="_blank">144bc43</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/d56d98cb3aa2e4768d272c382017bdc8889a06ed/analysis/index.Rmd" target="_blank">d56d98c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-05
</td>
<td>
add opp map
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/f53ab2772e3e897295f109389cb1cf19a7d71c4d/docs/index.html" target="_blank">f53ab27</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c48aedcbe8688bc762be9e2aa901a0a50606c3fc/analysis/index.Rmd" target="_blank">c48aedc</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-23
</td>
<td>
add multimap stat graphs
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/e7cb428de68cdf9db814b7fb859caa0e954338f1/docs/index.html" target="_blank">e7cb428</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/9fbd1fe61d1a16783614fab355cc522bf11cb279/analysis/index.Rmd" target="_blank">9fbd1fe</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-21
</td>
<td>
figure out sample swap
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/2e33ce93a1227b8cb84cacd5b3d914dcab1fc71c/docs/index.html" target="_blank">2e33ce9</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/17d5d33beaa8e717b613ff408d8948415d559837/analysis/index.Rmd" target="_blank">17d5d33</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-21
</td>
<td>
add diff splice without 2 samples
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/40db68c19e220862b8f27ddadb157ad7f9f17bd8/docs/index.html" target="_blank">40db68c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/7c206f8564758f532df70a03de7ba08ae322a15c/analysis/index.Rmd" target="_blank">7c206f8</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-19
</td>
<td>
add extra QC anaylsis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/9ecd7691d60ca3c6b742c1f3a4956f51832bd63e/docs/index.html" target="_blank">9ecd769</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/db85b26c94005e7cad373dc524afb78f7f2ad8fb/analysis/index.Rmd" target="_blank">db85b26</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-19
</td>
<td>
add remove 2 analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/dc91b0af79367ef559ad277a01090d4c950dcc88/docs/index.html" target="_blank">dc91b0a</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b5ba82eee24903f1da5a92da2559275ffc71912d/analysis/index.Rmd" target="_blank">b5ba82e</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-11
</td>
<td>
add diff expression and diff splicing
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/066d419290a2f4c3eef8bdf538216f3d348572fb/docs/index.html" target="_blank">066d419</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a255e22b9d32ad80884501ae03a7c4e162fdbba5/analysis/index.Rmd" target="_blank">a255e22</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-16
</td>
<td>
add code and annotations for dt plots
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/f4bcae9ac5f32e2f228bd29eaaf2ae525899ce22/docs/index.html" target="_blank">f4bcae9</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/25a8b1e42180e62d71b4ff4dd0400102843f1cec/analysis/index.Rmd" target="_blank">25a8b1e</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-15
</td>
<td>
fix name bug add number PAS analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/d9f241f464a962076d2b3378fc27d4167217ae49/docs/index.html" target="_blank">d9f241f</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/36d8ded5078c72dbd927d0f8c1c26d5febef08bd/analysis/index.Rmd" target="_blank">36d8ded</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-11
</td>
<td>
add correlation between ind analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/3f3c4d0bbcd2e09211f2fea1ac28e58777e3d00f/docs/index.html" target="_blank">3f3c4d0</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c4db4f0e862f52d0ff492d5ca928b9bc0ae2b345/analysis/index.Rmd" target="_blank">c4db4f0</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-10
</td>
<td>
add non norm pca and verify bam
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/b8ccfdba858ba7a58fd036f32380c6c91d513263/docs/index.html" target="_blank">b8ccfdb</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1f063297413d5951470981d2eb95540825326ce0/analysis/index.Rmd" target="_blank">1f06329</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-09
</td>
<td>
add PCA by total and nuclear
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/d0c98c261010b35d00259c9ca5488e2e46448332/docs/index.html" target="_blank">d0c98c2</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/14a3f665ce75def8370c8c6129603a205c5648d0/analysis/index.Rmd" target="_blank">14a3f66</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-09
</td>
<td>
add pca and human v chimp in nuc analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/c3aa529bce951a381624b7854c6e7522f3c14bff/docs/index.html" target="_blank">c3aa529</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/da85ad8c1c69856141190009e20c4ef98ed07b76/analysis/index.Rmd" target="_blank">da85ad8</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-07
</td>
<td>
add code to prepare human and chimp TvN
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/4ea25767ec9bed8fe46e366e324043d228e57f5e/docs/index.html" target="_blank">4ea2576</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/920abac02a19940bb87f056a3a06dcb35cd5016d/analysis/index.Rmd" target="_blank">920abac</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-04
</td>
<td>
add comaprison to old PAS
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/e0ac227e5aa1b09a1da0fa75aaab25b9f0eb749a/docs/index.html" target="_blank">e0ac227</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/e3f0cdffde5f9e8344479c42d2e16e725e73b22e/analysis/index.Rmd" target="_blank">e3f0cdf</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-03
</td>
<td>
add annotation analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/b5edd8e21fc0cf262cf1332d74ffc1917c2ff7f7/docs/index.html" target="_blank">b5edd8e</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-01
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b5a215119477fca80ff396449a2cbbbff36db2d1/analysis/index.Rmd" target="_blank">b5a2151</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-01
</td>
<td>
add analysis for liftover
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/0e8aa0da4367a78c4c39a2d082b6d235b78237e8/docs/index.html" target="_blank">0e8aa0d</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-01
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/7d81da82a31dd6f9169c49f13873a66afba68755/analysis/index.Rmd" target="_blank">7d81da8</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-01
</td>
<td>
add map stats to site
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/7a9cf749ef4c1a7f45bf19aab838396625c8284c/docs/index.html" target="_blank">7a9cf74</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a2fe9d6cca95411e4ce8b2cfd67d80c7b22c418d/analysis/index.Rmd" target="_blank">a2fe9d6</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-24
</td>
<td>
add annotation prep anaylsis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/03cb9ab5e687900eff39dc89fbad5178e6a10a6c/docs/index.html" target="_blank">03cb9ab</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/88ed6b434a8626d6210e8921e2154078dca095d6/docs/index.html" target="_blank">88ed6b4</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b6db8f3924282d211181aeaf4baa6851908f18aa/analysis/index.Rmd" target="_blank">b6db8f3</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-02
</td>
<td>
add project info to about and index
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/d03d7b001d5748f6f9162210b368fab2dac90662/docs/index.html" target="_blank">d03d7b0</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f914d4920107a1b4c48c590209133aa9ac5817de/analysis/index.Rmd" target="_blank">f914d49</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-02
</td>
<td>
Start workflowr project.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>I am interesting in understanding differential APA between human and chimpanzees. I used 3’ Seq to identify PAS for total and nuclear frations of LCL in both species.</p>
<div id="preparation-and-qc" class="section level2">
<h2>Preparation and QC:</h2>
<ul>
<li><p><a href="annotationInfo.html">Prepare annotation files for mapping</a></p></li>
<li><p><a href="removeSNOfromAnno.html">Remove SNORNAs from the annotations</a></p></li>
<li><p><a href="mapStats.html">Mapping statistics and plots</a></p></li>
<li><p><a href="liftoverPAS.html">Liftover PAS</a></p></li>
<li><p><a href="annotatePAS.html">Annotated PAS and make APA phenotype</a></p></li>
<li><p><a href="NromalizePhenos.html">PCA plot to look at variation in data</a></p></li>
<li><p><a href="CorrbetweenInd.html">Correlation between individuals</a></p></li>
<li><p><a href="verifyBAM.html">Verify sample identity</a></p></li>
<li><p><a href="EnrichmentEndofGenes.html">3’ Seq enrichement along gene bodies</a></p></li>
<li><p><a href="CrossChimpThreePrime.html">Cross map and prepare Chimp 3’ for Viz in IGV</a></p></li>
<li><p><a href="fixPeak2PAS.html">Make PAS bed for 5% peaks</a></p></li>
</ul>
</div>
<div id="choosing-cuttoffs-for-usage-and-expression-level" class="section level2">
<h2>Choosing cuttoffs for usage and expression level</h2>
<ul>
<li><p><a href="expressioncutoff.html">Use merged 3’ at gene level to look for a data quantity cutoff</a></p></li>
<li><p><a href="establishCutoffs.html">Subsampleing apaQTL lines (not redone)</a></p></li>
<li><p><a href="applyDoubleFilt.html">Apply the expression filter</a></p></li>
<li><p><a href="compareFilter.html">Compare before and after cutoff</a></p></li>
</ul>
</div>
<div id="catalog-pas" class="section level2">
<h2>Catalog PAS:</h2>
<ul>
<li><p><a href="PASnumberSpecies_doubleFilter.html">Number of PAS in each species double filter</a></p></li>
<li><p><a href="signalsites_doublefilter.html">Top 2 signal sites double filter</a></p></li>
<li><p><a href="dominantPAS_DF.html">Dominant PAS in Nuclear Fraction double filter</a></p></li>
<li><p><a href="DominantPASintronLoc_DF.html">Intronic location for dominant PAS double filter</a></p></li>
<li><p><a href="TotalDomStructure.html">Test for dominant PAS for total fraction</a></p></li>
<li><p><a href="DomTotalvNuclear.html">Compare dominance between fractions</a></p></li>
<li><p><a href="speciesSpecific_DF.html">Species Specificity in PAS double filter</a></p></li>
<li><p><a href="ConservationofPASNumber.html">PAS number distribution according to dominance</a></p></li>
<li><p><a href="ConservationofPASNumberTotal.html">PAS number distribution according to dominance in total</a></p></li>
<li><p><a href="ConservationPlots.html">Some plots about conservation (correaltions in usage)</a></p></li>
<li><p><a href="SpecBetweenfrac.html">Correlation between fraction in each species</a></p></li>
<li><p><a href="NewDomDef.html">Look at dominance by difference between dominant PAS and next highest used</a></p></li>
<li><p><a href="specSpecDom.html">New Dominance and species specific PAS</a></p></li>
<li><p><a href="DiffTop2SecondDom.html">Dominance structure looking at the difference for all PAS and establish cutoff at .4</a></p></li>
</ul>
</div>
<div id="differential-isoform-analysis" class="section level2">
<h2>Differential Isoform Analysis:</h2>
<ul>
<li><p><a href="Nuclear_HvC_DF.html">Nuclear Human vs. Chimp Analysis Double filter</a></p></li>
<li><p><a href="ExploredAPA_DF.html">Explore nuclear differential APA PAS and genes Double filter</a></p></li>
<li><p><a href="dAPAandapaQTL_DF.html">Enrichment for differencial APA and apaQTL double filter</a></p></li>
<li><p><a href="visualizePASUsage_DF.html">Visualization for all Nuclear DAPA genes double filter</a></p></li>
<li><p><a href="SS_diffUsage.html">Does presence or absense of signal explain differential usage</a></p></li>
<li><p><a href="PTM_analysis.html">Protein interactions</a></p></li>
<li><p><a href="CodingdPAS.html">Explore coding dPAS</a></p></li>
<li><p><a href="UsageForDomPAS.html">New Dominance method and usage or differential usage</a></p></li>
<li><p><a href="NewdomanddAPA.html">New dominant and dAPA</a></p></li>
</ul>
<div id="use-annotations-to-understand-mechanisms" class="section level3">
<h3>Use annotations to understand mechanisms</h3>
<ul>
<li><p><a href="ortho3UTR.html">Identify orthologous 3’UTR set</a></p></li>
<li><p><a href="LongShortUTR_2PAS.html">Look at differentially used PAS by length of UTR</a></p></li>
<li><p><a href="percetnUTR.html">Proportion of ortho 3’ UTR</a></p></li>
<li><p><a href="miRNAanalysis.html">Look at miRNA density</a></p></li>
</ul>
</div>
</div>
<div id="quantify-phenotypes-with-rna" class="section level2">
<h2>Quantify Phenotypes with RNA:</h2>
<ul>
<li><p><a href="diffSplicing.html">Differential Splicing</a></p></li>
<li><p><a href="diffExpression.html">Differential Expression</a></p>
<ul>
<li><p><a href="MetabySpecies.html">Metadata by species</a></p></li>
<li><p><a href="DEandeQTls.html">DE genes and eQTLs</a></p></li>
</ul></li>
<li><p><a href="DEvDS.html">Compare DE and DS genes</a></p></li>
</ul>
</div>
<div id="intersect-with-outside-molecular-phenotypes" class="section level2">
<h2>Intersect with outside molecular phenotypes:</h2>
<ul>
<li><p><a href="dProtanddRibo.html">Download data from differential protein and ribosomes</a></p></li>
<li><p><a href="DiffTransProtvsNuclearAPA_DF.html">Compare to nuclear differential APA with Double filter</a></p></li>
<li><p><a href="upsetter_DF.html">Use UpSetR to vizualize intersections Double filter</a></p></li>
<li><p><a href="correlationPhenos_DF.html">Effect size correlations double filter</a></p></li>
<li><p><a href="SpliceSiteStrength.html">Compare species 5’ splice site strength</a></p></li>
<li><p><a href="DomPatternandDE.html">Compare dominance structure and DE genes</a></p></li>
<li><p><a href="pol2.html">PolII</a></p></li>
<li><p><a href="NewDomandDE.html">New Dominance method and DE</a></p></li>
<li><p><a href="enrichDomdribodp.html">New Dominance and protein/translation</a></p></li>
</ul>
</div>
<div id="selection-exploration" class="section level2">
<h2>Selection exploration</h2>
<ul>
<li><p><a href="dAPA_Conservation.html">Explore PAS conservation</a></p></li>
<li><p><a href="DirSelectionKhan.html">Use directional selection from Khan</a></p></li>
<li><p><a href="ConsandDom.html">Conservation based on dominance structure</a></p></li>
<li><p><a href="ConsandDomNewDef.html">Conservation based on new dominance categories</a></p></li>
</ul>
</div>
<div id="explore-effect-size-relationships" class="section level2">
<h2>Explore effect size relationships</h2>
<ul>
<li><p><a href="ExploreAPAexpression.html">Explore more about APA expression relationship</a></p></li>
<li><p><a href="ExploreAPAexpressionOverlap.html">Explore more about APA expression relationship with overlaping significant genes</a></p></li>
<li><p><a href="DiffUsedIntronic.html">Apa and expression relationship for seperated by location of PAS</a></p></li>
<li><p><a href="DeESandDomgenes.hmtl">APA and expression relationship with dominance structure</a></p></li>
<li><p><a href="dAPAdPnotE.html">Look at genes that have differences in protein levels and APA levels but not mRNA expression</a></p></li>
<li><p><a href="EffectSizePlots.html">Plot the different effect size relationships together</a></p></li>
</ul>
</div>
<div id="domiance-and-de-test-for-mechanisms" class="section level2">
<h2>Domiance and DE (test for mechanisms)</h2>
<ul>
<li><p><a href="DiffDomDE_length.html">Could standardization with length account for differences?</a></p></li>
<li><p><a href="DiffDomandDELOC.html">Look at this by location</a></p></li>
<li><p><a href="ARE_element.html">Test for destabilization</a></p></li>
<li><p><a href="mRNADecay.html">Test differences in mRNA decay</a></p></li>
</ul>
</div>
<div id="information-content" class="section level2">
<h2>Information content</h2>
<ul>
<li><p><a href="infoContent.html">Calculate Information content measures</a></p></li>
<li><p><a href="DeandNumPAS.html">DE and number of PAS</a></p></li>
<li><p><a href="InfoanddAPA.html">Information content and dAPA</a></p></li>
</ul>
</div>
<div id="evaluate-reasons-for-inconsistant-dominance-structure-not-rerun-after-annotation-change-420" class="section level2">
<h2>Evaluate Reasons for inconsistant dominance structure (not rerun after annotation change 4/20)</h2>
<p>List of everything we tried for this section:</p>
<ul>
<li><a href="listFixes.html">List of Test dominance structure</a></li>
</ul>
<div id="number-of-ns" class="section level3">
<h3>Number of Ns</h3>
<ul>
<li><p><a href="index.html">Look at number of reads lost if you exclude reads with N</a></p></li>
<li><p><a href="BaseComp.html">Base composition</a></p></li>
</ul>
</div>
<div id="misprimming" class="section level3">
<h3>Misprimming</h3>
<ul>
<li><a href="changeMisprimcut.html">Change misprimming parameters</a></li>
</ul>
<p>Liftover</p>
<ul>
<li><p><a href="liftMis4.html">Liftover for 4</a></p></li>
<li><p><a href="liftMis5.html">Liftover for 5</a></p></li>
<li><p><a href="liftMis6.html">Liftover for 6</a></p></li>
</ul>
<p>Annotate and make phenotype</p>
<ul>
<li><p><a href="annoMis4.html">Annotation for 4</a></p></li>
<li><p><a href="annoMis5.html">Annotation for 5</a></p></li>
<li><p><a href="annoMis6.html">Annotation for 6</a></p></li>
</ul>
<p>Evaluate results:</p>
<ul>
<li><a href="NewFilterDomPAS.html">Dominant PAS distribution after adjusting filters</a></li>
</ul>
</div>
<div id="annotation-differences" class="section level3">
<h3>Annotation differences</h3>
<ul>
<li><a href="annotationBias.html">Test for annotation bias</a></li>
</ul>
</div>
<div id="other-ideas" class="section level3">
<h3>Other ideas:</h3>
<ul>
<li><p><a href="DomPASDiffUsed.html">Add usage cutoffs for the dominant PAS</a></p></li>
<li><p><a href="OverlappingPAS.html">Overlapping PAS</a></p></li>
<li><p><a href="TrynewQuantMethods.html">Test quantification methods</a></p></li>
<li><p><a href="checkResnoMM.html">Look at dominance sturcture without multimapping and SNOrnas</a></p></li>
<li><p><a href="DominantPAS_usingNorm.html">Distribution with normalized usage</a></p></li>
<li><p><a href="DominantNoTies.html">Filter out ties and genes with 1 PAS</a></p></li>
<li><p><a href="PPASqauntnorm.html">Quantile Normalzie</a></p></li>
</ul>
</div>
<div id="test-multimapping-and-fc-criteria" class="section level3">
<h3>Test multimapping and FC criteria</h3>
<ul>
<li><p><a href="multimapParameters.html">Look at new parameters for multimapping</a></p></li>
<li><p><a href="orthoexonAnno.html">Assess which PAS are in orthoexon file</a></p></li>
<li><p><a href="MMExpreiment.html">Experiemnt with quality and FC</a></p></li>
<li><p><a href="DomwithPrimary.html">Redo the quantification with primary reads</a></p></li>
</ul>
</div>
</div>
<div id="visualization" class="section level2">
<h2>Visualization</h2>
<ul>
<li><a href="GvizPlots.html">Try to use gviz</a></li>
</ul>
</div>
<div id="extra-qc-and-analysis" class="section level2">
<h2>Extra QC and Analysis</h2>
<ul>
<li><p><a href="comp2apaQTLPAS.html">Compare identified PAS with apaQTL PAS</a></p></li>
<li><p><a href="MergePASwithin.html">Test for overlaps within species</a></p></li>
<li><p><a href="resultswo18499.html">Check res after removing 18499</a></p></li>
</ul>
<div id="sample-switch-analysis" class="section level3">
<h3>Sample switch analysis</h3>
<ul>
<li><p><a href="InvestigateBadSamples.html">Investigate bad RNA seq samples</a></p>
<ul>
<li><p><a href="diffSplicing_removeBad.html">Differential splicing without bad samples</a></p></li>
<li><p><a href="diffExpressionRemove2.html">Differential Expression without bad samples</a></p></li>
</ul></li>
</ul>
</div>
<div id="mapping-problems-with-pantro5-and-multimapping" class="section level3">
<h3>Mapping problems with pantro5 and multimapping</h3>
<ul>
<li><p><a href="MapRNAhg19.html">Map RNA to hg19</a></p></li>
<li><p><a href="multiMap.html">Evaluate multimapping criteria</a></p></li>
<li><p><a href="OppositeMap.html">Map to opposite Genomes</a></p></li>
<li><p><a href="investigatePantro5.html">Evaluate Pantro 5 problems</a></p></li>
</ul>
<p>From before reseq:</p>
<ul>
<li><p><a href="NormalizePheno_total.html">Total Fraction</a></p></li>
<li><p><a href="NonNormPheno_total.html">Total Fraction- Prenormalized</a></p></li>
</ul>
</div>
<div id="chosing-between-pantro-files" class="section level3">
<h3>Chosing between pantro files</h3>
<ul>
<li><p><a href="compareOrthoExon.html">Compare pantro5 and pantro6 orthoexon</a></p></li>
<li><p><a href="diffExpressionPantro6.html">Differential Expression with Pantro6</a></p></li>
<li><p><a href="DE_pantro5vspantro6.html">Compare DE in pantro5 vs pantro6</a></p></li>
</ul>
</div>
<div id="extra-comparisons-or-analysis-not-redone-after-double-filter" class="section level3">
<h3>Extra comparisons or analysis: not redone after double filter</h3>
<ul>
<li><p><a href="Human_TvN.html">Human Total vs Nuclear Analysis</a></p></li>
<li><p><a href="Chimp_TvN.html">Chimp Total vs Nuclear Analysis</a></p></li>
<li><p><a href="dominantPAS_total.html">Dominant PAS in Total Fraction</a></p></li>
<li><p><a href="Total_HvC.html">Total Human vs. Chimp Analysis</a></p></li>
<li><p><a href="CompHC_totalandnuclear.html">Venn diagrams for genes in each differential APA set</a></p></li>
</ul>
<p>From the original nuclear dominant</p>
<ul>
<li><p><a href="ss_sameDomdiffDom.html">Does signal site explain trend?</a></p></li>
<li><p><a href="HumanIntronicDominant.html">Look at Intronic Dominant with DPAU and DE</a></p></li>
<li><p><a href="DapaineQTL.html">Enrichment for differencial APA and eQTL</a></p></li>
<li><p><a href="DominatPASandDIffAPA.html">Look at dAPA and genes from Dominant PAS analysis</a></p></li>
</ul>
</div>
<div id="predouble-filter-analysis" class="section level3">
<h3>PreDouble filter analysis</h3>
<ul>
<li><p><a href="PASnumperSpecies.html">Number of PAS in each species</a></p></li>
<li><p><a href="signalsites.html">Pressence of top 2 signal sites</a></p></li>
<li><p><a href="dominantPAS.html">Dominant PAS in Nuclear Fraction</a></p></li>
<li><p><a href="DominantPASintronLoc.html">Intronic location for dominant PAS</a></p></li>
<li><p><a href="speciesSpecific.html">Species Specificity in PAS</a></p></li>
<li><p><a href="Nuclear_HvC.html">Nuclear Human vs. Chimp Analysis</a></p></li>
<li><p><a href="Explore.html">Explore nuclear differential APA PAS and genes</a></p></li>
<li><p><a href="andapaQTL.html">Enrichment for differencial APA and apaQTL</a></p></li>
<li><p><a href="visualizePASUsage.html">Visualization for all Nuclear DAPA genes</a></p></li>
<li><p><a href="DiffTransProtvsNuclearAPA.html">Compare to nuclear differential APA</a></p></li>
<li><p><a href="upsetter.html">Use UpSetR to vizualize intersections</a></p></li>
<li><p><a href="correlationPhenos.html">Effect size correlations</a></p></li>
<li><p><a href="NormalizePheno_nuclear.html">Normalize Nuclear Fraction</a></p></li>
<li><p><a href="mediation.html">Mediation analysis</a></p></li>
</ul>
</div>
<div id="human-v-chimp-fraction-differences---pre--march-rerun" class="section level3">
<h3>Human v Chimp fraction differences - pre- march rerun</h3>
<ul>
<li><a href="TotalVNuclearBothSpecies.html">Differential isoform between fractons by species</a></li>
</ul>
<div id="mediation-tests" class="section level4">
<h4>Mediation tests</h4>
<ul>
<li><p><a href="NormalizePheno_nuclear_DF.html">Normalize (qqnrom) Nuclear Fraction double filter</a></p></li>
<li><p><a href="normalizeClusters.html">Normalize clusters from splicing analysis</a></p></li>
<li><p><a href="mediation_DF.html">Mediation analysis double filter</a></p></li>
<li><p><a href="mediation_prot.html">Mediation with protein</a></p></li>
</ul>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
