<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Home</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/journal.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Comparative_APA</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/brimittleman/Comparative_APA">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Home</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-01-22
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Comparative_APA/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.5.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtree07fd45952afd6e6586320a0f6ae9230c1e3f7ff0targetblank07fd459a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/brimittleman/Comparative_APA/tree/07fd45952afd6e6586320a0f6ae9230c1e3f7ff0" target="_blank">07fd459</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtree07fd45952afd6e6586320a0f6ae9230c1e3f7ff0targetblank07fd459a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    code/chimp_log/
    Ignored:    code/human_log/
    Ignored:    data/.DS_Store
    Ignored:    data/metadata_HCpanel.txt.sb-a5794dd2-i594qs/

Untracked files:
    Untracked:  ._.DS_Store
    Untracked:  Chimp/
    Untracked:  Human/
    Untracked:  analysis/CrossChimpThreePrime.Rmd
    Untracked:  analysis/DiffTransProtvsExpression.Rmd
    Untracked:  analysis/assessReadQual.Rmd
    Untracked:  analysis/diffExpressionPantro6.Rmd
    Untracked:  code/._ClassifyLeafviz.sh
    Untracked:  code/._Config_chimp.yaml
    Untracked:  code/._Config_chimp_full.yaml
    Untracked:  code/._Config_human.yaml
    Untracked:  code/._ConvertJunc2Bed.sh
    Untracked:  code/._CountNucleotides.py
    Untracked:  code/._CrossMapChimpRNA.sh
    Untracked:  code/._CrossMapThreeprime.sh
    Untracked:  code/._DiffSplice.sh
    Untracked:  code/._DiffSplicePlots.sh
    Untracked:  code/._DiffSplicePlots_gencode.sh
    Untracked:  code/._DiffSplice_gencode.sh
    Untracked:  code/._DiffSplice_removebad.sh
    Untracked:  code/._FindIntronForDomPAS.sh
    Untracked:  code/._FindIntronForDomPAS_DF.sh
    Untracked:  code/._GetMAPQscore.py
    Untracked:  code/._GetSecondaryMap.py
    Untracked:  code/._Lift5perPAS.sh
    Untracked:  code/._LiftFinalChimpJunc2Human.sh
    Untracked:  code/._LiftOrthoPAS2chimp.sh
    Untracked:  code/._MapBadSamples.sh
    Untracked:  code/._PAS_ATTAAA.sh
    Untracked:  code/._PAS_ATTAAA_df.sh
    Untracked:  code/._PASsequences.sh
    Untracked:  code/._PASsequences_DF.sh
    Untracked:  code/._PlotNuclearUsagebySpecies.R
    Untracked:  code/._PlotNuclearUsagebySpecies_DF.R
    Untracked:  code/._QuantMergedClusters.sh
    Untracked:  code/._ReverseLiftFilter.R
    Untracked:  code/._RunFixLeafCluster.sh
    Untracked:  code/._RunNegMCMediation.sh
    Untracked:  code/._Snakefile
    Untracked:  code/._SnakefilePAS
    Untracked:  code/._SnakefilePASfilt
    Untracked:  code/._SortIndexBadSamples.sh
    Untracked:  code/._bed215upbed.py
    Untracked:  code/._bed2SAF_gen.py
    Untracked:  code/._buildIndecpantro5
    Untracked:  code/._buildIndecpantro5.sh
    Untracked:  code/._buildLeafviz.sh
    Untracked:  code/._buildLeafviz_leadAnno.sh
    Untracked:  code/._buildStarIndex.sh
    Untracked:  code/._chimpChromprder.sh
    Untracked:  code/._cleanbed2saf.py
    Untracked:  code/._cluster.json
    Untracked:  code/._cluster2bed.py
    Untracked:  code/._clusterLiftReverse.sh
    Untracked:  code/._clusterLiftReverse_removebad.sh
    Untracked:  code/._clusterLiftprimary.sh
    Untracked:  code/._clusterLiftprimary_removebad.sh
    Untracked:  code/._converBam2Junc.sh
    Untracked:  code/._converBam2Junc_removeBad.sh
    Untracked:  code/._extraSnakefiltpas
    Untracked:  code/._filter5percPAS.py
    Untracked:  code/._filterNumChroms.py
    Untracked:  code/._filterPASforMP.py
    Untracked:  code/._filterPostLift.py
    Untracked:  code/._fixExonFC.py
    Untracked:  code/._fixLeafCluster.py
    Untracked:  code/._fixLiftedJunc.py
    Untracked:  code/._fixUTRexonanno.py
    Untracked:  code/._formathg38Anno.py
    Untracked:  code/._formatpantro6Anno.py
    Untracked:  code/._getRNAseqMapStats.sh
    Untracked:  code/._hg19MapStats.sh
    Untracked:  code/._humanChromorder.sh
    Untracked:  code/._intersectLiftedPAS.sh
    Untracked:  code/._liftJunctionFiles.sh
    Untracked:  code/._liftPAS19to38.sh
    Untracked:  code/._liftedchimpJunc2human.sh
    Untracked:  code/._makeNuclearDapaplots.sh
    Untracked:  code/._makeNuclearDapaplots_DF.sh
    Untracked:  code/._makeSamplyGroupsHuman_TvN.py
    Untracked:  code/._mapRNAseqhg19.sh
    Untracked:  code/._mapRNAseqhg19_newPipeline.sh
    Untracked:  code/._maphg19.sh
    Untracked:  code/._maphg19_subjunc.sh
    Untracked:  code/._mediation_test.R
    Untracked:  code/._mergeChimp3prime_inhg38.sh
    Untracked:  code/._mergedBam2BW.sh
    Untracked:  code/._nameClusters.py
    Untracked:  code/._negativeMediation_montecarlo.R
    Untracked:  code/._numMultimap.py
    Untracked:  code/._overlapapaQTLPAS.sh
    Untracked:  code/._prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/._prepareLeafvizAnno.sh
    Untracked:  code/._preparePAS4lift.py
    Untracked:  code/._primaryLift.sh
    Untracked:  code/._processhg38exons.py
    Untracked:  code/._quantJunc.sh
    Untracked:  code/._quantJunc_TEST.sh
    Untracked:  code/._quantJunc_removeBad.sh
    Untracked:  code/._quantMerged_seperatly.sh
    Untracked:  code/._recLiftchim2human.sh
    Untracked:  code/._revLiftPAShg38to19.sh
    Untracked:  code/._reverseLift.sh
    Untracked:  code/._runCheckReverseLift.sh
    Untracked:  code/._runChimpDiffIso.sh
    Untracked:  code/._runCountNucleotides.sh
    Untracked:  code/._runFilterNumChroms.sh
    Untracked:  code/._runHumanDiffIso.sh
    Untracked:  code/._runNuclearDiffIso_DF.sh
    Untracked:  code/._runNuclearDifffIso.sh
    Untracked:  code/._runTotalDiffIso.sh
    Untracked:  code/._run_chimpverifybam.sh
    Untracked:  code/._run_verifyBam.sh
    Untracked:  code/._snakemake.batch
    Untracked:  code/._snakemakePAS.batch
    Untracked:  code/._snakemakePASchimp.batch
    Untracked:  code/._snakemakePAShuman.batch
    Untracked:  code/._snakemake_chimp.batch
    Untracked:  code/._snakemake_human.batch
    Untracked:  code/._snakemakefiltPAS.batch
    Untracked:  code/._snakemakefiltPAS_chimp
    Untracked:  code/._snakemakefiltPAS_chimp.sh
    Untracked:  code/._snakemakefiltPAS_human.sh
    Untracked:  code/._submit-snakemake-chimp.sh
    Untracked:  code/._submit-snakemake-human.sh
    Untracked:  code/._submit-snakemakePAS-chimp.sh
    Untracked:  code/._submit-snakemakePAS-human.sh
    Untracked:  code/._submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/._submit-snakemakefiltPAS-human.sh
    Untracked:  code/._subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/._subset_diffisopheno_Nuclear_HvC_DF.py
    Untracked:  code/._subset_diffisopheno_Total_HvC.py
    Untracked:  code/._threeprimeOrthoFC.sh
    Untracked:  code/._transcriptDTplotsNuclear.sh
    Untracked:  code/._verifyBam4973.sh
    Untracked:  code/._verifyBam4973inHuman.sh
    Untracked:  code/._wrap_chimpverifybam.sh
    Untracked:  code/._wrap_verifyBam.sh
    Untracked:  code/._writeMergecode.py
    Untracked:  code/.snakemake/
    Untracked:  code/ClassifyLeafviz.sh
    Untracked:  code/Config_chimp.yaml
    Untracked:  code/Config_chimp_full.yaml
    Untracked:  code/Config_human.yaml
    Untracked:  code/ConvertJunc2Bed.err
    Untracked:  code/ConvertJunc2Bed.out
    Untracked:  code/ConvertJunc2Bed.sh
    Untracked:  code/CountNucleotides.py
    Untracked:  code/CrossMapChimpRNA.sh
    Untracked:  code/CrossMapThreeprime.sh
    Untracked:  code/CrossmapChimp3prime.err
    Untracked:  code/CrossmapChimp3prime.out
    Untracked:  code/CrossmapChimpRNA.err
    Untracked:  code/CrossmapChimpRNA.out
    Untracked:  code/DiffSplice.err
    Untracked:  code/DiffSplice.out
    Untracked:  code/DiffSplice.sh
    Untracked:  code/DiffSplicePlots.err
    Untracked:  code/DiffSplicePlots.out
    Untracked:  code/DiffSplicePlots.sh
    Untracked:  code/DiffSplicePlots_gencode.sh
    Untracked:  code/DiffSplice_gencode.sh
    Untracked:  code/DiffSplice_removebad.err
    Untracked:  code/DiffSplice_removebad.out
    Untracked:  code/DiffSplice_removebad.sh
    Untracked:  code/FilterReverseLift.err
    Untracked:  code/FilterReverseLift.out
    Untracked:  code/FindIntronForDomPAS.err
    Untracked:  code/FindIntronForDomPAS.out
    Untracked:  code/FindIntronForDomPAS.sh
    Untracked:  code/FindIntronForDomPAS_DF.sh
    Untracked:  code/GencodeDiffSplice.err
    Untracked:  code/GencodeDiffSplice.out
    Untracked:  code/GetMAPQscore.py
    Untracked:  code/GetSecondaryMap.py
    Untracked:  code/HchromOrder.err
    Untracked:  code/HchromOrder.out
    Untracked:  code/JunctionLift.err
    Untracked:  code/JunctionLift.out
    Untracked:  code/JunctionLiftFinalChimp.err
    Untracked:  code/JunctionLiftFinalChimp.out
    Untracked:  code/Lift5perPAS.sh
    Untracked:  code/Lift5perPASbed.err
    Untracked:  code/Lift5perPASbed.out
    Untracked:  code/LiftClustersFirst.err
    Untracked:  code/LiftClustersFirst.out
    Untracked:  code/LiftClustersFirst_remove.err
    Untracked:  code/LiftClustersFirst_remove.out
    Untracked:  code/LiftClustersSecond.err
    Untracked:  code/LiftClustersSecond.out
    Untracked:  code/LiftClustersSecond_remove.err
    Untracked:  code/LiftClustersSecond_remove.out
    Untracked:  code/LiftFinalChimpJunc2Human.sh
    Untracked:  code/LiftOrthoPAS2chimp.sh
    Untracked:  code/LiftorthoPAS.err
    Untracked:  code/LiftorthoPASt.out
    Untracked:  code/Log.out
    Untracked:  code/MapBadSamples.err
    Untracked:  code/MapBadSamples.out
    Untracked:  code/MapBadSamples.sh
    Untracked:  code/MapStats.err
    Untracked:  code/MapStats.out
    Untracked:  code/MergeClusters.err
    Untracked:  code/MergeClusters.out
    Untracked:  code/MergeClusters.sh
    Untracked:  code/PAS_ATTAAA.err
    Untracked:  code/PAS_ATTAAA.out
    Untracked:  code/PAS_ATTAAA.sh
    Untracked:  code/PAS_ATTAAADF.err
    Untracked:  code/PAS_ATTAAADF.out
    Untracked:  code/PAS_ATTAAA_df.sh
    Untracked:  code/PAS_sequence.err
    Untracked:  code/PAS_sequence.out
    Untracked:  code/PAS_sequenceDF.err
    Untracked:  code/PAS_sequenceDF.out
    Untracked:  code/PASsequences.sh
    Untracked:  code/PASsequences_DF.sh
    Untracked:  code/PlotNuclearUsagebySpecies.R
    Untracked:  code/PlotNuclearUsagebySpecies_DF.R
    Untracked:  code/QuantMergeClusters
    Untracked:  code/QuantMergeClusters.err
    Untracked:  code/QuantMergeClusters.out
    Untracked:  code/QuantMergedClusters.sh
    Untracked:  code/Rev_liftoverPAShg19to38.err
    Untracked:  code/Rev_liftoverPAShg19to38.out
    Untracked:  code/ReverseLiftFilter.R
    Untracked:  code/RunFixCluster.err
    Untracked:  code/RunFixCluster.out
    Untracked:  code/RunFixLeafCluster.sh
    Untracked:  code/RunNegMCMediation.err
    Untracked:  code/RunNegMCMediation.sh
    Untracked:  code/RunNegMCMediationr.out
    Untracked:  code/RunPosMCMediation.err
    Untracked:  code/RunPosMCMediation.sh
    Untracked:  code/RunPosMCMediationr.out
    Untracked:  code/SAF215upbed_gen.py
    Untracked:  code/Snakefile
    Untracked:  code/SnakefilePAS
    Untracked:  code/SnakefilePASfilt
    Untracked:  code/SortIndexBadSamples.err
    Untracked:  code/SortIndexBadSamples.out
    Untracked:  code/SortIndexBadSamples.sh
    Untracked:  code/TotalTranscriptDTplot.err
    Untracked:  code/TotalTranscriptDTplot.out
    Untracked:  code/Upstream10Bases_general.py
    Untracked:  code/apaQTLsnake.err
    Untracked:  code/apaQTLsnake.out
    Untracked:  code/apaQTLsnakePAS.err
    Untracked:  code/apaQTLsnakePAS.out
    Untracked:  code/apaQTLsnakePAShuman.err
    Untracked:  code/bam2junc.err
    Untracked:  code/bam2junc.out
    Untracked:  code/bam2junc_remove.err
    Untracked:  code/bam2junc_remove.out
    Untracked:  code/bed215upbed.py
    Untracked:  code/bed2SAF_gen.py
    Untracked:  code/bed2saf.py
    Untracked:  code/bg_to_cov.py
    Untracked:  code/buildIndecpantro5
    Untracked:  code/buildIndecpantro5.sh
    Untracked:  code/buildLeafviz.err
    Untracked:  code/buildLeafviz.out
    Untracked:  code/buildLeafviz.sh
    Untracked:  code/buildLeafviz_leadAnno.sh
    Untracked:  code/buildLeafviz_leafanno.err
    Untracked:  code/buildLeafviz_leafanno.out
    Untracked:  code/buildStarIndex.sh
    Untracked:  code/callPeaksYL.py
    Untracked:  code/chimpChromprder.sh
    Untracked:  code/chooseAnno2Bed.py
    Untracked:  code/chooseAnno2SAF.py
    Untracked:  code/chromOrder.err
    Untracked:  code/chromOrder.out
    Untracked:  code/classifyLeafviz.err
    Untracked:  code/classifyLeafviz.out
    Untracked:  code/cleanbed2saf.py
    Untracked:  code/cluster.json
    Untracked:  code/cluster2bed.py
    Untracked:  code/clusterLiftReverse.sh
    Untracked:  code/clusterLiftReverse_removebad.sh
    Untracked:  code/clusterLiftprimary.sh
    Untracked:  code/clusterLiftprimary_removebad.sh
    Untracked:  code/clusterPAS.json
    Untracked:  code/clusterfiltPAS.json
    Untracked:  code/comands2Mege.sh
    Untracked:  code/converBam2Junc.sh
    Untracked:  code/converBam2Junc_removeBad.sh
    Untracked:  code/convertNumeric.py
    Untracked:  code/environment.yaml
    Untracked:  code/extraSnakefiltpas
    Untracked:  code/filter5perc.R
    Untracked:  code/filter5percPAS.py
    Untracked:  code/filter5percPheno.py
    Untracked:  code/filterBamforMP.pysam2_gen.py
    Untracked:  code/filterJuncChroms.err
    Untracked:  code/filterJuncChroms.out
    Untracked:  code/filterMissprimingInNuc10_gen.py
    Untracked:  code/filterNumChroms.py
    Untracked:  code/filterPASforMP.py
    Untracked:  code/filterPostLift.py
    Untracked:  code/filterSAFforMP_gen.py
    Untracked:  code/filterSortBedbyCleanedBed_gen.R
    Untracked:  code/filterpeaks.py
    Untracked:  code/fixExonFC.py
    Untracked:  code/fixFChead.py
    Untracked:  code/fixFChead_bothfrac.py
    Untracked:  code/fixLeafCluster.py
    Untracked:  code/fixLiftedJunc.py
    Untracked:  code/fixUTRexonanno.py
    Untracked:  code/formathg38Anno.py
    Untracked:  code/generateStarIndex.err
    Untracked:  code/generateStarIndex.out
    Untracked:  code/generateStarIndexHuman.err
    Untracked:  code/generateStarIndexHuman.out
    Untracked:  code/getRNAseqMapStats.sh
    Untracked:  code/hg19MapStats.err
    Untracked:  code/hg19MapStats.out
    Untracked:  code/hg19MapStats.sh
    Untracked:  code/humanChromorder.sh
    Untracked:  code/humanFiles
    Untracked:  code/intersectAnno.err
    Untracked:  code/intersectAnno.out
    Untracked:  code/intersectAnnoExt.err
    Untracked:  code/intersectAnnoExt.out
    Untracked:  code/intersectLiftedPAS.sh
    Untracked:  code/leafcutter_merge_regtools_redo.py
    Untracked:  code/liftJunctionFiles.sh
    Untracked:  code/liftPAS19to38.sh
    Untracked:  code/liftoverPAShg19to38.err
    Untracked:  code/liftoverPAShg19to38.out
    Untracked:  code/log/
    Untracked:  code/make5percPeakbed.py
    Untracked:  code/makeFileID.py
    Untracked:  code/makeNuclearDapaplots.sh
    Untracked:  code/makeNuclearDapaplots_DF.sh
    Untracked:  code/makeNuclearPlots.err
    Untracked:  code/makeNuclearPlots.out
    Untracked:  code/makeNuclearPlotsDF.err
    Untracked:  code/makeNuclearPlotsDF.out
    Untracked:  code/makePheno.py
    Untracked:  code/makeSamplyGroupsChimp_TvN.py
    Untracked:  code/makeSamplyGroupsHuman_TvN.py
    Untracked:  code/mapRNAseqhg19.sh
    Untracked:  code/mapRNAseqhg19_newPipeline.sh
    Untracked:  code/maphg19.err
    Untracked:  code/maphg19.out
    Untracked:  code/maphg19.sh
    Untracked:  code/maphg19_new.err
    Untracked:  code/maphg19_new.out
    Untracked:  code/maphg19_sub.err
    Untracked:  code/maphg19_sub.out
    Untracked:  code/maphg19_subjunc.sh
    Untracked:  code/mediation_test.R
    Untracked:  code/merge.err
    Untracked:  code/mergeChimp3prime_inhg38.sh
    Untracked:  code/merge_leafcutter_clusters_redo.py
    Untracked:  code/mergeandsort_ChimpinHuman.err
    Untracked:  code/mergeandsort_ChimpinHuman.out
    Untracked:  code/mergedBam2BW.sh
    Untracked:  code/mergedbam2bw.err
    Untracked:  code/mergedbam2bw.out
    Untracked:  code/nameClusters.py
    Untracked:  code/namePeaks.py
    Untracked:  code/negativeMediation_montecarlo.R
    Untracked:  code/nuclearTranscriptDTplot.err
    Untracked:  code/nuclearTranscriptDTplot.out
    Untracked:  code/numMultimap.py
    Untracked:  code/overlapPAS.err
    Untracked:  code/overlapPAS.out
    Untracked:  code/overlapapaQTLPAS.sh
    Untracked:  code/overlapapaQTLPAS_extended.sh
    Untracked:  code/overlapapaQTLPAS_samples.sh
    Untracked:  code/peak2PAS.py
    Untracked:  code/pheno2countonly.R
    Untracked:  code/postiveMediation_montecarlo.R
    Untracked:  code/prepareAnnoLeafviz.err
    Untracked:  code/prepareAnnoLeafviz.out
    Untracked:  code/prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/prepareLeafvizAnno.sh
    Untracked:  code/preparePAS4lift.py
    Untracked:  code/prepare_phenotype_table.py
    Untracked:  code/primaryLift.err
    Untracked:  code/primaryLift.out
    Untracked:  code/primaryLift.sh
    Untracked:  code/processhg38exons.py
    Untracked:  code/quantJunc.sh
    Untracked:  code/quantJunc_TEST.sh
    Untracked:  code/quantJunc_removeBad.sh
    Untracked:  code/quantLiftedPAS.err
    Untracked:  code/quantLiftedPAS.out
    Untracked:  code/quantLiftedPAS.sh
    Untracked:  code/quatJunc.err
    Untracked:  code/quatJunc.out
    Untracked:  code/recChimpback2Human.err
    Untracked:  code/recChimpback2Human.out
    Untracked:  code/recLiftchim2human.sh
    Untracked:  code/revLift.err
    Untracked:  code/revLift.out
    Untracked:  code/revLiftPAShg38to19.sh
    Untracked:  code/reverseLift.sh
    Untracked:  code/runCheckReverseLift.sh
    Untracked:  code/runChimpDiffIso.sh
    Untracked:  code/runCountNucleotides.err
    Untracked:  code/runCountNucleotides.out
    Untracked:  code/runCountNucleotides.sh
    Untracked:  code/runCountNucleotidesPantro6.err
    Untracked:  code/runCountNucleotidesPantro6.out
    Untracked:  code/runCountNucleotides_pantro6.sh
    Untracked:  code/runFilterNumChroms.sh
    Untracked:  code/runHumanDiffIso.sh
    Untracked:  code/runNuclearDiffIso_DF.sh
    Untracked:  code/runNuclearDifffIso.sh
    Untracked:  code/runTotalDiffIso.sh
    Untracked:  code/run_Chimpleafcutter_ds.err
    Untracked:  code/run_Chimpleafcutter_ds.out
    Untracked:  code/run_Chimpverifybam.err
    Untracked:  code/run_Chimpverifybam.out
    Untracked:  code/run_Humanleafcutter_ds.err
    Untracked:  code/run_Humanleafcutter_ds.out
    Untracked:  code/run_Nuclearleafcutter_ds.err
    Untracked:  code/run_Nuclearleafcutter_ds.out
    Untracked:  code/run_Nuclearleafcutter_dsDF.err
    Untracked:  code/run_Nuclearleafcutter_dsDF.out
    Untracked:  code/run_Totalleafcutter_ds.err
    Untracked:  code/run_Totalleafcutter_ds.out
    Untracked:  code/run_chimpverifybam.sh
    Untracked:  code/run_verifyBam.sh
    Untracked:  code/run_verifybam.err
    Untracked:  code/run_verifybam.out
    Untracked:  code/slurm-62824013.out
    Untracked:  code/slurm-62825841.out
    Untracked:  code/slurm-62826116.out
    Untracked:  code/slurm-64108209.out
    Untracked:  code/slurm-64108521.out
    Untracked:  code/slurm-64108557.out
    Untracked:  code/snakePASChimp.err
    Untracked:  code/snakePASChimp.out
    Untracked:  code/snakePAShuman.out
    Untracked:  code/snakemake.batch
    Untracked:  code/snakemakeChimp.err
    Untracked:  code/snakemakeChimp.out
    Untracked:  code/snakemakeHuman.err
    Untracked:  code/snakemakeHuman.out
    Untracked:  code/snakemakePAS.batch
    Untracked:  code/snakemakePASFiltChimp.err
    Untracked:  code/snakemakePASFiltChimp.out
    Untracked:  code/snakemakePASFiltHuman.err
    Untracked:  code/snakemakePASFiltHuman.out
    Untracked:  code/snakemakePASchimp.batch
    Untracked:  code/snakemakePAShuman.batch
    Untracked:  code/snakemake_chimp.batch
    Untracked:  code/snakemake_human.batch
    Untracked:  code/snakemakefiltPAS.batch
    Untracked:  code/snakemakefiltPAS_chimp.sh
    Untracked:  code/snakemakefiltPAS_human.sh
    Untracked:  code/submit-snakemake-chimp.sh
    Untracked:  code/submit-snakemake-human.sh
    Untracked:  code/submit-snakemakePAS-chimp.sh
    Untracked:  code/submit-snakemakePAS-human.sh
    Untracked:  code/submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/submit-snakemakefiltPAS-human.sh
    Untracked:  code/subset_diffisopheno.py
    Untracked:  code/subset_diffisopheno_Chimp_tvN.py
    Untracked:  code/subset_diffisopheno_Huma_tvN.py
    Untracked:  code/subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/subset_diffisopheno_Nuclear_HvC_DF.py
    Untracked:  code/subset_diffisopheno_Total_HvC.py
    Untracked:  code/test
    Untracked:  code/threeprimeOrthoFC.out
    Untracked:  code/threeprimeOrthoFC.sh
    Untracked:  code/threeprimeOrthoFCcd.err
    Untracked:  code/transcriptDTplotsNuclear.sh
    Untracked:  code/transcriptDTplotsTotal.sh
    Untracked:  code/verifyBam4973.sh
    Untracked:  code/verifyBam4973inHuman.sh
    Untracked:  code/verifybam4973.err
    Untracked:  code/verifybam4973.out
    Untracked:  code/verifybam4973HumanMap.err
    Untracked:  code/verifybam4973HumanMap.out
    Untracked:  code/wrap_Chimpverifybam.err
    Untracked:  code/wrap_Chimpverifybam.out
    Untracked:  code/wrap_chimpverifybam.sh
    Untracked:  code/wrap_verifyBam.sh
    Untracked:  code/wrap_verifybam.err
    Untracked:  code/wrap_verifybam.out
    Untracked:  code/writeMergecode.py
    Untracked:  data/._.DS_Store
    Untracked:  data/._HC_filenames.txt
    Untracked:  data/._HC_filenames.txt.sb-4426323c-IKIs0S
    Untracked:  data/._HC_filenames.xlsx
    Untracked:  data/._MapPantro6_meta.txt
    Untracked:  data/._MapPantro6_meta.txt.sb-a5794dd2-Cskmlm
    Untracked:  data/._MapPantro6_meta.xlsx
    Untracked:  data/._OppositeSpeciesMap.txt
    Untracked:  data/._OppositeSpeciesMap.txt.sb-a5794dd2-mayWJf
    Untracked:  data/._OppositeSpeciesMap.xlsx
    Untracked:  data/._RNASEQ_metadata.txt
    Untracked:  data/._RNASEQ_metadata.txt.sb-4426323c-TE4ns3
    Untracked:  data/._RNASEQ_metadata.txt.sb-51f67ae1-HXp7Gq
    Untracked:  data/._RNASEQ_metadata_2Removed.txt
    Untracked:  data/._RNASEQ_metadata_2Removed.txt.sb-4426323c-a4lBwx
    Untracked:  data/._RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/._RNASEQ_metadata_stranded.txt
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-a5794dd2-D659m2
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-a5794dd2-ImNMoY
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-e4bf31f0-ZGnGgl
    Untracked:  data/._RNASEQ_metadata_stranded.xlsx
    Untracked:  data/._metadata_HCpanel.txt
    Untracked:  data/._metadata_HCpanel.txt.sb-a3d92a2d-b9cYoF
    Untracked:  data/._metadata_HCpanel.txt.sb-a5794dd2-i594qs
    Untracked:  data/._metadata_HCpanel.txt.sb-f4823d1e-qihGek
    Untracked:  data/._metadata_HCpanel.xlsx
    Untracked:  data/._metadata_HCpanel_frompantro5.xlsx
    Untracked:  data/._~$RNASEQ_metadata.xlsx
    Untracked:  data/._~$metadata_HCpanel.xlsx
    Untracked:  data/._.xlsx
    Untracked:  data/CompapaQTLpas/
    Untracked:  data/DTmatrix/
    Untracked:  data/DiffExpression/
    Untracked:  data/DiffIso_Nuclear/
    Untracked:  data/DiffIso_Nuclear_DF/
    Untracked:  data/DiffIso_Total/
    Untracked:  data/DiffSplice/
    Untracked:  data/DiffSplice_liftedJunc/
    Untracked:  data/DiffSplice_removeBad/
    Untracked:  data/DominantPAS/
    Untracked:  data/DominantPAS_DF/
    Untracked:  data/EvalPantro5/
    Untracked:  data/HC_filenames.txt
    Untracked:  data/HC_filenames.xlsx
    Untracked:  data/Khan_prot/
    Untracked:  data/Li_eqtls/
    Untracked:  data/MapPantro6_meta.txt
    Untracked:  data/MapPantro6_meta.xlsx
    Untracked:  data/MapStats/
    Untracked:  data/NormalizedClusters/
    Untracked:  data/NuclearHvC/
    Untracked:  data/NuclearHvC_DF/
    Untracked:  data/OppositeSpeciesMap.txt
    Untracked:  data/OppositeSpeciesMap.xlsx
    Untracked:  data/OverlapBenchmark/
    Untracked:  data/PAS/
    Untracked:  data/PAS_doubleFilter/
    Untracked:  data/Peaks_5perc/
    Untracked:  data/Pheno_5perc/
    Untracked:  data/Pheno_5perc_nuclear/
    Untracked:  data/Pheno_5perc_nuclear_old/
    Untracked:  data/Pheno_5perc_total/
    Untracked:  data/RNASEQ_metadata.txt
    Untracked:  data/RNASEQ_metadata_2Removed.txt
    Untracked:  data/RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/RNASEQ_metadata_stranded.txt
    Untracked:  data/RNASEQ_metadata_stranded.txt.sb-e4bf31f0-ZGnGgl/
    Untracked:  data/RNASEQ_metadata_stranded.xlsx
    Untracked:  data/SignalSites/
    Untracked:  data/SignalSites_doublefilter/
    Untracked:  data/Threeprime2Ortho/
    Untracked:  data/TotalHvC/
    Untracked:  data/TwoBadSampleAnalysis/
    Untracked:  data/Wang_ribo/
    Untracked:  data/apaQTLGenes/
    Untracked:  data/chainFiles/
    Untracked:  data/cleanPeaks_anno/
    Untracked:  data/cleanPeaks_byspecies/
    Untracked:  data/cleanPeaks_lifted/
    Untracked:  data/files4viz_nuclear/
    Untracked:  data/files4viz_nuclear_DF/
    Untracked:  data/leafviz/
    Untracked:  data/liftover_files/
    Untracked:  data/mediation/
    Untracked:  data/metadata_HCpanel.txt
    Untracked:  data/metadata_HCpanel.xlsx
    Untracked:  data/metadata_HCpanel_frompantro5.txt
    Untracked:  data/metadata_HCpanel_frompantro5.xlsx
    Untracked:  data/primaryLift/
    Untracked:  data/reverseLift/
    Untracked:  data/~$RNASEQ_metadata.xlsx
    Untracked:  data/~$metadata_HCpanel.xlsx
    Untracked:  data/.xlsx
    Untracked:  output/dtPlots/
    Untracked:  projectNotes.Rmd

Unstaged changes:
    Modified:   analysis/ExploredAPA.Rmd
    Modified:   analysis/OppositeMap.Rmd
    Modified:   analysis/annotationInfo.Rmd
    Modified:   analysis/comp2apaQTLPAS.Rmd
    Modified:   analysis/correlationPhenos.Rmd
    Modified:   analysis/establishCutoffs.Rmd
    Modified:   analysis/investigatePantro5.Rmd
    Modified:   analysis/multiMap.Rmd
    Modified:   analysis/speciesSpecific.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/07fd45952afd6e6586320a0f6ae9230c1e3f7ff0/analysis/index.Rmd" target="_blank">07fd459</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-22
</td>
<td>
DF for upset and translation protein ven
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/58002317b3430a3dfa938cc9e111cce4f41f77a4/docs/index.html" target="_blank">5800231</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/117fd632969209613fed6fed461293da502a2986/analysis/index.Rmd" target="_blank">117fd63</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-22
</td>
<td>
redo differential analysis with double filt
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/f00297802381bb687f60fb10c2ccea7e9cae8280/docs/index.html" target="_blank">f002978</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/46c489bf544f38874f9e7e15db4dbbd640136cb2/analysis/index.Rmd" target="_blank">46c489b</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-22
</td>
<td>
nuclear diff filtered
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/6c39a2a7656f78c3cef5b7cda1a5163d19f548f2/docs/index.html" target="_blank">6c39a2a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/010c249a1fac4b9e302603953455894bfd148d9e/analysis/index.Rmd" target="_blank">010c249</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
DF species specific
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/48db342468338de2ec45be8692fa7ff7aee80118/docs/index.html" target="_blank">48db342</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/414b47c494215de92521c0276f3847bd092ae798/analysis/index.Rmd" target="_blank">414b47c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
update dominant PAS
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/5525b39f555800466e8d97a5ca8e849940f68ac7/docs/index.html" target="_blank">5525b39</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/2e66af99c18acdebb901d0e367eeec84e940fbde/analysis/index.Rmd" target="_blank">2e66af9</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-21
</td>
<td>
add ss and PAS num DF
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/80f647a5782e2b58660ae49c0bb5d4a8cee5efbf/docs/index.html" target="_blank">80f647a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/96b46ba75ca9a73b88e4c38e83dc4b949baf42d6/analysis/index.Rmd" target="_blank">96b46ba</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-19
</td>
<td>
add enrichment in apapQTLs
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/c211e215cc82898901dd43dc9cfac50fbf10154d/docs/index.html" target="_blank">c211e21</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/7a9f60879915ace82b872f58d7209efd957b93a5/analysis/index.Rmd" target="_blank">7a9f608</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-18
</td>
<td>
finished benchmark
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/5eef3eb638ca2bf1784cce27cdf859279202a165/docs/index.html" target="_blank">5eef3eb</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/5c24c0cad611959f9d9c0266b270a52242768642/analysis/index.Rmd" target="_blank">5c24c0c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-16
</td>
<td>
add cutoff code files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/7e3ab25222fa694c8b875389b0ff50beea884583/docs/index.html" target="_blank">7e3ab25</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c0b3ef6823219aae50e3b6e8c85f9b1ba7ac9a79/analysis/index.Rmd" target="_blank">c0b3ef6</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-16
</td>
<td>
add correlation between pheons
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/0a3545a5989e5163fbd6abaf57a9715cdd295541/docs/index.html" target="_blank">0a3545a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/5dd5e3520a1db76aac47d0cd2ff68516635a8be6/analysis/index.Rmd" target="_blank">5dd5e35</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-15
</td>
<td>
add code to viz nuclear dapa
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/4f5d4929bd9915b744db58348b0e6abd8dbf17e9/docs/index.html" target="_blank">4f5d492</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/3cce3da7f1e29f6354c8ba13cfbf5670d31b266f/analysis/index.Rmd" target="_blank">3cce3da</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-14
</td>
<td>
start mediation analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/c74c3dff8fe4e0f556c85f6e87a52b98af045052/docs/index.html" target="_blank">c74c3df</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/e1c6d25cab3b2417ff0cb817baa5b2ce20c2bc18/analysis/index.Rmd" target="_blank">e1c6d25</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-14
</td>
<td>
add analysis on species spec PAS
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/1b9ae04751591afa424b45746658ede0a2fe8265/docs/index.html" target="_blank">1b9ae04</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/5ea47104fe4b593dc225c6982b9287a75d033568/analysis/index.Rmd" target="_blank">5ea4710</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-13
</td>
<td>
add questions about dAPA pas and genes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/b963392cd54baea2025e643b0f5c7fa5b8a22ce2/docs/index.html" target="_blank">b963392</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1121102d5c13611abe70c2f43c03cea96e44fbc0/analysis/index.Rmd" target="_blank">1121102</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-13
</td>
<td>
add dpau and de overlap with dom intronic
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/df07f1ae847e014b069b024db3499632d98b7199/docs/index.html" target="_blank">df07f1a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/928a30dd19be22274470defdc5d88b3e87a05048/analysis/index.Rmd" target="_blank">928a30d</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
add same and diff dom ss analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/ced196676b623df77391971e34c2804b6ec4d087/docs/index.html" target="_blank">ced1966</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1cdc21d6416fd4732da758fe59f13769a5a06c29/analysis/index.Rmd" target="_blank">1cdc21d</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
add initial signal site analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/dd02dabec8be8444f027abc10b667f1c6e8c5f3a/docs/index.html" target="_blank">dd02dab</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/a41141b42e3d041fb7643c6b4fe21e9144dddc58/docs/index.html" target="_blank">a41141b</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/93ced1e42aac3e088f3a548d93f2ca501a5b0736/analysis/index.Rmd" target="_blank">93ced1e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-09
</td>
<td>
peak 2 pas
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/6d9e3694a2b7c72dc2ed9d8647c74328504d1602/docs/index.html" target="_blank">6d9e369</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/203eae574eb54842cf37786d93397c01153ea3b3/analysis/index.Rmd" target="_blank">203eae5</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-08
</td>
<td>
first steps for intron loc analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/4c87e962435f44f7059b09a4b9647ac5aa9ab9b2/docs/index.html" target="_blank">4c87e96</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/6e3027ad7348b369c091f8fb903edfee7cd13b3d/analysis/index.Rmd" target="_blank">6e3027a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-07
</td>
<td>
upset plots for comparisons
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/776853582da60e8c2f6bb43a1a0b14e339c47463/docs/index.html" target="_blank">7768535</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/670020f30d49aae921fa8368b8e7db48a43d4ce5/analysis/index.Rmd" target="_blank">670020f</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-07
</td>
<td>
add translation protien overlap
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/dec23c17b58af879e35616b4e87db89ea5b422fe/docs/index.html" target="_blank">dec23c1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/54015c83b7d5a5583ee4c3cba8e9f12734ca0000/analysis/index.Rmd" target="_blank">54015c8</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-06
</td>
<td>
add overlap dom with dAPA
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/c633e63b7908ec3c5e4a1dd03454495fe8d80f38/docs/index.html" target="_blank">c633e63</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/5e857040d3c1c7aa994183714a5c0a7c84632596/analysis/index.Rmd" target="_blank">5e85704</a>
</td>
<td>
brimittleman
</td>
<td>
2020-01-06
</td>
<td>
add overlap with eQTLs
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/dc41b3bed4b09ecfbadc5582fe9344c478a36f26/docs/index.html" target="_blank">dc41b3b</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/6214ec5ac07800db9fcab129908c163965822c19/analysis/index.Rmd" target="_blank">6214ec5</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-30
</td>
<td>
add venn diagram for diff apa
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/42a8b12141c2eaa3041cbbb71c6923e210c46dc6/docs/index.html" target="_blank">42a8b12</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/385057f01b6b0ca687e6e47d3a4bc8203d56768b/analysis/index.Rmd" target="_blank">385057f</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-30
</td>
<td>
fix dom PAS on index
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/0176b490c9d47ce9a1a57debcb33dd33a8678879/docs/index.html" target="_blank">0176b49</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/5b25363cd6afdf6d3159d57a65fb785318f7fab7/analysis/index.Rmd" target="_blank">5b25363</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-30
</td>
<td>
add total and nuclear dominant PAS analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/26447479c3188b6643bf89ec83ac1c4042b4369b/docs/index.html" target="_blank">2644747</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/4c8973d5a093a6450fa78c96b3fc524bb3ca971a/analysis/index.Rmd" target="_blank">4c8973d</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-27
</td>
<td>
add total human vs chimp
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/ffecde802b87c4d1b1578945db9807fab043f0b4/docs/index.html" target="_blank">ffecde8</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/d541bb5b0b501844781d1db3cb73f9182dc492d6/analysis/index.Rmd" target="_blank">d541bb5</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-17
</td>
<td>
move extra pcas
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/8b074b88d8eddfe32f775bf287b0ee2308d0a16e/docs/index.html" target="_blank">8b074b8</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/de41e07c5a865e0fe881b5301917b46a3213b9ef/analysis/index.Rmd" target="_blank">de41e07</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-16
</td>
<td>
add gene comp 5v6
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/1b69793cd30f749ee4b0ba21d45783b7752cc282/docs/index.html" target="_blank">1b69793</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/3b64d8c2dfa1c6a7c2c5e3687a09e2963987fed6/analysis/index.Rmd" target="_blank">3b64d8c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-16
</td>
<td>
add compare ortho exon
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/cbadb32d4f2c53f97ffab3f48cd89c838f4ed583/docs/index.html" target="_blank">cbadb32</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/8068598fcdb9515d4c3f7b6b429971c0d2c7eec7/analysis/index.Rmd" target="_blank">8068598</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-12
</td>
<td>
meta data by species
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/3ee838285a224d7b74f683ccc93ff29a21a4333e/docs/index.html" target="_blank">3ee8382</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f04a8042f4b698ab19e130b931b5aa8075a125a4/analysis/index.Rmd" target="_blank">f04a804</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-08
</td>
<td>
add DE vs DS analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/b32d961465fc2811af401d0ac4ff09dee895107c/docs/index.html" target="_blank">b32d961</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1452408d077ed41af636026bbe45b5aaf6066893/analysis/index.Rmd" target="_blank">1452408</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-05
</td>
<td>
add investigation pantro5
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/144bc43946f728c856c1039f0d8b7d0f6eb6e936/docs/index.html" target="_blank">144bc43</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/d56d98cb3aa2e4768d272c382017bdc8889a06ed/analysis/index.Rmd" target="_blank">d56d98c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-12-05
</td>
<td>
add opp map
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/f53ab2772e3e897295f109389cb1cf19a7d71c4d/docs/index.html" target="_blank">f53ab27</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c48aedcbe8688bc762be9e2aa901a0a50606c3fc/analysis/index.Rmd" target="_blank">c48aedc</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-24
</td>
<td>
add multimap stat graphs
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/e7cb428de68cdf9db814b7fb859caa0e954338f1/docs/index.html" target="_blank">e7cb428</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/9fbd1fe61d1a16783614fab355cc522bf11cb279/analysis/index.Rmd" target="_blank">9fbd1fe</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-21
</td>
<td>
figure out sample swap
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/2e33ce93a1227b8cb84cacd5b3d914dcab1fc71c/docs/index.html" target="_blank">2e33ce9</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/17d5d33beaa8e717b613ff408d8948415d559837/analysis/index.Rmd" target="_blank">17d5d33</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-21
</td>
<td>
add diff splice without 2 samples
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/40db68c19e220862b8f27ddadb157ad7f9f17bd8/docs/index.html" target="_blank">40db68c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/7c206f8564758f532df70a03de7ba08ae322a15c/analysis/index.Rmd" target="_blank">7c206f8</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-20
</td>
<td>
add extra QC anaylsis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/9ecd7691d60ca3c6b742c1f3a4956f51832bd63e/docs/index.html" target="_blank">9ecd769</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/db85b26c94005e7cad373dc524afb78f7f2ad8fb/analysis/index.Rmd" target="_blank">db85b26</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-20
</td>
<td>
add remove 2 analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/dc91b0af79367ef559ad277a01090d4c950dcc88/docs/index.html" target="_blank">dc91b0a</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b5ba82eee24903f1da5a92da2559275ffc71912d/analysis/index.Rmd" target="_blank">b5ba82e</a>
</td>
<td>
brimittleman
</td>
<td>
2019-11-11
</td>
<td>
add diff expression and diff splicing
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/066d419290a2f4c3eef8bdf538216f3d348572fb/docs/index.html" target="_blank">066d419</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a255e22b9d32ad80884501ae03a7c4e162fdbba5/analysis/index.Rmd" target="_blank">a255e22</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-16
</td>
<td>
add code and annotations for dt plots
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/f4bcae9ac5f32e2f228bd29eaaf2ae525899ce22/docs/index.html" target="_blank">f4bcae9</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/25a8b1e42180e62d71b4ff4dd0400102843f1cec/analysis/index.Rmd" target="_blank">25a8b1e</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-15
</td>
<td>
fix name bug add number PAS analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/d9f241f464a962076d2b3378fc27d4167217ae49/docs/index.html" target="_blank">d9f241f</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/36d8ded5078c72dbd927d0f8c1c26d5febef08bd/analysis/index.Rmd" target="_blank">36d8ded</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-11
</td>
<td>
add correlation between ind analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/3f3c4d0bbcd2e09211f2fea1ac28e58777e3d00f/docs/index.html" target="_blank">3f3c4d0</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c4db4f0e862f52d0ff492d5ca928b9bc0ae2b345/analysis/index.Rmd" target="_blank">c4db4f0</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-10
</td>
<td>
add non norm pca and verify bam
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/b8ccfdba858ba7a58fd036f32380c6c91d513263/docs/index.html" target="_blank">b8ccfdb</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1f063297413d5951470981d2eb95540825326ce0/analysis/index.Rmd" target="_blank">1f06329</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-09
</td>
<td>
add PCA by total and nuclear
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/d0c98c261010b35d00259c9ca5488e2e46448332/docs/index.html" target="_blank">d0c98c2</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/14a3f665ce75def8370c8c6129603a205c5648d0/analysis/index.Rmd" target="_blank">14a3f66</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-09
</td>
<td>
add pca and human v chimp in nuc analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/c3aa529bce951a381624b7854c6e7522f3c14bff/docs/index.html" target="_blank">c3aa529</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/da85ad8c1c69856141190009e20c4ef98ed07b76/analysis/index.Rmd" target="_blank">da85ad8</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-07
</td>
<td>
add code to prepare human and chimp TvN
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/4ea25767ec9bed8fe46e366e324043d228e57f5e/docs/index.html" target="_blank">4ea2576</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/920abac02a19940bb87f056a3a06dcb35cd5016d/analysis/index.Rmd" target="_blank">920abac</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-04
</td>
<td>
add comaprison to old PAS
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/e0ac227e5aa1b09a1da0fa75aaab25b9f0eb749a/docs/index.html" target="_blank">e0ac227</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/e3f0cdffde5f9e8344479c42d2e16e725e73b22e/analysis/index.Rmd" target="_blank">e3f0cdf</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-03
</td>
<td>
add annotation analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/b5edd8e21fc0cf262cf1332d74ffc1917c2ff7f7/docs/index.html" target="_blank">b5edd8e</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b5a215119477fca80ff396449a2cbbbff36db2d1/analysis/index.Rmd" target="_blank">b5a2151</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-02
</td>
<td>
add analysis for liftover
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/0e8aa0da4367a78c4c39a2d082b6d235b78237e8/docs/index.html" target="_blank">0e8aa0d</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-01
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/7d81da82a31dd6f9169c49f13873a66afba68755/analysis/index.Rmd" target="_blank">7d81da8</a>
</td>
<td>
brimittleman
</td>
<td>
2019-10-01
</td>
<td>
add map stats to site
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/7a9cf749ef4c1a7f45bf19aab838396625c8284c/docs/index.html" target="_blank">7a9cf74</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a2fe9d6cca95411e4ce8b2cfd67d80c7b22c418d/analysis/index.Rmd" target="_blank">a2fe9d6</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-24
</td>
<td>
add annotation prep anaylsis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/03cb9ab5e687900eff39dc89fbad5178e6a10a6c/docs/index.html" target="_blank">03cb9ab</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/88ed6b434a8626d6210e8921e2154078dca095d6/docs/index.html" target="_blank">88ed6b4</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b6db8f3924282d211181aeaf4baa6851908f18aa/analysis/index.Rmd" target="_blank">b6db8f3</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-02
</td>
<td>
add project info to about and index
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/d03d7b001d5748f6f9162210b368fab2dac90662/docs/index.html" target="_blank">d03d7b0</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f914d4920107a1b4c48c590209133aa9ac5817de/analysis/index.Rmd" target="_blank">f914d49</a>
</td>
<td>
brimittleman
</td>
<td>
2019-09-02
</td>
<td>
Start workflowr project.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>I am interesting in understanding differential APA between human and chimpanzees. I used 3’ Seq to identify PAS for total and nuclear frations of LCL in both species.</p>
<div id="preparation-and-qc" class="section level2">
<h2>Preparation and QC:</h2>
<ul>
<li><p><a href="annotationInfo.html">Prepare annotation files for mapping</a></p></li>
<li><p><a href="mapStats.html">Mapping statistics and plots</a></p></li>
<li><p><a href="liftoverPAS.html">Liftover PAS</a></p></li>
<li><p><a href="annotatePAS.html">Annotated PAS and make APA phenotype</a></p></li>
<li><p><a href="comp2apaQTLPAS.html">Compare identified PAS with apaQTL PAS</a></p></li>
<li><p><a href="NromalizePhenos.html">PCA plot to look at variation in data</a></p></li>
<li><p><a href="CorrbetweenInd.html">Correlation between individuals</a></p></li>
<li><p><a href="verifyBAM.Rmd">Verify sample identity</a></p></li>
<li><p><a href="EnrichmentEndofGenes.html">3’ Seq enrichement along gene bodies</a></p></li>
<li><p><a href="CrossChimpThreePrime.html">Cross map and prepare Chimp 3’ for Viz in IGV</a></p></li>
<li><p><a href="fixPeak2PAS.html">Make PAS bed for 5% peaks</a></p></li>
</ul>
<div id="choosing-cuttoffs-for-usage-and-expression-level" class="section level3">
<h3>Choosing cuttoffs for usage and expression level</h3>
<ul>
<li><p><a href="expressioncutoff.html">Use merged 3’ at gene level to look for a data quantity cutoff</a></p></li>
<li><p><a href="establishCutoffs.html">Establish cutoffs by subsampleing apaQTL lines</a></p></li>
</ul>
</div>
</div>
<div id="catalog-pas" class="section level2">
<h2>Catalog PAS:</h2>
<ul>
<li><p><a href="PASnumperSpecies.html">Number of PAS in each species</a></p>
<ul>
<li><a href="PASnumberSpecies_doubleFilter.html">Number of PAS in each species double filter</a></li>
</ul></li>
<li><p><a href="signalsites.html">Pressence of top 2 signal sites</a></p>
<ul>
<li><a href="signalsites_doublefilter.html">Top 2 signal sites double filter</a></li>
</ul></li>
<li><p><a href="dominantPAS.html">Dominant PAS in Nuclear Fraction</a></p>
<pre><code>  * [Dominant PAS in Nuclear Fraction double filter](dominantPAS_DF.html)  </code></pre>
<ul>
<li><p><a href="DominantPASintronLoc.html">Intronic location for dominant PAS</a></p>
<ul>
<li><a href="DominantPASintronLoc_DF.html">Intronic location for dominant PAS double filter</a></li>
</ul></li>
</ul></li>
<li><p><a href="speciesSpecific.html">Species Specificity in PAS</a></p>
<ul>
<li><a href="speciesSpecific_DF.html">Species Specificity in PAS double filter</a></li>
</ul></li>
</ul>
</div>
<div id="differential-isoform-analysis" class="section level2">
<h2>Differential Isoform Analysis:</h2>
<ul>
<li><p><a href="Nuclear_HvC.html">Nuclear Human vs. Chimp Analysis</a></p>
<ul>
<li><a href="Nuclear_HvC_DF.html">Nuclear Human vs. Chimp Analysis Double filter</a></li>
</ul></li>
<li><p><a href="ExploredAPA.html">Explore nuclear differential APA PAS and genes</a></p>
<ul>
<li><a href="ExploredAPA_DF.html">Explore nuclear differential APA PAS and genes Double filter</a></li>
</ul></li>
<li><p><a href="dAPAandapaQTL.html">Enrichment for differencial APA and apaQTL</a></p>
<ul>
<li><a href="dAPAandapaQTL_DF.html">Enrichment for differencial APA and apaQTL double filter</a></li>
</ul></li>
<li><p><a href="visualizePASUsage.html">Visualization for all Nuclear DAPA genes</a></p>
<ul>
<li><a href="visualizePASUsage_DF.html">Visualization for all Nuclear DAPA genes double filter</a></li>
</ul></li>
</ul>
</div>
<div id="quantify-phenotypes-with-rna" class="section level2">
<h2>Quantify Phenotypes with RNA:</h2>
<ul>
<li><p><a href="diffSplicing.html">Differential Splicing</a></p></li>
<li><p><a href="diffExpression.html">Differential Expression</a></p>
<ul>
<li><p><a href="MetabySpecies.html">Metadata by species</a></p></li>
<li><p><a href="DEandeQTls.html">DE genes and eQTLs</a></p></li>
</ul></li>
<li><p><a href="DEvDS.html">Compare DE and DS genes</a></p></li>
</ul>
</div>
<div id="intersect-with-outside-molecular-phenotypes" class="section level2">
<h2>Intersect with outside molecular phenotypes:</h2>
<ul>
<li><p><a href="dProtanddRibo.html">Download data from differential protein and ribosomes</a></p></li>
<li><p><a href="DiffTransProtvsNuclearAPA.html">Compare to nuclear differential APA</a></p>
<ul>
<li><a href="DiffTransProtvsNuclearAPA_DF.html">Compare to nuclear differential APA with Double filter</a></li>
</ul></li>
<li><p><a href="upsetter.html">Use UpSetR to vizualize intersections</a></p>
<ul>
<li><a href="upsetter_DF.html">Use UpSetR to vizualize intersections Double filter</a></li>
</ul></li>
<li><p><a href="correlationPhenos.html">Effect size correlations</a></p></li>
</ul>
</div>
<div id="mediation" class="section level2">
<h2>Mediation</h2>
<ul>
<li><p><a href="NormalizePheno_nuclear.html">Normalize Nuclear Fraction</a></p></li>
<li><p><a href="normalizeClusters.html">Normalize clusters from splicing analysis</a></p></li>
<li><p><a href="mediation.html">Mediation analysis</a></p></li>
</ul>
</div>
<div id="extra-qc-and-analysis" class="section level2">
<h2>Extra QC and Analysis</h2>
<div id="sample-switch-analysis" class="section level3">
<h3>Sample switch analysis</h3>
<ul>
<li><p><a href="InvestigateBadSamples.html">Investigate bad RNA seq samples</a></p>
<ul>
<li><p><a href="diffSplicing_removeBad.html">Differential splicing without bad samples</a></p></li>
<li><p><a href="diffExpressionRemove2.html">Differential Expression without bad samples</a></p></li>
</ul></li>
</ul>
</div>
<div id="mapping-problems-with-pantro5-and-multimapping" class="section level3">
<h3>Mapping problems with pantro5 and multimapping</h3>
<ul>
<li><p><a href="MapRNAhg19.html">Map RNA to hg19</a></p></li>
<li><p><a href="multiMap.html">Evaluate multimapping criteria</a></p></li>
<li><p><a href="OppositeMap.html">Map to opposite Genomes</a></p></li>
<li><p><a href="investigatePantro5.html">Evaluate Pantro 5 problems</a></p></li>
</ul>
<p>From before reseq:</p>
<ul>
<li><p><a href="NormalizePheno_total.html">Total Fraction</a></p></li>
<li><p><a href="NonNormPheno_total.html">Total Fraction- Prenormalized</a></p></li>
</ul>
</div>
<div id="chosing-between-pantro-files" class="section level3">
<h3>Chosing between pantro files</h3>
<ul>
<li><p><a href="compareOrthoExon.html">Compare pantro5 and pantro6 orthoexon</a></p></li>
<li><p><a href="diffExpressionPantro6.html">Differential Expression with Pantro6</a></p></li>
<li><p><a href="DE_pantro5vspantro6.html">Compare DE in pantro5 vs pantro6</a></p></li>
</ul>
</div>
<div id="extra-comparisons-or-analysis-not-redone-after-double-filter" class="section level3">
<h3>Extra comparisons or analysis: not redone after double filter</h3>
<ul>
<li><p><a href="Human_TvN.html">Human Total vs Nuclear Analysis</a></p></li>
<li><p><a href="Chimp_TvN.html">Chimp Total vs Nuclear Analysis</a></p></li>
<li><p><a href="dominantPAS_total.html">Dominant PAS in Total Fraction</a></p></li>
<li><p><a href="Total_HvC.html">Total Human vs. Chimp Analysis</a></p></li>
<li><p><a href="CompHC_totalandnuclear.html">Venn diagrams for genes in each differential APA set</a></p></li>
</ul>
<p>From the original nuclear dominant</p>
<ul>
<li><p><a href="ss_sameDomdiffDom.html">Does signal site explain trend?</a></p></li>
<li><p><a href="HumanIntronicDominant.html">Look at Intronic Dominant with DPAU and DE</a></p></li>
<li><p><a href="DapaineQTL.html">Enrichment for differencial APA and eQTL</a></p></li>
<li><p><a href="DominatPASandDIffAPA.html">Look at dAPA and genes from Dominant PAS analysis</a></p></li>
</ul>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
