<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Briana Mittleman" />


<title>Main figures</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/journal.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Comparative_APA</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/brimittleman/Comparative_APA">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Main figures</h1>
<h4 class="author">Briana Mittleman</h4>
<h4 class="date">5/30/2020</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-07-25
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Comparative_APA/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20190902code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20190902)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20190902code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20190902)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtreedc22f1753a71f616779ab7e2ec7c5d40787ae600targetblankdc22f17a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/brimittleman/Comparative_APA/tree/dc22f1753a71f616779ab7e2ec7c5d40787ae600" target="_blank">dc22f17</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtreedc22f1753a71f616779ab7e2ec7c5d40787ae600targetblankdc22f17a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/brimittleman/Comparative_APA/tree/dc22f1753a71f616779ab7e2ec7c5d40787ae600" target="_blank">dc22f17</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    code/chimp_log/
    Ignored:    code/human_log/
    Ignored:    data/.DS_Store
    Ignored:    data/TrialFiltersMeta.txt.sb-9845453e-R58Y0Q/
    Ignored:    data/mediation_prot/
    Ignored:    data/metadata_HCpanel.txt.sb-284518db-RGf0kd/
    Ignored:    data/metadata_HCpanel.txt.sb-a5794dd2-i594qs/
    Ignored:    output/.DS_Store

Untracked files:
    Untracked:  ._.DS_Store
    Untracked:  Chimp/
    Untracked:  GEO/
    Untracked:  Human/
    Untracked:  analysis/ALUelements.Rmd
    Untracked:  analysis/AREstabilityScores.Rmd
    Untracked:  analysis/AllLoc_effectSizeCor.Rmd
    Untracked:  analysis/Conservation_bydAPAset.Rmd
    Untracked:  analysis/CrossChimpThreePrime.Rmd
    Untracked:  analysis/DiffTransProtvsExpression.Rmd
    Untracked:  analysis/DiffUsedUTR.Rmd
    Untracked:  analysis/GvizPlots.Rmd
    Untracked:  analysis/HandC.TvN
    Untracked:  analysis/PhenotypeOverlap10.Rmd
    Untracked:  analysis/ResultsNoUnlifted.md
    Untracked:  analysis/SuppTables.Rmd
    Untracked:  analysis/annotationBias.Rmd
    Untracked:  analysis/assessReadQual.Rmd
    Untracked:  analysis/diffExpressionPantro6.Rmd
    Untracked:  analysis/isoformdivandexp.Rmd
    Untracked:  code/._AlignmentScores.sh
    Untracked:  code/._BothFCMM.sh
    Untracked:  code/._BothFCMMPrim.sh
    Untracked:  code/._BothFCnewOInclusive.sh
    Untracked:  code/._ChimpStarMM2.sh
    Untracked:  code/._ClassifyLeafviz.sh
    Untracked:  code/._ClosestorthoEx.sh
    Untracked:  code/._Config_chimp.yaml
    Untracked:  code/._Config_chimp_full.yaml
    Untracked:  code/._Config_human.yaml
    Untracked:  code/._ConvertJunc2Bed.sh
    Untracked:  code/._CountNucleotides.py
    Untracked:  code/._CrossMapChimpRNA.sh
    Untracked:  code/._CrossMapThreeprime.sh
    Untracked:  code/._DiffSplice.sh
    Untracked:  code/._DiffSplicePlots.sh
    Untracked:  code/._DiffSplicePlots_gencode.sh
    Untracked:  code/._DiffSplice_gencode.sh
    Untracked:  code/._DiffSplice_removebad.sh
    Untracked:  code/._Filter255MM.sh
    Untracked:  code/._FilterPrimSec.sh
    Untracked:  code/._FindIntronForDomPAS.sh
    Untracked:  code/._FindIntronForDomPAS_DF.sh
    Untracked:  code/._GetMAPQscore.py
    Untracked:  code/._GetSecondaryMap.py
    Untracked:  code/._Lift5perPAS.sh
    Untracked:  code/._LiftFinalChimpJunc2Human.sh
    Untracked:  code/._LiftOrthoPAS2chimp.sh
    Untracked:  code/._MapBadSamples.sh
    Untracked:  code/._MismatchNumbers.sh
    Untracked:  code/._PAS_ATTAAA.sh
    Untracked:  code/._PAS_ATTAAA_df.sh
    Untracked:  code/._PAS_seqExpanded.sh
    Untracked:  code/._PASsequences.sh
    Untracked:  code/._PASsequences_DF.sh
    Untracked:  code/._PlotNuclearUsagebySpecies.R
    Untracked:  code/._PlotNuclearUsagebySpecies_DF.R
    Untracked:  code/._QuantMergedClusters.sh
    Untracked:  code/._RNATranscriptDTplot.sh
    Untracked:  code/._ReverseLiftFilter.R
    Untracked:  code/._RunFixLeafCluster.sh
    Untracked:  code/._RunNegMCMediation.sh
    Untracked:  code/._RunNegMCMediationDF.sh
    Untracked:  code/._RunPosMCMediationDF.err
    Untracked:  code/._RunPosMCMediationDF.sh
    Untracked:  code/._SAF2Bed.py
    Untracked:  code/._Snakefile
    Untracked:  code/._SnakefilePAS
    Untracked:  code/._SnakefilePASfilt
    Untracked:  code/._SortIndexBadSamples.sh
    Untracked:  code/._StarMM2.sh
    Untracked:  code/._TestFC.sh
    Untracked:  code/._assignPeak2Intronicregion
    Untracked:  code/._assignPeak2Intronicregion.sh
    Untracked:  code/._bed215upbed.py
    Untracked:  code/._bed2Bedbothstrand.py
    Untracked:  code/._bed2SAF_gen.py
    Untracked:  code/._buildIndecpantro5
    Untracked:  code/._buildIndecpantro5.sh
    Untracked:  code/._buildLeafviz.sh
    Untracked:  code/._buildLeafviz_leadAnno.sh
    Untracked:  code/._buildStarIndex.sh
    Untracked:  code/._chimpChromprder.sh
    Untracked:  code/._chimpMultiCov.sh
    Untracked:  code/._chimpMultiCov255.sh
    Untracked:  code/._chimpMultiCovInclusive.sh
    Untracked:  code/._chooseSignalSite.py
    Untracked:  code/._cleanbed2saf.py
    Untracked:  code/._cluster.json
    Untracked:  code/._cluster2bed.py
    Untracked:  code/._clusterLiftReverse.sh
    Untracked:  code/._clusterLiftReverse_removebad.sh
    Untracked:  code/._clusterLiftprimary.sh
    Untracked:  code/._clusterLiftprimary_removebad.sh
    Untracked:  code/._converBam2Junc.sh
    Untracked:  code/._converBam2Junc_removeBad.sh
    Untracked:  code/._extraSnakefiltpas
    Untracked:  code/._extractPhyloReg.py
    Untracked:  code/._extractPhyloRegGene.py
    Untracked:  code/._extractPhylopGeneral.ph
    Untracked:  code/._extractPhylopGeneral.py
    Untracked:  code/._extractPhylopReg200down.py
    Untracked:  code/._extractPhylopReg200up.py
    Untracked:  code/._filter5percPAS.py
    Untracked:  code/._filterNumChroms.py
    Untracked:  code/._filterPASforMP.py
    Untracked:  code/._filterPostLift.py
    Untracked:  code/._filterPrimaryread.py
    Untracked:  code/._filterSecondaryread.py
    Untracked:  code/._fixExonFC.py
    Untracked:  code/._fixFCheadforExp.py
    Untracked:  code/._fixLeafCluster.py
    Untracked:  code/._fixLiftedJunc.py
    Untracked:  code/._fixUTRexonanno.py
    Untracked:  code/._formathg38Anno.py
    Untracked:  code/._formatpantro6Anno.py
    Untracked:  code/._getRNAseqMapStats.sh
    Untracked:  code/._hg19MapStats.sh
    Untracked:  code/._humanChromorder.sh
    Untracked:  code/._humanMultiCov.sh
    Untracked:  code/._humanMultiCov255.sh
    Untracked:  code/._humanMultiCov_inclusive.sh
    Untracked:  code/._intersectLiftedPAS.sh
    Untracked:  code/._liftJunctionFiles.sh
    Untracked:  code/._liftPAS19to38.sh
    Untracked:  code/._liftedchimpJunc2human.sh
    Untracked:  code/._makeNuclearDapaplots.sh
    Untracked:  code/._makeNuclearDapaplots_DF.sh
    Untracked:  code/._makeSamplyGroupsHuman_TvN.py
    Untracked:  code/._mapRNAseqhg19.sh
    Untracked:  code/._mapRNAseqhg19_newPipeline.sh
    Untracked:  code/._maphg19.sh
    Untracked:  code/._maphg19_subjunc.sh
    Untracked:  code/._mediation_test.R
    Untracked:  code/._mergeChimp3prime_inhg38.sh
    Untracked:  code/._mergeandBWRNAseq.sh
    Untracked:  code/._mergedBam2BW.sh
    Untracked:  code/._nameClusters.py
    Untracked:  code/._negativeMediation_montecarlo.R
    Untracked:  code/._negativeMediation_montecarloDF.R
    Untracked:  code/._numMultimap.py
    Untracked:  code/._overlapMMandOrthoexon.sh
    Untracked:  code/._overlapPASandOrthoexon.sh
    Untracked:  code/._overlapapaQTLPAS.sh
    Untracked:  code/._parseHg38.py
    Untracked:  code/._postiveMediation_montecarlo_DF.R
    Untracked:  code/._prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/._prepareLeafvizAnno.sh
    Untracked:  code/._preparePAS4lift.py
    Untracked:  code/._primaryLift.sh
    Untracked:  code/._processhg38exons.py
    Untracked:  code/._quantJunc.sh
    Untracked:  code/._quantJunc_TEST.sh
    Untracked:  code/._quantJunc_removeBad.sh
    Untracked:  code/._quantLiftedPASPrimary.sh
    Untracked:  code/._quantMerged_seperatly.sh
    Untracked:  code/._recLiftchim2human.sh
    Untracked:  code/._revLiftPAShg38to19.sh
    Untracked:  code/._reverseLift.sh
    Untracked:  code/._runCheckReverseLift.sh
    Untracked:  code/._runChimpDiffIso.sh
    Untracked:  code/._runCountNucleotides.sh
    Untracked:  code/._runFilterNumChroms.sh
    Untracked:  code/._runHumanDiffIso.sh
    Untracked:  code/._runNuclearDiffIso_DF.sh
    Untracked:  code/._runNuclearDifffIso.sh
    Untracked:  code/._runTotalDiffIso.sh
    Untracked:  code/._run_chimpverifybam.sh
    Untracked:  code/._run_verifyBam.sh
    Untracked:  code/._snakemake.batch
    Untracked:  code/._snakemakePAS.batch
    Untracked:  code/._snakemakePASchimp.batch
    Untracked:  code/._snakemakePAShuman.batch
    Untracked:  code/._snakemake_chimp.batch
    Untracked:  code/._snakemake_human.batch
    Untracked:  code/._snakemakefiltPAS.batch
    Untracked:  code/._snakemakefiltPAS_chimp
    Untracked:  code/._snakemakefiltPAS_chimp.sh
    Untracked:  code/._snakemakefiltPAS_human.sh
    Untracked:  code/._spliceSite2Fasta.py
    Untracked:  code/._submit-snakemake-chimp.sh
    Untracked:  code/._submit-snakemake-human.sh
    Untracked:  code/._submit-snakemakePAS-chimp.sh
    Untracked:  code/._submit-snakemakePAS-human.sh
    Untracked:  code/._submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/._submit-snakemakefiltPAS-human.sh
    Untracked:  code/._subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/._subset_diffisopheno_Nuclear_HvC_DF.py
    Untracked:  code/._subset_diffisopheno_Total_HvC.py
    Untracked:  code/._threeprimeOrthoFC.sh
    Untracked:  code/._transcriptDTplotsNuclear.sh
    Untracked:  code/._verifyBam4973.sh
    Untracked:  code/._verifyBam4973inHuman.sh
    Untracked:  code/._wrap_chimpverifybam.sh
    Untracked:  code/._wrap_verifyBam.sh
    Untracked:  code/._writeMergecode.py
    Untracked:  code/.snakemake/
    Untracked:  code/ALLPAS_sequenceDF.err
    Untracked:  code/ALLPAS_sequenceDF.out
    Untracked:  code/AlignmentScores.err
    Untracked:  code/AlignmentScores.out
    Untracked:  code/AlignmentScores.sh
    Untracked:  code/BothFCMM.err
    Untracked:  code/BothFCMM.out
    Untracked:  code/BothFCMM.sh
    Untracked:  code/BothFCMMPrim.err
    Untracked:  code/BothFCMMPrim.out
    Untracked:  code/BothFCMMPrim.sh
    Untracked:  code/BothFCnewOInclusive.sh
    Untracked:  code/BothFCnewOInclusive.sh.err
    Untracked:  code/BothFCnewOInclusive.sh.out
    Untracked:  code/ChimpStarMM2.err
    Untracked:  code/ChimpStarMM2.out
    Untracked:  code/ChimpStarMM2.sh
    Untracked:  code/ClassifyLeafviz.sh
    Untracked:  code/ClosestorthoEx.err
    Untracked:  code/ClosestorthoEx.out
    Untracked:  code/ClosestorthoEx.sh
    Untracked:  code/Config_chimp.yaml
    Untracked:  code/Config_chimp_full.yaml
    Untracked:  code/Config_human.yaml
    Untracked:  code/ConvertJunc2Bed.err
    Untracked:  code/ConvertJunc2Bed.out
    Untracked:  code/ConvertJunc2Bed.sh
    Untracked:  code/CountNucleotides.py
    Untracked:  code/CrossMapChimpRNA.sh
    Untracked:  code/CrossMapThreeprime.sh
    Untracked:  code/CrossmapChimp3prime.err
    Untracked:  code/CrossmapChimp3prime.out
    Untracked:  code/CrossmapChimpRNA.err
    Untracked:  code/CrossmapChimpRNA.out
    Untracked:  code/DTUTR.sh
    Untracked:  code/DiffDom_RNAmotif_4.err
    Untracked:  code/DiffDom_RNAmotif_4.out
    Untracked:  code/DiffDom_RNAmotif_4.sh
    Untracked:  code/DiffDom_RNAmotif_4_splitDE.err
    Untracked:  code/DiffDom_RNAmotif_4_splitDE.out
    Untracked:  code/DiffDom_RNAmotif_4_splitDE.sh
    Untracked:  code/DiffSplice.err
    Untracked:  code/DiffSplice.out
    Untracked:  code/DiffSplice.sh
    Untracked:  code/DiffSplicePlots.err
    Untracked:  code/DiffSplicePlots.out
    Untracked:  code/DiffSplicePlots.sh
    Untracked:  code/DiffSplicePlots_gencode.sh
    Untracked:  code/DiffSplice_gencode.sh
    Untracked:  code/DiffSplice_removebad.err
    Untracked:  code/DiffSplice_removebad.out
    Untracked:  code/DiffSplice_removebad.sh
    Untracked:  code/Filter255.err
    Untracked:  code/Filter255.out
    Untracked:  code/Filter255MM.sh
    Untracked:  code/FilterPrimSec.err
    Untracked:  code/FilterPrimSec.out
    Untracked:  code/FilterPrimSec.sh
    Untracked:  code/FilterReverseLift.err
    Untracked:  code/FilterReverseLift.out
    Untracked:  code/FindDomXCutoff.py
    Untracked:  code/FindIntronForDomPAS.err
    Untracked:  code/FindIntronForDomPAS.out
    Untracked:  code/FindIntronForDomPAS.sh
    Untracked:  code/FindIntronForDomPAS_DF.sh
    Untracked:  code/GencodeDiffSplice.err
    Untracked:  code/GencodeDiffSplice.out
    Untracked:  code/GetMAPQscore.py
    Untracked:  code/GetSecondaryMap.py
    Untracked:  code/GetTopminus2Usage.py
    Untracked:  code/H3K36me3DTplot.err
    Untracked:  code/H3K36me3DTplot.out
    Untracked:  code/H3K36me3DTplot.sh
    Untracked:  code/H3K36me3DTplot_DiffIso.err
    Untracked:  code/H3K36me3DTplot_DiffIso.out
    Untracked:  code/H3K36me3DTplot_DiffIso.sh
    Untracked:  code/H3K36me3DTplot_Specific.err
    Untracked:  code/H3K36me3DTplot_Specific.out
    Untracked:  code/H3K36me3DTplot_Specific.sh
    Untracked:  code/H3K36me3DTplot_distalPAS.err
    Untracked:  code/H3K36me3DTplot_distalPAS.out
    Untracked:  code/H3K36me3DTplot_distalPAS.sh
    Untracked:  code/H3K36me3DTplot_transcript.err
    Untracked:  code/H3K36me3DTplot_transcript.out
    Untracked:  code/H3K36me3DTplot_transcript.sh
    Untracked:  code/H3K36me3DTplotwide.err
    Untracked:  code/H3K36me3DTplotwide.out
    Untracked:  code/H3K36me3DTplotwide.sh
    Untracked:  code/H3K9me3DTplot_transcript.err
    Untracked:  code/H3K9me3DTplot_transcript.out
    Untracked:  code/H3K9me3DTplot_transcript.sh
    Untracked:  code/H3K9me3_processandDT.sh
    Untracked:  code/HchromOrder.err
    Untracked:  code/HchromOrder.out
    Untracked:  code/InfoContentShannon.py
    Untracked:  code/InfoContentbyInd.py
    Untracked:  code/IntersectMMandOrtho.err
    Untracked:  code/IntersectMMandOrtho.out
    Untracked:  code/IntersectPASandOrtho.err
    Untracked:  code/IntersectPASandOrtho.out
    Untracked:  code/JunctionLift.err
    Untracked:  code/JunctionLift.out
    Untracked:  code/JunctionLiftFinalChimp.err
    Untracked:  code/JunctionLiftFinalChimp.out
    Untracked:  code/Lift5perPAS.sh
    Untracked:  code/Lift5perPASbed.err
    Untracked:  code/Lift5perPASbed.out
    Untracked:  code/LiftClustersFirst.err
    Untracked:  code/LiftClustersFirst.out
    Untracked:  code/LiftClustersFirst_remove.err
    Untracked:  code/LiftClustersFirst_remove.out
    Untracked:  code/LiftClustersSecond.err
    Untracked:  code/LiftClustersSecond.out
    Untracked:  code/LiftClustersSecond_remove.err
    Untracked:  code/LiftClustersSecond_remove.out
    Untracked:  code/LiftFinalChimpJunc2Human.sh
    Untracked:  code/LiftOrthoPAS2chimp.sh
    Untracked:  code/LiftorthoPAS.err
    Untracked:  code/LiftorthoPASt.out
    Untracked:  code/Log.out
    Untracked:  code/MapBadSamples.err
    Untracked:  code/MapBadSamples.out
    Untracked:  code/MapBadSamples.sh
    Untracked:  code/MapStats.err
    Untracked:  code/MapStats.out
    Untracked:  code/MaxEntCode/
    Untracked:  code/MergeClusters.err
    Untracked:  code/MergeClusters.out
    Untracked:  code/MergeClusters.sh
    Untracked:  code/MismatchNumbers.err
    Untracked:  code/MismatchNumbers.out
    Untracked:  code/MismatchNumbers.sh
    Untracked:  code/NuclearDTUTR.err
    Untracked:  code/NuclearDTUTRt.out
    Untracked:  code/NuclearPlotsDEandDiffDom_4.err
    Untracked:  code/NuclearPlotsDEandDiffDom_4.out
    Untracked:  code/NuclearPlotsDEandDiffDom_4.sh
    Untracked:  code/PAS_ATTAAA.err
    Untracked:  code/PAS_ATTAAA.out
    Untracked:  code/PAS_ATTAAA.sh
    Untracked:  code/PAS_ATTAAADF.err
    Untracked:  code/PAS_ATTAAADF.out
    Untracked:  code/PAS_ATTAAA_df.sh
    Untracked:  code/PAS_seqExpanded.sh
    Untracked:  code/PAS_sequence.err
    Untracked:  code/PAS_sequence.out
    Untracked:  code/PAS_sequenceDF.err
    Untracked:  code/PAS_sequenceDF.out
    Untracked:  code/PASexpanded_sequenceDF.err
    Untracked:  code/PASexpanded_sequenceDF.out
    Untracked:  code/PASsequences.sh
    Untracked:  code/PASsequences_DF.sh
    Untracked:  code/PlotNuclearUsagebySpecies.R
    Untracked:  code/PlotNuclearUsagebySpecies_DF.R
    Untracked:  code/PlotNuclearUsagebySpecies_DF_4DIC.R
    Untracked:  code/PlotNuclearUsagebySpecies_DF_DEout.R
    Untracked:  code/QuantMergeClusters
    Untracked:  code/QuantMergeClusters.err
    Untracked:  code/QuantMergeClusters.out
    Untracked:  code/QuantMergedClusters.sh
    Untracked:  code/RNATranscriptDTplot.err
    Untracked:  code/RNATranscriptDTplot.out
    Untracked:  code/RNATranscriptDTplot.sh
    Untracked:  code/RNAmotif_PAS.err
    Untracked:  code/RNAmotif_PAS.out
    Untracked:  code/RNAmotif_PAS.sh
    Untracked:  code/RNAmotif_PAS_chimp.err
    Untracked:  code/RNAmotif_PAS_chimp.out
    Untracked:  code/RNAmotif_PAS_chimp.sh
    Untracked:  code/Rev_liftoverPAShg19to38.err
    Untracked:  code/Rev_liftoverPAShg19to38.out
    Untracked:  code/ReverseLiftFilter.R
    Untracked:  code/RunFixCluster.err
    Untracked:  code/RunFixCluster.out
    Untracked:  code/RunFixLeafCluster.sh
    Untracked:  code/RunNegMCMediation.err
    Untracked:  code/RunNegMCMediation.sh
    Untracked:  code/RunNegMCMediationDF.err
    Untracked:  code/RunNegMCMediationDF.out
    Untracked:  code/RunNegMCMediationDF.sh
    Untracked:  code/RunNegMCMediationr.out
    Untracked:  code/RunNewDom.err
    Untracked:  code/RunNewDom.out
    Untracked:  code/RunPosMCMediation.err
    Untracked:  code/RunPosMCMediation.sh
    Untracked:  code/RunPosMCMediationDF.err
    Untracked:  code/RunPosMCMediationDF.out
    Untracked:  code/RunPosMCMediationDF.sh
    Untracked:  code/RunPosMCMediationr.out
    Untracked:  code/SAF215upbed_gen.py
    Untracked:  code/SAF2Bed.py
    Untracked:  code/Snakefile
    Untracked:  code/SnakefilePAS
    Untracked:  code/SnakefilePASfilt
    Untracked:  code/SortIndexBadSamples.err
    Untracked:  code/SortIndexBadSamples.out
    Untracked:  code/SortIndexBadSamples.sh
    Untracked:  code/StarMM2.err
    Untracked:  code/StarMM2.out
    Untracked:  code/StarMM2.sh
    Untracked:  code/TestFC.err
    Untracked:  code/TestFC.out
    Untracked:  code/TestFC.sh
    Untracked:  code/TotalTranscriptDTplot.err
    Untracked:  code/TotalTranscriptDTplot.out
    Untracked:  code/UTR2FASTA.py
    Untracked:  code/Upstream10Bases_general.py
    Untracked:  code/allPASSeq_df.sh
    Untracked:  code/apaQTLsnake.err
    Untracked:  code/apaQTLsnake.out
    Untracked:  code/apaQTLsnakePAS.err
    Untracked:  code/apaQTLsnakePAS.out
    Untracked:  code/apaQTLsnakePAShuman.err
    Untracked:  code/apaQTLsnakefiltPAS.err
    Untracked:  code/apaQTLsnakefiltPAS.out
    Untracked:  code/assignPeak2Intronicregion.err
    Untracked:  code/assignPeak2Intronicregion.out
    Untracked:  code/assignPeak2Intronicregion.sh
    Untracked:  code/bam2junc.err
    Untracked:  code/bam2junc.out
    Untracked:  code/bam2junc_remove.err
    Untracked:  code/bam2junc_remove.out
    Untracked:  code/bed215upbed.py
    Untracked:  code/bed2Bedbothstrand.py
    Untracked:  code/bed2SAF_gen.py
    Untracked:  code/bed2saf.py
    Untracked:  code/bg_to_cov.py
    Untracked:  code/buildIndecpantro5
    Untracked:  code/buildIndecpantro5.sh
    Untracked:  code/buildLeafviz.err
    Untracked:  code/buildLeafviz.out
    Untracked:  code/buildLeafviz.sh
    Untracked:  code/buildLeafviz_leadAnno.sh
    Untracked:  code/buildLeafviz_leafanno.err
    Untracked:  code/buildLeafviz_leafanno.out
    Untracked:  code/buildStarIndex.sh
    Untracked:  code/callPeaksYL.py
    Untracked:  code/chimpChromprder.sh
    Untracked:  code/chimpMultiCov.err
    Untracked:  code/chimpMultiCov.out
    Untracked:  code/chimpMultiCov.sh
    Untracked:  code/chimpMultiCov255.sh
    Untracked:  code/chimpMultiCovInclusive.err
    Untracked:  code/chimpMultiCovInclusive.out
    Untracked:  code/chimpMultiCovInclusive.sh
    Untracked:  code/chooseAnno2Bed.py
    Untracked:  code/chooseAnno2SAF.py
    Untracked:  code/chooseSignalSite.py
    Untracked:  code/chromOrder.err
    Untracked:  code/chromOrder.out
    Untracked:  code/classifyLeafviz.err
    Untracked:  code/classifyLeafviz.out
    Untracked:  code/cleanbed2saf.py
    Untracked:  code/cluster.json
    Untracked:  code/cluster2bed.py
    Untracked:  code/clusterLiftReverse.sh
    Untracked:  code/clusterLiftReverse_removebad.sh
    Untracked:  code/clusterLiftprimary.sh
    Untracked:  code/clusterLiftprimary_removebad.sh
    Untracked:  code/clusterPAS.json
    Untracked:  code/clusterfiltPAS.json
    Untracked:  code/comands2Mege.sh
    Untracked:  code/converBam2Junc.sh
    Untracked:  code/converBam2Junc_removeBad.sh
    Untracked:  code/convertNumeric.py
    Untracked:  code/extraSnakefiltpas
    Untracked:  code/extractPhaastConGeneral.py
    Untracked:  code/extractPhyloReg.py
    Untracked:  code/extractPhyloRegGene.py
    Untracked:  code/extractPhylopGeneral.py
    Untracked:  code/extractPhylopReg200down.py
    Untracked:  code/extractPhylopReg200up.py
    Untracked:  code/filter5perc.R
    Untracked:  code/filter5percPAS.py
    Untracked:  code/filter5percPheno.py
    Untracked:  code/filterBamforMP.pysam2_gen.py
    Untracked:  code/filterJuncChroms.err
    Untracked:  code/filterJuncChroms.out
    Untracked:  code/filterMissprimingInNuc10_gen.py
    Untracked:  code/filterNumChroms.py
    Untracked:  code/filterPASforMP.py
    Untracked:  code/filterPostLift.py
    Untracked:  code/filterPrimaryread.py
    Untracked:  code/filterSAFforMP_gen.py
    Untracked:  code/filterSecondaryread.py
    Untracked:  code/filterSortBedbyCleanedBed_gen.R
    Untracked:  code/filterpeaks.py
    Untracked:  code/fixExonFC.py
    Untracked:  code/fixFChead.py
    Untracked:  code/fixFChead_bothfrac.py
    Untracked:  code/fixFCheadforExp.py
    Untracked:  code/fixLeafCluster.py
    Untracked:  code/fixLiftedJunc.py
    Untracked:  code/fixUTRexonanno.py
    Untracked:  code/formathg38Anno.py
    Untracked:  code/generateStarIndex.err
    Untracked:  code/generateStarIndex.out
    Untracked:  code/generateStarIndexHuman.err
    Untracked:  code/generateStarIndexHuman.out
    Untracked:  code/getAlloverlap.py
    Untracked:  code/getRNAseqMapStats.sh
    Untracked:  code/hg19MapStats.err
    Untracked:  code/hg19MapStats.out
    Untracked:  code/hg19MapStats.sh
    Untracked:  code/humanChromorder.sh
    Untracked:  code/humanFiles
    Untracked:  code/humanMultiCov.err
    Untracked:  code/humanMultiCov.out
    Untracked:  code/humanMultiCov.sh
    Untracked:  code/humanMultiCov255.err
    Untracked:  code/humanMultiCov255.out
    Untracked:  code/humanMultiCov255.sh
    Untracked:  code/humanMultiCovInclusive.err
    Untracked:  code/humanMultiCovInclusive.out
    Untracked:  code/humanMultiCov_inclusive.sh
    Untracked:  code/infoContentSimpson.py
    Untracked:  code/intersectAnno.err
    Untracked:  code/intersectAnno.out
    Untracked:  code/intersectAnnoExt.err
    Untracked:  code/intersectAnnoExt.out
    Untracked:  code/intersectLiftedPAS.sh
    Untracked:  code/leafcutter_merge_regtools_redo.py
    Untracked:  code/liftJunctionFiles.sh
    Untracked:  code/liftPAS19to38.sh
    Untracked:  code/liftVCF.out
    Untracked:  code/liftVCF.sh
    Untracked:  code/liftoverPAShg19to38.err
    Untracked:  code/liftoverPAShg19to38.out
    Untracked:  code/lliftVCF.err
    Untracked:  code/log/
    Untracked:  code/make5percPeakbed.py
    Untracked:  code/makeDIC.err
    Untracked:  code/makeDIC.out
    Untracked:  code/makeFileID.py
    Untracked:  code/makeNuclearDapaplots.sh
    Untracked:  code/makeNuclearDapaplots_DF.sh
    Untracked:  code/makeNuclearPlots.err
    Untracked:  code/makeNuclearPlots.out
    Untracked:  code/makeNuclearPlotsDF.err
    Untracked:  code/makeNuclearPlotsDF.out
    Untracked:  code/makePheno.py
    Untracked:  code/makeSamplyGroupsChimp_TvN.py
    Untracked:  code/makeSamplyGroupsHuman_TvN.py
    Untracked:  code/makedICPlots_DF.sh
    Untracked:  code/mapRNAseqhg19.sh
    Untracked:  code/mapRNAseqhg19_newPipeline.sh
    Untracked:  code/maphg19.err
    Untracked:  code/maphg19.out
    Untracked:  code/maphg19.sh
    Untracked:  code/maphg19_new.err
    Untracked:  code/maphg19_new.out
    Untracked:  code/maphg19_sub.err
    Untracked:  code/maphg19_sub.out
    Untracked:  code/maphg19_subjunc.sh
    Untracked:  code/mediation_test.R
    Untracked:  code/merge.err
    Untracked:  code/mergeChimp3prime_inhg38.sh
    Untracked:  code/mergeChimpRNA.sh
    Untracked:  code/merge_leafcutter_clusters_redo.py
    Untracked:  code/mergeandBWRNAseq.sh
    Untracked:  code/mergeandsort_ChimpinHuman.err
    Untracked:  code/mergeandsort_ChimpinHuman.out
    Untracked:  code/mergeandsort_H3K9me3
    Untracked:  code/mergeandsort_h3k36me3
    Untracked:  code/mergeandsorth3k36me3.sh
    Untracked:  code/mergedBam2BW.sh
    Untracked:  code/mergedbam2bw.err
    Untracked:  code/mergedbam2bw.out
    Untracked:  code/mergedbamRNAand2bw.err
    Untracked:  code/mergedbamRNAand2bw.out
    Untracked:  code/nameClusters.py
    Untracked:  code/namePeaks.py
    Untracked:  code/negativeMediation_montecarlo.R
    Untracked:  code/negativeMediation_montecarloDF.R
    Untracked:  code/nuclearTranscriptDTplot.err
    Untracked:  code/nuclearTranscriptDTplot.out
    Untracked:  code/numMultimap.py
    Untracked:  code/overlapMMandOrthoexon.sh
    Untracked:  code/overlapPAS.err
    Untracked:  code/overlapPAS.out
    Untracked:  code/overlapPASandOrthoexon.sh
    Untracked:  code/overlapapaQTLPAS.sh
    Untracked:  code/overlapapaQTLPAS_extended.sh
    Untracked:  code/overlapapaQTLPAS_samples.sh
    Untracked:  code/parseHg38.py
    Untracked:  code/peak2PAS.py
    Untracked:  code/pheno2countonly.R
    Untracked:  code/postiveMediation_montecarlo.R
    Untracked:  code/postiveMediation_montecarlo_DF.R
    Untracked:  code/prepareAnnoLeafviz.err
    Untracked:  code/prepareAnnoLeafviz.out
    Untracked:  code/prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/prepareLeafvizAnno.sh
    Untracked:  code/preparePAS4lift.py
    Untracked:  code/prepare_phenotype_table.py
    Untracked:  code/primaryLift.err
    Untracked:  code/primaryLift.out
    Untracked:  code/primaryLift.sh
    Untracked:  code/processhg38exons.py
    Untracked:  code/quantJunc.sh
    Untracked:  code/quantJunc_TEST.sh
    Untracked:  code/quantJunc_removeBad.sh
    Untracked:  code/quantLiftedPAS.err
    Untracked:  code/quantLiftedPAS.out
    Untracked:  code/quantLiftedPAS.sh
    Untracked:  code/quantLiftedPASPrimary.err
    Untracked:  code/quantLiftedPASPrimary.out
    Untracked:  code/quantLiftedPASPrimary.sh
    Untracked:  code/quatJunc.err
    Untracked:  code/quatJunc.out
    Untracked:  code/recChimpback2Human.err
    Untracked:  code/recChimpback2Human.out
    Untracked:  code/recLiftchim2human.sh
    Untracked:  code/revLift.err
    Untracked:  code/revLift.out
    Untracked:  code/revLiftPAShg38to19.sh
    Untracked:  code/reverseLift.sh
    Untracked:  code/runCheckReverseLift.sh
    Untracked:  code/runChimpDiffIso.sh
    Untracked:  code/runChimpDiffIsoDF.sh
    Untracked:  code/runCountNucleotides.err
    Untracked:  code/runCountNucleotides.out
    Untracked:  code/runCountNucleotides.sh
    Untracked:  code/runCountNucleotidesPantro6.err
    Untracked:  code/runCountNucleotidesPantro6.out
    Untracked:  code/runCountNucleotides_pantro6.sh
    Untracked:  code/runFilterNumChroms.sh
    Untracked:  code/runHumanDiffIso.sh
    Untracked:  code/runHumanDiffIsoDF.sh
    Untracked:  code/runNewDom.sh
    Untracked:  code/runNuclearDiffIso_DF.sh
    Untracked:  code/runNuclearDifffIso.sh
    Untracked:  code/runTotalDiffIso.sh
    Untracked:  code/run_Chimpleafcutter_ds.err
    Untracked:  code/run_Chimpleafcutter_ds.out
    Untracked:  code/run_Chimpverifybam.err
    Untracked:  code/run_Chimpverifybam.out
    Untracked:  code/run_Humanleafcutter_dF.err
    Untracked:  code/run_Humanleafcutter_dF.out
    Untracked:  code/run_Humanleafcutter_ds.err
    Untracked:  code/run_Humanleafcutter_ds.out
    Untracked:  code/run_Nuclearleafcutter_ds.err
    Untracked:  code/run_Nuclearleafcutter_ds.out
    Untracked:  code/run_Nuclearleafcutter_dsDF.err
    Untracked:  code/run_Nuclearleafcutter_dsDF.out
    Untracked:  code/run_Totalleafcutter_ds.err
    Untracked:  code/run_Totalleafcutter_ds.out
    Untracked:  code/run_chimpverifybam.sh
    Untracked:  code/run_verifyBam.sh
    Untracked:  code/run_verifybam.err
    Untracked:  code/run_verifybam.out
    Untracked:  code/slurm-62824013.out
    Untracked:  code/slurm-62825841.out
    Untracked:  code/slurm-62826116.out
    Untracked:  code/slurm-64108209.out
    Untracked:  code/slurm-64108521.out
    Untracked:  code/slurm-64108557.out
    Untracked:  code/snakePASChimp.err
    Untracked:  code/snakePASChimp.out
    Untracked:  code/snakePAShuman.out
    Untracked:  code/snakemake.batch
    Untracked:  code/snakemakeChimp.err
    Untracked:  code/snakemakeChimp.out
    Untracked:  code/snakemakeHuman.err
    Untracked:  code/snakemakeHuman.out
    Untracked:  code/snakemakePAS.batch
    Untracked:  code/snakemakePASFiltChimp.err
    Untracked:  code/snakemakePASFiltChimp.out
    Untracked:  code/snakemakePASFiltHuman.err
    Untracked:  code/snakemakePASFiltHuman.out
    Untracked:  code/snakemakePAS_Human.batch
    Untracked:  code/snakemakePASchimp.batch
    Untracked:  code/snakemakePAShuman.batch
    Untracked:  code/snakemake_chimp.batch
    Untracked:  code/snakemake_human.batch
    Untracked:  code/snakemakefiltPAS.batch
    Untracked:  code/snakemakefiltPAS_chimp.sh
    Untracked:  code/snakemakefiltPAS_human.batch
    Untracked:  code/snakemakefiltPAS_human.sh
    Untracked:  code/spliceSite2Fasta.py
    Untracked:  code/submit-snakemake-chimp.sh
    Untracked:  code/submit-snakemake-human.sh
    Untracked:  code/submit-snakemakePAS-chimp.sh
    Untracked:  code/submit-snakemakePAS-human.sh
    Untracked:  code/submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/submit-snakemakefiltPAS-human.sh
    Untracked:  code/subset_diffisopheno.py
    Untracked:  code/subset_diffisopheno_Chimp_tvN.py
    Untracked:  code/subset_diffisopheno_Chimp_tvN_DF.py
    Untracked:  code/subset_diffisopheno_Huma_tvN.py
    Untracked:  code/subset_diffisopheno_Huma_tvN_DF.py
    Untracked:  code/subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/subset_diffisopheno_Nuclear_HvC_DF.py
    Untracked:  code/subset_diffisopheno_Total_HvC.py
    Untracked:  code/test
    Untracked:  code/test.txt
    Untracked:  code/threeprimeOrthoFC.out
    Untracked:  code/threeprimeOrthoFC.sh
    Untracked:  code/threeprimeOrthoFCcd.err
    Untracked:  code/transcriptDTplotsNuclear.sh
    Untracked:  code/transcriptDTplotsTotal.sh
    Untracked:  code/tripseq-analysis/
    Untracked:  code/verifyBam4973.sh
    Untracked:  code/verifyBam4973inHuman.sh
    Untracked:  code/verifybam4973.err
    Untracked:  code/verifybam4973.out
    Untracked:  code/verifybam4973HumanMap.err
    Untracked:  code/verifybam4973HumanMap.out
    Untracked:  code/wrap_Chimpverifybam.err
    Untracked:  code/wrap_Chimpverifybam.out
    Untracked:  code/wrap_chimpverifybam.sh
    Untracked:  code/wrap_verifyBam.sh
    Untracked:  code/wrap_verifybam.err
    Untracked:  code/wrap_verifybam.out
    Untracked:  code/writeMergecode.py
    Untracked:  data/._.DS_Store
    Untracked:  data/._HC_filenames.txt
    Untracked:  data/._HC_filenames.txt.sb-4426323c-IKIs0S
    Untracked:  data/._HC_filenames.xlsx
    Untracked:  data/._MapPantro6_meta.txt
    Untracked:  data/._MapPantro6_meta.txt.sb-a5794dd2-Cskmlm
    Untracked:  data/._MapPantro6_meta.xlsx
    Untracked:  data/._OppositeSpeciesMap.txt
    Untracked:  data/._OppositeSpeciesMap.txt.sb-a5794dd2-mayWJf
    Untracked:  data/._OppositeSpeciesMap.xlsx
    Untracked:  data/._RNASEQ_metadata.txt
    Untracked:  data/._RNASEQ_metadata.txt.sb-4426323c-TE4ns3
    Untracked:  data/._RNASEQ_metadata.txt.sb-51f67ae1-HXp7Gq
    Untracked:  data/._RNASEQ_metadata_2Removed.txt
    Untracked:  data/._RNASEQ_metadata_2Removed.txt.sb-4426323c-a4lBwx
    Untracked:  data/._RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/._RNASEQ_metadata_stranded.txt
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-a5794dd2-D659m2
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-a5794dd2-ImNMoY
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-e4bf31f0-ZGnGgl
    Untracked:  data/._RNASEQ_metadata_stranded.xlsx
    Untracked:  data/._TrialFiltersMeta.txt
    Untracked:  data/._TrialFiltersMeta.txt.sb-9845453e-R58Y0Q
    Untracked:  data/._metadata_HCpanel.txt
    Untracked:  data/._metadata_HCpanel.txt.sb-a3d92a2d-b9cYoF
    Untracked:  data/._metadata_HCpanel.txt.sb-a5794dd2-i594qs
    Untracked:  data/._metadata_HCpanel.txt.sb-f4823d1e-qihGek
    Untracked:  data/._metadata_HCpanel_frompantro5.xlsx
    Untracked:  data/._~$RNASEQ_metadata.xlsx
    Untracked:  data/._~$metadata_HCpanel.xlsx
    Untracked:  data/._.xlsx
    Untracked:  data/ALU/
    Untracked:  data/AREelements/
    Untracked:  data/BaseComp/
    Untracked:  data/CleanLiftedPeaks_FC_primary/
    Untracked:  data/CompapaQTLpas/
    Untracked:  data/DIC_Viz/
    Untracked:  data/DNDS/
    Untracked:  data/DTmatrix/
    Untracked:  data/DiffDomandDE_example/
    Untracked:  data/DiffExpression/
    Untracked:  data/DiffIso_Nuclear/
    Untracked:  data/DiffIso_Nuclear_DF/
    Untracked:  data/DiffIso_Total/
    Untracked:  data/DiffSplice/
    Untracked:  data/DiffSplice_liftedJunc/
    Untracked:  data/DiffSplice_removeBad/
    Untracked:  data/DifferentialPASUsageResults.txt
    Untracked:  data/DistTwoDom/
    Untracked:  data/DomDefGreaterX/
    Untracked:  data/DomStructure_4/
    Untracked:  data/DominantPAS/
    Untracked:  data/DominantPAS_DF/
    Untracked:  data/DoubleFilterUsageNumeric/
    Untracked:  data/EvalPantro5/
    Untracked:  data/H3K36me3/
    Untracked:  data/HC_filenames.txt
    Untracked:  data/HC_filenames.xlsx
    Untracked:  data/HumanMolPheno/
    Untracked:  data/HumanUTR/
    Untracked:  data/IndInfoContent/
    Untracked:  data/InfoContent/
    Untracked:  data/Khan_prot/
    Untracked:  data/Li_eqtls/
    Untracked:  data/MapPantro6_meta.txt
    Untracked:  data/MapPantro6_meta.xlsx
    Untracked:  data/MapStats/
    Untracked:  data/NormalizedClusters/
    Untracked:  data/NuclearHvC/
    Untracked:  data/NuclearHvC_DF/
    Untracked:  data/OppositeSpeciesMap.txt
    Untracked:  data/OppositeSpeciesMap.xlsx
    Untracked:  data/OrthoExonBed/
    Untracked:  data/OverlapBenchmark/
    Untracked:  data/OverlappingPAS/
    Untracked:  data/PAS/
    Untracked:  data/PAS_SAF/
    Untracked:  data/PAS_doubleFilter/
    Untracked:  data/PTM/
    Untracked:  data/Peaks_5perc/
    Untracked:  data/PhastCon/
    Untracked:  data/Pheno_5perc/
    Untracked:  data/Pheno_5perc_DF_nuclear/
    Untracked:  data/Pheno_5perc_nuclear/
    Untracked:  data/Pheno_5perc_nuclear_old/
    Untracked:  data/Pheno_5perc_total/
    Untracked:  data/PhyloP/
    Untracked:  data/Pol2Chip/
    Untracked:  data/QTLPASoverlap/
    Untracked:  data/RNASEQ_metadata.txt
    Untracked:  data/RNASEQ_metadata_2Removed.txt
    Untracked:  data/RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/RNASEQ_metadata_stranded.txt
    Untracked:  data/RNASEQ_metadata_stranded.txt.sb-e4bf31f0-ZGnGgl/
    Untracked:  data/RNASEQ_metadata_stranded.xlsx
    Untracked:  data/SignalSites/
    Untracked:  data/SignalSites_doublefilter/
    Untracked:  data/SpliceSite/
    Untracked:  data/TestAnnoBiasOE/
    Untracked:  data/TestMM2/
    Untracked:  data/TestMM2_AS/
    Untracked:  data/TestMM2_PrimaryRead/
    Untracked:  data/TestMM2_SeondaryRead/
    Untracked:  data/TestMM2_mismatch/
    Untracked:  data/TestMM2_quality/
    Untracked:  data/TestWithinMergePAS/
    Untracked:  data/Test_FC_methods/
    Untracked:  data/Threeprime2Ortho/
    Untracked:  data/TotalFractionPAS/
    Untracked:  data/TotalHvC/
    Untracked:  data/TrialFiltersMeta.txt
    Untracked:  data/TwoBadSampleAnalysis/
    Untracked:  data/UnliftedSites/
    Untracked:  data/UrichElements/
    Untracked:  data/Wang_ribo/
    Untracked:  data/apaQTLGenes/
    Untracked:  data/bioGRID/
    Untracked:  data/chainFiles/
    Untracked:  data/cleanPeaks_anno/
    Untracked:  data/cleanPeaks_byspecies/
    Untracked:  data/cleanPeaks_lifted/
    Untracked:  data/files4viz_nuclear/
    Untracked:  data/files4viz_nuclear_DF/
    Untracked:  data/gsea/
    Untracked:  data/gviz/
    Untracked:  data/hypoxia/
    Untracked:  data/leafviz/
    Untracked:  data/liftover_files/
    Untracked:  data/mediation/
    Untracked:  data/mediation_DF/
    Untracked:  data/metadata_HCpanel.txt
    Untracked:  data/metadata_HCpanel.xlsx
    Untracked:  data/metadata_HCpanel_extra.txt
    Untracked:  data/metadata_HCpanel_frompantro5.txt
    Untracked:  data/metadata_HCpanel_frompantro5.xlsx
    Untracked:  data/miRNA/
    Untracked:  data/multimap/
    Untracked:  data/orthoUTR/
    Untracked:  data/paiDecay/
    Untracked:  data/primaryLift/
    Untracked:  data/reverseLift/
    Untracked:  data/testQuant/
    Untracked:  data/utrDB/
    Untracked:  data/~$RNASEQ_metadata.xlsx
    Untracked:  data/~$metadata_HCpanel.xlsx
    Untracked:  data/.xlsx
    Untracked:  output/._.DS_Store
    Untracked:  output/AUcount_density.pdf
    Untracked:  output/AUcount_density_sm.pdf
    Untracked:  output/DEandAPA.txt
    Untracked:  output/DEandAPA_sig.txt
    Untracked:  output/DEandTEeffectsize
    Untracked:  output/DEandTEeffectsize.pdf
    Untracked:  output/DEeffectsize
    Untracked:  output/DEeffectsize.pdf
    Untracked:  output/FigureDF/
    Untracked:  output/FigurePresColors/
    Untracked:  output/PropSamesdom
    Untracked:  output/PropSamesdom.pdf
    Untracked:  output/SupplementaryTable/
    Untracked:  output/TEeffectsize
    Untracked:  output/TEeffectsize.pdf
    Untracked:  output/Total_DEeffectsize
    Untracked:  output/Total_DEeffectsize.pdf
    Untracked:  output/Total_DEeffectsizeNotJusttop.pdf
    Untracked:  output/Total_DErelationship.pdf
    Untracked:  output/Total_TEeffectsize
    Untracked:  output/Total_TEeffectsize.pdf
    Untracked:  output/Ubiqplot
    Untracked:  output/Ubiqplot.pdf
    Untracked:  output/dAPAandDomEnrich.png
    Untracked:  output/dEandDomEnrich.png
    Untracked:  output/dediffdom.pdf
    Untracked:  output/dpnotDE
    Untracked:  output/dtPlots/
    Untracked:  output/exandte
    Untracked:  output/fig1.pdf
    Untracked:  output/fig2.pdf
    Untracked:  output/fig3.pdf
    Untracked:  output/fig4.pdf
    Untracked:  output/fig5.pdf
    Untracked:  output/fig6.pdf
    Untracked:  output/piecharts
    Untracked:  output/piecharts.pdf
    Untracked:  output/removeUnilift_fig6 copy.pdf
    Untracked:  output/removeUnilift_fig6.pdf
    Untracked:  output/removeUnlift_fig3.pdf
    Untracked:  output/removeUnlift_fig4.pdf
    Untracked:  output/removeUnlift_fig5.pdf
    Untracked:  output/simpson.pdf
    Untracked:  output/simpson_alTogether.pdf
    Untracked:  output/supplement/
    Untracked:  output/whichSiteplot.pdf
    Untracked:  projectNotes.Rmd
    Untracked:  proteinModelSet.Rmd

Unstaged changes:
    Modified:   analysis/DeandNumPAS.Rmd
    Modified:   analysis/DirSelectionKhan.Rmd
    Modified:   analysis/ExploredAPA.Rmd
    Modified:   analysis/ExploredAPA_DF.Rmd
    Modified:   analysis/MMExpreiment.Rmd
    Modified:   analysis/OppositeMap.Rmd
    Modified:   analysis/PTM_analysis.Rmd
    Modified:   analysis/ResultsNoUnlifted.Rmd
    Modified:   analysis/TotalDomStructure.Rmd
    Modified:   analysis/TotalVNuclearBothSpecies.Rmd
    Modified:   analysis/TryTripSeqAnalysis.Rmd
    Modified:   analysis/annotationInfo.Rmd
    Modified:   analysis/changeMisprimcut.Rmd
    Modified:   analysis/comp2apaQTLPAS.Rmd
    Modified:   analysis/correlationPhenos.Rmd
    Modified:   analysis/dInforContent.Rmd
    Modified:   analysis/df_QC.Rmd
    Modified:   analysis/diffExpression.Rmd
    Modified:   analysis/establishCutoffs.Rmd
    Modified:   analysis/incorporateQTLsAncestral.Rmd
    Modified:   analysis/index.Rmd
    Modified:   analysis/infoContent.Rmd
    Modified:   analysis/investigatePantro5.Rmd
    Modified:   analysis/mRNADecay.Rmd
    Modified:   analysis/miRNAanalysis.Rmd
    Modified:   analysis/multiMap.Rmd
    Modified:   analysis/phastCon.Rmd
    Modified:   analysis/pol2.Rmd
    Modified:   analysis/speciesSpecific.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/mainFigures.Rmd</code>) and HTML (<code>docs/mainFigures.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/dc22f1753a71f616779ab7e2ec7c5d40787ae600/analysis/mainFigures.Rmd" target="_blank">dc22f17</a>
</td>
<td>
brimittleman
</td>
<td>
2020-07-25
</td>
<td>
edits for manuscrit updates 7.25
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/8cf811694aca24690632de4a92cef9efaa8078ec/docs/mainFigures.html" target="_blank">8cf8116</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/40aae53b39e4f647de4f36a48bd374bf0d2cc943/analysis/mainFigures.Rmd" target="_blank">40aae53</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-30
</td>
<td>
fix main fig typos
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/mainFigures.html" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/eeccce8f22ce887d8c0d8e881fa176e5d6a5caad/analysis/mainFigures.Rmd" target="_blank">eeccce8</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
<td>
remore last 3
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/6c4190ea07f3c0b3e307765308c601c2925c3ce4/docs/mainFigures.html" target="_blank">6c4190e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/faf89021589f8f045a073572e4cb44104e08d050/analysis/mainFigures.Rmd" target="_blank">faf8902</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-11
</td>
<td>
fix typos
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/a438fc1f8ea7690c6d9eddb2c435c2cf3f30c902/docs/mainFigures.html" target="_blank">a438fc1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/2c88fe0dd67f4763ae372c02162ddc16185fb0bb/analysis/mainFigures.Rmd" target="_blank">2c88fe0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
<td>
fig 5 and 6
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/mainFigures.html" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/033290cd8cbbc95cd609066fbddecfbfb4e7f3d7/analysis/mainFigures.Rmd" target="_blank">033290c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
<td>
add main figures 1-4
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>I would like to plot all of the main figures in one R markdown with cowplot. I will save the data frames in the other files as rds objects then load them here.</p>
<pre class="r"><code>library(cowplot)</code></pre>
<pre><code>
********************************************************</code></pre>
<pre><code>Note: As of version 1.0.0, cowplot does not change the</code></pre>
<pre><code>  default ggplot2 theme anymore. To recover the previous</code></pre>
<pre><code>  behavior, execute:
  theme_set(theme_cowplot())</code></pre>
<pre><code>********************************************************</code></pre>
<pre class="r"><code>library(tidyverse)</code></pre>
<pre><code>── Attaching packages ───────────────────────────────────────────────────────────────────────────── tidyverse 1.3.0 ──</code></pre>
<pre><code>✔ ggplot2 3.2.1     ✔ purrr   0.3.4
✔ tibble  2.1.3     ✔ dplyr   0.8.3
✔ tidyr   1.1.0     ✔ stringr 1.4.0
✔ readr   1.3.1     ✔ forcats 0.4.0</code></pre>
<pre><code>── Conflicts ──────────────────────────────────────────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()</code></pre>
<pre class="r"><code>library(ggpubr)</code></pre>
<pre><code>
Attaching package: &#39;ggpubr&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:cowplot&#39;:

    get_legend</code></pre>
<pre class="bash"><code>mkdir ../output/FigureDF</code></pre>
<div id="figure1" class="section level2">
<h2>Figure1</h2>
<p>1A- location pic charts</p>
<pre class="r"><code>fig1A_data=read.table(&quot;../output/FigureDF/nPAS_figure1.txt&quot;, header = T, stringsAsFactors = F) 

fig1A_data$species &lt;- factor(fig1A_data$species, labels = c(&quot;Chimpanzee~PAS&quot;, &quot;Human~PAS&quot;))

fig1A=ggplot(fig1A_data,aes(x=&quot;&quot;,y=prop, fill=loc)) + geom_bar(stat=&quot;identity&quot;,width=1, color=&quot;white&quot;)+  coord_polar(&quot;y&quot;, start=0) +theme_void() + facet_wrap(~species,labeller = label_parsed,strip.position = &quot;left&quot;) + scale_fill_brewer(palette = &quot;RdYlBu&quot;, name=&quot;Genic Location&quot;, labels=c(&quot;Coding&quot;, &quot;5KB downstream&quot;, &quot;Intronic&quot;,&quot;3&#39; UTR&quot;, &quot;5&#39; UTR&quot;)) + theme(legend.position = &quot;bottom&quot;,strip.text = element_text(size = 10, face = &quot;italic&quot;,angle=270), text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;)) +labs(title=&quot;PAS genic locations are conserved between species&quot;, x=&quot;&quot;, y=&quot;&quot;)


fig1A</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-3-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-3-1.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fig1B_data=read.table(&quot;../output/FigureDF/SeqConservation.txt&quot;,header = T,stringsAsFactors = F)
 
fig1B=ggplot(fig1B_data, aes(x=Set, by=Set, fill=region, y=PhyloP)) + geom_boxplot(notch = T) + scale_fill_brewer(palette = &quot;RdYlBu&quot;) + scale_x_discrete(labels=c(&quot;-600&quot;, &quot;-400&quot;, &quot;-200&quot;, &#39;0&#39;,&#39;200&#39;,&#39;400&#39;,&#39;600&#39;)) + labs(x=&quot;Basepairs&quot;, title=&quot;PAS are more conserved \nthan surrounding regions&quot;) + guides(fill = FALSE) + theme_classic()+ theme(plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;), axis.text.x = element_text(size=10),axis.text.y = element_text(size=10),text=element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))

fig1B</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-4-1">
Past versions of unnamed-chunk-4-1.png
</button>
</p>
<div id="fig-unnamed-chunk-4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-4-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-4-1.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Figure 1C:</p>
<pre class="r"><code>fig1c_data=read.table(&quot;../output/FigureDF/SignalSizeFigure.txt&quot;, header = T, stringsAsFactors = F)

fic1c=ggplot(fig1c_data, aes(y=propSS,by=Species,fill=Species,x=reorder(SS, -propSS))) + geom_bar(stat=&quot;identity&quot;, position = &quot;dodge&quot;) +  theme_classic() +theme(axis.text.x = element_text(size=10,angle = 90),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;),axis.text.y = element_text(size=10),text=element_text(size=10), legend.position = &quot;top&quot;,plot.margin = unit(c(0,0,0,0), &quot;cm&quot;)) + scale_fill_brewer(palette = &quot;Dark2&quot;, label=c(&quot;Chimp&quot;, &quot;Human&quot;))  + labs(y=&quot;Proportion of PAS&quot;,title=&quot;Proportion of PAS with a signal site&quot;, x=&quot;&quot;)</code></pre>
<p>figure 1 :</p>
<pre class="r"><code>row2=plot_grid(fig1B,fic1c, labels=c(&quot;b&quot;,&quot;c&quot;), scale = c(.9,.9), nrow = 1)

fig1all=plot_grid(fig1A,row2, labels = c(&quot;a&quot;, &quot;&quot;),nrow = 2,rel_heights=c(.75,1 ),hjust=-12)
fig1all</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-6-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-6-1.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pdf(&quot;../output/fig1.pdf&quot;, height=6, width=8)
fig1all
dev.off()</code></pre>
<pre><code>png 
  2 </code></pre>
</div>
<div id="figure-2" class="section level2">
<h2>Figure 2:</h2>
<p>Pie charts for differences</p>
<pre class="r"><code>PASlevel= read.table(&quot;../output/FigureDF/PASlevelDiff.txt&quot;, stringsAsFactors = F, header=T,col.names =c(&quot;sig&quot;, &quot;n&quot;,&quot;gene&quot;,&quot;prop&quot;)) %&gt;% mutate(set=&quot;PAS&quot;, test=c(&quot;Conserved&quot;, &quot;Not Tested&quot;, &quot;Divergent&quot;))
genelevel= read.table(&quot;../output/FigureDF/GenelevelDiff.txt&quot;, stringsAsFactors = F, header = T, col.names =c(&quot;sig&quot;, &quot;n&quot;,&quot;gene&quot;,&quot;prop&quot;)) %&gt;% mutate(set=&quot;gene&quot;, test=c(&quot;Conserved&quot;, &quot;Not Tested&quot;, &quot;Divergent&quot;))
isoformlevel= read.table(&quot;../output/FigureDF/IsoformlevelDiff.txt&quot;, stringsAsFactors = F, header = T, col.names =c(&quot;sig&quot;, &quot;n&quot;,&quot;gene&quot;,&quot;prop&quot;)) %&gt;% mutate(set=&quot;isoform&quot;,test=c(&quot;Conserved&quot;, &quot;Not Tested&quot;, &quot;Divergent&quot;))

fig2a=PASlevel %&gt;% bind_rows(genelevel) %&gt;% bind_rows(isoformlevel)
fig2a$set=factor(fig2a$set, levels=c(&quot;PAS&quot;, &quot;gene&quot;, &quot;isoform&quot;),labels = c(&quot;PAS~Differences&quot;, &quot;Genes~with~PAS~Differneces&quot;, &quot;Differences~isoform~diversity&quot;))
fig2a$test=factor(fig2a$test,levels= c(&quot;Conserved&quot;, &quot;Divergent&quot;,&quot;Not Tested&quot;))

useCOl &lt;- c(&quot;#d73027&quot;, &quot;#4575b4&quot;,&quot;#fee090&quot;)

fig2aplot=ggplot(fig2a,aes(by=test, y=prop, x=&quot;&quot;, fill=test)) + geom_bar(stat=&quot;identity&quot;,width=1, color=&quot;white&quot;)+  coord_polar(&quot;y&quot;, start=0) +theme_void() + scale_fill_manual(values=useCOl,name=&quot;&quot;) + facet_wrap(~set,labeller = label_parsed,strip.position = &quot;left&quot;) + theme(legend.position = &quot;bottom&quot;,strip.text = element_text(size = 10, angle=270), text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=12)) + labs(title=&quot;APA is functionally conserved at PAS and isoform diversity levels&quot;)</code></pre>
<p>Dominance plots</p>
<pre class="r"><code>fig2cd_data=read.table(&quot;../output/FigureDF/DominantPAS.txt&quot;, header = T, stringsAsFactors = F)
fig2cd_data$cut=as.factor(fig2cd_data$cut)

fig2d=ggplot(fig2cd_data,aes(x=cut,y=PropSame,fill=cut)) + geom_bar(stat=&quot;identity&quot;) +geom_text(aes(label=nDom), position=position_dodge(width=0.9), vjust=1,size=3)+ scale_fill_brewer(palette = &quot;RdYlBu&quot;)+theme_classic()+theme(legend.position = &quot;none&quot;,text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=12),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;)) + labs(title=&quot;Most genes with a dominant PAS\n share the same dominant PAS&quot;, y=&quot;Proportion of Genes in Set&quot;, x=&quot;Dominance Cutoff&quot;) 
fig2d</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-9-1">
Past versions of unnamed-chunk-9-1.png
</button>
</p>
<div id="fig-unnamed-chunk-9-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/8cf811694aca24690632de4a92cef9efaa8078ec/docs/figure/mainFigures.Rmd/unnamed-chunk-9-1.png" target="_blank">8cf8116</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-30
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-9-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/6c4190ea07f3c0b3e307765308c601c2925c3ce4/docs/figure/mainFigures.Rmd/unnamed-chunk-9-1.png" target="_blank">6c4190e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-9-1.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fig2c=ggplot(fig2cd_data,aes(x=cut,y=PropDom,fill=cut)) + geom_bar(stat=&quot;identity&quot;) +geom_text(aes(label=nDom), position=position_dodge(width=0.9), vjust=1, size=3) + scale_fill_brewer(palette = &quot;RdYlBu&quot;)+theme_classic()+theme(legend.position = &quot;none&quot;,text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=12),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;)) + labs(title=&quot;Proportion of tested Genes \nwith a dominant PAS&quot;, y=&quot;Proportion of Tested Genes&quot;, x=&quot;Dominance Cutoff&quot;)

fig2c</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-9-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-9-2">
Past versions of unnamed-chunk-9-2.png
</button>
</p>
<div id="fig-unnamed-chunk-9-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/8cf811694aca24690632de4a92cef9efaa8078ec/docs/figure/mainFigures.Rmd/unnamed-chunk-9-2.png" target="_blank">8cf8116</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-30
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-9-2.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-9-2.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fig2row2=plot_grid(fig2c,fig2d, labels=c(&quot;b&quot;,&quot;c&quot;), scale = c(.9,.9), nrow = 1)

fig2=plot_grid(fig2aplot,fig2row2, nrow = 2,labels = c(&quot;a&quot;,&quot;&quot;),rel_heights=c(.8,1 ),hjust=-5)

fig2</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-10-1">
Past versions of unnamed-chunk-10-1.png
</button>
</p>
<div id="fig-unnamed-chunk-10-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/8cf811694aca24690632de4a92cef9efaa8078ec/docs/figure/mainFigures.Rmd/unnamed-chunk-10-1.png" target="_blank">8cf8116</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-30
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-10-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/6c4190ea07f3c0b3e307765308c601c2925c3ce4/docs/figure/mainFigures.Rmd/unnamed-chunk-10-1.png" target="_blank">6c4190e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-10-1.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pdf(&quot;../output/fig2.pdf&quot;, height=6, width=8)
fig2
dev.off()</code></pre>
<pre><code>png 
  2 </code></pre>
</div>
<div id="figure-3" class="section level2">
<h2>figure 3</h2>
<pre class="r"><code>fig3_data=read.table(&quot;../output/DEandAPA.txt&quot;, header = T, stringsAsFactors = F)
fig3_data2=read.table(&quot;../output/DEandAPA_sig.txt&quot;, header = T, stringsAsFactors = F)</code></pre>
<pre class="r"><code>fig3a=ggplot(fig3_data,aes(y=deltaPAU, x=CorrectedlogFC)) + geom_point(alpha=.3) + geom_smooth(method=&quot;lm&quot;) + labs(title=&quot;APA v DE&quot;, x=&quot;DE log effect size&quot;, y=&quot;Difference in PAS Usage&quot;) + scale_color_brewer(palette = &quot;Set1&quot;,name=&quot;&quot;, labels=c(&quot;Intronic&quot;, &quot;3&#39; UTR&quot;))+ stat_cor(label.x = -8,label.y = -1) +theme_classic() + theme(text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=12),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))
fig3a</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-13-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-13-1">
Past versions of unnamed-chunk-13-1.png
</button>
</p>
<div id="fig-unnamed-chunk-13-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-13-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-13-1.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fig3b=ggplot(fig3_data,aes(y=deltaPAU, x=CorrectedlogFC, col=loc)) + geom_point(alpha=.3) + geom_smooth(aes(col=loc),method=&quot;lm&quot;) + labs(title=&quot;APA v DE&quot;, x=&quot;DE log effect size&quot;, y=&quot;Difference in PAS Usage&quot;) + scale_color_brewer(palette = &quot;Set1&quot;,name=&quot;&quot;, labels=c(&quot;Intronic&quot;, &quot;3&#39; UTR&quot;))+ stat_cor(aes(col=loc),label.x = -8,label.y = c(-1,1)) +theme_classic() + theme(legend.position = &quot;right&quot;,text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=12),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))
fig3b</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-13-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-13-2">
Past versions of unnamed-chunk-13-2.png
</button>
</p>
<div id="fig-unnamed-chunk-13-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-13-2.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-13-2.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fig3c=ggplot(fig3_data2,aes(y=deltaPAU, x=CorrectedlogFC)) + geom_point(alpha=.3) + geom_smooth(method=&quot;lm&quot;) + labs(title=&quot;Significant differences in \nAPA and DE&quot;, x=&quot;DE log effect size&quot;, y=&quot;Difference in PAS Usage&quot;) + scale_color_brewer(palette = &quot;Set1&quot;,name=&quot;&quot;, labels=c(&quot;Intronic&quot;, &quot;3&#39; UTR&quot;))+ stat_cor(label.x = -8,label.y = -1)+theme_classic()+ theme(legend.position = &quot;top&quot;,text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=12),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))
fig3c</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-13-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-13-3">
Past versions of unnamed-chunk-13-3.png
</button>
</p>
<div id="fig-unnamed-chunk-13-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-13-3.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-13-3.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fig3d=ggplot(fig3_data2,aes(y=deltaPAU, x=CorrectedlogFC, col=loc)) + geom_point(alpha=.3) + geom_smooth(aes(col=loc),method=&quot;lm&quot;) + labs(title=&quot;Significant differences in\n APA and DE&quot;, x=&quot;DE log effect size&quot;, y=&quot;Difference in PAS Usage&quot;) + scale_color_brewer(palette = &quot;Set1&quot;,name=&quot;&quot;, labels=c(&quot;Intronic&quot;, &quot;3&#39; UTR&quot;))+ stat_cor(aes(col=loc),label.x = -8,label.y = c(-1,1))+theme_classic()+ theme(legend.position = &quot;right&quot;,text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=12),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;) )

fig3d</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-13-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-13-4">
Past versions of unnamed-chunk-13-4.png
</button>
</p>
<div id="fig-unnamed-chunk-13-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-13-4.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-13-4.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fig3=plot_grid(fig3a, fig3b,fig3c,fig3d, scale = c(.9,.9,.9,.9), labels=c(&quot;a&quot;,&quot;b&quot;,&quot;c&quot;,&quot;d&quot;),rel_widths=c(1,1.3,1,1.3))

fig3</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-14-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-14-1">
Past versions of unnamed-chunk-14-1.png
</button>
</p>
<div id="fig-unnamed-chunk-14-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-14-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-14-1.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pdf(&quot;../output/fig3.pdf&quot;, height=8, width=8)
fig3
dev.off()</code></pre>
<pre><code>png 
  2 </code></pre>
</div>
<div id="figure-5" class="section level2">
<h2>figure 5</h2>
<pre class="r"><code>fig4Data=read.table(&quot;../output/FigureDF/DEandTEenrich.txt&quot;, header = T, stringsAsFactors = F)

fig4Data$Pval=as.numeric(fig4Data$Pval)
fig4Data$Enrichment=as.numeric(fig4Data$Enrichment)
fig4Data$Pheno=factor(fig4Data$Pheno, levels=c(&quot;Expression&quot;, &quot;Translation&quot;, &quot;Protein&quot;))</code></pre>
<pre class="r"><code>figure4a=ggplot(fig4Data,aes(x=sets,col=sets,y=Enrichment,label = round(Enrichment,3)))+ geom_bar(stat=&quot;identity&quot;,color=&quot;grey&quot;,aes(y=fig4Data$Enrichment),width=.01)+geom_point(size=10) + coord_flip() + geom_hline(yintercept = 1) + facet_grid(~Pheno)+scale_color_manual(values=useCOl) + labs( title=&quot;Genes with differences in APA are enriched\n in differentially expression and differentially translated genes&quot;,x=&quot;&quot;, y=&quot;Enrichment&quot;)+geom_text(color = &quot;black&quot;, size = 3) +scale_x_discrete(labels=c(Both=&quot;Both&quot;, OnlyAPA=&quot;PAS&quot;,OnlyIC= &quot;ID&quot;))+ theme_classic()+theme(legend.position = &quot;none&quot;, text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=12),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))
figure4a</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-17-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-17-1">
Past versions of unnamed-chunk-17-1.png
</button>
</p>
<div id="fig-unnamed-chunk-17-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-17-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/6c4190ea07f3c0b3e307765308c601c2925c3ce4/docs/figure/mainFigures.Rmd/unnamed-chunk-17-1.png" target="_blank">6c4190e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-17-1.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>figure4b=ggplot(fig4Data,aes(x=sets, y=-log10(Pval),fill=sets)) +geom_bar(stat = &quot;identity&quot;,position = &quot;dodge&quot;) +geom_hline(yintercept =1.3)+ labs(x=&quot;&quot;)+scale_x_discrete(labels=c(Both=&quot;Both&quot;, OnlyAPA=&quot;PAS&quot;,OnlyIC= &quot;ID&quot;))+ scale_fill_manual(values=useCOl,labels=c(&quot;Both&quot;, &quot;PAS Level&quot;, &quot;Isoform Diversity&quot;), name=&quot;&quot;)+facet_wrap(~Pheno)+ theme_classic()+ theme(legend.position = &quot;none&quot;,text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=12),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))

figure4b</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-17-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-17-2">
Past versions of unnamed-chunk-17-2.png
</button>
</p>
<div id="fig-unnamed-chunk-17-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-17-2.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a438fc1f8ea7690c6d9eddb2c435c2cf3f30c902/docs/figure/mainFigures.Rmd/unnamed-chunk-17-2.png" target="_blank">a438fc1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-17-2.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>row2fig4=plot_grid(NULL, figure4b,NULL, rel_widths=c(0.12,1,.1), nrow = 1 )
fig4=plot_grid(figure4a,row2fig4,nrow =2, scale = c(.9,.9),labels = c(&quot;a&quot;,&quot;b&quot;))
fig4</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-18-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-18-1">
Past versions of unnamed-chunk-18-1.png
</button>
</p>
<div id="fig-unnamed-chunk-18-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-18-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/6c4190ea07f3c0b3e307765308c601c2925c3ce4/docs/figure/mainFigures.Rmd/unnamed-chunk-18-1.png" target="_blank">6c4190e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a438fc1f8ea7690c6d9eddb2c435c2cf3f30c902/docs/figure/mainFigures.Rmd/unnamed-chunk-18-1.png" target="_blank">a438fc1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1c20472d43032f6e00b9bd4b1afd7970fe62e36/docs/figure/mainFigures.Rmd/unnamed-chunk-18-1.png" target="_blank">c1c2047</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pdf(&quot;../output/fig5.pdf&quot;, height=6, width=8,useKerning=F)
fig4
dev.off()</code></pre>
<pre><code>png 
  2 </code></pre>
</div>
<div id="figure-5-1" class="section level2">
<h2>figure 5</h2>
<pre class="r"><code>fig5data=read.table(&quot;../output/FigureDF/SameDiffDomEnrichinDE.txt&quot;,header = T, stringsAsFactors = F)
fig5data$set=as.factor(fig5data$set)

fig5a=ggplot(fig5data,aes(x=set,group=type,col=set,y=Enrich))+ geom_bar(stat=&quot;identity&quot;,col=&quot;grey&quot;,alpha=.3,width=.01)+geom_point(size=10) + coord_flip()+ geom_hline(yintercept = 1) +scale_color_brewer(palette=&quot;RdYlBu&quot;)+geom_text(col=&quot;black&quot;,aes(label = round(Enrich,2)))+ facet_grid(~type)+labs(x=&quot;Dominance Cutoff&quot;, y=&quot;Enrichment&quot;,title=&quot;Enrichment for differentially expressed genes in genes\n with the same and different dominant PAS&quot;) + theme_classic()+ theme(legend.position = &quot;none&quot;,  text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=12),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))


fig5a</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-20-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-20-1">
Past versions of unnamed-chunk-20-1.png
</button>
</p>
<div id="fig-unnamed-chunk-20-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-20-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a438fc1f8ea7690c6d9eddb2c435c2cf3f30c902/docs/figure/mainFigures.Rmd/unnamed-chunk-20-1.png" target="_blank">a438fc1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fig5b=ggplot(fig5data,aes(x=set, y=-log10(Pval),fill=set)) + geom_bar(stat=&quot;identity&quot;) +labs(title=&quot;&quot;,x=&quot;Dominance Cutoff&quot;)+ scale_fill_brewer(palette = &quot;RdYlBu&quot;) + theme(legend.position = &quot;none&quot;)+ geom_hline(yintercept = 1.30103) + facet_grid(~type)+ theme_classic()+ theme(legend.position = &quot;none&quot;,  text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=10),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))

fig5b</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-20-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-20-2">
Past versions of unnamed-chunk-20-2.png
</button>
</p>
<div id="fig-unnamed-chunk-20-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-20-2.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a438fc1f8ea7690c6d9eddb2c435c2cf3f30c902/docs/figure/mainFigures.Rmd/unnamed-chunk-20-2.png" target="_blank">a438fc1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fig5=plot_grid(fig5a,fig5b,nrow = 2,scale = c(.9,.9),labels = c(&quot;a&quot;,&quot;b&quot;), rel_heights = c(1,.8))

fig5</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-21-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-21-1">
Past versions of unnamed-chunk-21-1.png
</button>
</p>
<div id="fig-unnamed-chunk-21-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-21-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a438fc1f8ea7690c6d9eddb2c435c2cf3f30c902/docs/figure/mainFigures.Rmd/unnamed-chunk-21-1.png" target="_blank">a438fc1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pdf(&quot;../output/fig4.pdf&quot;, height=7, width=7,useKerning=F)
fig5
dev.off()</code></pre>
<pre><code>png 
  2 </code></pre>
<div id="fig-6" class="section level3">
<h3>fig 6</h3>
<pre class="r"><code>fig6topdata=read.table(&quot;../output/FigureDF/DPnotDE.txt&quot;,header = T, stringsAsFactors = F)
fig6middata=read.table(&quot;../output/FigureDF/DPnotDE_trans.txt&quot;,header = T, stringsAsFactors = F)

fig6dapa=read.table(&quot;../output/FigureDF/PinteractiondAPA.txt&quot;,header = T, stringsAsFactors = F)
fig6both=read.table(&quot;../output/FigureDF/Pinteractionboth.txt&quot;,header = T, stringsAsFactors = F)
fib6ic=read.table(&quot;../output/FigureDF/PinteractiondIC.txt&quot;,header = T, stringsAsFactors = F)</code></pre>
<pre class="r"><code>fig6a=ggplot(fig6topdata,aes(x=Set, fill=Set, y=Number))+ geom_bar(stat=&quot;identity&quot;)+ scale_fill_manual(values=useCOl)+geom_text(aes(label=Number), position=position_dodge(width=0.9), vjust=2)+ labs(title=&quot;Number of genes differentially expressed\n in protein but not mRNA&quot;, y=&quot;Number of Genes&quot;,x=&quot;&quot;)+scale_x_discrete(labels=c(Both=&quot;Both&quot;, OnlyAPA=&quot;PAS&quot;,OnlydIC= &quot;ID&quot;))+ theme_classic()+theme(legend.position = &quot;none&quot;,  text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=10),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))


fig6a</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-24-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-24-1">
Past versions of unnamed-chunk-24-1.png
</button>
</p>
<div id="fig-unnamed-chunk-24-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-24-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a438fc1f8ea7690c6d9eddb2c435c2cf3f30c902/docs/figure/mainFigures.Rmd/unnamed-chunk-24-1.png" target="_blank">a438fc1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fig6b=ggplot(fig6topdata,aes(x=Set, fill=Set, y=Prop))+ geom_bar(stat=&quot;identity&quot;)+ scale_fill_manual(values=useCOl) + labs(title=&quot;Proportions of genes differentially expressed\n in protein but not mRNA&quot;, y=&quot;Proportion of Genes&quot;,x=&quot;&quot;)+geom_text(aes(label=round(Prop,3)), position=position_dodge(width=0.9), vjust=2) +scale_x_discrete(labels=c(Both=&quot;Both&quot;, OnlyAPA=&quot;PAS&quot;,OnlydIC= &quot;ID&quot;))+ theme_classic()+theme(legend.position = &quot;none&quot;,  text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=10),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))

fig6b</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-24-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-24-2">
Past versions of unnamed-chunk-24-2.png
</button>
</p>
<div id="fig-unnamed-chunk-24-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-24-2.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a438fc1f8ea7690c6d9eddb2c435c2cf3f30c902/docs/figure/mainFigures.Rmd/unnamed-chunk-24-2.png" target="_blank">a438fc1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fig6c= ggplot(fig6middata, aes(x=set,by=dTE, y=ndTE,fill=set, alpha=dTE)) +geom_bar(stat=&quot;identity&quot;, position = &quot;dodge&quot;) + labs(title=&quot;&quot;, y=&quot;Number of Genes&quot;,x=&quot;&quot;) + scale_fill_manual(values = useCOl ) + scale_alpha_manual(values=c(.6, 1),name=&quot;Differentially translated&quot;) + geom_text(aes(label=ndTE), position=position_dodge(width=0.9), vjust=1)+guides(fill = FALSE)+scale_x_discrete(labels=c(Both=&quot;Both&quot;, OnlyAPA=&quot;PAS&quot;,OnlyIC= &quot;ID&quot;))+theme_classic()+ theme(legend.position = &quot;top&quot;, legend.justification = &quot;center&quot;, text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=10),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;)) 
fig6c</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-25-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-25-1">
Past versions of unnamed-chunk-25-1.png
</button>
</p>
<div id="fig-unnamed-chunk-25-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-25-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a438fc1f8ea7690c6d9eddb2c435c2cf3f30c902/docs/figure/mainFigures.Rmd/unnamed-chunk-25-1.png" target="_blank">a438fc1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fig6d=ggplot(fig6middata, aes(x=set,by=dTE, y=ndTE,fill=set, alpha=dTE)) +geom_bar(stat=&quot;identity&quot;, position = &quot;fill&quot;) + labs(title=&quot;&quot;, y=&quot;Proportion of Genes&quot;,x=&quot;&quot;) + scale_fill_manual(values = useCOl ) + scale_alpha_manual(values=c(.6, 1),name=&quot;Differentially translated&quot;) + guides(fill = FALSE)+scale_x_discrete(labels=c(Both=&quot;Both&quot;, OnlyAPA=&quot;PAS&quot;,OnlyIC= &quot;ID&quot;))+ theme(legend.position = &quot;top&quot;,legend.justification = &quot;center&quot;, text=element_text(size=10),plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=10),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))
fig6d</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-25-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-25-2">
Past versions of unnamed-chunk-25-2.png
</button>
</p>
<div id="fig-unnamed-chunk-25-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-25-2.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a438fc1f8ea7690c6d9eddb2c435c2cf3f30c902/docs/figure/mainFigures.Rmd/unnamed-chunk-25-2.png" target="_blank">a438fc1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#fig6e=ggplot(fig6both,aes(x=dPnotDE, y=log10(NormInter),fill=dPnotDE)) + geom_boxplot(notch = T) + stat_compare_means( label.x = 1.25, label.y = 1.5) + scale_fill_manual(values = c(&quot;grey&quot;, &quot;#d73027&quot;))+ labs(x=&quot;Differentally expressed in protein \n not in mRNA&quot;, y=&quot;log10(Interaction)&quot;, title=&quot;&quot;) + theme_classic() + theme(axis.text.x=element_text(angle=90, hjust=0, size = 10), text= element_text(size=10), legend.position = &quot;none&quot;,plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=10),axis.text.y = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))
#fig6e

#fig6f=ggplot(fig6dapa,aes(x=dPnotDE, y=log10(NormInter),fill=dPnotDE)) + geom_boxplot(notch = T) + stat_compare_means( label.x = 1.25, label.y = 1.5) + scale_fill_manual(values = c(&quot;grey&quot;, &quot;#4575b4&quot;)) + labs(x=&quot;Differentally expressed in protein \n not in mRNA&quot;, y=&quot;log10(Interaction)&quot;, title=&quot;&quot;)  + theme_classic() + theme(axis.text.x=element_text(angle=90, hjust=0, size = 10), text= element_text(size=10), legend.position = &quot;none&quot;,plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=10),axis.text.y = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))
#fig6f


#fig6g=ggplot(fib6ic,aes(x=dPnotDE, y=log10(NormInter),fill=dPnotDE)) + geom_boxplot(notch = T) + stat_compare_means( label.x = 1.25, label.y = 1.5) + scale_fill_manual(values = c(&quot;grey&quot;, &quot;#fee090&quot;)) + labs(x=&quot;Differentally expressed in protein \n not in mRNA&quot;, y=&quot;log10(Interaction)&quot;, title=&quot;&quot;) + theme_classic() + theme(axis.text.x=element_text(angle=90, hjust=0, size = 10), text= element_text(size=10), legend.position = &quot;none&quot;,plot.title = element_text(hjust = 0.5, face=&quot;bold&quot;,size=10),axis.text.y = element_text(size=10),plot.margin = unit(c(0,0,0,0), &quot;cm&quot;))

#fig6g</code></pre>
<pre class="r"><code>fig6top=plot_grid(fig6a,fig6b, labels = c(&quot;a&quot;,&quot;b&quot;), scale = c(.9,.9),nrow = 1)
fig6mid=plot_grid(fig6c,fig6d, labels = c(&quot;c&quot;,&quot;d&quot;), scale = c(.9,.9),nrow = 1)
#fig6botom=plot_grid(fig6e,fig6f, fig6g, labels = c(&quot;e&quot;,&quot;f&quot;,&quot;g&quot;), scale = c(.9,.9,.9),nrow = 1)

fig6=plot_grid(fig6top,fig6mid,nrow = 2, rel_heights = c(1,1))
fig6</code></pre>
<p><img src="figure/mainFigures.Rmd/unnamed-chunk-27-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-27-1">
Past versions of unnamed-chunk-27-1.png
</button>
</p>
<div id="fig-unnamed-chunk-27-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f31a5a05402e620290d200c0f7d6278c78d8cc05/docs/figure/mainFigures.Rmd/unnamed-chunk-27-1.png" target="_blank">f31a5a0</a>
</td>
<td>
brimittleman
</td>
<td>
2020-06-26
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a438fc1f8ea7690c6d9eddb2c435c2cf3f30c902/docs/figure/mainFigures.Rmd/unnamed-chunk-27-1.png" target="_blank">a438fc1</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pdf(&quot;../output/fig6.pdf&quot;, height=8, width=8,useKerning=F)
fig6
dev.off()</code></pre>
<pre><code>png 
  2 </code></pre>
<p>Draft July 25 switch figure 4 and 5</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] ggpubr_0.3.0    forcats_0.4.0   stringr_1.4.0   dplyr_0.8.3    
 [5] purrr_0.3.4     readr_1.3.1     tidyr_1.1.0     tibble_2.1.3   
 [9] ggplot2_3.2.1   tidyverse_1.3.0 cowplot_1.0.0  

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.3         lubridate_1.7.4    lattice_0.20-38   
 [4] assertthat_0.2.1   rprojroot_1.3-2    digest_0.6.20     
 [7] plyr_1.8.4         R6_2.4.0           cellranger_1.1.0  
[10] backports_1.1.4    reprex_0.3.0       evaluate_0.14     
[13] httr_1.4.1         pillar_1.4.2       rlang_0.4.6       
[16] lazyeval_0.2.2     curl_3.3           readxl_1.3.1      
[19] data.table_1.12.8  rstudioapi_0.10    whisker_0.3-2     
[22] car_3.0-5          rmarkdown_1.13     labeling_0.3      
[25] foreign_0.8-71     munsell_0.5.0      broom_0.5.2       
[28] compiler_3.6.1     httpuv_1.5.1       modelr_0.1.8      
[31] xfun_0.8           pkgconfig_2.0.2    htmltools_0.3.6   
[34] tidyselect_1.1.0   workflowr_1.6.2    rio_0.5.16        
[37] crayon_1.3.4       dbplyr_1.4.2       withr_2.1.2       
[40] later_0.8.0        grid_3.6.1         nlme_3.1-140      
[43] jsonlite_1.6       gtable_0.3.0       lifecycle_0.1.0   
[46] DBI_1.1.0          git2r_0.26.1       magrittr_1.5      
[49] scales_1.1.0       zip_2.0.3          carData_3.0-2     
[52] cli_1.1.0          stringi_1.4.3      reshape2_1.4.3    
[55] farver_2.0.1       ggsignif_0.5.0     fs_1.3.1          
[58] promises_1.0.1     xml2_1.3.2         vctrs_0.3.0       
[61] generics_0.0.2     openxlsx_4.1.0.1   RColorBrewer_1.1-2
[64] tools_3.6.1        glue_1.3.1         hms_0.5.3         
[67] abind_1.4-5        yaml_2.2.0         colorspace_1.4-1  
[70] rstatix_0.5.0      rvest_0.3.5        knitr_1.23        
[73] haven_2.3.1       </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
