<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Briana Mittleman" />


<title>Dominant PAS and DE</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/journal.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Comparative_APA</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/brimittleman/Comparative_APA">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Dominant PAS and DE</h1>
<h4 class="author"><em>Briana Mittleman</em></h4>
<h4 class="date"><em>4/16/2020</em></h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-05-17
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Comparative_APA/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity itâ€™s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20190902code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20190902)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20190902code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20190902)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g.Â subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtree4b2ff4a03156b795718666f84cd9c761b952f752targetblank4b2ff4aa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/brimittleman/Comparative_APA/tree/4b2ff4a03156b795718666f84cd9c761b952f752" target="_blank">4b2ff4a</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtree4b2ff4a03156b795718666f84cd9c761b952f752targetblank4b2ff4aa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    code/chimp_log/
    Ignored:    code/human_log/
    Ignored:    data/.DS_Store
    Ignored:    data/TrialFiltersMeta.txt.sb-9845453e-R58Y0Q/
    Ignored:    data/mediation_prot/
    Ignored:    data/metadata_HCpanel.txt.sb-284518db-RGf0kd/
    Ignored:    data/metadata_HCpanel.txt.sb-a5794dd2-i594qs/
    Ignored:    output/.DS_Store

Untracked files:
    Untracked:  ._.DS_Store
    Untracked:  Chimp/
    Untracked:  Human/
    Untracked:  analysis/AREstabilityScores.Rmd
    Untracked:  analysis/AllLoc_effectSizeCor.Rmd
    Untracked:  analysis/CrossChimpThreePrime.Rmd
    Untracked:  analysis/DiffTransProtvsExpression.Rmd
    Untracked:  analysis/DiffUsedUTR.Rmd
    Untracked:  analysis/GvizPlots.Rmd
    Untracked:  analysis/HandC.TvN
    Untracked:  analysis/PhenotypeOverlap10.Rmd
    Untracked:  analysis/annotationBias.Rmd
    Untracked:  analysis/assessReadQual.Rmd
    Untracked:  analysis/diffExpressionPantro6.Rmd
    Untracked:  code/._AlignmentScores.sh
    Untracked:  code/._BothFCMM.sh
    Untracked:  code/._BothFCMMPrim.sh
    Untracked:  code/._BothFCnewOInclusive.sh
    Untracked:  code/._ChimpStarMM2.sh
    Untracked:  code/._ClassifyLeafviz.sh
    Untracked:  code/._ClosestorthoEx.sh
    Untracked:  code/._Config_chimp.yaml
    Untracked:  code/._Config_chimp_full.yaml
    Untracked:  code/._Config_human.yaml
    Untracked:  code/._ConvertJunc2Bed.sh
    Untracked:  code/._CountNucleotides.py
    Untracked:  code/._CrossMapChimpRNA.sh
    Untracked:  code/._CrossMapThreeprime.sh
    Untracked:  code/._DiffSplice.sh
    Untracked:  code/._DiffSplicePlots.sh
    Untracked:  code/._DiffSplicePlots_gencode.sh
    Untracked:  code/._DiffSplice_gencode.sh
    Untracked:  code/._DiffSplice_removebad.sh
    Untracked:  code/._Filter255MM.sh
    Untracked:  code/._FilterPrimSec.sh
    Untracked:  code/._FindIntronForDomPAS.sh
    Untracked:  code/._FindIntronForDomPAS_DF.sh
    Untracked:  code/._GetMAPQscore.py
    Untracked:  code/._GetSecondaryMap.py
    Untracked:  code/._Lift5perPAS.sh
    Untracked:  code/._LiftFinalChimpJunc2Human.sh
    Untracked:  code/._LiftOrthoPAS2chimp.sh
    Untracked:  code/._MapBadSamples.sh
    Untracked:  code/._MismatchNumbers.sh
    Untracked:  code/._PAS_ATTAAA.sh
    Untracked:  code/._PAS_ATTAAA_df.sh
    Untracked:  code/._PAS_seqExpanded.sh
    Untracked:  code/._PASsequences.sh
    Untracked:  code/._PASsequences_DF.sh
    Untracked:  code/._PlotNuclearUsagebySpecies.R
    Untracked:  code/._PlotNuclearUsagebySpecies_DF.R
    Untracked:  code/._QuantMergedClusters.sh
    Untracked:  code/._RNATranscriptDTplot.sh
    Untracked:  code/._ReverseLiftFilter.R
    Untracked:  code/._RunFixLeafCluster.sh
    Untracked:  code/._RunNegMCMediation.sh
    Untracked:  code/._RunNegMCMediationDF.sh
    Untracked:  code/._RunPosMCMediationDF.err
    Untracked:  code/._RunPosMCMediationDF.sh
    Untracked:  code/._SAF2Bed.py
    Untracked:  code/._Snakefile
    Untracked:  code/._SnakefilePAS
    Untracked:  code/._SnakefilePASfilt
    Untracked:  code/._SortIndexBadSamples.sh
    Untracked:  code/._StarMM2.sh
    Untracked:  code/._TestFC.sh
    Untracked:  code/._assignPeak2Intronicregion
    Untracked:  code/._assignPeak2Intronicregion.sh
    Untracked:  code/._bed215upbed.py
    Untracked:  code/._bed2Bedbothstrand.py
    Untracked:  code/._bed2SAF_gen.py
    Untracked:  code/._buildIndecpantro5
    Untracked:  code/._buildIndecpantro5.sh
    Untracked:  code/._buildLeafviz.sh
    Untracked:  code/._buildLeafviz_leadAnno.sh
    Untracked:  code/._buildStarIndex.sh
    Untracked:  code/._chimpChromprder.sh
    Untracked:  code/._chimpMultiCov.sh
    Untracked:  code/._chimpMultiCov255.sh
    Untracked:  code/._chimpMultiCovInclusive.sh
    Untracked:  code/._chooseSignalSite.py
    Untracked:  code/._cleanbed2saf.py
    Untracked:  code/._cluster.json
    Untracked:  code/._cluster2bed.py
    Untracked:  code/._clusterLiftReverse.sh
    Untracked:  code/._clusterLiftReverse_removebad.sh
    Untracked:  code/._clusterLiftprimary.sh
    Untracked:  code/._clusterLiftprimary_removebad.sh
    Untracked:  code/._converBam2Junc.sh
    Untracked:  code/._converBam2Junc_removeBad.sh
    Untracked:  code/._extraSnakefiltpas
    Untracked:  code/._extractPhyloReg.py
    Untracked:  code/._extractPhyloRegGene.py
    Untracked:  code/._extractPhylopGeneral.ph
    Untracked:  code/._extractPhylopGeneral.py
    Untracked:  code/._extractPhylopReg200down.py
    Untracked:  code/._extractPhylopReg200up.py
    Untracked:  code/._filter5percPAS.py
    Untracked:  code/._filterNumChroms.py
    Untracked:  code/._filterPASforMP.py
    Untracked:  code/._filterPostLift.py
    Untracked:  code/._filterPrimaryread.py
    Untracked:  code/._filterSecondaryread.py
    Untracked:  code/._fixExonFC.py
    Untracked:  code/._fixFCheadforExp.py
    Untracked:  code/._fixLeafCluster.py
    Untracked:  code/._fixLiftedJunc.py
    Untracked:  code/._fixUTRexonanno.py
    Untracked:  code/._formathg38Anno.py
    Untracked:  code/._formatpantro6Anno.py
    Untracked:  code/._getRNAseqMapStats.sh
    Untracked:  code/._hg19MapStats.sh
    Untracked:  code/._humanChromorder.sh
    Untracked:  code/._humanMultiCov.sh
    Untracked:  code/._humanMultiCov255.sh
    Untracked:  code/._humanMultiCov_inclusive.sh
    Untracked:  code/._intersectLiftedPAS.sh
    Untracked:  code/._liftJunctionFiles.sh
    Untracked:  code/._liftPAS19to38.sh
    Untracked:  code/._liftedchimpJunc2human.sh
    Untracked:  code/._makeNuclearDapaplots.sh
    Untracked:  code/._makeNuclearDapaplots_DF.sh
    Untracked:  code/._makeSamplyGroupsHuman_TvN.py
    Untracked:  code/._mapRNAseqhg19.sh
    Untracked:  code/._mapRNAseqhg19_newPipeline.sh
    Untracked:  code/._maphg19.sh
    Untracked:  code/._maphg19_subjunc.sh
    Untracked:  code/._mediation_test.R
    Untracked:  code/._mergeChimp3prime_inhg38.sh
    Untracked:  code/._mergeandBWRNAseq.sh
    Untracked:  code/._mergedBam2BW.sh
    Untracked:  code/._nameClusters.py
    Untracked:  code/._negativeMediation_montecarlo.R
    Untracked:  code/._negativeMediation_montecarloDF.R
    Untracked:  code/._numMultimap.py
    Untracked:  code/._overlapMMandOrthoexon.sh
    Untracked:  code/._overlapPASandOrthoexon.sh
    Untracked:  code/._overlapapaQTLPAS.sh
    Untracked:  code/._parseHg38.py
    Untracked:  code/._postiveMediation_montecarlo_DF.R
    Untracked:  code/._prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/._prepareLeafvizAnno.sh
    Untracked:  code/._preparePAS4lift.py
    Untracked:  code/._primaryLift.sh
    Untracked:  code/._processhg38exons.py
    Untracked:  code/._quantJunc.sh
    Untracked:  code/._quantJunc_TEST.sh
    Untracked:  code/._quantJunc_removeBad.sh
    Untracked:  code/._quantLiftedPASPrimary.sh
    Untracked:  code/._quantMerged_seperatly.sh
    Untracked:  code/._recLiftchim2human.sh
    Untracked:  code/._revLiftPAShg38to19.sh
    Untracked:  code/._reverseLift.sh
    Untracked:  code/._runCheckReverseLift.sh
    Untracked:  code/._runChimpDiffIso.sh
    Untracked:  code/._runCountNucleotides.sh
    Untracked:  code/._runFilterNumChroms.sh
    Untracked:  code/._runHumanDiffIso.sh
    Untracked:  code/._runNuclearDiffIso_DF.sh
    Untracked:  code/._runNuclearDifffIso.sh
    Untracked:  code/._runTotalDiffIso.sh
    Untracked:  code/._run_chimpverifybam.sh
    Untracked:  code/._run_verifyBam.sh
    Untracked:  code/._snakemake.batch
    Untracked:  code/._snakemakePAS.batch
    Untracked:  code/._snakemakePASchimp.batch
    Untracked:  code/._snakemakePAShuman.batch
    Untracked:  code/._snakemake_chimp.batch
    Untracked:  code/._snakemake_human.batch
    Untracked:  code/._snakemakefiltPAS.batch
    Untracked:  code/._snakemakefiltPAS_chimp
    Untracked:  code/._snakemakefiltPAS_chimp.sh
    Untracked:  code/._snakemakefiltPAS_human.sh
    Untracked:  code/._spliceSite2Fasta.py
    Untracked:  code/._submit-snakemake-chimp.sh
    Untracked:  code/._submit-snakemake-human.sh
    Untracked:  code/._submit-snakemakePAS-chimp.sh
    Untracked:  code/._submit-snakemakePAS-human.sh
    Untracked:  code/._submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/._submit-snakemakefiltPAS-human.sh
    Untracked:  code/._subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/._subset_diffisopheno_Nuclear_HvC_DF.py
    Untracked:  code/._subset_diffisopheno_Total_HvC.py
    Untracked:  code/._threeprimeOrthoFC.sh
    Untracked:  code/._transcriptDTplotsNuclear.sh
    Untracked:  code/._verifyBam4973.sh
    Untracked:  code/._verifyBam4973inHuman.sh
    Untracked:  code/._wrap_chimpverifybam.sh
    Untracked:  code/._wrap_verifyBam.sh
    Untracked:  code/._writeMergecode.py
    Untracked:  code/.snakemake/
    Untracked:  code/ALLPAS_sequenceDF.err
    Untracked:  code/ALLPAS_sequenceDF.out
    Untracked:  code/AlignmentScores.err
    Untracked:  code/AlignmentScores.out
    Untracked:  code/AlignmentScores.sh
    Untracked:  code/BothFCMM.err
    Untracked:  code/BothFCMM.out
    Untracked:  code/BothFCMM.sh
    Untracked:  code/BothFCMMPrim.err
    Untracked:  code/BothFCMMPrim.out
    Untracked:  code/BothFCMMPrim.sh
    Untracked:  code/BothFCnewOInclusive.sh
    Untracked:  code/BothFCnewOInclusive.sh.err
    Untracked:  code/BothFCnewOInclusive.sh.out
    Untracked:  code/ChimpStarMM2.err
    Untracked:  code/ChimpStarMM2.out
    Untracked:  code/ChimpStarMM2.sh
    Untracked:  code/ClassifyLeafviz.sh
    Untracked:  code/ClosestorthoEx.err
    Untracked:  code/ClosestorthoEx.out
    Untracked:  code/ClosestorthoEx.sh
    Untracked:  code/Config_chimp.yaml
    Untracked:  code/Config_chimp_full.yaml
    Untracked:  code/Config_human.yaml
    Untracked:  code/ConvertJunc2Bed.err
    Untracked:  code/ConvertJunc2Bed.out
    Untracked:  code/ConvertJunc2Bed.sh
    Untracked:  code/CountNucleotides.py
    Untracked:  code/CrossMapChimpRNA.sh
    Untracked:  code/CrossMapThreeprime.sh
    Untracked:  code/CrossmapChimp3prime.err
    Untracked:  code/CrossmapChimp3prime.out
    Untracked:  code/CrossmapChimpRNA.err
    Untracked:  code/CrossmapChimpRNA.out
    Untracked:  code/DTUTR.sh
    Untracked:  code/DiffDom_RNAmotif_4.err
    Untracked:  code/DiffDom_RNAmotif_4.out
    Untracked:  code/DiffDom_RNAmotif_4.sh
    Untracked:  code/DiffDom_RNAmotif_4_splitDE.err
    Untracked:  code/DiffDom_RNAmotif_4_splitDE.out
    Untracked:  code/DiffDom_RNAmotif_4_splitDE.sh
    Untracked:  code/DiffSplice.err
    Untracked:  code/DiffSplice.out
    Untracked:  code/DiffSplice.sh
    Untracked:  code/DiffSplicePlots.err
    Untracked:  code/DiffSplicePlots.out
    Untracked:  code/DiffSplicePlots.sh
    Untracked:  code/DiffSplicePlots_gencode.sh
    Untracked:  code/DiffSplice_gencode.sh
    Untracked:  code/DiffSplice_removebad.err
    Untracked:  code/DiffSplice_removebad.out
    Untracked:  code/DiffSplice_removebad.sh
    Untracked:  code/Filter255.err
    Untracked:  code/Filter255.out
    Untracked:  code/Filter255MM.sh
    Untracked:  code/FilterPrimSec.err
    Untracked:  code/FilterPrimSec.out
    Untracked:  code/FilterPrimSec.sh
    Untracked:  code/FilterReverseLift.err
    Untracked:  code/FilterReverseLift.out
    Untracked:  code/FindDomXCutoff.py
    Untracked:  code/FindIntronForDomPAS.err
    Untracked:  code/FindIntronForDomPAS.out
    Untracked:  code/FindIntronForDomPAS.sh
    Untracked:  code/FindIntronForDomPAS_DF.sh
    Untracked:  code/GencodeDiffSplice.err
    Untracked:  code/GencodeDiffSplice.out
    Untracked:  code/GetMAPQscore.py
    Untracked:  code/GetSecondaryMap.py
    Untracked:  code/GetTopminus2Usage.py
    Untracked:  code/H3K36me3DTplot.err
    Untracked:  code/H3K36me3DTplot.out
    Untracked:  code/H3K36me3DTplot.sh
    Untracked:  code/H3K36me3DTplot_DiffIso.err
    Untracked:  code/H3K36me3DTplot_DiffIso.out
    Untracked:  code/H3K36me3DTplot_DiffIso.sh
    Untracked:  code/H3K36me3DTplot_Specific.err
    Untracked:  code/H3K36me3DTplot_Specific.out
    Untracked:  code/H3K36me3DTplot_Specific.sh
    Untracked:  code/H3K36me3DTplot_distalPAS.err
    Untracked:  code/H3K36me3DTplot_distalPAS.out
    Untracked:  code/H3K36me3DTplot_distalPAS.sh
    Untracked:  code/H3K36me3DTplot_transcript.err
    Untracked:  code/H3K36me3DTplot_transcript.out
    Untracked:  code/H3K36me3DTplot_transcript.sh
    Untracked:  code/H3K36me3DTplotwide.err
    Untracked:  code/H3K36me3DTplotwide.out
    Untracked:  code/H3K36me3DTplotwide.sh
    Untracked:  code/H3K9me3DTplot_transcript.err
    Untracked:  code/H3K9me3DTplot_transcript.out
    Untracked:  code/H3K9me3DTplot_transcript.sh
    Untracked:  code/H3K9me3_processandDT.sh
    Untracked:  code/HchromOrder.err
    Untracked:  code/HchromOrder.out
    Untracked:  code/InfoContentShannon.py
    Untracked:  code/InfoContentbyInd.py
    Untracked:  code/IntersectMMandOrtho.err
    Untracked:  code/IntersectMMandOrtho.out
    Untracked:  code/IntersectPASandOrtho.err
    Untracked:  code/IntersectPASandOrtho.out
    Untracked:  code/JunctionLift.err
    Untracked:  code/JunctionLift.out
    Untracked:  code/JunctionLiftFinalChimp.err
    Untracked:  code/JunctionLiftFinalChimp.out
    Untracked:  code/Lift5perPAS.sh
    Untracked:  code/Lift5perPASbed.err
    Untracked:  code/Lift5perPASbed.out
    Untracked:  code/LiftClustersFirst.err
    Untracked:  code/LiftClustersFirst.out
    Untracked:  code/LiftClustersFirst_remove.err
    Untracked:  code/LiftClustersFirst_remove.out
    Untracked:  code/LiftClustersSecond.err
    Untracked:  code/LiftClustersSecond.out
    Untracked:  code/LiftClustersSecond_remove.err
    Untracked:  code/LiftClustersSecond_remove.out
    Untracked:  code/LiftFinalChimpJunc2Human.sh
    Untracked:  code/LiftOrthoPAS2chimp.sh
    Untracked:  code/LiftorthoPAS.err
    Untracked:  code/LiftorthoPASt.out
    Untracked:  code/Log.out
    Untracked:  code/MapBadSamples.err
    Untracked:  code/MapBadSamples.out
    Untracked:  code/MapBadSamples.sh
    Untracked:  code/MapStats.err
    Untracked:  code/MapStats.out
    Untracked:  code/MaxEntCode/
    Untracked:  code/MergeClusters.err
    Untracked:  code/MergeClusters.out
    Untracked:  code/MergeClusters.sh
    Untracked:  code/MismatchNumbers.err
    Untracked:  code/MismatchNumbers.out
    Untracked:  code/MismatchNumbers.sh
    Untracked:  code/NuclearDTUTR.err
    Untracked:  code/NuclearDTUTRt.out
    Untracked:  code/NuclearPlotsDEandDiffDom_4.err
    Untracked:  code/NuclearPlotsDEandDiffDom_4.out
    Untracked:  code/NuclearPlotsDEandDiffDom_4.sh
    Untracked:  code/PAS_ATTAAA.err
    Untracked:  code/PAS_ATTAAA.out
    Untracked:  code/PAS_ATTAAA.sh
    Untracked:  code/PAS_ATTAAADF.err
    Untracked:  code/PAS_ATTAAADF.out
    Untracked:  code/PAS_ATTAAA_df.sh
    Untracked:  code/PAS_seqExpanded.sh
    Untracked:  code/PAS_sequence.err
    Untracked:  code/PAS_sequence.out
    Untracked:  code/PAS_sequenceDF.err
    Untracked:  code/PAS_sequenceDF.out
    Untracked:  code/PASexpanded_sequenceDF.err
    Untracked:  code/PASexpanded_sequenceDF.out
    Untracked:  code/PASsequences.sh
    Untracked:  code/PASsequences_DF.sh
    Untracked:  code/PlotNuclearUsagebySpecies.R
    Untracked:  code/PlotNuclearUsagebySpecies_DF.R
    Untracked:  code/PlotNuclearUsagebySpecies_DF_4DIC.R
    Untracked:  code/PlotNuclearUsagebySpecies_DF_DEout.R
    Untracked:  code/QuantMergeClusters
    Untracked:  code/QuantMergeClusters.err
    Untracked:  code/QuantMergeClusters.out
    Untracked:  code/QuantMergedClusters.sh
    Untracked:  code/RNATranscriptDTplot.err
    Untracked:  code/RNATranscriptDTplot.out
    Untracked:  code/RNATranscriptDTplot.sh
    Untracked:  code/RNAmotif_PAS.err
    Untracked:  code/RNAmotif_PAS.out
    Untracked:  code/RNAmotif_PAS.sh
    Untracked:  code/RNAmotif_PAS_chimp.err
    Untracked:  code/RNAmotif_PAS_chimp.out
    Untracked:  code/RNAmotif_PAS_chimp.sh
    Untracked:  code/Rev_liftoverPAShg19to38.err
    Untracked:  code/Rev_liftoverPAShg19to38.out
    Untracked:  code/ReverseLiftFilter.R
    Untracked:  code/RunFixCluster.err
    Untracked:  code/RunFixCluster.out
    Untracked:  code/RunFixLeafCluster.sh
    Untracked:  code/RunNegMCMediation.err
    Untracked:  code/RunNegMCMediation.sh
    Untracked:  code/RunNegMCMediationDF.err
    Untracked:  code/RunNegMCMediationDF.out
    Untracked:  code/RunNegMCMediationDF.sh
    Untracked:  code/RunNegMCMediationr.out
    Untracked:  code/RunNewDom.err
    Untracked:  code/RunNewDom.out
    Untracked:  code/RunPosMCMediation.err
    Untracked:  code/RunPosMCMediation.sh
    Untracked:  code/RunPosMCMediationDF.err
    Untracked:  code/RunPosMCMediationDF.out
    Untracked:  code/RunPosMCMediationDF.sh
    Untracked:  code/RunPosMCMediationr.out
    Untracked:  code/SAF215upbed_gen.py
    Untracked:  code/SAF2Bed.py
    Untracked:  code/Snakefile
    Untracked:  code/SnakefilePAS
    Untracked:  code/SnakefilePASfilt
    Untracked:  code/SortIndexBadSamples.err
    Untracked:  code/SortIndexBadSamples.out
    Untracked:  code/SortIndexBadSamples.sh
    Untracked:  code/StarMM2.err
    Untracked:  code/StarMM2.out
    Untracked:  code/StarMM2.sh
    Untracked:  code/TestFC.err
    Untracked:  code/TestFC.out
    Untracked:  code/TestFC.sh
    Untracked:  code/TotalTranscriptDTplot.err
    Untracked:  code/TotalTranscriptDTplot.out
    Untracked:  code/UTR2FASTA.py
    Untracked:  code/Upstream10Bases_general.py
    Untracked:  code/allPASSeq_df.sh
    Untracked:  code/apaQTLsnake.err
    Untracked:  code/apaQTLsnake.out
    Untracked:  code/apaQTLsnakePAS.err
    Untracked:  code/apaQTLsnakePAS.out
    Untracked:  code/apaQTLsnakePAShuman.err
    Untracked:  code/apaQTLsnakefiltPAS.err
    Untracked:  code/apaQTLsnakefiltPAS.out
    Untracked:  code/assignPeak2Intronicregion.err
    Untracked:  code/assignPeak2Intronicregion.out
    Untracked:  code/assignPeak2Intronicregion.sh
    Untracked:  code/bam2junc.err
    Untracked:  code/bam2junc.out
    Untracked:  code/bam2junc_remove.err
    Untracked:  code/bam2junc_remove.out
    Untracked:  code/bed215upbed.py
    Untracked:  code/bed2Bedbothstrand.py
    Untracked:  code/bed2SAF_gen.py
    Untracked:  code/bed2saf.py
    Untracked:  code/bg_to_cov.py
    Untracked:  code/buildIndecpantro5
    Untracked:  code/buildIndecpantro5.sh
    Untracked:  code/buildLeafviz.err
    Untracked:  code/buildLeafviz.out
    Untracked:  code/buildLeafviz.sh
    Untracked:  code/buildLeafviz_leadAnno.sh
    Untracked:  code/buildLeafviz_leafanno.err
    Untracked:  code/buildLeafviz_leafanno.out
    Untracked:  code/buildStarIndex.sh
    Untracked:  code/callPeaksYL.py
    Untracked:  code/chimpChromprder.sh
    Untracked:  code/chimpMultiCov.err
    Untracked:  code/chimpMultiCov.out
    Untracked:  code/chimpMultiCov.sh
    Untracked:  code/chimpMultiCov255.sh
    Untracked:  code/chimpMultiCovInclusive.err
    Untracked:  code/chimpMultiCovInclusive.out
    Untracked:  code/chimpMultiCovInclusive.sh
    Untracked:  code/chooseAnno2Bed.py
    Untracked:  code/chooseAnno2SAF.py
    Untracked:  code/chooseSignalSite.py
    Untracked:  code/chromOrder.err
    Untracked:  code/chromOrder.out
    Untracked:  code/classifyLeafviz.err
    Untracked:  code/classifyLeafviz.out
    Untracked:  code/cleanbed2saf.py
    Untracked:  code/cluster.json
    Untracked:  code/cluster2bed.py
    Untracked:  code/clusterLiftReverse.sh
    Untracked:  code/clusterLiftReverse_removebad.sh
    Untracked:  code/clusterLiftprimary.sh
    Untracked:  code/clusterLiftprimary_removebad.sh
    Untracked:  code/clusterPAS.json
    Untracked:  code/clusterfiltPAS.json
    Untracked:  code/comands2Mege.sh
    Untracked:  code/converBam2Junc.sh
    Untracked:  code/converBam2Junc_removeBad.sh
    Untracked:  code/convertNumeric.py
    Untracked:  code/environment.yaml
    Untracked:  code/extraSnakefiltpas
    Untracked:  code/extractPhaastConGeneral.py
    Untracked:  code/extractPhyloReg.py
    Untracked:  code/extractPhyloRegGene.py
    Untracked:  code/extractPhylopGeneral.py
    Untracked:  code/extractPhylopReg200down.py
    Untracked:  code/extractPhylopReg200up.py
    Untracked:  code/filter5perc.R
    Untracked:  code/filter5percPAS.py
    Untracked:  code/filter5percPheno.py
    Untracked:  code/filterBamforMP.pysam2_gen.py
    Untracked:  code/filterJuncChroms.err
    Untracked:  code/filterJuncChroms.out
    Untracked:  code/filterMissprimingInNuc10_gen.py
    Untracked:  code/filterNumChroms.py
    Untracked:  code/filterPASforMP.py
    Untracked:  code/filterPostLift.py
    Untracked:  code/filterPrimaryread.py
    Untracked:  code/filterSAFforMP_gen.py
    Untracked:  code/filterSecondaryread.py
    Untracked:  code/filterSortBedbyCleanedBed_gen.R
    Untracked:  code/filterpeaks.py
    Untracked:  code/fixExonFC.py
    Untracked:  code/fixFChead.py
    Untracked:  code/fixFChead_bothfrac.py
    Untracked:  code/fixFCheadforExp.py
    Untracked:  code/fixLeafCluster.py
    Untracked:  code/fixLiftedJunc.py
    Untracked:  code/fixUTRexonanno.py
    Untracked:  code/formathg38Anno.py
    Untracked:  code/generateStarIndex.err
    Untracked:  code/generateStarIndex.out
    Untracked:  code/generateStarIndexHuman.err
    Untracked:  code/generateStarIndexHuman.out
    Untracked:  code/getAlloverlap.py
    Untracked:  code/getRNAseqMapStats.sh
    Untracked:  code/hg19MapStats.err
    Untracked:  code/hg19MapStats.out
    Untracked:  code/hg19MapStats.sh
    Untracked:  code/humanChromorder.sh
    Untracked:  code/humanFiles
    Untracked:  code/humanMultiCov.err
    Untracked:  code/humanMultiCov.out
    Untracked:  code/humanMultiCov.sh
    Untracked:  code/humanMultiCov255.err
    Untracked:  code/humanMultiCov255.out
    Untracked:  code/humanMultiCov255.sh
    Untracked:  code/humanMultiCovInclusive.err
    Untracked:  code/humanMultiCovInclusive.out
    Untracked:  code/humanMultiCov_inclusive.sh
    Untracked:  code/infoContentSimpson.py
    Untracked:  code/intersectAnno.err
    Untracked:  code/intersectAnno.out
    Untracked:  code/intersectAnnoExt.err
    Untracked:  code/intersectAnnoExt.out
    Untracked:  code/intersectLiftedPAS.sh
    Untracked:  code/leafcutter_merge_regtools_redo.py
    Untracked:  code/liftJunctionFiles.sh
    Untracked:  code/liftPAS19to38.sh
    Untracked:  code/liftoverPAShg19to38.err
    Untracked:  code/liftoverPAShg19to38.out
    Untracked:  code/log/
    Untracked:  code/make5percPeakbed.py
    Untracked:  code/makeDIC.err
    Untracked:  code/makeDIC.out
    Untracked:  code/makeFileID.py
    Untracked:  code/makeNuclearDapaplots.sh
    Untracked:  code/makeNuclearDapaplots_DF.sh
    Untracked:  code/makeNuclearPlots.err
    Untracked:  code/makeNuclearPlots.out
    Untracked:  code/makeNuclearPlotsDF.err
    Untracked:  code/makeNuclearPlotsDF.out
    Untracked:  code/makePheno.py
    Untracked:  code/makeSamplyGroupsChimp_TvN.py
    Untracked:  code/makeSamplyGroupsHuman_TvN.py
    Untracked:  code/makedICPlots_DF.sh
    Untracked:  code/mapRNAseqhg19.sh
    Untracked:  code/mapRNAseqhg19_newPipeline.sh
    Untracked:  code/maphg19.err
    Untracked:  code/maphg19.out
    Untracked:  code/maphg19.sh
    Untracked:  code/maphg19_new.err
    Untracked:  code/maphg19_new.out
    Untracked:  code/maphg19_sub.err
    Untracked:  code/maphg19_sub.out
    Untracked:  code/maphg19_subjunc.sh
    Untracked:  code/mediation_test.R
    Untracked:  code/merge.err
    Untracked:  code/mergeChimp3prime_inhg38.sh
    Untracked:  code/mergeChimpRNA.sh
    Untracked:  code/merge_leafcutter_clusters_redo.py
    Untracked:  code/mergeandBWRNAseq.sh
    Untracked:  code/mergeandsort_ChimpinHuman.err
    Untracked:  code/mergeandsort_ChimpinHuman.out
    Untracked:  code/mergeandsort_H3K9me3
    Untracked:  code/mergeandsort_h3k36me3
    Untracked:  code/mergeandsorth3k36me3.sh
    Untracked:  code/mergedBam2BW.sh
    Untracked:  code/mergedbam2bw.err
    Untracked:  code/mergedbam2bw.out
    Untracked:  code/mergedbamRNAand2bw.err
    Untracked:  code/mergedbamRNAand2bw.out
    Untracked:  code/nameClusters.py
    Untracked:  code/namePeaks.py
    Untracked:  code/negativeMediation_montecarlo.R
    Untracked:  code/negativeMediation_montecarloDF.R
    Untracked:  code/nuclearTranscriptDTplot.err
    Untracked:  code/nuclearTranscriptDTplot.out
    Untracked:  code/numMultimap.py
    Untracked:  code/overlapMMandOrthoexon.sh
    Untracked:  code/overlapPAS.err
    Untracked:  code/overlapPAS.out
    Untracked:  code/overlapPASandOrthoexon.sh
    Untracked:  code/overlapapaQTLPAS.sh
    Untracked:  code/overlapapaQTLPAS_extended.sh
    Untracked:  code/overlapapaQTLPAS_samples.sh
    Untracked:  code/parseHg38.py
    Untracked:  code/peak2PAS.py
    Untracked:  code/pheno2countonly.R
    Untracked:  code/postiveMediation_montecarlo.R
    Untracked:  code/postiveMediation_montecarlo_DF.R
    Untracked:  code/prepareAnnoLeafviz.err
    Untracked:  code/prepareAnnoLeafviz.out
    Untracked:  code/prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/prepareLeafvizAnno.sh
    Untracked:  code/preparePAS4lift.py
    Untracked:  code/prepare_phenotype_table.py
    Untracked:  code/primaryLift.err
    Untracked:  code/primaryLift.out
    Untracked:  code/primaryLift.sh
    Untracked:  code/processhg38exons.py
    Untracked:  code/quantJunc.sh
    Untracked:  code/quantJunc_TEST.sh
    Untracked:  code/quantJunc_removeBad.sh
    Untracked:  code/quantLiftedPAS.err
    Untracked:  code/quantLiftedPAS.out
    Untracked:  code/quantLiftedPAS.sh
    Untracked:  code/quantLiftedPASPrimary.err
    Untracked:  code/quantLiftedPASPrimary.out
    Untracked:  code/quantLiftedPASPrimary.sh
    Untracked:  code/quatJunc.err
    Untracked:  code/quatJunc.out
    Untracked:  code/recChimpback2Human.err
    Untracked:  code/recChimpback2Human.out
    Untracked:  code/recLiftchim2human.sh
    Untracked:  code/revLift.err
    Untracked:  code/revLift.out
    Untracked:  code/revLiftPAShg38to19.sh
    Untracked:  code/reverseLift.sh
    Untracked:  code/runCheckReverseLift.sh
    Untracked:  code/runChimpDiffIso.sh
    Untracked:  code/runChimpDiffIsoDF.sh
    Untracked:  code/runCountNucleotides.err
    Untracked:  code/runCountNucleotides.out
    Untracked:  code/runCountNucleotides.sh
    Untracked:  code/runCountNucleotidesPantro6.err
    Untracked:  code/runCountNucleotidesPantro6.out
    Untracked:  code/runCountNucleotides_pantro6.sh
    Untracked:  code/runFilterNumChroms.sh
    Untracked:  code/runHumanDiffIso.sh
    Untracked:  code/runHumanDiffIsoDF.sh
    Untracked:  code/runNewDom.sh
    Untracked:  code/runNuclearDiffIso_DF.sh
    Untracked:  code/runNuclearDifffIso.sh
    Untracked:  code/runTotalDiffIso.sh
    Untracked:  code/run_Chimpleafcutter_ds.err
    Untracked:  code/run_Chimpleafcutter_ds.out
    Untracked:  code/run_Chimpverifybam.err
    Untracked:  code/run_Chimpverifybam.out
    Untracked:  code/run_Humanleafcutter_dF.err
    Untracked:  code/run_Humanleafcutter_dF.out
    Untracked:  code/run_Humanleafcutter_ds.err
    Untracked:  code/run_Humanleafcutter_ds.out
    Untracked:  code/run_Nuclearleafcutter_ds.err
    Untracked:  code/run_Nuclearleafcutter_ds.out
    Untracked:  code/run_Nuclearleafcutter_dsDF.err
    Untracked:  code/run_Nuclearleafcutter_dsDF.out
    Untracked:  code/run_Totalleafcutter_ds.err
    Untracked:  code/run_Totalleafcutter_ds.out
    Untracked:  code/run_chimpverifybam.sh
    Untracked:  code/run_verifyBam.sh
    Untracked:  code/run_verifybam.err
    Untracked:  code/run_verifybam.out
    Untracked:  code/slurm-62824013.out
    Untracked:  code/slurm-62825841.out
    Untracked:  code/slurm-62826116.out
    Untracked:  code/slurm-64108209.out
    Untracked:  code/slurm-64108521.out
    Untracked:  code/slurm-64108557.out
    Untracked:  code/snakePASChimp.err
    Untracked:  code/snakePASChimp.out
    Untracked:  code/snakePAShuman.out
    Untracked:  code/snakemake.batch
    Untracked:  code/snakemakeChimp.err
    Untracked:  code/snakemakeChimp.out
    Untracked:  code/snakemakeHuman.err
    Untracked:  code/snakemakeHuman.out
    Untracked:  code/snakemakePAS.batch
    Untracked:  code/snakemakePASFiltChimp.err
    Untracked:  code/snakemakePASFiltChimp.out
    Untracked:  code/snakemakePASFiltHuman.err
    Untracked:  code/snakemakePASFiltHuman.out
    Untracked:  code/snakemakePAS_Human.batch
    Untracked:  code/snakemakePASchimp.batch
    Untracked:  code/snakemakePAShuman.batch
    Untracked:  code/snakemake_chimp.batch
    Untracked:  code/snakemake_human.batch
    Untracked:  code/snakemakefiltPAS.batch
    Untracked:  code/snakemakefiltPAS_chimp.sh
    Untracked:  code/snakemakefiltPAS_human.batch
    Untracked:  code/snakemakefiltPAS_human.sh
    Untracked:  code/spliceSite2Fasta.py
    Untracked:  code/submit-snakemake-chimp.sh
    Untracked:  code/submit-snakemake-human.sh
    Untracked:  code/submit-snakemakePAS-chimp.sh
    Untracked:  code/submit-snakemakePAS-human.sh
    Untracked:  code/submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/submit-snakemakefiltPAS-human.sh
    Untracked:  code/subset_diffisopheno.py
    Untracked:  code/subset_diffisopheno_Chimp_tvN.py
    Untracked:  code/subset_diffisopheno_Chimp_tvN_DF.py
    Untracked:  code/subset_diffisopheno_Huma_tvN.py
    Untracked:  code/subset_diffisopheno_Huma_tvN_DF.py
    Untracked:  code/subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/subset_diffisopheno_Nuclear_HvC_DF.py
    Untracked:  code/subset_diffisopheno_Total_HvC.py
    Untracked:  code/test
    Untracked:  code/test.txt
    Untracked:  code/threeprimeOrthoFC.out
    Untracked:  code/threeprimeOrthoFC.sh
    Untracked:  code/threeprimeOrthoFCcd.err
    Untracked:  code/transcriptDTplotsNuclear.sh
    Untracked:  code/transcriptDTplotsTotal.sh
    Untracked:  code/verifyBam4973.sh
    Untracked:  code/verifyBam4973inHuman.sh
    Untracked:  code/verifybam4973.err
    Untracked:  code/verifybam4973.out
    Untracked:  code/verifybam4973HumanMap.err
    Untracked:  code/verifybam4973HumanMap.out
    Untracked:  code/wrap_Chimpverifybam.err
    Untracked:  code/wrap_Chimpverifybam.out
    Untracked:  code/wrap_chimpverifybam.sh
    Untracked:  code/wrap_verifyBam.sh
    Untracked:  code/wrap_verifybam.err
    Untracked:  code/wrap_verifybam.out
    Untracked:  code/writeMergecode.py
    Untracked:  data/._.DS_Store
    Untracked:  data/._HC_filenames.txt
    Untracked:  data/._HC_filenames.txt.sb-4426323c-IKIs0S
    Untracked:  data/._HC_filenames.xlsx
    Untracked:  data/._MapPantro6_meta.txt
    Untracked:  data/._MapPantro6_meta.txt.sb-a5794dd2-Cskmlm
    Untracked:  data/._MapPantro6_meta.xlsx
    Untracked:  data/._OppositeSpeciesMap.txt
    Untracked:  data/._OppositeSpeciesMap.txt.sb-a5794dd2-mayWJf
    Untracked:  data/._OppositeSpeciesMap.xlsx
    Untracked:  data/._RNASEQ_metadata.txt
    Untracked:  data/._RNASEQ_metadata.txt.sb-4426323c-TE4ns3
    Untracked:  data/._RNASEQ_metadata.txt.sb-51f67ae1-HXp7Gq
    Untracked:  data/._RNASEQ_metadata_2Removed.txt
    Untracked:  data/._RNASEQ_metadata_2Removed.txt.sb-4426323c-a4lBwx
    Untracked:  data/._RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/._RNASEQ_metadata_stranded.txt
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-a5794dd2-D659m2
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-a5794dd2-ImNMoY
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-e4bf31f0-ZGnGgl
    Untracked:  data/._RNASEQ_metadata_stranded.xlsx
    Untracked:  data/._TrialFiltersMeta.txt
    Untracked:  data/._TrialFiltersMeta.txt.sb-9845453e-R58Y0Q
    Untracked:  data/._metadata_HCpanel.txt
    Untracked:  data/._metadata_HCpanel.txt.sb-a3d92a2d-b9cYoF
    Untracked:  data/._metadata_HCpanel.txt.sb-a5794dd2-i594qs
    Untracked:  data/._metadata_HCpanel.txt.sb-f4823d1e-qihGek
    Untracked:  data/._metadata_HCpanel_frompantro5.xlsx
    Untracked:  data/._~$RNASEQ_metadata.xlsx
    Untracked:  data/._~$metadata_HCpanel.xlsx
    Untracked:  data/._ï€¦.xlsx
    Untracked:  data/AREelements/
    Untracked:  data/BaseComp/
    Untracked:  data/CleanLiftedPeaks_FC_primary/
    Untracked:  data/CompapaQTLpas/
    Untracked:  data/DIC_Viz/
    Untracked:  data/DNDS/
    Untracked:  data/DTmatrix/
    Untracked:  data/DiffDomandDE_example/
    Untracked:  data/DiffExpression/
    Untracked:  data/DiffIso_Nuclear/
    Untracked:  data/DiffIso_Nuclear_DF/
    Untracked:  data/DiffIso_Total/
    Untracked:  data/DiffSplice/
    Untracked:  data/DiffSplice_liftedJunc/
    Untracked:  data/DiffSplice_removeBad/
    Untracked:  data/DistTwoDom/
    Untracked:  data/DomDefGreaterX/
    Untracked:  data/DomStructure_4/
    Untracked:  data/DominantPAS/
    Untracked:  data/DominantPAS_DF/
    Untracked:  data/DoubleFilterUsageNumeric/
    Untracked:  data/EvalPantro5/
    Untracked:  data/H3K36me3/
    Untracked:  data/HC_filenames.txt
    Untracked:  data/HC_filenames.xlsx
    Untracked:  data/HumanMolPheno/
    Untracked:  data/IndInfoContent/
    Untracked:  data/InfoContent/
    Untracked:  data/Khan_prot/
    Untracked:  data/Li_eqtls/
    Untracked:  data/MapPantro6_meta.txt
    Untracked:  data/MapPantro6_meta.xlsx
    Untracked:  data/MapStats/
    Untracked:  data/NormalizedClusters/
    Untracked:  data/NuclearHvC/
    Untracked:  data/NuclearHvC_DF/
    Untracked:  data/OppositeSpeciesMap.txt
    Untracked:  data/OppositeSpeciesMap.xlsx
    Untracked:  data/OrthoExonBed/
    Untracked:  data/OverlapBenchmark/
    Untracked:  data/OverlappingPAS/
    Untracked:  data/PAS/
    Untracked:  data/PAS_SAF/
    Untracked:  data/PAS_doubleFilter/
    Untracked:  data/Peaks_5perc/
    Untracked:  data/PhastCon/
    Untracked:  data/Pheno_5perc/
    Untracked:  data/Pheno_5perc_DF_nuclear/
    Untracked:  data/Pheno_5perc_nuclear/
    Untracked:  data/Pheno_5perc_nuclear_old/
    Untracked:  data/Pheno_5perc_total/
    Untracked:  data/PhyloP/
    Untracked:  data/Pol2Chip/
    Untracked:  data/RNASEQ_metadata.txt
    Untracked:  data/RNASEQ_metadata_2Removed.txt
    Untracked:  data/RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/RNASEQ_metadata_stranded.txt
    Untracked:  data/RNASEQ_metadata_stranded.txt.sb-e4bf31f0-ZGnGgl/
    Untracked:  data/RNASEQ_metadata_stranded.xlsx
    Untracked:  data/SignalSites/
    Untracked:  data/SignalSites_doublefilter/
    Untracked:  data/SpliceSite/
    Untracked:  data/TestAnnoBiasOE/
    Untracked:  data/TestMM2/
    Untracked:  data/TestMM2_AS/
    Untracked:  data/TestMM2_PrimaryRead/
    Untracked:  data/TestMM2_SeondaryRead/
    Untracked:  data/TestMM2_mismatch/
    Untracked:  data/TestMM2_quality/
    Untracked:  data/TestWithinMergePAS/
    Untracked:  data/Test_FC_methods/
    Untracked:  data/Threeprime2Ortho/
    Untracked:  data/TotalFractionPAS/
    Untracked:  data/TotalHvC/
    Untracked:  data/TrialFiltersMeta.txt
    Untracked:  data/TwoBadSampleAnalysis/
    Untracked:  data/Wang_ribo/
    Untracked:  data/apaQTLGenes/
    Untracked:  data/bioGRID/
    Untracked:  data/chainFiles/
    Untracked:  data/cleanPeaks_anno/
    Untracked:  data/cleanPeaks_byspecies/
    Untracked:  data/cleanPeaks_lifted/
    Untracked:  data/files4viz_nuclear/
    Untracked:  data/files4viz_nuclear_DF/
    Untracked:  data/gviz/
    Untracked:  data/leafviz/
    Untracked:  data/liftover_files/
    Untracked:  data/mediation/
    Untracked:  data/mediation_DF/
    Untracked:  data/metadata_HCpanel.txt
    Untracked:  data/metadata_HCpanel.xlsx
    Untracked:  data/metadata_HCpanel_extra.txt
    Untracked:  data/metadata_HCpanel_frompantro5.txt
    Untracked:  data/metadata_HCpanel_frompantro5.xlsx
    Untracked:  data/miRNA/
    Untracked:  data/multimap/
    Untracked:  data/orthoUTR/
    Untracked:  data/paiDecay/
    Untracked:  data/primaryLift/
    Untracked:  data/reverseLift/
    Untracked:  data/testQuant/
    Untracked:  data/~$RNASEQ_metadata.xlsx
    Untracked:  data/~$metadata_HCpanel.xlsx
    Untracked:  data/ï€¦.xlsx
    Untracked:  output/._.DS_Store
    Untracked:  output/dAPAandDomEnrich.png
    Untracked:  output/dEandDomEnrich.png
    Untracked:  output/dtPlots/
    Untracked:  projectNotes.Rmd
    Untracked:  proteinModelSet.Rmd

Unstaged changes:
    Modified:   analysis/DICNotDEDP.Rmd
    Modified:   analysis/DeandNumPAS.Rmd
    Modified:   analysis/DiffTop2SecondDom.Rmd
    Modified:   analysis/DirSelectionKhan.Rmd
    Modified:   analysis/ExploredAPA.Rmd
    Modified:   analysis/MMExpreiment.Rmd
    Modified:   analysis/OppositeMap.Rmd
    Modified:   analysis/PTM_analysis.Rmd
    Modified:   analysis/TotalDomStructure.Rmd
    Modified:   analysis/TotalVNuclearBothSpecies.Rmd
    Modified:   analysis/annotationInfo.Rmd
    Modified:   analysis/changeMisprimcut.Rmd
    Modified:   analysis/comp2apaQTLPAS.Rmd
    Modified:   analysis/correlationPhenos.Rmd
    Modified:   analysis/dInforContent.Rmd
    Modified:   analysis/diffExpression.Rmd
    Modified:   analysis/establishCutoffs.Rmd
    Modified:   analysis/investigatePantro5.Rmd
    Modified:   analysis/mRNADecay.Rmd
    Modified:   analysis/multiMap.Rmd
    Modified:   analysis/phastCon.Rmd
    Modified:   analysis/pol2.Rmd
    Modified:   analysis/signalsites_doublefilter.Rmd
    Modified:   analysis/speciesSpecific.Rmd

</code></pre>
<p>
Note that any generated files, e.g.Â HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If youâ€™ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/4b2ff4a03156b795718666f84cd9c761b952f752/analysis/NewDomandDE.Rmd" target="_blank">4b2ff4a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-05-17
</td>
<td>
add loc enrich and diff dom plot
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/cb0a024668cd4fab10b941cf1cec571a37f2c341/docs/NewDomandDE.html" target="_blank">cb0a024</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c253b1bb854a604222ca6319260be90672640f17/analysis/NewDomandDE.Rmd" target="_blank">c253b1b</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-30
</td>
<td>
fix same no dapa bug, change simp color
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/69e8fd59816096069258f64e180d898ffffe3411/docs/NewDomandDE.html" target="_blank">69e8fd5</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/132a7169985a534210976cde05d888904699f9ea/analysis/NewDomandDE.Rmd" target="_blank">132a716</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-28
</td>
<td>
make my own dot plots
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/7725e4d79aabfd7c04579ab4a5c6bebfe73f1293/docs/NewDomandDE.html" target="_blank">7725e4d</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b653f27393255018959d3e1e880782a0f4ab1900/analysis/NewDomandDE.Rmd" target="_blank">b653f27</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-27
</td>
<td>
add simpson
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/961b808c7e75bdfb2870e317b6668ecab46999ca/docs/NewDomandDE.html" target="_blank">961b808</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/c1bc4966c75d6590a43f38c7a5090ef7358d0d02/analysis/NewDomandDE.Rmd" target="_blank">c1bc496</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-23
</td>
<td>
add interaction density, dapa and e and order proble
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/add6b2ab6ab218f0c331650bbb3d7507bd65a0c0/docs/NewDomandDE.html" target="_blank">add6b2a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/5168eee7f7d76f04b542b3eb9b3faf9bb417ac66/analysis/NewDomandDE.Rmd" target="_blank">5168eee</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-23
</td>
<td>
fix expected, and p
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/3790efa6842b4564ad18aefed223ab5c7608b89e/docs/NewDomandDE.html" target="_blank">3790efa</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/e513e9f83f244e91b95d8a0673a3721a691396b6/analysis/NewDomandDE.Rmd" target="_blank">e513e9f</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-23
</td>
<td>
add dot chart
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/53a75707083d909ad1c9cf54e2a9c8f94ce45bc3/docs/NewDomandDE.html" target="_blank">53a7570</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b4e617ede962869e7ef3784c37a96f64a025a527/analysis/NewDomandDE.Rmd" target="_blank">b4e617e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-22
</td>
<td>
add color and prop dom, add decay
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/fbc65999dd3844de9ac467ff3fbf6e03c9f2a3a1/docs/NewDomandDE.html" target="_blank">fbc6599</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/95685efb3a0c1ea3b1cbb510c7ed5c6f71862162/analysis/NewDomandDE.Rmd" target="_blank">95685ef</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-21
</td>
<td>
add length diff analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/a60094e6c07cea1e1dab986fd0bd36b27da8e301/docs/NewDomandDE.html" target="_blank">a60094e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/2b63d021b10c674a9631737203bfb525ff86a161/analysis/NewDomandDE.Rmd" target="_blank">2b63d02</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-21
</td>
<td>
add new dom and de/dapa
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/1dc519ab5e05ba948757bcda028136895a2271ae/docs/NewDomandDE.html" target="_blank">1dc519a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/9f8e75fc4bd144eb4cb016469f6a58b675fb4e26/analysis/NewDomandDE.Rmd" target="_blank">9f8e75f</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-17
</td>
<td>
add enrich
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/7c1a91eedcb1fb7005c94e97d78f68ac3ecb494a/docs/NewDomandDE.html" target="_blank">7c1a91e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/218a3d40d59418b055c05563e54f9ed0ccf5057b/docs/NewDomandDE.html" target="_blank">218a3d4</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/2461fb7bf628afc5e809ef902569507efeb154c2/analysis/NewDomandDE.Rmd" target="_blank">2461fb7</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-16
</td>
<td>
new dom integration
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>library(workflowr)</code></pre>
<pre><code>This is workflowr version 1.6.0
Run ?workflowr for help getting started</code></pre>
<pre class="r"><code>library(tidyverse)</code></pre>
<pre><code>â”€â”€ Attaching packages â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€ tidyverse 1.2.1 â”€â”€</code></pre>
<pre><code>âœ” ggplot2 3.1.1       âœ” purrr   0.3.2  
âœ” tibble  2.1.1       âœ” dplyr   0.8.0.1
âœ” tidyr   0.8.3       âœ” stringr 1.3.1  
âœ” readr   1.3.1       âœ” forcats 0.3.0  </code></pre>
<pre><code>â”€â”€ Conflicts â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€ tidyverse_conflicts() â”€â”€
âœ– dplyr::filter() masks stats::filter()
âœ– dplyr::lag()    masks stats::lag()</code></pre>
<pre class="r"><code>library(cowplot)</code></pre>
<pre><code>
Attaching package: &#39;cowplot&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:ggplot2&#39;:

    ggsave</code></pre>
<pre class="r"><code>library(ggpubr)</code></pre>
<pre><code>Loading required package: magrittr</code></pre>
<pre><code>
Attaching package: &#39;magrittr&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:purrr&#39;:

    set_names</code></pre>
<pre><code>The following object is masked from &#39;package:tidyr&#39;:

    extract</code></pre>
<pre><code>
Attaching package: &#39;ggpubr&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:cowplot&#39;:

    get_legend</code></pre>
<pre class="r"><code>library(RColorBrewer)</code></pre>
<p>I will ask if there if dominance and DE are related. First I can ask if genes with dominant PAS are enriched in the DE genes.</p>
<pre class="r"><code>PAS=read.table(&quot;../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt&quot;, header = T, stringsAsFactors = F)
MetaCol=colnames(PAS)

nameID=read.table(&quot;../../genome_anotation_data/ensemble_to_genename.txt&quot;,sep=&quot;\t&quot;, header = T, stringsAsFactors = F)
DE= read.table(&quot;../data/DiffExpression/DEtested_allres.txt&quot;,header=F, stringsAsFactors = F,col.names = c(&#39;Gene_stable_ID&#39;, &#39;logFC&#39; ,&#39;AveExpr&#39;, &#39;t&#39;, &#39;P.Value&#39;, &#39;adj.P.Val&#39;, &#39;B&#39;)) %&gt;% inner_join(nameID, by=&quot;Gene_stable_ID&quot;) %&gt;% dplyr::select(-Gene_stable_ID, -Source_of_gene_name) %&gt;% rename(&quot;gene&quot;=Gene.name) %&gt;% mutate(DE=ifelse(adj.P.Val&lt;=.05, &quot;Yes&quot;,&quot;No&quot;)) %&gt;% select(gene,DE)
DE_yes= DE %&gt;% filter(DE==&quot;Yes&quot;)</code></pre>
<p>Domiance</p>
<pre class="r"><code>#9
HumanDom9=read.table(&quot;../data/DomDefGreaterX/Human_.9_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Human9&quot;) %&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))
ChimpDom9=read.table(&quot;../data/DomDefGreaterX/Chimp_.9_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Chimp9&quot;) %&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))

#8 
HumanDom8=read.table(&quot;../data/DomDefGreaterX/Human_.8_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Human8&quot;)%&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))
ChimpDom8=read.table(&quot;../data/DomDefGreaterX/Chimp_.8_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Chimp8&quot;) %&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))

#7 
HumanDom7=read.table(&quot;../data/DomDefGreaterX/Human_.7_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Human7&quot;)%&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))
ChimpDom7=read.table(&quot;../data/DomDefGreaterX/Chimp_.7_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Chimp7&quot;)%&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))


#6 
HumanDom6=read.table(&quot;../data/DomDefGreaterX/Human_.6_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Human6&quot;) %&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))
ChimpDom6=read.table(&quot;../data/DomDefGreaterX/Chimp_.6_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Chimp6&quot;) %&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))

#5  
HumanDom5=read.table(&quot;../data/DomDefGreaterX/Human_.5_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Human5&quot;) %&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))
ChimpDom5=read.table(&quot;../data/DomDefGreaterX/Chimp_.5_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Chimp5&quot;)%&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))

#4 
HumanDom4=read.table(&quot;../data/DomDefGreaterX/Human_.4_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Human4&quot;)%&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))
ChimpDom4=read.table(&quot;../data/DomDefGreaterX/Chimp_.4_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Chimp4&quot;)%&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))

#3 
HumanDom3=read.table(&quot;../data/DomDefGreaterX/Human_.3_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Human3&quot;) %&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))
ChimpDom3=read.table(&quot;../data/DomDefGreaterX/Chimp_.3_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Chimp3&quot;)%&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))

#2
HumanDom2=read.table(&quot;../data/DomDefGreaterX/Human_.2_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Human2&quot;)%&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))
ChimpDom2=read.table(&quot;../data/DomDefGreaterX/Chimp_.2_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Chimp2&quot;)%&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))

#1
HumanDom1=read.table(&quot;../data/DomDefGreaterX/Human_.1_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Human1&quot;)%&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))
ChimpDom1=read.table(&quot;../data/DomDefGreaterX/Chimp_.1_dominantPAS.txt&quot;, col.names = MetaCol,stringsAsFactors = F) %&gt;% mutate(set=&quot;Chimp1&quot;) %&gt;% mutate(DE=ifelse(gene %in% DE_yes$gene,&quot;Yes&quot;,&quot;No&quot;))

#all  
HumanDomAll= HumanDom1 %&gt;% bind_rows(HumanDom2) %&gt;% bind_rows(HumanDom3) %&gt;% bind_rows(HumanDom4) %&gt;% bind_rows(HumanDom5) %&gt;% bind_rows(HumanDom6) %&gt;% bind_rows(HumanDom7) %&gt;% bind_rows(HumanDom8) %&gt;% bind_rows(HumanDom9) 
ChimpDomAll= ChimpDom1 %&gt;% bind_rows(ChimpDom2) %&gt;% bind_rows(ChimpDom3) %&gt;% bind_rows(ChimpDom4) %&gt;% bind_rows(ChimpDom5) %&gt;% bind_rows(ChimpDom6) %&gt;% bind_rows(ChimpDom7) %&gt;% bind_rows(ChimpDom8) %&gt;% bind_rows(ChimpDom9) </code></pre>
<pre class="r"><code>ChimpSet=c(&#39;Chimp1&#39;,&#39;Chimp2&#39;, &#39;Chimp3&#39;, &#39;Chimp4&#39;, &#39;Chimp5&#39;, &#39;Chimp6&#39;, &#39;Chimp7&#39;, &#39;Chimp8&#39;,&#39;Chimp9&#39;)
EnrichChimp=c()
PvalueChimp=c()
for (i in ChimpSet){
  x=nrow(ChimpDomAll %&gt;% filter(set==i, DE==&quot;Yes&quot;))
  m=nrow(DE_yes)
  n=nrow(DE) - nrow(DE_yes)
  k=nrow(ChimpDomAll %&gt;% filter(set==i))
  N=nrow(DE)
  PvalueChimp=c(PvalueChimp, phyper(x,m,n,k,lower.tail=F))
  enrich=(x/k)/(m/N)
  EnrichChimp=c(EnrichChimp, enrich)
}

PvalueChimp</code></pre>
<pre><code>[1] 1.0000000 1.0000000 1.0000000 0.9999842 0.9999501 0.9971507 0.9994521
[8] 0.9839957 0.5079290</code></pre>
<pre class="r"><code>EnrichChimp</code></pre>
<pre><code>[1] 0.8849697 0.8877147 0.8880516 0.9057939 0.8959653 0.9108136 0.8719019
[8] 0.8855119 0.9896978</code></pre>
<pre class="r"><code>HumanSet=c(&#39;Human1&#39;,&#39;Human2&#39;, &#39;Human3&#39;, &#39;Human4&#39;, &#39;Human5&#39;, &#39;Human6&#39;, &#39;Human7&#39;, &#39;Human8&#39;,&#39;Human9&#39;)
EnrichHuman=c()
PvalueHuman=c()
for (i in HumanSet){
  x=nrow(HumanDomAll %&gt;% filter(set==i, DE==&quot;Yes&quot;))
  m=nrow(DE_yes)
  n=nrow(DE) - nrow(DE_yes)
  k=nrow(HumanDomAll %&gt;% filter(set==i))
  N=nrow(DE)
  PvalueHuman=c(PvalueHuman, phyper(x,m,n,k,lower.tail=F))
  enrich=(x/k)/(m/N)
  EnrichHuman=c(EnrichHuman, enrich)
}

PvalueHuman</code></pre>
<pre><code>[1] 1.0000000 1.0000000 0.9999997 0.9999064 0.9992779 0.9991470 0.9853263
[8] 0.6526253 0.6747532</code></pre>
<pre class="r"><code>EnrichHuman</code></pre>
<pre><code>[1] 0.8885286 0.8749491 0.8850791 0.8975969 0.8974753 0.8785672 0.8903590
[8] 0.9674743 0.9054783</code></pre>
<p>No enrichment for these. The real question is if genes with different dominant PAS are DE. This requires chosing how to call different dominant.</p>
<div id="same-vs-different-dominant" class="section level2">
<h2>Same vs different dominant:</h2>
<p>Are genes with different dominant at the cutoff .4 cutoff enriched for DE:</p>
<pre class="r"><code>FourRes=read.table(&quot;../data/DomStructure_4/InclusiveDominantPASat4.txt&quot;, header = T,stringsAsFactors = F)
FourRes_diff= FourRes %&gt;% filter(Set==&quot;Different&quot;)
FourRes_same= FourRes %&gt;% filter(Set==&quot;Same&quot;)</code></pre>
<pre class="r"><code>x=length(intersect(FourRes_diff$gene,DE_yes$gene))
m=nrow(DE_yes)
n=nrow(DE) - nrow(DE_yes)
k=nrow(FourRes %&gt;% filter(Set==&quot;Different&quot;))
N=nrow(DE)
phyper(x,m,n,k,lower.tail=F)</code></pre>
<pre><code>[1] 0.09585894</code></pre>
<pre class="r"><code>(x/k)/(m/N)</code></pre>
<pre><code>[1] 1.117149</code></pre>
<pre class="r"><code>x=length(intersect(FourRes_same$gene,DE_yes$gene))
m=nrow(DE_yes)
n=nrow(DE) - nrow(DE_yes)
k=nrow(FourRes %&gt;% filter(Set==&quot;Same&quot;))
N=nrow(DE)
phyper(x,m,n,k,lower.tail=F)</code></pre>
<pre><code>[1] 0.9999999</code></pre>
<pre class="r"><code>(x/k)/(m/N)</code></pre>
<pre><code>[1] 0.883703</code></pre>
<p>This is conditioned on the gene having a dominant PAS.</p>
<p>do this based on a set tested both</p>
<pre class="r"><code>All4= FourRes %&gt;% select(gene,Set) %&gt;% inner_join(DE, by=&quot;gene&quot;)

x=nrow(All4 %&gt;% filter(Set==&quot;Different&quot;, DE==&quot;Yes&quot;))
m=nrow(All4 %&gt;% filter( DE==&quot;Yes&quot;))
n=nrow(All4 %&gt;% filter( DE==&quot;No&quot;))
k=nrow(All4 %&gt;% filter(Set==&quot;Different&quot;))
N=nrow(All4)
phyper(x,m,n,k,lower.tail=F)</code></pre>
<pre><code>[1] 0.0009822532</code></pre>
<pre class="r"><code>(x/k)/(m/N)</code></pre>
<pre><code>[1] 1.305745</code></pre>
<p>I am not sure what the set should be.</p>
<pre class="r"><code>x=nrow(All4 %&gt;% filter(Set==&quot;Same&quot;, DE==&quot;Yes&quot;))
m=nrow(All4 %&gt;% filter(DE==&quot;Yes&quot;))
n=nrow(All4 %&gt;% filter(DE==&quot;No&quot;))
k=nrow(All4 %&gt;% filter(Set==&quot;Same&quot;))
N=nrow(All4)
phyper(x,m,n,k,lower.tail=F)</code></pre>
<pre><code>[1] 0.9982958</code></pre>
<pre class="r"><code>(x/k)/(m/N)</code></pre>
<pre><code>[1] 0.9792784</code></pre>
</div>
<div id="robust-to-different-cutoffs" class="section level2">
<h2>Robust to different cutoffs</h2>
<pre class="r"><code>HumanRes=read.table(&quot;../data/DomDefGreaterX/Human_AllGenes_DiffTop.txt&quot;, col.names = c(&quot;Human_PAS&quot;, &quot;gene&quot;,&quot;Human_DiffDom&quot;),stringsAsFactors = F)

ChimpRes=read.table(&quot;../data/DomDefGreaterX/Chimp_AllGenes_DiffTop.txt&quot;, col.names = c(&quot;Chimp_PAS&quot;, &quot;gene&quot;,&quot;Chimp_DiffDom&quot;),stringsAsFactors = F)

BothRes=HumanRes %&gt;% inner_join(ChimpRes,by=&quot;gene&quot;)

BothRes_10=BothRes %&gt;% filter(Chimp_DiffDom &gt;=0.1 | Human_DiffDom&gt;=0.1) %&gt;% mutate(Set= ifelse(Human_PAS==Chimp_PAS,&quot;Same&quot;, &quot;Different&quot;),cut=10) 
BothRes_20=BothRes %&gt;% filter(Chimp_DiffDom &gt;=0.2 | Human_DiffDom&gt;=0.2) %&gt;% mutate(Set= ifelse(Human_PAS==Chimp_PAS,&quot;Same&quot;, &quot;Different&quot;),cut=20)
BothRes_30=BothRes %&gt;% filter(Chimp_DiffDom &gt;=0.3 | Human_DiffDom&gt;=0.3) %&gt;% mutate(Set= ifelse(Human_PAS==Chimp_PAS,&quot;Same&quot;, &quot;Different&quot;),cut=30)
BothRes_40=BothRes %&gt;% filter(Chimp_DiffDom &gt;=0.4 | Human_DiffDom&gt;=0.4) %&gt;% mutate(Set= ifelse(Human_PAS==Chimp_PAS,&quot;Same&quot;, &quot;Different&quot;),cut=40)
BothRes_50=BothRes %&gt;% filter(Chimp_DiffDom &gt;=0.5 | Human_DiffDom&gt;=0.5) %&gt;% mutate(Set= ifelse(Human_PAS==Chimp_PAS,&quot;Same&quot;, &quot;Different&quot;),cut=50)
BothRes_60=BothRes %&gt;% filter(Chimp_DiffDom &gt;=0.6 | Human_DiffDom&gt;=0.6) %&gt;% mutate(Set= ifelse(Human_PAS==Chimp_PAS,&quot;Same&quot;, &quot;Different&quot;),cut=60)
BothRes_70=BothRes %&gt;% filter(Chimp_DiffDom &gt;=0.7 | Human_DiffDom&gt;=0.7) %&gt;% mutate(Set= ifelse(Human_PAS==Chimp_PAS,&quot;Same&quot;, &quot;Different&quot;),cut=70)
BothRes_80=BothRes %&gt;% filter(Chimp_DiffDom &gt;=0.8 | Human_DiffDom&gt;=0.8) %&gt;% mutate(Set= ifelse(Human_PAS==Chimp_PAS,&quot;Same&quot;, &quot;Different&quot;),cut=80)
BothRes_90=BothRes %&gt;% filter(Chimp_DiffDom &gt;=0.9 | Human_DiffDom&gt;=0.9) %&gt;% mutate(Set= ifelse(Human_PAS==Chimp_PAS,&quot;Same&quot;, &quot;Different&quot;),cut=90)

BothResAll=BothRes_10 %&gt;% bind_rows(BothRes_20) %&gt;% bind_rows(BothRes_30) %&gt;% bind_rows(BothRes_40) %&gt;% bind_rows(BothRes_50) %&gt;% bind_rows(BothRes_60) %&gt;% bind_rows(BothRes_70) %&gt;% bind_rows(BothRes_80) %&gt;% bind_rows(BothRes_90)</code></pre>
<pre class="r"><code>Pval=c()
Enrich=c()
set=c(10,20,30,40,50,60,70,80,90)
expected=c()
actual=c()

All4= BothResAll  %&gt;% select(gene,cut,Set) %&gt;% inner_join(DE, by=&quot;gene&quot;)


for (i in set){
  x=nrow(All4 %&gt;% filter(cut==i, Set==&quot;Different&quot;, DE==&quot;Yes&quot;))
  actual=c(actual, x)
  m=nrow(All4 %&gt;% filter(cut==i, DE==&quot;Yes&quot;))
  n=nrow(All4 %&gt;% filter(cut==i, DE==&quot;No&quot;))
  k=nrow(All4 %&gt;% filter(cut==i, Set==&quot;Different&quot;))
  N=nrow(All4 %&gt;% filter(cut==i))
  val=phyper(x-1,m,n,k,lower.tail=F)
  Pval= c(Pval, val)
  en=(x/k)/(m/N)
  Enrich=c(Enrich, en)
  #ex=which(grepl(max(dhyper(1:x, m, n, k)), dhyper(1:x, m, n, k)))
  ex=k*(m/N)
  expected=c(expected,ex)
}

ResDF=as.data.frame(cbind(set,Pval,Enrich, actual, expected))
ResDF$set=as.factor(ResDF$set)
ResDF$Pval=as.numeric(as.character(ResDF$Pval))
ResDF$Enrich=as.numeric(as.character(ResDF$Enrich))

diffP=ggplot(ResDF,aes(x=set, y=-log10(Pval),fill=set)) + geom_bar(stat=&quot;identity&quot;) +labs(title=&quot;Enrichment pvalues for DE and different dominant&quot;)+ scale_fill_brewer(palette = &quot;RdYlBu&quot;) + theme(legend.position = &quot;none&quot;)+ geom_hline(yintercept = 1.30103)
diffE=ggplot(ResDF,aes(x=set, y=Enrich,fill=set)) + geom_bar(stat=&quot;identity&quot;) + geom_hline(yintercept = 1)+labs(title=&quot;Enrichment for DE and different dominant&quot;)+ scale_fill_brewer(palette = &quot;RdYlBu&quot;) + theme(legend.position = &quot;none&quot;)</code></pre>
<pre class="r"><code>PvalSame=c()
EnrichSame=c()
expectedSame=c()
actualSame=c()



for (i in set){
  x=nrow(All4 %&gt;% filter(cut==i, Set==&quot;Same&quot;, DE==&quot;Yes&quot;))
  actualSame=c(actualSame, x)
  m=nrow(All4 %&gt;% filter(cut==i, DE==&quot;Yes&quot;))
  n=nrow(All4 %&gt;% filter(cut==i, DE==&quot;No&quot;))
  k=nrow(All4 %&gt;% filter(cut==i, Set==&quot;Same&quot;))
  N=nrow(All4 %&gt;% filter(cut==i))
  val=phyper(x-1,m,n,k,lower.tail=F)
  PvalSame= c(PvalSame, val)
  en=(x/k)/(m/N)
  EnrichSame=c(EnrichSame, en)
  #ex=which(grepl(max(dhyper(1:x, m, n, k)), dhyper(1:x, m, n, k)))
  ex=k*(m/N)
  expectedSame=c(expectedSame,ex)
}

ResDFSame=as.data.frame(cbind(set,PvalSame,EnrichSame, actualSame,expectedSame))
ResDFSame$set=as.factor(ResDFSame$set)
ResDFSame$PvalSame=as.numeric(as.character(ResDFSame$PvalSame))
ResDFSame$EnrichSame=as.numeric(as.character(ResDFSame$EnrichSame))

Samep=ggplot(ResDFSame,aes(x=set, y=-log10(PvalSame),fill=set)) + geom_bar(stat=&quot;identity&quot;) +labs(title=&quot;Enrichment pvalues for DE and same dominant&quot;)+ scale_fill_brewer(palette = &quot;RdYlBu&quot;) + theme(legend.position = &quot;none&quot;)+ geom_hline(yintercept = 1.30103)
SameE=ggplot(ResDFSame,aes(x=set, y=EnrichSame,fill=set)) + geom_bar(stat=&quot;identity&quot;) + geom_hline(yintercept = 1)+labs(title=&quot;Enrichment for DE and same dominant&quot;)+ scale_fill_brewer(palette = &quot;RdYlBu&quot;) + theme(legend.position = &quot;none&quot;)</code></pre>
<pre class="r"><code>ResDF</code></pre>
<pre><code>  set        Pval   Enrich actual   expected
1  10 0.025369960 1.070079    430 401.839506
2  20 0.003500182 1.146763    221 192.716425
3  30 0.002404823 1.227587    116  94.494294
4  40 0.001704216 1.305745     76  58.204341
5  50 0.004173416 1.346146     51  37.885924
6  60 0.002304926 1.461459     36  24.632911
7  70 0.023483381 1.491590     18  12.067662
8  80 0.228627744 1.247994     11   8.814145
9  90 0.384878104 1.191860      6   5.034146</code></pre>
<pre class="r"><code>ResDFSame</code></pre>
<pre><code>  set  PvalSame EnrichSame actualSame expectedSame
1  10 0.9785161  0.9816444       1506   1534.16049
2  20 0.9973903  0.9779256       1253   1281.28357
3  30 0.9984228  0.9794500       1025   1046.50571
4  40 0.9990177  0.9792784        841    858.79566
5  50 0.9977739  0.9808864        673    686.11408
6  60 0.9990053  0.9786078        520    531.36709
7  70 0.9905316  0.9838326        361    366.93234
8  80 0.8790630  0.9902498        222    224.18586
9  90 0.8120858  0.9880709         80     80.96585</code></pre>
<p>Look at propotion of DE genes</p>
<pre class="r"><code>PropDE=as.data.frame(cbind(set, OverlapDE=actual, DE=rep(nrow(DE_yes),9))) %&gt;% mutate(Prop=OverlapDE/DE)
PropDE$set=as.factor(PropDE$set)
ggplot(PropDE, aes(x=set,y=Prop,fill=set)) + geom_bar(stat=&quot;identity&quot;)+ labs(title=&quot;Proportion of DE genes with different dominant PAS&quot;, y=&quot;Proportion of DE genes&quot;,x=&quot;Dominance Cuttoff&quot;)+ scale_fill_brewer(palette = &quot;RdYlBu&quot;) + theme(legend.position = &quot;none&quot;)+geom_text(aes(label=OverlapDE), position=position_dodge(width=0.9), vjust=1.5)</code></pre>
<p><img src="figure/NewDomandDE.Rmd/unnamed-chunk-15-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-15-1">
Past versions of unnamed-chunk-15-1.png
</button>
</p>
<div id="fig-unnamed-chunk-15-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/3790efa6842b4564ad18aefed223ab5c7608b89e/docs/figure/NewDomandDE.Rmd/unnamed-chunk-15-1.png" target="_blank">3790efa</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/53a75707083d909ad1c9cf54e2a9c8f94ce45bc3/docs/figure/NewDomandDE.Rmd/unnamed-chunk-15-1.png" target="_blank">53a7570</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/fbc65999dd3844de9ac467ff3fbf6e03c9f2a3a1/docs/figure/NewDomandDE.Rmd/unnamed-chunk-15-1.png" target="_blank">fbc6599</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>plot grid:</p>
<pre class="r"><code>plot_grid(diffE, SameE, diffP, Samep)</code></pre>
<p><img src="figure/NewDomandDE.Rmd/unnamed-chunk-16-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-16-1">
Past versions of unnamed-chunk-16-1.png
</button>
</p>
<div id="fig-unnamed-chunk-16-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/69e8fd59816096069258f64e180d898ffffe3411/docs/figure/NewDomandDE.Rmd/unnamed-chunk-16-1.png" target="_blank">69e8fd5</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/add6b2ab6ab218f0c331650bbb3d7507bd65a0c0/docs/figure/NewDomandDE.Rmd/unnamed-chunk-16-1.png" target="_blank">add6b2a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/53a75707083d909ad1c9cf54e2a9c8f94ce45bc3/docs/figure/NewDomandDE.Rmd/unnamed-chunk-16-1.png" target="_blank">53a7570</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid( diffP, Samep)</code></pre>
<p><img src="figure/NewDomandDE.Rmd/unnamed-chunk-16-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-16-2">
Past versions of unnamed-chunk-16-2.png
</button>
</p>
<div id="fig-unnamed-chunk-16-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/69e8fd59816096069258f64e180d898ffffe3411/docs/figure/NewDomandDE.Rmd/unnamed-chunk-16-2.png" target="_blank">69e8fd5</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggdotchart(ResDF,y=&quot;Enrich&quot;,x= &quot;set&quot;, color=&quot;set&quot;,add = &quot;segments&quot;, rotate = TRUE, dot.size = 10,  label = round(ResDF$Enrich,2), font.label = list(color = &quot;black&quot;, size = 10, vjust = 0.5),ggtheme = theme_pubr(),sort=&quot;d&quot;, legend=&quot;none&quot;, palette=&quot;RdYlBu&quot;, title=&quot;Enrichment for DE and dAPA Different Dominant&quot;) + geom_hline(yintercept = 1)

ggdotchart(ResDFSame,y=&quot;EnrichSame&quot;,x= &quot;set&quot;, color=&quot;set&quot;,add = &quot;segments&quot;, rotate = TRUE, dot.size = 10,  label = round(ResDFSame$EnrichSame,2), font.label = list(color = &quot;black&quot;, size = 10, vjust = 0.5),ggtheme = theme_pubr(),sort=&quot;d&quot;, legend=&quot;none&quot;, palette=&quot;RdYlBu&quot;, title=&quot;Enrichment for DE and dAPA Same Dominant&quot;) + geom_hline(yintercept = 1)</code></pre>
<p>Plot together:</p>
<pre class="r"><code>ResDFfix=ResDF %&gt;% mutate(type=&quot;Different&quot;)
ResDFSamefix=ResDFSame %&gt;% mutate(type=&quot;Same&quot;) 

colnames(ResDFSamefix)=colnames(ResDFfix)
ResDFfix_both=ResDFfix %&gt;% bind_rows(ResDFSamefix)</code></pre>
<pre class="r"><code>DEdomPlot=ggplot(ResDFfix_both,aes(x=set,group=type,col=set,y=Enrich))+ geom_bar(stat=&quot;identity&quot;,col=&quot;grey&quot;,alpha=.3,width=.01)+geom_point(size=10) + coord_flip()+ geom_hline(yintercept = 1) +scale_color_brewer(palette=&quot;RdYlBu&quot;)+geom_text(col=&quot;black&quot;,aes(label = round(Enrich,2)))+ facet_grid(~type)+labs(x=&quot;Dominance Cutoff&quot;, y=&quot;Enrichment&quot;,title=&quot;Enrichment for DE by Domianance&quot;) + theme(legend.position = &quot;none&quot;)
DEdomPlot</code></pre>
<p><img src="figure/NewDomandDE.Rmd/unnamed-chunk-19-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-19-1">
Past versions of unnamed-chunk-19-1.png
</button>
</p>
<div id="fig-unnamed-chunk-19-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/69e8fd59816096069258f64e180d898ffffe3411/docs/figure/NewDomandDE.Rmd/unnamed-chunk-19-1.png" target="_blank">69e8fd5</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/961b808c7e75bdfb2870e317b6668ecab46999ca/docs/figure/NewDomandDE.Rmd/unnamed-chunk-19-1.png" target="_blank">961b808</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="split-into-with-dapa-and-without" class="section level2">
<h2>Split into with dAPA and without</h2>
<p>Add info about DE to All4.</p>
<pre class="r"><code>diffIsoGenes= read.table(&quot;../data/DiffIso_Nuclear_DF/SignifianceEitherGENES_Nuclear.txt&quot;,stringsAsFactors = F, header = T)

All4DiffIso= All4 %&gt;% mutate(dAPA=ifelse(gene %in% diffIsoGenes$gene, &quot;Yes&quot;, &quot;No&quot;)) 

PvalSameapa=c()
pvalBothSideapa=c()
EnrichSameAPA=c()
ExpectedSameAPA=c()
actualSameapa=c()
#same dapa
for (i in set){
  x=nrow(All4DiffIso %&gt;% filter(cut==i, Set==&quot;Same&quot;, DE==&quot;Yes&quot;, dAPA==&quot;Yes&quot;))
  actualSameapa=c(actualSameapa, x)
  m=nrow(All4DiffIso %&gt;% filter(cut==i, DE==&quot;Yes&quot;, dAPA==&quot;Yes&quot;))
  n=nrow(All4DiffIso %&gt;% filter(cut==i, DE==&quot;No&quot;, dAPA==&quot;Yes&quot;))
  k=nrow(All4DiffIso %&gt;% filter(cut==i, Set==&quot;Same&quot;, dAPA==&quot;Yes&quot;))
  N=nrow(All4DiffIso %&gt;% filter(cut==i,dAPA==&quot;Yes&quot;))
  val=phyper(x-1,m,n,k,lower.tail=F)
  val2=phyper(x,m,n,k,lower.tail=T)
  pvalBothSideapa=c(pvalBothSideapa, val2)
  PvalSameapa= c(PvalSameapa, val)
  en=(x/k)/(m/N)
  EnrichSameAPA=c(EnrichSameAPA, en)
  #ex=which(grepl(max(dhyper(1:x, m, n, k)), dhyper(1:x, m, n, k)))
  ex=k*(m/N)
  ExpectedSameAPA=c(ExpectedSameAPA,ex)
}
actualSameapa</code></pre>
<pre><code>[1] 231 207 172 136 103  80  46  26   7</code></pre>
<pre class="r"><code>ExpectedSameAPA</code></pre>
<pre><code>[1] 234.67700 217.20264 187.27941 146.76411 110.98324  84.50000  48.48485
[8]  25.65333   6.50000</code></pre>
<pre class="r"><code>EnrichSameAPA</code></pre>
<pre><code>[1] 0.9843316 0.9530271 0.9184138 0.9266571 0.9280681 0.9467456 0.9487500
[8] 1.0135135 1.0769231</code></pre>
<pre class="r"><code>PvalSameapa</code></pre>
<pre><code>[1] 0.6908226 0.9271034 0.9951886 0.9867290 0.9779845 0.9302170 0.8874905
[8] 0.5307203 0.5000000</code></pre>
<pre class="r"><code>pvalBothSideapa</code></pre>
<pre><code>[1] 0.352383333 0.093628644 0.007647351 0.021604183 0.037785694 0.118754602
[7] 0.209980308 0.663841564 0.793175394</code></pre>
<pre class="r"><code>set</code></pre>
<pre><code>[1] 10 20 30 40 50 60 70 80 90</code></pre>
<pre class="r"><code>PvalSameNo=c()
EnrichSameNo=c()
ExpectedSameNo=c()
actualSameNo=c()
pvalBothSideno=c()
#no dapa
for (i in set){
  x=nrow(All4DiffIso %&gt;% filter(cut==i, Set==&quot;Same&quot;, DE==&quot;Yes&quot;, dAPA==&quot;No&quot;))
  actualSameNo=c(actualSameNo, x)
  m=nrow(All4DiffIso %&gt;% filter(cut==i, DE==&quot;Yes&quot;, dAPA==&quot;No&quot;))
  n=nrow(All4DiffIso %&gt;% filter(cut==i, DE==&quot;No&quot;, dAPA==&quot;No&quot;))
  k=nrow(All4DiffIso %&gt;% filter(cut==i, Set==&quot;Same&quot;, dAPA==&quot;No&quot;))
  N=nrow(All4DiffIso %&gt;% filter(cut==i,dAPA==&quot;No&quot;))
  val=phyper(x-1,m,n,k,lower.tail=F)
  val2=phyper(x,m,n,k,lower.tail=T)
  pvalBothSideno=c(pvalBothSideno, val2)
  PvalSameNo= c(PvalSameNo, val)
  en=(x/k)/(m/N)
  EnrichSameNo=c(EnrichSameNo, en)
  #ex=which(grepl(max(dhyper(1:x, m, n, k)), dhyper(1:x, m, n, k)))
  ex=k*(m/N)
  ExpectedSameNo=c(ExpectedSameNo,ex)
}
actualSameNo</code></pre>
<pre><code>[1] 1275 1046  853  705  570  440  315  196   73</code></pre>
<pre class="r"><code>ExpectedSameNo</code></pre>
<pre><code>[1] 1282.2282 1048.5219  849.3733  704.5488  570.0000  440.0000  315.0000
[8]  196.0000   73.0000</code></pre>
<pre class="r"><code>EnrichSameNo</code></pre>
<pre><code>[1] 0.9943628 0.9975948 1.0042699 1.0006404 1.0000000 1.0000000 1.0000000
[8] 1.0000000 1.0000000</code></pre>
<pre class="r"><code>PvalSameNo</code></pre>
<pre><code>[1] 0.78224457 0.71829130 0.08718982 0.54027531 1.00000000 1.00000000
[7] 1.00000000 1.00000000 1.00000000</code></pre>
<pre class="r"><code>pvalBothSideno</code></pre>
<pre><code>[1] 0.2483884 0.3483472 0.9662289 0.8354296 1.0000000 1.0000000 1.0000000
[8] 1.0000000 1.0000000</code></pre>
<p>pval bothsided:</p>
<pre class="r"><code>pvalBothSideno</code></pre>
<pre><code>[1] 0.2483884 0.3483472 0.9662289 0.8354296 1.0000000 1.0000000 1.0000000
[8] 1.0000000 1.0000000</code></pre>
<pre class="r"><code>PvalSameNo</code></pre>
<pre><code>[1] 0.78224457 0.71829130 0.08718982 0.54027531 1.00000000 1.00000000
[7] 1.00000000 1.00000000 1.00000000</code></pre>
<pre class="r"><code>pvalBothSideapa</code></pre>
<pre><code>[1] 0.352383333 0.093628644 0.007647351 0.021604183 0.037785694 0.118754602
[7] 0.209980308 0.663841564 0.793175394</code></pre>
<pre class="r"><code>ResDFSameAPA=as.data.frame(cbind(set,pval=PvalSameapa,pvalOpp=pvalBothSideapa,enrich=EnrichSameAPA, actual=actualSameapa,expected=ExpectedSameAPA)) %&gt;% mutate(dAPA=&quot;Yes&quot;)
ResDFSameNo=as.data.frame(cbind(set,pval=PvalSameNo,pvalOpp=pvalBothSideno,enrich=EnrichSameNo, actual=actualSameNo,expected=ExpectedSameNo))%&gt;% mutate(dAPA=&quot;No&quot;)


ResDFSameApaBoth= ResDFSameAPA %&gt;% bind_rows(ResDFSameNo)

ResDFSameApaBoth$set=as.factor(ResDFSameApaBoth$set)</code></pre>
<pre class="r"><code>ggplot(ResDFSameApaBoth,aes(x=set, by=dAPA, fill=set,alpha=dAPA, y=enrich)) + geom_bar(stat=&quot;identity&quot;, position=&quot;dodge&quot;) +scale_fill_brewer(palette=&quot;RdYlBu&quot;)+ scale_alpha_discrete(range = c(0.6, 1)) + geom_hline(yintercept = 1) + labs( title=&quot;Enrichment for DE by in genes with Same Dominant&quot;,x=&quot;Dominance Cutoff&quot;, y=&quot;Enrichement&quot;)</code></pre>
<pre><code>Warning: Using alpha for a discrete variable is not advised.</code></pre>
<p><img src="figure/NewDomandDE.Rmd/unnamed-chunk-23-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-23-1">
Past versions of unnamed-chunk-23-1.png
</button>
</p>
<div id="fig-unnamed-chunk-23-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/cb0a024668cd4fab10b941cf1cec571a37f2c341/docs/figure/NewDomandDE.Rmd/unnamed-chunk-23-1.png" target="_blank">cb0a024</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-30
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/69e8fd59816096069258f64e180d898ffffe3411/docs/figure/NewDomandDE.Rmd/unnamed-chunk-23-1.png" target="_blank">69e8fd5</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/7725e4d79aabfd7c04579ab4a5c6bebfe73f1293/docs/figure/NewDomandDE.Rmd/unnamed-chunk-23-1.png" target="_blank">7725e4d</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-27
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/961b808c7e75bdfb2870e317b6668ecab46999ca/docs/figure/NewDomandDE.Rmd/unnamed-chunk-23-1.png" target="_blank">961b808</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Try ggplot dot plot.</p>
<pre class="r"><code>ResDFSameApaBoth_len= ResDFSameApaBoth %&gt;% mutate(linelength=ifelse(dAPA==&quot;Yes&quot;, enrich, 0))

ggplot(ResDFSameApaBoth_len,aes(x=set,col=set,shape=dAPA, y=enrich,label = round(enrich,3)))+ geom_bar(stat=&quot;identity&quot;,color=&quot;grey&quot;,aes(y=linelength),width=.01)+geom_point(size=10) + coord_flip() + geom_hline(yintercept = 1) +scale_color_brewer(palette=&quot;RdYlBu&quot;)+ labs( title=&quot;Enrichment for DE by in genes with Same Dominant&quot;,x=&quot;Dominance Cutoff&quot;, y=&quot;Enrichement&quot;)+geom_text(color = &quot;black&quot;, size = 3) + theme(legend.position = &quot;bottom&quot;)</code></pre>
<p><img src="figure/NewDomandDE.Rmd/unnamed-chunk-24-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-24-1">
Past versions of unnamed-chunk-24-1.png
</button>
</p>
<div id="fig-unnamed-chunk-24-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/cb0a024668cd4fab10b941cf1cec571a37f2c341/docs/figure/NewDomandDE.Rmd/unnamed-chunk-24-1.png" target="_blank">cb0a024</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-30
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/69e8fd59816096069258f64e180d898ffffe3411/docs/figure/NewDomandDE.Rmd/unnamed-chunk-24-1.png" target="_blank">69e8fd5</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/7725e4d79aabfd7c04579ab4a5c6bebfe73f1293/docs/figure/NewDomandDE.Rmd/unnamed-chunk-24-1.png" target="_blank">7725e4d</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>enrichmentp=ggplot(ResDFSameApaBoth,aes(x=set, by=dAPA, fill=set, y=-log10(pval))) + geom_bar(stat=&quot;identity&quot;, position=&quot;dodge&quot;) +scale_fill_brewer(palette=&quot;RdYlBu&quot;) + labs( title=&quot;enrichment p-value &quot;,x=&quot;Dominance Cutoff&quot;, y=&quot;-log10(P-value)&quot;) + facet_grid(~dAPA) + theme(legend.position = &quot;none&quot;)+ geom_hline(yintercept = 1.30103)



oppenrichp=ggplot(ResDFSameApaBoth,aes(x=set, by=dAPA, fill=set, y=-log10(pvalOpp))) + geom_bar(stat=&quot;identity&quot;, position=&quot;dodge&quot;) +scale_fill_brewer(palette=&quot;RdYlBu&quot;) + labs( title=&quot;depletion p-value &quot;,x=&quot;Dominance Cutoff&quot;, y=&quot;-log10(P-value)&quot;) + facet_grid(~dAPA) + theme(legend.position = &quot;none&quot;)+ geom_hline(yintercept = 1.30103)</code></pre>
<pre class="r"><code>plot_grid(enrichmentp,oppenrichp, nrow=2)</code></pre>
<p><img src="figure/NewDomandDE.Rmd/unnamed-chunk-26-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-26-1">
Past versions of unnamed-chunk-26-1.png
</button>
</p>
<div id="fig-unnamed-chunk-26-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/69e8fd59816096069258f64e180d898ffffe3411/docs/figure/NewDomandDE.Rmd/unnamed-chunk-26-1.png" target="_blank">69e8fd5</a>
</td>
<td>
brimittleman
</td>
<td>
2020-04-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Plot pvalue with enriichemtn</p>
<pre class="r"><code>pvals=plot_grid( diffP, Samep)


plot_grid(DEdomPlot,pvals, nrow = 2)</code></pre>
<p><img src="figure/NewDomandDE.Rmd/unnamed-chunk-27-1.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.5.1 (2018-07-02)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] RColorBrewer_1.1-2 ggpubr_0.2         magrittr_1.5      
 [4] cowplot_0.9.4      forcats_0.3.0      stringr_1.3.1     
 [7] dplyr_0.8.0.1      purrr_0.3.2        readr_1.3.1       
[10] tidyr_0.8.3        tibble_2.1.1       ggplot2_3.1.1     
[13] tidyverse_1.2.1    workflowr_1.6.0   

loaded via a namespace (and not attached):
 [1] tidyselect_0.2.5 reshape2_1.4.3   haven_1.1.2      lattice_0.20-38 
 [5] colorspace_1.3-2 generics_0.0.2   htmltools_0.3.6  yaml_2.2.0      
 [9] rlang_0.4.0      later_0.7.5      pillar_1.3.1     glue_1.3.0      
[13] withr_2.1.2      modelr_0.1.2     readxl_1.1.0     plyr_1.8.4      
[17] munsell_0.5.0    gtable_0.2.0     cellranger_1.1.0 rvest_0.3.2     
[21] evaluate_0.12    labeling_0.3     knitr_1.20       httpuv_1.4.5    
[25] broom_0.5.1      Rcpp_1.0.4.6     promises_1.0.1   scales_1.0.0    
[29] backports_1.1.2  jsonlite_1.6     fs_1.3.1         hms_0.4.2       
[33] digest_0.6.18    stringi_1.2.4    grid_3.5.1       rprojroot_1.3-2 
[37] cli_1.1.0        tools_3.5.1      lazyeval_0.2.1   crayon_1.3.4    
[41] whisker_0.3-2    pkgconfig_2.0.2  xml2_1.2.0       lubridate_1.7.4 
[45] assertthat_0.2.0 rmarkdown_1.10   httr_1.3.1       rstudioapi_0.10 
[49] R6_2.3.0         nlme_3.1-137     git2r_0.26.1     compiler_3.5.1  </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
