<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Briana Mittleman" />


<title>Try different quantification methods</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/journal.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Comparative_APA</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/brimittleman/Comparative_APA">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Try different quantification methods</h1>
<h4 class="author"><em>Briana Mittleman</em></h4>
<h4 class="date"><em>3/19/2020</em></h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-03-28
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Comparative_APA/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity itâ€™s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20190902code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20190902)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20190902code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20190902)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g.Â subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtree706fa0d50038656a0ed1504362f62640173ba444targetblank706fa0da"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/brimittleman/Comparative_APA/tree/706fa0d50038656a0ed1504362f62640173ba444" target="_blank">706fa0d</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanComparativeAPAtree706fa0d50038656a0ed1504362f62640173ba444targetblank706fa0da" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    code/chimp_log/
    Ignored:    code/human_log/
    Ignored:    data/.DS_Store
    Ignored:    data/TrialFiltersMeta.txt.sb-9845453e-R58Y0Q/
    Ignored:    data/mediation_prot/
    Ignored:    data/metadata_HCpanel.txt.sb-a5794dd2-i594qs/
    Ignored:    output/.DS_Store

Untracked files:
    Untracked:  ._.DS_Store
    Untracked:  Chimp/
    Untracked:  Human/
    Untracked:  analysis/CrossChimpThreePrime.Rmd
    Untracked:  analysis/DiffTransProtvsExpression.Rmd
    Untracked:  analysis/DiffUsedUTR.Rmd
    Untracked:  analysis/GvizPlots.Rmd
    Untracked:  analysis/HandC.TvN
    Untracked:  analysis/PhenotypeOverlap10.Rmd
    Untracked:  analysis/annotationBias.Rmd
    Untracked:  analysis/assessReadQual.Rmd
    Untracked:  analysis/diffExpressionPantro6.Rmd
    Untracked:  analysis/pol2.Rmd
    Untracked:  code/._BothFCMM.sh
    Untracked:  code/._BothFCMMPrim.sh
    Untracked:  code/._BothFCnewOInclusive.sh
    Untracked:  code/._ChimpStarMM2.sh
    Untracked:  code/._ClassifyLeafviz.sh
    Untracked:  code/._Config_chimp.yaml
    Untracked:  code/._Config_chimp_full.yaml
    Untracked:  code/._Config_human.yaml
    Untracked:  code/._ConvertJunc2Bed.sh
    Untracked:  code/._CountNucleotides.py
    Untracked:  code/._CrossMapChimpRNA.sh
    Untracked:  code/._CrossMapThreeprime.sh
    Untracked:  code/._DiffSplice.sh
    Untracked:  code/._DiffSplicePlots.sh
    Untracked:  code/._DiffSplicePlots_gencode.sh
    Untracked:  code/._DiffSplice_gencode.sh
    Untracked:  code/._DiffSplice_removebad.sh
    Untracked:  code/._FindIntronForDomPAS.sh
    Untracked:  code/._FindIntronForDomPAS_DF.sh
    Untracked:  code/._GetMAPQscore.py
    Untracked:  code/._GetSecondaryMap.py
    Untracked:  code/._Lift5perPAS.sh
    Untracked:  code/._LiftFinalChimpJunc2Human.sh
    Untracked:  code/._LiftOrthoPAS2chimp.sh
    Untracked:  code/._MapBadSamples.sh
    Untracked:  code/._PAS_ATTAAA.sh
    Untracked:  code/._PAS_ATTAAA_df.sh
    Untracked:  code/._PAS_seqExpanded.sh
    Untracked:  code/._PASsequences.sh
    Untracked:  code/._PASsequences_DF.sh
    Untracked:  code/._PlotNuclearUsagebySpecies.R
    Untracked:  code/._PlotNuclearUsagebySpecies_DF.R
    Untracked:  code/._QuantMergedClusters.sh
    Untracked:  code/._RNATranscriptDTplot.sh
    Untracked:  code/._ReverseLiftFilter.R
    Untracked:  code/._RunFixLeafCluster.sh
    Untracked:  code/._RunNegMCMediation.sh
    Untracked:  code/._RunNegMCMediationDF.sh
    Untracked:  code/._RunPosMCMediationDF.err
    Untracked:  code/._RunPosMCMediationDF.sh
    Untracked:  code/._SAF2Bed.py
    Untracked:  code/._Snakefile
    Untracked:  code/._SnakefilePAS
    Untracked:  code/._SnakefilePASfilt
    Untracked:  code/._SortIndexBadSamples.sh
    Untracked:  code/._StarMM2.sh
    Untracked:  code/._TestFC.sh
    Untracked:  code/._assignPeak2Intronicregion
    Untracked:  code/._assignPeak2Intronicregion.sh
    Untracked:  code/._bed215upbed.py
    Untracked:  code/._bed2Bedbothstrand.py
    Untracked:  code/._bed2SAF_gen.py
    Untracked:  code/._buildIndecpantro5
    Untracked:  code/._buildIndecpantro5.sh
    Untracked:  code/._buildLeafviz.sh
    Untracked:  code/._buildLeafviz_leadAnno.sh
    Untracked:  code/._buildStarIndex.sh
    Untracked:  code/._chimpChromprder.sh
    Untracked:  code/._chimpMultiCov.sh
    Untracked:  code/._chimpMultiCov255.sh
    Untracked:  code/._chimpMultiCovInclusive.sh
    Untracked:  code/._chooseSignalSite.py
    Untracked:  code/._cleanbed2saf.py
    Untracked:  code/._cluster.json
    Untracked:  code/._cluster2bed.py
    Untracked:  code/._clusterLiftReverse.sh
    Untracked:  code/._clusterLiftReverse_removebad.sh
    Untracked:  code/._clusterLiftprimary.sh
    Untracked:  code/._clusterLiftprimary_removebad.sh
    Untracked:  code/._converBam2Junc.sh
    Untracked:  code/._converBam2Junc_removeBad.sh
    Untracked:  code/._extraSnakefiltpas
    Untracked:  code/._extractPhyloReg.py
    Untracked:  code/._extractPhyloRegGene.py
    Untracked:  code/._extractPhylopGeneral.ph
    Untracked:  code/._extractPhylopGeneral.py
    Untracked:  code/._extractPhylopReg200down.py
    Untracked:  code/._extractPhylopReg200up.py
    Untracked:  code/._filter5percPAS.py
    Untracked:  code/._filterNumChroms.py
    Untracked:  code/._filterPASforMP.py
    Untracked:  code/._filterPostLift.py
    Untracked:  code/._filterPrimaryread.py
    Untracked:  code/._filterSecondaryread.py
    Untracked:  code/._fixExonFC.py
    Untracked:  code/._fixLeafCluster.py
    Untracked:  code/._fixLiftedJunc.py
    Untracked:  code/._fixUTRexonanno.py
    Untracked:  code/._formathg38Anno.py
    Untracked:  code/._formatpantro6Anno.py
    Untracked:  code/._getRNAseqMapStats.sh
    Untracked:  code/._hg19MapStats.sh
    Untracked:  code/._humanChromorder.sh
    Untracked:  code/._humanMultiCov.sh
    Untracked:  code/._humanMultiCov255.sh
    Untracked:  code/._humanMultiCov_inclusive.sh
    Untracked:  code/._intersectLiftedPAS.sh
    Untracked:  code/._liftJunctionFiles.sh
    Untracked:  code/._liftPAS19to38.sh
    Untracked:  code/._liftedchimpJunc2human.sh
    Untracked:  code/._makeNuclearDapaplots.sh
    Untracked:  code/._makeNuclearDapaplots_DF.sh
    Untracked:  code/._makeSamplyGroupsHuman_TvN.py
    Untracked:  code/._mapRNAseqhg19.sh
    Untracked:  code/._mapRNAseqhg19_newPipeline.sh
    Untracked:  code/._maphg19.sh
    Untracked:  code/._maphg19_subjunc.sh
    Untracked:  code/._mediation_test.R
    Untracked:  code/._mergeChimp3prime_inhg38.sh
    Untracked:  code/._mergeandBWRNAseq.sh
    Untracked:  code/._mergedBam2BW.sh
    Untracked:  code/._nameClusters.py
    Untracked:  code/._negativeMediation_montecarlo.R
    Untracked:  code/._negativeMediation_montecarloDF.R
    Untracked:  code/._numMultimap.py
    Untracked:  code/._overlapMMandOrthoexon.sh
    Untracked:  code/._overlapapaQTLPAS.sh
    Untracked:  code/._parseHg38.py
    Untracked:  code/._postiveMediation_montecarlo_DF.R
    Untracked:  code/._prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/._prepareLeafvizAnno.sh
    Untracked:  code/._preparePAS4lift.py
    Untracked:  code/._primaryLift.sh
    Untracked:  code/._processhg38exons.py
    Untracked:  code/._quantJunc.sh
    Untracked:  code/._quantJunc_TEST.sh
    Untracked:  code/._quantJunc_removeBad.sh
    Untracked:  code/._quantMerged_seperatly.sh
    Untracked:  code/._recLiftchim2human.sh
    Untracked:  code/._revLiftPAShg38to19.sh
    Untracked:  code/._reverseLift.sh
    Untracked:  code/._runCheckReverseLift.sh
    Untracked:  code/._runChimpDiffIso.sh
    Untracked:  code/._runCountNucleotides.sh
    Untracked:  code/._runFilterNumChroms.sh
    Untracked:  code/._runHumanDiffIso.sh
    Untracked:  code/._runNuclearDiffIso_DF.sh
    Untracked:  code/._runNuclearDifffIso.sh
    Untracked:  code/._runTotalDiffIso.sh
    Untracked:  code/._run_chimpverifybam.sh
    Untracked:  code/._run_verifyBam.sh
    Untracked:  code/._snakemake.batch
    Untracked:  code/._snakemakePAS.batch
    Untracked:  code/._snakemakePASchimp.batch
    Untracked:  code/._snakemakePAShuman.batch
    Untracked:  code/._snakemake_chimp.batch
    Untracked:  code/._snakemake_human.batch
    Untracked:  code/._snakemakefiltPAS.batch
    Untracked:  code/._snakemakefiltPAS_chimp
    Untracked:  code/._snakemakefiltPAS_chimp.sh
    Untracked:  code/._snakemakefiltPAS_human.sh
    Untracked:  code/._spliceSite2Fasta.py
    Untracked:  code/._submit-snakemake-chimp.sh
    Untracked:  code/._submit-snakemake-human.sh
    Untracked:  code/._submit-snakemakePAS-chimp.sh
    Untracked:  code/._submit-snakemakePAS-human.sh
    Untracked:  code/._submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/._submit-snakemakefiltPAS-human.sh
    Untracked:  code/._subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/._subset_diffisopheno_Nuclear_HvC_DF.py
    Untracked:  code/._subset_diffisopheno_Total_HvC.py
    Untracked:  code/._threeprimeOrthoFC.sh
    Untracked:  code/._transcriptDTplotsNuclear.sh
    Untracked:  code/._verifyBam4973.sh
    Untracked:  code/._verifyBam4973inHuman.sh
    Untracked:  code/._wrap_chimpverifybam.sh
    Untracked:  code/._wrap_verifyBam.sh
    Untracked:  code/._writeMergecode.py
    Untracked:  code/.snakemake/
    Untracked:  code/BothFCMM.err
    Untracked:  code/BothFCMM.out
    Untracked:  code/BothFCMM.sh
    Untracked:  code/BothFCMMPrim.err
    Untracked:  code/BothFCMMPrim.out
    Untracked:  code/BothFCMMPrim.sh
    Untracked:  code/BothFCnewOInclusive.sh
    Untracked:  code/BothFCnewOInclusive.sh.err
    Untracked:  code/BothFCnewOInclusive.sh.out
    Untracked:  code/ChimpStarMM2.err
    Untracked:  code/ChimpStarMM2.out
    Untracked:  code/ChimpStarMM2.sh
    Untracked:  code/ClassifyLeafviz.sh
    Untracked:  code/Config_chimp.yaml
    Untracked:  code/Config_chimp_full.yaml
    Untracked:  code/Config_human.yaml
    Untracked:  code/ConvertJunc2Bed.err
    Untracked:  code/ConvertJunc2Bed.out
    Untracked:  code/ConvertJunc2Bed.sh
    Untracked:  code/CountNucleotides.py
    Untracked:  code/CrossMapChimpRNA.sh
    Untracked:  code/CrossMapThreeprime.sh
    Untracked:  code/CrossmapChimp3prime.err
    Untracked:  code/CrossmapChimp3prime.out
    Untracked:  code/CrossmapChimpRNA.err
    Untracked:  code/CrossmapChimpRNA.out
    Untracked:  code/DiffSplice.err
    Untracked:  code/DiffSplice.out
    Untracked:  code/DiffSplice.sh
    Untracked:  code/DiffSplicePlots.err
    Untracked:  code/DiffSplicePlots.out
    Untracked:  code/DiffSplicePlots.sh
    Untracked:  code/DiffSplicePlots_gencode.sh
    Untracked:  code/DiffSplice_gencode.sh
    Untracked:  code/DiffSplice_removebad.err
    Untracked:  code/DiffSplice_removebad.out
    Untracked:  code/DiffSplice_removebad.sh
    Untracked:  code/FilterReverseLift.err
    Untracked:  code/FilterReverseLift.out
    Untracked:  code/FindIntronForDomPAS.err
    Untracked:  code/FindIntronForDomPAS.out
    Untracked:  code/FindIntronForDomPAS.sh
    Untracked:  code/FindIntronForDomPAS_DF.sh
    Untracked:  code/GencodeDiffSplice.err
    Untracked:  code/GencodeDiffSplice.out
    Untracked:  code/GetMAPQscore.py
    Untracked:  code/GetSecondaryMap.py
    Untracked:  code/HchromOrder.err
    Untracked:  code/HchromOrder.out
    Untracked:  code/IntersectMMandOrtho.err
    Untracked:  code/IntersectMMandOrtho.out
    Untracked:  code/JunctionLift.err
    Untracked:  code/JunctionLift.out
    Untracked:  code/JunctionLiftFinalChimp.err
    Untracked:  code/JunctionLiftFinalChimp.out
    Untracked:  code/Lift5perPAS.sh
    Untracked:  code/Lift5perPASbed.err
    Untracked:  code/Lift5perPASbed.out
    Untracked:  code/LiftClustersFirst.err
    Untracked:  code/LiftClustersFirst.out
    Untracked:  code/LiftClustersFirst_remove.err
    Untracked:  code/LiftClustersFirst_remove.out
    Untracked:  code/LiftClustersSecond.err
    Untracked:  code/LiftClustersSecond.out
    Untracked:  code/LiftClustersSecond_remove.err
    Untracked:  code/LiftClustersSecond_remove.out
    Untracked:  code/LiftFinalChimpJunc2Human.sh
    Untracked:  code/LiftOrthoPAS2chimp.sh
    Untracked:  code/LiftorthoPAS.err
    Untracked:  code/LiftorthoPASt.out
    Untracked:  code/Log.out
    Untracked:  code/MapBadSamples.err
    Untracked:  code/MapBadSamples.out
    Untracked:  code/MapBadSamples.sh
    Untracked:  code/MapStats.err
    Untracked:  code/MapStats.out
    Untracked:  code/MaxEntCode/
    Untracked:  code/MergeClusters.err
    Untracked:  code/MergeClusters.out
    Untracked:  code/MergeClusters.sh
    Untracked:  code/PAS_ATTAAA.err
    Untracked:  code/PAS_ATTAAA.out
    Untracked:  code/PAS_ATTAAA.sh
    Untracked:  code/PAS_ATTAAADF.err
    Untracked:  code/PAS_ATTAAADF.out
    Untracked:  code/PAS_ATTAAA_df.sh
    Untracked:  code/PAS_seqExpanded.sh
    Untracked:  code/PAS_sequence.err
    Untracked:  code/PAS_sequence.out
    Untracked:  code/PAS_sequenceDF.err
    Untracked:  code/PAS_sequenceDF.out
    Untracked:  code/PASexpanded_sequenceDF.err
    Untracked:  code/PASexpanded_sequenceDF.out
    Untracked:  code/PASsequences.sh
    Untracked:  code/PASsequences_DF.sh
    Untracked:  code/PlotNuclearUsagebySpecies.R
    Untracked:  code/PlotNuclearUsagebySpecies_DF.R
    Untracked:  code/QuantMergeClusters
    Untracked:  code/QuantMergeClusters.err
    Untracked:  code/QuantMergeClusters.out
    Untracked:  code/QuantMergedClusters.sh
    Untracked:  code/RNATranscriptDTplot.err
    Untracked:  code/RNATranscriptDTplot.out
    Untracked:  code/RNATranscriptDTplot.sh
    Untracked:  code/Rev_liftoverPAShg19to38.err
    Untracked:  code/Rev_liftoverPAShg19to38.out
    Untracked:  code/ReverseLiftFilter.R
    Untracked:  code/RunFixCluster.err
    Untracked:  code/RunFixCluster.out
    Untracked:  code/RunFixLeafCluster.sh
    Untracked:  code/RunNegMCMediation.err
    Untracked:  code/RunNegMCMediation.sh
    Untracked:  code/RunNegMCMediationDF.err
    Untracked:  code/RunNegMCMediationDF.out
    Untracked:  code/RunNegMCMediationDF.sh
    Untracked:  code/RunNegMCMediationr.out
    Untracked:  code/RunPosMCMediation.err
    Untracked:  code/RunPosMCMediation.sh
    Untracked:  code/RunPosMCMediationDF.err
    Untracked:  code/RunPosMCMediationDF.out
    Untracked:  code/RunPosMCMediationDF.sh
    Untracked:  code/RunPosMCMediationr.out
    Untracked:  code/SAF215upbed_gen.py
    Untracked:  code/SAF2Bed.py
    Untracked:  code/Snakefile
    Untracked:  code/SnakefilePAS
    Untracked:  code/SnakefilePASfilt
    Untracked:  code/SortIndexBadSamples.err
    Untracked:  code/SortIndexBadSamples.out
    Untracked:  code/SortIndexBadSamples.sh
    Untracked:  code/StarMM2.err
    Untracked:  code/StarMM2.out
    Untracked:  code/StarMM2.sh
    Untracked:  code/TestFC.err
    Untracked:  code/TestFC.out
    Untracked:  code/TestFC.sh
    Untracked:  code/TotalTranscriptDTplot.err
    Untracked:  code/TotalTranscriptDTplot.out
    Untracked:  code/Upstream10Bases_general.py
    Untracked:  code/apaQTLsnake.err
    Untracked:  code/apaQTLsnake.out
    Untracked:  code/apaQTLsnakePAS.err
    Untracked:  code/apaQTLsnakePAS.out
    Untracked:  code/apaQTLsnakePAShuman.err
    Untracked:  code/assignPeak2Intronicregion.err
    Untracked:  code/assignPeak2Intronicregion.out
    Untracked:  code/assignPeak2Intronicregion.sh
    Untracked:  code/bam2junc.err
    Untracked:  code/bam2junc.out
    Untracked:  code/bam2junc_remove.err
    Untracked:  code/bam2junc_remove.out
    Untracked:  code/bed215upbed.py
    Untracked:  code/bed2Bedbothstrand.py
    Untracked:  code/bed2SAF_gen.py
    Untracked:  code/bed2saf.py
    Untracked:  code/bg_to_cov.py
    Untracked:  code/buildIndecpantro5
    Untracked:  code/buildIndecpantro5.sh
    Untracked:  code/buildLeafviz.err
    Untracked:  code/buildLeafviz.out
    Untracked:  code/buildLeafviz.sh
    Untracked:  code/buildLeafviz_leadAnno.sh
    Untracked:  code/buildLeafviz_leafanno.err
    Untracked:  code/buildLeafviz_leafanno.out
    Untracked:  code/buildStarIndex.sh
    Untracked:  code/callPeaksYL.py
    Untracked:  code/chimpChromprder.sh
    Untracked:  code/chimpMultiCov.err
    Untracked:  code/chimpMultiCov.out
    Untracked:  code/chimpMultiCov.sh
    Untracked:  code/chimpMultiCov255.sh
    Untracked:  code/chimpMultiCovInclusive.err
    Untracked:  code/chimpMultiCovInclusive.out
    Untracked:  code/chimpMultiCovInclusive.sh
    Untracked:  code/chooseAnno2Bed.py
    Untracked:  code/chooseAnno2SAF.py
    Untracked:  code/chooseSignalSite.py
    Untracked:  code/chromOrder.err
    Untracked:  code/chromOrder.out
    Untracked:  code/classifyLeafviz.err
    Untracked:  code/classifyLeafviz.out
    Untracked:  code/cleanbed2saf.py
    Untracked:  code/cluster.json
    Untracked:  code/cluster2bed.py
    Untracked:  code/clusterLiftReverse.sh
    Untracked:  code/clusterLiftReverse_removebad.sh
    Untracked:  code/clusterLiftprimary.sh
    Untracked:  code/clusterLiftprimary_removebad.sh
    Untracked:  code/clusterPAS.json
    Untracked:  code/clusterfiltPAS.json
    Untracked:  code/comands2Mege.sh
    Untracked:  code/converBam2Junc.sh
    Untracked:  code/converBam2Junc_removeBad.sh
    Untracked:  code/convertNumeric.py
    Untracked:  code/environment.yaml
    Untracked:  code/extraSnakefiltpas
    Untracked:  code/extractPhyloReg.py
    Untracked:  code/extractPhyloRegGene.py
    Untracked:  code/extractPhylopGeneral.py
    Untracked:  code/extractPhylopReg200down.py
    Untracked:  code/extractPhylopReg200up.py
    Untracked:  code/filter5perc.R
    Untracked:  code/filter5percPAS.py
    Untracked:  code/filter5percPheno.py
    Untracked:  code/filterBamforMP.pysam2_gen.py
    Untracked:  code/filterJuncChroms.err
    Untracked:  code/filterJuncChroms.out
    Untracked:  code/filterMissprimingInNuc10_gen.py
    Untracked:  code/filterNumChroms.py
    Untracked:  code/filterPASforMP.py
    Untracked:  code/filterPostLift.py
    Untracked:  code/filterPrimaryread.py
    Untracked:  code/filterSAFforMP_gen.py
    Untracked:  code/filterSecondaryread.py
    Untracked:  code/filterSortBedbyCleanedBed_gen.R
    Untracked:  code/filterpeaks.py
    Untracked:  code/fixExonFC.py
    Untracked:  code/fixFChead.py
    Untracked:  code/fixFChead_bothfrac.py
    Untracked:  code/fixLeafCluster.py
    Untracked:  code/fixLiftedJunc.py
    Untracked:  code/fixUTRexonanno.py
    Untracked:  code/formathg38Anno.py
    Untracked:  code/generateStarIndex.err
    Untracked:  code/generateStarIndex.out
    Untracked:  code/generateStarIndexHuman.err
    Untracked:  code/generateStarIndexHuman.out
    Untracked:  code/getRNAseqMapStats.sh
    Untracked:  code/hg19MapStats.err
    Untracked:  code/hg19MapStats.out
    Untracked:  code/hg19MapStats.sh
    Untracked:  code/humanChromorder.sh
    Untracked:  code/humanFiles
    Untracked:  code/humanMultiCov.err
    Untracked:  code/humanMultiCov.out
    Untracked:  code/humanMultiCov.sh
    Untracked:  code/humanMultiCov255.err
    Untracked:  code/humanMultiCov255.out
    Untracked:  code/humanMultiCov255.sh
    Untracked:  code/humanMultiCovInclusive.err
    Untracked:  code/humanMultiCovInclusive.out
    Untracked:  code/humanMultiCov_inclusive.sh
    Untracked:  code/intersectAnno.err
    Untracked:  code/intersectAnno.out
    Untracked:  code/intersectAnnoExt.err
    Untracked:  code/intersectAnnoExt.out
    Untracked:  code/intersectLiftedPAS.sh
    Untracked:  code/leafcutter_merge_regtools_redo.py
    Untracked:  code/liftJunctionFiles.sh
    Untracked:  code/liftPAS19to38.sh
    Untracked:  code/liftoverPAShg19to38.err
    Untracked:  code/liftoverPAShg19to38.out
    Untracked:  code/log/
    Untracked:  code/make5percPeakbed.py
    Untracked:  code/makeFileID.py
    Untracked:  code/makeNuclearDapaplots.sh
    Untracked:  code/makeNuclearDapaplots_DF.sh
    Untracked:  code/makeNuclearPlots.err
    Untracked:  code/makeNuclearPlots.out
    Untracked:  code/makeNuclearPlotsDF.err
    Untracked:  code/makeNuclearPlotsDF.out
    Untracked:  code/makePheno.py
    Untracked:  code/makeSamplyGroupsChimp_TvN.py
    Untracked:  code/makeSamplyGroupsHuman_TvN.py
    Untracked:  code/mapRNAseqhg19.sh
    Untracked:  code/mapRNAseqhg19_newPipeline.sh
    Untracked:  code/maphg19.err
    Untracked:  code/maphg19.out
    Untracked:  code/maphg19.sh
    Untracked:  code/maphg19_new.err
    Untracked:  code/maphg19_new.out
    Untracked:  code/maphg19_sub.err
    Untracked:  code/maphg19_sub.out
    Untracked:  code/maphg19_subjunc.sh
    Untracked:  code/mediation_test.R
    Untracked:  code/merge.err
    Untracked:  code/mergeChimp3prime_inhg38.sh
    Untracked:  code/merge_leafcutter_clusters_redo.py
    Untracked:  code/mergeandBWRNAseq.sh
    Untracked:  code/mergeandsort_ChimpinHuman.err
    Untracked:  code/mergeandsort_ChimpinHuman.out
    Untracked:  code/mergedBam2BW.sh
    Untracked:  code/mergedbam2bw.err
    Untracked:  code/mergedbam2bw.out
    Untracked:  code/mergedbamRNAand2bw.err
    Untracked:  code/mergedbamRNAand2bw.out
    Untracked:  code/nameClusters.py
    Untracked:  code/namePeaks.py
    Untracked:  code/negativeMediation_montecarlo.R
    Untracked:  code/negativeMediation_montecarloDF.R
    Untracked:  code/nuclearTranscriptDTplot.err
    Untracked:  code/nuclearTranscriptDTplot.out
    Untracked:  code/numMultimap.py
    Untracked:  code/overlapMMandOrthoexon.sh
    Untracked:  code/overlapPAS.err
    Untracked:  code/overlapPAS.out
    Untracked:  code/overlapapaQTLPAS.sh
    Untracked:  code/overlapapaQTLPAS_extended.sh
    Untracked:  code/overlapapaQTLPAS_samples.sh
    Untracked:  code/parseHg38.py
    Untracked:  code/peak2PAS.py
    Untracked:  code/pheno2countonly.R
    Untracked:  code/postiveMediation_montecarlo.R
    Untracked:  code/postiveMediation_montecarlo_DF.R
    Untracked:  code/prepareAnnoLeafviz.err
    Untracked:  code/prepareAnnoLeafviz.out
    Untracked:  code/prepareCleanLiftedFC_5perc4LC.py
    Untracked:  code/prepareLeafvizAnno.sh
    Untracked:  code/preparePAS4lift.py
    Untracked:  code/prepare_phenotype_table.py
    Untracked:  code/primaryLift.err
    Untracked:  code/primaryLift.out
    Untracked:  code/primaryLift.sh
    Untracked:  code/processhg38exons.py
    Untracked:  code/quantJunc.sh
    Untracked:  code/quantJunc_TEST.sh
    Untracked:  code/quantJunc_removeBad.sh
    Untracked:  code/quantLiftedPAS.err
    Untracked:  code/quantLiftedPAS.out
    Untracked:  code/quantLiftedPAS.sh
    Untracked:  code/quatJunc.err
    Untracked:  code/quatJunc.out
    Untracked:  code/recChimpback2Human.err
    Untracked:  code/recChimpback2Human.out
    Untracked:  code/recLiftchim2human.sh
    Untracked:  code/revLift.err
    Untracked:  code/revLift.out
    Untracked:  code/revLiftPAShg38to19.sh
    Untracked:  code/reverseLift.sh
    Untracked:  code/runCheckReverseLift.sh
    Untracked:  code/runChimpDiffIso.sh
    Untracked:  code/runChimpDiffIsoDF.sh
    Untracked:  code/runCountNucleotides.err
    Untracked:  code/runCountNucleotides.out
    Untracked:  code/runCountNucleotides.sh
    Untracked:  code/runCountNucleotidesPantro6.err
    Untracked:  code/runCountNucleotidesPantro6.out
    Untracked:  code/runCountNucleotides_pantro6.sh
    Untracked:  code/runFilterNumChroms.sh
    Untracked:  code/runHumanDiffIso.sh
    Untracked:  code/runHumanDiffIsoDF.sh
    Untracked:  code/runNuclearDiffIso_DF.sh
    Untracked:  code/runNuclearDifffIso.sh
    Untracked:  code/runTotalDiffIso.sh
    Untracked:  code/run_Chimpleafcutter_ds.err
    Untracked:  code/run_Chimpleafcutter_ds.out
    Untracked:  code/run_Chimpverifybam.err
    Untracked:  code/run_Chimpverifybam.out
    Untracked:  code/run_Humanleafcutter_dF.err
    Untracked:  code/run_Humanleafcutter_dF.out
    Untracked:  code/run_Humanleafcutter_ds.err
    Untracked:  code/run_Humanleafcutter_ds.out
    Untracked:  code/run_Nuclearleafcutter_ds.err
    Untracked:  code/run_Nuclearleafcutter_ds.out
    Untracked:  code/run_Nuclearleafcutter_dsDF.err
    Untracked:  code/run_Nuclearleafcutter_dsDF.out
    Untracked:  code/run_Totalleafcutter_ds.err
    Untracked:  code/run_Totalleafcutter_ds.out
    Untracked:  code/run_chimpverifybam.sh
    Untracked:  code/run_verifyBam.sh
    Untracked:  code/run_verifybam.err
    Untracked:  code/run_verifybam.out
    Untracked:  code/slurm-62824013.out
    Untracked:  code/slurm-62825841.out
    Untracked:  code/slurm-62826116.out
    Untracked:  code/slurm-64108209.out
    Untracked:  code/slurm-64108521.out
    Untracked:  code/slurm-64108557.out
    Untracked:  code/snakePASChimp.err
    Untracked:  code/snakePASChimp.out
    Untracked:  code/snakePAShuman.out
    Untracked:  code/snakemake.batch
    Untracked:  code/snakemakeChimp.err
    Untracked:  code/snakemakeChimp.out
    Untracked:  code/snakemakeHuman.err
    Untracked:  code/snakemakeHuman.out
    Untracked:  code/snakemakePAS.batch
    Untracked:  code/snakemakePASFiltChimp.err
    Untracked:  code/snakemakePASFiltChimp.out
    Untracked:  code/snakemakePASFiltHuman.err
    Untracked:  code/snakemakePASFiltHuman.out
    Untracked:  code/snakemakePASchimp.batch
    Untracked:  code/snakemakePAShuman.batch
    Untracked:  code/snakemake_chimp.batch
    Untracked:  code/snakemake_human.batch
    Untracked:  code/snakemakefiltPAS.batch
    Untracked:  code/snakemakefiltPAS_chimp.sh
    Untracked:  code/snakemakefiltPAS_human.sh
    Untracked:  code/spliceSite2Fasta.py
    Untracked:  code/submit-snakemake-chimp.sh
    Untracked:  code/submit-snakemake-human.sh
    Untracked:  code/submit-snakemakePAS-chimp.sh
    Untracked:  code/submit-snakemakePAS-human.sh
    Untracked:  code/submit-snakemakefiltPAS-chimp.sh
    Untracked:  code/submit-snakemakefiltPAS-human.sh
    Untracked:  code/subset_diffisopheno.py
    Untracked:  code/subset_diffisopheno_Chimp_tvN.py
    Untracked:  code/subset_diffisopheno_Chimp_tvN_DF.py
    Untracked:  code/subset_diffisopheno_Huma_tvN.py
    Untracked:  code/subset_diffisopheno_Huma_tvN_DF.py
    Untracked:  code/subset_diffisopheno_Nuclear_HvC.py
    Untracked:  code/subset_diffisopheno_Nuclear_HvC_DF.py
    Untracked:  code/subset_diffisopheno_Total_HvC.py
    Untracked:  code/test
    Untracked:  code/test.txt
    Untracked:  code/threeprimeOrthoFC.out
    Untracked:  code/threeprimeOrthoFC.sh
    Untracked:  code/threeprimeOrthoFCcd.err
    Untracked:  code/transcriptDTplotsNuclear.sh
    Untracked:  code/transcriptDTplotsTotal.sh
    Untracked:  code/verifyBam4973.sh
    Untracked:  code/verifyBam4973inHuman.sh
    Untracked:  code/verifybam4973.err
    Untracked:  code/verifybam4973.out
    Untracked:  code/verifybam4973HumanMap.err
    Untracked:  code/verifybam4973HumanMap.out
    Untracked:  code/wrap_Chimpverifybam.err
    Untracked:  code/wrap_Chimpverifybam.out
    Untracked:  code/wrap_chimpverifybam.sh
    Untracked:  code/wrap_verifyBam.sh
    Untracked:  code/wrap_verifybam.err
    Untracked:  code/wrap_verifybam.out
    Untracked:  code/writeMergecode.py
    Untracked:  data/._.DS_Store
    Untracked:  data/._HC_filenames.txt
    Untracked:  data/._HC_filenames.txt.sb-4426323c-IKIs0S
    Untracked:  data/._HC_filenames.xlsx
    Untracked:  data/._MapPantro6_meta.txt
    Untracked:  data/._MapPantro6_meta.txt.sb-a5794dd2-Cskmlm
    Untracked:  data/._MapPantro6_meta.xlsx
    Untracked:  data/._OppositeSpeciesMap.txt
    Untracked:  data/._OppositeSpeciesMap.txt.sb-a5794dd2-mayWJf
    Untracked:  data/._OppositeSpeciesMap.xlsx
    Untracked:  data/._RNASEQ_metadata.txt
    Untracked:  data/._RNASEQ_metadata.txt.sb-4426323c-TE4ns3
    Untracked:  data/._RNASEQ_metadata.txt.sb-51f67ae1-HXp7Gq
    Untracked:  data/._RNASEQ_metadata_2Removed.txt
    Untracked:  data/._RNASEQ_metadata_2Removed.txt.sb-4426323c-a4lBwx
    Untracked:  data/._RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/._RNASEQ_metadata_stranded.txt
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-a5794dd2-D659m2
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-a5794dd2-ImNMoY
    Untracked:  data/._RNASEQ_metadata_stranded.txt.sb-e4bf31f0-ZGnGgl
    Untracked:  data/._RNASEQ_metadata_stranded.xlsx
    Untracked:  data/._TrialFiltersMeta.txt
    Untracked:  data/._TrialFiltersMeta.txt.sb-9845453e-R58Y0Q
    Untracked:  data/._metadata_HCpanel.txt
    Untracked:  data/._metadata_HCpanel.txt.sb-a3d92a2d-b9cYoF
    Untracked:  data/._metadata_HCpanel.txt.sb-a5794dd2-i594qs
    Untracked:  data/._metadata_HCpanel.txt.sb-f4823d1e-qihGek
    Untracked:  data/._metadata_HCpanel.xlsx
    Untracked:  data/._metadata_HCpanel_frompantro5.xlsx
    Untracked:  data/._~$RNASEQ_metadata.xlsx
    Untracked:  data/._~$metadata_HCpanel.xlsx
    Untracked:  data/._ï€¦.xlsx
    Untracked:  data/BaseComp/
    Untracked:  data/CompapaQTLpas/
    Untracked:  data/DNDS/
    Untracked:  data/DTmatrix/
    Untracked:  data/DiffExpression/
    Untracked:  data/DiffIso_Nuclear/
    Untracked:  data/DiffIso_Nuclear_DF/
    Untracked:  data/DiffIso_Total/
    Untracked:  data/DiffSplice/
    Untracked:  data/DiffSplice_liftedJunc/
    Untracked:  data/DiffSplice_removeBad/
    Untracked:  data/DominantPAS/
    Untracked:  data/DominantPAS_DF/
    Untracked:  data/EvalPantro5/
    Untracked:  data/HC_filenames.txt
    Untracked:  data/HC_filenames.xlsx
    Untracked:  data/Khan_prot/
    Untracked:  data/Li_eqtls/
    Untracked:  data/MapPantro6_meta.txt
    Untracked:  data/MapPantro6_meta.xlsx
    Untracked:  data/MapStats/
    Untracked:  data/NormalizedClusters/
    Untracked:  data/NuclearHvC/
    Untracked:  data/NuclearHvC_DF/
    Untracked:  data/OppositeSpeciesMap.txt
    Untracked:  data/OppositeSpeciesMap.xlsx
    Untracked:  data/OrthoExonBed/
    Untracked:  data/OverlapBenchmark/
    Untracked:  data/OverlappingPAS/
    Untracked:  data/PAS/
    Untracked:  data/PAS_SAF/
    Untracked:  data/PAS_doubleFilter/
    Untracked:  data/Peaks_5perc/
    Untracked:  data/Pheno_5perc/
    Untracked:  data/Pheno_5perc_DF_nuclear/
    Untracked:  data/Pheno_5perc_nuclear/
    Untracked:  data/Pheno_5perc_nuclear_old/
    Untracked:  data/Pheno_5perc_total/
    Untracked:  data/PhyloP/
    Untracked:  data/RNASEQ_metadata.txt
    Untracked:  data/RNASEQ_metadata_2Removed.txt
    Untracked:  data/RNASEQ_metadata_2Removed.xlsx
    Untracked:  data/RNASEQ_metadata_stranded.txt
    Untracked:  data/RNASEQ_metadata_stranded.txt.sb-e4bf31f0-ZGnGgl/
    Untracked:  data/RNASEQ_metadata_stranded.xlsx
    Untracked:  data/SignalSites/
    Untracked:  data/SignalSites_doublefilter/
    Untracked:  data/SpliceSite/
    Untracked:  data/TestMM2/
    Untracked:  data/TestMM2_PrimaryRead/
    Untracked:  data/TestMM2_SeondaryRead/
    Untracked:  data/TestMM2_quality/
    Untracked:  data/Test_FC_methods/
    Untracked:  data/Threeprime2Ortho/
    Untracked:  data/TotalHvC/
    Untracked:  data/TrialFiltersMeta.txt
    Untracked:  data/TwoBadSampleAnalysis/
    Untracked:  data/Wang_ribo/
    Untracked:  data/apaQTLGenes/
    Untracked:  data/bioGRID/
    Untracked:  data/chainFiles/
    Untracked:  data/cleanPeaks_anno/
    Untracked:  data/cleanPeaks_byspecies/
    Untracked:  data/cleanPeaks_lifted/
    Untracked:  data/files4viz_nuclear/
    Untracked:  data/files4viz_nuclear_DF/
    Untracked:  data/gviz/
    Untracked:  data/leafviz/
    Untracked:  data/liftover_files/
    Untracked:  data/mediation/
    Untracked:  data/mediation_DF/
    Untracked:  data/metadata_HCpanel.txt
    Untracked:  data/metadata_HCpanel.xlsx
    Untracked:  data/metadata_HCpanel_frompantro5.txt
    Untracked:  data/metadata_HCpanel_frompantro5.xlsx
    Untracked:  data/multimap/
    Untracked:  data/primaryLift/
    Untracked:  data/reverseLift/
    Untracked:  data/testQuant/
    Untracked:  data/~$RNASEQ_metadata.xlsx
    Untracked:  data/~$metadata_HCpanel.xlsx
    Untracked:  data/ï€¦.xlsx
    Untracked:  output/._.DS_Store
    Untracked:  output/dtPlots/
    Untracked:  projectNotes.Rmd
    Untracked:  proteinModelSet.Rmd

Unstaged changes:
    Modified:   analysis/DiffUsedIntronic.Rmd
    Modified:   analysis/ExploredAPA.Rmd
    Modified:   analysis/ExploredAPA_DF.Rmd
    Modified:   analysis/OppositeMap.Rmd
    Modified:   analysis/SpliceSiteStrength.Rmd
    Modified:   analysis/TotalVNuclearBothSpecies.Rmd
    Modified:   analysis/annotationInfo.Rmd
    Modified:   analysis/changeMisprimcut.Rmd
    Modified:   analysis/comp2apaQTLPAS.Rmd
    Modified:   analysis/correlationPhenos.Rmd
    Modified:   analysis/dAPAandapaQTL_DF.Rmd
    Modified:   analysis/establishCutoffs.Rmd
    Modified:   analysis/investigatePantro5.Rmd
    Modified:   analysis/multiMap.Rmd
    Modified:   analysis/speciesSpecific.Rmd
    Modified:   analysis/upsetter_DF.Rmd

</code></pre>
<p>
Note that any generated files, e.g.Â HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If youâ€™ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/706fa0d50038656a0ed1504362f62640173ba444/analysis/TrynewQuantMethods.Rmd" target="_blank">706fa0d</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-28
</td>
<td>
add mm exp code
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/3d4dd4722d951a6205c8db1fdf2d6a513b4b402e/docs/TrynewQuantMethods.html" target="_blank">3d4dd47</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f92b89c48e3cdd34486ac237f56bef2d9e8acd5f/analysis/TrynewQuantMethods.Rmd" target="_blank">f92b89c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-26
</td>
<td>
add ortho exon and new mm
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/TrynewQuantMethods.html" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/4e6bd5a3c3a21ba0871f3be089facb1568d8240a/analysis/TrynewQuantMethods.Rmd" target="_blank">4e6bd5a</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
<td>
fix mm
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/478a5047eb6cee6f9843e177ee573e232a31e722/docs/TrynewQuantMethods.html" target="_blank">478a504</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/646844c41c822c5a0f471cab08180de22a6e741e/analysis/TrynewQuantMethods.Rmd" target="_blank">646844c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-24
</td>
<td>
add write out
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/f0c8049f4ef5f1802bbfe198a505bf8589f4b29a/docs/TrynewQuantMethods.html" target="_blank">f0c8049</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f11a628267f91ffe25509d8245bd587b282f92c4/analysis/TrynewQuantMethods.Rmd" target="_blank">f11a628</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
<td>
plots for multimap
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/cd2bced4f0f75c4e1611c95c547084f8faa4a620/docs/TrynewQuantMethods.html" target="_blank">cd2bced</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/0e5531891c4b65ebbc9ae088c63cd4909559312d/analysis/TrynewQuantMethods.Rmd" target="_blank">0e55318</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
<td>
diagnose multimap
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/02f5c26ea000102158d8e7c657d820215c0c81f6/docs/TrynewQuantMethods.html" target="_blank">02f5c26</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/a4d172b4808bc19df1fbbde4bd5f90907c1ff465/analysis/TrynewQuantMethods.Rmd" target="_blank">a4d172b</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-20
</td>
<td>
add mm res
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/Comparative_APA/246d60c22c9418dce848d46bf9ce2808ad3386e4/docs/TrynewQuantMethods.html" target="_blank">246d60c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/b6ccdbcbc61bfb23d707cbf1be10f9a6b23e0b1c/analysis/TrynewQuantMethods.Rmd" target="_blank">b6ccdbc</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-19
</td>
<td>
add new quat
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>library(tidyverse)</code></pre>
<pre><code>â”€â”€ Attaching packages â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€ tidyverse 1.2.1 â”€â”€</code></pre>
<pre><code>âœ” ggplot2 3.1.1       âœ” purrr   0.3.2  
âœ” tibble  2.1.1       âœ” dplyr   0.8.0.1
âœ” tidyr   0.8.3       âœ” stringr 1.3.1  
âœ” readr   1.3.1       âœ” forcats 0.3.0  </code></pre>
<pre><code>â”€â”€ Conflicts â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€ tidyverse_conflicts() â”€â”€
âœ– dplyr::filter() masks stats::filter()
âœ– dplyr::lag()    masks stats::lag()</code></pre>
<pre class="r"><code>library(ggpubr)</code></pre>
<pre><code>Loading required package: magrittr</code></pre>
<pre><code>
Attaching package: &#39;magrittr&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:purrr&#39;:

    set_names</code></pre>
<pre><code>The following object is masked from &#39;package:tidyr&#39;:

    extract</code></pre>
<pre class="r"><code>library(VennDiagram)</code></pre>
<pre><code>Loading required package: grid</code></pre>
<pre><code>Loading required package: futile.logger</code></pre>
<pre><code>
Attaching package: &#39;VennDiagram&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:ggpubr&#39;:

    rotate</code></pre>
<pre class="r"><code>library(Matrix)</code></pre>
<pre><code>
Attaching package: &#39;Matrix&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:tidyr&#39;:

    expand</code></pre>
<p>I am worried about using featureCounts. It looks like there are reads that are not counted. In this analysis, I will look at correlation between counts as well as the specific locations I have been worried about.</p>
<p>First, I will do this in human nuclear with the final PAS.</p>
<p>This is the fixed strand. The</p>
<pre class="r"><code>Meta=read.table(&quot;../data/PAS_doubleFilter/PAS_10perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt&quot;,header = T)
ChimpMeta=read.table(&quot;../data/PAS_doubleFilter/PAS_doublefilter_either_ChimpCoordChimpUsage.sort.bed&quot;, col.names = c(&quot;chr&quot;,&quot;start&quot;,&quot;end&quot;, &quot;PAS&quot;, &quot;score&quot;, &quot;strand&quot;),stringsAsFactors = F) %&gt;% select(PAS)
HumanCountsFeatureCounts=read.table(&quot;../Human/data/CleanLiftedPeaks_FC/ALLPAS_postLift_LocParsed_Human_fixed.fc&quot;, header = T) %&gt;%  separate(Geneid, into=c(&quot;disc&quot;, &quot;PAS&quot;, &quot;chr2&quot;, &quot;start2&quot;, &quot;end2&quot;, &#39;strand&#39;, &#39;geneid&#39;), sep=&quot;:&quot;)  %&gt;% select(-contains(&quot;_T&quot;)) %&gt;% filter(PAS %in% Meta$PAS) 

ChimpCountsFeatureCounts=read.table(&quot;../Chimp/data/CleanLiftedPeaks_FC/ALLPAS_postLift_LocParsed_Chimp_fixed.fc&quot;, header = T) %&gt;%  separate(Geneid, into=c(&quot;disc&quot;, &quot;PAS&quot;, &quot;chr2&quot;, &quot;start2&quot;, &quot;end2&quot;, &#39;strand&#39;, &#39;geneid&#39;), sep=&quot;:&quot;)  %&gt;% select(-contains(&quot;_T&quot;)) %&gt;% right_join(ChimpMeta,by=&quot;PAS&quot;)</code></pre>
<p>First use the bedtools method:</p>
<pre class="bash"><code>mkdir ../data/testQuant

sbatch humanMultiCov.sh

sbatch chimpMultiCov.sh</code></pre>
<pre class="r"><code>HumanDFMulticov=read.table(&quot;../data/testQuant/Human_DF_PAS.txt&quot;)</code></pre>
<p>Compare:</p>
<pre class="r"><code># cor(a,b)

HumanCountsFeatureCounts_mat= as.matrix(HumanCountsFeatureCounts %&gt;% select(contains(&quot;_N&quot;)))

HumanDFMulticov_mat=as.matrix(HumanDFMulticov[,7:12])
#PAS correlation 
Human_DFcor= cor(HumanCountsFeatureCounts_mat,HumanDFMulticov_mat)</code></pre>
<p>I care about the diagonal for the matrix.</p>
<pre class="r"><code>Human_DFcordiag= diag(Human_DFcor)

Human_DFcordiag</code></pre>
<pre><code>[1] 0.9839297 0.9551134 0.9668683 0.9742895 0.9781832 0.9821528</code></pre>
<p>There are the correlations for individual.</p>
<pre class="r"><code>ChimpDFMulticov=read.table(&quot;../data/testQuant/Chimp_DF_PAS.txt&quot;)

ChimpCountsFeatureCounts_mat= as.matrix(ChimpCountsFeatureCounts %&gt;% select(contains(&quot;_N&quot;)))

ChimpDFMulticov_mat=as.matrix(ChimpDFMulticov[,7:12])
#PAS correlation 
Chimp_DFcor= cor(ChimpCountsFeatureCounts_mat,ChimpDFMulticov_mat)

Chimp_DFcordiag= diag(Chimp_DFcor)

Chimp_DFcordiag</code></pre>
<pre><code>[1] 0.9900335 0.9727290 0.9819526 0.9805695 0.9751067 0.9772205</code></pre>
<p>Compare species</p>
<pre class="r"><code>DF_Multi=as.data.frame(cbind(Chimp=Chimp_DFcordiag,Human=Human_DFcordiag)) %&gt;% gather(&quot;Species&quot;, &quot;Correlation&quot;)


ggplot(DF_Multi,aes(x=Species, y=Correlation, fill=Species)) + geom_boxplot() + geom_jitter() + stat_compare_means() + labs(title=&quot;Double filtered Final PAS correlation \nbetween feature counts and multiCov&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-8-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-8-1">
Past versions of unnamed-chunk-8-1.png
</button>
</p>
<div id="fig-unnamed-chunk-8-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/246d60c22c9418dce848d46bf9ce2808ad3386e4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-8-1.png" target="_blank">246d60c</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Difference in the matricies:</p>
<pre class="r"><code>HumanDiff= HumanCountsFeatureCounts_mat-HumanDFMulticov_mat
ChimpDiff=ChimpCountsFeatureCounts_mat-ChimpDFMulticov_mat</code></pre>
<p>Extract the non</p>
<pre class="r"><code>nnzero(HumanDiff)</code></pre>
<pre><code>[1] 110127</code></pre>
<pre class="r"><code>sum(HumanDiff)</code></pre>
<pre><code>[1] -1787591</code></pre>
<pre class="r"><code>HumanDiff_g=as.data.frame(HumanDiff) %&gt;% gather(&quot;Ind&quot;, &quot;count&quot;)

ggplot(HumanDiff_g, aes(x=Ind, y=count,col=Ind))+ geom_boxplot() + labs(y=&quot;FeatureCount - Bedtools&quot;, title=&quot;Human Difference in Counts for final PAS&quot;)+ theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-10-1">
Past versions of unnamed-chunk-10-1.png
</button>
</p>
<div id="fig-unnamed-chunk-10-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-10-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/02f5c26ea000102158d8e7c657d820215c0c81f6/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-10-1.png" target="_blank">02f5c26</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Look at the examples with the min values.</p>
<pre class="r"><code>HumanDiff_rowS= rowSums(HumanDiff)
#min:  
minDiff=which(HumanDiff_rowS==min(HumanDiff_rowS))

#36964 (look at this row in the df)
HumanCountsFeatureCounts[minDiff,]</code></pre>
<pre><code>      disc         PAS chr2  start2    end2 strand    geneid  Chr   Start
36964 Both human299134 chr7 5527138 5527338      + ACTB_utr3 chr7 5527138
          End Strand Length X18498_N X18499_N X18502_N X18504_N X18510_N
36964 5527338      +    201     3833     4635     3328     2983     3460
      X18523_N
36964     4132</code></pre>
<pre class="r"><code>HumanDFMulticov[minDiff,]</code></pre>
<pre><code>        V1      V2      V3          V4        V5 V6    V7    V8    V9
36964 chr7 5527138 5527338 human299134 0.9833333  - 36243 45135 31796
        V10   V11   V12
36964 32458 33604 39187</code></pre>
<pre class="r"><code>nnzero(ChimpDiff)</code></pre>
<pre><code>[1] 112291</code></pre>
<pre class="r"><code>sum(ChimpDiff)</code></pre>
<pre><code>[1] -1904816</code></pre>
<pre class="r"><code>ChimpDiff_g=as.data.frame(ChimpDiff) %&gt;% gather(&quot;Ind&quot;, &quot;count&quot;)

ggplot(ChimpDiff_g, aes(x=Ind, y=count,col=Ind))+ geom_boxplot() + labs(y=&quot;FeatureCount - Bedtools&quot;, title=&quot;Chimp Difference in Counts for final PAS&quot;) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-12-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-12-1">
Past versions of unnamed-chunk-12-1.png
</button>
</p>
<div id="fig-unnamed-chunk-12-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-12-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/02f5c26ea000102158d8e7c657d820215c0c81f6/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-12-1.png" target="_blank">02f5c26</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Look at the examples with the min values.</p>
<pre class="r"><code>ChimpDiff_rowS= rowSums(ChimpDiff)
#min:  
minDiffChimp=which(ChimpDiff_rowS==min(ChimpDiff_rowS))

ChimpCountsFeatureCounts[minDiffChimp,]</code></pre>
<pre><code>      disc         PAS chr2  start2    end2 strand    geneid  Chr   Start
36910 Both human299134 chr7 5527138 5527338      + ACTB_utr3 chr7 5663709
          End Strand Length X18358_N X3622_N X3659_N X4973_N pt30_N pt91_N
36910 5663909      +    201     3626    7146    5153    6572   8693   7306</code></pre>
<pre class="r"><code>ChimpDFMulticov[minDiffChimp,]</code></pre>
<pre><code>        V1      V2      V3          V4    V5 V6    V7    V8    V9   V10
36910 chr7 5663709 5663909 human299134 0.985  - 31910 63880 44470 49514
        V11   V12
36910 73935 60607</code></pre>
<p>This shows that in general counts are lower in the features counts.</p>
<p>Examples:</p>
<p>human188040</p>
<pre class="r"><code>Meta %&gt;% filter(PAS==&quot;human188040&quot;)</code></pre>
<pre><code>          PAS  disc  gene loc  chr     start       end Chimp Human
1 human188040 Human PLCL1 cds chr2 198084933 198085133 0.095 0.025
  strandFix
1         +</code></pre>
<pre class="r"><code>ChimpDFMulticov %&gt;% filter(V4==&quot;human188040&quot;)</code></pre>
<pre><code>     V1       V2       V3          V4    V5 V6 V7 V8 V9 V10 V11 V12
1 chr2B 84660923 84661123 human188040 0.095  +  0  0  5   1   0   0</code></pre>
<pre class="r"><code>ChimpCountsFeatureCounts %&gt;% filter(PAS==&quot;human188040&quot;)</code></pre>
<pre><code>   disc         PAS chr2    start2      end2 strand    geneid   Chr
1 Human human188040 chr2 198084933 198085133      - PLCL1_cds chr2B
     Start      End Strand Length X18358_N X3622_N X3659_N X4973_N pt30_N
1 84660923 84661123      -    201        0       0       5       1      0
  pt91_N
1      0</code></pre>
<pre class="r"><code>HumanDFMulticov %&gt;% filter(V4==&quot;human188040&quot;)</code></pre>
<pre><code>    V1        V2        V3          V4    V5 V6 V7 V8 V9 V10 V11 V12
1 chr2 198084933 198085133 human188040 0.025  +  0  2  6   3   0   2</code></pre>
<pre class="r"><code>HumanCountsFeatureCounts %&gt;% filter(PAS==&quot;human188040&quot;)</code></pre>
<pre><code>   disc         PAS chr2    start2      end2 strand    geneid  Chr
1 Human human188040 chr2 198084933 198085133      - PLCL1_cds chr2
      Start       End Strand Length X18498_N X18499_N X18502_N X18504_N
1 198084933 198085133      -    201        0        2        6        3
  X18510_N X18523_N
1        0        2</code></pre>
<p>Same counts here. This is why it was not found in chimp. It seems it wouldnt pass the non zero filter.<br />
chimp171173</p>
<pre class="r"><code>Meta %&gt;% filter(PAS==&quot;chimp171173&quot;)</code></pre>
<pre><code>          PAS  disc gene  loc  chr   start     end     Chimp        Human
1 chimp171173 Chimp ADI1 utr3 chr2 3498436 3498639 0.5358333 0.0008333333
  strandFix
1         -</code></pre>
<pre class="r"><code>HumanDFMulticov %&gt;% filter(V4==&quot;chimp171173&quot;) </code></pre>
<pre><code>    V1      V2      V3          V4           V5 V6  V7 V8  V9 V10 V11 V12
1 chr2 3498436 3498639 chimp171173 0.0008333333  - 140 87 105  47 250 525</code></pre>
<pre class="r"><code>HumanCountsFeatureCounts %&gt;% filter(PAS==&quot;chimp171173&quot;)</code></pre>
<pre><code>   disc         PAS chr2  start2    end2 strand    geneid  Chr   Start
1 Chimp chimp171173 chr2 3498436 3498639      + ADI1_utr3 chr2 3498436
      End Strand Length X18498_N X18499_N X18502_N X18504_N X18510_N
1 3498639      +    204        0        0        0        0        0
  X18523_N
1        0</code></pre>
<p>chimp171172</p>
<pre class="r"><code>Meta %&gt;% filter(PAS==&quot;chimp171172&quot;)</code></pre>
<pre><code>          PAS  disc gene  loc  chr   start     end Chimp Human strandFix
1 chimp171172 Chimp ADI1 utr3 chr2 3497907 3498106 0.155     0         -</code></pre>
<pre class="r"><code>HumanDFMulticov %&gt;% filter(V4==&quot;chimp171172&quot;) </code></pre>
<pre><code>    V1      V2      V3          V4 V5 V6 V7 V8 V9 V10 V11 V12
1 chr2 3497907 3498106 chimp171172  0  - 24  9 10  15  27 110</code></pre>
<pre class="r"><code>HumanCountsFeatureCounts %&gt;% filter(PAS==&quot;chimp171172&quot;)</code></pre>
<pre><code>   disc         PAS chr2  start2    end2 strand    geneid  Chr   Start
1 Chimp chimp171172 chr2 3497907 3498106      + ADI1_utr3 chr2 3497907
      End Strand Length X18498_N X18499_N X18502_N X18504_N X18510_N
1 3498106      +    200        0        0        0        0        0
  X18523_N
1        0</code></pre>
<p>This is fixed. (ADI1)</p>
<p>LAT human124355</p>
<pre class="r"><code>Meta %&gt;% filter(PAS==&quot;human124355&quot;)</code></pre>
<pre><code>          PAS  disc gene  loc   chr    start      end     Chimp Human
1 human124355 Human  LAT utr3 chr16 28990265 28990465 0.4241667  0.52
  strandFix
1         +</code></pre>
<pre class="r"><code>ChimpDFMulticov %&gt;% filter(V4==&quot;human124355&quot;)</code></pre>
<pre><code>                    V1     V2     V3          V4        V5 V6 V7 V8 V9 V10
1 chrUn_NW_019937196v1 380529 380729 human124355 0.4241667  - 19 26 43  23
  V11 V12
1  34  25</code></pre>
<pre class="r"><code>ChimpCountsFeatureCounts %&gt;% filter(PAS==&quot;human124355&quot;)</code></pre>
<pre><code>   disc         PAS  chr2   start2     end2 strand   geneid
1 Human human124355 chr16 28990265 28990465      - LAT_utr3
                   Chr  Start    End Strand Length X18358_N X3622_N
1 chrUn_NW_019937196v1 380529 380729      +    201       19      23
  X3659_N X4973_N pt30_N pt91_N
1      43      23     32     24</code></pre>
<pre class="r"><code>HumanDFMulticov %&gt;% filter(V4==&quot;human124355&quot;)</code></pre>
<pre><code>     V1       V2       V3          V4   V5 V6 V7 V8 V9 V10 V11 V12
1 chr16 28990265 28990465 human124355 0.52  + 50 10 28  18  33  32</code></pre>
<pre class="r"><code>HumanCountsFeatureCounts %&gt;% filter(PAS==&quot;human124355&quot;)</code></pre>
<pre><code>   disc         PAS  chr2   start2     end2 strand   geneid   Chr    Start
1 Human human124355 chr16 28990265 28990465      - LAT_utr3 chr16 28990265
       End Strand Length X18498_N X18499_N X18502_N X18504_N X18510_N
1 28990465      -    201       50       10       28       16       32
  X18523_N
1       31</code></pre>
<p>Makes sense. Discovered only in human because on contig. Counting is ok.</p>
<p>Compare the PAS discovered in human:</p>
<pre class="r"><code>humanDisc=Meta %&gt;% filter(disc==&quot;Human&quot;)

ChimpFC_human= ChimpCountsFeatureCounts %&gt;% filter(PAS %in% humanDisc$PAS)
ChimpMC_human= ChimpDFMulticov %&gt;% filter(V4 %in% humanDisc$PAS)

#correlation  
ChimpFC_human_mat= as.matrix(ChimpFC_human[,13:18])
ChimpMC_human_mat=as.matrix(ChimpMC_human[,7:12])
#PAS correlation 
Chimp_DFcor_Humandisc= cor(ChimpFC_human_mat,ChimpMC_human_mat)

Chimp_DFcor_Humandiscdiag= diag(Chimp_DFcor_Humandisc)

Chimp_DFcor_Humandiscdiag</code></pre>
<pre><code>[1] 0.9994462 0.9949149 0.9990672 0.9974558 0.9984791 0.9988820</code></pre>
<p>Is the multicov higher?</p>
<pre class="r"><code>ChimpHumanDiscDiff= ChimpFC_human_mat-ChimpMC_human_mat

nnzero(ChimpHumanDiscDiff)</code></pre>
<pre><code>[1] 12166</code></pre>
<pre class="r"><code>sum(ChimpHumanDiscDiff)</code></pre>
<pre><code>[1] -212818</code></pre>
<pre class="r"><code>summary(ChimpHumanDiscDiff)</code></pre>
<pre><code>    X18358_N            X3622_N             X3659_N         
 Min.   :-2248.000   Min.   :-2444.000   Min.   :-1658.000  
 1st Qu.:   -1.000   1st Qu.:   -2.000   1st Qu.:   -1.000  
 Median :    0.000   Median :    0.000   Median :    0.000  
 Mean   :   -8.554   Mean   :   -8.259   Mean   :   -6.238  
 3rd Qu.:    0.000   3rd Qu.:    0.000   3rd Qu.:    0.000  
 Max.   :  345.000   Max.   :  198.000   Max.   :  213.000  
    X4973_N             pt30_N              pt91_N         
 Min.   :-2526.00   Min.   :-2935.000   Min.   :-2713.000  
 1st Qu.:   -2.00   1st Qu.:   -2.000   1st Qu.:   -2.000  
 Median :    0.00   Median :    0.000   Median :    0.000  
 Mean   :   -6.81   Mean   :   -8.015   Mean   :   -8.682  
 3rd Qu.:    0.00   3rd Qu.:    0.000   3rd Qu.:    0.000  
 Max.   :  192.00   Max.   :  134.000   Max.   :  119.000  </code></pre>
<pre class="r"><code>ChimpHumanDiscDiff_rowS= rowSums(ChimpHumanDiscDiff)
#min:  
minDiffChimpinH=which(ChimpHumanDiscDiff_rowS==min(ChimpHumanDiscDiff_rowS))

ChimpFC_human[minDiffChimpinH,]</code></pre>
<pre><code>     disc       PAS chr2   start2     end2 strand    geneid  Chr    Start
113 Human human8304 chr1 44778707 44778907      - RPS8_utr3 chr1 43588450
         End Strand Length X18358_N X3622_N X3659_N X4973_N pt30_N pt91_N
113 43588650      -    201      400     421     239     448    627    602</code></pre>
<pre class="r"><code>ChimpMC_human[minDiffChimpinH,]</code></pre>
<pre><code>      V1       V2       V3        V4  V5 V6   V7   V8   V9  V10  V11  V12
113 chr1 43588450 43588650 human8304 0.5  + 2648 2534 1757 2974 3562 3315</code></pre>
<p>chr 1 (.5X) RPS8 (snoRNAs near- could be problem )</p>
<p>Redo with different map quality</p>
<p>-q 255 this will not count multimapping reads.</p>
<pre class="bash"><code>sbatch chimpMultiCov255.sh
sbatch humanMultiCov255.sh
</code></pre>
<pre class="r"><code>HumanDFMulticov255=read.table(&quot;../data/testQuant/Human_DF_PAS_255.txt&quot;)
ChimpDFMulticov255=read.table(&quot;../data/testQuant/Chimp_DF_PAS_255.txt&quot;)


HumanDFMulticov255_mat=as.matrix(HumanDFMulticov255[,7:12])


#compare this to the earlier one    

HumanDiffMulti=HumanDFMulticov_mat- HumanDFMulticov255_mat

HumanDiffMulti_g=as.data.frame(HumanDiffMulti) %&gt;% gather(&quot;Ind&quot;, &quot;Count&quot;)

ggplot(HumanDiffMulti_g,aes(x=Ind, y=Count, col=Ind)) + geom_boxplot() + labs(y=&quot;With Multimap - No multimap&quot;, title=&quot;Difference in PAS read count with bedtools multimap filters&quot;, x=&quot;Human Individuals&quot;) +theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-22-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-22-1">
Past versions of unnamed-chunk-22-1.png
</button>
</p>
<div id="fig-unnamed-chunk-22-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-22-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/02f5c26ea000102158d8e7c657d820215c0c81f6/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-22-1.png" target="_blank">02f5c26</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#compare to FC  

HumanDiffw225= HumanCountsFeatureCounts_mat - HumanDFMulticov255_mat
HumanDiffw225_g=as.data.frame(HumanDiffw225) %&gt;% gather(&quot;Ind&quot;, &quot;Count&quot;)
ggplot(HumanDiffw225_g,aes(x=Ind, y=Count, col=Ind)) + geom_boxplot() + labs(x=&quot;Human Individual&quot;, y=&quot;Feature counts = Bedtools (no multimap)&quot;, title=&quot;Difference in PAS read count between featurecounts\n and bedtools without multimap&quot;) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-23-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-23-1">
Past versions of unnamed-chunk-23-1.png
</button>
</p>
<div id="fig-unnamed-chunk-23-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-23-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Check examples:</p>
<pre class="r"><code>Meta %&gt;% filter(PAS==&quot;chimp171172&quot;)</code></pre>
<pre><code>          PAS  disc gene  loc  chr   start     end Chimp Human strandFix
1 chimp171172 Chimp ADI1 utr3 chr2 3497907 3498106 0.155     0         -</code></pre>
<pre class="r"><code>HumanDFMulticov %&gt;% filter(V4==&quot;chimp171172&quot;) </code></pre>
<pre><code>    V1      V2      V3          V4 V5 V6 V7 V8 V9 V10 V11 V12
1 chr2 3497907 3498106 chimp171172  0  - 24  9 10  15  27 110</code></pre>
<pre class="r"><code>HumanDFMulticov255 %&gt;% filter(V4==&quot;chimp171172&quot;) </code></pre>
<pre><code>    V1      V2      V3          V4 V5 V6 V7 V8 V9 V10 V11 V12
1 chr2 3497907 3498106 chimp171172  0  -  1  3  0   0   0   0</code></pre>
<pre class="r"><code>HumanCountsFeatureCounts %&gt;% filter(PAS==&quot;chimp171172&quot;)</code></pre>
<pre><code>   disc         PAS chr2  start2    end2 strand    geneid  Chr   Start
1 Chimp chimp171172 chr2 3497907 3498106      + ADI1_utr3 chr2 3497907
      End Strand Length X18498_N X18499_N X18502_N X18504_N X18510_N
1 3498106      +    200        0        0        0        0        0
  X18523_N
1        0</code></pre>
<pre class="r"><code>ChimpDFMulticov %&gt;% filter(V4==&quot;chimp171172&quot;) </code></pre>
<pre><code>     V1      V2      V3          V4    V5 V6 V7 V8 V9 V10 V11 V12
1 chr2A 3477781 3477981 chimp171172 0.155  - 35 25 32  15  31  24</code></pre>
<pre class="r"><code>ChimpDFMulticov255 %&gt;% filter(V4==&quot;chimp171172&quot;) </code></pre>
<pre><code>     V1      V2      V3          V4    V5 V6 V7 V8 V9 V10 V11 V12
1 chr2A 3477781 3477981 chimp171172 0.155  - 35 25 32  15  31  24</code></pre>
<pre class="r"><code>ChimpCountsFeatureCounts %&gt;% filter(PAS==&quot;chimp171172&quot;)</code></pre>
<pre><code>   disc         PAS chr2  start2    end2 strand    geneid   Chr   Start
1 Chimp chimp171172 chr2 3497907 3498106      + ADI1_utr3 chr2A 3477781
      End Strand Length X18358_N X3622_N X3659_N X4973_N pt30_N pt91_N
1 3477981      +    201       35      25      32      15     30     24</code></pre>
<p>Ok it seems like maybe multimapping accounts for some of the differece.</p>
<p>Try including the multi map reads in feature counts. I will test the -M flag and the â€“primary flag to compare. The SAF file has all of the PAS.</p>
<pre class="bash"><code>sbatch BothFCMM.sh </code></pre>
<p>Grep the results: Chimp:chimp171172:chr2:3497907:3498106:+:ADI1_utr3 chr2 3497907 3498106 + 200 23 6 10 15 27 110</p>
<p>So this is the problem. The IGV.</p>
<div id="inclusive" class="section level2">
<h2>Inclusive</h2>
<p>Test also on the original calls. To see if it can handle this one.</p>
<p>I need to change bed to include both positive and negative. I can do this with python.</p>
<pre class="bash"><code>python bed2Bedbothstrand.py ../Human/data/inclusivePeaks/human_APApeaks.ALLChrom.bed ../data/testQuant/Human_ALLpeaks.Bothstrand.bed

python bed2Bedbothstrand.py ../Chimp/data/inclusivePeaks/chimp_APApeaks.ALLChrom.bed ../data/testQuant/Chimp_ALLpeaks.Bothstrand.bed

sbatch humanMultiCov_inclusive.sh
sbatch chimpMultiCovInclusive.sh


#look at difference with the -O flag  


sbatch BothFCnewOInclusive.sh</code></pre>
<p>All counts are 0. Fix this.</p>
</div>
<div id="address-multimaping-problem" class="section level2">
<h2>Address multimaping problem</h2>
<p>it looks like the problem is multimapping. I want to get rid of PAS that multi map in one species only. To do this I want to compare the featurecounts with and without the -M flag.</p>
<pre class="bash"><code>
sbatch BothFCMM.sh 


python fixFChead_bothfrac.py ../data/testQuant/ALLPAS_postLift_LocParsed_Human_BothFrac_Mflag ../data/testQuant/ALLPAS_postLift_LocParsed_Human_BothFrac_Mflag_fixed.fc  
</code></pre>
<p>Only look at the PAS in the double filtered for now. I want to know how many I would lose in both frac.</p>
<pre class="r"><code>HumanCountsNoMM=read.table(&quot;../Human/data/CleanLiftedPeaks_FC/ALLPAS_postLift_LocParsed_Human_fixed.fc&quot;, header = T) %&gt;%  separate(Geneid, into=c(&quot;disc&quot;, &quot;PAS&quot;, &quot;chr2&quot;, &quot;start2&quot;, &quot;end2&quot;, &#39;strand&#39;, &#39;geneid&#39;), sep=&quot;:&quot;)%&gt;% filter(PAS %in% Meta$PAS) 

HC_noMM=rowSums(as.matrix(HumanCountsNoMM[,13:24]) )

HumanCountMM=read.table(&quot;../data/testQuant/ALLPAS_postLift_LocParsed_Human_BothFrac_Mflag_fixed.fc&quot;, header = T)%&gt;%  separate(Geneid, into=c(&quot;disc&quot;, &quot;PAS&quot;, &quot;chr2&quot;, &quot;start2&quot;, &quot;end2&quot;, &#39;strand&#39;, &#39;geneid&#39;), sep=&quot;:&quot;) %&gt;% filter(PAS %in% Meta$PAS) 


HC_MM=rowSums(as.matrix(HumanCountMM[,13:24]) )

HC_diff=HC_MM-HC_noMM</code></pre>
<pre class="r"><code>plot(HC_diff,  main=&quot;Difference between featurecounts with and without -M flag&quot;, ylab=&quot;Sum Human multimap - Sum Human no multimap&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-30-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-30-1">
Past versions of unnamed-chunk-30-1.png
</button>
</p>
<div id="fig-unnamed-chunk-30-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-30-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>12 libraries. Lets say it needs to have at least 10 in each or 120</p>
<pre class="r"><code>sum(HC_diff&gt;120)</code></pre>
<pre><code>[1] 999</code></pre>
<pre class="r"><code>ChimpCountsNoMM=read.table(&quot;../Chimp/data/CleanLiftedPeaks_FC/ALLPAS_postLift_LocParsed_Chimp_fixed.fc&quot;, header = T)  %&gt;%  separate(Geneid, into=c(&quot;disc&quot;, &quot;PAS&quot;, &quot;chr2&quot;, &quot;start2&quot;, &quot;end2&quot;, &#39;strand&#39;, &#39;geneid&#39;), sep=&quot;:&quot;) %&gt;% right_join(ChimpMeta,by=&quot;PAS&quot;)

CC_noMM=rowSums(as.matrix(ChimpCountsNoMM[,13:24]) )

ChimpCountMM=read.table(&quot;../data/testQuant/ALLPAS_postLift_LocParsed_Chimp_BothFrac_Mflag&quot;, header = T) %&gt;%  separate(Geneid, into=c(&quot;disc&quot;, &quot;PAS&quot;, &quot;chr2&quot;, &quot;start2&quot;, &quot;end2&quot;, &#39;strand&#39;, &#39;geneid&#39;), sep=&quot;:&quot;)  %&gt;% right_join(ChimpMeta,by=&quot;PAS&quot;)


CC_MM=rowSums(as.matrix(ChimpCountMM[,13:24]) )



CC_diff=CC_MM-CC_noMM</code></pre>
<pre class="r"><code>plot(CC_diff,main=&quot;Difference between featurecounts with and without -M flag&quot;, ylab=&quot;Sum Chimp multimap - Sum Chimp no multimap&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-33-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-33-1">
Past versions of unnamed-chunk-33-1.png
</button>
</p>
<div id="fig-unnamed-chunk-33-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-33-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>sum(HC_diff&gt;120)</code></pre>
<pre><code>[1] 999</code></pre>
<pre class="r"><code>humanMM=Human=HumanCountsNoMM[which(HC_diff&gt;120),2]
chimpMM=ChimpCountsNoMM[which(CC_diff&gt;120),2]

bothMM=intersect(humanMM,chimpMM)

overlap &lt;- intersect(humanMM, chimpMM) 
H.unique &lt;- setdiff(humanMM,chimpMM) # n 1st NOT 2nd
C.unique &lt;- setdiff(chimpMM,humanMM)
full.list &lt;- unique(c(humanMM,chimpMM))</code></pre>
<pre class="r"><code>grid.newpage()
venn.plot &lt;- draw.pairwise.venn(area1 = length(humanMM),
                           area2 = length(chimpMM),
                           cross.area = length(overlap),
                           c(&quot;Human&quot;, &quot;Chimp&quot;), scaled = TRUE,
                           fill = c(&quot;green&quot;, &quot;blue&quot;),
                           cex = 1.5,
                           cat.cex = 1.5,
                           cat.pos = c(320, 25),
                           cat.dist = .05) </code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-36-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-36-1">
Past versions of unnamed-chunk-36-1.png
</button>
</p>
<div id="fig-unnamed-chunk-36-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-36-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>total remove if this filter:</p>
<pre class="r"><code>length(full.list)</code></pre>
<pre><code>[1] 1528</code></pre>
<p>Test a range of filters:</p>
<p>I will test how many PAS we would lose if we filtered at each value. I will see if there is natural cuttoff.</p>
<pre class="r"><code>filterRange=seq(0,500000,by=100)
ChimpDiffN=c()
HumanDiffN=c()

for (i in filterRange){
  ChimpDiffN= c(ChimpDiffN, sum(CC_diff&gt;i))
  HumanDiffN= c(HumanDiffN, sum(HC_diff&gt;i))
}


LostPASdf=as.data.frame(cbind(Filter=filterRange, Chimp=ChimpDiffN, Human=HumanDiffN))
#%&gt;% mutate(All=42318)
LostPASdf_g= LostPASdf %&gt;% gather(&quot;Species&quot;,&quot;PAS_lost&quot;, -Filter)</code></pre>
<p>Plot:</p>
<pre class="r"><code>ggplot(LostPASdf_g, aes(x=Filter, y=log10(PAS_lost),col=Species)) + geom_point(alpha=.5) + labs(y=&quot;log10(PAS lost due to filter)&quot;, x=&quot;Filter for difference in read count due to multimapping&quot;, title=&quot;PAS lost due to multimap filter by different filter&quot;) + scale_color_brewer(palette = &quot;Dark2&quot;)+geom_vline(xintercept = 20000) + annotate(&quot;text&quot;, label=&quot;Difference of 20,000 \ncounts across individuals&quot;, x=120000,y=3)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-39-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-39-1">
Past versions of unnamed-chunk-39-1.png
</button>
</p>
<div id="fig-unnamed-chunk-39-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-39-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggplot(LostPASdf_g, aes(x=log10(Filter), y=log10(PAS_lost),col=Species)) + geom_point(alpha=.5) + labs(y=&quot;log10(PAS lost due to filter)&quot;, x=&quot;log10(Filter for difference in read count due to multimapping)&quot;, title=&quot;PAS lost due to multimap filter by different filter&quot;) + scale_color_brewer(palette = &quot;Dark2&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-40-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-40-1">
Past versions of unnamed-chunk-40-1.png
</button>
</p>
<div id="fig-unnamed-chunk-40-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-40-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/cd2bced4f0f75c4e1611c95c547084f8faa4a620/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-40-1.png" target="_blank">cd2bced</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Look at proportion.</p>
<pre class="r"><code>LostPASdfPROP=LostPASdf_g%&gt;% mutate(All=42318, propLeft=PAS_lost/All, PropRemain=1-propLeft)</code></pre>
<pre class="r"><code>ggplot(LostPASdfPROP, aes(x=Filter, y=PropRemain,col=Species)) + geom_point(alpha=.3)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-42-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-42-1">
Past versions of unnamed-chunk-42-1.png
</button>
</p>
<div id="fig-unnamed-chunk-42-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-42-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/cd2bced4f0f75c4e1611c95c547084f8faa4a620/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-42-1.png" target="_blank">cd2bced</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Zoom in. Look at filter over 1000</p>
<pre class="r"><code>LostPASdfPROP_filt=LostPASdfPROP %&gt;% filter(Filter&gt;1000)


ggplot(LostPASdfPROP_filt, aes(x=Filter, y=PropRemain,col=Species)) + geom_point(alpha=.3) + geom_vline(xintercept = 20000) +  annotate(&quot;text&quot;, label=&quot;Difference of 20,000 \ncounts across individuals&quot;, x=120000,y=.998) + scale_color_brewer(palette = &quot;Dark2&quot;) + labs(x=&quot;Filter for difference in reads&quot;, y=&quot;Proportion of PAS remaining&quot;, title=&quot;Proportion of PAS remaining by species after multimap filter&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-43-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-43-1">
Past versions of unnamed-chunk-43-1.png
</button>
</p>
<div id="fig-unnamed-chunk-43-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-43-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Use this filter to make the boxplot</p>
<pre class="r"><code>humanMM2=Human=HumanCountsNoMM[which(HC_diff&gt;20000),2]
chimpMM2=ChimpCountsNoMM[which(CC_diff&gt;20000),2]

bothMM2=intersect(humanMM2,chimpMM2)
overlap2 &lt;- intersect(humanMM2, chimpMM2) 
grid.newpage()
venn.plot &lt;- draw.pairwise.venn(area1 = length(humanMM2),
                           area2 = length(chimpMM2),
                           cross.area = length(overlap2),
                           c(&quot;Human&quot;, &quot;Chimp&quot;), scaled = TRUE,
                           fill = c(&quot;orange&quot;, &quot;green&quot;),
                           cex = 1.5,
                           cat.cex = 1.5,
                           cat.pos = c(320, 25),
                           cat.dist = .05) </code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-44-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-44-1">
Past versions of unnamed-chunk-44-1.png
</button>
</p>
<div id="fig-unnamed-chunk-44-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-44-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>expect to loose chimp171173 in human</p>
<pre class="r"><code>HumanCountsNoMM[HumanCountsNoMM$PAS==&quot;chimp171173&quot;,]</code></pre>
<pre><code>       disc         PAS chr2  start2    end2 strand    geneid  Chr   Start
22206 Chimp chimp171173 chr2 3498436 3498639      + ADI1_utr3 chr2 3498436
          End Strand Length X18498_N X18498_T X18499_N X18499_T X18502_N
22206 3498639      +    204        0        0        0        0        0
      X18502_T X18504_N X18504_T X18510_N X18510_T X18523_N X18523_T
22206        0        0        0        0        1        0        0</code></pre>
<pre class="r"><code>HC_diff[22206]</code></pre>
<pre><code>[1] 3808</code></pre>
<p>Should it be the proportion of reads lost?</p>
<p>I need the sum for the reads with and without multimapping at each PAS. Proportion of reads in the mulimap set</p>
<pre class="r"><code>HumanPASMultiMap=as.data.frame(cbind(PAS=HumanCountMM$PAS,WithMultiMap=HC_MM, NoMultiMap=HC_noMM)) 
HumanPASMultiMap$WithMultiMap=as.numeric(as.character(HumanPASMultiMap$WithMultiMap))
HumanPASMultiMap$NoMultiMap=as.numeric(as.character(HumanPASMultiMap$NoMultiMap))
HumanPASMultiMapProp= HumanPASMultiMap%&gt;% mutate(Difference=WithMultiMap-NoMultiMap, PercentLost=Difference/WithMultiMap)</code></pre>
<p>check chimp171173</p>
<pre class="r"><code>HumanPASMultiMapProp %&gt;% filter(PAS==&quot;chimp171173&quot;)</code></pre>
<pre><code>          PAS WithMultiMap NoMultiMap Difference PercentLost
1 chimp171173         3809          1       3808   0.9997375</code></pre>
<p>Plot<br />
Color top 10%</p>
<pre class="r"><code>x=quantile(HumanPASMultiMapProp$PercentLost, seq(0,1, by=.05),na.rm = T)[20]

HumanPASMultiMapPropcol=HumanPASMultiMapProp %&gt;% mutate(color=ifelse(PercentLost&gt;=x, &quot;red&quot;, &quot;black&quot;)) %&gt;% arrange(PercentLost)</code></pre>
<pre class="r"><code>plot(HumanPASMultiMapPropcol$PercentLost,col=HumanPASMultiMapPropcol$color,ylab = &quot;Percent of reads multimapping&quot;,main=&quot;Percent of reads lost to multimap filter in each human PAS&quot;,xlab=&quot;42,318 PAS Sorted&quot;)
legend(&quot;topleft&quot;, legend=c(&quot;Bottom 95%&quot;, &quot;Top 5%&quot;),col=c(&quot;black&quot;, &quot;red&quot;), pch=16,bty = &#39;n&#39;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-49-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-49-1">
Past versions of unnamed-chunk-49-1.png
</button>
</p>
<div id="fig-unnamed-chunk-49-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-49-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Look at this in chimp</p>
<pre class="r"><code>ChimpPASMultiMap=as.data.frame(cbind(PAS=ChimpCountMM$PAS,WithMultiMap=CC_MM, NoMultiMap=CC_noMM)) 
ChimpPASMultiMap$WithMultiMap=as.numeric(as.character(ChimpPASMultiMap$WithMultiMap))
ChimpPASMultiMap$NoMultiMap=as.numeric(as.character(ChimpPASMultiMap$NoMultiMap))
ChimpPASMultiMapProp= ChimpPASMultiMap%&gt;% mutate(Difference=WithMultiMap-NoMultiMap, PercentLost=Difference/WithMultiMap)</code></pre>
<p>Plot<br />
Color top 5%</p>
<pre class="r"><code>y=quantile(ChimpPASMultiMapProp$PercentLost, seq(0,1, by=.05),na.rm = T)[20]

ChimpPASMultiMapPropcol=ChimpPASMultiMapProp %&gt;% mutate(color=ifelse(PercentLost&gt;=y, &quot;red&quot;, &quot;black&quot;)) %&gt;% arrange(PercentLost)</code></pre>
<pre class="r"><code>plot(ChimpPASMultiMapPropcol$PercentLost,col=ChimpPASMultiMapPropcol$color,ylab = &quot;Percent of reads multimapping&quot;,main=&quot;Percent of reads lost to multimap filter in each chimp PAS&quot;,xlab=&quot;42,318 PAS Sorted&quot;)
legend(&quot;topleft&quot;, legend=c(&quot;Bottom 95%&quot;, &quot;Top 5%&quot;),col=c(&quot;black&quot;, &quot;red&quot;), pch=16,bty = &#39;n&#39;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-52-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-52-1">
Past versions of unnamed-chunk-52-1.png
</button>
</p>
<div id="fig-unnamed-chunk-52-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-52-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/cd2bced4f0f75c4e1611c95c547084f8faa4a620/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-52-1.png" target="_blank">cd2bced</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Are these the same PAS :</p>
<p>Look at example.</p>
<pre class="r"><code>ChimpPASMultiMapProp %&gt;% filter(PAS==&quot;chimp171173&quot;)</code></pre>
<pre><code>          PAS WithMultiMap NoMultiMap Difference PercentLost
1 chimp171173         2219       2216          3  0.00135196</code></pre>
<p>So this shows that this PAS would not be lost in chimp</p>
<pre class="r"><code>ChimpTop5=ChimpPASMultiMapPropcol %&gt;% filter(color==&quot;red&quot;) %&gt;% select(PAS)
HumanTop5=HumanPASMultiMapPropcol %&gt;% filter(color==&quot;red&quot;) %&gt;% select(PAS)

bothLostProp=intersect(HumanTop5$PAS,ChimpTop5$PAS)
OverlapProp &lt;-intersect(HumanTop5$PAS, ChimpTop5$PAS) 
grid.newpage()
venn.plot &lt;- draw.pairwise.venn(area1 = length(HumanTop5$PAS),
                           area2 = length(ChimpTop5$PAS),
                           cross.area = length(OverlapProp),
                           c(&quot;Human&quot;, &quot;Chimp&quot;), scaled = TRUE,
                           fill = c(&quot;darkorange&quot;, &quot;darkgreen&quot;),
                           cex = 1.5,
                           cat.cex = 1.5,
                           cat.pos = c(320, 25),
                           cat.dist = .05) </code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-54-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-54-1">
Past versions of unnamed-chunk-54-1.png
</button>
</p>
<div id="fig-unnamed-chunk-54-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-54-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/cd2bced4f0f75c4e1611c95c547084f8faa4a620/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-54-1.png" target="_blank">cd2bced</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>This shows that multimapping is not equivalent between.</p>
<p>Color top 2% (get up to 13% difference )</p>
<pre class="r"><code>y=quantile(ChimpPASMultiMapProp$PercentLost, seq(0,1, by=.01),na.rm = T)[99]

ChimpPASMultiMapPropcol2=ChimpPASMultiMapProp %&gt;% mutate(color=ifelse(PercentLost&gt;=y, &quot;red&quot;, &quot;black&quot;)) %&gt;% arrange(PercentLost)

x=quantile(HumanPASMultiMapProp$PercentLost, seq(0,1, by=.01),na.rm = T)[99]

HumanPASMultiMapPropcol2=HumanPASMultiMapProp %&gt;% mutate(color=ifelse(PercentLost&gt;=y, &quot;red&quot;, &quot;black&quot;)) %&gt;% arrange(PercentLost)</code></pre>
<pre class="r"><code>plot(HumanPASMultiMapPropcol2$PercentLost,col=HumanPASMultiMapPropcol2$color,ylab = &quot;Percent of reads multimapping&quot;,main=&quot;Percent of reads lost to multimap filter in each human PAS&quot;,xlab=&quot;42,318 PAS Sorted&quot;)
legend(&quot;topleft&quot;, legend=c(&quot;Bottom 98%&quot;, &quot;Top 2%&quot;),col=c(&quot;black&quot;, &quot;red&quot;), pch=16,bty = &#39;n&#39;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-56-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-56-1">
Past versions of unnamed-chunk-56-1.png
</button>
</p>
<div id="fig-unnamed-chunk-56-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-56-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/cd2bced4f0f75c4e1611c95c547084f8faa4a620/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-56-1.png" target="_blank">cd2bced</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot(ChimpPASMultiMapPropcol2$PercentLost,col=ChimpPASMultiMapPropcol2$color,ylab = &quot;Percent of reads multimapping&quot;,main=&quot;Percent of reads lost to multimap filter in each chimp PAS&quot;,xlab=&quot;42,318 PAS Sorted&quot;)
legend(&quot;topleft&quot;, legend=c(&quot;Bottom 98%&quot;, &quot;Top 2%&quot;),col=c(&quot;black&quot;, &quot;red&quot;), pch=16,bty = &#39;n&#39;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-57-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-57-1">
Past versions of unnamed-chunk-57-1.png
</button>
</p>
<div id="fig-unnamed-chunk-57-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-57-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f0c8049f4ef5f1802bbfe198a505bf8589f4b29a/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-57-1.png" target="_blank">f0c8049</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ChimpTop2=ChimpPASMultiMapPropcol2 %&gt;% filter(color==&quot;red&quot;) %&gt;% select(PAS)
HumanTop2=HumanPASMultiMapPropcol2 %&gt;% filter(color==&quot;red&quot;) %&gt;% select(PAS)

bothLostProp=intersect(HumanTop2$PAS,ChimpTop2$PAS)
OverlapProp &lt;-intersect(HumanTop2$PAS, ChimpTop2$PAS) 
grid.newpage()
venn.plot &lt;- draw.pairwise.venn(area1 = length(HumanTop2$PAS),
                           area2 = length(ChimpTop2$PAS),
                           cross.area = length(OverlapProp),
                           c(&quot;Human&quot;, &quot;Chimp&quot;), scaled = TRUE,
                           fill = c(&quot;darkorange&quot;, &quot;darkgreen&quot;),
                           cex = 1.5,
                           cat.cex = 1.5,
                           cat.pos = c(320, 25),
                           cat.dist = .05) </code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-58-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-58-1">
Past versions of unnamed-chunk-58-1.png
</button>
</p>
<div id="fig-unnamed-chunk-58-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-58-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f0c8049f4ef5f1802bbfe198a505bf8589f4b29a/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-58-1.png" target="_blank">f0c8049</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>In this set there are more on the chimp side.</p>
<p>Look at where these are to see if this could explain the different distribution:</p>
<pre class="r"><code>Chimp2PerMeta= Meta %&gt;% filter(PAS %in% ChimpTop2$PAS) %&gt;% mutate(MultiMap=&quot;Chimp&quot;)

ggplot(Chimp2PerMeta,aes(x=loc,fill=loc)) +geom_bar(stat=&quot;count&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-59-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-59-1">
Past versions of unnamed-chunk-59-1.png
</button>
</p>
<div id="fig-unnamed-chunk-59-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-59-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Human2PerMeta= Meta %&gt;% filter(PAS %in% HumanTop2$PAS) %&gt;% mutate(MultiMap=&quot;Human&quot;)

ggplot(Human2PerMeta,aes(x=loc,fill=loc)) +geom_bar(stat=&quot;count&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-60-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-60-1">
Past versions of unnamed-chunk-60-1.png
</button>
</p>
<div id="fig-unnamed-chunk-60-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-60-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f0c8049f4ef5f1802bbfe198a505bf8589f4b29a/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-60-1.png" target="_blank">f0c8049</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Filterout the PAS with multimapping in both:</p>
<pre class="r"><code>Human.unique &lt;- setdiff(HumanTop2$PAS, ChimpTop2$PAS) # n 1st NOT 2nd

Chimp.unique &lt;- setdiff(ChimpTop2$PAS, HumanTop2$PAS) # n 1st NOT 2nd</code></pre>
<p>plot</p>
<pre class="r"><code>Human2PerUniqMeta= Meta %&gt;% filter(PAS %in% Human.unique) %&gt;% mutate(MultiMap=&quot;Human&quot;)

Chimp2PerUniqMeta= Meta %&gt;% filter(PAS %in% Chimp.unique) %&gt;%  mutate(MultiMap=&quot;Chimp&quot;)

Both2PerUniq=Human2PerUniqMeta %&gt;% bind_rows(Chimp2PerUniqMeta)


ggplot(Both2PerUniq, aes(x=loc, by=MultiMap, fill=MultiMap)) + geom_bar(stat=&quot;count&quot;, position = &quot;dodge&quot;) + scale_fill_brewer(palette = &quot;Dark2&quot;) + labs(y=&quot;Number of PAS&quot;, title=&quot;PAS likely wrong due to multimapping in 1 species \n 2% filter&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-62-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-62-1">
Past versions of unnamed-chunk-62-1.png
</button>
</p>
<div id="fig-unnamed-chunk-62-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-62-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/f0c8049f4ef5f1802bbfe198a505bf8589f4b29a/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-62-1.png" target="_blank">f0c8049</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Plot average usage in each species for these:</p>
<pre class="r"><code>Both2PerUniq_g= Both2PerUniq %&gt;%select(Chimp, Human, MultiMap, loc) %&gt;% gather(&quot;Species&quot;, &quot;Usage&quot;, -MultiMap, -loc )
ggplot(Both2PerUniq_g, aes(x=loc, y=Usage, by=Species, fill=Species)) + geom_boxplot() + facet_grid(~MultiMap) + scale_fill_brewer(palette = &quot;Dark2&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-63-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-63-1">
Past versions of unnamed-chunk-63-1.png
</button>
</p>
<div id="fig-unnamed-chunk-63-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-63-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Where were they discovered:</p>
<pre class="r"><code>ggplot(Both2PerUniq, aes(x=loc, by=disc,fill=disc)) + geom_bar(stat=&quot;count&quot;, position=&quot;dodge&quot;)+ facet_grid(~MultiMap) + scale_fill_brewer(palette = &quot;Dark2&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-64-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-64-1">
Past versions of unnamed-chunk-64-1.png
</button>
</p>
<div id="fig-unnamed-chunk-64-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-64-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Check to see if these are in the dominant set</p>
<p>Get the dominant and allow ties:</p>
<pre class="r"><code>allPAS= read.table(&quot;../data/PAS_doubleFilter/PAS_10perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt&quot;, header = T) 
ChimpPASwMean =allPAS %&gt;% dplyr::select(-Human)
HumanPASwMean =allPAS %&gt;% dplyr::select(-Chimp)

Chimp_Dom2= ChimpPASwMean %&gt;%
  group_by(gene) %&gt;%
  top_n(1,Chimp) %&gt;% 
  mutate(nPer=n()) 
nrow(Chimp_Dom2%&gt;% filter(nPer&gt;1) )</code></pre>
<pre><code>[1] 198</code></pre>
<pre class="r"><code>Human_Dom2= HumanPASwMean %&gt;%
  group_by(gene) %&gt;%
  top_n(1,Human) %&gt;% 
  mutate(nPer=n()) </code></pre>
<p>Is there an enrichment for dominant in these:</p>
<pre class="r"><code>allPAS_annoDomandMM= allPAS %&gt;% mutate(DomHuman=ifelse(PAS %in% Human_Dom2$PAS, &quot;Yes&quot;, &quot;No&quot;),DomChimp=ifelse(PAS %in% Chimp_Dom2$PAS, &quot;Yes&quot;, &quot;No&quot;), MM_human=ifelse(PAS %in% HumanTop2$PAS, &quot;Yes&quot;, &quot;No&quot;), MM_Chimp=ifelse(PAS %in% ChimpTop2$PAS, &quot;Yes&quot;, &quot;No&quot;), MMuniq_human=ifelse(PAS %in% Human.unique, &quot;Yes&quot;, &quot;No&quot;),MMuniq_chimp=ifelse(PAS %in% Chimp.unique, &quot;Yes&quot;, &quot;No&quot;))</code></pre>
<p>Enrichments:</p>
<p>Dom Human and MM human uniq</p>
<pre class="r"><code>x= nrow(allPAS_annoDomandMM %&gt;% filter(DomHuman ==&quot;Yes&quot;,MMuniq_chimp==&quot;Yes&quot; ))
m= nrow(allPAS_annoDomandMM %&gt;% filter(DomHuman ==&quot;Yes&quot;))
n=nrow(allPAS_annoDomandMM%&gt;% filter(DomHuman ==&quot;No&quot;))
k=nrow(allPAS_annoDomandMM%&gt;% filter(MMuniq_chimp ==&quot;Yes&quot;))
x</code></pre>
<pre><code>[1] 162</code></pre>
<pre class="r"><code>#expected
which(grepl(max(dhyper(1:x, m, n, k)), dhyper(1:x, m, n, k)))</code></pre>
<pre><code>[1] 144</code></pre>
<pre class="r"><code>phyper(x,m, n , k,lower.tail=F)</code></pre>
<pre><code>[1] 0.04561002</code></pre>
<pre class="r"><code>x= nrow(allPAS_annoDomandMM %&gt;% filter(DomChimp ==&quot;Yes&quot;,MMuniq_human==&quot;Yes&quot; ))
m= nrow(allPAS_annoDomandMM %&gt;% filter(DomChimp ==&quot;Yes&quot;))
n=nrow(allPAS_annoDomandMM%&gt;% filter(DomChimp ==&quot;No&quot;))
k=nrow(allPAS_annoDomandMM%&gt;% filter(MMuniq_human ==&quot;Yes&quot;))
x</code></pre>
<pre><code>[1] 163</code></pre>
<pre class="r"><code>#expected
which(grepl(max(dhyper(1:x, m, n, k)), dhyper(1:x, m, n, k)))</code></pre>
<pre><code>[1] 132</code></pre>
<pre class="r"><code>phyper(x,m, n , k,lower.tail=F)</code></pre>
<pre><code>[1] 0.001039761</code></pre>
<p>Not enriched but 79 and 62 still means something. Lets see where these are.</p>
<pre class="r"><code>ChimpDomandMM=allPAS_annoDomandMM %&gt;% filter(DomChimp ==&quot;Yes&quot;,MMuniq_human==&quot;Yes&quot;) %&gt;% mutate(DominantSpecies =&quot;Chimp&quot;)
HumanDomandMM=allPAS_annoDomandMM %&gt;% filter(DomHuman ==&quot;Yes&quot;,MMuniq_chimp==&quot;Yes&quot;)%&gt;% mutate(DominantSpecies=&quot;Human&quot;)


bothDomandMM= ChimpDomandMM %&gt;% bind_rows(HumanDomandMM)
#plot the location of dominant 

ggplot(bothDomandMM,aes(x=loc, fill=DominantSpecies,by=DominantSpecies)) + geom_bar(stat=&quot;count&quot;, position = &quot;dodge&quot;) + scale_fill_brewer(palette = &quot;Dark2&quot;) + labs(x=&quot;&quot;, y=&quot;Number of PAS&quot;, title=&quot;Location of Dominant PAS conditioned on multimapping problem in other species &quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-69-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-69-1">
Past versions of unnamed-chunk-69-1.png
</button>
</p>
<div id="fig-unnamed-chunk-69-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/1fad12e7675ba3b5cd0b9065c271d64aa12c88b4/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-69-1.png" target="_blank">1fad12e</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Write out these PAS to pull out in other analysis.</p>
<pre class="bash"><code>mkdir ../data/multimap
</code></pre>
<pre class="r"><code>allPAS_overlap= allPAS %&gt;% filter(PAS %in% OverlapProp) %&gt;% mutate(MultiMap=&quot;Both&quot;)


write.table(Chimp2PerUniqMeta, &quot;../data/multimap/Chimp_Uniq_multimapPAS.txt&quot;, col.names=T, row.names=F, quote=F)

write.table(Human2PerUniqMeta, &quot;../data/multimap/Human_Uniq_multimapPAS.txt&quot;, col.names=T, row.names=F, quote=F)
write.table(allPAS_overlap, &quot;../data/multimap/Both_multimapPAS.txt&quot;, col.names=T, row.names=F, quote=F)</code></pre>
<div id="try-the-primary-map-read" class="section level3">
<h3>Try the primary map read</h3>
<pre class="bash"><code>python fixFChead_bothfrac.py ../data/testQuant/ALLPAS_postLift_LocParsed_Human_Primary ../data/testQuant/ALLPAS_postLift_LocParsed_Human_Primary.fixed.fc 
python fixFChead_bothfrac.py ../data/testQuant/ALLPAS_postLift_LocParsed_Chimp_Primary ../data/testQuant/ALLPAS_postLift_LocParsed_Chimp_Primary.fixed.fc </code></pre>
<pre class="r"><code>HumanNPrimary=read.table(&quot;../data/testQuant/ALLPAS_postLift_LocParsed_Human_Primary.fixed.fc&quot;,header = T, stringsAsFactors = F) %&gt;% separate(Geneid, into=c(&quot;disc&quot;, &quot;PAS&quot;, &quot;chr2&quot;, &quot;start2&quot;, &quot;end2&quot;, &#39;strand&#39;, &#39;geneid&#39;), sep=&quot;:&quot;)%&gt;% filter(PAS %in% Meta$PAS) 
ChimpNPrimary=read.table(&quot;../data/testQuant/ALLPAS_postLift_LocParsed_Chimp_Primary.fixed.fc&quot;, header = T, stringsAsFactors = F) %&gt;% separate(Geneid, into=c(&quot;disc&quot;, &quot;PAS&quot;, &quot;chr2&quot;, &quot;start2&quot;, &quot;end2&quot;, &#39;strand&#39;, &#39;geneid&#39;), sep=&quot;:&quot;)%&gt;% filter(PAS %in% ChimpMeta$PAS) </code></pre>
<p>Compare this to the original FC.</p>
<pre class="r"><code>#HumanCountsFeatureCounts_mat

HumanPrimary_mat=as.matrix(HumanNPrimary %&gt;% select(contains(&quot;_N&quot;)))

HumanFCP=HumanPrimary_mat- HumanCountsFeatureCounts_mat



HumanFCP_g=as.data.frame(HumanFCP) %&gt;% gather(&quot;Ind&quot;, &quot;count&quot;)

ggplot(HumanFCP_g, aes(x=Ind, y=count,col=Ind))+ geom_boxplot() + labs(y=&quot;Primary - No MM&quot;, title=&quot;Human Difference in Counts for final PAS&quot;)+ theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-74-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-74-1">
Past versions of unnamed-chunk-74-1.png
</button>
</p>
<div id="fig-unnamed-chunk-74-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/3d4dd4722d951a6205c8db1fdf2d6a513b4b402e/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-74-1.png" target="_blank">3d4dd47</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Ok so the primary flag doesnâ€™t change anything for these.</p>
<p>try priamry and -m</p>
<pre class="bash"><code>sbatch BothFCMMPrim.sh
python fixFChead_bothfrac.py ../data/testQuant/ALLPAS_postLift_LocParsed_Human_PrimaryandM ../data/testQuant/ALLPAS_postLift_LocParsed_Human_PrimaryandM.fixed.fc 
python fixFChead_bothfrac.py ../data/testQuant/ALLPAS_postLift_LocParsed_Chimp_PrimaryandM ../data/testQuant/ALLPAS_postLift_LocParsed_Chimp_PrimaryandM.fixed.fc </code></pre>
<pre class="r"><code>HumanNPrimaryandM=read.table(&quot;../data/testQuant/ALLPAS_postLift_LocParsed_Human_PrimaryandM.fixed.fc&quot;,header = T, stringsAsFactors = F) %&gt;% separate(Geneid, into=c(&quot;disc&quot;, &quot;PAS&quot;, &quot;chr2&quot;, &quot;start2&quot;, &quot;end2&quot;, &#39;strand&#39;, &#39;geneid&#39;), sep=&quot;:&quot;)%&gt;% filter(PAS %in% Meta$PAS) 

HumanPrimaryandM_mat=as.matrix(HumanNPrimaryandM %&gt;% select(contains(&quot;_N&quot;)))

HumanFCPM=HumanPrimaryandM_mat- HumanCountsFeatureCounts_mat



HumanFCPM_g=as.data.frame(HumanFCPM) %&gt;% gather(&quot;Ind&quot;, &quot;count&quot;)

ggplot(HumanFCPM_g, aes(x=Ind, y=count,col=Ind))+ geom_boxplot() + labs(y=&quot;Primary with M - No MM&quot;, title=&quot;Human Difference in Counts for final PAS&quot;)+ theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-76-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-76-1">
Past versions of unnamed-chunk-76-1.png
</button>
</p>
<div id="fig-unnamed-chunk-76-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/Comparative_APA/blob/3d4dd4722d951a6205c8db1fdf2d6a513b4b402e/docs/figure/TrynewQuantMethods.Rmd/unnamed-chunk-76-1.png" target="_blank">3d4dd47</a>
</td>
<td>
brimittleman
</td>
<td>
2020-03-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Look at how many are not zero here. This tells me how many PAS have multimap.</p>
<pre class="r"><code>nrow(HumanFCPM)</code></pre>
<pre><code>[1] 42318</code></pre>
<pre class="r"><code>sum(rowSums(HumanFCPM) !=0)</code></pre>
<pre><code>[1] 10693</code></pre>
<pre class="r"><code>10693/42318</code></pre>
<pre><code>[1] 0.2526821</code></pre>
<p>25% have some MM.</p>
<p>I need the scores for the primary and secondary maps.</p>
<p>I will look at the mean for each PAS across individuals in each set. Then I can see the differences between the 3 sets. regular, -m, â€“primary</p>
<pre class="r"><code>#reg
HumanOriginal=HumanCountsFeatureCounts %&gt;% select(PAS,contains(&quot;_N&quot;)) %&gt;% gather(&quot;Ind&quot;, &quot;Counts&quot;, -PAS) %&gt;%group_by(PAS) %&gt;%  summarise(meanCount=mean(Counts)) %&gt;% mutate(Set=&quot;Original&quot;)

#multimap  

HumanPAS_MM=HumanCountMM %&gt;% select(PAS,contains(&quot;_N&quot;)) %&gt;% gather(&quot;Ind&quot;, &quot;Counts&quot;, -PAS) %&gt;%group_by(PAS) %&gt;%  summarise(meanCount=mean(Counts)) %&gt;% mutate(Set=&quot;MultiMap&quot;)


#Primary  

HumanPAS_primary=HumanNPrimaryandM  %&gt;% select(PAS,contains(&quot;_N&quot;)) %&gt;% gather(&quot;Ind&quot;, &quot;Counts&quot;, -PAS) %&gt;%group_by(PAS) %&gt;%  summarise(meanCount=mean(Counts)) %&gt;% mutate(Set=&quot;Primary&quot;)


ThreeCriteriaHuman=HumanOriginal %&gt;% bind_rows(HumanPAS_MM) %&gt;% bind_rows(HumanPAS_primary) %&gt;% spread(Set, meanCount) 

ThreeCriteriaHuman_var= HumanOriginal %&gt;% bind_rows(HumanPAS_MM) %&gt;% bind_rows(HumanPAS_primary) %&gt;% group_by(PAS) %&gt;% summarise(VarSet=var(meanCount))

ThreeCriteriaHuman_varnot0= ThreeCriteriaHuman_var %&gt;%  filter(VarSet!=0)</code></pre>
<pre class="r"><code>plot(sort(ThreeCriteriaHuman_varnot0$VarSet))</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-79-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggplot(ThreeCriteriaHuman,aes(x=Original,y=Primary)) + geom_point() + geom_abline(slope=1, intercept = 0) + geom_density2d(col=&quot;blue&quot;)+ coord_cartesian(xlim=c(0,2500),ylim=c(0,2500))</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-80-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggplot(ThreeCriteriaHuman,aes(x=Original,y=MultiMap)) + geom_point() + geom_abline(slope=1, intercept = 0) + geom_density2d(col=&quot;blue&quot;)+ coord_cartesian(xlim=c(0,2500),ylim=c(0,2500))</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-81-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggplot(ThreeCriteriaHuman,aes(x=Primary,y=MultiMap)) + geom_point() + geom_abline(slope=1, intercept = 0) + geom_density2d(col=&quot;blue&quot;)+ coord_cartesian(xlim=c(0,2500),ylim=c(0,2500))</code></pre>
<p><img src="figure/TrynewQuantMethods.Rmd/unnamed-chunk-82-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>I only let one location read. Multimap will be the same as primary.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.5.1 (2018-07-02)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] Matrix_1.2-15       VennDiagram_1.6.20  futile.logger_1.4.3
 [4] ggpubr_0.2          magrittr_1.5        forcats_0.3.0      
 [7] stringr_1.3.1       dplyr_0.8.0.1       purrr_0.3.2        
[10] readr_1.3.1         tidyr_0.8.3         tibble_2.1.1       
[13] ggplot2_3.1.1       tidyverse_1.2.1    

loaded via a namespace (and not attached):
 [1] tidyselect_0.2.5     reshape2_1.4.3       haven_1.1.2         
 [4] lattice_0.20-38      colorspace_1.3-2     generics_0.0.2      
 [7] htmltools_0.3.6      yaml_2.2.0           rlang_0.4.0         
[10] later_0.7.5          pillar_1.3.1         glue_1.3.0          
[13] withr_2.1.2          RColorBrewer_1.1-2   lambda.r_1.2.3      
[16] modelr_0.1.2         readxl_1.1.0         plyr_1.8.4          
[19] munsell_0.5.0        gtable_0.2.0         workflowr_1.6.0     
[22] cellranger_1.1.0     rvest_0.3.2          evaluate_0.12       
[25] labeling_0.3         knitr_1.20           httpuv_1.4.5        
[28] broom_0.5.1          Rcpp_1.0.2           formatR_1.5         
[31] promises_1.0.1       scales_1.0.0         backports_1.1.2     
[34] jsonlite_1.6         fs_1.3.1             hms_0.4.2           
[37] digest_0.6.18        stringi_1.2.4        rprojroot_1.3-2     
[40] cli_1.1.0            tools_3.5.1          lazyeval_0.2.1      
[43] futile.options_1.0.1 crayon_1.3.4         whisker_0.3-2       
[46] pkgconfig_2.0.2      MASS_7.3-51.1        xml2_1.2.0          
[49] lubridate_1.7.4      assertthat_0.2.0     rmarkdown_1.10      
[52] httr_1.3.1           rstudioapi_0.10      R6_2.3.0            
[55] nlme_3.1-137         git2r_0.26.1         compiler_3.5.1      </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
