---
title: "Comparison to PAS from apaQTL project"
author: "Briana Mittleman"
date: "10/4/2019"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


```{r}
library(ggpubr)
library(tidyverse)
library(reshape2)
```

In this analysis I will compare the PAS found at 5% in the human data in this project with the apaQTL project.  

Take the 5 perc PAS from the annotatePAS analysis and filter those with scores > .05 in human. I also want to remove PAS not in chr 1-22.  


```{r}
chroms=c('chr10', 'chr11', 'chr12', 'chr13', 'chr14', 'chr15', 'chr16', 'chr17', 'chr18', 'chr19', 'chr1', 'chr2', 'chr20', 'chr21', 'chr22', 'chr3', 'chr4', 'chr5', 'chr6', 'chr7','chr8', 'chr9')

compAPAPAS=read.table("../data/Peaks_5perc/Peaks_5perc_either_HumanCoordHummanUsage.bed", header =   T, stringsAsFactors = F)  %>%  filter(Human>=0.05, chr %in% chroms) 

```


I can write this bed out and use bedtools intersect. 

```{bash,eval=F}
mkdir ../data/CompapaQTLpas
```


```{r}
write.table(compAPAPAS,"../data/CompapaQTLpas/CompAPA_PAS_5percHuman.bed", col.names = F, row.names = F, quote = F, sep="\t")
```

```{bash,eval=F}
sort -k1,1 -k2,2n ../data/CompapaQTLpas/CompAPA_PAS_5percHuman.bed > ../data/CompapaQTLpas/CompAPA_PAS_5percHuman.sort.bed

sbatch overlapapaQTLPAS.sh
```


Compare the ones with overlaps to those without:

```{r}
wOverlap=read.table("../data/CompapaQTLpas/PAS_5percHuman.sort.Intersect.bed", col.names = colnames(compAPAPAS)) %>% mutate(overlap="yes")
noOverlap=read.table("../data/CompapaQTLpas/PAS_5percHuman.sort.Intersect.NoOverlap.bed", col.names = colnames(compAPAPAS)) %>% mutate(overlap="no")

AllwOinfo=as.data.frame(rbind(wOverlap, noOverlap))
nrow(AllwOinfo)
```


```{r}
ggplot(AllwOinfo, aes(x=overlap,fill=overlap)) +geom_bar(aes(y = (..count..)/sum(..count..)))+ scale_fill_brewer(palette = "Dark2")
```

```{r}
ggplot(AllwOinfo, aes(x=overlap, y=Human, fill=overlap)) + geom_boxplot() + scale_fill_brewer(palette = "Dark2") + stat_compare_means(method="t.test")
```


Look at actual counts to see if this is a noise problem:



```{r}
humanCounts=read.table("../Human/data/CleanLiftedPeaks_FC/ALLPAS_postLift_LocParsed_Human_fixed.fc",header = T, stringsAsFactors = F) %>% dplyr::select(-Chr, -Start,-End, -Strand, -Length)  %>% tidyr::separate(Geneid, sep = ":", into = c("disc", "PAS", "chr", "start", "end", "strand", "gene"))

humanCounts_mean=rowMeans(humanCounts[,8:19])

HumanMeanAnno= as.data.frame(cbind(PAS=humanCounts$PAS, meanCount=humanCounts_mean)) %>% inner_join(AllwOinfo, by= "PAS")

HumanMeanAnno$overlap=as.factor(HumanMeanAnno$overlap)
HumanMeanAnno$meanCount=as.integer(as.character(HumanMeanAnno$meanCount))
```

```{r}
ggplot(HumanMeanAnno, aes(x=overlap, y=log10(meanCount +1),fill=overlap)) + geom_boxplot() + scale_fill_brewer(palette = "Dark2")+ stat_compare_means(method="t.test") + labs(title="HC PAS counts by overlap with apaQTL PAS")
```

Get the percent overlap by usage filter:  

```{r}
overlap=c()
totalvec=c()
seq_usage=seq(0, .95, .01)
for (i in seq_usage){
  x=AllwOinfo %>% filter(Human>i) %>% group_by(overlap) %>% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlap=c(overlap,prop)
  totalvec=c(totalvec,total)
}

```

```{r}
plot(seq_usage,overlap,main="Overlap with apaQTL PAS by human average usage", ylab="Percent Overlap", xlab="Usage Cutoff")
abline(v=.05,col="red")
abline(v=.1,col="blue")
```

```{r}
plot(seq_usage,totalvec,main="Number of PAS",ylab="Number of PAS",xlab="Usage Cutoff" )
abline(v=.05,col="red")
abline(v=.1,col="blue")
```

Extended:  

```{r}
apaQTLPAS=read.table("../data/liftover_files/APAPAS_GeneLocAnno.5perc.hg19lifted.sorted.bed",stringsAsFactors = F,col.names = c("chr","start", "end", "PAS", "score","strand")) 
apaQTLPAS_ext= apaQTLPAS %>% mutate(start_new=start-50, end_new=end+50) %>% dplyr::select(chr,start_new, end_new, PAS, score, strand)

write.table(apaQTLPAS_ext,"../data/liftover_files/APAPAS_GeneLocAnno.5perc.hg19lifted_extended.bed",col.names = F, row.names = F, quote = F, sep="\t")
```

```{bash,eval=F}
sort -k1,1 -k2,2n ../data/liftover_files/APAPAS_GeneLocAnno.5perc.hg19lifted_extended.bed > ../data/liftover_files/APAPAS_GeneLocAnno.5perc.hg19lifted_extended.sort.bed

sbatch overlapapaQTLPAS_extended.sh
```


```{r}
wOverlapExt=read.table("../data/CompapaQTLpas/PAS_5percHuman.sort.Intersect_ext.bed", col.names = colnames(compAPAPAS)) %>% mutate(overlap="yes")
noOverlapExt=read.table("../data/CompapaQTLpas/PAS_5percHuman.sort.Intersect.NoOverlap_ext.bed", col.names = colnames(compAPAPAS)) %>% mutate(overlap="no")

AllwOinfoExt=as.data.frame(rbind(wOverlapExt, noOverlapExt))
nrow(AllwOinfoExt)
```

```{r}
ggplot(AllwOinfoExt, aes(x=overlap,fill=overlap)) +geom_bar(aes(y = (..count..)/sum(..count..)))+ scale_fill_brewer(palette = "Dark2")
```


```{r}
overlap=c()
totalvec=c()
seq_usage=seq(0, .95, .01)
for (i in seq_usage){
  x=AllwOinfoExt %>% filter(Human>i) %>% group_by(overlap) %>% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlap=c(overlap,prop)
  if (i==.05 | i==.1){
    print(prop)
    print(i)
  }
  totalvec=c(totalvec,total)
}
```

```{r}
plot(seq_usage,overlap,main="Overlap with apaQTL Extended PAS by human average usage", ylab="Percent Overlap", xlab="Usage Cutoff")
abline(v=.05,col="red")
abline(v=.1,col="blue")
```


```{r}
plot(seq_usage,totalvec,main="Number of PAS by human by usage",ylab="Number of PAS",xlab="Usage Cutoff" )
abline(v=.05,col="red")
abline(v=.1,col="blue")
```


##Expression Filter  
Filter down to those with higher expression: 

I want to only look at the genes that passed the expression filter to test for DE.  

```{r}
nameID=read.table("../../genome_anotation_data/ensemble_to_genename.txt",sep="\t", header = T, stringsAsFactors = F) %>% dplyr::select(Gene_stable_ID,Gene.name)
DEtested=read.table("../data/DiffExpression/DE_Testedgenes.txt", col.names = c("Gene_stable_ID"), stringsAsFactors = F) %>% inner_join(nameID, by="Gene_stable_ID")
```

Join gene names to the intersections:
```{r}
PASMeta=read.table("../data/PAS/PAS_5perc_either_HumanCoord_BothUsage_meta.txt",header = T,stringsAsFactors = F) %>% dplyr::select(PAS, gene, chr, start,end,disc)
AllwOinfo_gene=AllwOinfo %>% inner_join(PASMeta,by=c("PAS", "chr","start", "end"))
AllwOinfo_gene_filt=AllwOinfo_gene %>% filter(gene %in% DEtested$Gene.name )

nrow(AllwOinfo_gene_filt)
```

Now plot:  

```{r}
ggplot(AllwOinfo_gene_filt, aes(x=overlap,fill=overlap)) +geom_bar(aes(y = (..count..)/sum(..count..)))+ scale_fill_brewer(palette = "Dark2") + labs(title="Overlap with apaQTL PAS after filtering low expressed genes", y="percentage")

```

Remove those identified in chimp:

```{r}
AllwOinfo_gene_filt_humanident=AllwOinfo_gene_filt %>% filter(disc != "Chimp")
nrow(AllwOinfo_gene_filt_humanident)

ggplot(AllwOinfo_gene_filt_humanident, aes(x=overlap,fill=overlap)) +geom_bar(aes(y = (..count..)/sum(..count..)))+ scale_fill_brewer(palette = "Dark2") + labs(title="Overlap with apaQTL PAS after filtering low expressed genes\n and identified in chimp", y="percentage")
```


```{r}
AllwOinfoExt_gene=AllwOinfoExt %>% inner_join(PASMeta,by=c("PAS", "chr","start", "end"))
AllwOinfoExt_gene_filt=AllwOinfoExt_gene %>% filter(gene %in% DEtested$Gene.name )

ggplot(AllwOinfoExt_gene_filt, aes(x=overlap,fill=overlap)) +geom_bar(aes(y = (..count..)/sum(..count..)))+ scale_fill_brewer(palette = "Dark2") + labs(title="Overlap with apaQTL PAS after filtering low expressed genes \n extended PAS", y="percentage")
nrow(AllwOinfoExt_gene_filt)
```

```{r}
ggplot(AllwOinfoExt_gene_filt,aes(x=overlap,y=Human,fill=overlap))+ geom_boxplot() + stat_compare_means(method="t.test") + scale_fill_brewer(palette = "Dark2")
```

Get the percent overlap by usage filter:  

```{r}
overlap=c()
totalvec=c()
seq_usage=seq(0, .95, .01)
for (i in seq_usage){
  x=AllwOinfoExt_gene_filt %>% filter(Human>i) %>% group_by(overlap) %>% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
    if (i==.05 | i==.1){
    print(prop)
    print(i)
  }
  overlap=c(overlap,prop)
  totalvec=c(totalvec,total)
}

```

```{r}
plot(seq_usage,overlap,main="Overlap with apaQTL extended PAS by human \naverage usage filtered by gene expression", ylab="Percent Overlap", xlab="Usage Cutoff")
abline(v=.05,col="red")
abline(v=.1,col="blue")
```

```{r}
plot(seq_usage,totalvec,main="Number of PAS by human \naverage usage filtered by gene expression",ylab="Number of PAS",xlab="Usage Cutoff" )
abline(v=.05,col="red")
abline(v=.1,col="blue")
```
