---
title: "Difference in protein and APA not expression"
author: "Briana Mittleman"
date: "3/7/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

```{r}
library(tidyverse)
library(UpSetR)
library(ggpubr)
```

Upload:  

```{r}
Protein=read.table("../data/Khan_prot/HC_SigProtein.txt", header = T, stringsAsFactors = F)%>% dplyr::rename("gene"=gene.symbol)
nameID=read.table("../../genome_anotation_data/ensemble_to_genename.txt",sep="\t", header = T, stringsAsFactors = F)
DEgenes=read.table("../data/DiffExpression/DE_genes.txt", header = F,col.names = c("Gene_stable_ID"),stringsAsFactors = F) %>% inner_join(nameID, by="Gene_stable_ID") %>% dplyr::select(Gene.name) %>% dplyr::rename("gene"=Gene.name)
NucAPA=read.table("../data/DiffIso_Nuclear_DF/SignifianceEitherGENES_Nuclear.txt",header = T,stringsAsFactors = F)
```

I will do this first with these then I can start to look at it by significance.  

```{r}
apatested=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T) %>% select(gene) %>% unique()
```


```{r}
pnotE=Protein %>% anti_join(DEgenes,by="gene") 
nrow(pnotE)
nrow(Protein)
nrow(DEgenes)
apatested %>% inner_join(pnotE,by="gene") %>% nrow()

APAandPnotE= NucAPA %>% inner_join(Protein, by="gene") %>% anti_join(DEgenes,by="gene")
```

```{r}
listInput_nucOnly <- list(DE=DEgenes$gene, DAPA=NucAPA$gene, DP=Protein$gene)

#upset(fromList(listInput_nosplice), queries = list(list(query=intersects, params=list("DAPA", "DT", "DP"), color="red", active=T,query.name="APA, Ribo, Protein"),list(query=intersects, params=list("DE", "DT", "DP"), color="orange", active=T, query.name="Expression,Ribo, Protein"),list(query=intersects, params=list("DAPA", "DT"), color="blue", active=T, query.name="APA,Ribo") ,list(query=intersects, params=list("DAPA", "DP"), color="purple", active=T, query.name="APA, Protein"),list(query=intersects, params=list("DAPA", "DE"), color="green", active=T, query.name="APA, Expression")), order.by = "freq", query.legend = "bottom")




upset(fromList(listInput_nucOnly), order.by = "freq", keep.order = T,empty.intersections = "on", queries = list(list(query=intersects, params=list("DAPA", "DP"), color="darkorchid4", active=T,query.name="APA, Protein")))
      
      
      
      
```



90 of these genes.  

Learn about these genes. 

Selection:  


model.num.rna: : 1 = mRNA expression level pattern consistent with directional selection along human lineage, 2 = mRNA expression level pattern consistent with directional selection along chimpanzee lineage, 3 = undetermined pattern, 4 = patterns with no significant difference between mean expression levels; 5 = evidence for relaxation of constraint along human lineage, 6 = evidence of relaxation of constraint along chimpanzee lineage

model.num.protein: 1 = protein expression level pattern consistent with directional selection along human lineage, 2 = protein expression level pattern consistent with directional selection along chimpanzee lineage, 3 = undetermined pattern, 4 = patterns with no significant difference between mean expression levels; 5 = evidence for relaxation of constraint along human lineage, 6 = evidence of relaxation of constraint along chimpanzee lineage 


```{r}
KhanData=read.csv("../data/Khan_prot/Khan_TableS4.csv",stringsAsFactors = F)  %>% dplyr::select(gene.symbol,contains("model") ) %>% dplyr::rename("gene"=gene.symbol, "Protein"=model.num.protein, "RNA"=model.num.rna)


APAandPnotE_sel= APAandPnotE %>% inner_join(KhanData,by="gene")
```

Plot the information about the RNA and protein for these:  

```{r}
APAandPnotE_sel_g=APAandPnotE_sel %>% dplyr::select(gene, Protein, RNA) %>% gather("Set", "Model", -gene)


APAandPnotE_sel_g$Model= as.factor(APAandPnotE_sel_g$Model)
ggplot(APAandPnotE_sel_g,aes(x=Model, by=Set, fill=Set)) + geom_bar(stat="count", position="dodge") + scale_fill_brewer(palette = "RdYlBu")




```

Plot protein only:  

```{r}
APAandPnotE_sel_gOnlyP= APAandPnotE_sel_g %>% filter(Set=="Protein")

APAandPnotE_sel_gOnlyP$Model= as.factor(APAandPnotE_sel_gOnlyP$Model)

ggplot(APAandPnotE_sel_gOnlyP,aes(x=Model)) + geom_bar(stat="count", position="dodge", fill="darkorchid4") + labs(y="Number of Genes", x="Protein Selection Model", title="Protein and APA differences\n no difference in Expression") + scale_x_discrete( labels=c("Selection Human","Selection Chimp","Undetermined","No mean difference","Relaxation in Chimp"))+theme(axis.text.x=element_text(angle=90, hjust=0), text= element_text(size=16)) 
```


The genes in 1,2,5,6 are interesting.  

```{r}
APAandPnotE_selCalled= APAandPnotE_sel_g %>% filter(Set=="Protein", Model %in% c(1,2,5,6))
```
There are 20 of these genes: 

```{r}
APAandPnotE_selCalled
```


Where are the differential PAS in these genes:  

```{r}
#APAandPnotE_sel_gOnlyP
Meta=read.table("../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", header = T, stringsAsFactors = T) %>% dplyr::rename("ChimpUsage"=Chimp, "HumanUsage"=Human)
NucAPAres=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T,stringsAsFactors = F) %>% inner_join(Meta, by=c("chr", "start","end", "gene"))


NucAPAres_DP= NucAPAres %>% filter(gene %in%APAandPnotE_sel_gOnlyP$gene ) %>% filter(SigPAU2=="Yes")


NucAPAresSig=NucAPAres %>% filter(SigPAU2=="Yes")
```

THere are 154 PAS in this set:


```{r}
ggplot(NucAPAres_DP,aes(x=loc,fill=loc))+ geom_bar(stat="count") + scale_fill_brewer(palette = "RdYlBu")+theme(axis.text.x=element_text(angle=90, hjust=0), text= element_text(size=16), legend.position = "false") + labs(x="", y="Number of PAS", title="Expression independent PAS locations")
```


Enrichment for this:  

Compare to all of the significant in that location. 

```{r}

NucAPAres_sig= NucAPAres %>% filter(SigPAU2=="Yes") %>% mutate(dPnotE=ifelse(PAS %in% NucAPAres_DP$PAS,"Yes", "No"))


enrich=c()
pval=c()

for (i in c("cds", "end", "intron", "utr3")){
  x=nrow(NucAPAres_sig %>% filter(dPnotE=="Yes", loc==i))
  m=nrow(NucAPAres_sig %>% filter( loc==i))
  n=nrow(NucAPAres_sig %>% filter(loc!=i))
  k=nrow(NucAPAres_sig %>% filter(dPnotE=="Yes"))
  N=nrow(NucAPAres_sig)
  pval=c(pval, phyper(x-1,m,n,k,lower.tail=F))
  enrichval=(x/k)/(m/N)
  enrich=c(enrich, enrichval)
}
enrich
pval


NucAPAres_DPLocEnrich=NucAPAres_DP %>% group_by(loc) %>% summarise(n=n()) %>% bind_cols(enrichment=enrich, pvalue=pval)


locplot=ggplot(NucAPAres_DPLocEnrich, aes(x=loc, y=n, fill=loc)) + geom_bar(stat="identity") + scale_fill_brewer(palette = "RdYlBu")+theme(axis.text.x=element_text(angle=90, hjust=0), text= element_text(size=16), legend.position = "false") + labs(x="", y="Number of PAS", title="Expression independent PAS locations")+ geom_text(aes(label=paste("Enrichment=",round(enrichment,2), "X", sep=""), vjust=0)) +geom_text(aes(label=paste("Pval=",round(pvalue,3), sep=""), vjust=2))+ theme_classic() +  theme(legend.position = "none", axis.text.x = element_text(size=10),plot.title = element_text(hjust = 0.5, face="bold"),axis.text.y = element_text(size=10),text=element_text(size=10),plot.margin = unit(c(0,0,0,0), "cm"))
locplot
```
```{r, eval=F, include=F}
pdf("../output/supplement/Fig6-figSup1.pdf", height=6, width=6,useKerning=F)
locplot
dev.off()
```



Interactions:


Are there differences in protien interactions for these. 
```{r}
Interactions=read.table("../data/bioGRID/GeneswInteractions.txt",stringsAsFactors = F, header = T) 

OrthoUTR=read.table("../data/orthoUTR/HumanDistal3UTR.sort.bed", col.names = c("chr",'start','end','gene','score','strand'),stringsAsFactors = F) %>% mutate(length=end-start) %>% select(gene, length)


InteractionsAPA=Interactions %>%filter(gene %in% NucAPAresSig$gene) %>% mutate(dPnotE=ifelse(gene %in% NucAPAres_DP$gene, "Yes", "No"))%>% inner_join(OrthoUTR, by="gene") %>% mutate(density=nInt/length)


ggplot(InteractionsAPA,aes(x=dPnotE, y=log10(nInt+1),fill=dPnotE)) + geom_boxplot(notch = T) + stat_compare_means() + scale_fill_brewer(palette = "Set1")+theme(axis.text.x=element_text(angle=90, hjust=0), text= element_text(size=16), legend.position = "false") + labs(x="Gene in Expression independent set", y="log10(Number of Protein Interactions)", title="Protein Interactions for Expression \nindependent dAPA genes")
```
Plot density?  


```{r}
ggplot(InteractionsAPA,aes(x=dPnotE, y=log10(density),fill=dPnotE)) + geom_boxplot(notch = T) + stat_compare_means() + scale_fill_brewer(palette = "Set1")+theme(axis.text.x=element_text(angle=90, hjust=0), text= element_text(size=16), legend.position = "false") + labs(x="Gene in Expression independent set", y="log10(UTR density of interactions)", title="Protein Interactions for Expression \nindependent dAPA genes")
```



More likly to have one:  

```{r}
InteractionsAPA %>% mutate(HasInteraction=ifelse(nInt>0, "Yes", "No")) %>% group_by(dPnotE, HasInteraction) %>% summarise(nWithSet=n())
```

Set should be the interaction set dAPA, de, and dP. 

```{r}
Alldiff=Protein %>% inner_join(DEgenes,by="gene") %>% inner_join(NucAPA, by="gene") %>% dplyr::select(gene)
#This is 101 genes.  
geneAPAPnotEG=APAandPnotE %>% dplyr::select(gene)

GenesMatter= Alldiff %>% bind_rows(geneAPAPnotEG) %>% mutate(Ex=ifelse(gene %in% geneAPAPnotEG$gene, "No", "Yes")) %>% inner_join(Interactions, by="gene")
```


```{r}
ggplot(GenesMatter, aes(x=Ex, y=nInt, fill=Ex))+ geom_boxplot() + stat_compare_means() + scale_fill_brewer(palette = "RdYlBu")+theme(axis.text.x=element_text(angle=90, hjust=0), text= element_text(size=16), legend.position = "false") + labs(x="DE gene", y="Number of protein protein interactions", title="dAPA, DP, and DE")
```


Effect sizes : 

Look at the PAS effect sizes here and in protien, translation, and expression. 
```{r}
NucAPAres_sig_dpnotE = NucAPAres_sig %>% filter(dPnotE =="Yes")

#nameID=read.table("../../genome_anotation_data/ensemble_to_genename.txt",sep="\t", header = T, stringsAsFactors = F) %>% dplyr::select(Gene_stable_ID, Gene.name)
#DE data
DE=read.table("../data/DiffExpression/DEtested_allres.txt",stringsAsFactors = F,header = F, col.names = c("Gene_stable_ID" ,"logFC" ,"AveExpr" , "t" ,  "P.Value" ,  "adj.P.Val", "B"  )) %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::rename('gene'=Gene.name) %>% dplyr::select(-Gene_stable_ID)
#translation
Ribo=read.table("../data/Wang_ribo/Additionaltable5_translationComparisons.txt",header = T, stringsAsFactors = F) %>% dplyr::rename("Gene_stable_ID"= ENSG) %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::select(Gene.name, HvC.beta, HvC.pvalue, HvC.FDR) %>% dplyr::rename("gene"=Gene.name)
#prot  
Prot=read.table("../data/Khan_prot/ProtData_effectSize.txt", header = T, stringsAsFactors = F)
```


```{r}
APAandE=NucAPAres_sig_dpnotE %>% inner_join(DE, by="gene")

ggplot(APAandE, aes(x=logFC, y=deltaPAU)) + geom_point(alpha=.3) + geom_smooth(method="lm") +stat_cor()


ggplot(APAandE, aes(x=logFC, y=deltaPAU, col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")  +stat_cor(label.x = 1)

APAandRibo=NucAPAres_sig_dpnotE %>% inner_join(Ribo, by="gene")
ggplot(APAandRibo, aes(x=HvC.beta, y=deltaPAU)) + geom_point(alpha=.3) + geom_smooth(method="lm") +stat_cor()

APAandprot=NucAPAres_sig_dpnotE %>% inner_join(Prot, by="gene")

ggplot(APAandprot, aes(x=logEf, y=deltaPAU))+ geom_point(alpha=.3) + geom_smooth(method="lm") +stat_cor( )
ggplot(APAandprot, aes(x=logEf, y=deltaPAU, col=loc))+ geom_point(alpha=.3) + geom_smooth(method="lm") +stat_cor( )
```
None of these are significant.  


Check if any of these are genes with QTLs.  

I will pull in the genes with nuclear apaQTLs first.  

```{r}
apaQTLs=read.table("../../apaQTL/data/apaQTLs/Nuclear_apaQTLs4pc_5fdr.sort.bed",col.names = c('chr','start','end', 'PASid','score', 'strand')) %>% separate(PASid, into=c("gene", "PAS", "loc"),sep=":")

apaQTLGenes= apaQTLs %>% select(gene) %>% unique()
```


```{r}
APAandPnotE_apaQTL=APAandPnotE %>% mutate(apaQTL=ifelse(gene %in% apaQTLGenes$gene, "Yes", "No"))


APAandPnotE_apaQTL %>% group_by(apaQTL) %>% summarize(n=n())


APAandPnotE_apaQTL %>% filter(apaQTL=="Yes")
```

Background for enrichment is all of the dAPA genes.  
```{r}
x= nrow(APAandPnotE_apaQTL %>% filter(apaQTL =="Yes"))
m= nrow(APAandPnotE_apaQTL)
n=nrow(NucAPA)- nrow(APAandPnotE_apaQTL)
k=nrow(apaQTLGenes)
x

#expected
which(grepl(max(dhyper(1:x, m, n, k)), dhyper(1:x, m, n, k)))

phyper(x,m, n , k,lower.tail=F)

```
Not enriched for apaQTL.

pQTLs  

Using protien specific QTLs from Battle et al. 
```{r}
pQTLs=read.table("../../apaQTL/data/Battle_pQTL/psQTLGeneNames.txt")

APAandPnotE_pQTL=APAandPnotE %>% mutate(pQTL=ifelse(gene %in% pQTLs$V1, "Yes", "No"))


APAandPnotE_pQTL %>% group_by(pQTL) %>% summarize(n=n())

APAandPnotE_pQTL %>% filter(pQTL=="Yes")
```


```{r}
x= nrow(APAandPnotE_pQTL %>% filter(pQTL =="Yes"))
m= nrow(APAandPnotE_pQTL)
n=nrow(NucAPA)- nrow(APAandPnotE_pQTL)
k=nrow(pQTLs)
x

#expected
which(grepl(max(dhyper(1:x, m, n, k)), dhyper(1:x, m, n, k)))

phyper(x,m, n , k,lower.tail=F)

```


Are any of the these the diff dom set? Test .4 first: 

```{r}

HumanRes=read.table("../data/DomDefGreaterX/Human_AllGenes_DiffTop.txt", col.names = c("Human_PAS", "gene","Human_DiffDom"),stringsAsFactors = F)

ChimpRes=read.table("../data/DomDefGreaterX/Chimp_AllGenes_DiffTop.txt", col.names = c("Chimp_PAS", "gene","Chimp_DiffDom"),stringsAsFactors = F)

BothRes=HumanRes %>% inner_join(ChimpRes,by="gene")

BothRes_40=BothRes %>% filter(Chimp_DiffDom >=0.4 | Human_DiffDom>=0.4) %>% mutate(Set= ifelse(Human_PAS==Chimp_PAS,"Same", "Different"),cut=40)


NucAPAres_sig_sm= NucAPAres_sig %>% filter(dPnotE=="Yes")
BothRes_40_dp= BothRes_40 %>% filter(gene %in% NucAPAres_sig_sm$gene)


BothRes_40_dp %>% group_by(Set) %>% summarise(n())


metaSm= Meta %>% select(loc, PAS)
DiffHuman= BothRes_40_dp %>% filter(Set=="Different") %>% select(gene, Human_PAS)  %>% rename(PAS= Human_PAS)%>% inner_join(metaSm, by="PAS")

DiffChimp= BothRes_40_dp %>% filter(Set=="Different") %>% select(gene, Chimp_PAS)%>% rename(PAS= Chimp_PAS)%>% inner_join(metaSm, by="PAS")

```

```{r}
DiffHuman
DiffChimp
```

