---
title: "Conservation Questions"
author: "Briana Mittleman"
date: "2/28/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

In my initial exploration of dAPA PAS I saw they are enriched for negative phylop scores. I will explore this trend further here. I will see if intron location explain the differences.  

```{r}
library(tidyverse)
library(ggpubr)
library(reshape2)
```

```{r}
DiffUsage=read.table("../data/DiffIso_Nuclear_DF/SignifianceEitherPAS_2_Nuclear.txt", header = T, stringsAsFactors = F)

PASMeta=read.table("../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", header = T, stringsAsFactors = F) %>% dplyr::select(PAS, chr, start,end, gene, loc)

DiffUsagePAS=DiffUsage %>% inner_join(PASMeta, by=c("gene","chr", "start", "end"))
```

```{r}
phylores=read.table("../data/PhyloP/PAS_phyloP.txt", col.names = c("chr","start","end", "phyloP"), stringsAsFactors = F) %>% drop_na()
NucReswPhy=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T, stringsAsFactors = F) %>% inner_join(phylores, by=c("chr","start","end"))
```


```{r}
ggplot(NucReswPhy,aes(y=phyloP, x=SigPAU2,fill=SigPAU2)) + geom_boxplot() + stat_compare_means()+ scale_fill_brewer(palette = "Dark2", name="Signficant")


```

```{r}

ggplot(NucReswPhy,aes(x=phyloP, by=SigPAU2, fill=SigPAU2)) + geom_density(alpha=.5) + scale_fill_brewer(palette = "Dark2", name="Signficant PAS") + labs(title="Mean PhyloP scores for tested PAS") + annotate("text",label="Wilcoxan, p=1.4e -5",x=6,y=.75)
```


The significant PAS have on average lower phyloP scores. 


Positive scores — Measure conservation, which is slower evolution than expected, at sites that are predicted to be conserved.
Negative scores — Measure acceleration, which is faster evolution than expected, at sites that are predicted to be fast-evolving.


I can look at those with negative values:  

```{r}
x=nrow(NucReswPhy %>% filter(SigPAU2=="Yes", phyloP<0))
m= nrow(NucReswPhy %>% filter(phyloP<0))
n=nrow(NucReswPhy %>% filter(phyloP>=0))
k=nrow(NucReswPhy %>% filter(SigPAU2=="Yes"))


#expected
which(grepl(max(dhyper(1:x, m, n, k)), dhyper(1:x, m, n, k)))

#actual:
x

#pval
phyper(x,m,n,k,lower.tail=F)
```

```{r}
b=nrow(NucReswPhy %>% filter(SigPAU2=="Yes", phyloP<0))
n=nrow(NucReswPhy %>% filter(SigPAU2=="Yes"))
B=nrow(NucReswPhy %>% filter(phyloP<0))
N=nrow(NucReswPhy)

(b/n)/(B/N)
```



This means these regions are more likely to be fast evolving.  

Look at this by location: (is it driven by region)

```{r}
NucReswPhy_meta= NucReswPhy %>% inner_join(PASMeta, by=c("chr", "start", "end", "gene"))

ggplot(NucReswPhy_meta,aes(x=phyloP, by=SigPAU2, fill=SigPAU2)) + geom_density(alpha=.5) + scale_fill_brewer(palette = "Dark2") + facet_grid(~loc)
```
```{r}
NucReswPhy_meta_group=NucReswPhy_meta %>% group_by(loc,SigPAU2) %>% summarise(n=n(),meanPhylo=mean(phyloP))
NucReswPhy_meta_group
```

###Control sequence:upstream 

(upstream 200)

Look at the 200 basepairs upstream of each PAS as a control. 


```{r}
metaStrand=read.table("../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", header = T, stringsAsFactors = F) %>% select(chr, start,end, strandFix, PAS)

NucReswPhy_upstream=NucReswPhy %>% inner_join(metaStrand,by=c("chr", "start", "end")) %>% mutate(newStart=ifelse(strandFix=="+", start - 200, end), newEnd=ifelse(strandFix=="+", start, end +200))

NucReswPhy_upstreambed=NucReswPhy_upstream %>% select(chr, newStart, newEnd, PAS, Human, strandFix)

write.table(NucReswPhy_upstreambed,"../data/PhyloP/PAS_200upregions.bed",col.names = F,row.names = F,quote = F,sep="\t")
```


```{bash,eval=F}
python extractPhylopReg200up.py

```


```{r}
Phylo200UpContron=read.table("../data/PhyloP/PAS_phyloP_200upstream.txt",stringsAsFactors = F, col.names = c("chr", "start","end", "PAS","UpstreamControl_Phylop")) 

NucReswPhyandC=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T, stringsAsFactors = F) %>% inner_join(phylores, by=c("chr", "start","end")) %>% inner_join(metaStrand,by=c("chr", "start", "end"))%>% inner_join(Phylo200UpContron, by="PAS")  %>% drop_na()

NucReswPhyandCsmall=NucReswPhyandC %>% select(PAS,SigPAU2,phyloP ,UpstreamControl_Phylop ) %>% gather("set", "Phylop", -PAS, -SigPAU2)

wilcox.test(NucReswPhyandC$phyloP, NucReswPhyandC$UpstreamControl_Phylop, alternative = "greater")
```
Actual are greater than region upstream  

```{r}
ggplot(NucReswPhyandCsmall, aes(x=SigPAU2, by=set, fill=set, y=Phylop)) + geom_boxplot() + scale_fill_brewer(palette = "Dark2",labels=c('PAS', 'Control') ) + stat_compare_means()

```


```{r}
NucReswPhyandCsmall_noc= NucReswPhyandCsmall %>% filter(set!="UpstreamControl_Phylop")

ggplot(NucReswPhyandCsmall_noc, aes(x=SigPAU2, fill=SigPAU2, y=Phylop )) + geom_boxplot() + stat_compare_means() + scale_fill_brewer(palette = "OrRd") + labs(title="Significant PAS",x="")+ scale_x_discrete(labels=c("Not Significant", "Signficant"))+ theme(legend.position = "none",text= element_text(size=16))
```

Significant are lower than not significant:  

```{r}
NucReswPhyandCsmall_nocYES= NucReswPhyandCsmall_noc %>% filter(SigPAU2=="Yes")
NucReswPhyandCsmall_nocNO= NucReswPhyandCsmall_noc %>% filter(SigPAU2=="No")

wilcox.test(NucReswPhyandCsmall_nocYES$Phylop, NucReswPhyandCsmall_nocNO$Phylop, alternative ="less")
```

Significant have lower scores.  
 

##Negative enrichment 

Number of negative in each set?

```{r}
neg=NucReswPhyandCsmall %>% filter(Phylop <0) %>% group_by(set, SigPAU2) %>% summarise(nNeg=n())

pos=NucReswPhyandCsmall %>% filter(Phylop >0) %>% group_by(set, SigPAU2) %>% summarise(nPos=n())

both=neg %>% inner_join(pos,by= c('set', 'SigPAU2')) %>% mutate(PropNeg=nNeg/(nNeg+nPos))

both
```

More negative overall in actual. Is there an enrichment for negative in the control set?  

```{r}
x=nrow(NucReswPhyandC %>% filter(SigPAU2=="Yes", UpstreamControl_Phylop<0))
m= nrow(NucReswPhyandC %>% filter(UpstreamControl_Phylop<0))
n=nrow(NucReswPhyandC %>% filter(UpstreamControl_Phylop>=0))
k=nrow(NucReswPhyandC %>% filter(SigPAU2=="Yes"))


#expected
which(grepl(max(dhyper(1:x, m, n, k)), dhyper(1:x, m, n, k)))

#actual:
x

#pval
phyper(x,m,n,k,lower.tail=F)
```
```{r}
b=nrow(NucReswPhyandC %>% filter(SigPAU2=="Yes", UpstreamControl_Phylop<0))
n=nrow(NucReswPhyandC %>% filter(SigPAU2=="Yes"))
B=nrow(NucReswPhyandC %>% filter(UpstreamControl_Phylop<0))
N=nrow(NucReswPhyandC)

(b/n)/(B/N)
```

Stronger enrichement in the for negative in the real results compared to contol.
1.07x in control 1.11x in actual.



Maybe I need to move the control further up. 

Is this a better control? Dont want to go into an exon? What about downstream?  

##Control sequence (downstream 200): 

```{r}

NucReswPhy_downstream=NucReswPhy %>% inner_join(metaStrand,by=c("chr", "start", "end")) %>% mutate(newStart=ifelse(strandFix=="+",  end, start-200), newEnd=ifelse(strandFix=="+", end+200, start))

NucReswPhy_downstreambed=NucReswPhy_downstream %>% select(chr, newStart, newEnd, PAS, Human, strandFix)

write.table(NucReswPhy_downstreambed,"../data/PhyloP/PAS_200downpregions.bed",col.names = F,row.names = F,quote = F,sep="\t")
```



```{bash,eval=F}
python extractPhylopReg200down.py

```


```{r}
Phylo200downCont=read.table("../data/PhyloP/PAS_phyloP_200downstream.txt",stringsAsFactors = F, col.names = c("chr", "start","end", "PAS","DownstreamControl_Phylop")) 

NucReswPhyandbothC=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T, stringsAsFactors = F) %>% inner_join(phylores, by=c("chr", "start","end")) %>% inner_join(metaStrand,by=c("chr", "start", "end"))%>% inner_join(Phylo200UpContron, by="PAS")  %>% drop_na() %>%  inner_join(Phylo200downCont, by="PAS")  %>% drop_na()

NucReswPhyandCbothsmall=NucReswPhyandbothC %>% select(PAS,SigPAU2,phyloP ,UpstreamControl_Phylop,DownstreamControl_Phylop ) %>% gather("set", "Phylop", -PAS, -SigPAU2) %>% drop_na()


#difference in controls? 
wilcox.test(NucReswPhyandbothC$DownstreamControl_Phylop, NucReswPhyandbothC$UpstreamControl_Phylop,alternative = "greater")


```


```{r}

levels=NucReswPhyandCbothsmall$set %>% unique()
NucReswPhyandCbothsmall$set= factor(NucReswPhyandCbothsmall$set, levels = c("UpstreamControl_Phylop", "phyloP", "DownstreamControl_Phylop"))

my_comparisons <- list( c("DownsreamControl_Pylop", "phylopP"), c("DownsreamControl_Pylop", "UpstreamControl_Phylop"), c("phylopP", "UpstreamControl_Phylop") )
ggplot(NucReswPhyandCbothsmall, aes(x=set, by=set, fill=set, y=Phylop)) + geom_boxplot() + scale_fill_brewer(palette = "Dark2",labels=c("Upstream Control", "PAS", "Downstream Control") ) + stat_compare_means(ref.group = "phyloP",paired = FALSE,label = "p.signif")  + labs(x="", title="PAS conserved compared to surrounding regions" ) + scale_x_discrete( labels=c("Upstream Control", "PAS", "Downstream Control"))+ theme(legend.position = "none",text= element_text(size=16))
```

Same here. The actual region looks more conserved. 

```{r}
ggplot(NucReswPhyandCbothsmall, aes(x=SigPAU2, by=set, fill=set, y=Phylop)) + geom_boxplot() + scale_fill_brewer(palette = "Dark2",labels=c("Downstream Control", "PAS", "Upstream Control") )
```




###Extend 
2 more blocks up and downstream to add to plot. 

Extend downstream: 
```{r}

NucReswPhy_downstream2=NucReswPhy_downstream %>% mutate(newStart2=ifelse(strandFix=="+",  newEnd, newStart-200), newEnd2=ifelse(strandFix=="+", newEnd+200, newStart))

NucReswPhy_downstream2bed=NucReswPhy_downstream2 %>% select(chr, newStart2, newEnd2, PAS, Human, strandFix)

write.table(NucReswPhy_downstream2bed,"../data/PhyloP/PAS_200downpregions2.bed",col.names = F,row.names = F,quote = F,sep="\t")

NucReswPhy_downstream3=NucReswPhy_downstream2 %>% mutate(newStart3=ifelse(strandFix=="+",  newEnd2, newStart2-200), newEnd3=ifelse(strandFix=="+", newEnd2+200, newStart2))

NucReswPhy_downstream3bed=NucReswPhy_downstream3 %>% select(chr, newStart3, newEnd3, PAS, Human, strandFix)

write.table(NucReswPhy_downstream3bed,"../data/PhyloP/PAS_200downpregions3.bed",col.names = F,row.names = F,quote = F,sep="\t")
```

Extend upstream:  


```{r}
NucReswPhy_upstream2=NucReswPhy_upstream %>%  mutate(newStart2=ifelse(strandFix=="+", newStart - 200, newEnd), newEnd2=ifelse(strandFix=="+", newStart, newEnd +200))

NucReswPhy_upstreambed2=NucReswPhy_upstream2 %>% select(chr, newStart2, newEnd2, PAS, Human, strandFix)

write.table(NucReswPhy_upstreambed2,"../data/PhyloP/PAS_200upregions2.bed",col.names = F,row.names = F,quote = F,sep="\t")

NucReswPhy_upstream3=NucReswPhy_upstream2 %>%  mutate(newStart3=ifelse(strandFix=="+", newStart2 - 200, newEnd2), newEnd3=ifelse(strandFix=="+", newStart2, newEnd2 +200))

NucReswPhy_upstreambed3=NucReswPhy_upstream3 %>% select(chr, newStart3, newEnd3, PAS, Human, strandFix)

write.table(NucReswPhy_upstreambed3,"../data/PhyloP/PAS_200upregions3.bed",col.names = F,row.names = F,quote = F,sep="\t")
```

Run phylop for each of these:  

```{bash,eval=F}
python extractPhylopGeneral.py ../data/PhyloP/PAS_200downpregions2.bed ../data/PhyloP/PAS_phyloP_200downstream2.txt

python extractPhylopGeneral.py ../data/PhyloP/PAS_200downpregions3.bed ../data/PhyloP/PAS_phyloP_200downstream3.txt

python extractPhylopGeneral.py ../data/PhyloP/PAS_200upregions2.bed ../data/PhyloP/PAS_phyloP_200upstream2.txt

python extractPhylopGeneral.py ../data/PhyloP/PAS_200upregions3.bed ../data/PhyloP/PAS_phyloP_200upstream3.txt

```



```{r}
ResUpdown=NucReswPhyandbothC %>% select(PAS,SigPAU2,phyloP ,UpstreamControl_Phylop,DownstreamControl_Phylop ) 


Down2=read.table("../data/PhyloP/PAS_phyloP_200downstream2.txt",col.names = c("chr", "start", "end", "PAS", "Down2"),stringsAsFactors = F) %>% select(PAS, Down2)%>% drop_na()
Down3=read.table("../data/PhyloP/PAS_phyloP_200downstream3.txt",col.names = c("chr", "start", "end", "PAS", "Down3"),stringsAsFactors = F) %>% select(PAS, Down3)%>% drop_na()

Up2=read.table("../data/PhyloP/PAS_phyloP_200upstream2.txt",col.names = c("chr", "start", "end", "PAS", "Up2"),stringsAsFactors = F) %>% select(PAS, Up2)%>% drop_na()
Up3=read.table("../data/PhyloP/PAS_phyloP_200upstream3.txt",col.names = c("chr", "start", "end", "PAS", "Up3"),stringsAsFactors = F) %>% select(PAS, Up3)%>% drop_na()

ResUpdownAll= ResUpdown %>% inner_join(Down2, by="PAS")%>% inner_join(Down3, by="PAS") %>% inner_join(Up2, by="PAS") %>% inner_join(Up3, by="PAS")

ResUpdownAll_gather= ResUpdownAll %>% gather("Set", "PhyloP", -PAS, -SigPAU2)

ResUpdownAll_gather$Set=factor(ResUpdownAll_gather$Set, levels=c("Up3", "Up2","UpstreamControl_Phylop", "phyloP","DownstreamControl_Phylop", "Down2", "Down3" ))

```

```{r}
ggplot(ResUpdownAll_gather, aes(x=Set, by=Set, fill=Set, y=PhyloP)) + geom_boxplot() + scale_fill_brewer(palette = "Dark2") + scale_x_discrete(labels=c("-600", "-400", "-200", '0','200','400','600')) + labs(x="Basepairs", title="PAS are more conserved than surrounding regions") + theme(legend.position = "none")
```

```{r}
ggplot(ResUpdownAll_gather, aes(x=Set, by=Set, fill=Set, y=PhyloP)) + geom_boxplot() + scale_fill_brewer(palette = "Dark2") + facet_grid(~SigPAU2)+ scale_x_discrete(labels=c("-600", "-400", "-200", '0','200','400','600')) + labs(x="Basepairs", title="PAS are more conserved than surrounding regions") + theme(legend.position = "none")
```


Change colors: 

```{r}
ggplot(ResUpdownAll_gather, aes(x=Set, by=Set, fill=Set, y=PhyloP)) + geom_boxplot() + scale_fill_brewer(palette = "RdYlBu") + scale_x_discrete(labels=c("-600", "-400", "-200", '0','200','400','600')) + labs(x="Basepairs", title="PAS are more conserved than surrounding regions") + theme(legend.position = "none")+ theme_classic()+ guides(fill = FALSE)
```

Color just PAS and surrounding: 

```{r} 
ResUpdownAll_gather2= ResUpdownAll_gather %>% mutate(region=ifelse(Set=="phyloP", "Yes", "No"))

ggplot(ResUpdownAll_gather2, aes(x=Set, by=Set, fill=region, y=PhyloP)) + geom_boxplot(notch = T) + scale_fill_brewer(palette = "RdYlBu") + scale_x_discrete(labels=c("-600", "-400", "-200", '0','200','400','600')) + labs(x="Basepairs", title="PAS are more conserved than surrounding regions") + guides(fill = FALSE) + theme_classic()
```

```{r, plotcons, include=FALSE, dev="pdf",crop=FALSE}
ggplot(ResUpdownAll_gather2, aes(x=Set, by=Set, fill=region, y=PhyloP)) + geom_boxplot(notch = T) + scale_fill_brewer(palette = "RdYlBu") + scale_x_discrete(labels=c("-600", "-400", "-200", '0','200','400','600')) + labs(x="Basepairs", title="PAS are more conserved than surrounding regions") + guides(fill = FALSE) + theme_classic()
```

