---
title: "Differential Expression and APA"
author: "Briana Mittleman"
date: "1/6/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

```{r}
library(tidyverse)
library(workflowr)
```

In this analysis I will look at the overlap between DE genes and eQTLs identified in Li et al 2016. This is similar to the anaylsis I did for dAPA and I will use this as a way to benchmark those results.  


```{r}
explained=read.table("../data/Li_eqtls/explainedEgenes.txt",col.names = c("Gene.name"),stringsAsFactors = F)
unexplained=read.table("../data/Li_eqtls/UnexplainedEgenes.txt",col.names = c("Gene.name"),stringsAsFactors = F)
```


```{r}
nameID=read.table("../../genome_anotation_data/ensemble_to_genename.txt",sep="\t", header = T, stringsAsFactors = F)


DEgenes=read.table("../data/DiffExpression/DE_genes.txt", header = F,col.names = c("Gene_stable_ID"),stringsAsFactors = F) %>% inner_join(nameID, by="Gene_stable_ID") %>% dplyr::select(Gene.name)
```


```{r}
DE_explained=DEgenes %>% inner_join(explained, by="Gene.name")
DE_unexplained=DEgenes %>% inner_join(unexplained, by="Gene.name")
neither=DEgenes %>% anti_join(explained, by="Gene.name") %>% anti_join(unexplained, by="Gene.name")


cat=c("Explained", "Unexaplained", "Neither")
num=c(nrow(DE_explained), nrow(DE_unexplained), nrow(neither))

DEqtl=as.data.frame(cbind(cat,num))

DEqtl$cat=factor(DEqtl$cat, levels=c("Explained","Unexaplained", "Neither"), ordered=T)
DEqtl$num=as.numeric(as.character(DEqtl$num ))

ggplot(DEqtl,aes(x=cat, y=num)) +geom_bar(stat="identity") + geom_text(aes(label=num), vjust=1.6, color="white", size=3.5) + labs(title="Differential Expression and human eQTL", y="Number of genes", x="Overlap")
```

Proportion:  

```{r}
DEqtlProp=DEqtl %>% mutate(Prop=num/nrow(DEgenes))

ggplot(DEqtlProp,aes(x=cat, y=Prop)) +geom_bar(stat="identity") + geom_text(aes(label=round(Prop,digits = 3)), vjust=1.6, color="white", size=3.5) + labs(title="Differential Expression and human eQTL", y="Proportion of DE genes", x="Overlap")
```

Look for enrichment:

```{r}

permuteGenes <- function(InputGenes, nGenes, nTests){
  #InputGenes=NuclearAPAtested_genes
  #nGenes=nrow(sigNuclear_genes)
  #nTests=100
  explained=read.table("../data/Li_eqtls/explainedEgenes.txt",col.names = c("genes"),stringsAsFactors = F)
  unexplained=read.table("../data/Li_eqtls/UnexplainedEgenes.txt",col.names = c("genes"),stringsAsFactors = F)
  explainedOverlap=c()
  unexplainedOverlap=c()
  neitherOverlap=c()
  for (n in 1:nTests){
    genesTest=sample(InputGenes, nGenes)
    overlapE=intersect(genesTest, explained$genes) 
    overlapUN=intersect(genesTest, unexplained$genes) 
    diffs <- Reduce(setdiff,list(A = genesTest, B = explained$genes,C = unexplained$genes))
    explainedOverlap= c(explainedOverlap, length(overlapE))
    unexplainedOverlap=c(unexplainedOverlap,length(overlapUN))
    neitherOverlap=c(neitherOverlap, length(diffs))
  }
  DF=as.data.frame(cbind(explainedOverlap,unexplainedOverlap,neitherOverlap))
  return(DF)
}
```

I need to have all of the genes tested for eGenes.  

```{r}
DEtestedgenes=read.table("../data/DiffExpression/DE_Testedgenes.txt", header = F,col.names = c("Gene_stable_ID"),stringsAsFactors = F) %>% inner_join(nameID, by="Gene_stable_ID") %>% dplyr::select(Gene.name)
```


Perform permutations 1000 times
```{r}
Expression_1000tests=permuteGenes(DEtestedgenes$Gene.name, nrow(DEgenes),1000 )

EEx=Expression_1000tests %>% filter(explainedOverlap>=nrow(DE_explained)) %>% nrow() / 1000
EUnEx=Expression_1000tests %>% filter(unexplainedOverlap>=nrow(DE_unexplained)) %>% nrow() / 1000
```

Plot as histograms 

 Explained
```{r}

ggplot(Expression_1000tests,aes(x=explainedOverlap)) + geom_histogram(stat="count") + geom_vline(xintercept =nrow(DE_explained), col="red" )+labs(x="Number of Overlaps", title="DE overlap with explained eGenes") + annotate("text", x = 275, y = 30, label = paste("EmpPvalue=", EEx), col="red")

```
 Unexplained
```{r}
ggplot(Expression_1000tests,aes(x=unexplainedOverlap)) + geom_histogram(stat="count") + geom_vline(xintercept =nrow(DE_unexplained), col="red" ) +labs(x="Number of Overlaps", title="DE overlap with unexplained eGenes")+ annotate("text", x = 200, y = 30, label = paste("EmpPvalue=", EUnEx), col="red")
```
