---
title: "DIC not DE but DP"
author: "Briana Mittleman"
date: "5/7/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

```{r}
library(workflowr)
library(UpSetR)
library(cowplot)
library(ggpubr)
library(tidyverse)
```

I saw that dIC genes are enriched for translation. I want to look at the dIC genes that are not dAPA, are any of these DP but not DE?  

```{r}
Meta=read.table("../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", header = T, stringsAsFactors = F) 


Meta_genes= Meta %>% select(gene) %>% unique()

Meta_PAS=Meta %>%  select(PAS,gene)

dAPAGenes=read.table("../data/DiffIso_Nuclear_DF/SignifianceEitherGENES_Nuclear.txt", header = T, stringsAsFactors = F)
dAPAPAS=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T, stringsAsFactors = F) %>% inner_join(Meta, by=c("chr","start", "end","gene")) %>% select(PAS,gene,SigPAU2 ) 

dAPAPAS_genes= dAPAPAS %>% select(gene) %>% unique()

dAPATestedGenes= dAPAPAS  %>% select(gene) %>% unique() %>% mutate(dAPA=ifelse(gene %in% dAPAGenes$gene,"Yes", "No")) 

dICdata= read.table("../data/IndInfoContent/SimpsonMedianSignificance.txt", header = T, stringsAsFactors = F)%>% select(sIC,gene)

dAPAandDic= dICdata %>% inner_join(dAPATestedGenes,by="gene") %>% mutate(Both=ifelse(sIC=="Yes" & dAPA=="Yes", "Yes","No"),OnlyIC=ifelse(sIC=="Yes" & dAPA=="No", "Yes","No"),OnlyAPA=ifelse(sIC=="No" & dAPA=="Yes", "Yes","No"))

dIConly=dAPAandDic %>%filter(OnlyIC=="Yes")

```

How many of the dIConly genes are not DE? 

```{r}
nameID=read.table("../../genome_anotation_data/ensemble_to_genename.txt",sep="\t", header = T, stringsAsFactors = F) %>% dplyr::select(Gene_stable_ID, Gene.name)

DiffExp=read.table("../data/DiffExpression/DEtested_allres.txt",stringsAsFactors = F,header = F, col.names = c("Gene_stable_ID" ,"logFC" ,"AveExpr" , "t" ,  "P.Value" ,  "adj.P.Val", "B"  )) %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::rename('gene'=Gene.name) %>% dplyr::select(-Gene_stable_ID) %>% mutate(DE=ifelse(adj.P.Val<.05, "Yes", "No")) %>% select(gene,DE) %>% filter(DE=="Yes")

Prot= read.table("../data/Khan_prot/ProtData_effectSize.txt",header = T,stringsAsFactors = F)  %>% mutate(dP=ifelse(pval<0.05, "Yes", "No")) %>% filter(dP=="Yes")

Ribo=read.table("../data/Wang_ribo/Additionaltable5_translationComparisons.txt",header = T, stringsAsFactors = F) %>% rename("Gene_stable_ID"= ENSG) %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::select(Gene.name, HvC.beta, HvC.pvalue, HvC.FDR) %>% rename("gene"=Gene.name) %>% mutate(dTE=ifelse(HvC.FDR <0.05, "Yes","No")) %>% filter(dTE=="Yes")


RiboAll=read.table("../data/Wang_ribo/Additionaltable5_translationComparisons.txt",header = T, stringsAsFactors = F) %>% rename("Gene_stable_ID"= ENSG) %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::select(Gene.name, HvC.beta, HvC.pvalue, HvC.FDR) %>% rename("gene"=Gene.name) %>% mutate(dTE=ifelse(HvC.FDR <0.05, "Yes","No")) 
```

```{r}
listInput <- list(IC=dIConly$gene, DE=DiffExp$gene, DT=Ribo$gene, DP=Prot$gene)

upset(fromList(listInput), order.by = "freq", keep.order = T,empty.intersections = "on")
```

```{r}
listInput <- list(IC=dIConly$gene, DE=DiffExp$gene, DP=Prot$gene)

upset(fromList(listInput), order.by = "freq", keep.order = T,empty.intersections = "on")
```



So 40 genes that are dP and dIC not DE.  

Proportion of dIC genes: 

```{r}
40/nrow(dIConly)
```

Try only dAPA: 

```{r}
dAPAonly=dAPAandDic %>%filter(OnlyAPA=="Yes")
nrow(dAPAonly)
```
```{r}
listInput <- list(dAPA=dAPAonly$gene, DE=DiffExp$gene, DP=Prot$gene)

upset(fromList(listInput), order.by = "freq", keep.order = T,empty.intersections = "on")
```
76 in this set. 

```{r}
76/nrow(dAPAonly)
```
 Smaller proportion. No sure how to test this. 
 

Both: 

```{r}
Both=dAPAandDic %>%filter(Both=="Yes")
nrow(Both)
```
```{r}
listInput <- list(BothAPA=Both$gene, DE=DiffExp$gene, DP=Prot$gene)

upset(fromList(listInput), order.by = "freq", keep.order = T,empty.intersections = "on")
```
33 here:  

```{r}
33/nrow(Both)
```


Plot number in set and proprotion: 

```{r}
Set=c("OnlydIC", "Both", "OnlyAPA")
Number=c(40, 33,76)
SetSize=c(nrow(dIConly),nrow(Both),nrow(dAPAonly) )

useCOl <- c("#d73027", "#4575b4","#fee090")
DFres=data.frame(cbind(Set,Number,SetSize)) 
DFres$Number=as.numeric(as.character(DFres$Number))
DFres$SetSize=as.numeric(as.character(DFres$SetSize))

DFres_prop=DFres %>% mutate(Prop=Number/SetSize)

numberPlot=ggplot(DFres_prop,aes(x=Set, fill=Set, y=Number))+ geom_bar(stat="identity")+ scale_fill_manual(values=useCOl)+geom_text(aes(label=Number), position=position_dodge(width=0.9), vjust=2)+ theme(legend.position = "none") + labs(title="Number of dP not DE genes", y="Number of Genes",x="")+scale_x_discrete(labels=c(Both="Both", OnlyAPA="PAS\nLevel",OnlydIC= "Isoform\n Diversity"))

propPlot=ggplot(DFres_prop,aes(x=Set, fill=Set, y=Prop))+ geom_bar(stat="identity")+ scale_fill_manual(values=useCOl) + labs(title="Proportion of APA set that are dP not DE", y="Proportion of APA set",x="")+geom_text(aes(label=round(Prop,3)), position=position_dodge(width=0.9), vjust=2) + theme(legend.position = "none")+scale_x_discrete(labels=c(Both="Both", OnlyAPA="PAS\nLevel",OnlydIC= "Isoform\n Diversity"))

```



```{r}
numberprop=plot_grid(numberPlot,propPlot)
numberprop
```

Are there differences in number of protein for these sets? 

normalize interactions by length of gene.  
```{r}

ProtInfo=read.table("../data/PTM/ProtLength.txt", sep = "\t",stringsAsFactors = F,header = T,col.names = c("entry","organism", "nAA", "gene")) %>% select(nAA, gene)

Interactions=read.table("../data/bioGRID/GeneswInteractions.txt",stringsAsFactors = F, header = T) %>% inner_join(ProtInfo, by="gene")%>% mutate(NormInter=nInt/nAA)

#DiffExp$gene, DT=Ribo$gene, DP=Prot$gene

dAPAandDic_wP=dAPAandDic %>% mutate(dE=ifelse(gene %in%DiffExp$gene, "Yes", "No" ), dP=ifelse(gene %in%Prot$gene,"Yes","No" ), dPnotDE=ifelse(dE=="No"&dP=="Yes", "Yes","No")) %>% inner_join(Interactions, by="gene")

dAPAandDic_wP_dAPA= dAPAandDic_wP %>% filter(dAPA=="Yes")

dAPAandDic_wP_both= dAPAandDic_wP %>% filter(Both=="Yes")


dAPAandDic_wP_IC= dAPAandDic_wP %>% filter(OnlyIC=="Yes")
```

Plot for all 3 sets: 

```{r}
dapaProt=ggplot(dAPAandDic_wP_dAPA,aes(x=dPnotDE, y=log10(NormInter),fill=dPnotDE)) + geom_boxplot(notch = T) + stat_compare_means() + scale_fill_manual(values = c("grey", "#4575b4"))+theme(axis.text.x=element_text(angle=90, hjust=0), text= element_text(size=10), legend.position = "false") + labs(x="Expression independent set", y="log10(Normalized Interaction)", title="Protein Interactions \ngenes with site level differences") +geom_jitter(alpha=.1)


bothProt=ggplot(dAPAandDic_wP_both,aes(x=dPnotDE, y=log10(NormInter),fill=dPnotDE)) + geom_boxplot(notch = T) + stat_compare_means() + scale_fill_manual(values = c("grey", "#d73027"))+ theme(axis.text.x=element_text(angle=90, hjust=0), text= element_text(size=10), legend.position = "false") + labs(x="Expression independent set", y="log10(Normalized Interaction)", title="Protein Interactions \n with difference in both levles")+geom_jitter(alpha=.1)

dicProt=ggplot(dAPAandDic_wP_IC,aes(x=dPnotDE, y=log10(NormInter),fill=dPnotDE)) + geom_boxplot(notch = T) + stat_compare_means() + scale_fill_manual(values = c("grey", "#fee090"))+theme(axis.text.x=element_text(angle=90, hjust=0), text= element_text(size=10), legend.position = "false") + labs(x="Expression independent set", y="log10(Normalized Interaction)", title="Protein Interactions \n genes with isoform diversity differences")+geom_jitter(alpha=.1)

```

```{r}
interactionplots=plot_grid(bothProt,dapaProt,dicProt, nrow=1)
```
Plot full: 

```{r}
plot_grid(numberprop, interactionplots, nrow = 2)
```


##are these diff in translation:  
```{r}
nrow(RiboAll)
nrow(dAPAandDic_wP)
dAPAandDic_wP_trans= dAPAandDic_wP %>% inner_join(RiboAll,by="gene")



dAPAandDic_wP_trans_dpnote= dAPAandDic_wP_trans %>% filter(dPnotDE=="Yes") %>% select(gene,Both, OnlyIC, OnlyAPA,dTE)


dAPAandDic_wP_trans_dpnoteOnlyAPA=dAPAandDic_wP_trans_dpnote  %>% filter(OnlyAPA=="Yes") %>%  group_by(dTE) %>% summarise(ndTE=n()) %>% mutate(set="OnlyAPA")

dAPAandDic_wP_trans_dpnoteBoth=dAPAandDic_wP_trans_dpnote  %>% filter(Both=="Yes") %>%  group_by(dTE) %>% summarise(ndTE=n()) %>% mutate(set="Both")

dAPAandDic_wP_trans_dpnoteIC=dAPAandDic_wP_trans_dpnote  %>% filter(OnlyIC=="Yes") %>%  group_by(dTE) %>% summarise(ndTE=n()) %>% mutate(set="OnlyIC")


AllTenum= dAPAandDic_wP_trans_dpnoteOnlyAPA %>% bind_rows(dAPAandDic_wP_trans_dpnoteBoth) %>% bind_rows(dAPAandDic_wP_trans_dpnoteIC)
```

```{r}
numTE=ggplot(AllTenum, aes(x=set,by=dTE, y=ndTE,fill=set, alpha=dTE)) +geom_bar(stat="identity", position = "dodge") + labs(title="Most dP not dE genes are not dTE", y="Number of Genes",x="") + scale_fill_manual(values = useCOl ) + scale_alpha_manual(values=c(.6, 1)) + theme(legend.position = "bottom") + geom_text(aes(label=ndTE), position=position_dodge(width=0.9), vjust=1)+guides(fill = FALSE)+scale_x_discrete(labels=c(Both="Both", OnlyAPA="PAS\nLevel",OnlyIC= "Isoform\n Diversity"))

numTE
```
```{r}
propTe=ggplot(AllTenum, aes(x=set,by=dTE, y=ndTE,fill=set, alpha=dTE)) +geom_bar(stat="identity", position = "fill") + labs(title="Most dP not dE genes are not dTE", y="Proportion",x="") + scale_fill_manual(values = useCOl ) + scale_alpha_manual(values=c(.4, 1)) + theme(legend.position = "bottom")+guides(fill = FALSE)+scale_x_discrete(labels=c(Both="Both", OnlyAPA="PAS\nLevel",OnlyIC= "Isoform\n Diversity"))
propTe
```

Of 6477 we have data for each of the 3 phenotypes


from 149 to 142 
```{r}
dAPAandDic_wP_trans %>% filter(dPnotDE=="Yes") %>% filter(Both=="Yes" | OnlyIC=="Yes" | OnlyAPA=="Yes") %>% group_by(dTE) %>% summarise(n())
```
```{r}
teplots=plot_grid(numTE, propTe, nrow =1)
teplots
```


diff figures together: 

```{r}
fullplot=plot_grid(numberprop, teplots,interactionplots, nrow = 3)
fullplot
```
```{r,include=FALSE, eval=F}
pdf("../output/dpnotDE", height=12, width=14)
fullplot
dev.off()
```

```{r exandte, include=FALSE, dev="pdf", fig.height=12, fig.width=14, crop=FALSE}
fullplot
```


write data for main figures: 

```{r}
write.table(DFres_prop, "../output/FigureDF/DPnotDE.txt", col.names = T, row.names = F, quote = F)

write.table(AllTenum, "../output/FigureDF/DPnotDE_trans.txt", col.names = T, row.names = F, quote = F)

write.table(dAPAandDic_wP_dAPA, "../output/FigureDF/PinteractiondAPA.txt", col.names = T, row.names = F, quote = F)
write.table(dAPAandDic_wP_both, "../output/FigureDF/Pinteractionboth.txt", col.names = T, row.names = F, quote = F)
write.table(dAPAandDic_wP_IC, "../output/FigureDF/PinteractiondIC.txt", col.names = T, row.names = F, quote = F)


```

