---
title: "Differential Used Intronic"
author: "Briana Mittleman"
date: "2/23/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


```{r}
library(workflowr)
library(ggpubr)
library(tidyverse)
```

For this analysis I will look at the differentially used PAS in introns and ask if I can used information from DE and dribosome to better understand these. I subset intornic because I believe the intronic and utr mechanisms are different. 


```{r}
Meta=read.table("../data/PAS_doubleFilter/PAS_10perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", header = T, stringsAsFactors = F)  %>% select(PAS, chr, start,end, loc)
DiffIso= read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T,stringsAsFactors = F) %>% inner_join(Meta, by=c("chr", 'start','end')) %>% filter(loc %in% c("intron","utr3"))
DiffIsoSig= DiffIso %>% filter(SigPAU2=="Yes")


```

742 of the 11228 intronic PAS are significant. 

1659 of the 17012 3' UTR PAS are significant. 

I can compare the effect sizes with these genes in the DE.  

##Compare with expression
```{r}

nameID=read.table("../../genome_anotation_data/ensemble_to_genename.txt",sep="\t", header = T, stringsAsFactors = F) %>% select(Gene_stable_ID, Gene.name)

DE=read.table("../data/DiffExpression/DEtested_allres.txt",stringsAsFactors = F,header = F, col.names = c("Gene_stable_ID" ,"logFC" ,"AveExpr" , "t" ,  "P.Value" ,  "adj.P.Val", "B"  )) %>% inner_join(nameID,by="Gene_stable_ID") %>% rename('gene'=Gene.name) %>% select(-Gene_stable_ID)

```


First do all of the genes:  

```{r}
DeandAPA= DiffIso %>% inner_join(DE, by="gene")
```

This pas I will include each PAS 

```{r}
ggplot(DeandAPA,aes(y=deltaPAU, x=logFC,col=loc)) + geom_point(alpha=.3) + geom_smooth(aes(col=loc), method="lm") + labs(title="Intronic and 3' UTR APA v DE") + scale_color_brewer(palette = "Dark2")+ stat_cor(aes(color = loc), label.x = 3)

ggplot(DeandAPA,aes(y=deltaPAU, x=logFC)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="Intronic and 3' UTR APA v DE") + scale_color_brewer(palette = "Dark2")+ stat_cor( label.x = 3)
```




Just the genes with significant differences in PAS  

```{r}
DeandAPA_sigAPA= DeandAPA %>% filter(SigPAU2=="Yes")
```


```{r}
ggplot(DeandAPA_sigAPA,aes(y=deltaPAU, x=logFC, col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant Intronic and 3' UTR APA v DE")+ scale_color_brewer(palette = "Dark2")+ stat_cor(aes(color = loc), label.x = 3)


ggplot(DeandAPA_sigAPA,aes(y=deltaPAU, x=logFC)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant Intronic and 3' UTR APA v DE")+ scale_color_brewer(palette = "Dark2")+ stat_cor(label.x = 3)
```

Sig both:  


```{r}
DeandAPA_sigAPAandE= DeandAPA %>% filter(SigPAU2=="Yes",  adj.P.Val<.05)
```


```{r}
ggplot(DeandAPA_sigAPAandE,aes(y=deltaPAU, x=logFC,col=loc)) + geom_point() + geom_smooth(method="lm")+ labs(title="Significant Intronic and 3' UTR APA v Significant DE") + scale_color_brewer(palette = "Dark2")+ stat_cor(aes(color = loc), label.x = 3)

ggplot(DeandAPA_sigAPAandE,aes(y=deltaPAU, x=logFC)) + geom_point() + geom_smooth(method="lm")+ labs(title="Significant Intronic and 3' UTR APA v Significant DE") + scale_color_brewer(palette = "Dark2")+ stat_cor( label.x = 3)

```




###Choose most Sig PAS  

To break ties I will use the top average usage. I will not worry about location when chosing top PAS.  

```{r}
DeandAPA_topPAS= DeandAPA %>% mutate(AvgUsageBoth=(Human+Chimp)/2) %>% group_by(gene) %>% arrange(p.adjust,desc(AvgUsageBoth)) %>% slice(1) %>% ungroup()

#intron
nrow(DeandAPA_topPAS %>% filter(loc=="intron"))
nrow(DeandAPA_topPAS %>% filter(loc=="intron", SigPAU2=="Yes"))
#3 utr
nrow(DeandAPA_topPAS %>% filter(loc=="utr3"))
nrow(DeandAPA_topPAS %>% filter(loc=="utr3", SigPAU2=="Yes"))
```



from  11228 intronic to 1358 PAS (12%)
from 742 to 221 significant (30%)


from  17012 to 5993 3' UTR PAS. (35%)
from 1659 to 1088 significant (66%)

Plot the correlation in effect size  

```{r}
ggplot(DeandAPA_topPAS,aes(y=deltaPAU, x=logFC, col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")  + labs(title="Intronic and 3' UTR APA top PAS v DE") + scale_color_brewer(palette = "Dark2") + stat_cor(aes(color = loc), label.x = 3)
```




```{r}
DeandAPA_topPASsigAPA= DeandAPA_topPAS %>% filter(SigPAU2=="Yes")
```


```{r}
ggplot(DeandAPA_topPASsigAPA,aes(y=deltaPAU, x=logFC,col=loc)) + geom_point(alpha=.4) + geom_smooth(method="lm") + labs(title="Significant APA, Top PAS v DE ") + scale_color_brewer(palette = "Dark2")+ stat_cor(aes(color = loc), label.x = 3)
```



Sig both:  


```{r}
DeandAPA_topPASsigAPAandE= DeandAPA_topPASsigAPA %>% filter(SigPAU2=="Yes",  adj.P.Val<.05)
```


```{r}
ggplot(DeandAPA_topPASsigAPAandE,aes(y=deltaPAU, x=logFC,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA, Top PAS  v Significant DE") +  scale_color_brewer(palette = "Dark2")+ stat_cor(aes(color = loc), label.x = 3)

```


##Ribosome occupancy  

```{r}
Ribo=read.table("../data/Wang_ribo/Additionaltable5_translationComparisons.txt",header = T, stringsAsFactors = F) %>% rename("Gene_stable_ID"= ENSG) %>% inner_join(nameID,by="Gene_stable_ID") %>% select(Gene.name, HvC.beta, HvC.pvalue, HvC.FDR) %>% rename("gene"=Gene.name) 
```

Join with APA  

```{r}
RiboandAPA=DiffIso %>% inner_join(Ribo, by="gene")

RiboandAPA %>% group_by(gene) %>% n_distinct()

```

```{r}
ggplot(RiboandAPA,aes(y=deltaPAU, x=HvC.beta, col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="APA v Ribosome Occupany")+ scale_color_brewer(palette = "Dark2")+ stat_cor(aes(color = loc), label.x = 3)


ggplot(RiboandAPA,aes(y=deltaPAU, x=HvC.beta)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="APA v Ribosome Occupany")+ scale_color_brewer(palette = "Dark2")+ stat_cor( label.x = 3)
```



Just the genes with significant differences in PAS  

```{r}
RiboandAPA_sigAPA= RiboandAPA %>% filter(SigPAU2=="Yes")
```


```{r}
ggplot(RiboandAPA_sigAPA,aes(y=deltaPAU, x=HvC.beta,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA v Ribosome Occupany")+ scale_color_brewer(palette = "Dark2")+ stat_cor(aes(color = loc), label.x = 3)
```




Sig both:  


```{r}
RiboandAPA_sigAPAandR= RiboandAPA_sigAPA %>% filter(SigPAU2=="Yes",  HvC.FDR<.05)
```


```{r}
ggplot(RiboandAPA_sigAPAandR,aes(y=deltaPAU, x=HvC.beta,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA v Significant Ribosome Occupany") + scale_color_brewer(palette = "Dark2")+ stat_cor(aes(color = loc), label.x = 3)

```



The correlation in expression with intronic is not there in ribosome occupancy.  

###Choose most Sig PAS  

To break ties I will use the top average usage. I will not worry about location at first.  

```{r}
RiboandAPA_topPAS= RiboandAPA %>% mutate(AvgUsageBoth=(Human+Chimp)/2) %>% group_by(gene) %>% arrange(p.adjust,desc(AvgUsageBoth)) %>% slice(1) %>% ungroup()


nrow(RiboandAPA %>% filter(loc=="intron"))
nrow(RiboandAPA_topPAS %>% filter(loc=="intron"))


nrow(RiboandAPA %>% filter(loc=="utr3"))
nrow(RiboandAPA_topPAS %>% filter(loc=="utr3"))
```
UTR and ribo:
- All 13754
- in top used set- 5179

itron and ribo:
- All- 7405
- in top used set- 1180

Plot the correlation in effect size  

```{r}
ggplot(RiboandAPA_topPAS,aes(y=deltaPAU, x=HvC.beta,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")  + labs(title="APA, top PAS v Ribosome Occupany")+ scale_color_brewer(palette = "Dark2")+ stat_cor(aes(color = loc), label.x = 3)


```



Sig APA
```{r}
RiboandAPA_topPASsigAPA= RiboandAPA_topPAS %>% filter(SigPAU2=="Yes")

nrow(RiboandAPA_topPASsigAPA %>% filter(loc=="intron"))


nrow(RiboandAPA_topPASsigAPA %>% filter(loc=="utr3"))

```
192 intronic significant, 908 significant 3' utr

```{r}
ggplot(RiboandAPA_topPASsigAPA,aes(y=deltaPAU, x=HvC.beta,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant  APA, top PAS v Ribosome Occupany") +scale_color_brewer(palette = "Dark2")+ stat_cor(aes(color = loc), label.x = 3)

```




Sig both:  


```{r}
RiboandAPA_topPASsigAPAandR= RiboandAPA_topPASsigAPA %>% filter(SigPAU2=="Yes",  HvC.FDR<.05)

nrow(RiboandAPA_topPASsigAPAandR %>% filter(loc=="intron"))


nrow(RiboandAPA_topPASsigAPAandR %>% filter(loc=="utr3"))
```
43 PAS for intrnic 227 for 3' UTR

```{r}
ggplot(RiboandAPA_topPASsigAPAandR,aes(y=deltaPAU, x=HvC.beta,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant  APA, top PAS  v Significant Ribosome Occupany") + scale_color_brewer(palette = "Dark2")+ stat_cor(aes(color = loc), label.x = 3)

```


Correlation in UTR bnut not intronic. Not sure if this is due to the number of PAS.   
