---
title: "Differential Splicing"
author: "Briana Mittleman"
date: "11/11/2019"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


```{r}
library(tidyverse)
library(reshape2)
```

I want to use the RNA seq I collected to also perform a differential splicing analysis with leafcutter. I will follow the pipeline found at http://davidaknowles.github.io/leafcutter/articles/Usage.html. For a first pass I will use the bam files from the snakemake and differential expression analysis pipeline.  

I will get clusters in both species then perform reciprocal liftover. I can use a liftover pipeline similar to the one I used for the differnetial PAS analysis. 


Pipeline from example on leafcutter github.

```{bash,eval=F }


for bamfile in `ls run/geuvadis/*chr1.bam`; do
    echo Converting $bamfile to $bamfile.junc
    samtools index $bamfile
    regtools junctions extract -a 8 -m 50 -M 500000 $bamfile -o $bamfile.junc
    echo $bamfile.junc >> test_juncfiles.txt
done

python ../clustering/leafcutter_cluster_regtools.py -j test_juncfiles.txt -m 50 -o testYRIvsEU -l 500000
```

At this point I will be able to liftover the junctions. I can use the human corrdinates for the differential splicing step.  

```{bash,eval=F}
../scripts/leafcutter_ds.R --num_threads 4 ../example_data/testYRIvsEU_perind_numers.counts.gz example_geuvadis/groups_file.txt

```

I now have my RNA seq for each species. I can write a script that runs the junctions for each species.  


```{bash,eval=F}
sbatch converBam2Junc.sh
```


Create a script that only keeps the number chromosomes (2A and 2B for chimp). This means I will not have any of the chimp contigs.  


I should lift first then filter 

```{bash,eval=F}

mkdir ../data/DiffSplice_liftedJunc
sbatch liftJunctionFiles.sh

```


**liftover changes the junction files..**

For the humans I can simply filter the original junctions that pass the reciprocal liftover. For the chimps I need to figure out why the junctions change with liftover.  

Junction file format from regtools:
bed12 format:
- chrom
- start
- end
- junction name
- score- number of reads supporing junction
- strand
- thick start (same as chrom start)
- thick end (same as chrom end)
- itemRgb - default 255,0,0
- block cound- number blocks, defauld 2
- block size - comma- separated list of block sizes, number of items in list corresponds to blockCount
- block start- comma separated list of block starts, all block star positions should be calculated relative to chromStrat and the number of items in the list shoudl correspond to blockCount

I need to write a script that fixes the lifted files. I need to make column 9 255,0,0 and remove the commas from the last 2 columns. 

I can impliment the fix in the filter file. 
```{bash,eval=F}
sbatch runFilterNumChroms.sh
```


Make clusters: 

```{r}
juncfiles=read.table("../data/DiffSplice_liftedJunc/BothSpec_juncfiles.txt", header = F)
humanFiles=juncfiles %>% slice(1:6)
write.table(humanFiles, "../data/DiffSplice_liftedJunc/Human_juncfiles.txt", quote = F, col.names = F, row.names = F)
chimpFiles=juncfiles %>% slice(7:12)
write.table(chimpFiles, "../data/DiffSplice_liftedJunc/Chimp_juncfiles.txt", quote = F, col.names = F, row.names = F)
```

```{bash,eval=F}
sbatch quantJunc.sh
```


Now I can merge all of the culsters with: /project2/yangili1/yangili/leafcutter_scripts/merge_leafcutter_clusters.py


```{bash,eval=F}
sbatch MergeClusters.sh
sbatch QuantMergedClusters.sh
gunzip ../data/DiffSplice_liftedJunc/MergeCombined_perind_numers.counts.gz
```

Make the sample list:

```{r}
combinedCounts=read.table("../data/DiffSplice_liftedJunc/MergeCombined_perind_numers.counts", header=T)
x=colnames(combinedCounts)
#YG-BM-S8-18499H-Total_S8_R1_001-sort.bam
indiv=as.data.frame(x)  %>%  separate(x, into=c("yg", "bm","lane", "sample", "total",  "sort", "bam"), sep="[.]") %>% mutate(sample=paste(yg, "-", bm, "-", lane, "-", sample, "-", total, "-", sort, ".", bam, sep="")) %>% select(sample) %>%  mutate(Species=ifelse(grepl("H",sample), "Human", "Chimp"))


write.table(indiv, "../data/DiffSplice_liftedJunc/groups_file.txt", quote = F, col.names = F, row.names = F, sep = "\t")
```

fix to -Total instead of .Total


 
```{r}
counts=read.table('../data/DiffSplice_liftedJunc/MergeCombined_perind_numers.counts', header=T, check.names = F)
meta=read.table("../data/DiffSplice_liftedJunc/groups_file.txt", header=F, stringsAsFactors = F)
colnames(meta)[1:2]=c("sample","group")
counts=counts[,meta$sample]
#rownames(counts)
```

```{bash,eval=F}
gzip ../data/DiffSplice_liftedJunc/MergeCombined_perind_numers.counts
```

 
 I am going to write a python work around to change the cluster format. This will take the unziped version of the counts file. When I run it, I can unzip and zip the results in the bash script.  
 
 
```{bash,eval=F}

sbatch RunFixLeafCluster.sh

sbatch DiffSplice.sh



```
 
 
There are either not enough samples or min coverage problems.  Let me compare these to the numbers.  
```{r}
results=read.table("../data/DiffSplice_liftedJunc/MergedRes_cluster_significance.txt",stringsAsFactors = F, header = T, sep="\t") %>% separate(cluster, into=c("chrom", "clus"),sep=":") %>% filter(status=="Success") 
results$p.adjust=as.numeric(as.character(results$p.adjust))

results %>% filter(p.adjust < .05 ) %>% nrow()


results_sig= results %>% filter(p.adjust < .05 )
```


```{r}
qqplot(-log10(runif(nrow(results))), -log10(results$p.adjust),ylab="-log10 Total Adjusted Leafcutter pvalue", xlab="-log 10 Uniform expectation", main="Human Chimp differential splicing")
abline(0,1)
```
```{r}
effectsize=read.table("../data/DiffSplice_liftedJunc/MergedRes_effect_sizes.txt", stringsAsFactors = F,header=T)
plot(sort(effectsize$deltapsi), ,main="Human vs Chimp Effect sizes", ylab="Delta PSI", xlab="Cluster Index")
```


Use the leafcutter tool to visualize  

```{bash,eval=F}
sbatch DiffSplicePlots.sh 
```


prepare genes: 
```{r}
genes=results_sig %>% arrange(p.adjust) %>% dplyr::select(genes) %>% unique()
nrow(genes)
write.table(genes, "../data/DiffSplice_liftedJunc/orderedGenelist.txt", col.names = F, row.names = F, quote = F)

```


Build leafviz  

```{bash,eval=F}
sbatch prepareeafvizAnno.sh
sbatch buildLeafviz.sh
#with leafcutter annotation (https://github.com/davidaknowles/leafcutter/blob/master/leafviz/download_human_annotation_codes.sh)
sbatch buildLeafviz_leadAnno.sh
#classify clusters 
sbatch ClassifyLeafviz.sh
```

Run this from leafviz on my own computer,  

Convert juntions to bed to look at in IGV:

```{bash,eval=F}
sbatch ConvertJunc2Bed.sh
```

Lift chimp bams to look at in IGV as well.

Pantro5- hg38.  

```{bash,eval=F}
mkdir ../Chimp/data/RNAseq/Sort_hg38
sbatch CrossMapChimpRNA.sh 
```


Interesting spots:
- WSH3p= alt start
-RPL22 (NM_000983.3)
-RPL38
- chr3:129171277-129171446, NM_001127192.1, CNBP
-chr20:46394503-46406548 - ELMO2, NM_001318253.1
