---
title: "Differential Used Intronic"
author: "Briana Mittleman"
date: "2/23/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


```{r}
library(workflowr)
library(ggpubr)
library(limma)
library(qvalue)
library(tidyverse)
library(cowplot)
```

For this analysis I will look at the differentially used PAS in introns and ask if I can used information from DE and dribosome to better understand these. I subset intornic because I believe the intronic and utr mechanisms are different. 


```{r}
Meta=read.table("../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", header = T, stringsAsFactors = F)  %>% dplyr::select(PAS, chr, start,end, loc)
DiffIso= read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T,stringsAsFactors = F) %>% inner_join(Meta, by=c("chr", 'start','end')) %>% filter(loc %in% c("intron","utr3"))
DiffIsoSig= DiffIso %>% filter(SigPAU2=="Yes")


```


I can compare the effect sizes with these genes in the DE.  

##Compare with expression
```{r}

nameID=read.table("../../genome_anotation_data/ensemble_to_genename.txt",sep="\t", header = T, stringsAsFactors = F) %>% dplyr::select(Gene_stable_ID, Gene.name)

DE=read.table("../data/DiffExpression/DEtested_allres.txt",stringsAsFactors = F,header = F, col.names = c("Gene_stable_ID" ,"logFC" ,"AveExpr" , "t" ,  "P.Value" ,  "adj.P.Val", "B"  )) %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::rename('gene'=Gene.name) %>% dplyr::select(-Gene_stable_ID)

```


First do all of the genes:  

```{r}
DeandAPA= DiffIso %>% inner_join(DE, by="gene")
```

This pas I will include each PAS 

```{r}
ggplot(DeandAPA,aes(y=deltaPAU, x=logFC,col=loc)) + geom_point(alpha=.3) + geom_smooth(aes(col=loc), method="lm") + labs(title="Intronic and 3' UTR APA v DE") + scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 3)

ggplot(DeandAPA,aes(y=deltaPAU, x=logFC)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="Intronic and 3' UTR APA v DE") + scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 3)
```
```{r}

```




Just the genes with significant differences in PAS  

```{r}
DeandAPA_sigAPA= DeandAPA %>% filter(SigPAU2=="Yes")
```


```{r}
ggplot(DeandAPA_sigAPA,aes(y=deltaPAU, x=logFC, col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant Intronic and 3' UTR APA v DE")+ scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 3)


ggplot(DeandAPA_sigAPA,aes(y=deltaPAU, x=logFC)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant Intronic and 3' UTR APA v DE")+ scale_color_brewer(palette = "Set1")+ stat_cor(label.x = 3)
```

Sig both:  


```{r}
DeandAPA_sigAPAandE= DeandAPA %>% filter(SigPAU2=="Yes",  adj.P.Val<.05)
```


```{r}
ggplot(DeandAPA_sigAPAandE,aes(y=deltaPAU, x=logFC,col=loc)) + geom_point() + geom_smooth(method="lm")+ labs(title="Significant Intronic and 3' UTR APA v Significant DE") + scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 3)

ggplot(DeandAPA_sigAPAandE,aes(y=deltaPAU, x=logFC)) + geom_point() + geom_smooth(method="lm")+ labs(title="Significant Intronic and 3' UTR APA v Significant DE") + scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 3)

```




###Choose most Sig PAS  

To break ties I will use the top average usage. I will not worry about location when chosing top PAS.  

```{r}
DeandAPA_topPAS= DeandAPA %>% mutate(AvgUsageBoth=(Human+Chimp)/2) %>% group_by(gene) %>% arrange(p.adjust,desc(AvgUsageBoth)) %>% slice(1) %>% ungroup()

#intron
nrow(DeandAPA_topPAS %>% filter(loc=="intron"))
nrow(DeandAPA_topPAS %>% filter(loc=="intron", SigPAU2=="Yes"))
#3 utr
nrow(DeandAPA_topPAS %>% filter(loc=="utr3"))
nrow(DeandAPA_topPAS %>% filter(loc=="utr3", SigPAU2=="Yes"))
```



Plot the correlation in effect size  

```{r}
ggplot(DeandAPA_topPAS,aes(y=deltaPAU, x=logFC, col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")  + labs(title="Intronic and 3' UTR APA top PAS v DE") + scale_color_brewer(palette = "Set1") + stat_cor(aes(color = loc), label.x = 3)

ggplot(DeandAPA_topPAS,aes(y=deltaPAU, x=logFC)) + geom_point(alpha=.3) + geom_smooth(method="lm")  + stat_cor(label.x = 3)
```




```{r}
DeandAPA_topPASsigAPA= DeandAPA_topPAS %>% filter(SigPAU2=="Yes")
```


```{r}
ggplot(DeandAPA_topPASsigAPA,aes(y=deltaPAU, x=logFC,col=loc)) + geom_point(alpha=.4) + geom_smooth(method="lm") + labs(title="Significant APA, Top PAS v DE ") + scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 3)

ggplot(DeandAPA_topPASsigAPA,aes(y=deltaPAU, x=logFC)) + geom_point(alpha=.4) + geom_smooth(method="lm") + labs(title="Significant APA, Top PAS v DE ") + scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 3)
```



Sig both:  


```{r}
DeandAPA_topPASsigAPAandE= DeandAPA_topPASsigAPA %>% filter(SigPAU2=="Yes",  adj.P.Val<.05)
```


```{r}
ggplot(DeandAPA_topPASsigAPAandE,aes(y=deltaPAU, x=logFC,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA, Top PAS  v Significant DE") +  scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 3)

ggplot(DeandAPA_topPASsigAPAandE,aes(y=deltaPAU, x=logFC)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA, Top PAS  v Significant DE") +  scale_color_brewer(palette = "Set1")+ stat_cor(label.x = 3)

```

###Plot together: 


```{r}
allboth=ggplot(DeandAPA,aes(y=deltaPAU, x=logFC)) + geom_point(alpha=.3) + geom_smooth( method="lm") + labs(title="All APA and all expression") + scale_color_brewer(palette = "Set1")+ stat_cor(label.x = c(-6), label.y=-1) +theme_classic(base_size = 12) 
allSep= ggplot(DeandAPA_topPAS,aes(y=deltaPAU, x=logFC, col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")  + labs(title="All APA and all expression") + scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR")) + stat_cor(aes(color = loc), label.x = c(-8,1),label.y=-1)+theme_classic(base_size = 12) 
sigAPAboth=ggplot(DeandAPA_sigAPA,aes(y=deltaPAU, x=logFC)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA and all Expression")+ scale_color_brewer(palette = "Set1")+ stat_cor(label.x = -8,label.y=-1)+theme_classic(base_size = 12) 
sigAPSep=ggplot(DeandAPA_topPASsigAPA,aes(y=deltaPAU, x=logFC,col=loc)) + geom_point(alpha=.4) + geom_smooth(method="lm") + labs(title="Significant APA and all Expressio ") + scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR"))+ stat_cor(aes(color = loc), label.x = c(-8,1),label.y=-1)+theme_classic(base_size = 12) 
SigBoth= ggplot(DeandAPA_sigAPAandE,aes(y=deltaPAU, x=logFC)) + geom_point() + geom_smooth(method="lm")+ labs(title="Significant APA and Significant DE") + scale_color_brewer(palette = "Set1")+ stat_cor( label.x = -8,label.y=-1)+theme_classic(base_size = 12) 
SigSep=ggplot(DeandAPA_topPASsigAPAandE,aes(y=deltaPAU, x=logFC,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA and Significant DE") +  scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR"))+ stat_cor(aes(color = loc), label.x = c(-8,1),label.y=-1)+theme_classic(base_size = 12) 

```


```{r}
plot_grid(allboth,allSep,sigAPAboth,sigAPSep,SigBoth,SigSep, ncol=2)
```

```{r expression, include=FALSE, dev="pdf", fig.height=8, fig.width=11, crop=FALSE}

plot_grid(allboth,allSep,sigAPAboth,sigAPSep,SigBoth,SigSep, ncol=2)
```



##Ribosome occupancy  

```{r}
Ribo=read.table("../data/Wang_ribo/Additionaltable5_translationComparisons.txt",header = T, stringsAsFactors = F) %>% rename("Gene_stable_ID"= ENSG) %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::select(Gene.name, HvC.beta, HvC.pvalue, HvC.FDR) %>% rename("gene"=Gene.name)
```

Join with APA  

```{r}
RiboandAPA=DiffIso %>% inner_join(Ribo, by="gene")

RiboandAPA %>% group_by(gene) %>% n_distinct()

```

```{r}
ggplot(RiboandAPA,aes(y=deltaPAU, x=HvC.beta, col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="APA v Ribosome Occupany")+ scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 3)


ggplot(RiboandAPA,aes(y=deltaPAU, x=HvC.beta)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="APA v Ribosome Occupany")+ scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 3)
```



Just the genes with significant differences in PAS  

```{r}
RiboandAPA_sigAPA= RiboandAPA %>% filter(SigPAU2=="Yes")
```


```{r}
ggplot(RiboandAPA_sigAPA,aes(y=deltaPAU, x=HvC.beta,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA v Ribosome Occupany")+ scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 3)

ggplot(RiboandAPA_sigAPA,aes(y=deltaPAU, x=HvC.beta)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA v Ribosome Occupany")+ scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 3)
```




Sig both:  


```{r}
RiboandAPA_sigAPAandR= RiboandAPA_sigAPA %>% filter(SigPAU2=="Yes",  HvC.FDR<.05)
```


```{r}
ggplot(RiboandAPA_sigAPAandR,aes(y=deltaPAU, x=HvC.beta,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA v Significant Ribosome Occupany") + scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 3)

ggplot(RiboandAPA_sigAPAandR,aes(y=deltaPAU, x=HvC.beta)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA v Significant Ribosome Occupany") + scale_color_brewer(palette = "Set1")+ stat_cor(label.x = 3)

```



The correlation in expression with intronic is not there in ribosome occupancy.  

###Choose most Sig PAS  

To break ties I will use the top average usage. I will not worry about location at first.  

```{r}
RiboandAPA_topPAS= RiboandAPA %>% mutate(AvgUsageBoth=(Human+Chimp)/2) %>% group_by(gene) %>% arrange(p.adjust,desc(AvgUsageBoth)) %>% slice(1) %>% ungroup()

 

nrow(RiboandAPA %>% filter(loc=="intron"))
nrow(RiboandAPA_topPAS %>% filter(loc=="intron"))


nrow(RiboandAPA %>% filter(loc=="utr3"))
nrow(RiboandAPA_topPAS %>% filter(loc=="utr3"))
```


Plot the correlation in effect size  

```{r}
ggplot(RiboandAPA_topPAS,aes(y=deltaPAU, x=HvC.beta,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")  + labs(title="APA, top PAS v Ribosome Occupany")+ scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 3)

ggplot(RiboandAPA_topPAS,aes(y=deltaPAU, x=HvC.beta)) + geom_point(alpha=.3) + geom_smooth(method="lm")  + labs(title="APA, top PAS v Ribosome Occupany")+ scale_color_brewer(palette = "Set1")+ stat_cor(label.x = 3)


```



Sig APA
```{r}
RiboandAPA_topPASsigAPA= RiboandAPA_topPAS %>% filter(SigPAU2=="Yes")

nrow(RiboandAPA_topPASsigAPA %>% filter(loc=="intron"))


nrow(RiboandAPA_topPASsigAPA %>% filter(loc=="utr3"))

```
199 intronic significant, 910 significant 3' utr

```{r}
ggplot(RiboandAPA_topPASsigAPA,aes(y=deltaPAU, x=HvC.beta,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant  APA, top PAS v Ribosome Occupany") +scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 3)

ggplot(RiboandAPA_topPASsigAPA,aes(y=deltaPAU, x=HvC.beta)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant  APA, top PAS v Ribosome Occupany") +scale_color_brewer(palette = "Set1")+ stat_cor(label.x = 3)

```




Sig both:  


```{r}
RiboandAPA_topPASsigAPAandR= RiboandAPA_topPASsigAPA %>% filter(SigPAU2=="Yes",  HvC.FDR<.05)

nrow(RiboandAPA_topPASsigAPAandR %>% filter(loc=="intron"))


nrow(RiboandAPA_topPASsigAPAandR %>% filter(loc=="utr3"))
```
47 PAS for intrnic 229 for 3' UTR

```{r}
ggplot(RiboandAPA_topPASsigAPAandR,aes(y=deltaPAU, x=HvC.beta,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant  APA, top PAS  v Significant Ribosome Occupany") + scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 3)


ggplot(RiboandAPA_topPASsigAPAandR,aes(y=deltaPAU, x=HvC.beta)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant  APA, top PAS  v Significant Ribosome Occupany") + scale_color_brewer(palette = "Set1")+ stat_cor(label.x = 3)
```


Correlation in UTR but not intronic. Not sure if this is due to the number of PAS.   


###Plot all together 

```{r}
riboBoth=ggplot(RiboandAPA,aes(y=deltaPAU, x=HvC.beta)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="All APA and translation values")+ scale_color_brewer(palette = "Set1")+ stat_cor( label.x = c( -8, 5), label.y = -1)+theme_classic(base_size = 12) 
riboTop=ggplot(RiboandAPA_topPAS,aes(y=deltaPAU, x=HvC.beta,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")  + labs(title="All APA and translation values")+ scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3'"))+ stat_cor(aes(color = loc), label.x =c( -8, 1),label.y = -1)+theme_classic(base_size = 12) 

ribosigapaboth=ggplot(RiboandAPA_sigAPA,aes(y=deltaPAU, x=HvC.beta)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA and all Translation")+ scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR"))+ stat_cor( label.x = -8,label.y = -1)+theme_classic(base_size = 12) 

ribosigapasep=ggplot(RiboandAPA_topPASsigAPA,aes(y=deltaPAU, x=HvC.beta,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant  APA and Significant Translation") +scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR"))+ stat_cor(aes(color = loc), label.x =c( -8, 1),label.y = -1)+theme_classic(base_size = 12) 

sigapasigriboboth=ggplot(RiboandAPA_sigAPAandR,aes(y=deltaPAU, x=HvC.beta)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA and all Translation") + scale_color_brewer(palette = "Set1", name="")+ stat_cor(label.x = -8,label.y = -1)+theme_classic(base_size = 12) 

sigapasigribosep=ggplot(RiboandAPA_topPASsigAPAandR,aes(y=deltaPAU, x=HvC.beta,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant  APA and Significant Translation") + scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR"))+ stat_cor(aes(color = loc), label.x = c( -8, 1),label.y = -1)+theme_classic(base_size = 12) 


```


```{r}
plot_grid(riboBoth,riboTop,ribosigapaboth, ribosigapasep,sigapasigriboboth,sigapasigribosep, ncol=2)
```


```{r ribo, include=FALSE, dev="pdf", fig.height=8, fig.width=11, crop=FALSE}

plot_grid(riboBoth,riboTop,ribosigapaboth, ribosigapasep,sigapasigriboboth,sigapasigribosep, ncol=2)


```


```{r}
plot_grid(riboTop, ribosigapasep,sigapasigribosep)

```


```{r ribodiff, include=FALSE, dev="pdf", fig.height=8, fig.width=8, crop=FALSE}

plot_grid(riboBoth,riboTop,ribosigapaboth, ribosigapasep,sigapasigriboboth,sigapasigribosep, ncol=2)


```
##Protein  

I will use code from https://github.com/siddisis/project_primate_ribo to fit the linear model again and get effect sizes.  


load("../tables/fileS4.RData")

load("../rdas/HCR.protein.TMM.RData")


Put both of these in ../data/Khan_prot

```{r}
load("../data/Khan_prot/fileS4.RData")

load("../data/Khan_prot/HCR.protein.TMM.RData")
```


```{r}
expressed.gene.names <- as.character(HCR.protein.TMM.norm.ESNGlabeled[rownames(HCR.protein.TMM.norm.ESNGlabeled) %in% rownames(protein.expressed.data),16])
names(expressed.gene.names) <- rownames(protein.expressed.data)
```

Use to make design matrix  
```{r}

# HvC 
RNA.expressed.data.HC<-RNA.expressed.data[,1:10]



species.label <- substring(colnames(RNA.expressed.data.HC),1,1)

design <- model.matrix(~species.label)
colnames(design)<-c("Chimp","Human")
```


Protien 
```{r}
protein.expressed.data.HC<-protein.expressed.data[,1:10]

protein.fit<-lmFit(protein.expressed.data.HC ,design = design)


HvC.prot<- eBayes(protein.fit)

top.table <- topTable(HvC.prot, n = Inf)

volcanoplot(HvC.prot,coef=2,highlight=2)

effectsizeDF= as.data.frame(cbind(Gene_stable_ID=rownames(protein.expressed.data.HC),logEf=HvC.prot$coefficients[,2], pval=top.table$adj.P.Val))  %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::rename('gene'=Gene.name) %>% dplyr::select(-Gene_stable_ID)

write.table(effectsizeDF, "../data/Khan_prot/ProtData_effectSize.txt",col.names = T, row.names = F, quote = F)
```

```{r}
DPandAPA= DiffIso %>% inner_join(effectsizeDF, by="gene")
DPandAPA %>% group_by(gene) %>% summarise(n()) %>% nrow()

DPandAPA$logEf= as.numeric(as.character(DPandAPA$logEf))

DPandAPA$pval= as.numeric(as.character(DPandAPA$pval))
```

Looking at 2557 common genes.  


```{r}
ggplot(DPandAPA,aes(y=deltaPAU, x=logEf,col=loc)) + geom_point(alpha=.3) + geom_smooth(aes(col=loc), method="lm") + labs(title="Intronic and 3' UTR APA v DP") + scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 1)

ggplot(DPandAPA,aes(y=deltaPAU, x=logEf)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="Intronic and 3' UTR APA v DP") + scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 1)
```
Just the genes with significant differences in PAS  

```{r}
PandAPA_sigAPA= DPandAPA %>% filter(SigPAU2=="Yes")
```


```{r}
ggplot(PandAPA_sigAPA,aes(y=deltaPAU, x=logEf,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA v Protein")+ scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 1)

ggplot(PandAPA_sigAPA,aes(y=deltaPAU, x=logEf)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA v Protein")+ scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 1)
```
Sig both:  


```{r}
PandAPA_sigAPAandP= PandAPA_sigAPA %>% filter(SigPAU2=="Yes", pval <.05)
```



```{r}
ggplot(PandAPA_sigAPAandP,aes(y=deltaPAU, x=logEf,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA v Signficant Protein")+ scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 1)

ggplot(PandAPA_sigAPAandP,aes(y=deltaPAU, x=logEf)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA v Signficant Protein")+ scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 1)

```

###Choose most Sig PAS  

To break ties I will use the top average usage. I will not worry about location at first.  

```{r}
PandAPA_topPAS= DPandAPA %>% mutate(AvgUsageBoth=(Human+Chimp)/2) %>% group_by(gene) %>% arrange(p.adjust,desc(AvgUsageBoth)) %>% slice(1) %>% ungroup()
```


```{r}
ggplot(PandAPA_topPAS,aes(y=deltaPAU, x=logEf,col=loc)) + geom_point(alpha=.3) + geom_smooth(aes(col=loc), method="lm") + labs(title="Intronic and 3' UTR APA, top PAS v DP") + scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 1)

ggplot(PandAPA_topPAS,aes(y=deltaPAU, x=logEf)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="Intronic and 3' UTR APA v DP") + scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 1)
```
```{r}
PandAPA_topPAS_sigAPA= PandAPA_topPAS %>% filter(SigPAU2=="Yes")
```


```{r}
ggplot(PandAPA_topPAS_sigAPA,aes(y=deltaPAU, x=logEf,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA, top PAS v Protein")+ scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 1)

ggplot(PandAPA_topPAS_sigAPA,aes(y=deltaPAU, x=logEf)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA top PAS v Protein")+ scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 1)
```

Sig both:  


```{r}
PandAPA_topPAS_sigAPAandP= PandAPA_topPAS_sigAPA %>% filter(SigPAU2=="Yes", pval <.05)
```



```{r}
ggplot(PandAPA_topPAS_sigAPAandP,aes(y=deltaPAU, x=logEf,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA top PAS v Signficant Protein")+ scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 1)

ggplot(PandAPA_topPAS_sigAPAandP,aes(y=deltaPAU, x=logEf)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA top PAS v Signficant Protein")+ scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 1)

```



Check pvalues:  
```{r}

protKhan=read.csv("../data/Khan_prot/Khan_TableS4.csv",header = T)  %>% rename("Gene_stable_ID"= ENSG) %>% inner_join(nameID,by="Gene_stable_ID")  %>% rename("gene"=Gene.name) 

protKhanwmine= protKhan %>% inner_join(effectsizeDF, by="gene")
protKhanwmine$logEf=as.numeric(as.character(protKhanwmine$logEf))

protKhanwmine$pval=as.numeric(as.character(protKhanwmine$pval))
cor.test(protKhanwmine$pval,protKhanwmine$HC.pvalues.protein )
```
This is not good. Try the difference in means approach:  

Chimp-human 
```{r} 
protKhanSmall= protKhan %>% select(gene,mean.H.protein,mean.C.protein, HC.qvalues.rna) %>% mutate(Effect=mean.C.protein-mean.H.protein)
```

```{r}
deltaPandAPA= DiffIso %>% inner_join(protKhanSmall, by="gene")
deltaPandAPA %>% group_by(gene) %>% summarise(n()) %>% nrow()

```

2607 genes  

```{r}
ggplot(deltaPandAPA,aes(y=deltaPAU, x=Effect,col=loc)) + geom_point(alpha=.3) + geom_smooth(aes(col=loc), method="lm") + labs(title="Intronic and 3' UTR APA v DP") + scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 2)

ggplot(deltaPandAPA,aes(y=deltaPAU, x=Effect)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="Intronic and 3' UTR APA v DP") + scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 2)
```

Correlation effect sizes: 

```{r}
protKhanSmall_withmone= protKhanSmall %>% inner_join(effectsizeDF, by="gene")
protKhanSmall_withmone$logEf=as.numeric(as.character(protKhanSmall_withmone$logEf))


cor.test(protKhanSmall_withmone$logEf, protKhanSmall_withmone$Effect)
```
Ok this is equal but opposite.  So this is correct.  


**I need to check the direction  of the effects.**

###Plot together:  

```{r}
protboth=ggplot(DPandAPA,aes(y=deltaPAU, x=logEf)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="APA v Protein") + scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 2)+theme_classic(base_size = 12) 
protsep=ggplot(PandAPA_topPAS,aes(y=deltaPAU, x=logEf,col=loc)) + geom_point(alpha=.3) + geom_smooth(aes(col=loc), method="lm") + labs(title="APA, top PAS  v Protein") + scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 1)+theme_classic(base_size = 12) 
protsigapa=ggplot(PandAPA_sigAPA,aes(y=deltaPAU, x=logEf)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA v Protein")+ scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 1)+theme_classic(base_size = 12) 
protsigapasep=ggplot(PandAPA_topPAS_sigAPA,aes(y=deltaPAU, x=logEf,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA, top PAS v Protein")+ scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 1)+theme_classic(base_size = 12) 
protsigall=ggplot(PandAPA_sigAPAandP,aes(y=deltaPAU, x=logEf)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA v Signficant Protein")+ scale_color_brewer(palette = "Set1")+ stat_cor( label.x = 1)+theme_classic(base_size = 12) 
protsigallsep=ggplot(PandAPA_topPAS_sigAPAandP,aes(y=deltaPAU, x=logEf,col=loc)) + geom_point(alpha=.3) + geom_smooth(method="lm")+ labs(title="Significant APA top PAS v Signficant Protein")+ scale_color_brewer(palette = "Set1")+ stat_cor(aes(color = loc), label.x = 1)+theme_classic(base_size = 12) 


```


```{r}
plot_grid(protboth,protsep,protsigapa,protsigapasep,protsigall,protsigallsep, ncol=2)
```
```{r prot, include=FALSE, dev="pdf", fig.height=8, fig.width=11, crop=FALSE}

plot_grid(protboth,protsep,protsigapa,protsigapasep,protsigall,protsigallsep, ncol=2)



```
