---
title: "Intronic Location for Shared and Different Dominant PAS"
author: "Briana Mittleman"
date: "1/8/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


```{r}
library(workflowr)
library(tidyverse)
```

The goal of this analysis is to look at the distribution of intronic location for the shared and not shared dominant PAS. This will help me know if the pattern is due to annotation or not. In this analysis I will use the nuclear results.   


The first step is assigning each PAS to the intron it comes from.  

```{r}
HumanIntronicChimpUTR=read.table("../data/DominantPAS/Nuclear_HumanIntronicChimpUTR.txt",header = T, stringsAsFactors = F)%>% dplyr::select(gene, HumanPAS, HumanMean)
SameDomIntron=read.table("../data/DominantPAS/SameDominantIntronic.txt",header = T, stringsAsFactors = F)%>% dplyr::select(gene, HumanPAS,HumanMean)
```

```{r}
HumanPAS= read.table("../data/Pheno_5perc/Human_Pheno_5perc.txt", header = T,stringsAsFactors = F) %>% dplyr::select(chr, start, end, gene, strand, PAS) %>% rename("HumanPAS"=PAS)
```

Subset this file by those in the set and select it as a bed file for overlap with intron file. I will use the human mean as the score for now.    

```{r}
HumanPAS_samedom=HumanPAS %>% inner_join(SameDomIntron, by="HumanPAS") %>% mutate(GenePAS=paste(gene.x, HumanPAS, sep="_")) %>% dplyr::select(chr, start, end, GenePAS, HumanMean,strand)

HumanPAS_diffDom=HumanPAS %>% inner_join(HumanIntronicChimpUTR, by="HumanPAS") %>% mutate(GenePAS=paste(gene.x, HumanPAS, sep="_")) %>% dplyr::select(chr, start, end, GenePAS, HumanMean,strand)
```

I can write these out as bed files. 

```{r}
write.table(HumanPAS_samedom, "../data/DominantPAS/SameDominantPAS_intronic.bed", quote = F, row.names = F, col.names = F,sep = "\t")
write.table(HumanPAS_diffDom, "../data/DominantPAS/DifferentDominantPAS_intronic.bed", quote = F, row.names = F, col.names = F,sep = "\t")
```


I can use bedtools intersect to find the intron these are in.  


bedtools intersect -s -sorted -loj -a (PAS file) -b intron file  > output  

```{bash,eval=F}
sbatch FindIntronForDomPAS.sh
```


There are places where multiple transcripts with the intron included. this means the same PAS shows up multiple times ( i can look for uniq intron locations to take care of this)


```{r}
SameDomIntron=read.table("../data/DominantPAS/SameDominantPAS_intronic_mapped2Intron.txt", stringsAsFactors = F, col.names = c("PASchr", "PASstart", "PASend", "PASname", "PASusage", "PASstrand", "Intronchr", "IntronStart", "IntronEnd", "IntronName", "IntronScore", "IntronStrand"))
#group by intronname and keep top intron

SameDomIntronOne=SameDomIntron %>% group_by(PASname) %>% slice(1) %>% ungroup()



DiffDomIntron=read.table("../data/DominantPAS/DifferentDominantPAS_intronic_mapped2Intron.txt", stringsAsFactors = F, col.names = c("PASchr", "PASstart", "PASend", "PASname", "PASusage", "PASstrand", "Intronchr", "IntronStart", "IntronEnd", "IntronName", "IntronScore", "IntronStrand"))
DiffDomIntronOne=DiffDomIntron %>% group_by(PASname) %>% slice(1) %>% ungroup()
```

Now I need to find the midpoint for the PAS and get the percent distance to the start of the intron. I will use the intron strand because this is the genomic strand.  

```{r}
SameDomIntronOne_dist=SameDomIntronOne %>% mutate(centerPAS=PASstart +100, intronLength=IntronEnd-IntronStart, distance2PAS=ifelse(IntronStrand=="+", centerPAS -IntronStart, IntronEnd-centerPAS),propIntron=distance2PAS/intronLength)

DiffDomIntronOne_dist=DiffDomIntronOne %>% mutate(centerPAS=PASstart +100, intronLength=IntronEnd-IntronStart, distance2PAS=ifelse(IntronStrand=="+", centerPAS -IntronStart, IntronEnd-centerPAS),propIntron=distance2PAS/intronLength)
```

Plot both distributions, do percentage and absolute distance
```{r}
ggplot(SameDomIntronOne_dist, aes(x=distance2PAS)) + geom_histogram(bins=100)  + labs(x="Distance from intron start to PAS", title="Same Dominant Intron, absolute distance")

ggplot(SameDomIntronOne_dist, aes(x=propIntron)) + geom_histogram(bins=100) +labs(x="Proportion of intron", title="Same Dominant Intron, proportion of intron")
```
```{r}
SameDomIntronOne_dist_filt= SameDomIntronOne_dist %>% filter(distance2PAS<=100000)

ggplot(SameDomIntronOne_dist_filt, aes(x=distance2PAS)) + geom_histogram(bins=100)   +labs(x="Distance from intron start to PAS", title="Same Dominant Intron, absolute distance (less than 100kb)")
```



```{r}
ggplot(DiffDomIntronOne_dist, aes(x=distance2PAS)) + geom_histogram(bins=100)+ labs(x="Distance from intron start to PAS", title="Human Dominant Intronic, Chimp Dominant 3' UTR, absolute distance")
ggplot(DiffDomIntronOne_dist, aes(x=propIntron)) + geom_histogram(bins=100) +labs(x="Proportion of intron", title="Human Dominant Intronic, Chimp Dominant 3' UTR, Proportion of intron")


```


```{r}
DiffDomIntronOne_dist_filt= DiffDomIntronOne_dist %>% filter(distance2PAS<=100000)

ggplot(DiffDomIntronOne_dist_filt, aes(x=distance2PAS)) + geom_histogram(bins=100) +  labs(x="Distance from intron start to PAS", title="Human Dominant Intronic, Chimp Dominant 3' UTR \nabsolute distance (less than 100kb)")
```


Distributions look pretty similar.  

```{r}
ggplot(DiffDomIntronOne_dist, aes(x=propIntron)) +stat_ecdf(geom = "step", col="red") +stat_ecdf(data=SameDomIntronOne_dist, geom = "step", col="blue") + scale_colour_manual(name = 'Intron Set', values =c('red'='red','blue'='blue'), labels = c('c2','c1'),guide = 'legend')+ labs(x="PAS location by proprortion of Intron", title = "ecdf for Intronic location, red=Different Dom, blue=Same dom") 



```


```{r}
wilcox.test(DiffDomIntronOne_dist$propIntron, SameDomIntronOne_dist$propIntron)
wilcox.test(DiffDomIntronOne_dist$distance2PAS, SameDomIntronOne_dist$distance2PAS)
```

Not a significant difference in these distributions.  
