---
title: "Redo Expression Effect Size plot- Chimp Negative on both scales"
author: "Briana Mittleman"
date: "5/21/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

Untill now I have been plotting effect size correlations between human and chimp for expression, translation, and APA using inconsitent directions. 

I will flip the expression effect sizes to match the apa delta PAU. 


**Chimp will be negative**
**Human will be postivie**  

```{r}
library(ggpubr)
#stat cor is pearson 
library(qvalue)
library(tidyverse)
library("viridis") 
library(cowplot)
```

##Load data  
###APA data:  

```{r}
Meta=read.table("../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", header = T, stringsAsFactors = F)  %>% dplyr::select(PAS, chr, start,end, loc)
DiffIso= read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T,stringsAsFactors = F) %>% inner_join(Meta, by=c("chr", 'start','end')) %>% filter(loc %in% c("intron","utr3"))

DiffIsoTop=DiffIso %>% mutate(AvgUsageBoth=(Human+Chimp)/2) %>% group_by(gene) %>% arrange(p.adjust,desc(AvgUsageBoth)) %>% slice(1) %>% ungroup()


```

I will use the top used PAS for all plots (colored and not colored). This way there is no bias in the set.  
###Expression data    

```{r}
nameID=read.table("../../genome_anotation_data/ensemble_to_genename.txt",sep="\t", header = T, stringsAsFactors = F) %>% dplyr::select(Gene_stable_ID, Gene.name)

DE=read.table("../data/DiffExpression/DEtested_allres.txt",stringsAsFactors = F,header = F, col.names = c("Gene_stable_ID" ,"logFC" ,"AveExpr" , "t" ,  "P.Value" ,  "adj.P.Val", "B"  )) %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::rename('gene'=Gene.name) %>% dplyr::select(-Gene_stable_ID) %>% mutate(CorrectedlogFC=-1*logFC)
```


###Translation data:  
In this analysis - was already up regulated in chimp
```{r}
Ribo=read.table("../data/Wang_ribo/Additionaltable5_translationComparisons.txt",header = T, stringsAsFactors = F) %>% rename("Gene_stable_ID"= ENSG) %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::select(Gene.name, HvC.beta, HvC.pvalue, HvC.FDR) %>% rename("gene"=Gene.name)
```


##Plot data  

###APA and expression  

All data:  
```{r}
DeandAPA= DiffIsoTop %>% inner_join(DE, by="gene")

nrow(DeandAPA)
```


```{r}
DE_all=ggplot(DeandAPA,aes(y=deltaPAU, x=CorrectedlogFC)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="APA v DE", x="DE log effect size", y="Difference in PAS Usage") + scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR"))+ stat_cor(label.x = -8,label.y = -1) +theme_classic(base_size = 12)

DE_all
```
Seperate by size:  
```{r}
DE_split=ggplot(DeandAPA,aes(y=deltaPAU, x=CorrectedlogFC, col=loc)) + geom_point(alpha=.3) + geom_smooth(aes(col=loc),method="lm") + labs(title="APA v DE", x="DE log effect size", y="Difference in PAS Usage") + scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR"))+ stat_cor(aes(col=loc),label.x = c(-8,0),label.y = -1) +theme_classic(base_size = 12)

DE_split
```
```{r}
DeandAPA %>% group_by(loc) %>% summarise(n())
```



Subset to significant in both: 

```{r}
DeandAPASig= DeandAPA %>% filter(SigPAU2=="Yes", adj.P.Val<=0.05)

nrow(DeandAPASig)
```

```{r}
DE_sig_all=ggplot(DeandAPASig,aes(y=deltaPAU, x=CorrectedlogFC)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="Significant differences in APA and expression", x="DE log effect size", y="Difference in PAS Usage") + scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR"))+ stat_cor(label.x = -8,label.y = -1) +theme_classic(base_size = 12)

DE_sig_all
```
Seperate by size:  
```{r}
DE_sig_split=ggplot(DeandAPASig,aes(y=deltaPAU, x=CorrectedlogFC, col=loc)) + geom_point(alpha=.3) + geom_smooth(aes(col=loc),method="lm") + labs(title="Significant differences in APA and expression", x="DE log effect size", y="Difference in PAS Usage") + scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR"))+ stat_cor(aes(col=loc),label.x = c(-8,0),label.y = -1) +theme_classic(base_size = 12)

DE_sig_split
```
```{r}
DeandAPASig %>% group_by(loc) %>% summarise(n())
```

plot together: 

```{r}
plot_grid(DE_all, DE_split,DE_sig_all,DE_sig_split)
```



```{r,include=FALSE, eval=F}
pdf("../output/DEeffectsize", height=8, width=12)
plot_grid(DE_all, DE_split,DE_sig_all,DE_sig_split)
dev.off()
```

```{r DE, include=FALSE, dev="pdf", fig.height=8, fig.width=10, crop=FALSE}
plot_grid(DE_all, DE_split,DE_sig_all,DE_sig_split)
```



###APA and translation  

```{r}
RiboandAPA= DiffIsoTop %>% inner_join(Ribo, by="gene")

nrow(RiboandAPA)
```
```{r}
TE_all=ggplot(RiboandAPA,aes(y=deltaPAU, x=HvC.beta)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="APA v TE", x="TE effect size", y="Difference in PAS Usage") + scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR"))+ stat_cor(label.x = -8,label.y = -1) +theme_classic(base_size = 12)

TE_all
```
Seperate by size:  
```{r}
TE_split=ggplot(RiboandAPA,aes(y=deltaPAU, x=HvC.beta, col=loc)) + geom_point(alpha=.3) + geom_smooth(aes(col=loc),method="lm") + labs(title="APA v TE", x="TE effect size", y="Difference in PAS Usage") + scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR"))+ stat_cor(aes(col=loc),label.x = c(-8,0),label.y = -1) +theme_classic(base_size = 12)

TE_split
```

Number of intronic and 3' UTR  

```{r}
RiboandAPA %>% group_by(loc) %>% summarise(n())
```

Subset to significant in both: 

```{r}
RiboandAPASig= RiboandAPA %>% filter(SigPAU2=="Yes", HvC.FDR<=0.05)

nrow(RiboandAPASig)
```


```{r}
TE_sig_all=ggplot(RiboandAPASig,aes(y=deltaPAU, x=HvC.beta)) + geom_point(alpha=.3) + geom_smooth(method="lm") + labs(title="Significant differences in APA and translation", x="TE effect size", y="Difference in PAS Usage") + scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR"))+ stat_cor(label.x = -8,label.y = -1) +theme_classic(base_size = 12)

TE_sig_all
```


Seperate by size:  
```{r}
TE_sig_split=ggplot(RiboandAPASig,aes(y=deltaPAU, x=HvC.beta, col=loc)) + geom_point(alpha=.3) + geom_smooth(aes(col=loc),method="lm") + labs(title="Significant differences in APA and translation", x="TE effect size", y="Difference in PAS Usage") + scale_color_brewer(palette = "Set1",name="", labels=c("Intronic", "3' UTR"))+ stat_cor(aes(col=loc),label.x = c(-8,0),label.y = -1) +theme_classic(base_size = 12)

TE_sig_split
```


```{r}
RiboandAPASig %>% group_by(loc) %>% summarise(n())
```


plot together: 

```{r}
plot_grid(TE_all, TE_split,TE_sig_all,TE_sig_split)
```
```{r,include=FALSE, eval=F}
pdf("../output/TEeffectsize", height=8, width=10)
plot_grid(TE_all, TE_split,TE_sig_all,TE_sig_split)
dev.off()
```

```{r TE, include=FALSE, dev="pdf", fig.height=8, fig.width=10, crop=FALSE}
plot_grid(TE_all, TE_split,TE_sig_all,TE_sig_split)
```


###plot DE and TE  

```{r}
plot_grid(DE_sig_all,TE_sig_all,DE_sig_split, TE_sig_split)
```
```{r,include=FALSE, eval=F}
pdf("../output/DEandTEeffectsize", height=8, width=11)
plot_grid(DE_sig_all,TE_sig_all,DE_sig_split, TE_sig_split)
dev.off()
```



```{r DETE, include=FALSE, dev="pdf", fig.height=8, fig.width=11, crop=FALSE}
plot_grid(DE_sig_all,TE_sig_all,DE_sig_split, TE_sig_split)
```
