---
title: "Use apaQTL Individuals to establish cutoffs"
author: "Briana Mittleman"
date: "1/16/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

```{r}
library(tidyverse)
library(cowplot)
```

In this analyisis I will chose the 3 sets of 6 individuals randomly from my previous study to use for a benchmark analysis. 

I will need to rerun the snakefile each time.  

```{r}
indiv=read.table("../../apaQTL/data/MetaDataSequencing.txt", header= T, stringsAsFactors = F) %>% dplyr::select(line) %>% unique()
```


```{bash,eval=F}
mkdir ../data/OverlapBenchmark
```

Randomly choose 3 sets:  


```{r}
#sample1= sample(indiv$line, 6)
#sample2= sample(indiv$line, 6)
#sample3= sample(indiv$line, 6)

#save(sample1, sample2,sample3, file = "../data/OverlapBenchmark/samples.RData")


load("../data/OverlapBenchmark/samples.RData")
sample1
sample2
sample3

```

```{bash,eval=F}
mkdir ../../PAS_Sample1
mkdir ../../PAS_Sample1/code
mkdir ../../PAS_Sample1/data
mkdir ../../PAS_Sample1/data/fastq

mkdir ../../PAS_Sample2
mkdir ../../PAS_Sample2/code
mkdir ../../PAS_Sample2/data
mkdir ../../PAS_Sample2/data/fastq


mkdir ../../PAS_Sample3/
mkdir ../../PAS_Sample3/code/
mkdir ../../PAS_Sample3/data/
mkdir ../../PAS_Sample3/data/fastq

```
Move over these fastq files. 

I will also move the necessary snakefile and run files.  

Next steps:

* assign to genes and get pheno ratio (https://brimittleman.github.io/Comparative_APA/annotatePAS.html)  - run code in all 3 locations

'

```{bash,eval=F}
mkdir ../data/cleanPeaks_anno
bedtools map -a ../data/cleanPeaks/human_APApeaks.ALLChrom.Filtered.Named.Cleaned.bed -b  /project2/gilad/briana/genome_anotation_data/hg38_refseq_anno/hg38_ncbiRefseq_Formatted_Allannotation.sort.bed -c 4 -S -o distinct > ../data/cleanPeaks_anno/AllPAS_postLift.sort_LocAnno.bed 

python chooseAnno2Bed.py ../data/cleanPeaks_anno/AllPAS_postLift.sort_LocAnno.bed  ../data/cleanPeaks_anno/AllPAS_postLift.sort_LocAnnoPARSED.bed

python bed2SAF_gen.py ../data/cleanPeaks_anno/AllPAS_postLift.sort_LocAnnoPARSED.bed  ../data/cleanPeaks_anno/AllPAS_postLift.sort_LocAnnoPARSED.SAF

mkdir  ../data/CleanLiftedPeaks_FC/

sbatch quantLiftedPAS.sh 


###
python fixFChead_bothfrac.py ../data/CleanLiftedPeaks_FC/ALLPAS_postLift_LocParsed_Human ../data/CleanLiftedPeaks_FC/ALLPAS_postLift_LocParsed_Human_fixed.fc


python makeFileID.py ../data/CleanLiftedPeaks_FC/ALLPAS_postLift_LocParsed_Human ../data/CleanLiftedPeaks_FC/HumanFileID.txt


mkdir ../data/phenotype/

python makePheno.py  ../data/CleanLiftedPeaks_FC/ALLPAS_postLift_LocParsed_Human_fixed.fc ../data/CleanLiftedPeaks_FC/HumanFileID.txt ../data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno.txt

Rscript pheno2countonly.R -I ../data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno.txt -O ../data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno_countOnly.txt

python convertNumeric.py ../data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno_countOnly.txt ../data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno_countOnlyNumeric.txt

```


Pull these in, pull just the nuclear and get the mean:


Sample 1
```{r}
Sample1Anno=read.table("../../PAS_Sample1/data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno.txt", header = T, stringsAsFactors = F) %>% tidyr::separate(chrom, sep = ":", into = c("start1", "start", "end", "id")) %>% tidyr::separate(id, sep="_", into=c("gene", "strand", "peak"))  %>% separate(peak,into=c("loc", "PAS","chr"), sep="-")

Sample1Ind=colnames(Sample1Anno)[9:ncol(Sample1Anno)]

Sample1Usage=read.table("../../PAS_Sample1/data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno_countOnlyNumeric.txt", col.names = Sample1Ind) %>% dplyr::select(contains("_N")) 

Sample1All=as.data.frame(cbind(cbind(Sample1Anno[,1:8], Sample1=rowMeans(Sample1Usage))))
```

Sample 2:

```{r}
Sample2Anno=read.table("../../PAS_Sample2/data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno.txt", header = T, stringsAsFactors = F) %>% tidyr::separate(chrom, sep = ":", into = c("start1", "start", "end", "id")) %>% tidyr::separate(id, sep="_", into=c("gene", "strand", "peak"))  %>% separate(peak,into=c("loc", "PAS","chr"), sep="-")

Sample2Ind=colnames(Sample2Anno)[9:ncol(Sample2Anno)]

Sample2Usage=read.table("../../PAS_Sample2/data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno_countOnlyNumeric.txt", col.names = Sample2Ind) %>% dplyr::select(contains("_N")) 

Sample2All=as.data.frame(cbind(cbind(Sample2Anno[,1:8], Sample2=rowMeans(Sample2Usage))))
```


Sample 3  

```{r}
Sample3Anno=read.table("../../PAS_Sample3/data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno.txt", header = T, stringsAsFactors = F) %>% tidyr::separate(chrom, sep = ":", into = c("start1", "start", "end", "id")) %>% tidyr::separate(id, sep="_", into=c("gene", "strand", "peak"))  %>% separate(peak,into=c("loc", "PAS","chr"), sep="-")

Sample3Ind=colnames(Sample3Anno)[9:ncol(Sample3Anno)]

Sample3Usage=read.table("../../PAS_Sample3/data/phenotype/ALLPAS_postLift_LocParsed_Human_Pheno_countOnlyNumeric.txt", col.names = Sample3Ind) %>% dplyr::select(contains("_N")) 

Sample3All=as.data.frame(cbind(cbind(Sample3Anno[,1:8], Sample3=rowMeans(Sample3Usage))))
```


I need to make a bed file with these to overlap with the original PAS. I want 100 bp on each side of the end. Positive strand (actual negative strand) take 

```{r}
Sample1bed=Sample1All %>% mutate(PASName=paste(gene,PAS, sep="_"),newStart=ifelse(strand=="+", as.integer(start)-100, as.integer(end)-100), newEnd=ifelse(strand=="+",as.integer(start)+100, as.integer(end)+100)) %>%  dplyr::select(chr, newStart, newEnd, PASName, Sample1, strand)

#write.table(Sample1bed,"../data/OverlapBenchmark/sample1PAS.bed", col.names = F, row.names = F, sep="\t", quote = F )

Sample2bed=Sample2All %>% mutate(PASName=paste(gene,PAS, sep="_"),newStart=ifelse(strand=="+", as.integer(start)-100, as.integer(end)-100), newEnd=ifelse(strand=="+",as.integer(start)+100, as.integer(end)+100)) %>%  dplyr::select(chr, newStart, newEnd, PASName, Sample2, strand)

#write.table(Sample2bed,"../data/OverlapBenchmark/sample2PAS.bed", col.names = F, row.names = F, sep="\t", quote = F )

Sample3bed=Sample3All %>% mutate(PASName=paste(gene,PAS, sep="_"),newStart=ifelse(strand=="+", as.integer(start)-100, as.integer(end)-100), newEnd=ifelse(strand=="+",as.integer(start)+100, as.integer(end)+100)) %>%  dplyr::select(chr, newStart, newEnd, PASName, Sample3, strand)

#write.table(Sample3bed,"../data/OverlapBenchmark/sample3PAS.bed", col.names = F, row.names = F, sep="\t", quote = F )
```

```{bash,eval=F}
sort -k1,1 -k2,2n ../data/OverlapBenchmark/sample1PAS.bed > ../data/OverlapBenchmark/sample1PAS_sort.bed

sort -k1,1 -k2,2n ../data/OverlapBenchmark/sample2PAS.bed > ../data/OverlapBenchmark/sample2PAS_sort.bed

sort -k1,1 -k2,2n ../data/OverlapBenchmark/sample3PAS.bed > ../data/OverlapBenchmark/sample3PAS_sort.bed
```


Next step is to assess the overlap.  

```{bash,eval=F}
sbatch overlapapaQTLPAS_samples.sh
```

Sample 1 res:  

```{r}
wOverlap1=read.table("../data/OverlapBenchmark/sample1PAS_sort.Intersect.bed", col.names = colnames(Sample1bed)) %>% mutate(overlap="yes")
noOverlap1=read.table("../data/OverlapBenchmark/sample1PAS_sort.Intersect.NoOverlap.bed", col.names = colnames(Sample1bed)) %>% mutate(overlap="no")

AllwOinfo1=as.data.frame(rbind(wOverlap1, noOverlap1))
nrow(AllwOinfo1)
```

```{r}
overlap1=c()
totalvec1=c()
seq_usage=seq(0, .95, .01)
for (i in seq_usage){
  x=AllwOinfo1 %>% filter(Sample1>i) %>% group_by(overlap) %>% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlap1=c(overlap1,prop)
  totalvec1=c(totalvec1,total)
}

plot(seq_usage,overlap1,main="Sample 1", ylab="Percent Overlap", xlab="Usage Cutoff")
abline(v=.05,col="red")
abline(v=.1,col="blue")
```
Sample 2 


```{r}
wOverlap2=read.table("../data/OverlapBenchmark/sample2PAS_sort.Intersect.bed", col.names = colnames(Sample2bed)) %>% mutate(overlap="yes")
noOverlap2=read.table("../data/OverlapBenchmark/sample2PAS_sort.Intersect.NoOverlap.bed", col.names = colnames(Sample2bed)) %>% mutate(overlap="no")

AllwOinfo2=as.data.frame(rbind(wOverlap2, noOverlap2))
nrow(AllwOinfo2)
```

```{r}
overlap2=c()
totalvec2=c()
seq_usage=seq(0, .95, .01)
for (i in seq_usage){
  x=AllwOinfo2 %>% filter(Sample2>i) %>% group_by(overlap) %>% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlap2=c(overlap2,prop)
  totalve2c=c(totalvec2,total)
}

plot(seq_usage,overlap2,main="Sample 2", ylab="Percent Overlap", xlab="Usage Cutoff")
abline(v=.05,col="red")
abline(v=.1,col="blue")
```

Sample 3 


```{r}
wOverlap3=read.table("../data/OverlapBenchmark/sample3PAS_sort.Intersect.bed", col.names = colnames(Sample3bed)) %>% mutate(overlap="yes")
noOverlap3=read.table("../data/OverlapBenchmark/sample3PAS_sort.Intersect.NoOverlap.bed", col.names = colnames(Sample3bed)) %>% mutate(overlap="no")

AllwOinfo3=as.data.frame(rbind(wOverlap3, noOverlap3))
nrow(AllwOinfo3)
```

```{r}
overlap3=c()
totalvec3=c()
seq_usage=seq(0, .95, .01)
for (i in seq_usage){
  x=AllwOinfo3 %>% filter(Sample3>i) %>% group_by(overlap) %>% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlap3=c(overlap3,prop)
  totalvec3=c(totalvec3,total)
}

plot(seq_usage,overlap3,main="Sample 2", ylab="Percent Overlap", xlab="Usage Cutoff")
abline(v=.05,col="red")
abline(v=.1,col="blue")
```


Plot with all:  

```{r}
allSamp=as.data.frame(cbind(seq_usage,overlap1,overlap2,overlap3))
actualres= read.table("../data/CompapaQTLpas/ExtendedResoverlap.txt", header = T)

SampandRes=allSamp %>% inner_join(actualres,by="seq_usage") 

colnames(SampandRes)= c("Cutoff","Sample1", "Sample2", "Sample3", "CompPAS")

SampandRes_gather=SampandRes %>% gather(key="set", value="overlap",-Cutoff)
```

Plot:

```{r}
bench=ggplot(SampandRes_gather,aes(x=Cutoff,y=overlap, by=set, col=set))+ geom_line() + labs(title="Benchmark results with 3 sets of 6 samples") + geom_vline(xintercept=.05,col="red")+ geom_vline(xintercept=.1,col="blue")

bench
```

I will filter this to only genes passing the filter I established in the previous:  

```{r}
PassingGenes=read.table("../data/OverlapBenchmark/genesPassingCuttoff.txt", header = T, stringsAsFactors = F)
```



```{r}
AllwOinfo1_filt=AllwOinfo1 %>% separate(PASName, into=c("gene", "PAS"), sep="_")%>% filter(gene %in% PassingGenes$genes)


AllwOinfo2_filt=AllwOinfo2 %>% separate(PASName, into=c("gene", "PAS"), sep="_") %>% filter(gene %in% PassingGenes$genes)
AllwOinfo3_filt=AllwOinfo3 %>% separate(PASName, into=c("gene", "PAS"), sep="_") %>% filter(gene %in% PassingGenes$genes)


```

Actual results:

```{r}

chroms=c('chr10', 'chr11', 'chr12', 'chr13', 'chr14', 'chr15', 'chr16', 'chr17', 'chr18', 'chr19', 'chr1', 'chr2', 'chr20', 'chr21', 'chr22', 'chr3', 'chr4', 'chr5', 'chr6', 'chr7','chr8', 'chr9')

compAPAPAS=read.table("../data/Peaks_5perc/Peaks_5perc_either_HumanCoordHummanUsage.bed", header =   T, stringsAsFactors = F)  %>%  filter(Human>=0.05, chr %in% chroms) 
metaDataPAS=read.table("../data/PAS/PAS_5perc_either_HumanCoord_BothUsage_meta.txt", header = T, stringsAsFactors = F) %>% dplyr::select(PAS, gene)


wOverlapExt=read.table("../data/CompapaQTLpas/PAS_5percHuman.sort.Intersect_ext.bed", col.names = colnames(compAPAPAS)) %>% mutate(overlap="yes")
noOverlapExt=read.table("../data/CompapaQTLpas/PAS_5percHuman.sort.Intersect.NoOverlap_ext.bed", col.names = colnames(compAPAPAS)) %>% mutate(overlap="no")

AllwOinfoExt=as.data.frame(rbind(wOverlapExt, noOverlapExt)) %>% inner_join(metaDataPAS,by="PAS") %>% filter(gene %in% PassingGenes$genes)
nrow(AllwOinfoExt)
```


```{r}
overlapFilt1=c()
seq_usage=seq(0, .95, .01)
for (i in seq_usage){
  x=AllwOinfo1_filt %>% filter(Sample1>i) %>% group_by(overlap) %>% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlapFilt1=c(overlapFilt1,prop)
}

overlapFilt2=c()
for (i in seq_usage){
  x=AllwOinfo2_filt %>% filter(Sample2>i) %>% group_by(overlap) %>% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlapFilt2=c(overlapFilt2,prop)
}

overlapFilt3=c()
for (i in seq_usage){
  x=AllwOinfo3_filt %>% filter(Sample3>i) %>% group_by(overlap) %>% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlapFilt3=c(overlapFilt3,prop)
}

overlapFiltActual=c()
for (i in seq_usage){
  x=AllwOinfoExt %>% filter(Human>i) %>% group_by(overlap) %>% summarise(n=n())
  yes=as.numeric(x[2,2])
  total=as.numeric(x[2,2])+ as.numeric(x[1,2])
  prop=yes/total
  overlapFiltActual=c(overlapFiltActual,prop)
}

FilteredRes=as.data.frame(cbind(seq_usage,overlapFilt1,overlapFilt2,overlapFilt3,overlapFiltActual))
colnames(FilteredRes)= c("Cutoff","Sample1", "Sample2", "Sample3", "CompPAS")

FilteredRes_gather=FilteredRes %>% gather(key="set", value="overlap",-Cutoff)
```

```{r}
filteredbench=ggplot(FilteredRes_gather,aes(x=Cutoff,y=overlap, by=set, col=set))+ geom_line() + labs(title="Benchmark results with 3 sets of 6 samples, filtered genes") + geom_vline(xintercept=.05,col="red")+ geom_vline(xintercept=.1,col="blue")

filteredbench
```
Plot together:  


```{r}
plot_grid(bench,filteredbench)
```

