---
title: "Explore genes with dAPA double filter"
author: "Briana Mittleman"
date: "1/21/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


```{r}
library(tidyverse)
library(ggpubr)
library(reshape2)
```

I want to look more at the genes we found with dAPA. 

Question 1:

Do genes with differential APA have different numbers of PAS in each species?

```{r}
DiffUsage=read.table("../data/DiffIso_Nuclear_DF/SignifianceEitherPAS_2_Nuclear.txt", header = T, stringsAsFactors = F)

PASMeta=read.table("../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", header = T, stringsAsFactors = F) %>% dplyr::select(PAS, chr, start,end, gene, loc)

DiffUsagePAS=DiffUsage %>% inner_join(PASMeta, by=c("gene","chr", "start", "end"))
```

Number of PAS in each species: 

```{r}
PAS=read.table("../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", stringsAsFactors = F, header = T)
PAS_sm=PAS %>% dplyr::select(gene, Chimp, Human) 
PAS_m= melt(PAS_sm, id.var="gene", variable.name="species", value.name="meanUsage") %>% filter(meanUsage >=0.1) %>% group_by(species, gene) %>% summarise(nPAS=n())
```

Filter these by those with dAPA:

```{r}
PAS_m_dAPA= PAS_m %>% mutate(dAPA=ifelse(gene %in% DiffUsagePAS$gene, "Y", "N"))
```

```{r}
ggplot(PAS_m_dAPA,aes(by=dAPA, y=nPAS,x=species, fill=dAPA)) + geom_boxplot()  + stat_compare_means(method = "t.test") + scale_fill_brewer(palette = "Dark2") + labs(y="Number of PAS detected usage", title="Number of PAS detected by gene with differential usage") 
```

Question 2: Where are the differentially used PAS?

```{r}
ggplot(DiffUsagePAS,aes(x=loc, fill=loc)) + geom_bar(stat="count") +  theme(text= element_text(size=16), legend.position = "none") + scale_fill_brewer(palette = "Dark2") + labs(x="Genic Region", y="Number of dAPA PAS", title="Location of dAPA PAS") 

```
for supplement
```{r}
pdf("../output/supplement/Fig2_figSup1.pdf", height=4, width=4)
ggplot(DiffUsagePAS,aes(x=loc, fill=loc)) + geom_bar(stat="count") + scale_fill_brewer(palette = "RdYlBu") + labs(x="Genic Region", y="Number of PAS", title="Location of differentially used PAS") + theme_classic() + theme(axis.text.x = element_text(size=10, angle=90),plot.title = element_text(hjust = 0.5, face="bold"),axis.text.y = element_text(size=10),text=element_text(size=10), legend.position = "none",plot.margin = unit(c(0,0,0,0), "cm"))+scale_x_discrete(labels=c("Coding","5'kb downstream","Intronic", "3' UTR", "5' UTR"))
dev.off()

```



Seperate by location:  

```{r}
#negative deltaPAU is used more in chimp 
DiffUsagePAS_dir= DiffUsagePAS %>% mutate(direction=ifelse(deltaPAU >=0, "Human", "Chimp"))

ggplot(DiffUsagePAS_dir,aes(x=loc, fill=loc)) + geom_bar(stat="count")  + facet_grid(~direction)+ scale_fill_brewer(palette = "Dark2") + labs(x="Genic Region", y="Number of dAPA PAS", title="Location of dAPA PAS")+ theme(text= element_text(size=16), legend.position = "none") 
```
This is opposite of the results using just the dominant PAS.  I probably shouldn't put too much into that.  

Question 3: Does locaiton of the PAS effect the absolute value of the effect size  

```{r}
ggplot(DiffUsagePAS_dir,aes(x=loc, y=abs(deltaPAU), fill=loc)) + geom_violin() 
```



Explore conservation:  

https://www.ultraconserved.org

https://useast.ensembl.org/info/genome/compara/conservation_and_constrained.html


phylo p from genomebrowser  

```{bash,eval=F}
mkdir ../data/PhyloP
mkdir ../data/DNDS

```
PhyloP: Column #1 contains a one-based position coordinate. Column #2 contains a score showing the posterior probability that the phylogenetic hidden Markov model (HMM) of phastCons is in its most conserved state at that base position.  

I want to get the average score for each of the tested PAS. I can use pybigwig.  

```{bash,eval=F}
python extractPhyloReg.py
```

```{r}
phylores=read.table("../data/PhyloP/PAS_phyloP.txt", col.names = c("chr","start","end", "phyloP"), stringsAsFactors = F) %>% drop_na()
NucReswPhy=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T, stringsAsFactors = F) %>% inner_join(phylores, by=c("chr","start","end"))
```

Plot:  

```{r}
ggplot(NucReswPhy,aes(y=phyloP, x=SigPAU2,fill=SigPAU2)) + geom_boxplot() + stat_compare_means(method.args = list(alternative = "l"))+ scale_fill_brewer(palette = "Dark2", name="Signficant")


```

```{r}

ggplot(NucReswPhy,aes(x=phyloP, by=SigPAU2, fill=SigPAU2)) + geom_density(alpha=.5) + scale_fill_brewer(palette = "Dark2", name="Signficant PAS") + labs(title="Mean PhyloP scores for tested PAS") 
```

The significant PAS have on average lower phyloP scores. 


Positive scores — Measure conservation, which is slower evolution than expected, at sites that are predicted to be conserved.
Negative scores — Measure acceleration, which is faster evolution than expected, at sites that are predicted to be fast-evolving.


I can look at those with negative values:  

```{r}
x=nrow(NucReswPhy %>% filter(SigPAU2=="Yes", phyloP<0))
m= nrow(NucReswPhy %>% filter(phyloP<0))
n=nrow(NucReswPhy %>% filter(phyloP>=0))
k=nrow(NucReswPhy %>% filter(SigPAU2=="Yes"))

#actual:
x

#pval
phyper(x-1,m,n,k,lower.tail=F)
```
This means these regions are more likely to be fast evolving.  

Look at this by location: (is it driven by region)

```{r}
NucReswPhy_meta= NucReswPhy %>% inner_join(PASMeta, by=c("chr", "start", "end", "gene"))

ggplot(NucReswPhy_meta,aes(x=phyloP, by=SigPAU2, fill=SigPAU2)) + geom_density(alpha=.5) + scale_fill_brewer(palette = "Dark2") + facet_grid(~loc)
```
```{r}
NucReswPhy_meta_group=NucReswPhy_meta %>% group_by(loc,SigPAU2) %>% summarise(n=n(),meanPhylo=mean(phyloP))
NucReswPhy_meta_group
```

Compare this to the genes that are expressed for this I will need to make a bedfile with the genes. I will pull them in as well as the gene annotation:  

```{r}

DAPAGeneSig=read.table("../data/DiffIso_Nuclear_DF/SignifianceEitherGENES_Nuclear.txt", stringsAsFactors = F, header = T) 

DAPAGene=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", stringsAsFactors = F, header = T) %>% dplyr::select(gene) %>% unique() %>% mutate(Sig=ifelse(gene %in% DAPAGeneSig$gene,"Yes","No"))
```
To be safe ill use the longest transcripts from table browser refseq.  

This will be easiest in a python dictionary.  
```{bash,eval=F}
python parseHg38.py
sort -k1,1 -k2,2n ../../genome_anotation_data/hg38_refseq_anno/hg38_ncbiRefseq_GenesParsed.bed > ../../genome_anotation_data/hg38_refseq_anno/hg38_ncbiRefseq_GenesParsed_sort.bed
```

```{r}
genes=read.table("../../genome_anotation_data/hg38_refseq_anno/hg38_ncbiRefseq_GenesParsed_sort.bed", col.names = c("chr", "start", "end","name","score","strand"),stringsAsFactors = F) %>% filter(name %in% DAPAGene$gene ) %>% rename("gene"=name)
genesWithSig= genes %>% inner_join(DAPAGene, by="gene")
write.table(genes, "../data/PhyloP/NuclearSigGenes.bed", col.names = F, row.names = F, quote=F, sep="\t")

```

Get the mean phyloP scores.  

```{bash,eval=F}
 python extractPhyloRegGene.py 
```

```{r}
phyloresG=read.table("../data/PhyloP/PAS_phyloP_genes.txt", col.names = c("chr","start","end", "phyloP"), stringsAsFactors = F) %>% drop_na()
GenesPhy=genesWithSig %>% inner_join(phyloresG, by=c("chr","start","end"))
```



```{r}
ggplot(GenesPhy,aes(x=phyloP, by=Sig, fill=Sig)) + geom_density(alpha=.5)+ scale_fill_brewer(palette = "Dark2", name="Gene with \nSignificant PAS") + labs(title="Mean PhyloP scores for tested Genes") + annotate("text",label="Wilcoxan, p=2.6.4e -12",x=4,y=1.5)




ggplot(GenesPhy,aes(y=phyloP, x=Sig,fill=Sig)) + geom_boxplot() + stat_compare_means()+ scale_fill_brewer(palette = "Dark2")

```
These are also genes with a shift. See if more likely to have - value.  

```{r}
x=nrow(GenesPhy %>% filter(Sig=="Yes", phyloP<0))
m= nrow(GenesPhy %>% filter(phyloP<0))
n=nrow(GenesPhy %>% filter(phyloP>=0))
k=nrow(GenesPhy %>% filter(Sig=="Yes"))


#expected
which(grepl(max(dhyper(1:x, m, n, k)), dhyper(1:x, m, n, k)))

#actual:
x

#pval
phyper(x,m,n,k,lower.tail=F)
```
Enriched for genes with - scores.  

DN (non synonymous) /DS (synonymous): from ensamble site  - ratio of substitution rate (quick and dirty way to look at evo), ration >1 usually evidence for positive selection. values are in ../data/DNDS/HumanChimp_DNDS.csv 

Remove NA values 
```{r}
DNDS= read.csv("../data/DNDS/HumanChimp_DNDS.csv", header = T,stringsAsFactors = F) %>% drop_na() %>% group_by(Gene.name) %>% slice(1) %>% ungroup() %>% mutate(DNDSratio= dN.with.Chimpanzee/dS.with.Chimpanzee) %>% dplyr::select(Gene.name, dN.with.Chimpanzee,dS.with.Chimpanzee,DNDSratio) %>% rename("gene"=Gene.name)
```

Join with all results then subset based on significance:  

I will get all genes,

```{r}
NucResGenes=read.table("../data/DiffIso_Nuclear_DF/SignifianceEitherGENES_Nuclear.txt",header = T)
NucResAll=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T, stringsAsFactors = F) %>% dplyr::select(gene) %>% unique() %>% mutate(SigPASinGene=ifelse(gene %in% NucResGenes$gene, "yes", "no")) 

NucResDNDS= NucResAll %>% inner_join(DNDS,by="gene") 
```
We do not have information for 1236 of the genes. I will assess results on the 7308 with data. There  are also genes with ratio problems due to zero in the ds column. If it is infinity, i can make it 1 for now because there are only fixed non syn mutations fixing. If both are 0 I will make it 0. 

```{r}
NucResDNDS_fix=NucResDNDS %>% mutate(DNDSratio = replace_na(DNDSratio,0))

NucResDNDS_fix[NucResDNDS_fix == Inf] <- 1

summary(NucResDNDS_fix$DNDSratio)

NucResDNDS_fix %>% group_by(SigPASinGene) %>% summarise(n=n())
```
Plot this.  

```{r}

ggplot(NucResDNDS_fix,aes(y=log10(DNDSratio+1), x=SigPASinGene, fill=SigPASinGene))+ geom_boxplot() + stat_compare_means( label.y.npc = "middle") + scale_fill_brewer(palette = "Dark2") + labs(x="dAPA in Nuclear") + annotate("text", label="Yes=1451 \n No=5865", y=1.8,x=2)


```
I can ask if they are more likely to be above 1. I can do this with a hypergeo. 

```{r}

x=nrow(NucResDNDS_fix %>% filter(SigPASinGene=="yes", DNDSratio>=1))
m= nrow(NucResDNDS_fix %>% filter(DNDSratio>=1))
n=nrow(NucResDNDS_fix %>% filter(DNDSratio<1))
k=nrow(NucResDNDS_fix %>% filter(SigPASinGene=="yes"))


#expected
which(grepl(max(dhyper(1:x, m, n, k)), dhyper(1:x, m, n, k)))

#actual:
x

#pval
phyper(x,m,n,k,lower.tail=F)
```



No enrichment for positive selected genes.  

Gene ontology: 
Need a ranked list of genes. I can do this for the differential apa genes by pvalue.  
http://cbl-gorilla.cs.technion.ac.il

```{r}
NucRes=read.table("../data/DiffIso_Nuclear/SignifianceEitherPAS_2_Nuclear.txt",header = T,stringsAsFactors = F) %>% arrange(p.adjust) %>% dplyr::select(gene) %>% unique()


write.table(NucRes,"../data/DiffIso_Nuclear/SignifianceGenes_orderPval.txt",col.names = F, row.names = F, quote = F)
```



