---
title: "Home"
site: workflowr::wflow_site
output:
  workflowr::wflow_html:
    toc: false
editor_options:
  chunk_output_type: console
---

I am interesting in understanding differential APA between human and chimpanzees. I used 3' Seq to identify PAS for total and nuclear frations of LCL in both species. 


##Preparation and QC:  

* [Prepare annotation files for mapping ](annotationInfo.html)  

* [Remove SNORNAs from the annotations](removeSNOfromAnno.html)

* [Mapping statistics and plots](mapStats.html)

* [Liftover PAS](liftoverPAS.html)

* [Annotated PAS and make APA phenotype](annotatePAS.html)

* [PCA plot to look at variation in data](NromalizePhenos.html)
      
* [Correlation between individuals](CorrbetweenInd.html)
      
* [Verify sample identity](verifyBAM.html)

* [3' Seq enrichement along gene bodies](EnrichmentEndofGenes.html)

* [Cross map and prepare Chimp 3' for Viz in IGV](CrossChimpThreePrime.html)

* [Make PAS bed for 5% peaks](fixPeak2PAS.html)


## Choosing cuttoffs for usage and expression level  

  * [Use merged 3' at gene level to look for a data quantity cutoff](expressioncutoff.html)  

  * [Subsampleing apaQTL lines (not redone)](establishCutoffs.html)  
  
  * [Apply the expression filter](applyDoubleFilt.html)
  
  * [Compare before and after cutoff](compareFilter.html)  
  
  * [QC metrics on filtered](df_QC.html)
  
  * [Check how unlifted PAS would contribute to analysis](unliftedsites.html)

##Catalog PAS:

* [Number of PAS in each species double filter](PASnumberSpecies_doubleFilter.html)   

* [Top 2 signal sites double filter](signalsites_doublefilter.html)  

* [Dominant PAS in Nuclear Fraction double filter](dominantPAS_DF.html)  

* [Intronic location for dominant PAS double filter](DominantPASintronLoc_DF.html)  

* [Test for dominant PAS for total fraction](TotalDomStructure.html)

* [Compare dominance between fractions](DomTotalvNuclear.html)
    
* [Species Specificity in PAS double filter](speciesSpecific_DF.html)  

* [PAS number distribution according to dominance](ConservationofPASNumber.html)

* [PAS number distribution according to dominance in total](ConservationofPASNumberTotal.html)

* [Some plots about conservation (correaltions in usage)](ConservationPlots.html)

* [Correlation between fraction in each species](SpecBetweenfrac.html)

* [Look at dominance by difference between dominant PAS and next highest used](NewDomDef.html)

* [New Dominance and species specific PAS](specSpecDom.html)

* [Dominance structure looking at the difference for all PAS and establish cutoff at .4](DiffTop2SecondDom.html)

##Differential Isoform Analysis:

* [Nuclear Human vs. Chimp Analysis Double filter](Nuclear_HvC_DF.html)  

* [Explore nuclear differential APA PAS and genes Double filter](ExploredAPA_DF.html)  

* [Enrichment for differencial APA and apaQTL double filter](dAPAandapaQTL_DF.html)   

* [Visualization for all Nuclear DAPA genes double filter](visualizePASUsage_DF.html)  

* [Does presence or absense of signal explain differential usage](SS_diffUsage.html)

* [Protein interactions](PTM_analysis.html)  

* [Explore coding dPAS](CodingdPAS.html)  

* [New Dominance method and usage or differential usage](UsageForDomPAS.html)

* [New dominant and dAPA](NewdomanddAPA.html)

* [pie charts for 3 levels of dAPA](piecharts.html)

* [Test enrichment in gene sets](gsea.html)


###Use annotations to understand mechanisms

* [Identify orthologous 3'UTR set](ortho3UTR.html)  

* [Look at differentially used PAS by length of UTR](LongShortUTR_2PAS.html)  

* [Proportion of ortho 3' UTR](percetnUTR.html)  

* [Look at miRNA density](miRNAanalysis.html)  

* [3' UTR biology- length and AU](AT_richUTRs.html)

* [Expression levels](ExpressionLevelsdAPA.html)  

* [U rich up stream and downstream of site](UrichElements.html)

* [Which PAS within Ortho UTRs and seq between for these](WhichPASinOrtho.html)

* [Differntial usage, usage of site, and location in UTR](LikelihoodDiffUsebyUse.html)


* [AU rich elements with TrpSeq method](TryTripSeqAnalysis.html)

##Quantify Phenotypes with RNA:   

* [Differential Splicing](diffSplicing.html) 

* [Differential Expression](diffExpression.html)
  
    * [Metadata by species](MetabySpecies.html)
  
    * [DE genes and eQTLs](DEandeQTls.html)
  
* [Compare DE and DS genes](DEvDS.html)

  
##Intersect with outside molecular phenotypes:

* [Download data from differential protein and ribosomes](dProtanddRibo.html)  

* [Compare to nuclear differential APA with Double filter](DiffTransProtvsNuclearAPA_DF.html)  

* [Use UpSetR to vizualize intersections Double filter](upsetter_DF.html)  

* [Effect size correlations double filter](correlationPhenos_DF.html)  

* [Compare species 5' splice site strength](SpliceSiteStrength.html)  

* [Compare dominance structure and DE genes](DomPatternandDE.html)

* [PolII](pol2.html)

* [New Dominance method and DE](NewDomandDE.html)

* [New Dominance and translation](NewDomandTE.html)  

* [H3K36me3](h3k36me3.html)

* [Sequence between dominant PAS and DE status](seqBetweenDomPAS.html)


##Selection exploration  

* [Explore PAS conservation](dAPA_Conservation.html)  

* [Use directional selection from Khan](DirSelectionKhan.html)  

* [Conservation based on dominance structure](ConsandDom.html)

* [Conservation based on new dominance categories](ConsandDomNewDef.html)

* [Orthgonal analysis with PhastCon](phastCon.html)

##Explore effect size relationships  

* [Explore more about APA expression relationship](ExploreAPAexpression.html)

* [Explore more about APA expression relationship with overlaping significant genes](ExploreAPAexpressionOverlap.html)  

* [Apa and expression relationship for seperated by location of PAS](DiffUsedIntronic.html) 

* [APA and expression relationship with dominance structure](DeESandDomgenes.hmtl)

* [Look at genes that have differences in protein levels and APA levels but not mRNA expression](dAPAdPnotE.html)

* [Plot the different effect size relationships by dominance](EffectSizePlots.html)

* [Test usage restriction in effect size plot](ExploreIntronicEffectsize.html)  

* [Effect size plots with same direction for betas](EffectSizeRelationshipFixDir.html)

* [try to incorporate ancestral allele from QTL](incorporateQTLsAncestral.html)

* [Test for differences in protein marks](protMod.html)

* [Test relationship is driven by switching 3' UTR pas and intronic](IntronUTRSwitch.html)

##Domiance and DE (test for mechanisms)  

* [Could standardization with length account for differences?](DiffDomDE_length.html)  

* [Look at this by location](DiffDomandDELOC.html)

* [Test for destabilization](ARE_element.html)

* [Test differences in mRNA decay](mRNADecay.html)

* [PolyA factors and hypoxia](hypoxiaresponse.html)


##Verify effect sizes in total  

* [Total Human vs. Chimp Analysis](Total_HvC.html)  

* [Total effect size relationships](Total_dapavDE.html)

##Information content  

* [Calculate Information content measures](infoContent.html)  

* [DE and number of PAS](DeandNumPAS.html)

* [Information content and dAPA](InfoanddAPA.html)

* [Information content and various other measure from previous analysis](InfoandMoreVars.html)

* [Info Content by individual to look for species differences](dInforContent.html)

* [dInfoContent and other phenotypes](dICandotherD.html)  

* [Isoform diversity and DE direction](DirICandDE.html)  

* [Seperate dAPA and dIC to relate to other phenotpyes](SetsdAPADIC.html)

* [Genes that are dIC and dP but not DE](DICNotDEDP.html)


###Main figures  

* [Main figures](mainFigures.html)

* [Supplemental Tables](SuppTables.html)

Test for robustness to unlifted sites: 

* [Figures without 353 unlifted genes](ResultsNoUnlifted.html)


###Analysis for Revisions  

* [Which Chromosomes are the dAPA sites on](chromosomes.html)

* [Conservation analysis with PhyloP 20 ways](ConsPhlop20.html)

* [Conservation compared to other regulatory elements](phylopRegElements.html)

* [Additional Gene set enrichments](additionalGSEA.html)

* [Null distribution for figure 6](nullDistribution6.html)


##Evaluate Reasons for inconsistant dominance structure (not rerun after annotation change 4/20)

List of everything we tried for this section:  

* [List of Test dominance structure](listFixes.html)

###Number of Ns  

* [Look at number of reads lost if you exclude reads with N](index.html)

* [Base composition](BaseComp.html)


###Misprimming  
* [Change misprimming parameters](changeMisprimcut.html)  

Liftover  

* [Liftover for 4](liftMis4.html)  

* [Liftover for 5](liftMis5.html)  

* [Liftover for 6](liftMis6.html)  

Annotate and make phenotype  

* [Annotation for 4](annoMis4.html)

* [Annotation for 5](annoMis5.html)

* [Annotation for 6](annoMis6.html)

Evaluate results:  

* [Dominant PAS distribution after adjusting filters](NewFilterDomPAS.html)


###Annotation differences  

* [Test for annotation bias](annotationBias.html)

###Other ideas:  

* [Add usage cutoffs for the dominant PAS](DomPASDiffUsed.html)  

* [Overlapping PAS](OverlappingPAS.html)

* [Test quantification methods](TrynewQuantMethods.html)  

* [Look at dominance sturcture without multimapping and SNOrnas](checkResnoMM.html)

* [Distribution with normalized usage](DominantPAS_usingNorm.html)

* [Filter out ties and genes with 1 PAS](DominantNoTies.html)

* [Quantile Normalzie](PPASqauntnorm.html)


###Test multimapping and FC criteria

* [Look at new parameters for multimapping](multimapParameters.html)

* [Assess which PAS are in orthoexon file](orthoexonAnno.html)

* [Experiemnt with quality and FC](MMExpreiment.html)

* [Redo the quantification with primary reads](DomwithPrimary.html)



##Visualization  

* [Try to use gviz](GvizPlots.html)


##Extra QC and Analysis   

* [Compare identified PAS with apaQTL PAS](comp2apaQTLPAS.html)

* [Test for overlaps within species](MergePASwithin.html)

* [Check res after removing 18499](resultswo18499.html)

### Sample switch analysis  

* [Investigate bad RNA seq samples](InvestigateBadSamples.html)  

    * [Differential splicing without bad samples](diffSplicing_removeBad.html)  
    
    * [Differential Expression without bad samples](diffExpressionRemove2.html)  
    
### Mapping problems with pantro5 and multimapping

* [Map RNA to hg19](MapRNAhg19.html)

* [Evaluate multimapping criteria](multiMap.html)

* [Map to opposite Genomes](OppositeMap.html)

* [Evaluate Pantro 5 problems](investigatePantro5.html)

From before reseq:

* [Total Fraction](NormalizePheno_total.html)  
      
* [Total Fraction- Prenormalized](NonNormPheno_total.html)
        

### Chosing between pantro files

  * [Compare pantro5 and pantro6 orthoexon](compareOrthoExon.html)
    
  * [Differential Expression with Pantro6](diffExpressionPantro6.html)
  
  * [Compare DE in pantro5 vs pantro6](DE_pantro5vspantro6.html)


### Extra comparisons or analysis: not redone after double filter

* [Human Total vs Nuclear Analysis](Human_TvN.html)  

* [Chimp Total vs Nuclear Analysis](Chimp_TvN.html)  

* [Dominant PAS in Total Fraction](dominantPAS_total.html)  

* [Total Human vs. Chimp Analysis](Total_HvC.html)  

* [Venn diagrams for genes in each differential APA set](CompHC_totalandnuclear.html)  

From the original nuclear dominant  

   * [Does signal site explain trend?](ss_sameDomdiffDom.html)  

   * [Look at Intronic Dominant with DPAU and DE](HumanIntronicDominant.html)  

* [Enrichment for differencial APA and eQTL](DapaineQTL.html)  

* [Look at dAPA and genes from Dominant PAS analysis](DominatPASandDIffAPA.html)


###PreDouble filter analysis  


* [Number of PAS in each species](PASnumperSpecies.html)  

* [Pressence of top 2 signal sites](signalsites.html)  

* [Dominant PAS in Nuclear Fraction](dominantPAS.html)  

* [Intronic location for dominant PAS](DominantPASintronLoc.html)  

* [Species Specificity in PAS](speciesSpecific.html)  

* [Nuclear Human vs. Chimp Analysis](Nuclear_HvC.html)  

* [Explore nuclear differential APA PAS and genes](Explore.html)  

* [Enrichment for differencial APA and apaQTL](andapaQTL.html)  

* [Visualization for all Nuclear DAPA genes](visualizePASUsage.html)  

* [Compare to nuclear differential APA](DiffTransProtvsNuclearAPA.html)   

* [Use UpSetR to vizualize intersections](upsetter.html)  

* [Effect size correlations](correlationPhenos.html)  

* [Normalize Nuclear Fraction](NormalizePheno_nuclear.html)  

* [Mediation analysis](mediation.html)    

###Human v Chimp fraction differences - pre- march rerun  

* [Differential isoform between fractons by species](TotalVNuclearBothSpecies.html)

####Mediation tests  

* [Normalize (qqnrom) Nuclear Fraction double filter](NormalizePheno_nuclear_DF.html)    

* [Normalize clusters from splicing analysis](normalizeClusters.html)

* [Mediation analysis double filter](mediation_DF.html)  
    
* [Mediation with protein](mediation_prot.html)
