---
title: "Home"
site: workflowr::wflow_site
output:
  workflowr::wflow_html:
    toc: false
editor_options:
  chunk_output_type: console
---

I am interesting in understanding differential APA between human and chimpanzees. I used 3' Seq to identify PAS for total and nuclear frations of LCL in both species. 


##Preparation and QC:  

* [Prepare annotation files for mapping ](annotationInfo.html)  

* [Mapping statistics and plots](mapStats.html)

* [Liftover PAS](liftoverPAS.html)

* [Annotated PAS and make APA phenotype](annotatePAS.html)

* [Compare identified PAS with apaQTL PAS](comp2apaQTLPAS.html)

* [PCA plot to look at variation in data](NromalizePhenos.html)
      
* [Correlation between individuals](CorrbetweenInd.html)
      
* [Verify sample identity](verifyBAM.html)

* [3' Seq enrichement along gene bodies](EnrichmentEndofGenes.html)

* [Cross map and prepare Chimp 3' for Viz in IGV](CrossChimpThreePrime.html)

* [Make PAS bed for 5% peaks](fixPeak2PAS.html)


## Choosing cuttoffs for usage and expression level  

  * [Use merged 3' at gene level to look for a data quantity cutoff](expressioncutoff.html)  

  * [Establish cutoffs by subsampleing apaQTL lines](establishCutoffs.html)  
  
  * [Compare before and after cutoff](compareFilter.html)  

##Catalog PAS:

* [Number of PAS in each species](PASnumperSpecies.html)   

    * [Number of PAS in each species double filter](PASnumberSpecies_doubleFilter.html)   

* [Pressence of top 2 signal sites](signalsites.html)  

    * [Top 2 signal sites double filter](signalsites_doublefilter.html)  

* [Dominant PAS in Nuclear Fraction](dominantPAS.html)  

    * [Dominant PAS in Nuclear Fraction double filter](dominantPAS_DF.html)  

* [Intronic location for dominant PAS](DominantPASintronLoc.html)  
      
    * [Intronic location for dominant PAS double filter](DominantPASintronLoc_DF.html)  
    
* [Species Specificity in PAS](speciesSpecific.html)  

     * [Species Specificity in PAS double filter](speciesSpecific_DF.html)  
  


##Differential Isoform Analysis:

* [Nuclear Human vs. Chimp Analysis](Nuclear_HvC.html)  

     * [Nuclear Human vs. Chimp Analysis Double filter](Nuclear_HvC_DF.html)  

* [Explore nuclear differential APA PAS and genes](Explore.html)  

     * [Explore nuclear differential APA PAS and genes Double filter](Explore_DF.html)  

* [Enrichment for differencial APA and apaQTL](andapaQTL.html)  

     * [Enrichment for differencial APA and apaQTL double filter](dAPAandapaQTL_DF.html)   

* [Visualization for all Nuclear DAPA genes](visualizePASUsage.html)  
   
     * [Visualization for all Nuclear DAPA genes double filter](visualizePASUsage_DF.html)  

From here on, I did everything double filtered:  

* [Does presence or absense of signal explain differential usage](SS_diffUsage.html)

* [Protein interactions](PTM_analysis.html)  

* [Explore coding dPAS](CodingdPAS.html)  



##Human v Chimp fraction differences  

* [Differential isoform between fractons by species](TotalVNuclearBothSpecies.html)

##Quantify Phenotypes with RNA:   

* [Differential Splicing](diffSplicing.html) 

* [Differential Expression](diffExpression.html)
  
    * [Metadata by species](MetabySpecies.html)
  
    * [DE genes and eQTLs](DEandeQTls.html)
  
* [Compare DE and DS genes](DEvDS.html)

  
##Intersect with outside molecular phenotypes:

* [Download data from differential protein and ribosomes](dProtanddRibo.html)  

* [Compare to nuclear differential APA](DiffTransProtvsNuclearAPA.html)   

     * [Compare to nuclear differential APA with Double filter](DiffTransProtvsNuclearAPA_DF.html)  

* [Use UpSetR to vizualize intersections](upsetter.html)  

    * [Use UpSetR to vizualize intersections Double filter](upsetter_DF.html)  

* [Effect size correlations](correlationPhenos.html)  

    * [Effect size correlations double filter](correlationPhenos_DF.html)  
All with double filter:  

* [Compare species 5' splice site strength](SpliceSiteStrength.html)  



##Selection exploration  

* [Explore PAS conservation](dAPA_Conservation.html)  

* [Use directional selection from Khan](DirSelectionKhan.html)  

* [Conservation based on dominance structure](ConsandDom.html)

##Mediation  

* [Normalize Nuclear Fraction](NormalizePheno_nuclear.html)  

    * [Normalize (qqnrom) Nuclear Fraction double filter](NormalizePheno_nuclear_DF.html)    

* [Normalize clusters from splicing analysis](normalizeClusters.html)

* [Mediation analysis](mediation.html)    

    * [Mediation analysis double filter](mediation_DF.html)  
    
From here on, I did everything double filtered:  
    
* [Mediation with protein](mediation_prot.html)

* [Explore more about APA expression relationship](ExploreAPAexpression.html)

* [Explore more about APA expression relationship with overlaping significant genes](ExploreAPAexpressionOverlap.html)  

* [Apa and expression relationship for seperated by location of PAS](DiffUsedIntronic.html)  

* [Look at genes that have differences in protein levels and APA levels but not mRNA expression](dAPAdPnotE.html)


##Evaluate Reasons for inconsistant dominance structure 

* [Add usage cutoffs for the dominant PAS](DomPASDiffUsed.html)  

###Number of Ns  

* [Look at number of reads lost if you exclude reads with N](index.html)

* [Base composition](BaseComp.html)


###Misprimming  
* [Change misprimming parameters](changeMisprimcut.html)  

Liftover  

* [Liftover for 4](liftMis4.html)  

* [Liftover for 5](liftMis5.html)  

* [Liftover for 6](liftMis6.html)  

Annotate and make phenotype  

* [Annotation for 4](annoMis4.html)

* [Annotation for 5](annoMis5.html)

* [Annotation for 6](annoMis6.html)

Evaluate results:  

* [Dominant PAS distribution after adjusting filters](NewFilterDomPAS.html)


###Annotation differences  

* [Test for annotation bias](annotationBias.html)

###Other ideas:  

* [Overlapping PAS](OverlappingPAS.html)

* [Test quantification methods](TrynewQuantMethods.html)  

* [Remove SNORNAs from the annotations](removeSNOfromAnno.html)

* [Look at dominance sturcture without multimapping and SNOrnas](checkResnoMM.html)



##Visualization  

* [Try to use gviz](GvizPlots.html)


##Extra QC and Analysis   

### Sample switch analysis  

* [Investigate bad RNA seq samples](InvestigateBadSamples.html)  

    * [Differential splicing without bad samples](diffSplicing_removeBad.html)  
    
    * [Differential Expression without bad samples](diffExpressionRemove2.html)  
    
### Mapping problems with pantro5 and multimapping

* [Map RNA to hg19](MapRNAhg19.html)

* [Evaluate multimapping criteria](multiMap.html)

* [Map to opposite Genomes](OppositeMap.html)

* [Evaluate Pantro 5 problems](investigatePantro5.html)

From before reseq:

* [Total Fraction](NormalizePheno_total.html)  
      
* [Total Fraction- Prenormalized](NonNormPheno_total.html)
        

### Chosing between pantro files

  * [Compare pantro5 and pantro6 orthoexon](compareOrthoExon.html)
    
  * [Differential Expression with Pantro6](diffExpressionPantro6.html)
  
  * [Compare DE in pantro5 vs pantro6](DE_pantro5vspantro6.html)


### Extra comparisons or analysis: not redone after double filter

* [Human Total vs Nuclear Analysis](Human_TvN.html)  

* [Chimp Total vs Nuclear Analysis](Chimp_TvN.html)  

* [Dominant PAS in Total Fraction](dominantPAS_total.html)  

* [Total Human vs. Chimp Analysis](Total_HvC.html)  

* [Venn diagrams for genes in each differential APA set](CompHC_totalandnuclear.html)  

From the original nuclear dominant  

   * [Does signal site explain trend?](ss_sameDomdiffDom.html)  

   * [Look at Intronic Dominant with DPAU and DE](HumanIntronicDominant.html)  

* [Enrichment for differencial APA and eQTL](DapaineQTL.html)  

* [Look at dAPA and genes from Dominant PAS analysis](DominatPASandDIffAPA.html)

