---
title: "Home"
site: workflowr::wflow_site
output:
  workflowr::wflow_html:
    toc: false
editor_options:
  chunk_output_type: console
---

I am interesting in understanding differential APA between human and chimpanzees. I used 3' Seq to identify PAS for total and nuclear frations of LCL in both species. 



Preparation and QC:  

* [Prepare annotation files for mapping ](annotationInfo.html)  

* [Mapping statistics and plots](mapStats.html)

* [Liftover PAS](liftoverPAS.html)

* [Annotated PAS and make APA phenotype](annotatePAS.html)

* [Compare identified PAS with apaQTL PAS](comp2apaQTLPAS.html)

* [PCA plot to look at variation in data](NromalizePhenos.html)

      * [Total Fraction](NormalizePheno_total.html)  
      
      * [Total Fraction- Prenormalized](NonNormPheno_total.html)
        
      * [Nuclear Fraction](NormalizePheno_nuclear.html)  
      
* [Correlation between individuals](CorrbetweenInd.html)
      
* [Verify sample identity](verifyBAM.Rmd)

* [3' Seq enrichement along gene bodies](EnrichmentEndofGenes.html)

Catalog PAS:

* [Number of PAS in each species](PASnumperSpecies.html)  


Differential Isoform Analysis:

* [Human Total vs Nuclear Analysis](Human_TvN.html)  

* [Chimp Total vs Nuclear Analysis](Chimp_TvN.html)  

* [Nuclear Human vs. Chimp Analysis](Nuclear_HvC.html)

Quantify Phenotypes with RNA:   

* [Differnetial Splicing](diffSplicing.html) 
     * [Differential splicing without bad samples](diffSplicing_removeBad.html)

* [Differential Expression](diffExpression.html)

     * [Differential Expression without bad samples](diffExpressionRemove2.html)  


Extra QC and Analysis   

* [Investigate bad RNA seq samples](InvestigateBadSamples.html)  

* [Map RNA to hg19](MapRNAhg19.html)


