---
title: "Home"
site: workflowr::wflow_site
output:
  workflowr::wflow_html:
    toc: false
editor_options:
  chunk_output_type: console
---

I am interesting in understanding differential APA between human and chimpanzees. I used 3' Seq to identify PAS for total and nuclear frations of LCL in both species. 



##Preparation and QC:  

* [Prepare annotation files for mapping ](annotationInfo.html)  

* [Mapping statistics and plots](mapStats.html)

* [Liftover PAS](liftoverPAS.html)

* [Annotated PAS and make APA phenotype](annotatePAS.html)

* [Compare identified PAS with apaQTL PAS](comp2apaQTLPAS.html)

* [PCA plot to look at variation in data](NromalizePhenos.html)
      
* [Correlation between individuals](CorrbetweenInd.html)
      
* [Verify sample identity](verifyBAM.Rmd)

* [3' Seq enrichement along gene bodies](EnrichmentEndofGenes.html)

* [Cross map and prepare Chimp 3' for Viz in IGV](CrossChimpThreePrime.html)

* [Make PAS bed for 5% peaks](fixPeak2PAS.html)

##Catalog PAS:

* [Number of PAS in each species](PASnumperSpecies.html)   

* [Pressence of top 2 signal sites](signalsites.html)  

* [Dominant PAS in Total Fraction](dominantPAS_total.html)  

* [Dominant PAS in Nuclear Fraction](dominantPAS.html)  

    - [Intronics location for dominant PAS](DominantPASintronLoc.html)  
  
    - [Does signal site explain trend?](ss_sameDomdiffDom.html)  

    - [Look at Intronic Dominant with DPAU and DE](HumanIntronicDominant.html)  
    
* [Species Specificity in PAS](speciesSpecific.html)  
  


##Differential Isoform Analysis:

* [Human Total vs Nuclear Analysis](Human_TvN.html)  

* [Chimp Total vs Nuclear Analysis](Chimp_TvN.html)  

* [Nuclear Human vs. Chimp Analysis](Nuclear_HvC.html)

* [Total Human vs. Chimp Analysis](Total_HvC.html)

* [Venn diagrams for genes in each differential APA set](CompHC_totalandnuclear.html)

* [Explore nuclear differential APA PAS and genes](ExploredAPA.html)

* [Differential APA and eQTL overlap](DapaineQTL.html)

* [Look at dAPA and genes from Dominant PAS analysis](DominatPASandDIffAPA.html)



##Quantify Phenotypes with RNA:   

* [Differential Splicing](diffSplicing.html) 

* [Differential Expression](diffExpression.html)
  
  * [Metadata by species](MetabySpecies.html)
  
  * [DE genes and eQTLs](DEandeQTls.html)
  
* [Compare DE and DS genes](DEvDS.html)

  
##Intersect with outside molecular phenotypes:

* [Download data from differential protein and ribosomes](dProtanddRibo.html)  

* [Compare to nuclear differential APA](DiffTransProtvsNuclearAPA.html)   

* [Use UpSetR to vizualize intersections](upsetter.html) 


##Mediation  

* [Normalize Nuclear Fraction](NormalizePheno_nuclear.html)  

* [Normalize clusters from splicing analysis](normalizeClusters.html)

* [Mediation analysis](mediation.html)

##Extra QC and Analysis   

### Sample switch analysis  

* [Investigate bad RNA seq samples](InvestigateBadSamples.html)  

    * [Differential splicing without bad samples](diffSplicing_removeBad.html)  
    
    * [Differential Expression without bad samples](diffExpressionRemove2.html)  
    
### Mapping problems with pantro5 and multimapping

* [Map RNA to hg19](MapRNAhg19.html)

* [Evaluate multimapping criteria](multiMap.html)

* [Map to opposite Genomes](OppositeMap.html)

* [Evaluate Pantro 5 problems](investigatePantro5.html)

From before reseq:

* [Total Fraction](NormalizePheno_total.html)  
      
* [Total Fraction- Prenormalized](NonNormPheno_total.html)
        

### Chosing between pantro files

  * [Compare pantro5 and pantro6 orthoexon](compareOrthoExon.html)
    
  * [Differential Expression with Pantro6](diffExpressionPantro6.html)
  
  * [Compare DE in pantro5 vs pantro6](DE_pantro5vspantro6.html)

