---
title: "Cutoffs for multimapping"
author: "Briana Mittleman"
date: "3/26/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


```{r}
library(workflowr)
library(tidyverse)
```


I want to look at the parameters for multimapping. I will look at the paremeters such as include a read when the first vs second map difference is 10%, 5% ect. I want to get to a cutoff where the PAS lost are not in the ortho exon. 

I will look at the featurecounts documentation for this or I can look directly at the star info.   


−−primary: If specified, only primary alignments will be counted. Primary and secondary alignments are identified using bit 0x100 in the Flag field of SAM/BAM files. All primary alignments in a dataset will be counted no matter they are from multi- mapping reads or not (ie. ‘-M’ is ignored).

For multi-mappers, all alignments except one are marked with 0x100 (secondary alignment) in the FLAG (column 2 of the SAM

I used options:

 --outFilterMultimapNmax 10 --outSAMmultNmax 1 
 
 
 --outSAMmultNmax 1 will output exactly one SAM line for each mapped read
 
 I only allowed for 1 read to be mapped. I want the secondary one as well. I want to mess with the secondary read.  
 
 Remap human and chimp reads with the new filter. Just to look first, don't worry about misspriming.  
 
```{bash,eval=F}
mkdir ../data/TestMM2/
sbatch StarMM2.sh
sbatch ChimpStarMM2.sh
```
 
 chimp_combined_18358_N.MM2.sort.bam   is done. I can use this to look at the seondary mapped reads.  
 
 
 HI:i:i Query hit index, indicating the alignment record is the i-th one stored in SAM. 
 
 the HI:i:2 reads are the secondary maped reads.  
 
 NM:i:count  number of mismatches  
 
 256 is the secondary allignment 
 
 I can use pysam to filter secondary reads.  
 
```{bash,eval=F}
samtools view chimp_combined_18358_N.MM2.sort.bam  | cut -f5 | sort -n | uniq -c
```

1860484 0
2727360 1
3182906 3
39284071 255  


without multmap:  

 930242 0
1363680 1
1591453 3
39284071 255


I want to filter just secondary reads and get the scores-
secondary alignment score is in column 2

```{bash,eval=F}
samtools view chimp_combined_18358_N.MM2.sort.bam  | cut -f2 | sort -n | uniq -c
```
Primary (by strand)
23943403 0
19226043 16

seconday (by strand)  
1571873 256
2313502 272


"AS" SAM tag express the "Alignment score generated by aligner"
-scores between 53,85

get scores for primary and secondary  


Mismatches  (with secondary)
34622475 nM:i:0
7600018 nM:i:1
  57259 nM:i:10
2337437 nM:i:2
1045189 nM:i:3
 547184 nM:i:4
 327976 nM:i:5
 213004 nM:i:6
 138913 nM:i:7
  96974 nM:i:8
  68392 nM:i:9
  
Primary  

14717503 nM:i:0
3042728 nM:i:1
  21887 nM:i:10
 943180 nM:i:2
 406973 nM:i:3
 217371 nM:i:4
 132942 nM:i:5
  88071 nM:i:6
  54904 nM:i:7
  38671 nM:i:8
  26097 nM:i:9
  
  
Seperate secondary, write code for any file:  

```{bash,eval=F}
mkdir ../data/TestMM2_SeondaryRead
mkdir ../data/TestMM2_PrimaryRead

python filterSecondaryread.py ../data/TestMM2/chimp_combined_18358_N.MM2.sort.bam  ../data/TestMM2_SeondaryRead/chimp_combined_18358_N.MM2_secondary.bam


python filterPrimaryread.py ../data/TestMM2/chimp_combined_18358_N.MM2.sort.bam  ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.bam


samtools sort -o ../data/TestMM2_SeondaryRead/chimp_combined_18358_N.MM2_secondary.sort.bam -O bam  ../data/TestMM2_SeondaryRead/chimp_combined_18358_N.MM2_secondary.bam

samtools sort -o ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.sort.bam -O bam ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.bam

samtools index ../data/TestMM2_SeondaryRead/chimp_combined_18358_N.MM2_secondary.sort.bam

samtools index ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.sort.bam



             
```

scores 53-84  

2426470 nM:i:0
 919015 nM:i:1
   9997 nM:i:10
 253038 nM:i:2
 108368 nM:i:3
  59802 nM:i:4
  39210 nM:i:5
  26476 nM:i:6
  18892 nM:i:7
  13544 nM:i:8
  10563 nM:i:9
  
This means a lot of the secondary reads also have 0 mismatch. These are probably the reads I would want to filter.  


Compare the porportion.  

```{bash,eval=F}
mkdir ../data/TestMM2_quality

samtools view chimp_combined_18358_N.MM2_secondary.sort.bam | cut -f15 | sort -n | uniq -c > ../TestMM2_quality/chimp18358_secondaryMismatch.txt

samtools view chimp_combined_18358_N.MM2.sort.bam | cut -f15 | sort -n | uniq -c > ../TestMM2_quality/chimp18358_PrimaryMismatch.txt

samtools view ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.sort.bam | cut -f15 | sort -n | uniq -c > ../data/TestMM2_quality/chimp18358_PrimaryOnlyMismatch.txt
```


Compare this:  

```{r}
SecMismatch=read.table("../data/TestMM2_quality/chimp18358_secondaryMismatch.txt",col.names = c("Reads","Mismatch"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="Secondary")
PrimaryOnlyMismatch= read.table("../data/TestMM2_quality/chimp18358_PrimaryOnlyMismatch.txt",col.names = c("Reads","Mismatch"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="Primary")
PrimaryMismatch=read.table("../data/TestMM2_quality/chimp18358_PrimaryMismatch.txt",col.names = c("Reads","Mismatch"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="All")

BothMiss= SecMismatch %>% bind_rows(PrimaryOnlyMismatch)


ggplot(BothMiss, aes(x=Mismatch, y=PropReads, by=set, fill=set))+ geom_bar(stat="identity", position="dodge")+ labs(title="Primary v Seconday Mismatch Chimp 18358")
```
Do this for scores as well  

```{bash,eval=F}
samtools view ../data/TestMM2_SeondaryRead/chimp_combined_18358_N.MM2_secondary.sort.bam | cut -f14 | sort -n | uniq -c > ../data/TestMM2_quality/chimp18358_secondaryScores.txt

samtools view ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.sort.bam | cut -f14 | sort -n | uniq -c > ../data/TestMM2_quality/chimp18358_PrimaryOnlyScores.txt
```

```{r}
SecScore=read.table("../data/TestMM2_quality/chimp18358_secondaryScores.txt",col.names = c("Reads","Score"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="Secondary")
PrimaryOnlyScore= read.table("../data/TestMM2_quality/chimp18358_PrimaryOnlyScores.txt",col.names = c("Reads","Score"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="Primary")

BothScore= SecScore %>% bind_rows(PrimaryOnlyScore)

ggplot(BothScore, aes(x=Score, y=PropReads, by=set, fill=set))+ geom_bar(stat="identity", position="dodge") +  theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(title="Primary v Seconday Scores Chimp 18358")

```


Feature counts change the -Q for quality. what score is this? 

may q scores?  

```{bash,eval=F}
samtools view ../data/TestMM2_SeondaryRead/chimp_combined_18358_N.MM2_secondary.sort.bam | cut -f5 | sort -n | uniq -c > ../data/TestMM2_quality/chimp18358_secondaryMAPScores.txt

samtools view ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.sort.bam | cut -f5 | sort -n | uniq -c > ../data/TestMM2_quality/chimp18358_PrimaryOnlyMAPScores.txt
```


```{r}
SecMAPScore=read.table("../data/TestMM2_quality/chimp18358_secondaryMAPScores.txt",col.names = c("Reads","Score"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="Secondary")
PrimaryMAPOnlyScore= read.table("../data/TestMM2_quality/chimp18358_PrimaryOnlyMAPScores.txt",col.names = c("Reads","Score"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="Primary")

BothMAPScore= SecMAPScore %>% bind_rows(PrimaryMAPOnlyScore)

ggplot(BothMAPScore, aes(x=Score, y=PropReads, by=set, fill=set))+ geom_bar(stat="identity", position="dodge") +  theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(title="Primary v Seconday Scores Chimp 18358")

```

Mapping reads with the Fc.  

18358: 

Standard: 9195763
Primary: 9440757
Multimap: 9650073
9195763 for each score. 


##Mismatch scores for all primary and secondary  

I want to compare the primary and secondary reads for all libraries. I can then look at the mismatch distribution.  

First I need to wrap the primary and secondary read code.  

```{bash,eval=F}
sbatch FilterPrimSec.sh
mkdir ../data/TestMM2_mismatch
sbatch MismatchNumbers.sh
```

Load in results:

Chimp
```{r}
Chimp_18358_primary=read.table("../data/TestMM2_mismatch/chimp_combined_18358_N_primary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Primary", species="Chimp", line="18358")

Chimp_18358_secondary=read.table("../data/TestMM2_mismatch/chimp_combined_18358_N_secondary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Secondary", species="Chimp", line="18358")

Chimp_3622_primary=read.table("../data/TestMM2_mismatch/chimp_combined_3622_N_primary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Primary", species="Chimp", line="3622")

Chimp_3622_secondary=read.table("../data/TestMM2_mismatch/chimp_combined_3622_N_secondary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Secondary", species="Chimp", line="3622")

Chimp_3659_primary=read.table("../data/TestMM2_mismatch/chimp_combined_3659_N_primary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Primary", species="Chimp", line="3659")

Chimp_3659_secondary=read.table("../data/TestMM2_mismatch/chimp_combined_3659_N_secondary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Secondary", species="Chimp", line="3659")

Chimp_4973_primary=read.table("../data/TestMM2_mismatch/chimp_combined_4973_N_primary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Primary", species="Chimp",line="4973")

Chimp_4973_secondary=read.table("../data/TestMM2_mismatch/chimp_combined_4973_N_secondary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Secondary", species="Chimp",line="4973")

Chimp_pt30_primary=read.table("../data/TestMM2_mismatch/chimp_combined_pt30_N_primary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Primary", species="Chimp", line="pt30")

Chimp_pt30_secondary=read.table("../data/TestMM2_mismatch/chimp_combined_pt30_N_secondary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Secondary", species="Chimp",line="pt30")

Chimp_pt91_primary=read.table("../data/TestMM2_mismatch/chimp_combined_pt91_N_primary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Primary", species="Chimp",line="pt91")

Chimp_pt91_secondary=read.table("../data/TestMM2_mismatch/chimp_combined_pt91_N_secondary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Secondary", species="Chimp", line="pt91")
```


```{r}
allChimpmismatch=Chimp_18358_primary %>% 
  bind_rows(Chimp_18358_secondary)  %>% 
  bind_rows(Chimp_3622_primary) %>% 
  bind_rows(Chimp_3622_secondary) %>% 
  bind_rows(Chimp_3659_primary) %>% 
  bind_rows(Chimp_3659_secondary) %>% 
  bind_rows(Chimp_4973_primary) %>% 
  bind_rows(Chimp_4973_secondary) %>% 
  bind_rows(Chimp_pt30_primary) %>% 
  bind_rows(Chimp_pt30_secondary) %>% 
  bind_rows(Chimp_pt91_primary) %>% 
  bind_rows(Chimp_pt91_secondary) %>% 
  group_by(line, set) %>% 
  mutate(TotalReads=sum(Reads)) %>%
  ungroup() %>% 
  mutate(PropReads=Reads/TotalReads)

allChimpmismatch$Mis=factor(allChimpmismatch$Mis, levels=c("nM:i:0",  "nM:i:1" , "nM:i:2", "nM:i:3" , "nM:i:4", "nM:i:5" , "nM:i:6",  "nM:i:7" , "nM:i:8" , "nM:i:9","nM:i:10"))


ggplot(allChimpmismatch, aes(x=Mis,y=PropReads,by=set, fill=set)) + geom_bar(stat="identity",position = "dodge") + facet_grid(~line)+  theme(axis.text.x = element_text(angle = 90, hjust = 1))  +labs(title="Chimp Mismatch in primary and secondary")
ggplot(allChimpmismatch, aes(x=Mis,y=Reads,by=set, fill=set)) + geom_bar(stat="identity",position = "dodge") + facet_grid(~line)+  theme(axis.text.x = element_text(angle = 90, hjust = 1))  +labs(title="Chimp Mismatch in primary and secondary")
  
```

Human: 

```{r}
Human_18498_primary=read.table("../data/TestMM2_mismatch/human_combined_18498_N_primary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Primary", species="Human", line="18498")

Human_18498_secondary=read.table("../data/TestMM2_mismatch/human_combined_18498_N_secondary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Secondary", species="Human", line="18498")


Human_18499_primary=read.table("../data/TestMM2_mismatch/human_combined_18499_N_primary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Primary", species="Human", line="18499")

Human_18499_secondary=read.table("../data/TestMM2_mismatch/human_combined_18499_N_secondary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Secondary", species="Human", line="18499")

Human_18502_primary=read.table("../data/TestMM2_mismatch/human_combined_18502_N_primary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Primary", species="Human", line="18502")

Human_18502_secondary=read.table("../data/TestMM2_mismatch/human_combined_18502_N_secondary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Secondary", species="Human", line="18502")

Human_18504_primary=read.table("../data/TestMM2_mismatch/human_combined_18504_N_primary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Primary", species="Human", line="18504")

Human_18504_secondary=read.table("../data/TestMM2_mismatch/human_combined_18504_N_secondary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Secondary", species="Human", line="18504")

Human_18510_primary=read.table("../data/TestMM2_mismatch/human_combined_18510_N_primary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Primary", species="Human", line="18510")

Human_18510_secondary=read.table("../data/TestMM2_mismatch/human_combined_18510_N_secondary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Secondary", species="Human", line="18510")

Human_18523_primary=read.table("../data/TestMM2_mismatch/human_combined_18523_N_primary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Primary", species="Human", line="18523")

Human_18523_secondary=read.table("../data/TestMM2_mismatch/human_combined_18523_N_secondary_mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Secondary", species="Human", line="18523")
```
All human:  

```{r}
allHumanmismatch=Human_18498_primary %>% 
  bind_rows(Human_18498_secondary)  %>% 
  bind_rows(Human_18499_primary) %>% 
  bind_rows(Human_18499_secondary) %>% 
  bind_rows(Human_18502_primary) %>% 
  bind_rows(Human_18502_secondary) %>% 
  bind_rows(Human_18504_primary) %>% 
  bind_rows(Human_18504_secondary) %>% 
  bind_rows(Human_18510_primary) %>% 
  bind_rows(Human_18510_secondary) %>% 
  bind_rows(Human_18523_primary) %>% 
  bind_rows(Human_18523_secondary)%>% 
  group_by(line, set) %>% 
  mutate(TotalReads=sum(Reads)) %>% 
  ungroup() %>%
  mutate(PropReads=Reads/TotalReads)
  
allHumanmismatch$Mis=factor(allHumanmismatch$Mis, levels=c("nM:i:0",  "nM:i:1" , "nM:i:2", "nM:i:3" , "nM:i:4", "nM:i:5" , "nM:i:6",  "nM:i:7" , "nM:i:8" , "nM:i:9","nM:i:10"))
```

Plot:  
```{r}
ggplot(allHumanmismatch, aes(x=Mis,y=PropReads,by=set, fill=set)) + geom_bar(stat="identity",position = "dodge") + facet_grid(~line)+  theme(axis.text.x = element_text(angle = 90, hjust = 1))  +labs(title="Human Mismatch in primary and secondary")
ggplot(allHumanmismatch, aes(x=Mis,y=Reads,by=set, fill=set)) + geom_bar(stat="identity",position = "dodge") + facet_grid(~line)+  theme(axis.text.x = element_text(angle = 90, hjust = 1))  +labs(title="Human Mismatch in primary and secondary")
```

This shows there is not an easy way to seperate the reads. Primary and secondary reads have mistmatches.  

##Mismatch score for unique reads. 
Seperate the reads map uniq. 
```{bash,eval=F}
sbatch Filter255MM.sh
```
Look at the results:  

```{r}
Chimp_18358_Unique=read.table("../data/TestMM2_mismatch/chimp_combined_18358_N_255mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Unique", species="Chimp", line="18358")


Chimp_3622_Unique=read.table("../data/TestMM2_mismatch/chimp_combined_3622_N_255mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Unique", species="Chimp", line="3622")



Chimp_3659_Unique=read.table("../data/TestMM2_mismatch/chimp_combined_3659_N_255mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Unique", species="Chimp", line="3659")


Chimp_4973_Unique=read.table("../data/TestMM2_mismatch/chimp_combined_4973_N_255mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Unique", species="Chimp",line="4973")


Chimp_pt30_Unique=read.table("../data/TestMM2_mismatch/chimp_combined_pt30_N_255mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Unique", species="Chimp", line="pt30")


Chimp_pt91_Unique=read.table("../data/TestMM2_mismatch/chimp_combined_pt91_N_255mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Unique", species="Chimp",line="pt91")

```

```{r}
Human_18498_Unique=read.table("../data/TestMM2_mismatch/human_combined_18498_N_255mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Unique", species="Human", line="18498")


Human_18499_Unique=read.table("../data/TestMM2_mismatch/human_combined_18499_N_255mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Unique", species="Human", line="18499")


Human_18502_Unique=read.table("../data/TestMM2_mismatch/human_combined_18502_N_255mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Unique", species="Human", line="18502")


Human_18504_Unique=read.table("../data/TestMM2_mismatch/human_combined_18504_N_255mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Unique", species="Human", line="18504")


Human_18510_Unique=read.table("../data/TestMM2_mismatch/human_combined_18510_N_255mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Unique", species="Human", line="18510")


Human_18523_Unique=read.table("../data/TestMM2_mismatch/human_combined_18523_N_255mismatch.txt", stringsAsFactors = F, col.names = c("Reads", "Mis")) %>% mutate(set="Unique", species="Human", line="18523")

```

Look at these.  

```{r}
allUniqmismatch=Human_18498_Unique %>% 
  bind_rows(Human_18499_Unique)  %>% 
  bind_rows(Human_18502_Unique) %>% 
  bind_rows(Human_18504_Unique) %>% 
  bind_rows(Human_18510_Unique) %>% 
  bind_rows(Human_18523_Unique) %>% 
  bind_rows(Chimp_18358_Unique) %>% 
  bind_rows(Chimp_3622_Unique) %>% 
  bind_rows(Chimp_3659_Unique) %>% 
  bind_rows(Chimp_4973_Unique) %>% 
  bind_rows(Chimp_pt30_Unique)%>% 
  bind_rows(Chimp_pt91_Unique)%>% 
  group_by(line, set) %>% 
  mutate(TotalReads=sum(Reads)) %>% 
  ungroup() %>%
  mutate(PropReads=Reads/TotalReads)

allUniqmismatch$Mis=factor(allUniqmismatch$Mis, levels=c("nM:i:0",  "nM:i:1" , "nM:i:2", "nM:i:3" , "nM:i:4", "nM:i:5" , "nM:i:6",  "nM:i:7" , "nM:i:8" , "nM:i:9","nM:i:10"))
```
```{r}
ggplot(allUniqmismatch, aes(x=Mis,y=Reads,by=species, fill=species)) + geom_bar(stat="identity",position = "dodge") + facet_grid(~line)+  theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(title="Unique Read mismatch")

ggplot(allUniqmismatch, aes(x=Mis,y=PropReads,by=species, fill=species)) + geom_bar(stat="identity",position = "dodge") + facet_grid(~line)+  theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(title="Unique Read mismatch")
```
So even uniq reads have mismatches.  
##Alignment scores   
"AS" SAM tag express the "Alignment score generated by aligner", 14th column  

Do this for the primary, secondary and unique mapping reads.  
```{bash,eval=F}
mkdir ../data/TestMM2_AS/

sbatch AlignmentScores.sh  
```

Chimp:  
```{r}
Chimp_18358_ASprimary=read.table("../data/TestMM2_AS/chimp_combined_18358_N_Primary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Primary", species="Chimp", line="18358")

Chimp_18358_ASsecondary=read.table("../data/TestMM2_AS/chimp_combined_18358_N_Secondary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Secondary", species="Chimp", line="18358")

Chimp_18358_ASuniq=read.table("../data/TestMM2_AS/chimp_combined_18358_N_255_AS.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Unique", species="Chimp", line="18358")

Chimp_3622_ASprimary=read.table("../data/TestMM2_AS/chimp_combined_3622_N_Primary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Primary", species="Chimp", line="3622")

Chimp_3622_ASsecondary=read.table("../data/TestMM2_AS/chimp_combined_3622_N_Secondary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Secondary", species="Chimp", line="3622")

Chimp_3622_ASuniq=read.table("../data/TestMM2_AS/chimp_combined_3622_N_255_AS.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Unique", species="Chimp", line="3622")


Chimp_3659_ASprimary=read.table("../data/TestMM2_AS/chimp_combined_3659_N_Primary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Primary", species="Chimp", line="3659")

Chimp_3659_ASsecondary=read.table("../data/TestMM2_AS/chimp_combined_3659_N_Secondary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Secondary", species="Chimp", line="3659")

Chimp_3659_ASuniq=read.table("../data/TestMM2_AS/chimp_combined_3659_N_255_AS.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Unique", species="Chimp", line="3659")


Chimp_4973_ASprimary=read.table("../data/TestMM2_AS/chimp_combined_4973_N_Primary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Primary", species="Chimp",line="4973")

Chimp_4973_ASsecondary=read.table("../data/TestMM2_AS/chimp_combined_4973_N_Secondary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Secondary", species="Chimp",line="4973")

Chimp_4973_ASuniq=read.table("../data/TestMM2_AS/chimp_combined_4973_N_255_AS.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Unique", species="Chimp", line="4973")


Chimp_pt30_ASprimary=read.table("../data/TestMM2_AS/chimp_combined_pt30_N_Primary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Primary", species="Chimp", line="pt30")

Chimp_pt30_ASsecondary=read.table("../data/TestMM2_AS/chimp_combined_pt30_N_Secondary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Secondary", species="Chimp",line="pt30")

Chimp_pt30_ASuniq=read.table("../data/TestMM2_AS/chimp_combined_pt30_N_255_AS.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Unique", species="Chimp", line="pt30")

Chimp_pt91_ASprimary=read.table("../data/TestMM2_AS/chimp_combined_pt91_N_Primary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Primary", species="Chimp",line="pt91")

Chimp_pt91_ASsecondary=read.table("../data/TestMM2_AS/chimp_combined_pt91_N_Secondary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Secondary", species="Chimp", line="pt91")

Chimp_pt91_ASuniq=read.table("../data/TestMM2_AS/chimp_combined_pt91_N_255_AS.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Unique", species="Chimp", line="pt91")

```

```{r}
ChimpAS=Chimp_18358_ASprimary %>% 
  bind_rows(Chimp_18358_ASsecondary) %>% 
  bind_rows(Chimp_18358_ASuniq) %>% 
  bind_rows(Chimp_3622_ASprimary) %>% 
  bind_rows(Chimp_3622_ASsecondary) %>% 
  bind_rows(Chimp_3622_ASuniq) %>% 
  bind_rows(Chimp_3659_ASprimary) %>% 
  bind_rows(Chimp_3659_ASsecondary) %>% 
  bind_rows(Chimp_3659_ASuniq) %>% 
  bind_rows(Chimp_4973_ASprimary) %>% 
  bind_rows(Chimp_4973_ASsecondary) %>% 
  bind_rows(Chimp_4973_ASuniq) %>% 
  bind_rows(Chimp_pt30_ASprimary) %>% 
  bind_rows(Chimp_pt30_ASsecondary) %>% 
  bind_rows(Chimp_pt30_ASuniq) %>% 
  bind_rows(Chimp_pt91_ASprimary) %>% 
  bind_rows(Chimp_pt91_ASsecondary) %>% 
  bind_rows(Chimp_pt91_ASuniq) %>% 
  group_by(line, set) %>% 
  mutate(TotalReads=sum(Reads)) %>% 
  ungroup() %>%
  mutate(PropReads=Reads/TotalReads)
```

```{r}
ggplot(ChimpAS, aes(x=AS,y=PropReads,by=set, fill=set)) + geom_bar(stat="identity",position = "dodge") + facet_grid(~line)+  theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(title="Alignment scores")

```
Filter 1 individual to see better:

```{r}
ChimpAS18358= ChimpAS %>% filter(line=="18358")

ggplot(ChimpAS18358, aes(x=AS,y=PropReads,by=set, fill=set)) + geom_bar(stat="identity",position = "dodge")+ theme(axis.text.x = element_text(angle = 90, hjust = 1)) + scale_fill_brewer(palette = "Dark2") + labs(tite="Alignment scores for Chimp 18358",x="",y="proportion of reads")
```

Try human: 

```{r}
Human_18498_ASprimary=read.table("../data/TestMM2_AS/human_combined_18498_N_Primary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Primary", species="Human", line="18498")

Human_18498_ASsecondary=read.table("../data/TestMM2_AS/human_combined_18498_N_Secondary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Secondary", species="Human", line="18498")

Human_18498_ASuniq=read.table("../data/TestMM2_AS/human_combined_18498_N_255_AS.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Unique", species="Human", line="18498")


Human_18499_ASprimary=read.table("../data/TestMM2_AS/human_combined_18499_N_Primary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Primary", species="Human", line="18499")

Human_18499_ASsecondary=read.table("../data/TestMM2_AS/human_combined_18499_N_Secondary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Secondary", species="Human", line="18499")

Human_18499_ASuniq=read.table("../data/TestMM2_AS/human_combined_18499_N_255_AS.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Unique", species="Human", line="18499")


Human_18502_ASprimary=read.table("../data/TestMM2_AS/human_combined_18502_N_Primary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Primary", species="Human", line="18502")

Human_18502_ASsecondary=read.table("../data/TestMM2_AS/human_combined_18502_N_Secondary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Secondary", species="Human", line="18502")

Human_18502_ASuniq=read.table("../data/TestMM2_AS/human_combined_18502_N_255_AS.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Unique", species="Human", line="18502")


Human_18504_ASprimary=read.table("../data/TestMM2_AS/human_combined_18504_N_Primary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Primary", species="Human", line="18504")

Human_18504_ASsecondary=read.table("../data/TestMM2_AS/human_combined_18504_N_Secondary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Secondary", species="Human", line="18504")

Human_18504_ASuniq=read.table("../data/TestMM2_AS/human_combined_18504_N_255_AS.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Unique", species="Human", line="18504")


Human_18510_ASprimary=read.table("../data/TestMM2_AS/human_combined_18510_N_Primary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Primary", species="Human", line="18510")

Human_18510_ASsecondary=read.table("../data/TestMM2_AS/human_combined_18510_N_Secondary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Secondary", species="Human", line="18510")

Human_18510_ASuniq=read.table("../data/TestMM2_AS/human_combined_18510_N_255_AS.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Unique", species="Human", line="18510")


Human_18523_ASprimary=read.table("../data/TestMM2_AS/human_combined_18523_N_Primary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Primary", species="Human", line="18523")

Human_18523_ASsecondary=read.table("../data/TestMM2_AS/human_combined_18523_N_Secondary.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Secondary", species="Human", line="18523")

Human_18523_ASuniq=read.table("../data/TestMM2_AS/human_combined_18523_N_255_AS.txt", stringsAsFactors = F, col.names = c("Reads", "AS")) %>% mutate(set="Unique", species="Human", line="18523")


```

```{r}
Human_AS=Human_18498_ASprimary %>% 
  bind_rows(Human_18498_ASsecondary) %>% 
  bind_rows(Human_18498_ASuniq) %>% 
  bind_rows( Human_18499_ASprimary) %>% 
  bind_rows(Human_18499_ASsecondary) %>% 
  bind_rows(Human_18499_ASuniq) %>% 
  bind_rows( Human_18502_ASprimary) %>% 
  bind_rows(Human_18502_ASsecondary) %>% 
  bind_rows(Human_18502_ASuniq) %>%
  bind_rows( Human_18504_ASprimary) %>% 
  bind_rows(Human_18504_ASsecondary) %>% 
  bind_rows(Human_18504_ASuniq) %>%
  bind_rows( Human_18510_ASprimary) %>% 
  bind_rows(Human_18510_ASsecondary) %>% 
  bind_rows(Human_18510_ASuniq) %>%  
  bind_rows( Human_18523_ASprimary) %>% 
  bind_rows(Human_18523_ASsecondary) %>% 
  bind_rows(Human_18523_ASuniq) %>%
    group_by(line, set) %>% 
    mutate(TotalReads=sum(Reads)) %>% 
   ungroup() %>%
    mutate(PropReads=Reads/TotalReads)
```

```{r}
ggplot(Human_AS, aes(x=AS,y=PropReads,by=set, fill=set)) + geom_bar(stat="identity",position = "dodge")+ theme(axis.text.x = element_text(angle = 90, hjust = 1)) + scale_fill_brewer(palette = "Dark2") + labs(tite="Alignment scores for Human Lines",x="",y="proportion of reads")+facet_grid(~line)
```

One line:  


```{r}
HumanAS18523= Human_AS %>% filter(line=="18523")

ggplot(HumanAS18523, aes(x=AS,y=PropReads,by=set, fill=set)) + geom_bar(stat="identity",position = "dodge")+ theme(axis.text.x = element_text(angle = 90, hjust = 1)) + scale_fill_brewer(palette = "Dark2") + labs(tite="Alignment scores for Human 18523",x="",y="proportion of reads")
```
This does not differentiate good and bad reads either.  
