---
title: "Cutoffs for multimapping"
author: "Briana Mittleman"
date: "3/26/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


```{r}
library(workflowr)
library(tidyverse)
```


I want to look at the parameters for multimapping. I will look at the paremeters such as include a read when the first vs second map difference is 10%, 5% ect. I want to get to a cutoff where the PAS lost are not in the ortho exon. 

I will look at the featurecounts documentation for this or I can look directly at the star info.   


−−primary: If specified, only primary alignments will be counted. Primary and secondary alignments are identified using bit 0x100 in the Flag field of SAM/BAM files. All primary alignments in a dataset will be counted no matter they are from multi- mapping reads or not (ie. ‘-M’ is ignored).

For multi-mappers, all alignments except one are marked with 0x100 (secondary alignment) in the FLAG (column 2 of the SAM

I used options:

 --outFilterMultimapNmax 10 --outSAMmultNmax 1 
 
 
 --outSAMmultNmax 1 will output exactly one SAM line for each mapped read
 
 I only allowed for 1 read to be mapped. I want the secondary one as well. I want to mess with the secondary read.  
 
 Remap human and chimp reads with the new filter. Just to look first, don't worry about misspriming.  
 
```{bash,eval=F}
mkdir ../data/TestMM2/
sbatch StarMM2.sh
sbatch ChimpStarMM2.sh
```
 
 chimp_combined_18358_N.MM2.sort.bam   is done. I can use this to look at the seondary mapped reads.  
 
 
 HI:i:i Query hit index, indicating the alignment record is the i-th one stored in SAM. 
 
 the HI:i:2 reads are the secondary maped reads.  
 
 NM:i:count  number of mismatches  
 
 256 is the secondary allignment 
 
 I can use pysam to filter secondary reads.  
 
```{bash,eval=F}
samtools view chimp_combined_18358_N.MM2.sort.bam  | cut -f5 | sort -n | uniq -c
```

1860484 0
2727360 1
3182906 3
39284071 255  


without multmap:  

 930242 0
1363680 1
1591453 3
39284071 255


I want to filter just secondary reads and get the scores-
secondary alignment score is in column 2

```{bash,eval=F}
samtools view chimp_combined_18358_N.MM2.sort.bam  | cut -f2 | sort -n | uniq -c
```
Primary (by strand)
23943403 0
19226043 16

seconday (by strand)  
1571873 256
2313502 272


"AS" SAM tag express the "Alignment score generated by aligner"
-scores between 53,85

get scores for primary and secondary  


Mismatches  (with secondary)
34622475 nM:i:0
7600018 nM:i:1
  57259 nM:i:10
2337437 nM:i:2
1045189 nM:i:3
 547184 nM:i:4
 327976 nM:i:5
 213004 nM:i:6
 138913 nM:i:7
  96974 nM:i:8
  68392 nM:i:9
  
Primary  

14717503 nM:i:0
3042728 nM:i:1
  21887 nM:i:10
 943180 nM:i:2
 406973 nM:i:3
 217371 nM:i:4
 132942 nM:i:5
  88071 nM:i:6
  54904 nM:i:7
  38671 nM:i:8
  26097 nM:i:9
  
  
Seperate secondary, write code for any file:  

```{bash,eval=F}
mkdir ../data/TestMM2_SeondaryRead
mkdir ../data/TestMM2_PrimaryRead

python filterSecondaryread.py ../data/TestMM2/chimp_combined_18358_N.MM2.sort.bam  ../data/TestMM2_SeondaryRead/chimp_combined_18358_N.MM2_secondary.bam


python filterPrimaryread.py ../data/TestMM2/chimp_combined_18358_N.MM2.sort.bam  ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.bam


samtools sort -o ../data/TestMM2_SeondaryRead/chimp_combined_18358_N.MM2_secondary.sort.bam -O bam  ../data/TestMM2_SeondaryRead/chimp_combined_18358_N.MM2_secondary.bam

samtools sort -o ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.sort.bam -O bam ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.bam

samtools index ../data/TestMM2_SeondaryRead/chimp_combined_18358_N.MM2_secondary.sort.bam

samtools index ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.sort.bam



             
```

scores 53-84  

2426470 nM:i:0
 919015 nM:i:1
   9997 nM:i:10
 253038 nM:i:2
 108368 nM:i:3
  59802 nM:i:4
  39210 nM:i:5
  26476 nM:i:6
  18892 nM:i:7
  13544 nM:i:8
  10563 nM:i:9
  
This means a lot of the secondary reads also have 0 mismatch. These are probably the reads I would want to filter.  


Compare the porportion.  

```{bash,eval=F}
mkdir ../data/TestMM2_quality

samtools view chimp_combined_18358_N.MM2_secondary.sort.bam | cut -f15 | sort -n | uniq -c > ../TestMM2_quality/chimp18358_secondaryMismatch.txt

samtools view chimp_combined_18358_N.MM2.sort.bam | cut -f15 | sort -n | uniq -c > ../TestMM2_quality/chimp18358_PrimaryMismatch.txt

samtools view ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.sort.bam | cut -f15 | sort -n | uniq -c > ../data/TestMM2_quality/chimp18358_PrimaryOnlyMismatch.txt
```


Compare this:  

```{r}
SecMismatch=read.table("../data/TestMM2_quality/chimp18358_secondaryMismatch.txt",col.names = c("Reads","Mismatch"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="Secondary")
PrimaryOnlyMismatch= read.table("../data/TestMM2_quality/chimp18358_PrimaryOnlyMismatch.txt",col.names = c("Reads","Mismatch"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="Primary")
PrimaryMismatch=read.table("../data/TestMM2_quality/chimp18358_PrimaryMismatch.txt",col.names = c("Reads","Mismatch"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="All")

BothMiss= SecMismatch %>% bind_rows(PrimaryOnlyMismatch)


ggplot(BothMiss, aes(x=Mismatch, y=PropReads, by=set, fill=set))+ geom_bar(stat="identity", position="dodge")+ labs(title="Primary v Seconday Mismatch Chimp 18358")
```
Do this for scores as well  

```{bash,eval=F}
samtools view ../data/TestMM2_SeondaryRead/chimp_combined_18358_N.MM2_secondary.sort.bam | cut -f14 | sort -n | uniq -c > ../data/TestMM2_quality/chimp18358_secondaryScores.txt

samtools view ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.sort.bam | cut -f14 | sort -n | uniq -c > ../data/TestMM2_quality/chimp18358_PrimaryOnlyScores.txt
```

```{r}
SecScore=read.table("../data/TestMM2_quality/chimp18358_secondaryScores.txt",col.names = c("Reads","Score"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="Secondary")
PrimaryOnlyScore= read.table("../data/TestMM2_quality/chimp18358_PrimaryOnlyScores.txt",col.names = c("Reads","Score"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="Primary")

BothScore= SecScore %>% bind_rows(PrimaryOnlyScore)

ggplot(BothScore, aes(x=Score, y=PropReads, by=set, fill=set))+ geom_bar(stat="identity", position="dodge") +  theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(title="Primary v Seconday Scores Chimp 18358")

```


Feature counts change the -Q for quality. what score is this? 

may q scores?  

```{bash,eval=F}
samtools view ../data/TestMM2_SeondaryRead/chimp_combined_18358_N.MM2_secondary.sort.bam | cut -f5 | sort -n | uniq -c > ../data/TestMM2_quality/chimp18358_secondaryMAPScores.txt

samtools view ../data/TestMM2_PrimaryRead/chimp_combined_18358_N.MM2_primary.sort.bam | cut -f5 | sort -n | uniq -c > ../data/TestMM2_quality/chimp18358_PrimaryOnlyMAPScores.txt
```


```{r}
SecMAPScore=read.table("../data/TestMM2_quality/chimp18358_secondaryMAPScores.txt",col.names = c("Reads","Score"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="Secondary")
PrimaryMAPOnlyScore= read.table("../data/TestMM2_quality/chimp18358_PrimaryOnlyMAPScores.txt",col.names = c("Reads","Score"),stringsAsFactors = F) %>% mutate(TotalRead=sum(Reads),PropReads=Reads/TotalRead,set="Primary")

BothMAPScore= SecMAPScore %>% bind_rows(PrimaryMAPOnlyScore)

ggplot(BothMAPScore, aes(x=Score, y=PropReads, by=set, fill=set))+ geom_bar(stat="identity", position="dodge") +  theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(title="Primary v Seconday Scores Chimp 18358")

```

secondary score in column 2  

```{bash,eval=F}
samtools view ../data/TestMM2_SeondaryRead/chimp_combined_18358_N.MM2_secondary.sort.bam | cut -f2 | sort -n | uniq -c > ../data/TestMM2_quality/chimp18358_secondaryExtraScores.txt
```

