---
title: "PhyloP regulatory elements"
author: "Briana Mittleman"
date: "12/21/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

```{r}
library(workflowr)
library(cowplot)
library(tidyverse)
```

I will download regulatory elements and compare them to the PAS regions. I will run this for phylop20 and phlylop100

/project2/gilad/briana/Comparative_APA/data/RegElements

Download from table browser:

* refseq elements 
* sno and miRNAs 



```{bash,eval=F}


python extractPhylopGeneral_20way.py   ../data/RegElements/hg38_RefSeqElements.bed ../data/RegElements/hg38_RefSeqElements.phyloP20.txt

python extractPhylopGeneral_20way.py  ../data/RegElements/hg38_snomiRNA.bed ../data/RegElements/hg38_snomiRNAphyloP20.txt 



python extractPhylopGeneral.py   ../data/RegElements/hg38_RefSeqElements.bed ../data/RegElements/hg38_RefSeqElements.phyloP100.txt

python extractPhylopGeneral.py  ../data/RegElements/hg38_snomiRNA.bed ../data/RegElements/hg38_snomiRNAphyloP100.txt 



```


```{r}
PAS_20=read.table("../data/PhyloP/PAS_Phylop20.txt", col.names = c("chr","start","end","PAS", "phyloP20"), stringsAsFactors = F) %>% drop_na() %>% mutate(element="PAS") %>% select(element,phyloP20)

PAS_100=read.table("../data/PhyloP/PAS_phyloP.txt", col.names = c("chr","start","end", "phyloP100"), stringsAsFactors = F) %>% drop_na()  %>%  mutate(element="PAS") %>% select(element,phyloP100)
```

Reg elements:

```{r}
Reg_20= read.table("../data/RegElements/hg38_RefSeqElements.phyloP20.txt", col.names = c('chr', 'start', 'end', 'element', 'phyloP20' ), header = F, stringsAsFactors = F) %>% drop_na()%>% select(element, phyloP20)

RNAs_20=read.table("../data/RegElements/hg38_snomiRNAphyloP20.txt", col.names = c('chr', 'start', 'end', 'elementName', 'phyloP20' ), header = F, stringsAsFactors = F)%>% mutate(element="smallRNA") %>% drop_na()%>% select(element, phyloP20)

Reg_100= read.table("../data/RegElements/hg38_RefSeqElements.phyloP100.txt", col.names = c('chr', 'start', 'end', 'element', 'phyloP100' ), header = F, stringsAsFactors = F) %>% drop_na()%>% select(element, phyloP100)

RNAs_100=read.table("../data/RegElements/hg38_snomiRNAphyloP100.txt", col.names = c('chr', 'start', 'end', 'element_name', 'phyloP100' ), header = F, stringsAsFactors = F) %>% mutate(element="smallRNA") %>% drop_na() %>% select(element, phyloP100)
```


Look at the elements in the reg and decide which to include: 

```{r}
elementcount=Reg_20 %>% group_by(element) %>% summarise(nele=n()) %>% arrange(desc(nele))

regs_use=c("conserved_region", "enhancer", "CpG_island", "promoter", "silencer", "repeat_instability_region")
```
Will include: 
conserved_region, enhancer,CpG_island, promoter, silencer, repeat_instability_region

```{r}
Reg_20_sm=Reg_20 %>% filter(element %in% regs_use)
Reg_100_sm=Reg_100 %>% filter(element %in% regs_use)


RegandRNA_20=Reg_20_sm %>% bind_rows(RNAs_20) %>% bind_rows(PAS_20) %>% mutate(fillcol=ifelse(element=="PAS", "yes", "no"))

RegandRNA_100=Reg_100_sm %>% bind_rows(RNAs_100) %>% bind_rows(PAS_100)%>% mutate(fillcol=ifelse(element=="PAS", "yes", "no"))
```


Plot: 

```{r}
phylop20plot=ggplot(RegandRNA_20, aes(x=element, y=phyloP20,fill=fillcol)) + geom_boxplot() + theme_classic() + labs(title="Regulatory element conservation", y="PhyloP 20 ways", x="")+ theme(axis.text.x = element_text(angle = 90), legend.position ="none")+ scale_fill_brewer(palette = "RdYlBu")
phylop20plot
```

```{r}
phylop100plot= ggplot(RegandRNA_100, aes(x=element, y=phyloP100,fill=fillcol)) + geom_boxplot() + theme_classic() + labs(title="Regulatory element conservation", y="PhyloP 100 ways", x="")+ theme(axis.text.x = element_text(angle = 90), legend.position ="none")+ scale_fill_brewer(palette = "RdYlBu")
phylop100plot
```


```{r}
both_together=plot_grid(phylop20plot,phylop100plot, nrow = 2, labels=c("a", "b"))
both_together
```

```{r}
pdf("../output/supplement/Revision-phylopRed.pdf", height=8, width=6,useKerning=F)
both_together
dev.off()
```


Add introns and exon:

get chr 1-22 from intron and exon file

subsetChr.py
```{python,eval=F}

main(fin, fout):
  small=open(fout, "w")
  chrkeep=["chr1", "chr2", 'chr3', 'chr4', 'chr5', 'chr6', 'chr7', 'chr8', 'chr9', 'chr10', 'chr11', 'chr12', 'chr13', 'chr14', 'chr15', 'chr16', 'chr17', 'chr18', 'chr19', 'chr20', 'chr21','chr22']
  for ln in open(fin,"r"):
    chrom=ln.split()[0]
    if chrom in chrkeep:
        small.write(ln)
  small.close()


if __name__ == "__main__":
    import sys
    infile = sys.argv[1]
    Outfile =sys.argv[2]
    main(infile, Outfile)
```

```{bash,eval=F}
python subsetChr.py ../data/RegElements/hg38_ncbiRefseq_Formatted_Allannotation_noSNO.Resort.bed ../data/RegElements/hg38_ncbiRefseq_Formatted_Allannotation_noSNO.Resort.1.22.bed



python extractPhylopGeneral_20way.py  ../data/RegElements/hg38_ncbiRefseq_Formatted_Allannotation_noSNO.Resort.1.22.bed ../data/RegElements/phylop20_ncbi.txt

```

```{r}
Allloc_20=read.table("../data/RegElements/phylop20_ncbi.txt", stringsAsFactors = F, header = F, col.names = c("chr", "start","end", "name", "phyloP20")) %>% separate(name, into=c("element", "gene"),sep=":") %>% drop_na() 

ggplot(Allloc_20, aes(x=element, y=phyloP20)) + geom_boxplot() + theme_classic()
```

```{r}
Intron_20=Allloc_20 %>% filter(element=="intron") %>% select(element, phyloP20)

RegandRNAintron_20=Reg_20_sm %>% bind_rows(RNAs_20) %>% bind_rows(PAS_20) %>% bind_rows(Intron_20) %>%  mutate(fillcol=ifelse(element=="PAS", "yes", "no"))

ggplot(RegandRNAintron_20, aes(x=element, y=phyloP20,fill=fillcol)) + geom_boxplot() + theme_classic() + labs(title="Regulatory element conservation", y="PhyloP 20 ways", x="")+ theme(axis.text.x = element_text(angle = 90), legend.position ="none")+ scale_fill_brewer(palette = "RdYlBu")
```
Significance: 
```{r}
promoter20= RegandRNA_20 %>% filter(element=="promoter")
enhan20= RegandRNA_20 %>% filter(element=="enhancer")
consv20= RegandRNA_20 %>% filter(element=="conserved_region")
cpg20= RegandRNA_20 %>% filter(element=="CpG_island")
repeate20= RegandRNA_20 %>% filter(element=="repeat_instability_region")
silencer20= RegandRNA_20 %>% filter(element=="silencer")
smallRNA20= RegandRNA_20 %>% filter(element=="smallRNA")


wilcox.test(PAS_20$phyloP20,promoter20$phyloP20, alternative = "greater")
wilcox.test(PAS_20$phyloP20,enhan20$phyloP20, alternative = "greater")
wilcox.test(PAS_20$phyloP20,consv20$phyloP20, alternative = "greater")
wilcox.test(PAS_20$phyloP20,cpg20$phyloP20, alternative = "greater")
wilcox.test(PAS_20$phyloP20,repeate20$phyloP20, alternative = "greater")
wilcox.test(PAS_20$phyloP20,silencer20$phyloP20, alternative = "greater")
wilcox.test(PAS_20$phyloP20,smallRNA20$phyloP20, alternative = "greater")


```


```{r}
Allloc_100=read.table("../data/RegElements/phylop100_ncbi.txt", stringsAsFactors = F, header = F, col.names = c("chr", "start","end", "name", "phyloP100")) %>% separate(name, into=c("element", "gene"),sep=":") %>% drop_na()


Intron_100=Allloc_100 %>% filter(element=="intron") %>% select(element, phyloP100)

RegandRNAintron_100=Reg_100_sm %>% bind_rows(RNAs_100) %>% bind_rows(PAS_100) %>% bind_rows(Intron_100) %>%  mutate(fillcol=ifelse(element=="PAS", "yes", "no"))

ggplot(RegandRNAintron_100, aes(x=element, y=phyloP100,fill=fillcol)) + geom_boxplot() + theme_classic() + labs(title="Regulatory element conservation", y="PhyloP 100 ways", x="")+ theme(axis.text.x = element_text(angle = 90), legend.position ="none")+ scale_fill_brewer(palette = "RdYlBu")
```

```{r}

promoter100= RegandRNA_100 %>% filter(element=="promoter")
enhan100= RegandRNA_100 %>% filter(element=="enhancer")
consv100= RegandRNA_100 %>% filter(element=="conserved_region")
cpg100= RegandRNA_100 %>% filter(element=="CpG_island")
repeate100= RegandRNA_100 %>% filter(element=="repeat_instability_region")
silencer100= RegandRNA_100 %>% filter(element=="silencer")
smallRNA100= RegandRNA_100 %>% filter(element=="smallRNA")


wilcox.test(PAS_100$phyloP100,promoter100$phyloP100, alternative = "greater")
wilcox.test(PAS_100$phyloP100,enhan100$phyloP100, alternative = "greater")
wilcox.test(PAS_100$phyloP100,consv100$phyloP100, alternative = "greater")
wilcox.test(PAS_100$phyloP100,cpg100$phyloP100, alternative = "greater")
wilcox.test(PAS_100$phyloP100,repeate100$phyloP100, alternative = "greater")
wilcox.test(PAS_100$phyloP100,silencer100$phyloP100, alternative = "greater")
wilcox.test(PAS_100$phyloP100,smallRNA100$phyloP100, alternative = "greater")



```


