---
title: "Change Misprimming to exclude reads with Ns"
author: "Briana Mittleman"
date: "3/12/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

The chimp genome has more reads with N's. In order to make sure this is not biasing the results I will reimpliment the pipeline with a new restriction for any reads with an N. I will see if the number of reads are significantly different after this or if the top PAS distribution.  
```{r}
library(tidyverse)
```


```{bash,eval=F}
mkdir ../../Comparative_NoNs
cd Comparative_NoNs
mkdir Human
mkdir Human/data
mkdir Human/data/fastq

mkdir Chimp
mkdir Chimp/data
mkdir Chimp/data/fastq

#copy fastqs 
```

Files to copy over from code.  

cluster.json
Config_chimp.yaml
Config_human.yaml
filterBamforMP.pysam2_gen.py
filterMissprimingInNuc10_gen.py
filterSortBedbyCleanedBed_gen.R
Snakefile
snakemake_chimp.batch
snakemake_human.batch
submit-snakemake-chimp.sh
submit-snakemake-human.sh
Upstream10Bases_general.py

Change the general upstream clean to remove reads with any Ns.  

Change config files to new dir.  

Remove RNA seq rules.  

New filter code:  
Add a output with the number of reads filtered due to number of Ns 

```{bash,eval=F}
filterMissprimingInNuc10_genNoNs.py
```
Create a log dir.  

```{bash,eval=F}
sbatch snakemake_chimp.batch
sbatch snakemake_human.batch
```

First Pass will be quantifing how many reads are lost due to this.  

Look at the results  
```{r}
Mapping=read.csv("../../Comparative_NoNs/data/HC_metadataNoNs.csv", header = T) %>% mutate(PercMap=NewClean/Reads)
```

```{r}
ggplot(Mapping,aes(x=Line, fill=Species, y=PercMap)) + geom_bar(stat="identity") + facet_grid(~Fraction) +theme(axis.text.x = element_text(angle = 90)) + labs(y= "Map proportion", title="Map Stats") + scale_fill_brewer(palette = "Dark2")
```
```{r}
ggplot(Mapping,aes(x=Line, fill=Species, y=ReadsLost)) + geom_bar(stat="identity") + facet_grid(~Fraction) +theme(axis.text.x = element_text(angle = 90)) + labs(y= "Number Reads", title="Reads filtered due to N's") + scale_fill_brewer(palette = "Dark2")
```

Look at the number of Ns in the reads.  

I will run bedtools nuc on the full reads.  

Do this for the clean bed and not 
```{bash,eval=F}
mkdir ../Chimp/data/readSeq
mkdir ../Chimp/data/readSeqClean
mkdir ../Human/data/readSeq
mkdir ../Human/data/readSeqClean
sbatch NucForAllReads.sh

```

I will want to plot the number of reads with an N I can write a python script that writes a line the name of the sample and the number of reads with an N (remember .upper)

Run this script on all of the files. 

```{bash,eval=F}
sbatch runCountNs.sh
```


```{r}

type=c(rep("Pre", 12), rep("Post", 12 ),rep("Pre", 12), rep("Post", 12 ))

Nres=read.table("../../Comparative_NoNs/data/CountsforNs.txt", col.names = c("library","NumNs", "NumReads"),stringsAsFactors = F) %>% separate(library, into=c("species", "comb", "line", "fraction"),sep="_")  %>% mutate(PropReadsN=NumNs/NumReads) %>% bind_cols("type"=type)

ggplot(Nres, aes(x=line,y=PropReadsN,fill=species))+ geom_bar(stat="identity") + facet_grid(type~fraction) +theme(axis.text.x = element_text(angle = 90)) + labs(y= "Proprotion of Reads", title="Proportion of Reads with N's") + scale_fill_brewer(palette = "Dark2")


ggplot(Nres, aes(x=line,y=NumNs,fill=species))+ geom_bar(stat="identity") + facet_grid(type~fraction) +theme(axis.text.x = element_text(angle = 90)) + labs(y= "Reads", title=" Reads with N's") + scale_fill_brewer(palette = "Dark2")

```

This is interesting because it is opposite of the expectation. When I compared the 2 genomes I looked at Ns vs Ts in the upstream 10. I will make the same plots here.  

```{sbatch,eval=F}
sbatch runCountNucleotides.sh
```

```{r}
countRes=read.table("../../Comparative_NoNs/data/CountsforNucleotides.txt", stringsAsFactors = F, header = F, col.names = c("library", "Nucleotide", "numLetter","NumReads")) %>% separate(library, into=c("species", "com", "individual", "fraction")) %>% mutate(NormN=numLetter/(NumReads*10), Fraction=ifelse(fraction=="N", "Nuclear", "Total")) 

countRes_N= countRes %>% filter(Nucleotide=="N")
countRes_T= countRes %>% filter(Nucleotide=="T")


ggplot(countRes_N, aes(x=individual, y=NormN, fill=species)) + geom_bar(stat="identity") + facet_grid(~Fraction)+ theme(axis.text.x = element_text(angle = 90)) +  labs(title="N nucleotides per base",y="N nucleotide per base")+scale_fill_brewer(palette = "Dark2")
```
```{r}
ggplot(countRes_N, aes(x=individual, y=NormN, fill=species)) + geom_bar(stat="identity") + facet_grid(~Fraction)+ theme(axis.text.x = element_text(angle = 90)) +  labs(title="N nucleotides per base",y="N nucleotide per base") +scale_y_log10()+scale_fill_brewer(palette = "Dark2")
```

```{r}
ggplot(countRes_T, aes(x=individual, y=NormN, fill=species)) + geom_bar(stat="identity") + facet_grid(~Fraction) + theme(axis.text.x = element_text(angle = 90)) + labs(title="T nucleotides per base", y="T nucleotide per base")+ scale_fill_brewer(palette = "Dark2")
```

```{r}
ggplot(countRes_N, aes(x=individual, y=numLetter, fill=species)) + geom_bar(stat="identity") + facet_grid(~Fraction)+ theme(axis.text.x = element_text(angle = 90)) +  labs(title="N nucleotides per base",y="N nucleotide per base")+scale_fill_brewer(palette = "Dark2")
```

