---
title: "dAPA dIC together and sep"
author: "Briana Mittleman"
date: "5/7/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

```{r}
library(workflowr)
library(UpSetR)
library(VennDiagram)
library(tidyverse)
library(cowplot)

```

I want to look at regulatory phenotype regulation based on dAPA, both, or dIC.  


For this analysis I will use dIC at 5% FDR. Numbers are smaller but overlaps suggest it is more biological.  

I will look at genes tested in all analysis then proportion results to only dAPA, dIC and dAPA, or dIC only. I will test for enrichement in each of these sets with expression, translation, and protein.  


Load APA data:

For apa I reduce to gene level and count it as sig if at least one PAS is different. 
```{r}
Meta=read.table("../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", header = T, stringsAsFactors = F) 


Meta_genes= Meta %>% select(gene) %>% unique()

Meta_PAS=Meta %>%  select(PAS,gene)

dAPAGenes=read.table("../data/DiffIso_Nuclear_DF/SignifianceEitherGENES_Nuclear.txt", header = T, stringsAsFactors = F)
dAPAPAS=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T, stringsAsFactors = F) %>% inner_join(Meta, by=c("chr","start", "end","gene")) %>% select(PAS,gene,SigPAU2 ) 

dAPAPAS_genes= dAPAPAS %>% select(gene) %>% unique()

dAPATestedGenes= dAPAPAS  %>% select(gene) %>% unique() %>% mutate(dAPA=ifelse(gene %in% dAPAGenes$gene,"Yes", "No")) 


```


```{r}
dICdata= read.table("../data/IndInfoContent/SimpsonMedianSignificance.txt", header = T, stringsAsFactors = F)%>% select(sIC,gene)
dICdata_sig= dICdata %>% filter(sIC=="Yes")
```


```{r}
dAPAandDic= dICdata %>% inner_join(dAPATestedGenes,by="gene") %>% mutate(Both=ifelse(sIC=="Yes" & dAPA=="Yes", "Yes","No"),OnlyIC=ifelse(sIC=="Yes" & dAPA=="No", "Yes","No"),OnlyAPA=ifelse(sIC=="No" & dAPA=="Yes", "Yes","No"))

nrow(dAPAandDic)
```

Make an upsetter plot first:  

```{upset, dev="pdf", fig.height=6, fig.width=6, crop=FALSE}
#useCOl <- c("#d73027", "#4575b4","#fee090")


listInput <- list(SiteLevel=dAPAGenes$gene, IsoformDiversity=dICdata_sig$gene)

upset(fromList(listInput), order.by = "freq", empty.intersections = "on")


```


```{r,include=F, eval=F}
pdf("../output/supplement/Fig2-figSup7.pdf")
upset(fromList(listInput), order.by = "freq", empty.intersections = "on")
dev.off()
```


Ven diagram:

```{r}

overlap=intersect(dAPAGenes$gene,dICdata_sig$gene)

grid.newpage()
venn.plot <- draw.pairwise.venn(area1 = length(dAPAGenes$gene),
                           area2 = length(dICdata_sig$gene),
                           cross.area = length(overlap),
                           c("Site Level", "Isoform Diversity"), scaled = TRUE,
                           fill = c("#d73027", "#fee090"),
                           cex = 1.5,
                           cat.cex = 1.5,
                           cat.pos = c(320, 25),
                           cat.dist = .05) 
```



##Expression  

```{r}
nameID=read.table("../../genome_anotation_data/ensemble_to_genename.txt",sep="\t", header = T, stringsAsFactors = F) %>% dplyr::select(Gene_stable_ID, Gene.name)

DiffExp=read.table("../data/DiffExpression/DEtested_allres.txt",stringsAsFactors = F,header = F, col.names = c("Gene_stable_ID" ,"logFC" ,"AveExpr" , "t" ,  "P.Value" ,  "adj.P.Val", "B"  )) %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::rename('gene'=Gene.name) %>% dplyr::select(-Gene_stable_ID) %>% mutate(DE=ifelse(adj.P.Val<.05, "Yes", "No")) %>% select(gene,DE)
```

```{r}
DEandAPA=DiffExp %>% inner_join(dAPAandDic,by="gene")
nrow(DEandAPA)
```

Erichment for only APA:  


```{r}
sets=c("OnlyAPA", "OnlyIC", "Both")
DE_pval=c()
DE_enrich=c()
```

```{r}
x=nrow(DEandAPA %>% filter(OnlyAPA=="Yes", DE=="Yes"))
m=nrow(DEandAPA %>% filter(DE=="Yes"))
n=nrow(DEandAPA %>% filter(DE=="No"))
k=nrow(DEandAPA %>% filter(OnlyAPA=="Yes"))
N=nrow(DEandAPA)
phyper(x-1,m,n,k,lower.tail=F)
DE_pval=c(DE_pval, phyper(x-1,m,n,k,lower.tail=F))
x
DE_enrich=c(DE_enrich, (x/k)/(m/N))
(x/k)/(m/N)

```

Only dIC

```{r}
x=nrow(DEandAPA %>% filter(OnlyIC=="Yes", DE=="Yes"))
m=nrow(DEandAPA %>% filter(DE=="Yes"))
n=nrow(DEandAPA %>% filter(DE=="No"))
k=nrow(DEandAPA %>% filter(OnlyIC=="Yes"))
N=nrow(DEandAPA)
phyper(x-1,m,n,k,lower.tail=F)

DE_pval=c(DE_pval, phyper(x-1,m,n,k,lower.tail=F))
x
DE_enrich=c(DE_enrich, (x/k)/(m/N))
(x/k)/(m/N)
```

Both: 

```{r}
x=nrow(DEandAPA %>% filter(Both=="Yes", DE=="Yes"))
m=nrow(DEandAPA %>% filter(DE=="Yes"))
n=nrow(DEandAPA %>% filter(DE=="No"))
k=nrow(DEandAPA %>% filter(Both=="Yes"))
N=nrow(DEandAPA)
phyper(x-1,m,n,k,lower.tail=F)

DE_pval=c(DE_pval, phyper(x-1,m,n,k,lower.tail=F))
x
DE_enrich=c(DE_enrich, (x/k)/(m/N))
(x/k)/(m/N)

```
All de res: 

```{r}
DEdf=as.data.frame(cbind(sets,DE_pval, DE_enrich)) %>% rename(Pval=DE_pval, Enrichment=DE_enrich) %>% mutate(Pheno="Expression")
DEdf
```

##Translation  

```{r}
Ribo=read.table("../data/Wang_ribo/Additionaltable5_translationComparisons.txt",header = T, stringsAsFactors = F) %>% rename("Gene_stable_ID"= ENSG) %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::select(Gene.name, HvC.beta, HvC.pvalue, HvC.FDR) %>% rename("gene"=Gene.name) %>% mutate(dTE=ifelse(HvC.FDR <0.05, "Yes","No"))
RiboSmall= Ribo %>% select(gene,dTE)

```


```{r}
DTandAPA=Ribo %>% inner_join(dAPAandDic,by="gene")
nrow(DTandAPA)
```
```{r}
#sets=c("OnlyAPA", "OnlyIC", "Both")
DT_pval=c()
DT_enrich=c()
```

only APA

```{r}
x=nrow(DTandAPA %>% filter(OnlyAPA=="Yes", dTE=="Yes"))
m=nrow(DTandAPA %>% filter(dTE=="Yes"))
n=nrow(DTandAPA %>% filter(dTE=="No"))
k=nrow(DTandAPA %>% filter(OnlyAPA=="Yes"))
N=nrow(DTandAPA)
phyper(x-1,m,n,k,lower.tail=F)
DT_pval=c(DT_pval, phyper(x-1,m,n,k,lower.tail=F))
x
DT_enrich=c(DT_enrich, (x/k)/(m/N))
(x/k)/(m/N)

```
Only dIC

```{r}
x=nrow(DTandAPA %>% filter(OnlyIC=="Yes", dTE=="Yes"))
m=nrow(DTandAPA %>% filter(dTE=="Yes"))
n=nrow(DTandAPA %>% filter(dTE=="No"))
k=nrow(DTandAPA %>% filter(OnlyIC=="Yes"))
N=nrow(DTandAPA)
phyper(x-1,m,n,k,lower.tail=F)

DT_pval=c(DT_pval, phyper(x-1,m,n,k,lower.tail=F))
x
DT_enrich=c(DT_enrich, (x/k)/(m/N))
(x/k)/(m/N)
```


```{r}
x=nrow(DTandAPA %>% filter(Both=="Yes", dTE=="Yes"))
m=nrow(DTandAPA %>% filter(dTE=="Yes"))
n=nrow(DTandAPA %>% filter(dTE=="No"))
k=nrow(DTandAPA %>% filter(Both=="Yes"))
N=nrow(DTandAPA)
phyper(x-1,m,n,k,lower.tail=F)

DT_pval=c(DT_pval, phyper(x-1,m,n,k,lower.tail=F))
x
DT_enrich=c(DT_enrich, (x/k)/(m/N))
(x/k)/(m/N)

```


```{r}
DTdf=as.data.frame(cbind(sets,DT_pval, DT_enrich)) %>% rename(Pval=DT_pval, Enrichment=DT_enrich) %>% mutate(Pheno="Translation")
DTdf
```

##Protein  
 
(pval is adjusted already)
```{r}
Prot= read.table("../data/Khan_prot/ProtData_effectSize.txt",header = T,stringsAsFactors = F)  %>% mutate(dP=ifelse(pval<0.05, "Yes", "No"))


ProtSmall=Prot %>% select(gene, dP)
```


```{r}
DPandAPA=Prot %>% inner_join(dAPAandDic,by="gene")
nrow(DPandAPA)
```


```{r}
#sets=c("OnlyAPA", "OnlyIC", "Both")
DP_pval=c()
DP_enrich=c()
```
only APA

```{r}
x=nrow(DPandAPA %>% filter(OnlyAPA=="Yes", dP=="Yes"))
m=nrow(DPandAPA %>% filter(dP=="Yes"))
n=nrow(DPandAPA %>% filter(dP=="No"))
k=nrow(DPandAPA %>% filter(OnlyAPA=="Yes"))
N=nrow(DPandAPA)
phyper(x-1,m,n,k,lower.tail=F)
DP_pval=c(DP_pval, phyper(x-1,m,n,k,lower.tail=F))
x
DP_enrich=c(DP_enrich, (x/k)/(m/N))
(x/k)/(m/N)

```
Only dIC

```{r}
x=nrow(DPandAPA %>% filter(OnlyIC=="Yes", dP=="Yes"))
m=nrow(DPandAPA %>% filter(dP=="Yes"))
n=nrow(DPandAPA %>% filter(dP=="No"))
k=nrow(DPandAPA %>% filter(OnlyIC=="Yes"))
N=nrow(DPandAPA)
phyper(x-1,m,n,k,lower.tail=F)

DP_pval=c(DP_pval, phyper(x-1,m,n,k,lower.tail=F))
x
DP_enrich=c(DP_enrich, (x/k)/(m/N))
(x/k)/(m/N)
```


```{r}
x=nrow(DPandAPA %>% filter(Both=="Yes", dP=="Yes"))
m=nrow(DPandAPA %>% filter(dP=="Yes"))
n=nrow(DPandAPA %>% filter(dP=="No"))
k=nrow(DPandAPA %>% filter(Both=="Yes"))
N=nrow(DPandAPA)
phyper(x-1,m,n,k,lower.tail=F)

DP_pval=c(DP_pval, phyper(x-1,m,n,k,lower.tail=F))
x
DP_enrich=c(DP_enrich, (x/k)/(m/N))
(x/k)/(m/N)

```



```{r}
DPdf=as.data.frame(cbind(sets,DP_pval, DP_enrich)) %>% rename(Pval=DP_pval, Enrichment=DP_enrich) %>% mutate(Pheno="Protein")
DPdf
```


##Plot together:  

```{r}
AllDF= DEdf %>% bind_rows(DTdf) %>% bind_rows(DPdf)

AllDF$Pval=as.numeric(AllDF$Pval)
AllDF$Enrichment=as.numeric(AllDF$Enrichment)
AllDF$Pheno=factor(AllDF$Pheno, levels=c("Expression", "Translation", "Protein"))
useCOl <- c("#d73027", "#4575b4","#fee090")

enrichplot=ggplot(AllDF,aes(x=Pheno, by=sets, y=Enrichment,fill=sets)) +geom_bar(stat = "identity",position = "dodge") +geom_hline(yintercept =1) + scale_fill_manual(values=useCOl)

enrichplot

enrichpoint=ggplot(AllDF,aes(x=sets,col=sets,y=Enrichment,label = round(Enrichment,3)))+ geom_bar(stat="identity",color="grey",aes(y=AllDF$Enrichment),width=.01)+geom_point(size=10) + coord_flip() + geom_hline(yintercept = 1) + facet_grid(~Pheno)+scale_color_manual(values=useCOl)+ labs( title="Enrichment for APA phenotype differences in other regulatory phenotypes",x="Set", y="Enrichment")+geom_text(color = "black", size = 3) + theme(legend.position = "none")


enrichpoint


pvalplot=ggplot(AllDF,aes(x=Pheno, by=sets, y=-log10(Pval),fill=sets)) +geom_bar(stat = "identity",position = "dodge") +geom_hline(yintercept =1.3)+ scale_fill_manual(values=useCOl)+  theme(legend.position = "bottom")

pvalplot
```



plot together: 

```{r}
plot_grid(enrichpoint,pvalplot, nrow=2)
```

Plot without protien: 

```{r}
DETEDF= DEdf %>% bind_rows(DTdf)

DETEDF$Pval=as.numeric(DETEDF$Pval)
DETEDF$Enrichment=as.numeric(DETEDF$Enrichment)
DETEDF$Pheno=factor(DETEDF$Pheno, levels=c("Expression", "Translation", "Protein"))


enrichpointnoP=ggplot(DETEDF,aes(x=sets,col=sets,y=Enrichment,label = round(Enrichment,3)))+ geom_bar(stat="identity",color="grey",aes(y=DETEDF$Enrichment),width=.01)+geom_point(size=10) + coord_flip() + geom_hline(yintercept = 1) + facet_grid(~Pheno)+scale_color_manual(values=useCOl) + labs( title="Enrichment for APA phenotype \nin other regulatory phenotypes",x="", y="Enrichment")+geom_text(color = "black", size = 3) + theme(legend.position = "none")+scale_x_discrete(labels=c(Both="Both", OnlyAPA="PAS Level",OnlyIC= "Isoform Diversity"))


enrichpointnoP


pvalplotnoP=ggplot(DETEDF,aes(x=Pheno, by=sets, y=-log10(Pval),fill=sets)) +geom_bar(stat = "identity",position = "dodge") +geom_hline(yintercept =1.3)+ scale_fill_manual(values=useCOl,labels=c("Both", "PAS Level", "Isoform Diversity"), name="")+  theme(legend.position = "bottom")


pvalplotnoP
```

```{r}
exandte=plot_grid(enrichpointnoP,pvalplotnoP, nrow=2)
exandte
```
```{r,include=FALSE, eval=F}
pdf("../output/exandte", height=6, width=10)
exandte
dev.off()
```

```{r exandte, include=FALSE, dev="pdf", fig.height=6, fig.width=10, crop=FALSE}
exandte
```

Write out data for figures  
```{r}
write.table(DETEDF, "../output/FigureDF/DEandTEenrich.txt",col.names = T,row.names = F, quote = F)
```


Examples:  

Only dIC
```{r}
dIConly=dAPAandDic %>% filter(OnlyIC=="Yes") 
dIConly_translation=dIConly  %>% inner_join(Ribo, by="gene") %>% filter(dTE =="Yes")
```


* CLECL1 chimp uses 2 more often human uses 1 most often  

* GRHPR- human intronic just enough to change the utr ratio 

* hadha- human proximal, chimp 2 UTR  

* IVNS1ABP- chimp 1, human more 

* OGFOD3 - chimp more PAS used (good igv example)  

* ZNF512B human more spread 

```{r}
dIC_both= dAPAandDic %>% filter(Both=="Yes")
```


##Selection sets from Khan  

```{r}
KhanData=read.csv("../data/Khan_prot/Khan_TableS4.csv",stringsAsFactors = F)  %>% select(gene.symbol,contains("model") ) %>% rename("gene"=gene.symbol, "Protein"=model.num.protein, "RNA"=model.num.rna)

KhanData_g=KhanData %>% gather("Set", "Model", -gene)

KhanData_g$Model= as.factor(KhanData_g$Model)

KhanData_g_RNA= KhanData_g %>% filter(Set=="RNA")


KhanData_g_Prot= KhanData_g %>% filter(Set=="Protein")
```
Join with all of the tested gene.  

```{r}
KhanWithapa=dAPAandDic %>% inner_join(KhanData_g_RNA, by="gene")
```


Test only APA:  

```{r}
Model=seq(1,6)
EnrichmentRNA_apaOnly=c()
PvalueRNA_apaOnly=c()
for (i in seq(1:6)){
  x=nrow(KhanWithapa %>% filter(OnlyAPA=="Yes", Model==i))
  m=nrow(KhanWithapa %>% filter(Model==i))
  n=nrow(KhanWithapa %>% filter(Model!=i))
  k=nrow(KhanWithapa %>% filter(OnlyAPA=="Yes"))
  N=nrow(KhanWithapa)
  PvalueRNA_apaOnly=c(PvalueRNA_apaOnly, phyper(x-1,m,n,k,lower.tail=F))
  enrich=(x/k)/(m/N)
  EnrichmentRNA_apaOnly=c(EnrichmentRNA_apaOnly, enrich)
}

EnrichmentRNA_apaOnly
PvalueRNA_apaOnly
```
```{r}
EnrichmentRNA_ICOnly=c()
PvalueRNA_ICOnly=c()
for (i in seq(1:6)){
  x=nrow(KhanWithapa %>% filter(OnlyIC=="Yes", Model==i))
  m=nrow(KhanWithapa %>% filter(Model==i))
  n=nrow(KhanWithapa %>% filter(Model!=i))
  k=nrow(KhanWithapa %>% filter(OnlyIC=="Yes"))
  N=nrow(KhanWithapa)
  PvalueRNA_ICOnly=c(PvalueRNA_ICOnly, phyper(x-1,m,n,k,lower.tail=F))
  enrich=(x/k)/(m/N)
  EnrichmentRNA_ICOnly=c(EnrichmentRNA_ICOnly, enrich)
}

EnrichmentRNA_ICOnly
PvalueRNA_ICOnly
```
Both  

```{r}
actualRNAboth=c()
EnrichmentRNA_both=c()
expected_RNA_both=c()
PvalueRNA_both=c()
for (i in seq(1:6)){
  x=nrow(KhanWithapa %>% filter(Both=="Yes", Model==i))
  m=nrow(KhanWithapa %>% filter(Model==i))
  n=nrow(KhanWithapa %>% filter(Model!=i))
  k=nrow(KhanWithapa %>% filter(Both=="Yes"))
  N=nrow(KhanWithapa)
  actualRNAboth=c(actualRNAboth,x)
  PvalueRNA_both=c(PvalueRNA_both, phyper(x-1,m,n,k,lower.tail=F))
  expec=k*(m/N)
  expected_RNA_both=c(expected_RNA_both, expec)
  enrich=(x/k)/(m/N)
  EnrichmentRNA_both=c(EnrichmentRNA_both, enrich)
}
actualRNAboth
expected_RNA_both
EnrichmentRNA_both
PvalueRNA_both

```

Try enrichment for 18 + 22:  

```{r}
  x=nrow(KhanWithapa %>% filter(Both=="Yes", Model %in% c(1,2)))
  m=nrow(KhanWithapa %>% filter(Model %in% c(1,2)))
  n=nrow(KhanWithapa %>% filter(Model %in% c(3,4,5,6)))
  k=nrow(KhanWithapa %>% filter(Both=="Yes"))
  N=nrow(KhanWithapa)
  x
  pval=phyper(x-1,m,n,k,lower.tail=F)
  pval
  expec=k*(m/N)
  expec
  enrich=(x/k)/(m/N)
  enrich
```


Prot:  

```{r}
KhanPWithapa=dAPAandDic %>% inner_join(KhanData_g_Prot, by="gene")

```

Only APA  

```{r}

EnrichmentProt_apaOnly=c()
PvalueProp_apaOnly=c()
for (i in seq(1:6)){
  x=nrow(KhanPWithapa %>% filter(OnlyAPA=="Yes", Model==i))
  m=nrow(KhanPWithapa %>% filter(Model==i))
  n=nrow(KhanPWithapa %>% filter(Model!=i))
  k=nrow(KhanPWithapa %>% filter(OnlyAPA=="Yes"))
  N=nrow(KhanPWithapa)
  PvalueProp_apaOnly=c(PvalueProp_apaOnly, phyper(x-1,m,n,k,lower.tail=F))
  enrich=(x/k)/(m/N)
  EnrichmentProt_apaOnly=c(EnrichmentProt_apaOnly, enrich)
}

EnrichmentProt_apaOnly
PvalueProp_apaOnly
```
```{r}
EnrichmentProt_ICOnly=c()
PvalueProt_ICOnly=c()
for (i in seq(1:6)){
  x=nrow(KhanPWithapa %>% filter(OnlyIC=="Yes", Model==i))
  m=nrow(KhanPWithapa %>% filter(Model==i))
  n=nrow(KhanPWithapa %>% filter(Model!=i))
  k=nrow(KhanPWithapa %>% filter(OnlyIC=="Yes"))
  N=nrow(KhanPWithapa)
  PvalueProt_ICOnly=c(PvalueProt_ICOnly, phyper(x-1,m,n,k,lower.tail=F))
  enrich=(x/k)/(m/N)
  EnrichmentProt_ICOnly=c(EnrichmentProt_ICOnly, enrich)
}

EnrichmentProt_ICOnly
PvalueProt_ICOnly
```

Both:  

```{r}
EnrichmentProt_both=c()
PvalueProt_both=c()
for (i in seq(1:6)){
  x=nrow(KhanPWithapa %>% filter(Both=="Yes", Model==i))
  m=nrow(KhanPWithapa %>% filter(Model==i))
  n=nrow(KhanPWithapa %>% filter(Model!=i))
  k=nrow(KhanPWithapa %>% filter(Both=="Yes"))
  N=nrow(KhanPWithapa)
  PvalueProt_both=c(PvalueProt_both, phyper(x-1,m,n,k,lower.tail=F))
  enrich=(x/k)/(m/N)
  EnrichmentProt_both=c(EnrichmentProt_both, enrich)
}

EnrichmentProt_both
PvalueProt_both
```


Significant: 

Both RNA:  2 (1.6X, 0.01547134)

mRNA expression level pattern consistent with directional selection along chimpanzee lineage


```{r}
nrow(KhanWithapa %>% filter(Both=="Yes", Model==2))
```

 6 = evidence of relaxation of constraint along chimpanzee lineage
 
```{r}
nrow(KhanPWithapa %>% filter(Both=="Yes", Model==6))
```
 

Both Protien:  6 (3.87X, 0.038)

##location enrichment:  

For the dAPA only and both. where are the PAS contibuting to the relationship. are they enriched among dAPA PAS 

want those that are DE genes 

```{r}
dAPAPAS_wLoc=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T, stringsAsFactors = F) %>% inner_join(Meta, by=c("chr","start", "end","gene")) %>% select(PAS, loc, SigPAU2, gene)

DiffExpSig= DiffExp %>% filter(DE=='Yes')
BothAPAgenes= dAPAandDic %>% filter(Both=="Yes") %>% inner_join(DiffExpSig, by="gene")
OnlyAPAgenes=dAPAandDic %>% filter(OnlyAPA=="Yes")%>% inner_join(DiffExpSig, by="gene")

dAPAPAS_wLocAndBoth= dAPAPAS_wLoc %>% filter(SigPAU2=="Yes") %>% mutate(dAPAonly=ifelse(gene %in%OnlyAPAgenes$gene,"Yes","No"), Both=ifelse(gene %in% BothAPAgenes$gene, "Yes", "No"))


```


```{r}
enrichdAPAloc=c()
pvaldAPAloc=c()
expectddAPA=c()
pvaldAPAlocDep=c()
actualdAPA=c()
for (i in c("cds", "end", "intron", "utr3", "utr5")){
  x=nrow(dAPAPAS_wLocAndBoth %>% filter(dAPAonly=="Yes", loc==i))
  m=nrow(dAPAPAS_wLocAndBoth %>% filter(loc==i))
  n=nrow(dAPAPAS_wLocAndBoth %>% filter(loc!=i))
  k=nrow(dAPAPAS_wLocAndBoth %>% filter(dAPAonly=="Yes"))
  N=nrow(dAPAPAS_wLocAndBoth)
  actualdAPA=c(actualdAPA, x)
  exp=k*(m/N)
  expectddAPA=c(expectddAPA,exp)
  pvaldAPAloc=c(pvaldAPAloc, phyper(x-1,m,n,k,lower.tail=F))
  pvaldAPAlocDep=c(pvaldAPAlocDep, phyper(x,m,n,k,lower.tail=T))
  enrichval=(x/k)/(m/N)
  enrichdAPAloc=c(enrichdAPAloc, enrichval)
}

loc=c("cds", "end", "intron", "utr3", "utr5")
enrichdAPAloc
pvaldAPAloc

enrichdapalocdf=as.data.frame(cbind(loc, Actual=actualdAPA,Expected=expectddAPA, Enrichment=enrichdAPAloc,PvalEn=pvaldAPAloc, PvalDep=pvaldAPAlocDep)) %>% mutate(set="dAPAOnly")
enrichdapalocdf

```

```{r}
enrichBothloc=c()
pvalBothloc=c()
pvalBothDepletloc=c()
expectBoth=c()
actualBoth=c()
for (i in c("cds", "end", "intron", "utr3", "utr5")){
  x=nrow(dAPAPAS_wLocAndBoth %>% filter(Both=="Yes", loc==i))
  m=nrow(dAPAPAS_wLocAndBoth %>% filter(loc==i))
  n=nrow(dAPAPAS_wLocAndBoth %>% filter(loc!=i))
  k=nrow(dAPAPAS_wLocAndBoth %>% filter(Both=="Yes"))
  N=nrow(dAPAPAS_wLocAndBoth)
  actualBoth=c(actualBoth, x)
  exp=k*(m/N)
  expectBoth=c(expectBoth,exp)
  pvalBothDepletloc=c(pvalBothDepletloc, phyper(x,m,n,k,lower.tail=T))
  pvalBothloc=c(pvalBothloc, phyper(x-1,m,n,k,lower.tail=F))
  enrichval=(x/k)/(m/N)
  enrichBothloc=c(enrichBothloc, enrichval)
}

loc=c("cds", "end", "intron", "utr3", "utr5")
enrichBothloc
pvalBothloc



enrichbothlocdf=as.data.frame(cbind(loc,Actual=actualBoth,Expected=expectBoth, Enrichment=enrichBothloc,PvalEn=pvalBothloc,PvalDep=pvalBothDepletloc))%>% mutate(set="Both")
enrichbothlocdf

```


Join both and plot: 
```{r}
deLocALl=enrichdapalocdf %>% bind_rows(enrichbothlocdf)
deLocALl
```

```{r}
#ggplot(ResDFfix_both,aes(x=set,group=type,col=set,y=Enrich))+ geom_bar(stat="identity",col="grey",alpha=.3,width=.01)+geom_point(size=10) + coord_flip()+ geom_hline(yintercept = 1) +scale_color_brewer(palette="RdYlBu")+geom_text(col="black",aes(label = round(Enrich,2)))+ facet_grid(~type)+labs(x="Dominance Cutoff", y="Enrichment",title="Enrichment for DE by Domianance") + theme(legend.position = "none")
deLocALl$Enrichment=as.numeric(deLocALl$Enrichment)
ggplot(deLocALl, aes(x=loc, col=loc, y=Enrichment)) + geom_bar(stat="identity",col="grey",alpha=.3,width=.01)+geom_point(size=10) + coord_flip()+ geom_hline(yintercept = 1) +scale_color_brewer(palette="RdYlBu")+geom_text(col="black",aes(label = round(Enrichment,2)))+ facet_grid(~set)
```

Check any dAPA:  

```{r}

dAPAPAS_wLocAndBoth_either =dAPAPAS_wLocAndBoth %>% mutate(anydAPA=ifelse(Both=="Yes" | dAPAonly=="Yes", "Yes","No"))
enrichEither=c()
pvalEither=c()
pvalEitherDep=c()
expectEither=c()
actualEither=c()
for (i in c("cds", "end", "intron", "utr3", "utr5")){
  x=nrow(dAPAPAS_wLocAndBoth_either %>% filter(anydAPA=="Yes", loc==i))
  m=nrow(dAPAPAS_wLocAndBoth_either %>% filter(loc==i))
  n=nrow(dAPAPAS_wLocAndBoth_either %>% filter(loc!=i))
  k=nrow(dAPAPAS_wLocAndBoth_either %>% filter(anydAPA=="Yes"))
  N=nrow(dAPAPAS_wLocAndBoth_either)
  actualEither=c(actualEither, x)
  exp=k*(m/N)
  expectEither=c(expectEither,exp)
  pvalEitherDep=c(pvalEitherDep, phyper(x,m,n,k,lower.tail=T))
  pvalEither=c(pvalEither, phyper(x-1,m,n,k,lower.tail=F))
  enrichval=(x/k)/(m/N)
  enrichEither=c(enrichEither, enrichval)
}

loc=c("cds", "end", "intron", "utr3", "utr5")


enrichEitherDf=as.data.frame(cbind(loc,Actual=actualEither,Expected=expectEither, Enrichment=enrichEither,PvalEn=pvalEither,PvalDep=pvalEitherDep))%>% mutate(set="Both", Location=c("Coding", "End", "intronic","3'UTR", "5'UTR"))
enrichEitherDf
```

Plot:  

```{r}
enrichEitherDf$Enrichment=as.numeric(as.character(enrichEitherDf$Enrichment))
ggplot(enrichEitherDf, aes(x=Location, col=Location, y=Enrichment)) + geom_bar(stat="identity",col="grey",alpha=.3,width=.01)+geom_point(size=10) + coord_flip()+ geom_hline(yintercept = 1) +scale_color_brewer(palette="RdYlBu")+geom_text(col="black",aes(label = round(Enrichment,2))) + theme(legend.position = "none") +labs(title="Genic location enrichment for dAPA PAS in DE genes")


```

Barplot:  

```{r}
ggplot(enrichEitherDf, aes(x=Location, fill=Location, y=Enrichment)) + geom_bar(stat="identity") +scale_fill_brewer(palette="RdYlBu") + geom_hline(yintercept = 1)+geom_text(col="black",aes(label = round(Enrichment,2), vjust=2)) + theme(legend.position = "none") +labs(title="Genic location enrichment for dAPA PAS in DE genes")
```


Either dIC and TE:  


```{r}
RiboSmallSig= RiboSmall %>% filter(dTE=='Yes')
BothgeneTE= dAPAandDic %>% filter(Both=="Yes") %>% inner_join(RiboSmallSig, by="gene")
OnlyICgenes=dAPAandDic %>% filter(OnlyIC=="Yes")%>% inner_join(RiboSmallSig, by="gene")

dAPAPAS_wLocAndBothTE= dAPAPAS_wLoc %>% filter(SigPAU2=="Yes") %>% mutate(dIConly=ifelse(gene %in% OnlyICgenes$gene,"Yes","No"), Both=ifelse(gene %in% BothgeneTE$gene, "Yes", "No"))%>% mutate(anydIC=ifelse(Both=="Yes" | dIConly=="Yes", "Yes","No"))
```

```{r}
enrichEitherTE=c()
pvalEitherTE=c()
pvalEitherDepTE=c()
expectEitherTE=c()
actualEitherTE=c()
for (i in c("cds", "end", "intron", "utr3", "utr5")){
  x=nrow(dAPAPAS_wLocAndBothTE %>% filter(anydIC=="Yes", loc==i))
  m=nrow(dAPAPAS_wLocAndBothTE %>% filter(loc==i))
  n=nrow(dAPAPAS_wLocAndBothTE %>% filter(loc!=i))
  k=nrow(dAPAPAS_wLocAndBothTE %>% filter(anydIC=="Yes"))
  N=nrow(dAPAPAS_wLocAndBothTE)
  actualEitherTE=c(actualEitherTE, x)
  exp=k*(m/N)
  expectEitherTE=c(expectEitherTE,exp)
  pvalEitherDepTE=c(pvalEitherDepTE, phyper(x,m,n,k,lower.tail=T))
  pvalEitherTE=c(pvalEitherTE, phyper(x-1,m,n,k,lower.tail=F))
  enrichval=(x/k)/(m/N)
  enrichEitherTE=c(enrichEitherTE, enrichval)
}

loc=c("cds", "end", "intron", "utr3", "utr5")


enrichEitherDfTE=as.data.frame(cbind(loc,Actual=actualEitherTE,Expected=expectEitherTE, Enrichment=enrichEitherTE,PvalEn=pvalEitherTE,PvalDep=pvalEitherDepTE))%>% mutate(set="Both", Location=c("Coding", "End", "intronic","3'UTR", "5'UTR"))
enrichEitherDfTE
```

```{r}

enrichEitherDfTE$Enrichment=as.numeric(as.character(enrichEitherDfTE$Enrichment))
ggplot(enrichEitherDfTE, aes(x=Location, fill=Location, y=Enrichment)) + geom_bar(stat="identity") +scale_fill_brewer(palette="RdYlBu") + geom_hline(yintercept = 1)+geom_text(col="black",aes(label = round(Enrichment,2), vjust=2)) + theme(legend.position = "none") +labs(title="Genic location enrichment for dAPA PAS in dT genes")

```

