---
title: "5' Spice Site strength"
author: "Briana Mittleman"
date: "2/21/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


```{r}
library(workflowr)
library(tidyverse)
```

There is a hypothesis that increased 5' splice site strength is assocaited with decreased usage of intronic PAS. This is relate to competition and binding of the U1 snurp. I will ask if there are differences in 5' splcie sites for humans and chimp. 

Need to be careful about orthology here. To be conservative. I will only look at regions that map downstream of an ortho exon. 

First step is to map each intronic PAS to a human intron annotation.



I created a transcript minus exon file for my previos project. I will lift this file over andcheck it. I can remake it 

```{bash,eval=F}
mkdir ../data/SpliceSite  

liftOver /project2/gilad/briana/apaQTL/data/intron_analysis/transcriptsMinusExons.sort.bed  ../data/liftover_files/hg19ToHg38.over.chain.gz ../data/SpliceSite/transcriptMinusexon_hg38.bed ../data/SpliceSite/UnliftedIntron.bed 

```

These look really good, they line up well.  


Pull out intronic PAS  
```{r}
PAS_metaIntron=read.table("../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", header = T, stringsAsFactors = F) %>% filter(loc=="intron")
PAS=read.table("../data/PAS_doubleFilter/PAS_doublefilter_either_HumanCoordHummanUsage.bed", col.names = c("chr", "start", "end", "PAS", "score", "strand"),stringsAsFactors = F) %>% semi_join(PAS_metaIntron, by="PAS")

write.table(PAS, "../data/SpliceSite/IntronicPAS_humanCoord.bed", col.names = F, row.names = F, quote = F, sep="\t")
```


```{bash,eval=F}
sbatch assignPeak2Intronicregion.sh
```

Get the 5' splice sites for all of these.  

(lose ~800)



```{r}
PAS2Intron=read.table("../data/SpliceSite/IntronincPAS2Introns_humanCoord.bed",col.names = c("IntronChr", "IntronStart", "IntronEnd", "Gene", "Score", "Strand", "PASChr", "PASStart","PASEnd", "PAS", "humanUsage", "passtrand"),stringsAsFactors = F)

Lost= PAS %>% anti_join(PAS2Intron, by="PAS")

write.table(Lost, "../data/SpliceSite/LostinIntersect.bed", col.names = F, row.names = F, quote =F, sep = "\t")

```



Lose some with multiple isoforms. Downstream of a gene may be an intron in one. It is probably not possible to get perfect annotation.  


```{r}
PAS2Intron_pos= PAS2Intron %>% filter(Strand=="+") %>% mutate(start=IntronStart-3, end= IntronStart +6) %>% select(IntronChr, start,end, PAS,humanUsage, Strand)
PAS2Intron_neg=PAS2Intron %>% filter(Strand=="-") %>% mutate(start=IntronEnd-6, end= IntronEnd +3) %>% select(IntronChr, start,end, PAS,humanUsage, Strand)
PAS_5SS_both= PAS2Intron_neg %>% bind_rows(PAS2Intron_pos)

write.table(PAS_5SS_both, "../data/SpliceSite/IntronicPAS_SS_humanCoord.bed", col.names = F, row.names = F, quote=F, sep="\t")
```

Sort and assign to ortho exon. I need a small amount of overlap with the human ortho exon file. This comes from Kenneth's work.  /project2/gilad/kenneth/OrthoExonPartialMapping/human.noM.gtf

Ortho exon needs to be converted to bed to intersect.  

```{bash,eval=F}
sort -k1,1 -k2,2n ../data/SpliceSite/IntronicPAS_SS_humanCoord.bed > ../data/SpliceSite/IntronicPAS_SS_humanCoord.sort.bed 


bedtools intersect -a ../data/SpliceSite/IntronicPAS_SS_humanCoord.sort.bed  -b /project2/gilad/kenneth/OrthoExonPartialMapping/human.noM.gtf -s -wao > ../data/SpliceSite/IntronicPAS_SS_intersectOrthoExon.txt

#looking for 3 base overlap with splice sites  

```

```{r}
IntersectRes=read.table("../data/SpliceSite/IntronicPAS_SS_intersectOrthoExon.txt",stringsAsFactors = F,sep="\t", col.names = c("chr",'ssstart','ssend','PAS', 'humanusage','passtrand', 'file', 'loc','exonchr', 'enonstart','exonend', 'score', 'strand', 'score2', 'geneinfo', 'overlap')) %>% filter(overlap==3)


IntersectRes_group= IntersectRes %>% group_by(PAS) %>% summarise(nExon=n())
nrow(IntersectRes_group)
```




Filter :  


```{r}
PAS_5SS_both_filt= PAS_5SS_both %>% semi_join(IntersectRes_group, by="PAS") 

PAS_5SS_both_filt %>% group_by(PAS) %>% summarise(n=n()) %>% filter(n>1)

PAS_5SS_both_filt %>% group_by(PAS) %>% summarise(n=n()) %>% filter(n>1) %>% nrow()
```
228 map to 2 introns. Count each site for now.  


Write these out to sort and liftover.
```{r}
write.table(PAS_5SS_both_filt, "../data/SpliceSite/IntronicPAS_SS_humanCoord_filterOotho.bed", col.names = F, row.names = F, quote = F, sep="\t")
```

```{bash,eval=F}
sort -k1,1 -k2,2n ../data/SpliceSite/IntronicPAS_SS_humanCoord_filterOotho.bed > ../data/SpliceSite/IntronicPAS_SS_humanCoord_filterOotho.sort.bed


liftOver ../data/SpliceSite/IntronicPAS_SS_humanCoord_filterOotho.sort.bed ../data/chainFiles/hg38ToPanTro6.over.chain ../data/SpliceSite/IntronicPAS_SS_ChimpCoord_filterOotho.sort.bed ../data/SpliceSite/ChimpCoordUnliftedSS.txt 
```


Remove unlifted from human

```{r}
unliftedSS=read.table("../data/SpliceSite/ChimpCoordUnliftedSS.txt",col.names = c("chr", 'start','end','PAS', 'humanscore', 'strand'), stringsAsFactors = F)
#check still 9 bases
liftedSS=read.table("../data/SpliceSite/IntronicPAS_SS_ChimpCoord_filterOotho.sort.bed",col.names = c("chr", 'start','end','PAS', 'humanscore', 'strand'), stringsAsFactors = F) %>% mutate(legnth=end-start) 

liftedSS_wrongsize= liftedSS %>%  filter(legnth!=9)


nrow(liftedSS_wrongsize)
nrow(unliftedSS)

BADSS= as.data.frame(cbind(PAS=c(liftedSS_wrongsize$PAS,unliftedSS$PAS)))
```
Remove the 36 that to not lift or lift to the wrong size. 

```{r}
ChimpSS=liftedSS %>% filter(legnth==9) %>% select(-legnth)
nrow(ChimpSS)

HumanSS=PAS_5SS_both_filt %>% anti_join(BADSS, by="PAS")
nrow(HumanSS)
```
  

Next step is to use bedtools nuc to get the strand specific basepairs.  


```{r}
write.table(ChimpSS, "../data/SpliceSite/Chimp_SS.bed", col.names = F, row.names = F, quote = F, sep="\t")

write.table(HumanSS, "../data/SpliceSite/Human_SS.bed", col.names = F, row.names = F, quote = F, sep="\t")

```


```{bash,eval=F}
sort -k1,1 -k2,2n ../data/SpliceSite/Chimp_SS.bed > ../data/SpliceSite/Chimp_SS_sort.bed

sort -k1,1 -k2,2n ../data/SpliceSite/Human_SS.bed > ../data/SpliceSite/Human_SS.sort.bed

#bedtools nuc -fi /project2/gilad/briana/genome_anotation_data/genome/Homo_sapiens.GRCh37.75.dna_sm.all.fa -bed ../data/splicesite/AllPASSS.sort.noChr.bed -seq -s > ../data/splicesite/AllPASSS.sort.Nuc.txt

bedtools nuc -fi /project2/gilad/briana/genome_anotation_data/Chimp_genome/panTro6.fa -bed ../data/SpliceSite/Chimp_SS_sort.bed -seq -s > ../data/SpliceSite/Chimp_SS_sort.Nuc.txt

bedtools nuc -fi /project2/gilad/kenneth/References/human/genome/hg38.fa -bed ../data/SpliceSite/Human_SS.sort.bed -seq -s > ../data/SpliceSite/Human_SS.sort.Nuc.txt

#parse
python spliceSite2Fasta.py ../data/SpliceSite/Chimp_SS_sort.Nuc.txt ../data/SpliceSite/Chimp_SS_sort.Nuc.fasta

python spliceSite2Fasta.py ../data/SpliceSite/Human_SS.sort.Nuc.txt ../data/SpliceSite/Human_SS_sort.Nuc.fasta

#run ss maxent  
cd /MaxEntCode/fordownload
perl score5.pl ../../../data/SpliceSite/Chimp_SS_sort.Nuc.fasta >  ../../../data/SpliceSite/Chimp_SS_sort.Nuc.MaxentScores.txt
perl score5.pl  ../../../data/SpliceSite/Human_SS_sort.Nuc.fasta >  ../../../data/SpliceSite/Human_SS_sort.Nuc.MaxentScore.txt
```

```{r}
ChimpSS=read.table("../data/SpliceSite/Chimp_SS_sort.bed", col.names = c("chr",'start','end','PAS', 'HumanUsage', 'strand'), stringsAsFactors = F) %>% select(PAS,HumanUsage)
ChimpRES=read.table("../data/SpliceSite/Chimp_SS_sort.Nuc.MaxentScores.txt", col.names =c("Chimpseq", "ChimpScore"))
ChimpSSandRes=ChimpSS %>% bind_cols(ChimpRES)

HumanSS=read.table("../data/SpliceSite/Human_SS.sort.bed", col.names = c("chr",'start','end','PAS', 'HumanUsage', 'strand'), stringsAsFactors = F)%>% select(PAS,HumanUsage)
HumanRES=read.table("../data/SpliceSite/Human_SS_sort.Nuc.MaxentScore.txt", col.names = c("Humanseq", "HumanScore")) 
HumanSSandRes=HumanSS %>% bind_cols(HumanRES)


BothSSandRes= ChimpSSandRes %>% inner_join(HumanSSandRes, by=c('PAS','HumanUsage'))


```
Add mean chimp  

```{r}
ChimpPASUsage=read.table("../data/PAS_doubleFilter/PAS_doublefilter_either_ChimpCoordChimpUsage.sort.bed",col.names = c('chr','start','end',"PAS", 'ChimpUsage','strand' ),stringsAsFactors = F) %>% select(PAS, ChimpUsage)

BothSSandReswUsage=BothSSandRes %>% inner_join(ChimpPASUsage,by='PAS')
```

```{r}

ggplot(BothSSandReswUsage, aes(x=HumanScore, y=HumanUsage)) +geom_point(col="blue", alpha=.3) + geom_point(data=BothSSandReswUsage, aes(x=ChimpScore, y=ChimpUsage), alpha=.3,col="orange") 
```

```{r}
ggplot(BothSSandReswUsage,aes(x=ChimpScore, y=HumanScore)) + geom_point() + geom_density2d(col="green") + geom_smooth(method="lm",col="orange") + annotate("text",label="Pearsons Correlation = .98", x=-10, y=10) + labs(title="5' Splice Site Strength for Intronic PAS")+ theme(text= element_text(size=16))

cor.test(BothSSandReswUsage$ChimpScore,BothSSandReswUsage$HumanScore)

```

Plot this by if the PAS is signfiicantly different
```{r}
PASMeta=read.table("../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", header = T, stringsAsFactors = F) %>% dplyr::select(PAS, chr, start,end, gene, loc)

DiffUsagePAS=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T, stringsAsFactors = F) %>% inner_join(PASMeta, by=c("gene","chr", "start", "end"))  %>% select(PAS, SigPAU2)
BothSSandReswUsage_withSig=BothSSandReswUsage %>% inner_join(DiffUsagePAS, by="PAS")
```

```{r}
ggplot(BothSSandReswUsage_withSig,aes(x=ChimpScore, y=HumanScore, col=SigPAU2)) + geom_point(alpha=.5) + annotate("text",label="Pearsons Correlation = .98", x=-10, y=10) + labs(title="5' Splice Site Strength for Intronic PAS",col="Differentially \nused PAS")+ theme(text= element_text(size=16)) + scale_color_brewer(palette = "Dark2")

cor.test(BothSSandReswUsage$ChimpScore,BothSSandReswUsage$HumanScore)
```


```{r}
ggplot(BothSSandReswUsage_withSig,aes(x=ChimpUsage, y=HumanUsage,col=SigPAU2)) + geom_point(alpha=.5)  + geom_smooth(method="lm",col="orange") + annotate("text", label="Pearsons Correlation= 0.54", x=.65,y=.8) + theme(text= element_text(size=16)) + scale_color_brewer(palette = "Dark2") + labs(title="Usage of PAS by species",col="Differentially \nused PAS") 



cor.test(BothSSandReswUsage$ChimpUsage,BothSSandReswUsage$HumanUsage )
```

How many have different score: 

```{r}
BothSSandReswUsage_diff= BothSSandReswUsage_withSig %>% filter(ChimpScore!=HumanScore)

nrow(BothSSandReswUsage_diff)
```

229/8298 PAS have different scores in human and chimp  

I expect higher scores to have lower usage 

Plot difference in score and diff in usage  

```{r}
BothSSandReswUsage_diff_score= BothSSandReswUsage_diff %>% mutate(DiffScore=HumanScore-ChimpScore, DiffUsage=HumanUsage-ChimpUsage)

ggplot(BothSSandReswUsage_diff_score, aes(x=DiffScore, y=DiffUsage)) + geom_point(aes(col=SigPAU2)) + geom_smooth(method="lm") +theme(text= element_text(size=16)) + scale_color_brewer(palette = "Dark2") + labs(title="Relationship between differene in \nusage and difference in 5' SS score",col="Differentially \nused PAS") 

summary(lm(BothSSandReswUsage_diff_score$DiffScore ~ BothSSandReswUsage_diff_score$DiffUsage))

```

No correlation  

Are any of these the differentially used PAS.  

```{r}


Meta=read.table("../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", header = T, stringsAsFactors = F) %>% select(PAS,chr, loc, start, end)

DiffIsoRes=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T,stringsAsFactors = F) %>% inner_join(Meta, by=c("chr", 'start','end')) %>% select(PAS,SigPAU2 )
```




