---
title: "TripSeq annotation"
author: "Briana Mittleman"
date: "6/8/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


```{r}
library(tidyverse)
library(ggpubr)
```

THe trip seq paper has a method to explore 3' UTR characteristics from a bed file 


https://github.com/stephenfloor/tripseq-analysis
Test transcriptome_properties.py

I need the name of the genome and a bed file. I can start by testing on the ortho 3' UTRs  

clone code/tripseq-analysis
python 2
```{bash,eval=F}
source ~/activate_anaconda_python2.sh
module load bedtools
#in trip
python transcriptome_properties.py -i ../../data/orthoUTR/HumanDistal3UTR.sort.bed -g /project2/gilad/kenneth/References/human/genome/hg38.fa   --au-elements 

```

This gets the au rich element %. 


can do this with human and chimp:  
```{bash,eval=F}
python transcriptome_properties.py -i ../../data/orthoUTR/HumanDistal3UTR.sort.bed -g /project2/gilad/kenneth/References/human/genome/hg38.fa   --au-elements  -o ../../data/orthoUTR/HumanOrthoUTR_AUrich

python transcriptome_properties.py -i ../../data/orthoUTR/ChimpDistal3UTR.sort.bed -g /project2/gilad/briana/genome_anotation_data/Chimp_genome/panTro6.fa --au-elements  -o ../../data/orthoUTR/ChimpOrthoUTR_AUrich
```


```{r}
HumanOrthUTR_au=read.csv("../data/orthoUTR/HumanOrthoUTR_AUrich_au_elements.csv",header = T, stringsAsFactors = F) 
ChimpOrthoUTR_au=read.csv("../data/orthoUTR/ChimpOrthoUTR_AUrich_au_elements.csv",header = T, stringsAsFactors = F)
```

Corr: 

```{r}
HumanOrthUTR_au_sm=HumanOrthUTR_au %>% select(transcriptID, au_element_frac) %>% rename(HumanAU=au_element_frac)
ChimpOrthUTR_au_sm=ChimpOrthoUTR_au %>% select(transcriptID, au_element_frac) %>% rename(ChimpAU=au_element_frac)

BothAU=HumanOrthUTR_au_sm %>% inner_join(ChimpOrthUTR_au_sm, by="transcriptID")

cor.test(BothAU$ChimpAU, BothAU$HumanAU)

```
AU number:

```{r}
HumanOrthUTR_au_Numsm=HumanOrthUTR_au %>% select(transcriptID, au_element_count) %>% rename(HumanAU=au_element_count)
ChimpOrthUTR_au_Numsm=ChimpOrthoUTR_au %>% select(transcriptID, au_element_count) %>% rename(ChimpAU=au_element_count)

BothAUNum=HumanOrthUTR_au_Numsm %>% inner_join(ChimpOrthUTR_au_Numsm, by="transcriptID")

cor.test(BothAU$ChimpAU, BothAU$HumanAU)
```
same high correlation... 


```{r}
ggplot(BothAU, aes(x=HumanAU, y=ChimpAU)) + geom_point() + geom_abline(slope=1) + geom_density_2d()
```


Color by dAPA and diff iso diversity.

```{r}
Meta=read.table("../data/PAS_doubleFilter/PAS_5perc_either_HumanCoord_BothUsage_meta_doubleFilter.txt", header = T, stringsAsFactors = F) 
Meta_genes= Meta %>% select(gene) %>% unique()

Meta_PAS=Meta %>%  select(PAS,gene)

dAPAGenes=read.table("../data/DiffIso_Nuclear_DF/SignifianceEitherGENES_Nuclear.txt", header = T, stringsAsFactors = F)
dAPAPAS=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt", header = T, stringsAsFactors = F) %>% inner_join(Meta, by=c("chr","start", "end","gene")) %>% select(PAS,gene,SigPAU2 ) 

dAPAPAS_genes= dAPAPAS %>% select(gene) %>% unique()

dAPATestedGenes= dAPAPAS  %>% select(gene) %>% unique() %>% mutate(dAPA=ifelse(gene %in% dAPAGenes$gene,"Yes", "No")) 

dICdata= read.table("../data/IndInfoContent/SimpsonMedianSignificance.txt", header = T, stringsAsFactors = F)%>% select(sIC,gene)
dICdata_sig= dICdata %>% filter(sIC=="Yes")

dAPAandDic= dICdata %>% inner_join(dAPATestedGenes,by="gene") %>% mutate(Both=ifelse(sIC=="Yes" & dAPA=="Yes", "Yes","No"),OnlyIC=ifelse(sIC=="Yes" & dAPA=="No", "Yes","No"),OnlyAPA=ifelse(sIC=="No" & dAPA=="Yes", "Yes","No"))

dAPAonly=dAPAandDic %>% filter(dAPA=="Yes") %>% select(gene) %>% mutate(set="dAPA")
both=dAPAandDic %>% filter(Both=="Yes") %>% select(gene)%>% mutate(set="Both")
IDonly=dAPAandDic %>% filter(OnlyIC=="Yes") %>% select(gene)%>% mutate(set="ID")



Allset=dAPAonly %>% bind_rows(both) %>% bind_rows(IDonly)
Nonegenes= Meta_PAS %>% select(gene) %>% unique()%>% anti_join(Allset, by="gene") %>% mutate(set="None")
#no diff:  
Allset_andnon=Allset %>% bind_rows(Nonegenes)

```


```{r}


BothAU_fix= BothAU %>% separate(transcriptID, into=c("gene", "strand"), sep="\\(") %>% inner_join(Allset_andnon, by="gene") %>% mutate(anyDiff=ifelse(set=="None", "No", "Yes"))

BothAU_fix %>% group_by(set) %>% summarise(n())
```

Do dAPA genes have higher AU 
```{r}

BothAU_fix_g= BothAU_fix %>% gather("Species", "AU", -gene, -strand, -set,-anyDiff)


ggplot(BothAU_fix_g, aes(x=AU,col=set)) +stat_ecdf() + facet_grid(~Species)


```
 
 
box plots: 

```{r}
ggplot(BothAU_fix_g, aes(y=AU,x=set,fill=set)) +geom_boxplot() + facet_grid(~Species) + stat_compare_means()

ggplot(BothAU_fix_g, aes(y=AU,x=anyDiff,fill=anyDiff)) +geom_boxplot() + facet_grid(~Species) + stat_compare_means(method.args = list(alternative = "greater"))

```
```{r}
BothAU_fix_g %>% group_by(Species,anyDiff) %>% summarise(mean(AU))
```
```{r}
BothAU_fix_g %>% group_by(Species,set) %>% summarise(mean(AU))
```


dAPA are the higher genes:  

genes with expression diff: 

```{r}
nameID=read.table("../../genome_anotation_data/ensemble_to_genename.txt",sep="\t", header = T, stringsAsFactors = F) %>% dplyr::select(Gene_stable_ID, Gene.name)
DiffExp=read.table("../data/DiffExpression/DEtested_allres.txt",stringsAsFactors = F,header = F, col.names = c("Gene_stable_ID" ,"logFC" ,"AveExpr" , "t" ,  "P.Value" ,  "adj.P.Val", "B"  )) %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::rename('gene'=Gene.name) %>% dplyr::select(-Gene_stable_ID) %>% mutate(DE=ifelse(adj.P.Val<.05, "Yes", "No")) %>% select(gene,DE)

BothAU_fix_g_DE= BothAU_fix_g %>% inner_join(DiffExp,by="gene")

```

```{r}
ggplot(BothAU_fix_g_DE, aes(y=AU,x=DE,fill=DE)) +geom_boxplot() + facet_grid(anyDiff~Species) + stat_compare_means()
```

 genes with differential translation:  
 
```{r}
Ribo=read.table("../data/Wang_ribo/Additionaltable5_translationComparisons.txt",header = T, stringsAsFactors = F) %>% rename("Gene_stable_ID"= ENSG) %>% inner_join(nameID,by="Gene_stable_ID") %>% dplyr::select(Gene.name, HvC.beta, HvC.pvalue, HvC.FDR) %>% rename("gene"=Gene.name) %>% mutate(dTE=ifelse(HvC.FDR <0.05, "Yes","No"))
RiboSmall= Ribo %>% select(gene,dTE)

BothAU_fix_g_TE= BothAU_fix_g %>% inner_join(RiboSmall,by="gene")


ggplot(BothAU_fix_g_TE, aes(y=AU,x=dTE,fill=dTE)) +geom_boxplot() + facet_grid(anyDiff~Species) + stat_compare_means()
```
 
 AU for the ortho UTRs are not different based on DE or translation status.  
 
 
 filter to dAPA genes and see if they are different in expression and translation: 
 
```{r}
BothAU_fix_g_DE_dAPA= BothAU_fix_g_DE %>% filter(set=="dAPA")
ggplot(BothAU_fix_g_DE_dAPA, aes(y=AU,x=DE,fill=DE)) +geom_boxplot() + facet_grid(~Species) + stat_compare_means()


BothAU_fix_g_TE_dAPA= BothAU_fix_g_TE %>% filter(set=="dAPA")
ggplot(BothAU_fix_g_TE_dAPA, aes(y=AU,x=dTE,fill=dTE)) +geom_boxplot() + facet_grid(~Species) + stat_compare_means()
```


**ok so only significant relationship here is higher AU proportion in ortho 3' UTRs for dAPA genes. **


difference in dominance:  

```{r}
HumanRes=read.table("../data/DomDefGreaterX/Human_AllGenes_DiffTop.txt", col.names = c("Human_PAS", "gene","Human_DiffDom"),stringsAsFactors = F)

ChimpRes=read.table("../data/DomDefGreaterX/Chimp_AllGenes_DiffTop.txt", col.names = c("Chimp_PAS", "gene","Chimp_DiffDom"),stringsAsFactors = F)

BothRes=HumanRes %>% inner_join(ChimpRes,by="gene")

BothRes_10=BothRes %>% filter(Chimp_DiffDom >=0.1 | Human_DiffDom>=0.1) %>% mutate(Set= ifelse(Human_PAS==Chimp_PAS,"Same", "Different"),cut=10) 
BothRes_20=BothRes %>% filter(Chimp_DiffDom >=0.2 | Human_DiffDom>=0.2) %>% mutate(Set= ifelse(Human_PAS==Chimp_PAS,"Same", "Different"),cut=20)
BothRes_30=BothRes %>% filter(Chimp_DiffDom >=0.3 | Human_DiffDom>=0.3) %>% mutate(Set= ifelse(Human_PAS==Chimp_PAS,"Same", "Different"),cut=30)
BothRes_40=BothRes %>% filter(Chimp_DiffDom >=0.4 | Human_DiffDom>=0.4) %>% mutate(Set= ifelse(Human_PAS==Chimp_PAS,"Same", "Different"),cut=40)
BothRes_50=BothRes %>% filter(Chimp_DiffDom >=0.5 | Human_DiffDom>=0.5) %>% mutate(Set= ifelse(Human_PAS==Chimp_PAS,"Same", "Different"),cut=50)
BothRes_60=BothRes %>% filter(Chimp_DiffDom >=0.6 | Human_DiffDom>=0.6) %>% mutate(Set= ifelse(Human_PAS==Chimp_PAS,"Same", "Different"),cut=60)
BothRes_70=BothRes %>% filter(Chimp_DiffDom >=0.7 | Human_DiffDom>=0.7) %>% mutate(Set= ifelse(Human_PAS==Chimp_PAS,"Same", "Different"),cut=70)
BothRes_80=BothRes %>% filter(Chimp_DiffDom >=0.8 | Human_DiffDom>=0.8) %>% mutate(Set= ifelse(Human_PAS==Chimp_PAS,"Same", "Different"),cut=80)
BothRes_90=BothRes %>% filter(Chimp_DiffDom >=0.9 | Human_DiffDom>=0.9) %>% mutate(Set= ifelse(Human_PAS==Chimp_PAS,"Same", "Different"),cut=90)

BothResAll=BothRes_10 %>% bind_rows(BothRes_20) %>% bind_rows(BothRes_30) %>% bind_rows(BothRes_40) %>% bind_rows(BothRes_50) %>% bind_rows(BothRes_60) %>% bind_rows(BothRes_70) %>% bind_rows(BothRes_80) %>% bind_rows(BothRes_90) 

```


plotting au in human for each: 

```{r}
BothResAll_au= BothResAll %>% inner_join(BothAU_fix,by="gene")
BothResAll_au$cut=as.factor(BothResAll_au$cut)
ggplot(BothResAll_au,aes(x=cut, by=Set, y=HumanAU,fill=Set)) + geom_boxplot() + stat_compare_means(aes(label = ..p.signif..))
```


DP not DE:  

```{r}
ProtInfo=read.table("../data/PTM/ProtLength.txt", sep = "\t",stringsAsFactors = F,header = T,col.names = c("entry","organism", "nAA", "gene")) %>% select(nAA, gene)
Prot= read.table("../data/Khan_prot/ProtData_effectSize.txt",header = T,stringsAsFactors = F)  %>% mutate(dP=ifelse(pval<0.05, "Yes", "No")) %>% filter(dP=="Yes")
Interactions=read.table("../data/bioGRID/GeneswInteractions.txt",stringsAsFactors = F, header = T) %>% inner_join(ProtInfo, by="gene")%>% mutate(NormInter=nInt/nAA)

#DiffExp$gene, DT=Ribo$gene, DP=Prot$gene

dAPAandDic_wP=dAPAandDic %>% mutate(dE=ifelse(gene %in%DiffExp$gene, "Yes", "No" ), dP=ifelse(gene %in%Prot$gene,"Yes","No" ), dPnotDE=ifelse(dE=="No"&dP=="Yes", "Yes","No")) %>% inner_join(Interactions, by="gene") %>% inner_join(BothAU_fix,by="gene")
```


```{r}
ggplot(dAPAandDic_wP, aes(x=dPnotDE, y=HumanAU)) + geom_boxplot()
```
only one gene with the pattern and an ortho 3' UTR. 

Human 3' UTRs:  

Merge from refseq anno: 
```{bash,eval=F}
mkdir ../data/HumanUTR
bedtools merge -s -i ../data/orthoUTR/g38_ncbiRefseq_Formatted_Allannotation_UTR3.bed -c 4,5,6 -o distinct,mean,distinct > ../data/HumanUTR/Human3UTR.merged.bed
```

still multiple per genes.  take most 3' ? 

```{r}
humanMergeutr= read.table("../data/HumanUTR/Human3UTR.merged.bed", col.names = c('chr','start','end','gene','score','strand'), stringsAsFactors = F) %>% group_by(gene)

humanMergeutrpos= humanMergeutr %>% filter(strand=="+") %>% group_by(gene) %>% top_n(1,start)

humanMergeutrneg= humanMergeutr %>% filter(strand=="-") %>% group_by(gene) %>% top_n(-1,start)

humanDistalboth=humanMergeutrpos %>% bind_rows(humanMergeutrneg)

write.table(humanDistalboth, "../data/HumanUTR/HumanDistal3UTR.bed", col.names = F, row.names = F, quote = F, sep="\t")
```


Run these through AU:  

```{bash,eval=F}
python transcriptome_properties.py -i../../data/HumanUTR/HumanDistal3UTR.bed -g /project2/gilad/kenneth/References/human/genome/hg38.fa   --au-elements  -o ../../data/HumanUTR/HumanAllUTR_AUrich
```

```{r}

HumanALLUTR_au=read.csv("../data/HumanUTR/HumanAllUTR_AUrich_au_elements.csv",header = T, stringsAsFactors = F) %>%  separate(transcriptID, into=c("geneloc", "strand"), sep="\\(") %>% separate(geneloc, into=c("loc","gene"),sep=":") %>% na.omit()

```
missing pieces were overlapping.  

```{r}
dAPAandDic_wPAU=dAPAandDic %>% mutate(dE=ifelse(gene %in%DiffExp$gene, "Yes", "No" ), dP=ifelse(gene %in%Prot$gene,"Yes","No" ), dPnotDE=ifelse(dE=="No"&dP=="Yes", "Yes","No")) %>% inner_join(Interactions, by="gene") %>% inner_join(HumanALLUTR_au, by="gene")

dAPAandDic_wPAU$au_element_frac= as.numeric(as.character(dAPAandDic_wPAU$au_element_frac))
```

```{r}
ggplot(dAPAandDic_wPAU, aes(x=dPnotDE, y=au_element_frac)) + geom_boxplot()
```
```{r}
dAPAandDic_wPAU %>% group_by(dPnotDE) %>% summarise(n())
```

only 2...


```{r}
Allset_andnon_au= Allset_andnon %>% inner_join(HumanALLUTR_au, by="gene")
Allset_andnon_au$au_element_frac= as.numeric(as.character(Allset_andnon_au$au_element_frac))


ggplot(Allset_andnon_au, aes(x=set,fill=set,y=au_element_frac)) + geom_boxplot() + stat_compare_means()


Allset_andnon_au %>% group_by(set ) %>% summarise(mean(au_element_frac))
```

```{r}
Allset_andnon_aude=Allset_andnon_au %>% inner_join(DiffExp,by="gene")

ggplot(Allset_andnon_aude, aes(x=set,fill=set,y=au_element_frac,by=DE)) + geom_boxplot() + stat_compare_means()
```
```{r}
Allset_andnon_auTE=Allset_andnon_au  %>% inner_join(RiboSmall,by="gene")

ggplot(Allset_andnon_auTE, aes(x=set,fill=set,y=au_element_frac,by=dTE)) + geom_boxplot() + stat_compare_means()
```


ok there are no interesting relationships with this... I would have to proportion the UTRs probably... 


number:  
```{r}
Allset_andnon_au$au_element_count=as.numeric(Allset_andnon_au$au_element_count)

ggplot(Allset_andnon_au, aes(x=set,fill=set,y=au_element_count)) + geom_boxplot() + stat_compare_means() +ylim(c(0,100))
```
```{r}
Allset_andnon_auany= Allset_andnon_au %>% mutate(AnyAPA=ifelse(set=="None","No", "Yes"))

ggplot(Allset_andnon_auany, aes(x=AnyAPA,fill=AnyAPA,y=au_element_count)) + geom_boxplot() + stat_compare_means() +ylim(c(0,100))

ggplot(Allset_andnon_auany, aes(x=AnyAPA,fill=AnyAPA,y=au_element_frac)) + geom_boxplot() + stat_compare_means(method.args = list(alternative = "less")) 
```



```{r}
Allset_andnon_auany_de= Allset_andnon_auany %>% inner_join(DiffExp,by="gene")

ggplot(Allset_andnon_auany_de, aes(x=DE,fill=AnyAPA,y=au_element_count)) + geom_boxplot() + stat_compare_means() +ylim(c(0,100))
```
dAPA have more AU elements but the other relationships dont make much sense. nothing is strong.. on hold for now.



genes with 3' UTR difference: 

```{r}

MetaSm=Meta %>% select(PAS, chr, start, end, loc)
DiffIso=read.table("../data/DiffIso_Nuclear_DF/AllPAS_withGeneSig.txt",header = T, stringsAsFactors = F)  %>% inner_join(MetaSm, by=c("chr","start", "end")) 
DiffIsoSitUTR= DiffIso %>% filter(loc=="utr3", SigPAU2=="Yes")

DiffIsoSitUTRGenes=DiffIsoSitUTR %>% select(gene) %>% unique()


HumanALLUTR_au_UTRdiff= HumanALLUTR_au %>% mutate(dAPA=ifelse(gene %in%DiffIsoSitUTRGenes$gene, "Yes", "No" ))

HumanALLUTR_au_UTRdiff$au_element_frac=as.numeric(HumanALLUTR_au_UTRdiff$au_element_frac)
HumanALLUTR_au_UTRdiff$au_element_count=as.numeric(HumanALLUTR_au_UTRdiff$au_element_count)
```


```{r}
elementprop=ggplot(HumanALLUTR_au_UTRdiff, aes(x=dAPA, fill=dAPA, y=au_element_frac)) + geom_boxplot() + stat_compare_means() + scale_fill_brewer(palette = "Set1") + labs(title="Genes with 3' UTR differences \n have higher AU element %", y="Proportion of UTR in AU element", x="Gene with differentially used PAS") + theme_classic() + theme(legend.position = "none")

elementprop
HumanALLUTR_au_UTRdiff %>% group_by(dAPA) %>% summarise(mean(au_element_frac))
```
with density: 

```{r}

col=c("#eb825b", "#1e75bc")

ggplot(HumanALLUTR_au_UTRdiff, aes(x=dAPA, fill=dAPA, y=au_element_frac)) + geom_boxplot() + stat_compare_means() + scale_fill_brewer(palette = "Set1") + labs(title="Genes with 3' UTR differences have higher AU element %") + theme_classic() + theme(legend.position = "none")
ggplot(HumanALLUTR_au_UTRdiff, aes( fill=dAPA, x=au_element_frac)) + geom_density() + scale_fill_manual(values =col) + labs(title="Genes with 3' UTR differences have higher AU element %") + theme_classic() + theme(legend.position = "none")
```




Seperate by DE: 

```{r}
HumanALLUTR_au_UTRdiff_de= HumanALLUTR_au_UTRdiff %>% inner_join(DiffExp,by="gene")



ggplot(HumanALLUTR_au_UTRdiff_de, aes(by=dAPA, fill=dAPA, x=DE, y=au_element_frac)) + geom_boxplot() + stat_compare_means() + scale_fill_brewer(palette = "Set1")+ labs(title="AU % not associated with expression variation") + theme_classic() 
```


```{r}
ggplot(HumanALLUTR_au_UTRdiff, aes(x=dAPA, fill=dAPA, y=au_element_count)) + geom_boxplot() + stat_compare_means() + ylim(c(0, 100)) + scale_fill_brewer(palette = "Set1")+ labs(title="Genes with 3' UTR differences have more AU elements") + theme_classic() + theme(legend.position = "none")

meanDensity=ggplot(HumanALLUTR_au_UTRdiff, aes(x=dAPA, fill=dAPA, y=au_element_count)) + geom_boxplot() + stat_compare_means() + scale_fill_brewer(palette = "Set1")+ labs(title="Genes with 3' UTR differences have\n more AU elements", y="Number of 3' UTR AU elements", x="Gene with differentially used PAS") + theme_classic() + theme(legend.position = "none") 

col=c("#eb825b", "#1e75bc")

density_sm=ggplot(HumanALLUTR_au_UTRdiff, aes(fill=dAPA, x=au_element_count)) + geom_density(alpha=.5)  + scale_fill_manual(values =col )+ labs(title="") + theme_classic()+xlim(c(0, 50)) +theme(legend.position = "none", text=element_text(size=10),plot.title = element_text(hjust = 0.5, face="bold",size=12),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), "cm"))
density_sm

density=ggplot(HumanALLUTR_au_UTRdiff, aes(fill=dAPA, x=au_element_count)) + geom_density(alpha=.5)  + scale_fill_manual(values = col )+ labs(title="Genes with 3' UTR differences have more AU elements") + theme_classic() + theme(legend.position = "bottom", text=element_text(size=10),plot.title = element_text(hjust = 0.5, face="bold",size=12),axis.text.y = element_text(size=10),axis.text.x = element_text(size=10),plot.margin = unit(c(0,0,0,0), "cm"))
density
HumanALLUTR_au_UTRdiff %>% group_by(dAPA) %>% summarise(mean(au_element_count))
```



```{r}
pdf("../output/AUcount_density_sm.pdf",width = 4, height = 4)
density_sm
dev.off()
```

```{r}
pdf("../output/AUcount_density.pdf",width = 6, height = 6)
density
dev.off()
```

```{r}
ggplot(HumanALLUTR_au_UTRdiff_de, aes(by=dAPA, fill=dAPA, x=DE, y=au_element_count)) + geom_boxplot() + stat_compare_means() + scale_fill_brewer(palette = "Set1")+ labs(title="AU elements not associated with expression variation") + theme_classic() 
```

Supp. figure: 

```{r}
supplotAU=plot_grid(meanDensity,elementprop, labels = c("A","B"), scale = .9)
```

```{r}
pdf("../output/supplement/Fig3-figsup2-AU_new.pdf",width = 10, height = 6)
supplotAU
dev.off()
```
